<html class=" js flexbox canvas canvastext webgl no-touch geolocation postmessage no-websqldatabase indexeddb hashchange history draganddrop websockets rgba hsla multiplebgs backgroundsize borderimage borderradius boxshadow textshadow opacity cssanimations csscolumns cssgradients no-cssreflections csstransforms csstransforms3d csstransitions fontface generatedcontent video audio localstorage sessionstorage webworkers applicationcache svg inlinesvg smil svgclippaths js flexbox flexboxlegacy canvas canvastext webgl no-touch geolocation postmessage no-websqldatabase indexeddb hashchange history draganddrop websockets rgba hsla multiplebgs backgroundsize borderimage borderradius boxshadow textshadow opacity cssanimations csscolumns cssgradients no-cssreflections csstransforms csstransforms3d csstransitions fontface generatedcontent video audio localstorage sessionstorage webworkers applicationcache svg inlinesvg smil svgclippaths mediaqueries lastchild fullscreen cssremunit" style=""><!--<![endif]--><head><script type="text/javascript" async="async" src="https://stanfordmedicine.d2.sc.omtrdc.net/id?d_visid_ver=1.5.2&amp;callback=s_c_il%5B0%5D._setAnalyticsFields&amp;mcorgid=2C6958B4534410520A490D45%40AdobeOrg&amp;mid=65288271855724865442002099643359755654"></script><script type="text/javascript" async="async" src="https://dpm.demdex.net/id?d_visid_ver=1.5.2&amp;d_rtbd=json&amp;d_ver=2&amp;d_orgid=2C6958B4534410520A490D45%40AdobeOrg&amp;d_nsid=0&amp;d_cb=s_c_il%5B0%5D._setMarketingCloudFields"></script>
    <meta charset="utf-8">
    <title></title>
    <meta name="description" content="Julien&nbsp;Sage is on CAP Network. CAP Network is a virtual workspace, to support collaboration among faculty, graduate students, postdocs and staff.">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"><![endif]-->
    <!--[if IE 7]><link rel="stylesheet" href="/profiles/styles/font-awesome/scss/font-awesome-ie7.css;jsessionid=BC8E491FBE7A1060992D875F939A81E1.tc-cap-som-07"><![endif]-->
    <!--[if lte IE 8]><script src="/profiles/scripts/vendor/respond/respond.min.js;jsessionid=BC8E491FBE7A1060992D875F939A81E1.tc-cap-som-07"></script><![endif]-->
    <script src="/profiles/scripts/vendor/modernizr.min.js;jsessionid=BC8E491FBE7A1060992D875F939A81E1.tc-cap-som-07"></script>
    <meta name="robots" content="noindex">
    
    

      <!-- 
      dept = hesc
      baseUrl = https://med.stanford.edu
      appUrl = https://med.stanford.edu/profiles
      contextPath = 
      templatesUrl = https://med.stanford.edu/Templates/
      deptTemplatesUrl = https://med.stanford.edu/Templates/hesc/
      appTemplatesUrl = https://med.stanford.edu/Templates/profiles/
      somTemplatesUrl = https://med.stanford.edu/Templates/som_all_sm/
      deptIncludesUrl = https://med.stanford.edu/hesc/includes/
      shcSearchIncludesUrl = https://med.stanford.edu/hesc/search/includes/
      somIncludesUrl = https://med.stanford.edu/includes/
      findPhysicianIncludesUrl = https://med.stanford.edu/stanfordhospital/search/includes/
      prdadminUrl = https://med.stanford.edu/Templates/prdadmin/
      suffix = 
      isCancerSite = false
      isCancerResearcher = 
      isCancerPhysician = false
      isStanfordHospitalSite = false
      pageSpecificJs = 
      pageSpecificJs2 = 
      pageSpecificJsSig = 
      templatesIncludeURL = https://med.stanford.edu/Templates/profiles/includes/
      -->
       <!-- BEGIN HEAD CODE -->
<!--meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7" /-->
<meta http-equiv="X-UA-Compatible" content="chrome=1">
<meta http-equiv="content-type" content="text/html; charset=iso-8859-1">
<meta name="viewport" content="width=770">
<script type="text/javascript" src="/Templates/yui-2.7.0/build/yahoo-dom-event/yahoo-dom-event.js"></script>
<script type="text/javascript" src="/Templates/AC_RunActiveContent.js"></script>
<script type="text/javascript" src="/Templates/swfobject.js"></script>
<script type="text/javascript" src="/Templates/lib/moment/moment.js"></script>
<link rel="stylesheet" type="text/css" href="https://med.stanford.edu/etc/clientlibs/sm/custom/external.css">
<script type="text/javascript" src="https://med.stanford.edu/etc/clientlibs/sm/custom/external.js"></script>
<script type="text/javascript">
	var dataLayer = {
		"specVersion": "1.0.0",
		"site": {
			"name": "",
			"url": "",
			"platform": "Static",
			"locale": "",
			"google": {
				"analytics": ""
			}
		},
		"page": {
			"id": "Templates-hesc-headcode.htm",
			"title": "",
			"navTitle": "",
			"url": "http://med.stanford.edu/Templates/hesc/headcode.htm",
			"description": "",
			"keywords": [],
			"template": "Legacy",
			"publishedDate": "2012-08-10T14:03:01Z",
			"modifiedDate": "2012-08-10T14:03:01Z",
			"referrer": "",
			"content": {
				"title": ""
			}
		}							
	};
</script>

<!-- Use media="screen and (min-device-width: 481px)" to prevent uptake by iPhone -->
<link id="mainStyle" rel="stylesheet" type="text/css" href="/Templates/main_sm.css">
<link id="iPhoneStyle" media="only screen and (max-device-width: 480px)" href="/Templates/iPhone.css" type="text/css" rel="stylesheet">
<link rel="stylesheet" type="text/css" href="/Templates/hesc/styles.css">
<script type="text/javascript" src="/Templates/hesc/siteparams.js"></script>
<script type="text/javascript" src="/Templates/basic_functions_sm.js"></script>
<link rel="stylesheet" type="text/css" media="print" href="/Templates/print.css">
<script src="//assets.adobedtm.com/2b02a6b36f7b0182d4b9286f1307f16aa7d507a3/satelliteLib-31367a551e48d2f5c09eef9ae33fbefa973adbbb.js"></script><script src="https://assets.adobedtm.com/2b02a6b36f7b0182d4b9286f1307f16aa7d507a3/mbox-contents-6096c37e249258b28d4c020efef2602ca5a2c341.js"></script><script src="https://ssl.google-analytics.com/analytics.js"></script><style>.mboxDefault { visibility:hidden; }</style>
<!-- END HEAD CODE -->

     
    <link rel="stylesheet" type="text/css" href="/Templates/yui-2.7.0/build/container/assets/skins/sam/container.css">
    <script type="text/javascript" src="/Templates/yui-2.7.0/build/yahoo-dom-event/yahoo-dom-event.js"></script>
    <script type="text/javascript" src="/Templates/yui-2.7.0/build/container/container-min.js"></script>
    <script type="text/javascript" src="/Templates/yui-2.7.0/build/menu/menu-min.js"></script>
    <script type="text/javascript" src="/Templates/yui-2.7.0/build/json/json-min.js"></script>
    
    <script type="text/javascript" src="/Templates/yui-2.9.0/build/connection/connection-min.js"></script>
    

    
      <link rel="search" type="application/opensearchdescription+xml" title="Stanford CAP" href="/profiles/cap.v4/search.xml;jsessionid=BC8E491FBE7A1060992D875F939A81E1.tc-cap-som-07">
      <link rel="stylesheet" type="text/css" href="/Templates/profiles16/cap_components.css">
      
      <!-- Components for "loading..." interstitital attached to form submits -->
      <script type="text/javascript" src="/Templates/ui/interstitials/loading.js"></script>
      <script type="text/javascript" src="/Templates/ui/interstitials/genericSearchingMessage.js"></script>
      <!-- End "loading" components -->
    


    <link type="text/css" href="/profiles/cap.v4/wdgt.css;jsessionid=BC8E491FBE7A1060992D875F939A81E1.tc-cap-som-07" rel="stylesheet">
    <script type="text/javascript" src="/profiles/cap.v4/wdgt.js;jsessionid=BC8E491FBE7A1060992D875F939A81E1.tc-cap-som-07"></script>
    
    <link type="text/css" href="/profiles/cap.v4/searchDetails.css;jsessionid=BC8E491FBE7A1060992D875F939A81E1.tc-cap-som-07" rel="stylesheet">
    <script type="text/javascript">var mainContextPath = '';</script>
    <script type="text/javascript" src="/profiles/cap.v4/searchDetails.js;jsessionid=BC8E491FBE7A1060992D875F939A81E1.tc-cap-som-07"></script>

    <link rel="stylesheet" href="/profiles/styles/cap.v4.styles.css;jsessionid=BC8E491FBE7A1060992D875F939A81E1.tc-cap-som-07">
  <script type="text/javascript" charset="utf-8" async="" data-requirecontext="_" data-requiremodule="main" src="/profiles/scripts;jsessionid=BC8E491FBE7A1060992D875F939A81E1.tc-cap-som-07/main.js"></script><script src="https://assets.adobedtm.com/2b02a6b36f7b0182d4b9286f1307f16aa7d507a3/s-code-contents-2edb16d46900a24880139e9328bbb8c312264c89.js"></script><script src="https://assets.adobedtm.com/2b02a6b36f7b0182d4b9286f1307f16aa7d507a3/scripts/satellite-579fdd0964746d595d0004a4.js"></script><script type="text/javascript" charset="utf-8" async="" data-requirecontext="_" data-requiremodule="ga" src="//www.google-analytics.com/analytics.js"></script></head>
<!--[if lt IE 7]>  <body class="ie6 profile hesc"> <![endif]-->
<!--[if IE 7]>     <body class="ie7 profile hesc"> <![endif]-->
<!--[if IE 8]>     <body class="ie8 profile hesc"> <![endif]-->
<!--[if IE 9]>     <body class="ie9 profile hesc"> <![endif]-->
<!--[if !IE]><!--> <body class="profile hesc firefox noflash yui-skin-sam"><iframe id="_yuiResizeMonitor" title="Text Resize Monitor" style="position: absolute; visibility: visible; width: 2em; height: 2em; top: -25px; left: 0px; border-width: 0px;"></iframe><div class="yui-panel-container shadow yui-overlay-hidden" id="yuiLoadingPanel_c" style="visibility: hidden; left: 232px; top: 209px; z-index: 1000;"><div id="yuiLoadingPanel" class="yui-module yui-overlay yui-panel tcc_lightboxLoader" style="visibility: inherit; width: 240px;"><div class="bd" style="text-align: center;">One moment please, searching...<img src="/Templates/images/rel_interstitial_loading.gif"><br><a style="cursor: pointer;">Cancel</a></div></div></div> <!--<![endif]-->
    <div id="container">
      <header class="navbar " id="headerContent">
        


        
        
          
          
          <!-- BEGIN TOPCODE -->
 


<div id="brand" class="x_logo">
  <a href="http://med.stanford.edu/"><img src="/Templates/images/sm-logo.png" alt="Stanford School of Medicine" width="200" height="50"></a>
  <img src="/Templates/images/sm-logo.png" alt="Stanford School of Medicine" class="printLogo" width="1" height="1">
</div>
<div id="brandedSiteName">
  <a href="http://hesc.stanford.edu/">HESC<span class="nodisplay">â€“</span>
  <span id="brandedSiteNameSubtitle">Center for Human Embryonic Stem Cell<br>Research and Education</span></a>
</div>

<!-- SMSTAGING Stanford Medicine Version 1.0 6/20/2008: -->
<div id="search">
  <form action="//med.stanford.edu/searchsm/" method="get" name="searchform" id="searchform">
    <input id="numgm" name="numgm" value="0" type="hidden">
    <input id="searchid" name="searchid" value="" type="hidden">
    <div id="searchBox">
      <input id="q" name="q" size="12" value="Search Stanford Medical Sites" class="" type="text">
      <button class="searchSubmitButton" type="submit">Search</button>
    </div>
    <div id="searchExtras"><!-- optional links, search controls, etc. -->
      <!-- as_oq value reflects site or sites to be searched on first button -->
      <div id="searchControls">
        <ul id="searchControlsList">
          <li id="thisSiteOnly"><input name="as_oq" id="as_oq_1" value="site:hesc.stanford.edu" type="radio"><label for="as_oq_1" id="as_oq_1_label">This Site Only</label></li>
          <li id="allStanfordMedicine"><input name="as_oq" id="as_oq_2" value="" checked="checked" type="radio"><label for="as_oq_2" id="as_oq_2_label">Stanford Medical Sites</label></li>
        </ul>
      </div>
      <div id="searchLinks">
        <ul id="searchLinksList">
          <li id="waysToGive" class="x_firstChild"><a href="http://med.stanford.edu/development/">Ways to Give</a></li>
          <li id="findAPerson"><a href="http://med.stanford.edu/contacts/">Find a Person</a></li>
        </ul>
      </div>
      <div id="searchLinksLong">
        <ul id="searchLinksListLong">
          <li class="x_firstChild"><a href="http://med.stanford.edu/alumni/">Alumni</a></li>
          <li><a href="http://lane.stanford.edu/">Lane Library</a></li>
          <li id="waysToGiveLong"><a href="http://med.stanford.edu/development/">Ways to Give</a></li>
          <li id="findAPersonLong"><a href="http://med.stanford.edu/contacts/">Find a Person</a></li>
        </ul>
      </div>
    </div>
  </form>
</div>

<!-- END TOPCODE -->


          
        
         
      </header>

      <div id="mainContent">
        <section id="mastheadContainer" class="banner profile">
  <div class="container">
  <div class="row-fluid">
    <div class="span12">
      <div class="image-holder">
        <img src="https://cap.stanford.edu/profiles/viewImage?profileId=6009&amp;type=square" alt="Julien&nbsp;Sage" data-empty-src="/profiles/images/profile-large-blank.png">
      </div>
      <div class="nameAndTitle">
        <h1>Julien&nbsp;Sage</h1>
        <h2>Professor of Pediatrics (Hematology/Oncology) and of Genetics</h2>
          <h3>Pediatrics - Hematology &amp; Oncology</h3>
      </div>
      <ul class="unstyled">
          <li>
            <a href="https://cap.stanford.edu/profiles/frdActionServlet?choiceId=printerprofile&amp;profileversion=full&amp;profileId=6009" target="_blank" rel="nofollow">
              <span class="icon-background">
                <i class="icon-print"></i>
              </span>Print Profile
            </a>
          </li>
        <li>
          <a href="mailto:?subject=Julien&nbsp;Sage's Profile&amp;body=I would like to share the profile of Julien&nbsp;Sage with you. https://profiles.stanford.edu/julien-sage" rel="nofollow" data-bypass="">
            <span class="icon-background">
              <i class="icon-envelope-alt"></i>
            </span>Email Profile
          </a>
        </li>
      </ul>
    </div>
  </div>
</div>
</section>
<section id="tabbedLayoutContainer">
  <nav class="navbar">
  <div class="navbar-inner">
    <div class="container">
     
      <!-- .btn-navbar is used as the toggle for collapsed navbar content -->
      <a class="btn btn-navbar" data-toggle="collapse" data-target="#tabbedLayoutContainer .nav-collapse">
        <span class="nav-label">Profile Tabs Menu</span>
        <i class="icon-chevron-down"></i>
      </a>

      <div class="nav-collapse collapse">
        <ul class="nav nav-tabs tabs4">
            <li class="active">
              <a id="bioTabLink" href="/profiles/hesc/julien-sage?tab=bio" class="no-trigger" rel="nofollow">Bio</a>
            </li>
            <li class="">
              <a id="researchAndScholarshipTabLink" href="/profiles/hesc/julien-sage?tab=research-and-scholarship" class="no-trigger" rel="nofollow">Research &amp; Scholarship</a>
            </li>
            <li class="">
              <a id="teachingTabLink" href="/profiles/hesc/julien-sage?tab=teaching" class="no-trigger" rel="nofollow">Teaching</a>
            </li>
            <li class="">
              <a id="publicationsTabLink" href="/profiles/hesc/julien-sage?tab=publications" class="no-trigger" rel="nofollow">Publications</a>
            </li>
        </ul>
      </div>
     
    </div>
  </div>
</nav>

<div class="container breakout-phone">
  <div class="tab-content">
      <div id="bio" class="card tab-pane active">
        <h2>Bio</h2>

<div class="row-fluid">
  <div class="span8 page-content">
 
       
    
    
      <div id="academicAppointmentsContent" class="content-section">
        <h3>Academic Appointments</h3>
<hr>
<ul class="section-listing">
    <li class="section-list-item">
  <div class="description">Professor, <a href="http://pedhemonc.stanford.edu/"> Pediatrics - Hematology &amp; Oncology</a></div>
</li>
    <li class="section-list-item">
  <div class="description">Professor, <a href="http://genetics.stanford.edu/"> Genetics</a></div>
</li>
    <li class="section-list-item">
  <div class="description">Member, <a href="http://biox.stanford.edu/"> Bio-X</a></div>
</li>
    <li class="section-list-item">
  <div class="description">Member, <a href="http://chri.stanford.edu/"> Child Health Research Institute</a></div>
</li>
    <li class="section-list-item">
  <div class="description">Member, <a href="http://cancer.stanford.edu"> Stanford Cancer Institute</a></div>
</li>
    <li class="section-list-item">
  <div class="description">Faculty Fellow, <a href="https://chemh.stanford.edu/"> Stanford ChEM-H</a></div>
</li>
</ul>
      </div>
    
      <div id="administrativeAppointmentsContent" class="content-section">
        <h3>Administrative Appointments</h3>
<hr>
<ul class="section-listing">
    <li class="section-list-item">
  <div class="description">Member, Institute for Stem Cell Biology and Regenerative Medicine (2006 - Present)</div>
</li>
    <li class="section-list-item">
  <div class="description">co-Director, Cancer Biology PhD program (2016 - Present)</div>
</li>
</ul>
      </div>
    
    
      <div id="honorsAndAwardsContent" class="content-section">
        <h3>Honors &amp; Awards</h3>
<hr>
<ul class="section-listing">
    <li class="section-list-item">
  <div class="description">Scholar Award, Damon Runyon Cancer Research Foundation (2005-2008)</div>
</li>
    <li class="section-list-item">
  <div class="description">Scholar Award, Leukemia and Lymphoma Society (2009-2014)</div>
</li>
    <li class="section-list-item">
  <div class="description">Morgridge Faculty Scholar, Lucille Packard Foundation for Children's Health (2008-2013)</div>
</li>
    <li class="section-list-item">
  <div class="description">Harriet and Mary Zelencik Scientist in Children's Cancer and Blood Diseases, Lucille Packard Foundation for Children's Health (Since 2013)</div>
</li>
</ul>
      </div>
    
    
    
      <div id="professionalEducationContent" class="content-section">
        <h3>Professional Education</h3>
<hr>
<ul class="section-listing">
    <li class="section-list-item">
  <div class="description">B.S., Ecole Normale Superieure, France, Biology  (1993)</div>
</li>
    <li class="section-list-item">
  <div class="description">Ph.D., Nice University, France, Biology  (1998)</div>
</li>
</ul>
      </div>
    
    
    
    
    
    
    
    
  </div>
  <div class="span4 side-bar">
      <div id="contactInfoContent" class="content-section contact-info bg-icon-user">
        <h3>Contact</h3>
<hr>
<ul class="unstyled section-listing use-link-icons">
      <li>
        <div class="contact-info primary">
            <label>Academic</label>
          <a href="mailto:julsage@stanford.edu" class="email" rel="nofollow" data-bypass="">julsage@stanford.edu</a>
          <ul class="unstyled phone-number">
          </ul>
        </div>
      </li>
    <li>
      <div class="contact-info alternate">
        <label>Alternate Contact</label>
        <span class="name">Alyssa Ray</span>
        <span class="title">Research Administrator</span>
        <a href="mailto:alyssar@stanford.edu" class="email" rel="nofollow" data-bypass="">alyssar@stanford.edu</a>
        <ul class="unstyled phone-number">
            <li><span>Tel: (650) 724-9246</span></li>
        </ul>
      </div>
    </li>
</ul>

      </div>
    
      <div id="linksContent" class="content-section links bg-icon-link">
        <h3>Links</h3>
<hr>
<ul class="unstyled section-listing use-link-icons">
    <li class="external-link"><a href="http://med.stanford.edu/sage" class="Lab Site" target="_blank" data-bypass="">the Sage lab</a></li>
</ul>
      </div>
  </div>
</div>
      </div>
      <div id="research-and-scholarship" class="card tab-pane ">
        <h2>Research &amp; Scholarship</h2>

<div class="row-fluid">
  <div class="span8 page-content">
      <div id="currentResearchAndScholarlyInterestsContent" class="content-section">
        <h3>Current Research and Scholarly Interests</h3>
<hr>
<p>Our research program focuses on the mechanisms that control the proliferation of mammalian cells under normal and pathological conditions (regeneration, cancer), with a particular emphasis on stem cells and gene regulatory networks. We combine genetic, genomics, and proteomics approaches to identify and investigate genes and pathways involved in cancer initiation and progression. We use genome-editing strategies to develop and study genetically-engineered mouse models for human cancers, including lung cancer, pancreatic cancer, and liver cancer. Our work spans the investigation of fundamental biological processes to the implementation of clinical trials based on our findings in pre-clinical models.</p>
      </div>
    
    
      <div id="clinicalTrialsContentOnResearchAndScholarship" class="content-section">
        <h3>Clinical Trials</h3>
<hr>
<ul class="section-listing">
    <li class="clinical-trial">
  <span class="title">Cancer Biology of Retinoblastoma</span>
    <strong class="recruiting-status">Not Recruiting</strong>
  <div class="more-info detail">
    <p>
      Many children with the childhood cancer, Retinoblastoma, have surgery to remove the tumor
      and sometimes the entire eye. The purpose of this study is to collect the extra tissue from
      patients who undergo tumor removal for laboratory experiments that will help us understand
      not only what occurs in retinoblastoma cells but also how cells normally function. Some of
      these studies will include an evaluation of how cells control the way that genes are
      expressed, how cells "know" to become retinal cells, how cells remain retinal cells, how
      cells lose their identity as retinal cells, what changes make retinoblastoma cells different
      from normal retinal cells, and what changes make some retinoblastomas worse than others.
    </p>
      <p>
        <strong>Stanford is currently not accepting patients for this trial.</strong>
          <span>For more information, please contact   Julien Sage, 650-723-5113.
      </span></p>
    <div class="collaborators"></div>
    <div class="investigators"></div>
    <p style="margin-top: 1em;">
      <a href="https://med.stanford.edu/clinicaltrials/trials/NCT01642823" target="_blank"><i class="icon-external-link"></i> View full details</a>
    </p>
  </div>
</li>
    <li class="clinical-trial">
  <span class="title">Phase 2a Desipramine in Small Cell Lung Cancer and Other High-Grade Neuroendocrine Tumors</span>
    <strong class="recruiting-status">Not Recruiting</strong>
  <div class="more-info detail">
    <p>
      Intrapatient dose escalation study of desipramine in subjects with small cell lung cancer
      (SCLC) and other high-grade neuroendocrine tumors.
    </p>
      <p>
        <strong>Stanford is currently not accepting patients for this trial.</strong>
          <span>For more information, please contact   CCTO, 650-498-7061.
      </span></p>
    <div class="collaborators"></div>
    <div class="investigators"></div>
    <p style="margin-top: 1em;">
      <a href="https://med.stanford.edu/clinicaltrials/trials/NCT01719861" target="_blank"><i class="icon-external-link"></i> View full details</a>
    </p>
  </div>
</li>
</ul>
      </div>
    
    
    
    
  </div>
</div>
      </div>
      <div id="teaching" class="card tab-pane ">
        <h2>Teaching</h2>

<div class="row-fluid">
  <div class="span8 page-content">
      <div id="coursesContent" class="content-section">
        <h3>2016-17 Courses</h3>
<hr>
<ul class="section-listing courses">
    <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=CBIO 245: Lecture Seminar Series in Cancer Biology Program&amp;academicYear=20162017">Lecture Seminar Series in Cancer Biology Program</a><br><span>CBIO 245 (Aut, Win, Spr)</span></li>
    <li class="unstyled">
      <a class="independent-studies-label">Independent Studies (13)</a>
      <ul class="section-listing independent-studies">
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=CBIO 299: Directed Reading in Cancer Biology&amp;academicYear=20162017">Directed Reading in Cancer Biology</a><br><span>CBIO 299 (Aut, Win, Spr, Sum)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=GENE 299: Directed Reading in Genetics&amp;academicYear=20162017">Directed Reading in Genetics</a><br><span>GENE 299 (Win, Spr, Sum)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=PEDS 299: Directed Reading in Pediatrics&amp;academicYear=20162017">Directed Reading in Pediatrics</a><br><span>PEDS 299 (Aut, Win, Spr, Sum)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=PEDS 280: Early Clinical Experience&amp;academicYear=20162017">Early Clinical Experience</a><br><span>PEDS 280 (Aut, Win, Spr, Sum)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=CBIO 399: Graduate Research&amp;academicYear=20162017">Graduate Research</a><br><span>CBIO 399 (Aut, Win, Spr, Sum)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=GENE 399: Graduate Research&amp;academicYear=20162017">Graduate Research</a><br><span>GENE 399 (Win, Spr, Sum)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=PEDS 399: Graduate Research&amp;academicYear=20162017">Graduate Research</a><br><span>PEDS 399 (Aut, Win, Spr, Sum)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=GENE 370: Medical Scholars Research&amp;academicYear=20162017">Medical Scholars Research</a><br><span>GENE 370 (Aut, Win, Spr, Sum)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=PEDS 370: Medical Scholars Research&amp;academicYear=20162017">Medical Scholars Research</a><br><span>PEDS 370 (Aut, Win, Spr, Sum)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=GENE 260: Supervised Study&amp;academicYear=20162017">Supervised Study</a><br><span>GENE 260 (Win, Spr, Sum)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=CBIO 260: Teaching in Cancer Biology&amp;academicYear=20162017">Teaching in Cancer Biology</a><br><span>CBIO 260 (Spr)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=PEDS 199: Undergraduate Directed Reading/Research&amp;academicYear=20162017">Undergraduate Directed Reading/Research</a><br><span>PEDS 199 (Aut, Win, Spr, Sum)</span></li>
          <li class="course"><a href="http://explorecourses.stanford.edu/search?view=catalog&amp;filter-coursestatus-Active=on&amp;q=GENE 199: Undergraduate Research&amp;academicYear=20162017">Undergraduate Research</a><br><span>GENE 199 (Aut, Win, Spr, Sum)</span></li>
      </ul>
    </li>
  </ul>
      </div>
    
      <div id="stanfordAdviseesContent" class="content-section">
        <h3>Stanford Advisees</h3>
<hr>
<ul class="section-listing">
    <li class="advisees-list-item">
	  <span style="font-weight:600">Doctoral Dissertation Reader (AC)</span>
		<br>
		<span style="display:block" data-toggle="collapsePartial"><a href="https://profiles.stanford.edu/kevin-meng">Kevin Meng</a>, <a href="https://profiles.stanford.edu/youcef-ouadah">Youcef Ouadah</a>, <a href="https://profiles.stanford.edu/elizabeth-qin">Elizabeth Qin</a>, <a href="https://profiles.stanford.edu/miranda-stratton">Miranda Stratton</a>, <a href="https://profiles.stanford.edu/ian-winters">Ian Winters</a>, <a href="https://profiles.stanford.edu/nora-yucel">Nora Yucel</a></span>
</li>

    <li class="advisees-list-item">
	  <span style="font-weight:600">Postdoctoral Faculty Sponsor</span>
		<br>
		<span style="display:block" data-toggle="collapsePartial"><a href="https://profiles.stanford.edu/julia-arand">Julia Arand</a>, <a href="https://profiles.stanford.edu/garry-coles">Garry Coles</a>, <a href="https://profiles.stanford.edu/fangfei-qu">Fangfei Qu</a>, <a href="https://profiles.stanford.edu/jamie-webster">Jamie Webster</a></span>
</li>

    <li class="advisees-list-item">
	  <span style="font-weight:600">Doctoral Dissertation Advisor (AC)</span>
		<br>
		<span style="display:block" data-toggle="collapsePartial"><a href="https://profiles.stanford.edu/dian-yang">Dian Yang</a></span>
</li>

</ul>
      </div>
    
      <div id="graduateAndFellowshipProgramsContent" class="content-section">
        <h3>Graduate and Fellowship Programs</h3>
<hr>
<ul class="section-listing">
    <li class="section-list-item">
  <div class="description"><a href="http://medcatalog.stanford.edu/php/dept.php?DNo=730">Cancer Biology (Phd Program)</a></div>
</li>
    <li class="section-list-item">
  <div class="description"><a href="http://medcatalog.stanford.edu/php/dept.php?DNo=770">Genetics (Phd Program)</a></div>
</li>
    <li class="section-list-item">
  <div class="description">Hematology (Fellowship Program)</div>
</li>
    <li class="section-list-item">
  <div class="description">Pediatric Hem/Onc (Fellowship Program)</div>
</li>
    <li class="section-list-item">
  <div class="description"><a href="http://stemcell.stanford.edu/">Stem Cell Biology and Regenerative Medicine (Phd Program)</a></div>
</li>
</ul>
      </div>
  </div>
</div>
      </div>
      <div id="publications" class="card tab-pane ">
        <h2>Publications</h2>

<div class="row-fluid">
  <div class="span8 page-content">
    <div id="featuredPublicationsContent"></div>
    <div id="allPublicationsContent">
            <h3>All Publications</h3>
<hr>
<ul class="section-listing articles" data-count="85">
    <li class="publication  article" id="pubId-438256">
  <cite><span class="title">
  <span>Chemosensitive Relapse in Small Cell Lung Cancer Proceeds through an EZH2-SLFN11 Axis.</span>
  <i>Cancer cell</i>
</span>
<span class="authors">Gardner, E. E., Lok, B. H., Schneeberger, V. E., Desmeules, P., Miles, L. A., Arnold, P. K., Ni, A., Khodos, I., de Stanchina, E., Nguyen, T., Sage, J., Campbell, J. E., Ribich, S., Rekhtman, N., Dowlati, A., Massion, P. P., Rudin, C. M., Poirier, J. T.</span>

<span class="details">
   <span class="year">2017</span>; <span class="volume">31 (2)</span><span class="pages">: 286-299</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Small cell lung cancer is initially highly responsive to cisplatin and etoposide but in almost every case becomes rapidly chemoresistant, leading to death within 1 year. We modeled acquired chemoresistance in&nbsp;vivo using a series of patient-derived xenografts to generate paired chemosensitive and chemoresistant cancers. Multiple chemoresistant models demonstrated suppression of SLFN11, a factor implicated in DNA-damage repair deficiency. In&nbsp;vivo silencing of SLFN11 was associated with marked deposition of H3K27me3, a histone modification placed by EZH2, within the gene body of SLFN11, inducing local chromatin condensation and gene silencing. Inclusion of an EZH2 inhibitor with standard cytotoxic therapies prevented emergence of acquired resistance and augmented chemotherapeutic efficacy in both chemosensitive and chemoresistant models of small cell lung cancer.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1016/j.ccell.2017.01.006">DOI 10.1016/j.ccell.2017.01.006</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/28196596">PubMedID 28196596</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-426898">
  <cite><span class="title">
  <span>Novel functions for the transcription factor E2F4 in development and disease.</span>
  <i>Cell cycle </i>
</span>
<span class="authors">Hsu, J., Sage, J.</span>

<span class="details">
   <span class="year">2016</span><span class="pages">: 1-8</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The E2F family of transcription factors is a key determinant of cell proliferation in response to extra- and intra-cellular signals. Within this family, E2F4 is a transcriptional repressor whose activity is critical to engage and maintain cell cycle arrest in G0/G1 in conjunction with members of the retinoblastoma (RB) family. However, recent observations challenge this paradigm and indicate that E2F4 has a multitude of functions in cells besides this cell cycle regulatory role, including in embryonic and adult stem cells, during regenerative processes, and in cancer. Some of these new functions are independent of the RB family and involve direct activation of target genes. Here we review the canonical functions of E2F4 and discuss recent evidence expanding the role of this transcription factor, with a focus on cell fate decisions in tissue homeostasis and regeneration.</p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/27753528">PubMedID 27753528</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5176148">PubMedCentralID PMC5176148</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-421260">
  <cite><span class="title">
  <span>Essential role for the planarian intestinal GATA transcription factor in stem cells and regeneration.</span>
  <i>Developmental biology</i>
</span>
<span class="authors">Flores, N. M., Oviedo, N. J., Sage, J.</span>

<span class="details">
   <span class="year">2016</span>; <span class="volume">418 (1)</span><span class="pages">: 179-188</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The cellular turnover of adult tissues and injury-induced repair proceed through an exquisite integration of proliferation, differentiation, and survival signals that involve stem/progenitor cell populations, their progeny, and differentiated tissues. GATA factors are DNA binding proteins that control stem cells and the development of tissues by activating or repressing transcription. Here we examined the role of GATA transcription factors in Schmidtea mediterranea, a freshwater planarian that provides an excellent model to investigate gene function in adult stem cells, regeneration, and differentiation. Smed-gata4/5/6, the homolog of the three mammalian GATA-4,-5,-6 factors is expressed at high levels in differentiated gut cells but also at lower levels in neoblast populations, the planarian stem cells. Smed-gata4/5/6 knock-down results in broad differentiation defects, especially in response to injury. These defects are not restricted to the intestinal lineage. In particular, at late time points during the response to injury, loss of Smed-gata4/5/6 leads to decreased neoblast proliferation and to gene expression changes in several neoblast subpopulations. Thus, Smed-gata4/5/6 plays a key evolutionary conserved role in intestinal differentiation in planarians. These data further support a model in which defects in the intestinal lineage can indirectly affect other differentiation pathways in planarians.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1016/j.ydbio.2016.08.015">DOI 10.1016/j.ydbio.2016.08.015</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/27542689">PubMedID 27542689</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-417630">
  <cite><span class="title">
  <span>Is the Canonical RAF/MEK/ERK Signaling Pathway a&nbsp;Therapeutic Target in SCLC?</span>
  <i>Journal of thoracic oncology </i>
</span>
<span class="authors">Cristea, S., Sage, J.</span>

<span class="details">
   <span class="year">2016</span>; <span class="volume">11 (8)</span><span class="pages">: 1233-1241</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The activity of the RAF/MEK/ERK signaling pathway is critical for the proliferation of normal and cancerous cells. Oncogenic mutations driving the development of lung adenocarcinoma often activate this signaling pathway. In contrast, pathway activity levels and their biological roles are not well established in small cell lung cancer (SCLC), a fast-growing neuroendocrine lung cancer subtype. Here we discuss the function of the RAF/MEK/ERK kinase pathway and the mechanisms leading to its activation in SCLC cells. In particular, we argue that activation of this pathway may be beneficial to the survival, proliferation, and spread of SCLC cells in response to multiple stimuli. We also consider evidence that high levels of RAF/MEK/ERK pathway activity may be detrimental to SCLC tumors, including in part by&nbsp;interfering with their neuroendocrine fate. On the basis of these observations, we examined when small molecules targeting kinases in the RAF/MEK/ERK pathway may be useful therapeutically in patients with SCLC, including in combination with other therapeutic agents.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1016/j.jtho.2016.04.018">DOI 10.1016/j.jtho.2016.04.018</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/27133774">PubMedID 27133774</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5391975">PubMedCentralID PMC5391975</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-416504">
  <cite><span class="title">
  <span>Identification and Targeting of Long-Term Tumor-Propagating Cells in Small Cell Lung Cancer.</span>
  <i>Cell reports</i>
</span>
<span class="authors">Jahchan, N. S., Lim, J. S., Bola, B., Morris, K., Seitz, G., Tran, K. Q., Xu, L., Trapani, F., Morrow, C. J., Cristea, S., Coles, G. L., Yang, D., Vaka, D., Kareta, M. S., George, J., Mazur, P. K., Nguyen, T., Anderson, W. C., Dylla, S. J., Blackhall, F., Peifer, M., Dive, C., Sage, J.</span>

<span class="details">
   <span class="year">2016</span>; <span class="volume">16 (3)</span><span class="pages">: 644-656</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Small cell lung cancer (SCLC) is a neuroendocrine lung cancer characterized by fast growth, early dissemination, and rapid resistance to chemotherapy. We identified a population of long-term tumor-propagating cells (TPCs) in a mouse model of SCLC. This population, marked by high levels of EpCAM and CD24, is also prevalent in human primary SCLC tumors. Murine SCLC TPCs are numerous and highly proliferative but not intrinsically chemoresistant, indicating that not all clinical features of SCLC&nbsp;are linked to TPCs. SCLC TPCs possess a distinct transcriptional profile compared to non-TPCs, including elevated MYC activity. Genetic and pharmacological inhibition of MYC in SCLC cells to non-TPC levels inhibits long-term propagation but not short-term growth. These studies identify a highly tumorigenic population of SCLC cells in mouse models, cell lines, and patient tumors and a means to target them in this most fatal form of lung cancer.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1016/j.celrep.2016.06.021">DOI 10.1016/j.celrep.2016.06.021</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/27373157">PubMedID 27373157</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4956576">PubMedCentralID PMC4956576</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-413068">
  <cite><span class="title">
  <span>Nfib Promotes Metastasis through a Widespread Increase in Chromatin Accessibility</span>
  <i>CELL</i>
</span>
<span class="authors">Denny, S. K., Yang, D., Chuang, C., Brady, J. J., Lim, J. S., Gruner, B. M., Chiou, S., Schep, A. N., Baral, J., Hamard, C., Antoine, M., Wislez, M., Kong, C. S., Connolly, A. J., Park, K., Sage, J., Greenleaf, W. J., Winslow, M. M.</span>

<span class="details">
   <span class="year">2016</span>; <span class="volume">166 (2)</span><span class="pages">: 328-342</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Metastases are the main cause of cancer deaths, but the mechanisms underlying metastatic progression remain poorly understood. We isolated pure populations of cancer cells from primary tumors and metastases from a genetically engineered mouse model of human small cell lung cancer (SCLC) to investigate the mechanisms that drive the metastatic spread of&nbsp;this lethal cancer. Genome-wide characterization of chromatin accessibility revealed the opening of large numbers of distal regulatory elements across the genome during metastatic progression. These changes correlate with copy number amplification of the Nfib locus, and differentially accessible sites were highly enriched for Nfib transcription factor binding sites. Nfib is necessary and sufficient to increase chromatin accessibility at a large subset of the intergenic regions. Nfib promotes pro-metastatic neuronal gene expression programs and drives the metastatic ability of SCLC cells. The identification of widespread chromatin changes during SCLC progression reveals an unexpected global reprogramming during metastatic progression.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1016/j.cell.2016.05.052">DOI 10.1016/j.cell.2016.05.052</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000380255400012">Web of Science ID 000380255400012</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/27374332">PubMedID 27374332</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5004630">PubMedCentralID PMC5004630</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-416393">
  <cite><span class="title">
  <span>CD47-blocking immunotherapies stimulate macrophage-mediated destruction of small-cell lung cancer</span>
  <i>JOURNAL OF CLINICAL INVESTIGATION</i>
</span>
<span class="authors">Weiskopf, K., Jahchan, N. S., Schnorr, P. J., Cristea, S., Ring, A. M., Maute, R. L., Volkmer, A. K., Volkmer, J., Liu, J., Lim, J. S., Yang, D., Seitz, G., Thuyen Nguyen, T., Wu, D., Jude, K., Guerston, H., Barkal, A., Trapani, F., George, J., Poirier, J. T., Gardner, E. E., Miles, L. A., de Stanchina, E., Lofgren, S. M., Vogel, H., Winslow, M. M., Dive, C., Thomas, R. K., Rudin, C. M., van de Rijn, M., Majeti, R., Garcia, K. C., Weissman, I. L., Sage, J.</span>

<span class="details">
   <span class="year">2016</span>; <span class="volume">126 (7)</span><span class="pages">: 2610-2620</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Small-cell lung cancer (SCLC) is a highly aggressive subtype of lung cancer with limited treatment options. CD47 is a cell-surface molecule that promotes immune evasion by engaging signal-regulatory protein alpha (SIRP?), which serves as an inhibitory receptor on macrophages. Here, we found that CD47 is highly expressed on the surface of human SCLC cells; therefore, we investigated CD47-blocking immunotherapies as a potential approach for SCLC treatment. Disruption of the interaction of CD47 with SIRP? using anti-CD47 antibodies induced macrophage-mediated phagocytosis of human SCLC patient cells in culture. In a murine model, administration of CD47-blocking antibodies or targeted inactivation of the Cd47 gene markedly inhibited SCLC tumor growth. Furthermore, using comprehensive antibody arrays, we identified several possible therapeutic targets on the surface of SCLC cells. Antibodies to these targets, including CD56/neural cell adhesion molecule (NCAM), promoted phagocytosis in human SCLC cell lines that was enhanced when combined with CD47-blocking therapies. In light of recent clinical trials for CD47-blocking therapies in cancer treatment, these findings identify disruption of the CD47/SIRP? axis as a potential immunotherapeutic strategy for SCLC. This approach could enable personalized immunotherapeutic regimens in patients with SCLC and other cancers.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1172/JCI81603">DOI 10.1172/JCI81603</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000379094800024">Web of Science ID 000379094800024</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/27294525">PubMedID 27294525</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4922696">PubMedCentralID PMC4922696</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-403318">
  <cite><span class="title">
  <span>Identification of tumorigenic cells and therapeutic targets in pancreatic neuroendocrine tumors</span>
  <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i>
</span>
<span class="authors">Krampitz, G. W., George, B. M., Willingham, S. B., Volkmer, J., Weiskopf, K., Jahchan, N., Newman, A. M., Sahoo, D., Zemek, A. J., Yanovsky, R. L., Nguyen, J. K., Schnorr, P. J., Mazur, P. K., Sage, J., Longacre, T. A., Visser, B. C., Poultsides, G. A., Norton, J. A., Weissman, I. L.</span>

<span class="details">
   <span class="year">2016</span>; <span class="volume">113 (16)</span><span class="pages">: 4464-4469</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Pancreatic neuroendocrine tumors (PanNETs) are a type of pancreatic cancer with limited therapeutic options. Consequently, most patients with advanced disease die from tumor progression. Current evidence indicates that a subset of cancer cells is responsible for tumor development, metastasis, and recurrence, and targeting these tumor-initiating cells is necessary to eradicate tumors. However, tumor-initiating cells and the biological processes that promote pathogenesis remain largely uncharacterized in PanNETs. Here we profile primary and metastatic tumors from an index patient and demonstrate that MET proto-oncogene activation is important for tumor growth in PanNET xenograft models. We identify a highly tumorigenic cell population within several independent surgically acquired PanNETs characterized by increased cell-surface protein CD90 expression and aldehyde dehydrogenase A1 (ALDHA1) activity, and provide in vitro and in vivo evidence for their stem-like properties. We performed proteomic profiling of 332 antigens in two cell lines and four primary tumors, and showed that CD47, a cell-surface protein that acts as a "don't eat me" signal co-opted by cancers to evade innate immune surveillance, is ubiquitously expressed. Moreover, CD47 coexpresses with MET and is enriched in CD90(hi)cells. Furthermore, blocking CD47 signaling promotes engulfment of tumor cells by macrophages in vitro and inhibits xenograft tumor growth, prevents metastases, and prolongs survival in vivo.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1073/pnas.1600007113">DOI 10.1073/pnas.1600007113</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000374393800063">Web of Science ID 000374393800063</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/27035983">PubMedID 27035983</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4843455">PubMedCentralID PMC4843455</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-405438">
  <cite><span class="title">
  <span>Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer</span>
  <i>GENES &amp; DEVELOPMENT</i>
</span>
<span class="authors">Reynoird, N., Mazur, P. K., Stellfeld, T., Flores, N. M., Lofgren, S. M., Carlson, S. M., Brambilla, E., Hainaut, P., Kaznowska, E. B., Arrowsmith, C. H., Khatri, P., Stresemann, C., Gozani, O., Sage, J.</span>

<span class="details">
   <span class="year">2016</span>; <span class="volume">30 (7)</span><span class="pages">: 772-785</span>
</span></cite>
    <div class="detail"><p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1101/gad.275529.115">DOI 10.1101/gad.275529.115</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000373194300003">Web of Science ID 000373194300003</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-405926">
  <cite><span class="title">
  <span>Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer.</span>
  <i>Genes &amp; development</i>
</span>
<span class="authors">Reynoird, N., Mazur, P. K., Stellfeld, T., Flores, N. M., Lofgren, S. M., Carlson, S. M., Brambilla, E., Hainaut, P., Kaznowska, E. B., Arrowsmith, C. H., Khatri, P., Stresemann, C., Gozani, O., Sage, J.</span>

<span class="details">
   <span class="year">2016</span>; <span class="volume">30 (7)</span><span class="pages">: 772-785</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Pancreatic ductal adenocarcinoma (PDAC) is a lethal form of cancer with few therapeutic options. We found that levels of the lysine methyltransferase SMYD2 (SET and MYND domain 2) are elevated in PDAC and that genetic and pharmacological inhibition of SMYD2 restricts PDAC growth. We further identified the stress response kinase MAPKAPK3 (MK3) as a new physiologic substrate of SMYD2 in PDAC cells. Inhibition of MAPKAPK3 impedes PDAC growth, identifying a potential new kinase target in PDAC. Finally, we show that inhibition of SMYD2 cooperates with standard chemotherapy to treat PDAC cells and tumors. These findings uncover a pivotal role for SMYD2 in promoting pancreatic cancer.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1101/gad.275529.115">DOI 10.1101/gad.275529.115</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/26988419">PubMedID 26988419</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-407717">
  <cite><span class="title">
  <span>Control of Proliferation and Cancer Growth by the Hippo Signaling Pathway.</span>
  <i>Molecular cancer research </i>
</span>
<span class="authors">Ehmer, U., Sage, J.</span>

<span class="details">
   <span class="year">2016</span>; <span class="volume">14 (2)</span><span class="pages">: 127-140</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The control of cell division is essential for normal development and the maintenance of cellular homeostasis. Abnormal cell proliferation is associated with multiple pathological states, including cancer. Although the Hippo/YAP signaling pathway was initially thought to control organ size and growth, increasing evidence indicates that this pathway also plays a major role in the control of proliferation independent of organ size control. In particular, accumulating evidence indicates that the Hippo/YAP signaling pathway functionally interacts with multiple other cellular pathways and serves as a central node in the regulation of cell division, especially in cancer cells. Here, recent observations are highlighted that connect Hippo/YAP signaling to transcription, the basic cell-cycle machinery, and the control of cell division. Furthermore, the oncogenic and tumor-suppressive attributes of YAP/TAZ are reviewed, which emphasizes the relevance of the Hippo pathway in cancer. Mol Cancer Res; 14(2); 127-40. Â©2015 AACR.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1158/1541-7786.MCR-15-0305">DOI 10.1158/1541-7786.MCR-15-0305</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/26432795">PubMedID 26432795</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-407718">
  <cite><span class="title">
  <span>Loss of Pten Disrupts the Thymic Epithelium and Alters Thymic Function.</span>
  <i>PloS one</i>
</span>
<span class="authors">Garfin, P. M., Nguyen, T., Sage, J.</span>

<span class="details">
   <span class="year">2016</span>; <span class="volume">11 (2)</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The thymus is the site of T cell development and selection. In addition to lymphocytes, the thymus is composed of several types of stromal cells that are exquisitely organized to create the appropriate environment and microenvironment to support the development and selection of maturing T cells. Thymic epithelial cells (TECs) are one of the more important cell types in the thymic stroma, and they play a critical role in selecting functional T cell clones and supporting their development. In this study, we used a mouse genetics approach to investigate the consequences of deleting the Pten tumor suppressor gene in the TEC compartment of the developing thymus. We found that PTEN deficiency in TECs results in a smaller thymus with significantly disordered architecture and histology. Accordingly, loss of PTEN function also results in decreased T cells with a shift in the distribution of T cell subtypes towards CD8+ T cells. These experiments demonstrate that PTEN is critically required for the development of a functional thymic epithelium in mice. This work may help better understand the effects that certain medical conditions or clinical interventions have upon the thymus and immune function.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1371/journal.pone.0149430">DOI 10.1371/journal.pone.0149430</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/26914657">PubMedID 26914657</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-394915">
  <cite><span class="title">
  <span>Crosstalk between stem cell and cell cycle machineries</span>
  <i>CURRENT OPINION IN CELL BIOLOGY</i>
</span>
<span class="authors">Kareta, M. S., Sage, J., Wernig, M.</span>

<span class="details">
   <span class="year">2015</span>; <span class="volume">37</span><span class="pages">: 68-74</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Pluripotent stem cells, defined by an unlimited self-renewal capacity and an undifferentiated state, are best typified by embryonic stem cells. These cells have a unique cell cycle compared to somatic cells as defined by a rapid progression through the cell cycle and a minimal time spent in G1. Recent reports indicate that pluripotency and cell cycle regulation are mechanistically linked. In this review, we discuss the reciprocal co-regulation of these processes, how this co-regulation may prevent differentiation, and how cellular reprogramming can re-establish the unique cell cycle regulation in induced pluripotent stem cells.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1016/j.ceb.2015.10.001">DOI 10.1016/j.ceb.2015.10.001</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000367112800010">Web of Science ID 000367112800010</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/26520682">PubMedID 26520682</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-398340">
  <cite><span class="title">
  <span>Combined inhibition of BET family proteins and histone deacetylases as a potential epigenetics-based therapy for pancreatic ductal adenocarcinoma.</span>
  <i>Nature medicine</i>
</span>
<span class="authors">Mazur, P. K., Herner, A., Mello, S. S., Wirth, M., Hausmann, S., SÃ¡nchez-Rivera, F. J., Lofgren, S. M., Kuschma, T., Hahn, S. A., Vangala, D., Trajkovic-Arsic, M., Gupta, A., Heid, I., NoÃ«l, P. B., Braren, R., Erkan, M., Kleeff, J., Sipos, B., Sayles, L. C., Heikenwalder, M., HeÃŸmann, E., Ellenrieder, V., Esposito, I., Jacks, T., Bradner, J. E., Khatri, P., Sweet-Cordero, E. A., Attardi, L. D., Schmid, R. M., Schneider, G., Sage, J., Siveke, J. T.</span>

<span class="details">
   <span class="year">2015</span>; <span class="volume">21 (10)</span><span class="pages">: 1163-1171</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal human cancers and shows resistance to any therapeutic strategy used. Here we tested small-molecule inhibitors targeting chromatin regulators as possible therapeutic agents in PDAC. We show that JQ1, an inhibitor of the bromodomain and extraterminal (BET) family of proteins, suppresses PDAC development in mice by inhibiting both MYC activity and inflammatory signals. The histone deacetylase (HDAC) inhibitor SAHA synergizes with JQ1 to augment cell death and more potently suppress advanced PDAC. Finally, using a CRISPR-Cas9-based method for gene editing directly in the mouse adult pancreas, we show that de-repression of p57 (also known as KIP2 or CDKN1C) upon combined BET and HDAC inhibition is required for the induction of combination therapy-induced cell death in PDAC. SAHA is approved for human use, and molecules similar to JQ1 are being tested in clinical trials. Thus, these studies identify a promising epigenetic-based therapeutic strategy that may be rapidly implemented in fatal human tumors.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1038/nm.3952">DOI 10.1038/nm.3952</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/26390243">PubMedID 26390243</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-391110">
  <cite><span class="title">
  <span>Comprehensive genomic profiles of small cell lung cancer.</span>
  <i>Nature</i>
</span>
<span class="authors">George, J., Lim, J. S., Jang, S. J., Cun, Y., Ozretic, L., Kong, G., Leenders, F., Lu, X., FernÃ¡ndez-Cuesta, L., Bosco, G., MÃ¼ller, C., Dahmen, I., Jahchan, N. S., Park, K., Yang, D., Karnezis, A. N., Vaka, D., Torres, A., Wang, M. S., Korbel, J. O., Menon, R., Chun, S., Kim, D., Wilkerson, M., Hayes, N., Engelmann, D., PÃ¼tzer, B., Bos, M., Michels, S., Vlasic, I., Seidel, D., Pinther, B., Schaub, P., Becker, C., AltmÃ¼ller, J., Yokota, J., Kohno, T., Iwakawa, R., Tsuta, K., Noguchi, M., Muley, T., Hoffmann, H., Schnabel, P. A., Petersen, I., Chen, Y., Soltermann, A., Tischler, V., Choi, C., Kim, Y., Massion, P. P., Zou, Y., Jovanovic, D., Kontic, M., Wright, G. M., Russell, P. A., Solomon, B., Koch, I., Lindner, M., Muscarella, L. A., la Torre, A., Field, J. K., Jakopovic, M., Knezevic, J., CastaÃ±os-VÃ©lez, E., Roz, L., Pastorino, U., Brustugun, O., Lund-Iversen, M., Thunnissen, E., KÃ¶hler, J., Schuler, M., Botling, J., Sandelin, M., Sanchez-Cespedes, M., Salvesen, H. B., Achter, V., Lang, U., Bogus, M., Schneider, P. M., Zander, T., AnsÃ©n, S., Hallek, M., Wolf, J., Vingron, M., Yatabe, Y., Travis, W. D., NÃ¼rnberg, P., Reinhardt, C., Perner, S., Heukamp, L., BÃ¼ttner, R., Haas, S. A., Brambilla, E., Peifer, M., Sage, J., Thomas, R. K.</span>

<span class="details">
   <span class="year">2015</span>; <span class="volume">524 (7563)</span><span class="pages">: 47-53</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">We have sequenced the genomes of 110 small cell lung cancers (SCLC), one of the deadliest human cancers. In nearly all the tumours analysed we found bi-allelic inactivation of TP53 and RB1, sometimes by complex genomic rearrangements. Two tumours with wild-type RB1 had evidence of chromothripsis leading to overexpression of cyclin D1 (encoded by the CCND1 gene), revealing an alternative mechanism of Rb1 deregulation. Thus, loss of the tumour suppressors TP53 and RB1 is obligatory in SCLC. We discovered somatic genomic rearrangements of TP73 that create an oncogenic version of this gene, TP73?ex2/3. In rare cases, SCLC tumours exhibited kinase gene mutations, providing a possible therapeutic opportunity for individual patients. Finally, we observed inactivating mutations in NOTCH family genes in 25% of human SCLC. Accordingly, activation of Notch signalling in a pre-clinical SCLC mouse model strikingly reduced the number of tumours and extended the survival of the mutant mice. Furthermore, neuroendocrine gene expression was abrogated by Notch activity in SCLC cells. This first comprehensive study of somatic genome alterations in SCLC uncovers several key biological processes and identifies candidate therapeutic targets in this highly lethal form of cancer.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1038/nature14664">DOI 10.1038/nature14664</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/26168399">PubMedID 26168399</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4861069">PubMedCentralID PMC4861069</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-391969">
  <cite><span class="title">
  <span>Comprehensive genomic profiles of small cell lung cancer</span>
  <i>NATURE</i>
</span>
<span class="authors">George, J., Lim, J. S., Jang, S. J., Cun, Y., Ozretic, L., Kong, G., Leenders, F., Lu, X., Fernandez-Cuesta, L., Bosco, G., Mueller, C., Dahmen, I., Jahchan, N. S., Park, K., Yang, D., Karnezis, A. N., Vaka, D., Torres, A., Wang, M. S., Korbel, J. O., Menon, R., Chun, S., Kim, D., Wilkerson, M., Hayes, N., Engelmann, D., Puetzer, B., Bos, M., Michels, S., Vlasic, I., Seidel, D., Pinther, B., Schaub, P., Becker, C., Altmueller, J., Yokota, J., Kohno, T., Iwakawa, R., Tsuta, K., Noguchi, M., Muley, T., Hoffmann, H., Schnabel, P. A., Petersen, I., Chen, Y., Soltermann, A., Tischler, V., Choi, C., Kim, Y., Massion, P. P., Zou, Y., Jovanovic, D., Kontic, M., Wright, G. M., Russell, P. A., Solomon, B., Koch, I., Lindner, M., Muscarella, L. A., la Torre, A., Field, J. K., Jakopovic, M., Knezevic, J., Castanos-Velez, E., Roz, L., Pastorino, U., Brustugun, O., Lund-Iversen, M., Thunnissen, E., Koehler, J., Schuler, M., Botling, J., Sandelin, M., Sanchez-Cespedes, M., Salvesen, H. B., Achter, V., Lang, U., Bogus, M., Schneider, P. M., Zander, T., Ansen, S., Hallek, M., Wolf, J., Vingron, M., Yatabe, Y., Travis, W. D., Nuernberg, P., Reinhardt, C., Perner, S., Heukamp, L., Buettner, R., Haas, S. A., Brambilla, E., Peifer, M., Sage, J., Thomas, R. K.</span>

<span class="details">
   <span class="year">2015</span>; <span class="volume">524 (7563)</span><span class="pages">: 47-U73</span>
</span></cite>
    <div class="detail"><p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1038/nature14664">DOI 10.1038/nature14664</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000359002300029">Web of Science ID 000359002300029</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/26168399">PubMedID 26168399</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-346411">
  <cite><span class="title">
  <span>Inhibition of pluripotency networks by the rb tumor suppressor restricts reprogramming and tumorigenesis.</span>
  <i>Cell stem cell</i>
</span>
<span class="authors">Kareta, M. S., Gorges, L. L., Hafeez, S., Benayoun, B. A., Marro, S., Zmoos, A., Cecchini, M. J., Spacek, D., Batista, L. F., O'Brien, M., Ng, Y., Ang, C. E., Vaka, D., Artandi, S. E., Dick, F. A., Brunet, A., Sage, J., Wernig, M.</span>

<span class="details">
   <span class="year">2015</span>; <span class="volume">16 (1)</span><span class="pages">: 39-50</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Mutations in the retinoblastoma tumor suppressor gene Rb are involved in many forms of human cancer. In this study, we investigated the early consequences of inactivating Rb in the context of cellular reprogramming. We found that Rb inactivation promotes the reprogramming of differentiated cells to a pluripotent state. Unexpectedly, this effect is cell cycle independent, and instead reflects direct binding of Rb to pluripotency genes, including Sox2 and Oct4, which leads to a repressed chromatin state. More broadly, this regulation of pluripotency networks and Sox2 in particular is critical for the initiation of tumors upon loss of Rb in mice. These studies therefore identify Rb as a global transcriptional repressor of pluripotency networks, providing a molecular basis for previous reports about its involvement in cell fate pliability, and implicate misregulation of pluripotency factors such as Sox2 in tumorigenesis related to loss of Rb function.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1016/j.stem.2014.10.019">DOI 10.1016/j.stem.2014.10.019</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/25467916">PubMedID 25467916</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4389904">PubMedCentralID PMC4389904</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-322199">
  <cite><span class="title">
  <span>Genomic analysis of fibrolamellar hepatocellular carcinoma.</span>
  <i>Human molecular genetics</i>
</span>
<span class="authors">Xu, L., Hazard, F. K., Zmoos, A., Jahchan, N., Chaib, H., Garfin, P. M., Rangaswami, A., Snyder, M. P., Sage, J.</span>

<span class="details">
   <span class="year">2015</span>; <span class="volume">24 (1)</span><span class="pages">: 50-63</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Pediatric tumors are relatively infrequent but are often associated with significant lethality and lifelong morbidity. A major goal of pediatric cancer research has been to identify key drivers of tumorigenesis to eventually develop targeted therapies to enhance cure rate and minimize acute and long-term toxic effects. Here we used genomics approaches to identify biomarkers and candidate drivers for fibrolamellar hepatocellular carcinoma (FL-HCC), a very rare subtype of pediatric liver cancer for which limited therapeutic options exist. In-depth genomics analyses of one tumor followed by immunohistochemistry validation on seven other tumors showed expression of neuroendocrine markers in FL-HCC. DNA and RNA sequencing data further showed that common cancer pathways are not visibly altered in FL-HCC but identified two novel structural variants, both resulting in fusion transcripts. The first, a 400kb deletion, results in a DNAJ1-PRKCA fusion transcript, which leads to increased PKA activity in the index tumor case and other FL-HCC cases compared to normal liver. This PKA fusion protein is oncogenic in HCC cells. The second gene fusion event, a translocation between the CLPTML1 and GLIS3 genes, generates a transcript whose product also promotes cancer phenotypes in HCC cell lines. These experiments further highlight the tumorigenic role of gene fusions in the etiology of pediatric solid tumors and identify both candidate biomarkers and possible therapeutic targets for this lethal pediatric disease.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1093/hmg/ddu418">DOI 10.1093/hmg/ddu418</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/25122662">PubMedID 25122662</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-322996">
  <cite><span class="title">
  <span>Organ Size Control Is Dominant over Rb Family Inactivation to Restrict Proliferation In&nbsp;Vivo.</span>
  <i>Cell reports</i>
</span>
<span class="authors">Ehmer, U., Zmoos, A., Auerbach, R. K., Vaka, D., Butte, A. J., Kay, M. A., Sage, J.</span>

<span class="details">
   <span class="year">2014</span>; <span class="volume">8 (2)</span><span class="pages">: 371-381</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">In mammals, a cell's decision to divide is thought to&nbsp;be under the control of the Rb/E2F pathway. We previously found that inactivation of the Rb family of cell cycle inhibitors (Rb, p107, and p130) in quiescent liver progenitors leads to uncontrolled division and cancer initiation. Here, we show that, in contrast, deletion of the entire Rb gene family in mature hepatocytes is not sufficient for their long-term proliferation. The cell cycle block in Rb family mutant hepatocytes is independent of the Arf/p53/p21 checkpoint but can be abrogated upon decreasing liver size. At the molecular level, we identify YAP, a transcriptional regulator involved in organ size control, as a factor required for the sustained expression of cell cycle genes in hepatocytes. These experiments identify a higher level of regulation of the cell cycle in&nbsp;vivo in which signals regulating organ size are dominant regulators of the core cell cycle machinery.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1016/j.celrep.2014.06.025">DOI 10.1016/j.celrep.2014.06.025</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/25017070">PubMedID 25017070</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-317914">
  <cite><span class="title">
  <span>SMYD3 links lysine methylation of MAP3K2 to Ras-driven cancer.</span>
  <i>Nature</i>
</span>
<span class="authors">Mazur, P. K., Reynoird, N., Khatri, P., Jansen, P. W., Wilkinson, A. W., Liu, S., Barbash, O., Van Aller, G. S., Huddleston, M., Dhanak, D., Tummino, P. J., Kruger, R. G., Garcia, B. A., Butte, A. J., Vermeulen, M., Sage, J., Gozani, O.</span>

<span class="details">
   <span class="year">2014</span>; <span class="volume">510 (7504)</span><span class="pages">: 283-287</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Deregulation of lysine methylation signalling has emerged as a common aetiological factor in cancer pathogenesis, with inhibitors of several histone lysine methyltransferases (KMTs) being developed as chemotherapeutics. The largely cytoplasmic KMT SMYD3 (SET and MYND domain containing protein 3) is overexpressed in numerous human tumours. However, the molecular mechanism by which SMYD3 regulates cancer pathways and its relationship to tumorigenesis in vivo are largely unknown. Here we show that methylation of MAP3K2 by SMYD3 increases MAP kinase signalling and promotes the formation of Ras-driven carcinomas. Using mouse models for pancreatic ductal adenocarcinoma and lung adenocarcinoma, we found that abrogating SMYD3 catalytic activity inhibits tumour development in response to oncogenic Ras. We used protein array technology to identify the MAP3K2 kinase as a target of SMYD3. In cancer cell lines, SMYD3-mediated methylation of MAP3K2 at lysine 260 potentiates activation of the Ras/Raf/MEK/ERK signalling module and SMYD3 depletion synergizes with a MEK inhibitor to block Ras-driven tumorigenesis. Finally, the PP2A phosphatase complex, a key negative regulator of the MAP kinase pathway, binds to MAP3K2 and this interaction is blocked by methylation. Together, our results elucidate a new role for lysine methylation in integrating cytoplasmic kinase-signalling cascades and establish a pivotal role for SMYD3 in the regulation of oncogenic Ras signalling.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1038/nature13320">DOI 10.1038/nature13320</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/24847881">PubMedID 24847881</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-308307">
  <cite><span class="title">
  <span>From Fly Wings to Targeted Cancer Therapies: A Centennial for Notch Signaling</span>
  <i>CANCER CELL</i>
</span>
<span class="authors">Ntziachristos, P., Lim, J. S., Sage, J., Aifantis, I.</span>

<span class="details">
   <span class="year">2014</span>; <span class="volume">25 (3)</span><span class="pages">: 318-334</span>
</span></cite>
    <div class="detail"><p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1016/j.ccr.2014.02.018">DOI 10.1016/j.ccr.2014.02.018</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000333233400010">Web of Science ID 000333233400010</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-285360">
  <cite><span class="title">
  <span>A Drug Repositioning Approach Identifies Tricyclic Antidepressants as Inhibitors of Small Cell Lung Cancer and Other Neuroendocrine Tumors</span>
  <i>CANCER DISCOVERY</i>
</span>
<span class="authors">Jahchan, N. S., Dudley, J. T., Mazur, P. K., Flores, N., Yang, D., Palmerton, A., Zmoos, A., Vaka, D., Tran, K. Q., Zhou, M., Krasinska, K., Riess, J. W., Neal, J. W., Khatri, P., Park, K. S., Butte, A. J., Sage, J.</span>

<span class="details">
   <span class="year">2013</span>; <span class="volume">3 (12)</span><span class="pages">: 1364-1377</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Small cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer with high mortality. We used a systematic drug repositioning bioinformatics approach querying a large compendium of gene expression profiles to identify candidate U.S. Food and Drug Administration (FDA)-approved drugs to treat SCLC. We found that tricyclic antidepressants and related molecules potently induce apoptosis in both chemonaÃ¯ve and chemoresistant SCLC cells in culture, in mouse and human SCLC tumors transplanted into immunocompromised mice, and in endogenous tumors from a mouse model for human SCLC. The candidate drugs activate stress pathways and induce cell death in SCLC cells, at least in part by disrupting autocrine survival signals involving neurotransmitters and their G protein-coupled receptors. The candidate drugs inhibit the growth of other neuroendocrine tumors, including pancreatic neuroendocrine tumors and Merkel cell carcinoma. These experiments identify novel targeted strategies that can be rapidly evaluated in patients with neuroendocrine tumors through the repurposing of approved drugs.Our work shows the power of bioinformatics-based drug approaches to rapidly repurpose FDA-approved drugs and identifies a novel class of molecules to treat patients with SCLC, a cancer for which no effective novel systemic treatments have been identified in several decades. In addition, our experiments highlight the importance of novel autocrine mechanisms in promoting the growth of neuroendocrine tumor cells.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1158/2159-8290.CD-13-0183">DOI 10.1158/2159-8290.CD-13-0183</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000328257500023">Web of Science ID 000328257500023</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/24078773">PubMedID 24078773</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3864571">PubMedCentralID PMC3864571</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-132009">
  <cite><span class="title">
  <span>Inactivation of the RB family prevents thymus involution and promotes thymic function by direct control of Foxn1 expression.</span>
  <i>journal of experimental medicine</i>
</span>
<span class="authors">Garfin, P. M., Min, D., Bryson, J. L., Serwold, T., Edris, B., Blackburn, C. C., Richie, E. R., Weinberg, K. I., Manley, N. R., Sage, J., Viatour, P.</span>

<span class="details">
   <span class="year">2013</span>; <span class="volume">210 (6)</span><span class="pages">: 1087-1097</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Thymic involution during aging is a major cause of decreased production of T cells and reduced immunity. Here we show that inactivation of Rb family genes in young mice prevents thymic involution and results in an enlarged thymus competent for increased production of naive T cells. This phenotype originates from the expansion of functional thymic epithelial cells (TECs). In RB family mutant TECs, increased activity of E2F transcription factors drives increased expression of Foxn1, a central regulator of the thymic epithelium. Increased Foxn1 expression is required for the thymic expansion observed in Rb family mutant mice. Thus, the RB family promotes thymic involution and controls T cell production via a bone marrow-independent mechanism, identifying a novel pathway to target to increase thymic function in patients.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1084/jem.20121716">DOI 10.1084/jem.20121716</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/23669396">PubMedID 23669396</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-131405">
  <cite><span class="title">
  <span>IQGAP1 scaffold-kinase interaction blockade selectively targets RAS-MAP kinase-driven tumors.</span>
  <i>Nature medicine</i>
</span>
<span class="authors">Jameson, K. L., Mazur, P. K., Zehnder, A. M., Zhang, J., Zarnegar, B., Sage, J., Khavari, P. A.</span>

<span class="details">
   <span class="year">2013</span>; <span class="volume">19 (5)</span><span class="pages">: 626-630</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Upregulation of the ERK1 and ERK2 (ERK1/2) MAP kinase (MAPK) cascade occurs in &gt;30% of cancers, often through mutational activation of receptor tyrosine kinases or other upstream genes, including KRAS and BRAF. Efforts to target endogenous MAPKs are challenged by the fact that these kinases are required for viability in mammals. Additionally, the effectiveness of new inhibitors of mutant BRAF has been diminished by acquired tumor resistance through selection for BRAF-independent mechanisms of ERK1/2 induction. Furthermore, recently identified ERK1/2-inducing mutations in MEK1 and MEK2 (MEK1/2) MAPK genes in melanoma confer resistance to emerging therapeutic MEK inhibitors, underscoring the challenges facing direct kinase inhibition in cancer. MAPK scaffolds, such as IQ motif-containing GTPase activating protein 1 (IQGAP1), assemble pathway kinases to affect signal transmission, and disrupting scaffold function therefore offers an orthogonal approach to MAPK cascade inhibition. Consistent with this, we found a requirement for IQGAP1 in RAS-driven tumorigenesis in mouse and human tissue. In addition, the ERK1/2-binding IQGAP1 WW domain peptide disrupted IQGAP1-ERK1/2 interactions, inhibited RAS- and RAF-driven tumorigenesis, bypassed acquired resistance to the BRAF inhibitor vemurafenib (PLX-4032) and acted as a systemically deliverable therapeutic to significantly increase the lifespan of tumor-bearing mice. Scaffold-kinase interaction blockade acts by a mechanism distinct from direct kinase inhibition and may be a strategy to target overactive oncogenic kinase cascades in cancer.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1038/nm.3165">DOI 10.1038/nm.3165</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/23603816">PubMedID 23603816</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-131847">
  <cite><span class="title">
  <span>RB goes mitochondrial.</span>
  <i>Genes &amp; development</i>
</span>
<span class="authors">Attardi, L. D., Sage, J.</span>

<span class="details">
   <span class="year">2013</span>; <span class="volume">27 (9)</span><span class="pages">: 975-979</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The retinoblastoma tumor suppressor RB is well known for its capacity to restrict cell cycle progression at the G1/S transition of the cell cycle by controlling the transcription of cell cycle genes. In this issue of Genes &amp; Development, Hilgendorf and colleagues (pp. 1003-1015) have identified a novel tumor suppressor function for RB independent of its role as a transcriptional regulator, in which RB directly activates the apoptosis regulator Bax at the mitochondria to promote cell death.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1101/gad.219451.113">DOI 10.1101/gad.219451.113</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/23651852">PubMedID 23651852</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-127456">
  <cite><span class="title">
  <span>The RB family is required for the self-renewal and survival of human embryonic stem cells</span>
  <i>NATURE COMMUNICATIONS</i>
</span>
<span class="authors">Conklin, J. F., Baker, J., Sage, J.</span>

<span class="details">
   <span class="year">2012</span>; <span class="volume">3</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The mechanisms ensuring the long-term self-renewal of human embryonic stem cells are still only partly understood, limiting their use in cellular therapies. Here we found that increased activity of the RB cell cycle inhibitor in human embryonic stem cells induces cell cycle arrest, differentiation and cell death. Conversely, inactivation of the entire RB family (RB, p107 and p130) in human embryonic stem cells triggers G2/M arrest and cell death through functional activation of the p53 pathway and the cell cycle inhibitor p21. Differences in E2F target gene activation upon loss of RB family function between human embryonic stem cells, mouse embryonic stem cells and human fibroblasts underscore key differences in the cell cycle regulatory networks of human embryonic stem cells. Finally, loss of RB family function promotes genomic instability in both human and mouse embryonic stem cells, uncoupling cell cycle defects from chromosomal instability. These experiments indicate that a homeostatic level of RB activity is essential for the self-renewal and the survival of human embryonic stem cells.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1038/ncomms2254">DOI 10.1038/ncomms2254</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000316356700012">Web of Science ID 000316356700012</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/23212373">PubMedID 23212373</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-242868">
  <cite><span class="title">
  <span>Inactivating All Three Rb Family Pocket Proteins Is Insufficient to Initiate Cervical Cancer</span>
  <i>CANCER RESEARCH</i>
</span>
<span class="authors">Shin, M., Sage, J., Lambert, P. F.</span>

<span class="details">
   <span class="year">2012</span>; <span class="volume">72 (20)</span><span class="pages">: 5418-5427</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Human papillomavirus-16 (HPV-16) is associated etiologically with many human cervical cancers. It encodes 3 oncogenes E5, E6, and E7. Of these oncogenes, E7 has been found to be the dominant driver of cervical cancer in mice. More than 100 cellular proteins have been reported to associate with HPV-16 E7, which is thought to dysregulate the cell cycle in part by binding and inducing the degradation of pRb and its related pocket protein family members, p107 and p130. The ability of E7 to inactivate the pRb family correlates with its ability to induce head and neck cancers in mice. We previously showed that the inactivation of pRb is itself not sufficient to recapitulate the oncogenic properties of E7 in cervical carcinogenesis. In this study, we evaluated mice that were deficient in multiple pocket proteins, including mice that lacked pRb, p107, and p130. Strikingly, combined loss of two or all 3 pocket proteins resulted in development of high-grade cervical intraepithelial neoplasia, but not frank cervical carcinoma. These findings strongly argue that the oncogenic properties of HPV-16 E7 in human cervical carcinogenesis may involve disruption of E7 binding proteins beyond simply the pRb family members.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1158/0008-5472.CAN-12-2083">DOI 10.1158/0008-5472.CAN-12-2083</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000309972700030">Web of Science ID 000309972700030</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/22942253">PubMedID 22942253</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-187872">
  <cite><span class="title">
  <span>Integrative genome analyses identify key somatic driver mutations of small-cell lung cancer</span>
  <i>NATURE GENETICS</i>
</span>
<span class="authors">Peifer, M., Fernandez-Cuesta, L., Sos, M. L., George, J., Seidel, D., Kasper, L. H., Plenker, D., Leenders, F., Sun, R., Zander, T., Menon, R., Koker, M., Dahmen, I., Mueller, C., Di Cerbo, V., Schildhaus, H., Altmueller, J., Baessmann, I., Becker, C., De Wilde, B., Vandesompele, J., Boehm, D., Ansen, S., Gabler, F., Wilkening, I., Heynck, S., Heuckmann, J. M., Lu, X., Carter, S. L., Cibulskis, K., Banerji, S., Getz, G., Park, K., Rauh, D., Gruetter, C., Fischer, M., Pasqualucci, L., Wright, G., Wainer, Z., Russell, P., Petersen, I., Chen, Y., Stoelben, E., Ludwig, C., Schnabel, P., Hoffmann, H., Muley, T., Brockmann, M., Engel-Riedel, W., Muscarella, L. A., Fazio, V. M., Groen, H., Timens, W., Sietsma, H., Thunnissen, E., Smit, E., Heideman, D. A., Snijders, P. J., Cappuzzo, F., Ligorio, C., Damiani, S., Field, J., Solberg, S., Brustugun, O. T., Lund-Iversen, M., Saenger, J., Clement, J. H., Soltermann, A., Moch, H., Weder, W., Solomon, B., Soria, J., Validire, P., Besse, B., Brambilla, E., Brambilla, C., Lantuejoul, S., Lorimier, P., Schneider, P. M., Hallek, M., Pao, W., Meyerson, M., Sage, J., Shendure, J., Schneider, R., Buettner, R., Wolf, J., Nuernberg, P., Perner, S., Heukamp, L. C., Brindle, P. K., Haas, S., Thomas, R. K.</span>

<span class="details">
   <span class="year">2012</span>; <span class="volume">44 (10)</span><span class="pages">: 1104-?</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Small-cell lung cancer (SCLC) is an aggressive lung tumor subtype with poor prognosis. We sequenced 29 SCLC exomes, 2 genomes and 15 transcriptomes and found an extremely high mutation rate of 7.4Â±1 protein-changing mutations per million base pairs. Therefore, we conducted integrated analyses of the various data sets to identify pathogenetically relevant mutated genes. In all cases, we found evidence for inactivation of TP53 and RB1 and identified recurrent mutations in the CREBBP, EP300 and MLL genes that encode histone modifiers. Furthermore, we observed mutations in PTEN, SLIT2 and EPHA7, as well as focal amplifications of the FGFR1 tyrosine kinase gene. Finally, we detected many of the alterations found in humans in SCLC tumors from Tp53 and Rb1 double knockout mice. Our study implicates histone modification as a major feature of SCLC, reveals potentially therapeutically tractable genomic alterations and provides a generalizable framework for the identification of biologically relevant genes in the context of high mutational background.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1038/ng.2396">DOI 10.1038/ng.2396</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000309550200009">Web of Science ID 000309550200009</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/22941188">PubMedID 22941188</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-122469">
  <cite><span class="title">
  <span>The retinoblastoma tumor suppressor and stem cell biology</span>
  <i>GENES &amp; DEVELOPMENT</i>
</span>
<span class="authors">Sage, J.</span>

<span class="details">
   <span class="year">2012</span>; <span class="volume">26 (13)</span><span class="pages">: 1409-1420</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Stem cells play a critical role during embryonic development and in the maintenance of homeostasis in adult individuals. A better understanding of stem cell biology, including embryonic and adult stem cells, will allow the scientific community to better comprehend a number of pathologies and possibly design novel approaches to treat patients with a variety of diseases. The retinoblastoma tumor suppressor RB controls the proliferation, differentiation, and survival of cells, and accumulating evidence points to a central role for RB activity in the biology of stem and progenitor cells. In some contexts, loss of RB function in stem or progenitor cells is a key event in the initiation of cancer and determines the subtype of cancer arising from these pluripotent cells by altering their fate. In other cases, RB inactivation is often not sufficient to initiate cancer but may still lead to some stem cell expansion, raising the possibility that strategies aimed at transiently inactivating RB might provide a novel way to expand functional stem cell populations. Future experiments dedicated to better understanding how RB and the RB pathway control a stem cell's decisions to divide, self-renew, or give rise to differentiated progeny may eventually increase our capacity to control these decisions to enhance regeneration or help prevent cancer development.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1101/gad.193730.112">DOI 10.1101/gad.193730.112</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000306175400002">Web of Science ID 000306175400002</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/22751497">PubMedID 22751497</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-118651">
  <cite><span class="title">
  <span>Smyd3 regulates cancer cell phenotypes and catalyzes histone H4 lysine 5 methylation</span>
  <i>EPIGENETICS</i>
</span>
<span class="authors">Van Aller, G. S., Reynoird, N., Barbash, O., Huddleston, M., Liu, S., Zmoos, A., McDevitt, P., Sinnamon, R., Le, B., Mas, G., Annan, R., Sage, J., Garcia, B. A., Tummino, P. J., Gozani, O., Kruger, R. G.</span>

<span class="details">
   <span class="year">2012</span>; <span class="volume">7 (4)</span><span class="pages">: 340-343</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Smyd3 is a lysine methyltransferase implicated in chromatin and cancer regulation. Here we show that Smyd3 catalyzes histone H4 methylation at lysine 5 (H4K5me). This novel histone methylation mark is detected in diverse cell types and its formation is attenuated by depletion of Smyd3 protein. Further, Smyd3-driven cancer cell phenotypes require its enzymatic activity. Thus, Smyd3, via H4K5 methylation, provides a potential new link between chromatin dynamics and neoplastic disease.</p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000302493300004">Web of Science ID 000302493300004</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/22419068">PubMedID 22419068</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-242869">
  <cite><span class="title">
  <span>GENOMICS The path to retinoblastoma</span>
  <i>NATURE</i>
</span>
<span class="authors">Sage, J., Cleary, M. L.</span>

<span class="details">
   <span class="year">2012</span>; <span class="volume">481 (7381)</span><span class="pages">: 269-270</span>
</span></cite>
    <div class="detail"><p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000299210600023">Web of Science ID 000299210600023</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/22258599">PubMedID 22258599</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-113515">
  <cite><span class="title">
  <span>A crucial requirement for Hedgehog signaling in small cell lung cancer</span>
  <i>NATURE MEDICINE</i>
</span>
<span class="authors">Park, K., Martelotto, L. G., Peifer, M., Sos, M. L., Karnezis, A. N., Mahjoub, M. R., Bernard, K., Conklin, J. F., Szczepny, A., Yuan, J., Guo, R., Ospina, B., Falzon, J., Bennett, S., Brown, T. J., Markovic, A., Devereux, W. L., Ocasio, C. A., Chen, J. K., Stearns, T., Thomas, R. K., Dorsch, M., Buonamici, S., Watkins, D. N., Peacock, C. D., Sage, J.</span>

<span class="details">
   <span class="year">2011</span>; <span class="volume">17 (11)</span><span class="pages">: 1504-U1506</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Small-cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer for which there is no effective treatment. Using a mouse model in which deletion of Rb1 and Trp53 in the lung epithelium of adult mice induces SCLC, we found that the Hedgehog signaling pathway is activated in SCLC cells independently of the lung microenvironment. Constitutive activation of the Hedgehog signaling molecule Smoothened (Smo) promoted the clonogenicity of human SCLC in vitro and the initiation and progression of mouse SCLC in vivo. Reciprocally, deletion of Smo in Rb1 and Trp53-mutant lung epithelial cells strongly suppressed SCLC initiation and progression in mice. Furthermore, pharmacological blockade of Hedgehog signaling inhibited the growth of mouse and human SCLC, most notably following chemotherapy. These findings show a crucial cell-intrinsic role for Hedgehog signaling in the development and maintenance of SCLC and identify Hedgehog pathway inhibition as a therapeutic strategy to slow the progression of disease and delay cancer recurrence in individuals with SCLC.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1038/nm.2473">DOI 10.1038/nm.2473</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000296779300043">Web of Science ID 000296779300043</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/21983857">PubMedID 21983857</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3380617">PubMedCentralID PMC3380617</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-113634">
  <cite><span class="title">
  <span>PDGF signalling controls age-dependent proliferation in pancreatic beta-cells</span>
  <i>NATURE</i>
</span>
<span class="authors">Chen, H., Gu, X., Liu, Y., Wang, J., Wirt, S. E., Bottino, R., Schorle, H., Sage, J., Kim, S. K.</span>

<span class="details">
   <span class="year">2011</span>; <span class="volume">478 (7369)</span><span class="pages">: 349-?</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Determining the signalling pathways that direct tissue expansion is a principal goal of regenerative biology. Vigorous pancreatic ?-cell replication in juvenile mice and humans declines with age, and elucidating the basis for this decay may reveal strategies for inducing ?-cell expansion, a long-sought goal for diabetes therapy. Here we show that platelet-derived growth factor receptor (Pdgfr) signalling controls age-dependent ?-cell proliferation in mouse and human pancreatic islets. With age, declining ?-cell Pdgfr levels were accompanied by reductions in ?-cell enhancer of zeste homologue 2 (Ezh2) levels and ?-cell replication. Conditional inactivation of the Pdgfra gene in ?-cells accelerated these changes, preventing mouse neonatal ?-cell expansion and adult ?-cell regeneration. Targeted human PDGFR-? activation in mouse ?-cells stimulated Erk1/2 phosphorylation, leading to Ezh2-dependent expansion of adult ?-cells. Adult human islets lack PDGF signalling competence, but exposure of juvenile human islets to PDGF-AA stimulated ?-cell proliferation. The discovery of a conserved pathway controlling age-dependent ?-cell proliferation indicates new strategies for ?-cell expansion.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1038/nature10502">DOI 10.1038/nature10502</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000296021100038">Web of Science ID 000296021100038</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/21993628">PubMedID 21993628</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-242870">
  <cite><span class="title">
  <span>Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data (vol 3, 96ra77, 2011)</span>
  <i>SCIENCE TRANSLATIONAL MEDICINE</i>
</span>
<span class="authors">Sirota, M., DUDLEY, J. T., Kim, J., Chiang, A. P., Morgan, A. A., Sweet-Cordero, A., Sage, J., Butte, A. J.</span>

<span class="details">
   <span class="year">2011</span>; <span class="volume">3 (102)</span>
</span></cite>
    <div class="detail"><p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1126/scitranslmed.3003215">DOI 10.1126/scitranslmed.3003215</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000295323200013">Web of Science ID 000295323200013</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-112237">
  <cite><span class="title">
  <span>Notch signaling inhibits hepatocellular carcinoma following inactivation of the RB pathway</span>
  <i>JOURNAL OF EXPERIMENTAL MEDICINE</i>
</span>
<span class="authors">Viatour, P., Ehmer, U., Saddic, L. A., Dorrell, C., Andersen, J. B., Lin, C., Zmoos, A., Mazur, P. K., Schaffer, B. E., Ostermeier, A., Vogel, H., Sylvester, K. G., Thorgeirsson, S. S., Grompe, M., Sage, J.</span>

<span class="details">
   <span class="year">2011</span>; <span class="volume">208 (10)</span><span class="pages">: 1963-1976</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Hepatocellular carcinoma (HCC) is the third cancer killer worldwide with &gt;600,000 deaths every year. Although the major risk factors are known, therapeutic options in patients remain limited in part because of our incomplete understanding of the cellular and molecular mechanisms influencing HCC development. Evidence indicates that the retinoblastoma (RB) pathway is functionally inactivated in most cases of HCC by genetic, epigenetic, and/or viral mechanisms. To investigate the functional relevance of this observation, we inactivated the RB pathway in the liver of adult mice by deleting the three members of the Rb (Rb1) gene family: Rb, p107, and p130. Rb family triple knockout mice develop liver tumors with histopathological features and gene expression profiles similar to human HCC. In this mouse model, cancer initiation is associated with the specific expansion of populations of liver stem/progenitor cells, indicating that the RB pathway may prevent HCC development by maintaining the quiescence of adult liver progenitor cells. In addition, we show that during tumor progression, activation of the Notch pathway via E2F transcription factors serves as a negative feedback mechanism to slow HCC growth. The level of Notch activity is also able to predict survival of HCC patients, suggesting novel means to diagnose and treat HCC.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1084/jem.20110198">DOI 10.1084/jem.20110198</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000295318900005">Web of Science ID 000295318900005</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/21875955">PubMedID 21875955</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-112607">
  <cite><span class="title">
  <span>Lung Cancer Signatures in Plasma Based on Proteome Profiling of Mouse Tumor Models</span>
  <i>CANCER CELL</i>
</span>
<span class="authors">Taguchi, A., Politi, K., Pitteri, S. J., Lockwood, W. W., Faca, V. M., Kelly-Spratt, K., Wong, C., Zhang, Q., Chin, A., Park, K., Goodman, G., Gazdar, A. F., Sage, J., Dinulescu, D. M., Kucherlapati, R., DePinho, R. A., Kemp, C. J., Varmus, H. E., Hanash, S. M.</span>

<span class="details">
   <span class="year">2011</span>; <span class="volume">20 (3)</span><span class="pages">: 289-299</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">We investigated the potential of in-depth quantitative proteomics to reveal plasma protein signatures that reflect lung tumor biology. We compared plasma protein profiles of four mouse models of lung cancer with profiles of models of pancreatic, ovarian, colon, prostate, and breast cancer and two models of inflammation. A protein signature for Titf1/Nkx2-1, a known lineage-survival oncogene in lung cancer, was found in plasmas of mouse models of lung adenocarcinoma. An EGFR signature was found in plasma of an EGFR mutant model, and a distinct plasma signature related to neuroendocrine development was uncovered in the small-cell lung cancer model. We demonstrate relevance to human lung cancer of the protein signatures identified on the basis of mouse models.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1016/j.ccr.2011.08.007">DOI 10.1016/j.ccr.2011.08.007</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000295205700006">Web of Science ID 000295205700006</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/21907921">PubMedID 21907921</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-112259">
  <cite><span class="title">
  <span>Newly identified aspects of tumor suppression by RB</span>
  <i>DISEASE MODELS &amp; MECHANISMS</i>
</span>
<span class="authors">Viatour, P., Sage, J.</span>

<span class="details">
   <span class="year">2011</span>; <span class="volume">4 (5)</span><span class="pages">: 581-585</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The retinoblastoma (RB) tumor suppressor belongs to a cellular pathway that plays a crucial role in restricting the G1-S transition of the cell cycle in response to a large number of extracellular and intracellular cues. Research in the last decade has highlighted the complexity of regulatory networks that ensure proper cell cycle progression, and has also identified multiple cellular functions beyond cell cycle regulation for RB and its two family members, p107 and p130. Here we review some of the recent evidence pointing to a role of RB as a molecular adaptor at the crossroads of multiple pathways, ensuring cellular homeostasis in different contexts. In particular, we discuss the pro- and anti-tumorigenic roles of RB during the early stages of cancer, as well as the importance of the RB pathway in stem cells and cell fate decisions.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1242/dmm.008060">DOI 10.1242/dmm.008060</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000294411800005">Web of Science ID 000294411800005</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/21878458">PubMedID 21878458</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-111881">
  <cite><span class="title">
  <span>Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data</span>
  <i>SCIENCE TRANSLATIONAL MEDICINE</i>
</span>
<span class="authors">Sirota, M., Dudley, J. T., Kim, J., Chiang, A. P., Morgan, A. A., Sweet-Cordero, A., Sage, J., Butte, A. J.</span>

<span class="details">
   <span class="year">2011</span>; <span class="volume">3 (96)</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The application of established drug compounds to new therapeutic indications, known as drug repositioning, offers several advantages over traditional drug development, including reduced development costs and shorter paths to approval. Recent approaches to drug repositioning use high-throughput experimental approaches to assess a compound's potential therapeutic qualities. Here, we present a systematic computational approach to predict novel therapeutic indications on the basis of comprehensive testing of molecular signatures in drug-disease pairs. We integrated gene expression measurements from 100 diseases and gene expression measurements on 164 drug compounds, yielding predicted therapeutic potentials for these drugs. We recovered many known drug and disease relationships using computationally derived therapeutic potentials and also predict many new indications for these 164 drugs. We experimentally validated a prediction for the antiulcer drug cimetidine as a candidate therapeutic in the treatment of lung adenocarcinoma, and demonstrate its efficacy both in vitro and in vivo using mouse xenograft models. This computational method provides a systematic approach for repositioning established drugs to treat a wide range of human diseases.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1126/scitranslmed.3001318">DOI 10.1126/scitranslmed.3001318</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000293953100005">Web of Science ID 000293953100005</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/21849665">PubMedID 21849665</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3502016">PubMedCentralID PMC3502016</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-179727">
  <cite><span class="title">
  <span>Coexpression of Normally Incompatible Developmental Pathways in Retinoblastoma Genesis</span>
  <i>CANCER CELL</i>
</span>
<span class="authors">McEvoy, J., Flores-Otero, J., Zhang, J., Nemeth, K., Brennan, R., Bradley, C., Krafcik, F., Rodriguez-Galindo, C., Wilson, M., Xiong, S., Lozano, G., Sage, J., Fu, L., Louhibi, L., Trimarchi, J., Pani, A., Smeyne, R., Johnson, D., Dyer, M. A.</span>

<span class="details">
   <span class="year">2011</span>; <span class="volume">20 (2)</span><span class="pages">: 260-275</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">It is widely believed that the molecular and cellular features of a tumor reflect its cell of origin and can thus provide clues about treatment targets. The retinoblastoma cell of origin has been debated for over a century. Here, we report that human and mouse retinoblastomas have molecular, cellular, and neurochemical features of multiple cell classes, principally amacrine/horizontal interneurons, retinal progenitor cells, and photoreceptors. Importantly, single-cell gene expression array analysis showed that these multiple cell type-specific developmental programs are coexpressed in individual retinoblastoma cells, which creates a progenitor/neuronal hybrid cell. Furthermore, neurotransmitter receptors, transporters, and biosynthetic enzymes are expressed in human retinoblastoma, and targeted disruption of these pathways reduces retinoblastoma growth in&nbsp;vivo and in&nbsp;vitro.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1016/j.ccr.2011.07.005">DOI 10.1016/j.ccr.2011.07.005</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000294099700013">Web of Science ID 000294099700013</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/21840489">PubMedID 21840489</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-111910">
  <cite><span class="title">
  <span>miR than meets the eye</span>
  <i>GENES &amp; DEVELOPMENT</i>
</span>
<span class="authors">Sage, J., Ventura, A.</span>

<span class="details">
   <span class="year">2011</span>; <span class="volume">25 (16)</span><span class="pages">: 1663-1667</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Retinoblastoma is a rare pediatric cancer that has served as a paradigm to investigate the mechanisms of tumorigenesis. In this issue of Genes &amp; Development, Conkrite and colleagues (pp. 1734-1745) found high levels of the miR-17~92 and miR-106b-25 microRNAs in primary retinoblastomas and show that overexpression of miR-17~92 accelerates retinoblastoma development in mice by promoting proliferation, in part by reducing expression of the cell cycle inhibitor p21. These experiments identify the RB/miR-17~92/p21 axis as a critical regulator of retinoblastoma tumorigenesis and potentially many other cancers.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1101/gad.17454011">DOI 10.1101/gad.17454011</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000294003800001">Web of Science ID 000294003800001</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/21852531">PubMedID 21852531</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-111550">
  <cite><span class="title">
  <span>Characterization of the cell of origin for small cell lung cancer</span>
  <i>CELL CYCLE</i>
</span>
<span class="authors">Park, K., Liang, M., Raiser, D. M., Zamponi, R., Roach, R. R., Curtis, S. J., Walton, Z., Schaffer, B. E., Roake, C. M., Zmoos, A., Kriegel, C., Wong, K., Sage, J., Kim, C. F.</span>

<span class="details">
   <span class="year">2011</span>; <span class="volume">10 (16)</span><span class="pages">: 2806-2815</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Small cell lung carcinoma (SCLC) is a neuroendocrine subtype of lung cancer that affects more than 200,000 people worldwide every year with a very high mortality rate. Here, we used a mouse genetics approach to characterize the cell of origin for SCLC; in this mouse model, tumors are initiated by the deletion of the Rb and p53 tumor suppressor genes in the lung epithelium of adult mice. We found that mouse SCLCs often arise in the lung epithelium, where neuroendocrine cells are located, and that the majority of early lesions were composed of proliferating neuroendocrine cells. In addition, mice in which Rb and p53 are deleted in a variety of non-neuroendocrine lung epithelial cells did not develop SCLC. These data indicate that SCLC likely arises from neuroendocrine cells in the lung.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.4161/cc.10.16.17012">DOI 10.4161/cc.10.16.17012</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000294155600035">Web of Science ID 000294155600035</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/21822053">PubMedID 21822053</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3219544">PubMedCentralID PMC3219544</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-107867">
  <cite><span class="title">
  <span>Functional Interactions between Retinoblastoma and c-MYC in a Mouse Model of Hepatocellular Carcinoma</span>
  <i>PLOS ONE</i>
</span>
<span class="authors">Saddic, L. A., Wirt, S., Vogel, H., Felsher, D. W., Sage, J.</span>

<span class="details">
   <span class="year">2011</span>; <span class="volume">6 (5)</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Inactivation of the RB tumor suppressor and activation of the MYC family of oncogenes are frequent events in a large spectrum of human cancers. Loss of RB function and MYC activation are thought to control both overlapping and distinct cellular processes during cell cycle progression. However, how these two major cancer genes functionally interact during tumorigenesis is still unclear. Here, we sought to test whether loss of RB function would affect cancer development in a mouse model of c-MYC-induced hepatocellular carcinoma (HCC), a deadly cancer type in which RB is frequently inactivated and c-MYC often activated. We found that RB inactivation has minimal effects on the cell cycle, cell death, and differentiation features of liver tumors driven by increased levels of c-MYC. However, combined loss of RB and activation of c-MYC led to an increase in polyploidy in mature hepatocytes before the development of tumors. There was a trend for decreased survival in double mutant animals compared to mice developing c-MYC-induced tumors. Thus, loss of RB function does not provide a proliferative advantage to c-MYC-expressing HCC cells but the RB and c-MYC pathways may cooperate to control the polyploidy of mature hepatocytes.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1371/journal.pone.0019758">DOI 10.1371/journal.pone.0019758</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000290305600045">Web of Science ID 000290305600045</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/21573126">PubMedID 21573126</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-105633">
  <cite><span class="title">
  <span>MicroRNA programs in normal and aberrant stem and progenitor cells</span>
  <i>GENOME RESEARCH</i>
</span>
<span class="authors">Arnold, C. P., Tan, R., Zhou, B., Yue, S., Schaffert, S., Biggs, J. R., Doyonnas, R., Lo, M., Perry, J. M., Renault, V. M., Sacco, A., Somervaille, T., Viatour, P., Brunet, A., Cleary, M. L., Li, L., Sage, J., Zhang, D., Blau, H. M., Chen, C., Chen, C.</span>

<span class="details">
   <span class="year">2011</span>; <span class="volume">21 (5)</span><span class="pages">: 798-810</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Emerging evidence suggests that microRNAs (miRNAs), an abundant class of ?22-nucleotide small regulatory RNAs, play key roles in controlling the post-transcriptional genetic programs in stem and progenitor cells. Here we systematically examined miRNA expression profiles in various adult tissue-specific stem cells and their differentiated counterparts. These analyses revealed miRNA programs that are common or unique to blood, muscle, and neural stem cell populations and miRNA signatures that mark the transitions from self-renewing and quiescent stem cells to proliferative and differentiating progenitor cells. Moreover, we identified a stem/progenitor transition miRNA (SPT-miRNA) signature that predicts the effects of genetic perturbations, such as loss of PTEN and the Rb family, AML1-ETO9a expression, and MLL-AF10 transformation, on self-renewal and proliferation potentials of mutant stem/progenitor cells. We showed that some of the SPT-miRNAs control the self-renewal of embryonic stem cells and the reconstitution potential of hematopoietic stem cells (HSCs). Finally, we demonstrated that SPT-miRNAs coordinately regulate genes that are known to play roles in controlling HSC self-renewal, such as Hoxb6 and Hoxa4. Together, these analyses reveal the miRNA programs that may control key processes in normal and aberrant stem and progenitor cells, setting the foundations for dissecting post-transcriptional regulatory networks in stem cells.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1101/gr.111385.110">DOI 10.1101/gr.111385.110</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000290088000018">Web of Science ID 000290088000018</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/21451113">PubMedID 21451113</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-102830">
  <cite><span class="title">
  <span>RB deletion disrupts coordination between DNA replication licensing and mitotic entry in vivo</span>
  <i>MOLECULAR BIOLOGY OF THE CELL</i>
</span>
<span class="authors">Bourgo, R. J., Ehmer, U., Sage, J., Knudsen, E. S.</span>

<span class="details">
   <span class="year">2011</span>; <span class="volume">22 (7)</span><span class="pages">: 931-939</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The integrity of the retinoblastoma tumor suppressor (RB) pathway is critical for restraining inappropriate proliferation and suppressing tumor development in a plethora of tissues. Here adenovirus-mediated RB deletion in the liver of adult mice led to DNA replication in the absence of productive mitotic condensation. The replication induced by RB loss was E2F-mediated and associated with the induction of DNA damage and a nontranscriptional G2/M checkpoint that targeted the accumulation of Cyclin B1. In the context of RB deletion or E2F activation, there was an increase in hepatocyte ploidy that was accompanied by hyperphysiological assembly of prereplication complexes. In keeping with this dysregulation, initiation of DNA replication was readily observed in hepatocytes that were phenotypically in G2/M. Under such conditions, uncoupling of replication initiation from mitotic progression led to altered genome ploidy in the liver. Interestingly, these findings in hepatocytes were not recapitulated in the basally proliferative tissues of the gastrointestinal tract, where RB deletion, while increasing DNA replication, did not lead to a profound uncoupling from mitosis. Combined, these findings demonstrate the critical role of RB in controlling cell-cycle transitions and underscore the importance of intrinsic tissue environments in resultant phenotypes.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1091/mbc.E10-11-0895">DOI 10.1091/mbc.E10-11-0895</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000288995800005">Web of Science ID 000288995800005</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/21289097">PubMedID 21289097</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-96868">
  <cite><span class="title">
  <span>Methylation of the Retinoblastoma Tumor Suppressor by SMYD2</span>
  <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i>
</span>
<span class="authors">Saddic, L. A., West, L. E., Aslanian, A., Yates, J. R., Rubin, S. M., Gozani, O., Sage, J.</span>

<span class="details">
   <span class="year">2010</span>; <span class="volume">285 (48)</span><span class="pages">: 37733-37740</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The retinoblastoma tumor suppressor (RB) is a central cell cycle regulator and tumor suppressor. RB cellular functions are known to be regulated by a diversity of post-translational modifications such as phosphorylation and acetylation, raising the possibility that RB may also be methylated in cells. Here we demonstrate that RB can be methylated by SMYD2 at lysine 860, a highly conserved and novel site of modification. This methylation event occurs in vitro and in cells, and it is regulated during cell cycle progression, cellular differentiation, and in response to DNA damage. Furthermore, we show that RB monomethylation at lysine 860 provides a direct binding site for the methyl-binding domain of the transcriptional repressor L3MBTL1. These results support the idea that a code of post-translational modifications exists for RB and helps guide its functions in mammalian cells.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1074/jbc.M110.137612">DOI 10.1074/jbc.M110.137612</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000284424000065">Web of Science ID 000284424000065</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/20870719">PubMedID 20870719</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-98964">
  <cite><span class="title">
  <span>G1 arrest and differentiation can occur independently of Rb family function</span>
  <i>JOURNAL OF CELL BIOLOGY</i>
</span>
<span class="authors">Wirt, S. E., Adler, A. S., Gebala, V., Weimann, J. M., Schaffer, B. E., Saddic, L. A., Viatour, P., Vogel, H., Chang, H. Y., Meissner, A., Sage, J.</span>

<span class="details">
   <span class="year">2010</span>; <span class="volume">191 (4)</span><span class="pages">: 809-825</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The ability of progenitor cells to exit the cell cycle is essential for proper embryonic development and homeostasis, but the mechanisms governing cell cycle exit are still not fully understood. Here, we tested the requirement for the retinoblastoma (Rb) protein and its family members p107 and p130 in G0/G1 arrest and differentiation in mammalian cells. We found that Rb family triple knockout (TKO) mouse embryos survive until days 9-11 of gestation. Strikingly, some TKO cells, including in epithelial and neural lineages, are able to exit the cell cycle in G0/G1 and differentiate in teratomas and in culture. This ability of TKO cells to arrest in G0/G1 is associated with the repression of key E2F target genes. Thus, G1 arrest is not always dependent on Rb family members, which illustrates the robustness of cell cycle regulatory networks during differentiation and allows for the identification of candidate pathways to inhibit the expansion of cancer cells with mutations in the Rb pathway.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1083/jcb.201003048">DOI 10.1083/jcb.201003048</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000284737200014">Web of Science ID 000284737200014</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/21059851">PubMedID 21059851</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-94994">
  <cite><span class="title">
  <span>Transient Inactivation of Rb and ARF Yields Regenerative Cells from Postmitotic Mammalian Muscle</span>
  <i>CELL STEM CELL</i>
</span>
<span class="authors">Pajcini, K. V., Corbel, S. Y., Sage, J., Pomerantz, J. H., Blau, H. M.</span>

<span class="details">
   <span class="year">2010</span>; <span class="volume">7 (2)</span><span class="pages">: 198-213</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">An outstanding biological question is why tissue regeneration in mammals is limited, whereas urodele amphibians and teleost fish regenerate major structures, largely by cell cycle reentry. Upon inactivation of Rb, proliferation of postmitotic urodele skeletal muscle is induced, whereas in mammalian muscle this mechanism does not exist. We postulated that a tumor suppressor present in mammals but absent in regenerative vertebrates, the Ink4a product ARF (alternative reading frame), is a regeneration suppressor. Concomitant inactivation of Arf and Rb led to mammalian muscle cell cycle reentry, loss of differentiation properties, and upregulation of cytokinetic machinery. Single postmitotic myocytes were isolated by laser micro-dissection-catapulting, and transient suppression of Arf and Rb yielded myoblast colonies that retained the ability to differentiate and fuse into myofibers upon transplantation in vivo. These results show that differentiation of mammalian cells is reversed by inactivation of Arf and Rb and support the hypothesis that Arf evolved at the expense of regeneration.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1016/j.stem.2010.05.022">DOI 10.1016/j.stem.2010.05.022</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000281107400012">Web of Science ID 000281107400012</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/20682446">PubMedID 20682446</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-93482">
  <cite><span class="title">
  <span>RB's original CIN?</span>
  <i>GENES &amp; DEVELOPMENT</i>
</span>
<span class="authors">Sage, J., Straight, A. F.</span>

<span class="details">
   <span class="year">2010</span>; <span class="volume">24 (13)</span><span class="pages">: 1329-1333</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The retinoblastoma tumor suppressor RB is the downstream mediator of a cellular pathway that is thought to prevent cancer by controlling the ability of cells to enter or exit the cell cycle in G0/G1. Recently, however, accumulating evidence has suggested that RB, its family members p107 and p130, and their partners, the E2F family of transcription factors, may have important cellular functions beyond the G1/S transition of the cell cycle, including during DNA replication and at the transition into mitosis. In this issue of Genes &amp; Development, three studies demonstrate a critical role for RB in proper chromosome condensation, centromeric function, and chromosome stability in mammalian cells, and link these cellular functions of RB to tumor suppression in mice. Here we discuss how transcriptional and post-transcriptional mechanisms under the control of the RB pathway ensure accurate progression through mitosis, thereby preventing cancer development.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1101/gad.1948010">DOI 10.1101/gad.1948010</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000279405000001">Web of Science ID 000279405000001</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/20551167">PubMedID 20551167</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-93888">
  <cite><span class="title">
  <span>Tandem E2F Binding Sites in the Promoter of the p107 Cell Cycle Regulator Control p107 Expression and Its Cellular Functions</span>
  <i>PLOS GENETICS</i>
</span>
<span class="authors">Burkhart, D. L., Wirt, S. E., Zmoos, A., Kareta, M. S., Sage, J.</span>

<span class="details">
   <span class="year">2010</span>; <span class="volume">6 (6)</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The retinoblastoma tumor suppressor (Rb) is a potent and ubiquitously expressed cell cycle regulator, but patients with a germline Rb mutation develop a very specific tumor spectrum. This surprising observation raises the possibility that mechanisms that compensate for loss of Rb function are present or activated in many cell types. In particular, p107, a protein related to Rb, has been shown to functionally overlap for loss of Rb in several cellular contexts. To investigate the mechanisms underlying this functional redundancy between Rb and p107 in vivo, we used gene targeting in embryonic stem cells to engineer point mutations in two consensus E2F binding sites in the endogenous p107 promoter. Analysis of normal and mutant cells by gene expression and chromatin immunoprecipitation assays showed that members of the Rb and E2F families directly bound these two sites. Furthermore, we found that these two E2F sites controlled both the repression of p107 in quiescent cells and also its activation in cycling cells, as well as in Rb mutant cells. Cell cycle assays further indicated that activation of p107 transcription during S phase through the two E2F binding sites was critical for controlled cell cycle progression, uncovering a specific role for p107 to slow proliferation in mammalian cells. Direct transcriptional repression of p107 by Rb and E2F family members provides a molecular mechanism for a critical negative feedback loop during cell cycle progression and tumorigenesis. These experiments also suggest novel therapeutic strategies to increase the p107 levels in tumor cells.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1371/journal.pgen.1001003">DOI 10.1371/journal.pgen.1001003</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000279805200032">Web of Science ID 000279805200032</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/20585628">PubMedID 20585628</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-91948">
  <cite><span class="title">
  <span>Loss of p130 Accelerates Tumor Development in a Mouse Model for Human Small-Cell Lung Carcinoma</span>
  <i>CANCER RESEARCH</i>
</span>
<span class="authors">Schaffer, B. E., Park, K., Yiu, G., Conklin, J. F., Lin, C., Burkhart, D. L., Karnezis, A. N., Sweet-Cordero, E. A., Sage, J.</span>

<span class="details">
   <span class="year">2010</span>; <span class="volume">70 (10)</span><span class="pages">: 3877-3883</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Small-cell lung carcinoma (SCLC) is a neuroendocrine subtype of lung cancer. Although SCLC patients often initially respond to therapy, tumors nearly always recur, resulting in a 5-year survival rate of less than 10%. A mouse model has been developed based on the fact that the RB and p53 tumor suppressor genes are mutated in more than 90% of human SCLCs. Emerging evidence in patients and mouse models suggests that p130, a gene related to RB, may act as a tumor suppressor in SCLC cells. To test this idea, we used conditional mutant mice to delete p130 in combination with Rb and p53 in adult lung epithelial cells. We found that loss of p130 resulted in increased proliferation and significant acceleration of SCLC development in this triple-knockout mouse model. The histopathologic features of the triple-mutant mouse tumors closely resembled that of human SCLC. Genome-wide expression profiling experiments further showed that Rb/p53/p130-mutant mouse tumors were similar to human SCLC. These findings indicate that p130 plays a key tumor suppressor role in SCLC. Rb/p53/p130-mutant mice provide a novel preclinical mouse model to identify novel therapeutic targets against SCLC.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1158/0008-5472.CAN-09-4228">DOI 10.1158/0008-5472.CAN-09-4228</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000278486300004">Web of Science ID 000278486300004</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/20406986">PubMedID 20406986</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-91932">
  <cite><span class="title">
  <span>Complex transcriptional regulatory networks around the retinoblastoma tumor suppressor gene</span>
  <i>CELL CYCLE</i>
</span>
<span class="authors">Burkhart, D. L., Sage, J.</span>

<span class="details">
   <span class="year">2010</span>; <span class="volume">9 (9)</span><span class="pages">: 1663-1664</span>
</span></cite>
    <div class="detail"><p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000277402200002">Web of Science ID 000277402200002</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/20404509">PubMedID 20404509</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-91257">
  <cite><span class="title">
  <span>p107 in the public eye: an Rb understudy and more</span>
  <i>CELL DIVISION</i>
</span>
<span class="authors">Wirt, S. E., Sage, J.</span>

<span class="details">
   <span class="year">2010</span>; <span class="volume">5</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">p107 and its related family members Rb and p130 are critical regulators of cellular proliferation and tumorigenesis. Due to the extent of functional overlap within the Rb family, it has been difficult to assess which functions are exclusive to individual members and which are shared. Like its family members, p107 can bind a variety of cellular proteins to affect the expression of many target genes during cell cycle progression. Unlike Rb and p130, p107 is most highly expressed during the G1 to S phase transition of the cell cycle in actively dividing cells and accumulating evidence suggests a role for p107 during DNA replication. The specific roles for p107 during differentiation and development are less clear, although emerging studies suggest that it can cooperate with other Rb family members to control differentiation in multiple cell lineages. As a tumor suppressor, p107 is not as potent as Rb, yet studies in knockout mice have revealed some tumor suppressor functions in mice, depending on the context. In this review, we identify the unique and overlapping functions of p107 during the cell cycle, differentiation, and tumorigenesis.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1186/1747-1028-5-9">DOI 10.1186/1747-1028-5-9</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000283795500001">Web of Science ID 000283795500001</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/20359370">PubMedID 20359370</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-89130">
  <cite><span class="title">
  <span>Regulation of RB Transcription In Vivo by RB Family Members</span>
  <i>MOLECULAR AND CELLULAR BIOLOGY</i>
</span>
<span class="authors">Burkhart, D. L., Ngai, L. K., Roake, C. M., Viatour, P., Thangavel, C., Ho, V. M., Knudsen, E. S., Sage, J.</span>

<span class="details">
   <span class="year">2010</span>; <span class="volume">30 (7)</span><span class="pages">: 1729-1745</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">In cancer cells, the retinoblastoma tumor suppressor RB is directly inactivated by mutation in the RB gene or functionally inhibited by abnormal activation of cyclin-dependent kinase activity. While variations in RB levels may also provide an important means of controlling RB function in both normal and cancer cells, little is known about the mechanisms regulating RB transcription. Here we show that members of the RB and E2F families bind directly to the RB promoter. To investigate how the RB/E2F pathway may regulate Rb transcription, we generated reporter mice carrying an eGFP transgene inserted into a bacterial artificial chromosome containing most of the Rb gene. Expression of eGFP largely parallels that of Rb in transgenic embryos and adult mice. Using these reporter mice and mutant alleles for Rb, p107, and p130, we found that RB family members modulate Rb transcription in specific cell populations in vivo and in culture. Interestingly, while Rb is a target of the RB/E2F pathway in mouse and human cells, Rb expression does not strictly correlate with the cell cycle status of these cells. These experiments identify novel regulatory feedback mechanisms within the RB pathway in mammalian cells.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1128/MCB.00952-09">DOI 10.1128/MCB.00952-09</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000275302000013">Web of Science ID 000275302000013</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/20100864">PubMedID 20100864</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2838078">PubMedCentralID PMC2838078</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-85005">
  <cite><span class="title">
  <span>Keeping an Eye on Retinoblastoma Control of Human Embryonic Stem Cells</span>
  <i>JOURNAL OF CELLULAR BIOCHEMISTRY</i>
</span>
<span class="authors">Conklin, J. F., Sage, J.</span>

<span class="details">
   <span class="year">2009</span>; <span class="volume">108 (5)</span><span class="pages">: 1023-1030</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Human embryonic stem cells (hESCs) hold great promise in regenerative medicine. However, before the full potential of these cells is achieved, major basic biological questions need to be addressed. In particular, there are still gaps in our knowledge of the molecular mechanisms underlying the derivation of hESCs from blastocysts, the regulation of the undifferentiated, pluripotent state, and the control of differentiation into specific lineages. Furthermore, we still do not fully understand the tumorigenic potential of hESCs, limiting their use in regenerative medicine. The RB pathway is a key signaling module that controls cellular proliferation, cell survival, chromatin structure, and cellular differentiation in mammalian cells. Members of the RB pathway are important regulators of hESC biology and manipulation of the activity of this pathway may provide novel means to control the fate of hESCs. Here we review what is known about the expression and function of members of the RB pathway in hESCs and discuss areas of interest in this field.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1002/jcb.22342">DOI 10.1002/jcb.22342</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000272640900001">Web of Science ID 000272640900001</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/19760644">PubMedID 19760644</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-242871">
  <cite><span class="title">
  <span>Novel roles for A-type lamins in telomere biology and the DNA damage response pathway</span>
  <i>EMBO JOURNAL</i>
</span>
<span class="authors">Gonzalez-Suarez, I., Redwood, A. B., Perkins, S. M., Vermolen, B., Lichtensztejin, D., Grotsky, D. A., Morgado-Palacin, L., Gapud, E. J., Sleckman, B. P., Sullivan, T., Sage, J., Stewart, C. L., Mai, S., Gonzalo, S.</span>

<span class="details">
   <span class="year">2009</span>; <span class="volume">28 (16)</span><span class="pages">: 2414-2427</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">A-type lamins are intermediate filament proteins that provide a scaffold for protein complexes regulating nuclear structure and function. Mutations in the LMNA gene are linked to a variety of degenerative disorders termed laminopathies, whereas changes in the expression of lamins are associated with tumourigenesis. The molecular pathways affected by alterations of A-type lamins and how they contribute to disease are poorly understood. Here, we show that A-type lamins have a key role in the maintenance of telomere structure, length and function, and in the stabilization of 53BP1, a component of the DNA damage response (DDR) pathway. Loss of A-type lamins alters the nuclear distribution of telomeres and results in telomere shortening, defects in telomeric heterochromatin, and increased genomic instability. In addition, A-type lamins are necessary for the processing of dysfunctional telomeres by non-homologous end joining, putatively through stabilization of 53BP1. This study shows new functions for A-type lamins in the maintenance of genomic integrity, and suggests that alterations of telomere biology and defects in DDR contribute to the pathogenesis of lamin-related diseases.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1038/emboj.2009.196">DOI 10.1038/emboj.2009.196</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000269074400010">Web of Science ID 000269074400010</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/19629036">PubMedID 19629036</a></span>
  </p>
  <p class="pub-med-central">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2735177">PubMedCentralID PMC2735177</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-80596">
  <cite><span class="title">
  <span>The role of the retinoblastoma/E2F1 tumor suppressor pathway in the lesion recognition step of nucleotide excision repair</span>
  <i>DNA REPAIR</i>
</span>
<span class="authors">Lin, P. S., McPherson, L. A., Chen, A. Y., Sage, J., Ford, J. M.</span>

<span class="details">
   <span class="year">2009</span>; <span class="volume">8 (7)</span><span class="pages">: 795-802</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The retinoblastoma Rb/E2F tumor suppressor pathway plays a major role in the regulation of mammalian cell cycle progression. The pRb protein, along with closely related proteins p107 and p130, exerts its anti-proliferative effects by binding to the E2F family of transcription factors known to regulate essential genes throughout the cell cycle. We sought to investigate the role of the Rb/E2F1 pathway in the lesion recognition step of nucleotide excision repair (NER) in mouse embryonic fibroblasts (MEFs). Rb-/-, p107-/-, p130-/- MEFs repaired both cyclobutane pyrimidine dimers (CPDs) and 6-4 photoproducts (6-4PPs) at higher efficiency than did wildtype cells following UV-C irradiation. The expression of damaged DNA binding gene DDB2 involved in the DNA lesion recognition step was elevated in the Rb family-deficient MEFs. To determine if the enhanced DNA repair in the absence of the Rb gene family is due to the derepression of E2F1, we assayed the ability of E2F1-deficient cells to repair damaged DNA and demonstrated that E2F1-/- MEFs are impaired for the removal of both CPDs and 6-4PPs. Furthermore, wildtype cells induced a higher expression of DDB2 and xeroderma pigmentosum gene XPC transcript levels than did E2F1-/- cells following UV-C irradiation. Using an E2F SiteScan algorithm, we uncovered a putative E2F-responsive element in the XPC promoter upstream of the transcription start site. We showed with chromatin immunoprecipitation assays the binding of E2F1 to the XPC promoter in a UV-dependent manner, suggesting that E2F1 is a transcriptional regulator of XPC. Our study identifies a novel E2F1 gene target and further supports the growing body of evidence that the Rb/E2F1 tumor suppressor pathway is involved in the regulation of the DNA lesion recognition step of nucleotide excision repair.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1016/j.dnarep.2009.03.003">DOI 10.1016/j.dnarep.2009.03.003</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000268086300003">Web of Science ID 000268086300003</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/19376752">PubMedID 19376752</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-77918">
  <cite><span class="title">
  <span>The retinoblastoma gene Rb and its family member p130 suppress lung adenocarcinoma induced by oncogenic K-Ras</span>
  <i>ONCOGENE</i>
</span>
<span class="authors">Ho, V. M., Schaffer, B. E., Karnezis, A. N., Park, K. S., Sage, J.</span>

<span class="details">
   <span class="year">2009</span>; <span class="volume">28 (10)</span><span class="pages">: 1393-1399</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Mutations of the retinoblastoma tumor suppressor gene RB are frequently observed in human cancers, but rarely in non-small cell lung carcinomas (NSCLCs). Emerging evidence also suggests that the RB-related gene p130 is inactivated in a subset of human NSCLCs. To directly test the specific tumor suppressor roles of RB and p130 in NSCLC, we crossed Rb and p130 conditional mutant mice to mice carrying a conditional oncogenic K-Ras allele. In this model, controlled oncogenic K-Ras activation leads to the development of adenocarcinoma, a major subtype of NSCLC. We found that loss of p130 accelerated the death of mice, providing direct evidence in vivo that p130 is a tumor suppressor gene, albeit a weak one in this context. Loss of Rb increased the efficiency of lung cancer initiation and resulted in the development of high-grade adenocarcinomas and rapid death. Thus, despite the low frequency of RB mutations in human NSCLCs and reports that K-Ras activation and loss of RB function are rarely found in the same human tumors, loss of Rb clearly cooperates with activation of oncogenic K-Ras in lung adenocarcinoma development in mice.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1038/onc.2008.491">DOI 10.1038/onc.2008.491</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000264116000012">Web of Science ID 000264116000012</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/19151761">PubMedID 19151761</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-75095">
  <cite><span class="title">
  <span>Hematopoietic Stem Cell Quiescence Is Maintained by Compound Contributions of the Retinoblastoma Gene Family</span>
  <i>CELL STEM CELL</i>
</span>
<span class="authors">Viatour, P., Somervaille, T. C., Venkatasubrahmanyam, S., Kogan, S., McLaughlin, M. E., Weissman, I. L., Butte, A. J., Passegue, E., Sage, J.</span>

<span class="details">
   <span class="year">2008</span>; <span class="volume">3 (4)</span><span class="pages">: 416-428</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Individual members of the retinoblastoma (Rb) tumor suppressor gene family serve critical roles in the control of cellular proliferation and differentiation, but the extent of their contributions is masked by redundant and compensatory mechanisms. Here we employed a conditional knockout strategy to simultaneously inactivate all three members, Rb, p107, and p130, in adult hematopoietic stem cells (HSCs). Rb family triple knockout (TKO) mice develop a cell-intrinsic myeloproliferation that originates from hyperproliferative early hematopoietic progenitors and is accompanied by increased apoptosis in lymphoid progenitor populations. Loss of quiescence in the TKO HSC pool is associated with an expansion of these mutant stem cells but also with an enhanced mobilization and an impaired reconstitution potential upon transplantation. The presence of a single p107 allele is sufficient to largely rescue these defects. Thus, Rb family members collectively maintain HSC quiescence and the balance between lymphoid and myeloid cell fates in the hematopoietic system.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1016/j.stem.2008.07.009">DOI 10.1016/j.stem.2008.07.009</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000260149800012">Web of Science ID 000260149800012</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/18940733">PubMedID 18940733</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-72213">
  <cite><span class="title">
  <span>Cellular mechanisms of tumour suppression by the retinoblastoma gene</span>
  <i>NATURE REVIEWS CANCER</i>
</span>
<span class="authors">Burkhart, D. L., Sage, J.</span>

<span class="details">
   <span class="year">2008</span>; <span class="volume">8 (9)</span><span class="pages">: 671-682</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The retinoblastoma (RB) tumour suppressor gene is functionally inactivated in a broad range of paediatric and adult cancers, and a plethora of cellular functions and partners have been identified for the RB protein. Data from human tumours and studies from mouse models indicate that loss of RB function contributes to both cancer initiation and progression. However, we still do not know the identity of the cell types in which RB normally prevents cancer initiation in vivo, and the specific functions of RB that suppress distinct aspects of the tumorigenic process are poorly understood.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1038/nrc2399">DOI 10.1038/nrc2399</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000258662900012">Web of Science ID 000258662900012</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/18650841">PubMedID 18650841</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-73217">
  <cite><span class="title">
  <span>GFP reporter mice for the retinoblastoma-related cell cycle regulator p107</span>
  <i>CELL CYCLE</i>
</span>
<span class="authors">Burkhart, D. L., Viatour, P., Ho, V. M., Sage, J.</span>

<span class="details">
   <span class="year">2008</span>; <span class="volume">7 (16)</span><span class="pages">: 2544-2552</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The RB tumor suppressor gene is mutated in a broad range of human cancers, including pediatric retinoblastoma. Strikingly, however, Rb mutant mice develop tumors of the pituitary and thyroid glands, but not retinoblastoma. Mouse genetics experiments have demonstrated that p107, a protein related to pRB, is capable of preventing retinoblastoma, but not pituitary tumors, in Rb-deficient mice. Evidence suggests that the basis for this compensatory function of p107 is increased transcription of the p107 gene in response to Rb inactivation. To begin to address the context-dependency of this compensatory role of p107 and to follow p107 expression in vivo, we have generated transgenic mice carrying an enhanced GFP (eGFP) reporter inserted into a bacterial artificial chromosome (BAC) containing the mouse p107 gene. Expression of the eGFP transgene parallels that of p107 in these transgenic mice and identifies cells with a broad range of expression level for p107, even within particular organs or tissues. We also show that loss of Rb results in the upregulation of p107 transcription in specific cell populations in vivo, including subpopulations of hematopoietic cells. Thus, p107 BAC-eGFP transgenic mice serve as a useful tool to identify distinct cell types in which p107 is expressed and may have key functions in vivo, and to characterize changes in cellular networks accompanying Rb deficiency.</p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000258829900016">Web of Science ID 000258829900016</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/18719374">PubMedID 18719374</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-56229">
  <cite><span class="title">
  <span>Hope in sight for retinoblastoma</span>
  <i>NATURE MEDICINE</i>
</span>
<span class="authors">Sage, J.</span>

<span class="details">
   <span class="year">2007</span>; <span class="volume">13 (1)</span><span class="pages">: 30-31</span>
</span></cite>
    <div class="detail"><p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000243301800024">Web of Science ID 000243301800024</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/17206131">PubMedID 17206131</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-217807">
  <cite><span class="title">
  <span>pRB family proteins are required for H3K27 trimethylation and polycomb repression complexes binding to and silencing p16(INK4a) tumor suppressor gene</span>
  <i>GENES &amp; DEVELOPMENT</i>
</span>
<span class="authors">Kotake, Y., Cao, R., Viatour, P., Sage, J., Zhang, Y., Xiong, Y.</span>

<span class="details">
   <span class="year">2007</span>; <span class="volume">21 (1)</span><span class="pages">: 49-54</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Genetic studies have demonstrated that Bmi1 promotes cell proliferation and stem cell self-renewal with a correlative decrease of p16(INK4a) expression. Here, we demonstrate that Polycomb genes EZH2 and BMI1 repress p16 expression in human and mouse primary cells, but not in cells deficient for pRB protein function. The p16 locus is H3K27-methylated and bound by BMI1, RING2, and SUZ12. Inactivation of pRB family proteins abolishes H3K27 methylation and disrupts BMI1, RING2, and SUZ12 binding to the p16 locus. These results suggest a model in which pRB proteins recruit PRC2 to trimethylate p16, priming the BMI1-containing PRC1L ubiquitin ligase complex to silence p16.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1101/gad.1499407">DOI 10.1101/gad.1499407</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000243382700005">Web of Science ID 000243382700005</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/17210787">PubMedID 17210787</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-242872">
  <cite><span class="title">
  <span>The related retinoblastoma (pRb) and p130 proteins cooperate to regulate homeostasis in the intestinal epithelium</span>
  <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i>
</span>
<span class="authors">Haigis, K., Sage, J., Glickman, J., Shafer, S., Jacks, T.</span>

<span class="details">
   <span class="year">2006</span>; <span class="volume">281 (1)</span><span class="pages">: 638-647</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">pRb, p107, and p130 are related proteins that play a central role in the regulation of cell cycle progression and terminal differentiation in mammalian cells. Nevertheless, it is still largely unclear how these proteins achieve this regulation in vivo. The intestinal epithelium is an ideal in vivo system in which to study the molecular pathways that regulate proliferation and differentiation because it exists in a constant state of development throughout an animal's lifetime. We studied the phenotypic effects on the intestinal epithelium of mutating Rb and p107 or p130. Although mutating these genes singly had little or no effect, loss of pRb and p107 or p130 together produced chronic hyperplasia and dysplasia of the small intestinal and colonic epithelium. In Rb/p130 double mutants this hyperplasia was associated with defects in terminal differentiation of specific cell types and was dependent on the increased proliferation seen in the epithelium of mutant animals. At the molecular level, dysregulation of the Rb pathway led to an increase in the expression of Math1, Cdx1, Cdx2, transcription factors that regulate proliferation and differentiation in the intestinal epithelium. The absence of Cdx1 function in Rb/p130 double mutant mice partially reverted the histologic phenotype by suppressing ectopic mitosis in the epithelium. These studies implicate the Rb pathway as a regulator of epithelial homeostasis in the intestine.</p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000234307200076">Web of Science ID 000234307200076</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/16258171">PubMedID 16258171</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-44096">
  <cite><span class="title">
  <span>C/EBP beta cooperates with RB : E2F to implement Ras(V12)-induced cellular senescence</span>
  <i>EMBO JOURNAL</i>
</span>
<span class="authors">Sebastian, T., Malik, R., Thomas, S., Sage, J., Johnson, P. F.</span>

<span class="details">
   <span class="year">2005</span>; <span class="volume">24 (18)</span><span class="pages">: 3301-3312</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">In primary cells, overexpression of oncogenes such as Ras(V12) induces premature senescence rather than transformation. Senescence is an irreversible form of G1 arrest that requires the p19ARF/p53 and p16INK4a/pRB pathways and may suppress tumorigenesis in vivo. Here we show that the transcription factor C/EBPbeta is required for Ras(V12)-induced senescence. C/EBPbeta-/- mouse embryo fibroblasts (MEFs) expressing Ras(V12) continued to proliferate despite unimpaired induction of p19ARF and p53, and lacked morphological features of senescent fibroblasts. Enforced C/EBPbeta expression inhibited proliferation of wild-type MEFs and also slowed proliferation of p19Arf-/- and p53-/- cells, indicating that C/EBPbeta acts downstream or independently of p19ARF/p53 to suppress growth. C/EBPbeta was unable to inhibit proliferation of MEFs lacking all three RB family proteins or wild-type cells expressing dominant negative E2F-1 and, instead, stimulated their growth. C/EBPbeta decreased expression of several E2F target genes and was associated with their promoters in chromatin immunoprecipitation assays, suggesting that C/EBPbeta functions by repressing genes required for cell cycle progression. C/EBPbeta is therefore a novel component of the RB:E2F-dependent senescence program activated by oncogenic stress in primary cells.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1038/sj.emboj.7600789">DOI 10.1038/sj.emboj.7600789</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000232551600014">Web of Science ID 000232551600014</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/16107878">PubMedID 16107878</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-44091">
  <cite><span class="title">
  <span>Making young tumors old: a new weapon against cancer?</span>
  <i>Science of aging knowledge environment : SAGE KE</i>
</span>
<span class="authors">Sage, J.</span>

<span class="details">
   <span class="year">2005</span>; <span class="volume">2005 (33)</span><span class="pages">: pe25-?</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">As the population of industrial nations ages, the incidence of cancer and cancer mortality is increasing. Intuitively, older organisms may be less able to cope with accumulated damage and thus be more prone to develop cancer. However, so far, the links between aging and cancer have been only partially explored. Strikingly, four recent studies now indicate that premature senescence accompanied by cell cycle arrest occurs in tumors initiated by an oncogenic mutation. Thus, senescence may act as a key tumor suppressor mechanism in young tumors in vivo.</p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/16107660">PubMedID 16107660</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-34410">
  <cite><span class="title">
  <span>Cell type-specific effects of Rb deletion in the murine retina</span>
  <i>GENES &amp; DEVELOPMENT</i>
</span>
<span class="authors">Macpherson, D., Sage, J., Kim, T., Ho, D., McLaughlin, M. E., Jacks, T.</span>

<span class="details">
   <span class="year">2004</span>; <span class="volume">18 (14)</span><span class="pages">: 1681-1694</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Certain cells of the human retina are extremely sensitive to loss of function of the retinoblastoma tumor suppressor gene RB. Retinoblastomas develop early in life and at high frequency in individuals heterozygous for a germ-line RB mutation, and sporadic retinoblastomas invariably have somatic mutation in the RB gene. In contrast, retinoblastomas do not develop in Rb+/- mice. Although retinoblastoma is thought to have developmental origins, the function of Rb in retinal development has not been fully characterized. Here we studied the role of Rb in normal retinal development and in retinoblastoma using conditional Rb mutations in the mouse. In late embryogenesis, Rb-deficient retinas exhibited ectopic S-phase and high levels of p53-independent apoptosis, particularly in the differentiating retinal ganglion cell layer. During postnatal retinal development, loss of Rb led to more widespread retinal apoptosis, and adults showed loss of photoreceptors and bipolar cells. Conditional Rb mutation in the retina did not result in retinoblastoma formation even in a p53-mutant background. However, on a p107- or p130-deficient background, Rb mutation in the retina caused retinal dysplasia or retinoblastoma.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1101/gad.1203304">DOI 10.1101/gad.1203304</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000222742100005">Web of Science ID 000222742100005</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/15231717">PubMedID 15231717</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-242873">
  <cite><span class="title">
  <span>Discrete signaling pathways participate in RB-dependent responses to chemotherapeutic agents</span>
  <i>ONCOGENE</i>
</span>
<span class="authors">Mayhew, C. N., Perkin, L. M., Zhang, X. P., Sage, J., Jacks, T., Knudsen, E. S.</span>

<span class="details">
   <span class="year">2004</span>; <span class="volume">23 (23)</span><span class="pages">: 4107-4120</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The retinoblastoma (RB) tumor suppressor has been proposed to function as a key mediator of cell cycle checkpoints induced by chemotherapeutic agents. However, these prior studies have relied on embryonic fibroblasts harboring chronic loss of RB, a condition under which compensation of RB functions is known to occur. Here we utilized primary adult fibroblasts derived from mice harboring loxP sites flanking exon 3 of the Rb gene to delineate the action of RB in the chemotherapeutic response. In this system we find that targeted disruption of Rb leads to little overt change in cell cycle distribution. However, these cells exhibited deregulation of RB/E2F target genes and became aneuploid following culture in the absence of RB. When challenged with both DNA damaging and antimetabolite chemotherapeutics, RB was required for primary adult cells to undergo DNA damage checkpoint responses and loss of RB resulted in enhanced aneuploidy following challenge. In contrast, following spontaneous immortalization and the loss of functional p53 signaling, the antimetabolite 5-fluorouracil (5-FU) failed to induce arrest despite the presence of RB. In these immortal cultures RB/E2F targets were deregulated in a complex, gene-specific manner and RB was required for the checkpoint response to camptothecin (CPT). Mechanistic analyses of the checkpoint responses in primary cells indicated that loss of RB leads to increased p53 signaling and decreased viability following both CPT and 5-FU treatment. However, the mechanism through which these agents act to facilitate cell cycle inhibition through RB were distinct. These studies underscore the critical role of RB in DNA-damage checkpoint signaling and demonstrate that RB mediates chemotherapeutic-induced cell cycle inhibition in adult fibroblasts by distinct mechanisms.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1038/sj.onc.1207503">DOI 10.1038/sj.onc.1207503</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000221520200009">Web of Science ID 000221520200009</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/15064736">PubMedID 15064736</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-33256">
  <cite><span class="title">
  <span>Cyclin C makes an entry into the cell cycle</span>
  <i>DEVELOPMENTAL CELL</i>
</span>
<span class="authors">Sage, J.</span>

<span class="details">
   <span class="year">2004</span>; <span class="volume">6 (5)</span><span class="pages">: 607-608</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">From yeast to humans, cell cycle progression is orchestrated by the oscillation of kinase activities associated with cyclins. In an article published recently in Cell, Ren and Rollins investigate mechanisms controlling the G0/G1 transition in quiescent cells and identify new cyclin C/Cdk3 complexes as key regulators of cell cycle reentry in human cells.</p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000222443100001">Web of Science ID 000222443100001</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/15130482">PubMedID 15130482</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-242874">
  <cite><span class="title">
  <span>RB signaling prevents replication-dependent DNA double-strand breaks following genotoxic insult</span>
  <i>NUCLEIC ACIDS RESEARCH</i>
</span>
<span class="authors">Bosco, E. E., Mayhew, C. N., Hennigan, R. F., Sage, J., Jacks, T., Knudsen, E. S.</span>

<span class="details">
   <span class="year">2004</span>; <span class="volume">32 (1)</span><span class="pages">: 25-34</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Cell cycle checkpoints induced by DNA damage play an integral role in preservation of genomic stability by allowing cells to limit the propagation of deleterious mutations. The retinoblastoma tumor suppressor (RB) is crucial for the maintenance of the DNA damage checkpoint function because it elicits cell cycle arrest in response to a variety of genotoxic stresses. Although sporadic loss of RB is characteristic of most cancers and results in the bypass of the DNA damage checkpoint, the consequence of RB loss upon chemotherapeutic responsiveness has been largely uninvestigated. Here, we employed a conditional knockout approach to ablate RB in adult fibroblasts. This system enabled us to examine the DNA damage response of adult cells following acute RB deletion. Using this system, we demonstrated that loss of RB disrupted the DNA damage checkpoint elicited by either cisplatin or camptothecin exposure. Strikingly, this bypass was not associated with enhanced repair, but rather the accumulation of phosphorylated H2AX (gammaH2AX) foci, which indicate DNA double-strand breaks. The formation of gammaH2AX foci was due to ongoing replication following chemotherapeutic treatment in the RB-deficient cells. Additionally, peak gammaH2AX accumulation occurred in S-phase cells undergoing DNA replication in the presence of damage, and these gammaH2AX foci co-localized with replication foci. These results demonstrate that acute RB loss abrogates DNA damage-induced cell cycle arrest to induce gammaH2AX foci formation. Thus, secondary genetic lesions induced by RB loss have implications for the chemotherapeutic response and the development of genetic instability.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1093/nar/gkg919">DOI 10.1093/nar/gkg919</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000188988700018">Web of Science ID 000188988700018</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/14704340">PubMedID 14704340</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-242875">
  <cite><span class="title">
  <span>Recapitulation of the effects of the human papillomavirus type 16 E7 oncogene on mouse epithelium by somatic Rb deletion and detection of pRb-independent effects of E7 in vivo</span>
  <i>MOLECULAR AND CELLULAR BIOLOGY</i>
</span>
<span class="authors">Balsitis, S. J., Sage, J., Duensing, S., Munger, K., Jacks, T., Lambert, P. F.</span>

<span class="details">
   <span class="year">2003</span>; <span class="volume">23 (24)</span><span class="pages">: 9094-9103</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Although the human papillomavirus (HPV) E7 oncogene is known to contribute to the development of human cervical cancer, the mechanisms of its carcinogenesis are poorly understood. The first identified and most recognized function of E7 is its binding to and inactivation of the retinoblastoma tumor suppressor (pRb), but at least 18 other biological activities have also been reported for E7. Thus, it remains unclear which of these many activities contribute to the oncogenic potential of E7. We used a Cre-lox system to abolish pRb expression in the epidermis of transgenic mice and compared the outcome with the effects of E7 expression in the same tissue at early ages. Mice lacking pRb in epidermis showed epithelial hyperplasia, aberrant DNA synthesis, and improper differentiation. In addition, Rb-deleted epidermis (i.e., epidermis composed of cells with Rb deleted) exhibited centrosomal abnormalities and failed to arrest the cell cycle in response to ionizing radiation. Transgenic mice expressing E7 in skin display the same range of phenotypes. In sum, few differences were detected between Rb-deleted epidermis and E7-expressing epidermis in young mice. However, when both E7 was expressed and Rb was deleted in the same tissue, increased hyperplasia and dysplasia were observed. These findings indicate that inactivation of the Rb pathway can largely account for E7's phenotypes at an early age, but that pRb-independent activities of E7 are detectable in vivo.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1128/MBC.23.24.9094-9103.2003">DOI 10.1128/MBC.23.24.9094-9103.2003</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000187040400017">Web of Science ID 000187040400017</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/14645521">PubMedID 14645521</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-29469">
  <cite><span class="title">
  <span>Perp is a mediator of p53-dependent apoptosis in diverse cell types</span>
  <i>CURRENT BIOLOGY</i>
</span>
<span class="authors">Ihrie, R. A., Reczek, E., Horner, J. S., Khachatrian, L., Sage, J., Jacks, T., Attardi, L. D.</span>

<span class="details">
   <span class="year">2003</span>; <span class="volume">13 (22)</span><span class="pages">: 1985-1990</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The induction of apoptosis by the p53 protein is critical for its activity as a tumor suppressor. Although it is clear that p53 induces apoptosis at least in part by direct transcriptional activation of target genes, the set of p53 target genes that mediate p53 function in apoptosis in vivo remains to be well defined. The Perp (p53 apoptosis effector related to PMP-22) gene is highly expressed in cells undergoing p53-dependent apoptosis as compared to cells undergoing p53-dependent G1 arrest. Perp is a direct p53 target, and its overexpression is sufficient to induce cell death in fibroblasts, implicating it as an important component of p53 apoptotic function. Here, through the generation of Perp-deficient mice, we analyze the role of Perp in the p53 apoptosis pathway in multiple primary cell types by comparing the cell death responses of Perp null cells to those of wild-type and p53 null cells. These experiments demonstrate the involvement of Perp in p53-mediated cell death in thymocytes and neurons but not in E1A-expressing MEFs, indicating a cell type-specific role for Perp in the p53 cell death pathway. In addition, we show that Perp is not required for proliferation-associated functions of p53. Thus, Perp selectively mediates the p53 apoptotic response, and the requirement for Perp is dictated by cellular context.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1016/j.cub.2003.10.055">DOI 10.1016/j.cub.2003.10.055</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000186558100026">Web of Science ID 000186558100026</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/14614825">PubMedID 14614825</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-24327">
  <cite><span class="title">
  <span>Acute mutation of retinoblastoma gene function is sufficient for cell cycle re-entry</span>
  <i>NATURE</i>
</span>
<span class="authors">Sage, J., Miller, A. L., Perez-Mancera, P. A., Wysocki, J. M., Jacks, T.</span>

<span class="details">
   <span class="year">2003</span>; <span class="volume">424 (6945)</span><span class="pages">: 223-228</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Cancer cells arise from normal cells through the acquisition of a series of mutations in oncogenes and tumour suppressor genes. Mouse models of human cancer often rely on germline alterations that activate or inactivate genes of interest. One limitation of this approach is that germline mutations might have effects other than somatic mutations, owing to developmental compensation. To model sporadic cancers associated with inactivation of the retinoblastoma (RB) tumour suppressor gene in humans, we have produced a conditional allele of the mouse Rb gene. We show here that acute loss of Rb in primary quiescent cells is sufficient for cell cycle entry and has phenotypic consequences different from germline loss of Rb function. This difference is explained in part by functional compensation by the Rb-related gene p107. We also show that acute loss of Rb in senescent cells leads to reversal of the cellular senescence programme. Thus, the use of conditional knockout strategies might refine our understanding of gene function and help to model human cancer more accurately.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1038/nature01764">DOI 10.1038/nature01764</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000184032700049">Web of Science ID 000184032700049</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/12853964">PubMedID 12853964</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-21084">
  <cite><span class="title">
  <span>Conditional mutation of Rb causes cell cycle defects without apoptosis in the central nervous system</span>
  <i>MOLECULAR AND CELLULAR BIOLOGY</i>
</span>
<span class="authors">Macpherson, D., Sage, J., Crowley, D., Trumpp, A., Bronson, R. T., Jacks, T.</span>

<span class="details">
   <span class="year">2003</span>; <span class="volume">23 (3)</span><span class="pages">: 1044-1053</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Targeted disruption of the retinoblastoma gene in mice leads to embryonic lethality in midgestation accompanied by defective erythropoiesis. Rb(-/-) embryos also exhibit inappropriate cell cycle activity and apoptosis in the central nervous system (CNS), peripheral nervous system (PNS), and ocular lens. Loss of p53 can prevent the apoptosis in the CNS and lens; however, the specific signals leading to p53 activation have not been determined. Here we test the hypothesis that hypoxia caused by defective erythropoiesis in Rb-null embryos contributes to p53-dependent apoptosis. We show evidence of hypoxia in CNS tissue from Rb(-/-) embryos. The Cre-loxP system was then used to generate embryos in which Rb was deleted in the CNS, PNS and lens, in the presence of normal erythropoiesis. In contrast to the massive CNS apoptosis in Rb-null embryos at embryonic day 13.5 (E13.5), conditional mutants did not have elevated apoptosis in this tissue. There was still significant apoptosis in the PNS and lens, however. Rb(-/-) cells in the CNS, PNS, and lens underwent inappropriate S-phase entry in the conditional mutants at E13.5. By E18.5, conditional mutants had increased brain size and weight as well as defects in skeletal muscle development. These data support a model in which hypoxia is a necessary cofactor in the death of CNS neurons in the developing Rb mutant embryo.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1128/MCB.23.3.1044-1053.2003">DOI 10.1128/MCB.23.3.1044-1053.2003</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000180673600026">Web of Science ID 000180673600026</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/12529408">PubMedID 12529408</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-242876">
  <cite><span class="title">
  <span>An induced Ets repressor complex regulates growth arrest during terminal macrophage differentiation</span>
  <i>CELL</i>
</span>
<span class="authors">Klappacher, G. W., Lunyak, V. V., Sykes, D. B., Sawka-Verhelle, D., Sage, J., Brard, G., Ngo, S. D., Gangadharan, D., Jacks, T., Kamps, M. P., Rose, D. W., Rosenfeld, M. G., Glass, C. K.</span>

<span class="details">
   <span class="year">2002</span>; <span class="volume">109 (2)</span><span class="pages">: 169-180</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Defining the molecular mechanisms that coordinately regulate proliferation and differentiation is a central issue in development. Here, we describe a mechanism in which induction of the Ets repressor METS/PE1 links terminal differentiation to cell cycle arrest. Using macrophages as a model, we provide evidence that METS/PE1 blocks Ras-dependent proliferation without inhibiting Ras-dependent expression of cell type-specific genes by selectively replacing Ets activators on the promoters of cell cycle control genes. Antiproliferative effects of METS require its interaction with DP103, a DEAD box-containing protein that assembles a novel corepressor complex. Functional interactions between the METS/DP103 complex and E2F/ pRB family proteins are also necessary for inhibition of cellular proliferation, suggesting a combinatorial code that directs permanent cell cycle exit during terminal differentiation.</p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000175082600007">Web of Science ID 000175082600007</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/12007404">PubMedID 12007404</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-242877">
  <cite><span class="title">
  <span>Targeted point mutations of p53 lead to dominant-negative inhibition of wild-type p53 function</span>
  <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i>
</span>
<span class="authors">de Vries, A., Flores, E. R., Miranda, B., Hsieh, H. M., van Oostrom, C. T., Sage, J., Jacks, T.</span>

<span class="details">
   <span class="year">2002</span>; <span class="volume">99 (5)</span><span class="pages">: 2948-2953</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The p53 tumor suppressor gene is the most frequently mutated gene in human cancers, and germ-line p53 mutations cause a familial predisposition for cancer. Germ-line or sporadic p53 mutations are usually missense and typically affect the central DNA-binding domain of the protein. Because p53 functions as a tetrameric transcription factor, mutant p53 is thought to inhibit the function of wild-type p53 protein. Here, we studied the possible dominant-negative inhibition of wild-type p53 protein by two different, frequently occurring point mutations. The R270H and P275S mutations were targeted into the genome of mouse embryonic stem cells to allow the analysis of the effects of the mutant proteins expressed in normal cells at single-copy levels. In embryonic stem cells, the presence of a heterozygous point-mutated allele resulted in delayed transcriptional activation of several p53 downstream target genes on exposure to gamma irradiation. Doxorubicin-induced apoptosis was severely affected in the mutant embryonic stem cells compared with wild-type cells. Heterozygous mutant thymocytes had a severe defect in p53-dependent apoptotic pathways after treatment with gamma irradiation or doxorubicin, whereas p53-independent apoptotic pathways were intact. Together these data demonstrate that physiological expression of point-mutated p53 can strongly limit overall cellular p53 function, supporting the dominant-negative action of such mutants. Also, cells heterozygous for such mutations may be compromised in terms of tumor suppression and response to chemotherapeutic agents.</p>
  <p class="doi">
    <span>View details for <a href="https://dx.doi.org/10.1073/pnas.052713099">DOI 10.1073/pnas.052713099</a></span>
  </p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000174284600065">Web of Science ID 000174284600065</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/11867759">PubMedID 11867759</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-242878">
  <cite><span class="title">
  <span>The RB gene family and control of the cell cycle</span>
  <i>BULLETIN DU CANCER</i>
</span>
<span class="authors">Sage, J.</span>

<span class="details">
   <span class="year">2001</span>; <span class="volume">88 (6)</span><span class="pages">: 541-543</span>
</span></cite>
    <div class="detail"><p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000169993400001">Web of Science ID 000169993400001</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/11459699">PubMedID 11459699</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-8961">
  <cite><span class="title">
  <span>Targeted disruption of the three Rb-related genes leads to loss of G(1) control and immortalization</span>
  <i>GENES &amp; DEVELOPMENT</i>
</span>
<span class="authors">Sage, J., Mulligan, G. J., Attardi, L. D., Miller, A., Chen, S. Q., Williams, B., Theodorou, E., Jacks, T.</span>

<span class="details">
   <span class="year">2000</span>; <span class="volume">14 (23)</span><span class="pages">: 3037-3050</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The retinoblastoma protein, pRB, and the closely related proteins p107 and p130 are important regulators of the mammalian cell cycle. Biochemical and genetic studies have demonstrated overlapping as well as distinct functions for the three proteins in cell cycle control and mouse development. However, the role of the pRB family as a whole in the regulation of cell proliferation, cell death, or cell differentiation is not known. We generated embryonic stem (ES) cells and other cell types mutant for all three genes. Triple knock-out mouse embryonic fibroblasts (TKO MEFs) had a shorter cell cycle than wild-type, single, or double knock-out control cells. TKO cells were resistant to G(1) arrest following DNA damage, despite retaining functional p53 activity. They were also insensitive to G(1) arrest signals following contact inhibition or serum starvation. Finally, TKO MEFs did not undergo senescence in culture and do possess some characteristics of transformed cells. Our results confirm the essential role of the Rb family in the control of the G(1)/S transition, place the three Rb family members downstream of multiple cell cycle control pathways, and further the link between loss of cell cycle control and tumorigenesis.</p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000165788600011">Web of Science ID 000165788600011</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/11114892">PubMedID 11114892</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-242879">
  <cite><span class="title">
  <span>Sex hormone-induced carcinogenesis in Rb-deficient prostate tissue</span>
  <i>CANCER RESEARCH</i>
</span>
<span class="authors">Wang, Y. H., Hayward, S. W., Donjacour, A. A., Young, P., Jacks, T., Sage, J., Dahiya, R., Cardiff, R. D., Day, M. L., Cunha, G. R.</span>

<span class="details">
   <span class="year">2000</span>; <span class="volume">60 (21)</span><span class="pages">: 6008-6017</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The retinoblastoma (Rb) gene product is a prototypic tumor suppressor. Mice lacking the Rb gene are not viable and die in utero at approximately 13 days of gestation. In this study, we have rescued Rb-/- prostates by grafting pelvic organ rudiments from Rb-/- mouse embryos under the renal capsule of adult male nude mouse hosts. Grafts of embryonic pelvic organs developed into functional prostatic tissue. Some of the prostatic tissue generated was further used to construct chimeric prostatic tissue recombinants by combining wild-type rat urogenital mesenchyme (rUGM) with Rb-/- and Rb+/+ prostatic epithelium (PRE). The tissue recombinants were grown as subcapsular renal grafts and treated from the time of grafting with Silastic capsules containing 25 mg of testosterone plus 2.5 mg of estradiol. During 5-8 weeks of hormone treatment, rUGM+Rb+/+PRE tissue recombinants developed prostatic hyperplasia, whereas PRE in rUGM+Rb-/-PRE tissue recombinants developed hyperplasia, atypical hyperplasia, and carcinoma. During carcinogenesis in rUGM+Rb-/-PRE tissue recombinants, prostatic epithelial cells of the basal lineage disappeared, whereas the luminal cells underwent carcinogenesis. Epithelial E-cadherin almost totally disappeared. In all cases, epithelial PCNA labeling was elevated in tissue recombinants containing Rb-/- versus Rb+/+ epithelium. These epithelial changes were associated with almost total loss of smooth muscle cells in the stroma. In contrast, in untreated hosts rUGM+Rb+/+PRE tissue recombinants developed normally, and rUGM+Rb-/-PRE tissue recombinants developed mild epithelial hyperplasia. The results of this study demonstrate that Rb-/- prostatic tissue can be rescued from embryonic lethal mice and used to test its susceptibility to hormonal carcinogenesis. Deletion of the Rb gene predisposes prostatic epithelium to hyperplasia and increases proliferative activity Susceptibility to hormonal carcinogenesis in response to exogenous testosterone + estradiol is manifested in the progression from atypica hyperplasia to carcinoma. Thus, these findings demonstrate that the absence of the Rb tumor suppressor gene may predispose prostatic epithelial cells to carcinogenesis. Rescue of organs from Rb-/- embryos not only provides an opportunity to analyze the Rb gene pathway in the development and progression of prostate cancer but also provides an opportunity for specifically evaluating the role of the Rb pathway in development and carcinogenesis in other organs, such as the mammary gland and colon. Because rUGM greatly stimulates prostatic epithelial proliferation, the tissue recombinant model is a particularly useful tool for assessing the functional role of other genes in prostatic carcinogenesis through use of the appropriate transgenic or gene knockout mice.</p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000165230300021">Web of Science ID 000165230300021</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/11085521">PubMedID 11085521</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-242880">
  <cite><span class="title">
  <span>NF-kappa B is developmentally regulated during spermatogenesis in mice</span>
  <i>DEVELOPMENTAL DYNAMICS</i>
</span>
<span class="authors">Lilienbaum, A., Sage, J., Memet, S., Rassoulzadegan, M., Cuzin, F., Israel, A.</span>

<span class="details">
   <span class="year">2000</span>; <span class="volume">219 (3)</span><span class="pages">: 333-340</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">To analyze NF-kappa B activity in the testis, we used murine transgenic lines carrying a LacZ reporter gene under the control of a NF-kappa B-responsive promoter (Schmidt-Ullrich et al. [1996] Dev 122:2117-2128). We constructed three independent lines containing the promoter of the gene encoding p105, the precursor of the p50 subunit. This promoter contains three NF-kappa B-binding sites in its proximal part. Our results show that in adult mice, the beta-galactosidase activity which reflects nuclear NF-kappa B activity, is first detected in spermatocytes at the pachytene stage and remains activated in the following steps of germ cell differentiation and maturation. Using transgenic mice carrying a p105nlslacZ construct with the 3 NF-kappa B sites mutated in the p105 promoter, we found a significant reduction in the transgene activity, confirming the important role of NF-kappa B in the activation of the transgene. To confirm the stage of induction during spermatogenesis, we analysed the beta-galactosidase activity in the testes from prepuberal mice in which cells synchrouneously enter meiosis. We detected the transgene activity at 18 days after birth, corresponding to the pachytene stage in spermatocytes. In nuclear extracts prepared from prepuberal mice, we found a peak of NF-kappa B DNA-binding activity made of p50 and p65 subunits at day 18 after birth, which remains high in the later stages. Further analysis showed that I kappa B alpha and beta, but not epsilon are expressed in the testes. Altogether, these data suggest that NF-kappa B factors are stage specifically controlled and may play a role during the development of sperm cells.</p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000165122200005">Web of Science ID 000165122200005</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/11066090">PubMedID 11066090</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-242881">
  <cite><span class="title">
  <span>Cell cycle inhibition by the anti-angiogenic agent TNP-470 is mediated by p53 and p21(WAF1/CIP1)</span>
  <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i>
</span>
<span class="authors">Zhang, Y., Griffith, E. C., Sage, J., Jacks, T., Liu, J. O.</span>

<span class="details">
   <span class="year">2000</span>; <span class="volume">97 (12)</span><span class="pages">: 6427-6432</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Angiogenesis has been demonstrated to be essential for tumor growth and metastasis, and inhibition of angiogenesis is emerging as a promising strategy for treating cancer. Among the most potent inhibitors of angiogenesis is the fumagillin family of natural products. An analog of fumagillin, known as TNP-470 or AGM-1470, has been undergoing clinical trials for treating a variety of cancers. TNP-470 has been shown to block endothelial cell cycle progression in the late G(1) phase. Although the direct molecular target for TNP-470 has been identified as the type 2 methionine aminopeptidase (MetAP2), how inhibition of this enzyme leads to cell cycle arrest has remained unclear. We report that treatment of endothelial and other drug-sensitive cell types leads to the activation of the p53 pathway, causing an accumulation of the G(1) cyclin-dependent kinase inhibitor p21(WAF1/CIP1). The requirement of p53 and p21(WAF1/CIP1) for the cell cycle inhibition by TNP-470 is underscored by the observation that cells deficient in p53 and p21(WAF1/CIP1) are resistant to TNP-470. These results shed significant light on the mechanism of cell cycle inhibition by TNP-470 and suggest an alternative method of activating p53 in endothelial cells to halt angiogenesis and tumor progression.</p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000087526300037">Web of Science ID 000087526300037</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/10841547">PubMedID 10841547</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-242883">
  <cite><span class="title">
  <span>Temporal and spatial control of the Sycp1 gene transcription in the mouse meiosis: regulatory elements active in the male are not sufficient for expression in the female gonad</span>
  <i>MECHANISMS OF DEVELOPMENT</i>
</span>
<span class="authors">Sage, J., Martin, L., Meuwissen, R., Heyting, C., Cuzin, F., Rassoulzadegan, M.</span>

<span class="details">
   <span class="year">1999</span>; <span class="volume">80 (1)</span><span class="pages">: 29-39</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Transcription controls active at the initial stages of meiosis are clearly key elements in the regulation of germinal differentiation. Transcription of the Sycp1 gene (synaptonemal complex protein 1) starts as early as the leptotene and zygotene stages. Constructs with Sycp1 5' upstream sequences directed the expression of reporter genes to pachytene spermatocytes in transgenic mice. A short fragment encompassing the transcription start (n.t. -54 to +102) was sufficient for stage-specific expression in the adult male and for temporal regulation during development. Upstream enhancer element(s) quantitatively regulating expression were localized in the region between -54 and -260. The gene is normally expressed both in the male and female gonads, but none of the promoter sequences active in the testis allowed the expression of reporter genes during meiosis in the ovary.</p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000078473700003">Web of Science ID 000078473700003</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/10096061">PubMedID 10096061</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-242886">
  <cite><span class="title">
  <span>Cre expression in primary spermatocytes: A tool for genetic engineering of the germ line</span>
  <i>MOLECULAR REPRODUCTION AND DEVELOPMENT</i>
</span>
<span class="authors">Vidal, F., Sage, J., Cuzin, F., Rassoulzadegan, M.</span>

<span class="details">
   <span class="year">1998</span>; <span class="volume">51 (3)</span><span class="pages">: 274-280</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Transgenic mice were generated expressing a testicular Cre recombinase driven by promoter sequences derived from the gene encoding Synaptonemal Complex Protein 1 (Sycp1), expressed at an early stage of the male meiosis (leptotene to zygotene). Recombination at target LoxP sites was examined during germinal differentiation in mice harboring Sycp1-Cre and a second transgene where LoxP sites flank either the beta geo coding region, the Pgk1 promoter, or a tk-neo cassette inserted into the Rxr alpha locus. The LoxP-flanked transgenes were stably maintained in the somatic tissues of the double transgenic animals, as well as in the progeny of the females. Mice born after mating the double-transgenic males with normal females showed extensive deletions of the LoxP-flanked sequences. When the males were hemizygous for the Sycp1-Cre transgene, the deletions were observed even in the fraction of the offspring which had not inherited the Cre gene, thus demonstrating that expression occurred in the male parent during spermatogenesis. The high efficiency of excision at the LoxP sites makes the Sycp1-Cre transgenic males suitable for evaluating the role of defined gene functions in the germinal differentiation process.</p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000076145500006">Web of Science ID 000076145500006</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/9771647">PubMedID 9771647</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-242885">
  <cite><span class="title">
  <span>Stage-specific expression of the Kit receptor and its ligand (KL) during male gametogenesis in the mouse: a Kit-KL interaction critical for meiosis</span>
  <i>DEVELOPMENT</i>
</span>
<span class="authors">Vincent, S., Segretain, D., Nishikawa, S., Nishikawa, S., Sage, J., Cuzin, F., Rassoulzadegan, M.</span>

<span class="details">
   <span class="year">1998</span>; <span class="volume">125 (22)</span><span class="pages">: 4585-4593</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">The Kit receptor and its ligand KL, which together constitute an essential effector at various stages of embryonic development, are both present during adult gametogenesis. In the testis, KL is expressed in Sertoli cells, and Kit in germ cells, starting at the premeiotic stages. A series of observations indicated previously a role in spermatogonia survival, without excluding a possible function at later stages. We identified a complex pattern of expression of the two components in the adult murine testis, suggestive of a role in the meiotic progression of spermatocytes. At stages VII-VIII of the cycle of the seminiferous epithelium, the time when spermatocytes enter meiosis, the membrane-associated form of KL extends on the Sertoli cell from the peripheral to the adluminal compartment of the tubule. We also found that the receptor is present on the surface of germ cells up to the pachytene stage. The availability of differentiated Sertoli cell lines, which express the KL protein and support part of the maturation of germ cells in coculture, allowed us to ask whether, in the in vitro reconstructed system, transit of spermatocytes through meiosis requires the Kit-KL interaction. Addition of a blocking monoclonal antibody against the Kit receptor (ACK2) inhibited extensively the appearance of haploid cells and the expression of a haploid-phase-specific gene (Prm1). Recognition of the supporting Sertoli cell by germ cells was not affected, indicating a requirement for the activity of the receptor for either entering or completing meiosis. Involvement of the membrane-associated form of the ligand was suggested by the observation that addition of the soluble form of KL was equally inhibitory.</p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000077649200022">Web of Science ID 000077649200022</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/9778516">PubMedID 9778516</a></span>
  </p></div>
</li>
    <li class="publication  article" id="pubId-242889">
  <cite><span class="title">
  <span>Stage-specific signals in germ line differentiation: Control of Sertoli cell phagocytic activity by spermatogenic cells</span>
  <i>DEVELOPMENTAL BIOLOGY</i>
</span>
<span class="authors">Grandjean, V., Sage, J., Ranc, F., Cuzin, F., Rassoulzadegan, M.</span>

<span class="details">
   <span class="year">1997</span>; <span class="volume">184 (1)</span><span class="pages">: 165-174</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">Differentiation of male germ cells requires a continuous cross-talk with their somatic support, the Sertoli cell. An in vitro model of Sertoli cells was recently provided by established cell lines which maintain Sertoli-specific characteristics, among which is a regulated phagocytic capacity. In vivo, Sertoli cells take up the residual cytoplasm expelled from the maturing sperm, a process restricted to a limited period of germinal maturation, and they also eliminate abnormally differentiated germ cells in case of hormonal deficiency. Cells of the Sertoli line efficiently take up latex beads, as well as dead cells in the cultures. A semiquantitative assay of phagocytosis was developed, based on the uptake of fluorescent latex beads. 15P-1 cultures were found to contain a minor fraction of active phagocytes. After addition of a defined fraction of germ cells, however, all the cells internalized beads as efficiently as macrophages. The inducing cell was identified as the pachytene spermatocyte, a cell type which, in vivo, is associated with Sertoli cells when they express their phagocytic potential. These inducing meiotic cells were not internalized themselves. Rather, they interacted with Sertoli cells via a surface signal that was resistant to formaldehyde fixation. The whole induction process does not involve changes in Sertoli gene expression, since it occurs even in the presence of high doses of cycloheximide. After the required initial contact, further maintenance of the activity was dependent on factor(s) secreted in the medium of the activated culture. Phagocytosis was, on the other hand, abrogated in the presence of factor(s) secreted by a distinct fraction of germ cells, enriched in the late stages (second division) of meiosis.</p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/A1997WV15600015">Web of Science ID A1997WV15600015</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/9142992">PubMedID 9142992</a></span>
  </p></div>
</li>
    <li class="publication featured article" id="pubId-242891">
  <cite><span class="title">
  <span>TRANSMEIOTIC DIFFERENTIATION OF MALE GERM-CELLS IN CULTURE</span>
  <i>CELL</i>
</span>
<span class="authors">Rassoulzadegan, M., PAQUISFLUCKLINGER, V., Bertino, B., Sage, J., Jasin, M., Miyagawa, K., VANHEYNINGEN, V., Besmer, P., Cuzin, F.</span>

<span class="details">
   <span class="year">1993</span>; <span class="volume">75 (5)</span><span class="pages">: 997-1006</span>
</span></cite>
    <div class="detail"><h4>Abstract</h4>
  <p class="abstract">A cell culture system that supports the differentiation of male germ cells through meiosis is described. It takes advantage of the properties of a cell line, 15P-1, established from testicular cells of transgenic mice that express the large T protein of polyoma virus in the seminiferous epithelium. This line exhibits features characteristics of Sertoli cells, including transcription of the Wilms' tumor (WT1) and Steel genes. Cells of the 15P-1 type support the meiotic and postmeiotic differentiation in cocultures of diploid premeiotic germ cells into haploid spermatids expressing the protamine (Prm-1) gene. When cocultured with 15P-1 cells, testicular cells explanted from immature 9-day-old animals, before the onset of the first meiosis, generated tetrads of haploid cells with the morphology of round spermatids and initiated protamine transcription.</p>
  <p class="wos">
    <span>View details for <a href="https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/A1993MK96600019">Web of Science ID A1993MK96600019</a></span>
  </p>
  <p class="pub-med">
    <span>View details for <a href="https://www.ncbi.nlm.nih.gov/pubmed/7504588">PubMedID 7504588</a></span>
  </p></div>
</li>
</ul>
    </div>
    <div id="journalArticlesContent"></div>
    <div id="booksContent"></div>
    <div id="proceedingsContent"></div>
    <div id="presentationsContent"></div>
    <div id="paginationContent"></div>
  </div>
  <div class="span4 side-bar">
    <div id="navigatorContent" class="content-section navigator color-scheme-pub bg-icon-pub"></div>
    <div id="similarProfilesContent" class="content-section similar-profiles"></div>
    <div id="tagCloudContent" class="content-section tag-cloud"></div>
  </div>
</div>
      </div>
  </div>
</div>
</section>
      </div>
      
      <div id="breadcrumbs">
        <h4 class="nodisplay">You are Here:</h4>
        


      
      
      
        
      
        
        
        
        
        
        
        
        
          
        
        
          <h5 class="nodisplay">Current Tab: <span id="tabOverride">6</span></h5>
<a href="http://med.stanford.edu/">School of Medicine</a> Â» <a href="/profiles/">Community Academic Profiles </a>


        
        
        
        
      
      
      </div>

      <div id="subNav">
        


<!-- subNav: leftNavDept -->


	
	
    
    
    
       
    
	



        <!-- subNav: leftNavFaculty -->
        
        


        <!-- subNav: leftNavStatic -->
        
        
        
      </div>


      



        
        <div id="nav">
	<h4 class="nodisplay">Site Navigation:</h4>
	<ul id="navList"><li class="x_firstChild"><a href="http://hesc.stanford.edu/">Home</a></li>
      <li><a href="http://hesc.stanford.edu/aboutus/">About Us</a>
			<ul>
				<li><a href="http://hesc.stanford.edu/aboutus/">Overview</a></li>
				<li><a href="http://hesc.stanford.edu/aboutus/profiles.html">Faculty Profiles</a></li>
				<li><a href="http://hesc.stanford.edu/aboutus/resources.html">Links</a></li>		
			    <li><a href="http://hesc.stanford.edu/aboutus/contact.html">Contact Us</a></li>
			    <li><a href="http://hesc.stanford.edu/aboutus/sitemap.html">Site Map</a></li>
			</ul>
		</li>
		<li><a href="http://hesc.stanford.edu/education/">Education</a>
		  <ul>
				<li><a href="http://hesc.stanford.edu/education/">Overview</a></li>
			    <li class="x_hasChildren"><a href="http://hesc.stanford.edu/education/courses/">Training Courses</a>
			      <ul>
			        <li><a href="http://hesc.stanford.edu/education/courses/stemcell.html">Basic hESC Biology</a></li>
				    <li><a href="http://hesc.stanford.edu/education/courses/scnt.html">Reprogramming and SCNT</a></li>
				    <li><a href="http://hesc.stanford.edu/education/courses/derivation.html">Stem Cell Derivation</a></li>
				      <li><a href="http://hesc.stanford.edu/education/courses/embryology.html">Human Embryology</a></li>
		          </ul>
	        </li>
		        <li><a href="http://hesc.stanford.edu/education/schedules.html">Training Schedules</a></li>
		        <li><a href="http://hesc.stanford.edu/education/apply.html">Apply for Training</a></li>
		        <li><a href="http://hesc.stanford.edu/education/seminars.html">Seminars and Other Academic Courses</a></li>
		  </ul>
		</li>
		<li><a href="http://hesc.stanford.edu/research/">Research</a>
			<ul>
				<li><a href="http://hesc.stanford.edu/research/">Overview</a></li>
				<li class="x_hasChildren"><a href="http://hesc.stanford.edu/research/programs/"> Areas of Focus</a>
				  <ul id="arbitraryNavListResearch1">
				    <li><a href="http://hesc.stanford.edu/research/programs/human.html">Early Human Embryo Development</a></li>
			        <li><a href="http://hesc.stanford.edu/research/programs/stemcell.html">Human ES Cell Biology</a></li>
				    <li><a href="http://hesc.stanford.edu/research/programs/scnt.html">Reprogramming and SCNT</a></li>
				    <li><a href="http://hesc.stanford.edu/research/programs/germcell.html">Germ Cell Development</a></li>
			      </ul>
				</li>
				<li class="x_hasChildren"><a href="http://hesc.stanford.edu/research/corefacility/">Core Facility</a>
				  <ul id="arbitraryNavListResearch2">
				    <li><a href="http://hesc.stanford.edu/research/corefacility/distributions.html">hESC Line Distribution</a></li>
			        <li><a href="http://hesc.stanford.edu/research/corefacility/bank.html">Human Embryo, Oocyte, hESC and Somatic Cell Bank</a></li>
				    <li><a href="http://hesc.stanford.edu/research/corefacility/sky.html">Spectral Karyotype (SKY)</a></li>
				    <li><a href="http://hesc.stanford.edu/research/corefacility/downloads.html">MTAs &amp; Downloads</a></li>
				  </ul>
				</li>		
		    </ul>
		</li>
		<li class="x_lastChild"><a href="http://hesc.stanford.edu/donations/">Donations</a>
	    <ul>
				<li><a href="http://hesc.stanford.edu/donations/">Giving to the Institute</a></li>
				<li><a href="http://hesc.stanford.edu/donations/tissue.html">Tissue Donations</a></li>
			</ul>
		</li>
	</ul>
</div>

        <div id="localFooter">
	<h4 class="nodisplay">Footer Links:</h4>
	<ul id="localFooterList">
		<li class="x_firstChild"><a href="http://hesc.stanford.edu/aboutus/contact.html">Contact Us</a></li>
	  <li><a href="http://hesc.stanford.edu/aboutus/faqs.html">FAQs</a></li>
	  <li class="x_lastChild"><a href="http://hesc.stanford.edu/donations/">How to Help</a></li>
  </ul>
</div>



        
<div id="globalFooter">
	<ul id="globalFooterList">
		<li class="x_firstChild">Â©<a href="http://med.stanford.edu/"><span class="currentYear">2017</span> Stanford Medicine</a></li>
		<li><a href="http://www.stanford.edu/site/terms.html">Terms of Use</a></li>
	</ul>
</div>


      <div id="superheader">
	<h4 class="nodisplay">Stanford Medicine Resources:</h4>
	<ul id="superheaderList">
		<li id="stanfordMedicineBrand"><a href="http://med.stanford.edu/" title="Stanford Medicine"><b>Stanford Medicine</b></a></li>
		<li id="sh_gettingCare" class="x_hasChildren">
			<a href="http://med.stanford.edu/health-care.html">Getting Care</a>
			<div class="persist">
				<ul>
					<li><a href="http://med.stanford.edu/health-care.html">Overview</a></li>
					<li><a href="http://med.stanford.edu/health-care.html">Find a Physician</a></li>
					<li><a href="http://med.stanford.edu/health-care.html">Find a Clinical Care Center</a></li>
					<li><a href="http://stanfordhospital.org/">Stanford Hospital &amp; Clinics</a></li>
					<li><a href="http://www.stanfordchildrens.org/">Lucile Packard Children's Hospital</a></li>
					<li><a href="http://stanfordhospital.org/clinicsmedServices/medicalServices/emergencyroom/">Emergency</a></li>
				</ul>
			</div>
		</li>
		<li id="sh_research" class="x_hasChildren">
			<a href="http://med.stanford.edu/research.html">Research</a>
			<div class="persist">
				<ul>
					<li><a href="http://med.stanford.edu/research.html">Overview</a></li>
					<li><a href="http://med.stanford.edu/school.html">School of Medicine</a></li>
					<li><a href="http://med.stanford.edu/news.html">News &amp; Resources</a></li>
					<li><a href="http://med.stanford.edu/clinicaltrials/">Clinical Trials</a></li>
					<li><a href="http://med.stanford.edu/school/directory.html">Departments, Institutes &amp; Centers</a></li>
					<li><a href="https://med.stanford.edu/profiles/">Faculty Profiles</a></li>
				</ul>
			</div>
		</li>
		<li id="sh_education" class="x_hasChildren">
			<a href="http://med.stanford.edu/education.html">Education &amp; Training</a>
			<div class="persist">
				<ul>
					<li><a href="http://med.stanford.edu/education.html">Overview</a></li>
					<li><a href="http://med.stanford.edu/school.html">School of Medicine</a></li>
					<li><a href="http://med.stanford.edu/education.html">Programs</a></li>
					<li><a href="http://med.stanford.edu/education.html">Admissions</a></li>
					<li><a href="http://cme.stanford.edu/">Continuing Medical Education</a></li>
					<li><a href="http://med.stanford.edu/alumni/">Alumni</a></li>
					<li><a href="http://lane.stanford.edu/">Lane Medical Library</a></li>
				</ul>
			</div>
		</li>
		<li id="sh_community" class="x_hasChildren">
			<a href="http://med.stanford.edu/about/community-engagement.html">Community</a>
			<div class="persist">
				<ul>
					<li><a href="http://med.stanford.edu/about/community-engagement.html">Overview</a></li>
					<li><a href="http://healthlibrary.stanford.edu/">Stanford Health Library</a></li>
					<li><a href="http://stanfordmedicine.org/communitynews/">Community Newsletter</a></li>
					<li><a href="http://med.stanford.edu/about/how-to-help.html">Volunteering</a></li>
					<li><a href="http://med.stanford.edu/about/community-engagement.html">Public Service &amp; Community Partnerships</a></li>
					<li><a href="http://med.stanford.edu/about/community-engagement.html">Renewal &amp; New Building Projects</a></li>
				</ul>
			</div>
		</li>
		<li id="sh_aboutUs" class="x_hasChildren">
			<a href="http://med.stanford.edu/about.html">About Us</a>
			<div class="persist">
				<ul>
					<li><a href="http://med.stanford.edu/about.html">Overview</a></li>
					<li><a href="http://med.stanford.edu/news.html">News</a></li>
					<li><a href="http://stanfordcareers.stanford.edu/">Careers</a></li>
					<li><a href="http://med.stanford.edu/about/how-to-help.html">Ways to Give</a></li>
					<li><a href="http://med.stanford.edu/about/contacts.html">Find a Person</a></li>
					<li><a href="http://med.stanford.edu/about/contacts.html">Contact Us</a></li>
					<li><a href="http://med.stanford.edu/about/contacts/maps-directions.html">Maps &amp; Directions</a></li>
				</ul>
			</div>
		</li>
		<li id="stanfordUniversityBrand">
			<a href="http://www.stanford.edu/" title="Stanford University"><b>Stanford University</b></a>
		</li>
	</ul>
</div>

    </div>
    
	    <script>
	      window.isBannerSwapped = true;
	      var require = {
	        baseUrl: "/profiles/scripts;jsessionid=BC8E491FBE7A1060992D875F939A81E1.tc-cap-som-07"
	      };
	    </script>
	    <script src="/profiles/scripts/vendor/require.js;jsessionid=BC8E491FBE7A1060992D875F939A81E1.tc-cap-som-07"></script>
	    <script>
	      define('app-config', {
	        
	          viewJson: {"data":{"academicOffices":[{"address":"Stanford University, School of Medicine, Department of Pediatrics, 265 Campus Drive, SIM1 Lokey Building, Room 2078a","affiliationType":"University - Faculty","city":"Stanford","department":"Pediatrics - Hematology/Oncology","displayAddress":{"html":"Stanford University, School of Medicine<br/> Department of Pediatrics<br/> 265 Campus Drive, SIM1 Lokey Building, Room 2078a","text":"Stanford University, School of Medicine| Department of Pediatrics| 265 Campus Drive, SIM1 Lokey Building, Room 2078a"},"officeName":"office","position":"Professor","state":"California","type":"academic","zip":"94305-5457"}],"administrativeAppointments":[{"appointment":"Member","endYear":{"html":"Present","text":"Present"},"label":{"html":"Member, Institute for Stem Cell Biology and Regenerative Medicine (2006 - Present)","text":"Member, Institute for Stem Cell Biology and Regenerative Medicine (2006 - Present)"},"organization":"Institute for Stem Cell Biology and Regenerative Medicine","startYear":{"html":"2006","text":"2006","value":"2006-01-01T00:00:00.000-08:00"}},{"appointment":"co-Director","endYear":{"html":"Present","text":"Present"},"label":{"html":"co-Director, Cancer Biology PhD program (2016 - Present)","text":"co-Director, Cancer Biology PhD program (2016 - Present)"},"organization":"Cancer Biology PhD program","startYear":{"html":"2016","text":"2016","value":"2016-01-01T00:00:00.000-08:00"}}],"advisees":[{"advisee":{"alias":"julia-arand","firstName":"Julia","label":{"html":"<a href=\"https://profiles.stanford.edu/julia-arand\">Julia Arand</a>","text":"Julia Arand"},"lastName":"Arand","links":[{"href":"https://cap.stanford.edu/cap-api/api/profiles/v1/38372","rel":"https://cap.stanford.edu/rel/self"},{"href":"https://profiles.stanford.edu/julia-arand","rel":"https://cap.stanford.edu/rel/public"},{"href":"https://cap.stanford.edu/profiles/auth/frdActionServlet?choiceId=facProfile&profileId=38372","rel":"https://cap.stanford.edu/rel/intranet"}],"profileId":38372},"code":"PDFS","role":"Postdoctoral Faculty Sponsor"},{"advisee":{"alias":"garry-coles","firstName":"Garry","label":{"html":"<a href=\"https://profiles.stanford.edu/garry-coles\">Garry Coles</a>","text":"Garry Coles"},"lastName":"Coles","links":[{"href":"https://cap.stanford.edu/cap-api/api/profiles/v1/71266","rel":"https://cap.stanford.edu/rel/self"},{"href":"https://profiles.stanford.edu/garry-coles","rel":"https://cap.stanford.edu/rel/public"},{"href":"https://cap.stanford.edu/profiles/auth/frdActionServlet?choiceId=facProfile&profileId=71266","rel":"https://cap.stanford.edu/rel/intranet"}],"profileId":71266},"code":"PDFS","role":"Postdoctoral Faculty Sponsor"},{"advisee":{"alias":"kevin-meng","firstName":"Kevin","label":{"html":"<a href=\"https://profiles.stanford.edu/kevin-meng\">Kevin Meng</a>","text":"Kevin Meng"},"lastName":"Meng","links":[{"href":"https://cap.stanford.edu/cap-api/api/profiles/v1/39665","rel":"https://cap.stanford.edu/rel/self"},{"href":"https://profiles.stanford.edu/kevin-meng","rel":"https://cap.stanford.edu/rel/public"},{"href":"https://cap.stanford.edu/profiles/auth/frdActionServlet?choiceId=facProfile&profileId=39665","rel":"https://cap.stanford.edu/rel/intranet"}],"profileId":39665},"code":"DRDR","role":"Doctoral Dissertation Reader (AC)"},{"advisee":{"alias":"youcef-ouadah","firstName":"Youcef","label":{"html":"<a href=\"https://profiles.stanford.edu/youcef-ouadah\">Youcef Ouadah</a>","text":"Youcef Ouadah"},"lastName":"Ouadah","links":[{"href":"https://cap.stanford.edu/cap-api/api/profiles/v1/23110","rel":"https://cap.stanford.edu/rel/self"},{"href":"https://profiles.stanford.edu/youcef-ouadah","rel":"https://cap.stanford.edu/rel/public"},{"href":"https://cap.stanford.edu/profiles/auth/frdActionServlet?choiceId=facProfile&profileId=23110","rel":"https://cap.stanford.edu/rel/intranet"}],"profileId":23110},"code":"DRDR","role":"Doctoral Dissertation Reader (AC)"},{"advisee":{"alias":"elizabeth-qin","firstName":"Elizabeth","label":{"html":"<a href=\"https://profiles.stanford.edu/elizabeth-qin\">Elizabeth Qin</a>","text":"Elizabeth Qin"},"lastName":"Qin","links":[{"href":"https://cap.stanford.edu/cap-api/api/profiles/v1/34870","rel":"https://cap.stanford.edu/rel/self"},{"href":"https://profiles.stanford.edu/elizabeth-qin","rel":"https://cap.stanford.edu/rel/public"},{"href":"https://cap.stanford.edu/profiles/auth/frdActionServlet?choiceId=facProfile&profileId=34870","rel":"https://cap.stanford.edu/rel/intranet"}],"profileId":34870},"code":"DRDR","role":"Doctoral Dissertation Reader (AC)"},{"advisee":{"alias":"fangfei-qu","firstName":"Fangfei","label":{"html":"<a href=\"https://profiles.stanford.edu/fangfei-qu\">Fangfei Qu</a>","text":"Fangfei Qu"},"lastName":"Qu","links":[{"href":"https://cap.stanford.edu/cap-api/api/profiles/v1/164789","rel":"https://cap.stanford.edu/rel/self"},{"href":"https://profiles.stanford.edu/fangfei-qu","rel":"https://cap.stanford.edu/rel/public"},{"href":"https://cap.stanford.edu/profiles/auth/frdActionServlet?choiceId=facProfile&profileId=164789","rel":"https://cap.stanford.edu/rel/intranet"}],"profileId":164789},"code":"PDFS","role":"Postdoctoral Faculty Sponsor"},{"advisee":{"alias":"miranda-stratton","firstName":"Miranda","label":{"html":"<a href=\"https://profiles.stanford.edu/miranda-stratton\">Miranda Stratton</a>","text":"Miranda Stratton"},"lastName":"Stratton","links":[{"href":"https://cap.stanford.edu/cap-api/api/profiles/v1/39617","rel":"https://cap.stanford.edu/rel/self"},{"href":"https://profiles.stanford.edu/miranda-stratton","rel":"https://cap.stanford.edu/rel/public"},{"href":"https://cap.stanford.edu/profiles/auth/frdActionServlet?choiceId=facProfile&profileId=39617","rel":"https://cap.stanford.edu/rel/intranet"}],"profileId":39617},"code":"DRDR","role":"Doctoral Dissertation Reader (AC)"},{"advisee":{"alias":"jamie-webster","firstName":"Jamie","label":{"html":"<a href=\"https://profiles.stanford.edu/jamie-webster\">Jamie Webster</a>","text":"Jamie Webster"},"lastName":"Webster","links":[{"href":"https://cap.stanford.edu/cap-api/api/profiles/v1/84097","rel":"https://cap.stanford.edu/rel/self"},{"href":"https://profiles.stanford.edu/jamie-webster","rel":"https://cap.stanford.edu/rel/public"},{"href":"https://cap.stanford.edu/profiles/auth/frdActionServlet?choiceId=facProfile&profileId=84097","rel":"https://cap.stanford.edu/rel/intranet"}],"profileId":84097},"code":"PDFS","role":"Postdoctoral Faculty Sponsor"},{"advisee":{"alias":"ian-winters","firstName":"Ian","label":{"html":"<a href=\"https://profiles.stanford.edu/ian-winters\">Ian Winters</a>","text":"Ian Winters"},"lastName":"Winters","links":[{"href":"https://cap.stanford.edu/cap-api/api/profiles/v1/34096","rel":"https://cap.stanford.edu/rel/self"},{"href":"https://profiles.stanford.edu/ian-winters","rel":"https://cap.stanford.edu/rel/public"},{"href":"https://cap.stanford.edu/profiles/auth/frdActionServlet?choiceId=facProfile&profileId=34096","rel":"https://cap.stanford.edu/rel/intranet"}],"profileId":34096},"code":"DRDR","role":"Doctoral Dissertation Reader (AC)"},{"advisee":{"alias":"dian-yang","firstName":"Dian","label":{"html":"<a href=\"https://profiles.stanford.edu/dian-yang\">Dian Yang</a>","text":"Dian Yang"},"lastName":"Yang","links":[{"href":"https://cap.stanford.edu/cap-api/api/profiles/v1/23711","rel":"https://cap.stanford.edu/rel/self"},{"href":"https://profiles.stanford.edu/dian-yang","rel":"https://cap.stanford.edu/rel/public"},{"href":"https://cap.stanford.edu/profiles/auth/frdActionServlet?choiceId=facProfile&profileId=23711","rel":"https://cap.stanford.edu/rel/intranet"}],"profileId":23711},"code":"DDIS","role":"Doctoral Dissertation Advisor (AC)"},{"advisee":{"alias":"nora-yucel","firstName":"Nora","label":{"html":"<a href=\"https://profiles.stanford.edu/nora-yucel\">Nora Yucel</a>","text":"Nora Yucel"},"lastName":"Yucel","links":[{"href":"https://cap.stanford.edu/cap-api/api/profiles/v1/20189","rel":"https://cap.stanford.edu/rel/self"},{"href":"https://profiles.stanford.edu/nora-yucel","rel":"https://cap.stanford.edu/rel/public"},{"href":"https://cap.stanford.edu/profiles/auth/frdActionServlet?choiceId=facProfile&profileId=20189","rel":"https://cap.stanford.edu/rel/intranet"}],"profileId":20189},"code":"DRDR","role":"Doctoral Dissertation Reader (AC)"}],"affiliations":{"capFaculty":true,"capStaff":false,"capPostdoc":false,"capMdStudent":false,"capMsStudent":false,"capPhdStudent":false,"physician":false,"capRegistry":false,"capResident":false,"capFellow":false,"capOther":false},"alias":"julien-sage","alternateContact":{"email":"alyssar@stanford.edu","label":"Alternate Contact","name":"Alyssa Ray","phoneNumbers":["(650) 724-9246"],"title":"Research Administrator"},"clinicalTrialsUrl":"https://med.stanford.edu/clinicaltrials/api/v1/trials/search?limit=2000&investigator=julsage","community":"public","coursesTaught":[{"academicYear":"2016-17","courses":[{"academicYear":"2016-17","label":{"html":"<a href='http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=CBIO 245: Lecture Seminar Series in Cancer Biology Program&academicYear=20162017'>Lecture Seminar Series in Cancer Biology Program</a><br/><span>CBIO 245 (Aut, Win, Spr)</span>","text":"CBIO 245 (Aut, Win, Spr)"},"quarter":"Aut, Win, Spr","title":"Lecture Seminar Series in Cancer Biology Program","url":"http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=CBIO 245: Lecture Seminar Series in Cancer Biology Program&academicYear=20162017"}],"independentStudyCourses":[{"academicYear":"2016-17","label":{"html":"<a href='http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=CBIO 299: Directed Reading in Cancer Biology&academicYear=20162017'>Directed Reading in Cancer Biology</a><br/><span>CBIO 299 (Aut, Win, Spr, Sum)</span>","text":"CBIO 299 (Aut, Win, Spr, Sum)"},"quarter":"Aut, Win, Spr, Sum","title":"Directed Reading in Cancer Biology","url":"http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=CBIO 299: Directed Reading in Cancer Biology&academicYear=20162017"},{"academicYear":"2016-17","label":{"html":"<a href='http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=GENE 299: Directed Reading in Genetics&academicYear=20162017'>Directed Reading in Genetics</a><br/><span>GENE 299 (Win, Spr, Sum)</span>","text":"GENE 299 (Win, Spr, Sum)"},"quarter":"Win, Spr, Sum","title":"Directed Reading in Genetics","url":"http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=GENE 299: Directed Reading in Genetics&academicYear=20162017"},{"academicYear":"2016-17","label":{"html":"<a href='http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=PEDS 299: Directed Reading in Pediatrics&academicYear=20162017'>Directed Reading in Pediatrics</a><br/><span>PEDS 299 (Aut, Win, Spr, Sum)</span>","text":"PEDS 299 (Aut, Win, Spr, Sum)"},"quarter":"Aut, Win, Spr, Sum","title":"Directed Reading in Pediatrics","url":"http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=PEDS 299: Directed Reading in Pediatrics&academicYear=20162017"},{"academicYear":"2016-17","label":{"html":"<a href='http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=PEDS 280: Early Clinical Experience&academicYear=20162017'>Early Clinical Experience</a><br/><span>PEDS 280 (Aut, Win, Spr, Sum)</span>","text":"PEDS 280 (Aut, Win, Spr, Sum)"},"quarter":"Aut, Win, Spr, Sum","title":"Early Clinical Experience","url":"http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=PEDS 280: Early Clinical Experience&academicYear=20162017"},{"academicYear":"2016-17","label":{"html":"<a href='http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=CBIO 399: Graduate Research&academicYear=20162017'>Graduate Research</a><br/><span>CBIO 399 (Aut, Win, Spr, Sum)</span>","text":"CBIO 399 (Aut, Win, Spr, Sum)"},"quarter":"Aut, Win, Spr, Sum","title":"Graduate Research","url":"http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=CBIO 399: Graduate Research&academicYear=20162017"},{"academicYear":"2016-17","label":{"html":"<a href='http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=GENE 399: Graduate Research&academicYear=20162017'>Graduate Research</a><br/><span>GENE 399 (Win, Spr, Sum)</span>","text":"GENE 399 (Win, Spr, Sum)"},"quarter":"Win, Spr, Sum","title":"Graduate Research","url":"http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=GENE 399: Graduate Research&academicYear=20162017"},{"academicYear":"2016-17","label":{"html":"<a href='http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=PEDS 399: Graduate Research&academicYear=20162017'>Graduate Research</a><br/><span>PEDS 399 (Aut, Win, Spr, Sum)</span>","text":"PEDS 399 (Aut, Win, Spr, Sum)"},"quarter":"Aut, Win, Spr, Sum","title":"Graduate Research","url":"http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=PEDS 399: Graduate Research&academicYear=20162017"},{"academicYear":"2016-17","label":{"html":"<a href='http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=GENE 370: Medical Scholars Research&academicYear=20162017'>Medical Scholars Research</a><br/><span>GENE 370 (Aut, Win, Spr, Sum)</span>","text":"GENE 370 (Aut, Win, Spr, Sum)"},"quarter":"Aut, Win, Spr, Sum","title":"Medical Scholars Research","url":"http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=GENE 370: Medical Scholars Research&academicYear=20162017"},{"academicYear":"2016-17","label":{"html":"<a href='http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=PEDS 370: Medical Scholars Research&academicYear=20162017'>Medical Scholars Research</a><br/><span>PEDS 370 (Aut, Win, Spr, Sum)</span>","text":"PEDS 370 (Aut, Win, Spr, Sum)"},"quarter":"Aut, Win, Spr, Sum","title":"Medical Scholars Research","url":"http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=PEDS 370: Medical Scholars Research&academicYear=20162017"},{"academicYear":"2016-17","label":{"html":"<a href='http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=GENE 260: Supervised Study&academicYear=20162017'>Supervised Study</a><br/><span>GENE 260 (Win, Spr, Sum)</span>","text":"GENE 260 (Win, Spr, Sum)"},"quarter":"Win, Spr, Sum","title":"Supervised Study","url":"http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=GENE 260: Supervised Study&academicYear=20162017"},{"academicYear":"2016-17","label":{"html":"<a href='http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=CBIO 260: Teaching in Cancer Biology&academicYear=20162017'>Teaching in Cancer Biology</a><br/><span>CBIO 260 (Spr)</span>","text":"CBIO 260 (Spr)"},"quarter":"Spr","title":"Teaching in Cancer Biology","url":"http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=CBIO 260: Teaching in Cancer Biology&academicYear=20162017"},{"academicYear":"2016-17","label":{"html":"<a href='http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=PEDS 199: Undergraduate Directed Reading/Research&academicYear=20162017'>Undergraduate Directed Reading/Research</a><br/><span>PEDS 199 (Aut, Win, Spr, Sum)</span>","text":"PEDS 199 (Aut, Win, Spr, Sum)"},"quarter":"Aut, Win, Spr, Sum","title":"Undergraduate Directed Reading/Research","url":"http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=PEDS 199: Undergraduate Directed Reading/Research&academicYear=20162017"},{"academicYear":"2016-17","label":{"html":"<a href='http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=GENE 199: Undergraduate Research&academicYear=20162017'>Undergraduate Research</a><br/><span>GENE 199 (Aut, Win, Spr, Sum)</span>","text":"GENE 199 (Aut, Win, Spr, Sum)"},"quarter":"Aut, Win, Spr, Sum","title":"Undergraduate Research","url":"http://explorecourses.stanford.edu/search?view=catalog&filter-coursestatus-Active=on&q=GENE 199: Undergraduate Research&academicYear=20162017"}]}],"currentResearchInterests":{"fullText":{"html":"Our research program focuses on the mechanisms that control the proliferation of mammalian cells under normal and pathological conditions (regeneration, cancer), with a particular emphasis on stem cells and gene regulatory networks. We combine genetic, genomics, and proteomics approaches to identify and investigate genes and pathways involved in cancer initiation and progression. We use genome-editing strategies to develop and study genetically-engineered mouse models for human cancers, including lung cancer, pancreatic cancer, and liver cancer. Our work spans the investigation of fundamental biological processes to the implementation of clinical trials based on our findings in pre-clinical models.","text":"Our research program focuses on the mechanisms that control the proliferation of mammalian cells under normal and pathological conditions (regeneration, cancer), with a particular emphasis on stem cells and gene regulatory networks. We combine genetic, genomics, and proteomics approaches to identify and investigate genes and pathways involved in cancer initiation and progression. We use genome-editing strategies to develop and study genetically-engineered mouse models for human cancers, including lung cancer, pancreatic cancer, and liver cancer. Our work spans the investigation of fundamental biological processes to the implementation of clinical trials based on our findings in pre-clinical models."},"shortText":{"html":"We investigate the mechanisms by which normal cells become tumor cells, and we combine genetics, genomics, and proteomics approaches to investigate the differences between the proliferative response in response to injury and the hyperproliferative phenotype of cancer cells and to identify novel therapeutic targets in cancer cells.","text":"We investigate the mechanisms by which normal cells become tumor cells, and we combine genetics, genomics, and proteomics approaches to investigate the differences between the proliferative response in response to injury and the hyperproliferative phenotype of cancer cells and to identify novel therapeutic targets in cancer cells."},"terseText":{"html":"Cellular and molecular mechanisms of tumorigenesis and regeneration","text":"Cellular and molecular mechanisms of tumorigenesis and regeneration"}},"displayName":"JulienÂ Sage","education":[{"degree":"B.S.","fieldOfStudy":"Biology","label":{"html":"B.S., Ecole Normale Superieure, France, Biology  (1993)","text":"B.S., Ecole Normale Superieure, France, Biology  (1993)"},"organization":"Ecole Normale Superieure, France","yearIssued":"1993"},{"degree":"Ph.D.","fieldOfStudy":"Biology","label":{"html":"Ph.D., Nice University, France, Biology  (1998)","text":"Ph.D., Nice University, France, Biology  (1998)"},"organization":"Nice University, France","yearIssued":"1998"}],"graduateAndFellowshipPrograms":[{"description":"Cancer Biology","label":{"html":"<a href=\"http://medcatalog.stanford.edu/php/dept.php?DNo=730\">Cancer Biology (Phd Program)</a>","text":"Cancer Biology (Phd Program)"},"type":"Phd Program","url":"http://medcatalog.stanford.edu/php/dept.php?DNo=730"},{"description":"Genetics","label":{"html":"<a href=\"http://medcatalog.stanford.edu/php/dept.php?DNo=770\">Genetics (Phd Program)</a>","text":"Genetics (Phd Program)"},"type":"Phd Program","url":"http://medcatalog.stanford.edu/php/dept.php?DNo=770"},{"description":"Hematology","label":{"html":"Hematology (Fellowship Program)","text":"Hematology (Fellowship Program)"},"type":"Fellowship Program"},{"description":"Pediatric Hem/Onc","label":{"html":"Pediatric Hem/Onc (Fellowship Program)","text":"Pediatric Hem/Onc (Fellowship Program)"},"type":"Fellowship Program"},{"description":"Stem Cell Biology and Regenerative Medicine","label":{"html":"<a href=\"http://stemcell.stanford.edu/\">Stem Cell Biology and Regenerative Medicine (Phd Program)</a>","text":"Stem Cell Biology and Regenerative Medicine (Phd Program)"},"type":"Phd Program","url":"http://stemcell.stanford.edu/"}],"honorsAndAwards":[{"dates":"2005-2008","label":{"html":"Scholar Award, Damon Runyon Cancer Research Foundation (2005-2008)","text":"Scholar Award, Damon Runyon Cancer Research Foundation (2005-2008)"},"organization":"Damon Runyon Cancer Research Foundation","title":"Scholar Award"},{"dates":"2009-2014","label":{"html":"Scholar Award, Leukemia and Lymphoma Society (2009-2014)","text":"Scholar Award, Leukemia and Lymphoma Society (2009-2014)"},"organization":"Leukemia and Lymphoma Society","title":"Scholar Award"},{"dates":"2008-2013","label":{"html":"Morgridge Faculty Scholar, Lucille Packard Foundation for Children's Health (2008-2013)","text":"Morgridge Faculty Scholar, Lucille Packard Foundation for Children's Health (2008-2013)"},"organization":"Lucille Packard Foundation for Children's Health","title":"Morgridge Faculty Scholar"},{"dates":"Since 2013","label":{"html":"Harriet and Mary Zelencik Scientist in Children's Cancer and Blood Diseases, Lucille Packard Foundation for Children's Health (Since 2013)","text":"Harriet and Mary Zelencik Scientist in Children's Cancer and Blood Diseases, Lucille Packard Foundation for Children's Health (Since 2013)"},"organization":"Lucille Packard Foundation for Children's Health","title":"Harriet and Mary Zelencik Scientist in Children's Cancer and Blood Diseases"}],"internetLinks":[{"label":{"html":"<a href='http://med.stanford.edu/sage'>the Sage lab</a>","text":"the Sage lab"},"type":"Lab Site","url":"http://med.stanford.edu/sage"}],"keywords":["cell cycle, cancer, genetics, stem cell, regenerative biology, retinoblastoma, lung cancer, liver cancer, skin cancer, pancreas cancer, planaria, mouse models, Notch, Hedgehog, ES cells, Cell Biology , Gene Expression , Genetics , Stem Cell Biology , Cancer"],"ldapAdditionalInfo":{"mailCode":"5457","webPage":"http://web.stanford.edu/group/sage"},"longTitle":{"academicInstituteDisplay":false,"affiliation":"capFaculty","bannerDisplay":true,"label":{"html":"Professor of Pediatrics (Hematology/Oncology) and of Genetics","text":"Professor of Pediatrics (Hematology/Oncology) and of Genetics"},"organization":{"interDepartmentalProgram":false,"orgCode":"NULL"},"title":"Professor of Pediatrics (Hematology/Oncology) and of Genetics","type":"faculty"},"meta":{"etag":"05522605df15686f56d53bbd48014717","links":[{"href":"https://cap.stanford.edu/cap-api/api/profiles/v1/6009","rel":"https://cap.stanford.edu/rel/self"},{"href":"https://profiles.stanford.edu/julien-sage","rel":"https://cap.stanford.edu/rel/public"},{"href":"https://cap.stanford.edu/profiles/auth/frdActionServlet?choiceId=facProfile&profileId=6009","rel":"https://cap.stanford.edu/rel/intranet"},{"href":"https://cap.stanford.edu/profiles/frdActionServlet?choiceId=printerprofile&profileversion=full&profileId=6009","rel":"https://cap.stanford.edu/rel/pdf"},{"href":"https://cap.stanford.edu/cap-api/api/cap/v1/schemas/ProfileDto","rel":"https://cap.stanford.edu/rel/schema"}]},"names":{"legal":{"firstName":"Julien","lastName":"Sage"},"preferred":{"firstName":"Julien","lastName":"Sage"}},"organizations":[{"affiliation":"capFaculty","label":{"html":"<a href='http://cancer.stanford.edu'>Stanford Cancer Institute</a>","text":"Stanford Cancer Institute"},"organization":{"alias":"school-of-medicine/stanford-cancer-institute/cancer-biology","interDepartmentalProgram":false,"label":{"html":"<a href='http://cancer.stanford.edu'>Stanford Cancer Institute</a>","text":"Stanford Cancer Institute"},"org":"Stanford Cancer Institute","orgCode":"SCC3","orgUrl":"http://cancer.stanford.edu"},"type":"affiliation"},{"affiliation":"capFaculty","label":{"html":"NULL","text":"NULL"},"organization":{"interDepartmentalProgram":false,"label":{"html":"NULL","text":"NULL"},"org":"NULL","orgCode":"NULL"},"type":"affiliation"},{"affiliation":"capFaculty","label":{"html":"<a href='http://genetics.stanford.edu/'>Genetics</a>","text":"Genetics"},"organization":{"alias":"school-of-medicine/genetics","interDepartmentalProgram":false,"label":{"html":"<a href='http://genetics.stanford.edu/'>Genetics</a>","text":"Genetics"},"org":"Genetics","orgCode":"WAZC","orgUrl":"http://genetics.stanford.edu/"},"type":"affiliation"},{"affiliation":"capFaculty","label":{"html":"<a href='http://biox.stanford.edu/'>Bio-X</a>","text":"Bio-X"},"organization":{"interDepartmentalProgram":false,"label":{"html":"<a href='http://biox.stanford.edu/'>Bio-X</a>","text":"Bio-X"},"org":"Bio-X","orgCode":"BIOX","orgUrl":"http://biox.stanford.edu/"},"type":"affiliation"},{"affiliation":"capFaculty","label":{"html":"<a href='http://chri.stanford.edu/'>Child Health Research Institute</a>","text":"Child Health Research Institute"},"organization":{"alias":"school-of-medicine/child-health-research-institute","interDepartmentalProgram":false,"label":{"html":"<a href='http://chri.stanford.edu/'>Child Health Research Institute</a>","text":"Child Health Research Institute"},"org":"Child Health Research Institute","orgCode":"I001","orgUrl":"http://chri.stanford.edu/"},"type":"affiliation"},{"affiliation":"capFaculty","label":{"html":"<a href='http://pedhemonc.stanford.edu/'>Pediatrics - Hematology/Oncology</a>","text":"Pediatrics - Hematology/Oncology"},"organization":{"alias":"school-of-medicine/pediatrics/hematology-oncology","interDepartmentalProgram":false,"label":{"html":"<a href='http://pedhemonc.stanford.edu/'>Pediatrics - Hematology/Oncology</a>","text":"Pediatrics - Hematology/Oncology"},"org":"Pediatrics - Hematology/Oncology","orgCode":"XIXC","orgUrl":"http://pedhemonc.stanford.edu/"},"type":"affiliation"},{"affiliation":"capFaculty","label":{"html":"<a href='https://chemh.stanford.edu/'>Stanford ChEM-H</a>","text":"Stanford ChEM-H"},"organization":{"interDepartmentalProgram":false,"label":{"html":"<a href='https://chemh.stanford.edu/'>Stanford ChEM-H</a>","text":"Stanford ChEM-H"},"org":"Stanford ChEM-H","orgCode":"NOQ1","orgUrl":"https://chemh.stanford.edu/"},"type":"affiliation"},{"affiliation":"capFaculty","label":{"html":"<a href='http://pedhemonc.stanford.edu/'>Pediatrics - Hematology/Oncology</a>","text":"Pediatrics - Hematology/Oncology"},"organization":{"alias":"school-of-medicine/pediatrics/hematology-oncology","interDepartmentalProgram":false,"label":{"html":"<a href='http://pedhemonc.stanford.edu/'>Pediatrics - Hematology/Oncology</a>","text":"Pediatrics - Hematology/Oncology"},"org":"Pediatrics - Hematology/Oncology","orgCode":"XIXC","orgUrl":"http://pedhemonc.stanford.edu/"},"type":"institute"},{"affiliation":"capFaculty","label":{"html":"<a href='http://genetics.stanford.edu/'>Genetics</a>","text":"Genetics"},"organization":{"alias":"school-of-medicine/genetics","interDepartmentalProgram":false,"label":{"html":"<a href='http://genetics.stanford.edu/'>Genetics</a>","text":"Genetics"},"org":"Genetics","orgCode":"WAZC","orgUrl":"http://genetics.stanford.edu/"},"type":"institute"},{"affiliation":"capFaculty","label":{"html":"<a href='http://biox.stanford.edu/'>Bio-X</a>","text":"Bio-X"},"organization":{"interDepartmentalProgram":false,"label":{"html":"<a href='http://biox.stanford.edu/'>Bio-X</a>","text":"Bio-X"},"org":"Bio-X","orgCode":"BIOX","orgUrl":"http://biox.stanford.edu/"},"type":"institute"},{"affiliation":"capFaculty","label":{"html":"<a href='http://chri.stanford.edu/'>Child Health Research Institute</a>","text":"Child Health Research Institute"},"organization":{"alias":"school-of-medicine/child-health-research-institute","interDepartmentalProgram":false,"label":{"html":"<a href='http://chri.stanford.edu/'>Child Health Research Institute</a>","text":"Child Health Research Institute"},"org":"Child Health Research Institute","orgCode":"I001","orgUrl":"http://chri.stanford.edu/"},"type":"institute"},{"affiliation":"capFaculty","label":{"html":"<a href='http://cancer.stanford.edu'>Stanford Cancer Institute</a>","text":"Stanford Cancer Institute"},"organization":{"alias":"school-of-medicine/stanford-cancer-institute/cancer-biology","interDepartmentalProgram":false,"label":{"html":"<a href='http://cancer.stanford.edu'>Stanford Cancer Institute</a>","text":"Stanford Cancer Institute"},"org":"Stanford Cancer Institute","orgCode":"SCC3","orgUrl":"http://cancer.stanford.edu"},"type":"institute"},{"affiliation":"capFaculty","label":{"html":"<a href='https://chemh.stanford.edu/'>Stanford ChEM-H</a>","text":"Stanford ChEM-H"},"organization":{"interDepartmentalProgram":false,"label":{"html":"<a href='https://chemh.stanford.edu/'>Stanford ChEM-H</a>","text":"Stanford ChEM-H"},"org":"Stanford ChEM-H","orgCode":"NOQ1","orgUrl":"https://chemh.stanford.edu/"},"type":"institute"}],"postdoctoralAdvisees":[{"alias":"julia-arand","firstName":"Julia","label":{"html":"<a href=\"https://profiles.stanford.edu/julia-arand\">Julia Arand</a>","text":"Julia Arand"},"lastName":"Arand","links":[{"href":"https://cap.stanford.edu/cap-api/api/profiles/v1/38372","rel":"https://cap.stanford.edu/rel/self"},{"href":"https://profiles.stanford.edu/julia-arand","rel":"https://cap.stanford.edu/rel/public"},{"href":"https://cap.stanford.edu/profiles/auth/frdActionServlet?choiceId=facProfile&profileId=38372","rel":"https://cap.stanford.edu/rel/intranet"}],"profileId":38372},{"alias":"garry-coles","firstName":"Garry","label":{"html":"<a href=\"https://profiles.stanford.edu/garry-coles\">Garry Coles</a>","text":"Garry Coles"},"lastName":"Coles","links":[{"href":"https://cap.stanford.edu/cap-api/api/profiles/v1/71266","rel":"https://cap.stanford.edu/rel/self"},{"href":"https://profiles.stanford.edu/garry-coles","rel":"https://cap.stanford.edu/rel/public"},{"href":"https://cap.stanford.edu/profiles/auth/frdActionServlet?choiceId=facProfile&profileId=71266","rel":"https://cap.stanford.edu/rel/intranet"}],"profileId":71266},{"alias":"fangfei-qu","firstName":"Fangfei","label":{"html":"<a href=\"https://profiles.stanford.edu/fangfei-qu\">Fangfei Qu</a>","text":"Fangfei Qu"},"lastName":"Qu","links":[{"href":"https://cap.stanford.edu/cap-api/api/profiles/v1/164789","rel":"https://cap.stanford.edu/rel/self"},{"href":"https://profiles.stanford.edu/fangfei-qu","rel":"https://cap.stanford.edu/rel/public"},{"href":"https://cap.stanford.edu/profiles/auth/frdActionServlet?choiceId=facProfile&profileId=164789","rel":"https://cap.stanford.edu/rel/intranet"}],"profileId":164789},{"alias":"jamie-webster","firstName":"Jamie","label":{"html":"<a href=\"https://profiles.stanford.edu/jamie-webster\">Jamie Webster</a>","text":"Jamie Webster"},"lastName":"Webster","links":[{"href":"https://cap.stanford.edu/cap-api/api/profiles/v1/84097","rel":"https://cap.stanford.edu/rel/self"},{"href":"https://profiles.stanford.edu/jamie-webster","rel":"https://cap.stanford.edu/rel/public"},{"href":"https://cap.stanford.edu/profiles/auth/frdActionServlet?choiceId=facProfile&profileId=84097","rel":"https://cap.stanford.edu/rel/intranet"}],"profileId":84097}],"primaryContact":{"email":"julsage@stanford.edu","label":"Academic","name":"JulienÂ Sage","title":"Professor of Pediatrics (Hematology/Oncology) and of Genetics","type":"primary"},"profileId":6009,"profilePhotos":{"big":{"contentType":"image/jpeg","lastModified":"2015-01-26T21:40:16.355-08:00","placeholder":false,"type":"big","url":"https://cap.stanford.edu/profiles/viewImage?profileId=6009&type=big"},"bigger":{"contentType":"image/jpeg","lastModified":"2015-01-26T21:40:16.355-08:00","placeholder":false,"type":"bigger","url":"https://cap.stanford.edu/profiles/viewImage?profileId=6009&type=bigger"},"small":{"contentType":"image/jpeg","lastModified":"2015-01-26T21:40:16.355-08:00","placeholder":false,"type":"small","url":"https://cap.stanford.edu/profiles/viewImage?profileId=6009&type=small"},"square":{"contentType":"image/jpeg","lastModified":"2015-01-26T21:40:16.355-08:00","placeholder":false,"type":"square","url":"https://cap.stanford.edu/profiles/viewImage?profileId=6009&type=square"}},"profileTags":{"bioscienceHomeProgram":[{"@class":"edu.stanford.irt.cap.api.client.dto.BioscienceHomeProgramTag","description":"The mission of the Stanford Cancer Biology Graduate Home Program is to provide our students with education and training that will enable them to make significant contributions to this remarkable field. Our thriving, interdisciplinary program includes significant exposure to cancer biology as a formal discipline. Graduate students perform their research in many of the clinical and basic science departments in the School of Medicine, with main concentrations in the Departments of Pathology, Radiation Oncology, and Medicine.","imagePath":"/profiles/biosci/images/cancerbio.jpg","name":"Cancer Biology","url":"http://cancerbio.stanford.edu"},{"@class":"edu.stanford.irt.cap.api.client.dto.BioscienceHomeProgramTag","description":"An underlying theme in the Stanford Genetics Graduate Home Program is that genetics is not merely a set of tools but a coherent and fruitful way of thinking about biology and medicine. To this end, we emphasize a spectrum of approaches based on molecules, organisms, populations, and genomes. The mission of the Department of Genetics includes education and teaching as well as research, and graduates from our Home Program pursue careers in many different venues, including research in academic or industrial settings, health care, health policy, and education.","imagePath":"/profiles/biosci/images/genetics.jpg","name":"Genetics","url":"http://genetics.stanford.edu"}]},"publicationTags":[{"apiLabel":{"html":"Base Sequence","text":"Base Sequence"},"weight":11},{"apiLabel":{"html":"Cell Cycle","text":"Cell Cycle"},"weight":14},{"apiLabel":{"html":"Cell Cycle Proteins","text":"Cell Cycle Proteins"},"weight":11},{"apiLabel":{"html":"Cell Differentiation","text":"Cell Differentiation"},"weight":13},{"apiLabel":{"html":"Cell Line","text":"Cell Line"},"weight":11},{"apiLabel":{"html":"Cell Proliferation","text":"Cell Proliferation"},"weight":11},{"apiLabel":{"html":"Cells, Cultured","text":"Cells, Cultured"},"weight":12},{"apiLabel":{"html":"DNA-Binding Proteins","text":"DNA-Binding Proteins"},"weight":11},{"apiLabel":{"html":"Disease Models, Animal","text":"Disease Models, Animal"},"weight":11},{"apiLabel":{"html":"E2F Transcription Factors","text":"E2F Transcription Factors"},"weight":11},{"apiLabel":{"html":"Gene Deletion","text":"Gene Deletion"},"weight":11},{"apiLabel":{"html":"Gene Expression","text":"Gene Expression"},"weight":11},{"apiLabel":{"html":"Gene Expression Profiling","text":"Gene Expression Profiling"},"weight":11},{"apiLabel":{"html":"Genes, Retinoblastoma","text":"Genes, Retinoblastoma"},"weight":12},{"apiLabel":{"html":"Lung Neoplasms","text":"Lung Neoplasms"},"weight":11},{"apiLabel":{"html":"Mice, Inbred C57BL","text":"Mice, Inbred C57BL"},"weight":11},{"apiLabel":{"html":"Mice, Knockout","text":"Mice, Knockout"},"weight":14},{"apiLabel":{"html":"Mice, Transgenic","text":"Mice, Transgenic"},"weight":12},{"apiLabel":{"html":"Molecular Sequence Data","text":"Molecular Sequence Data"},"weight":11},{"apiLabel":{"html":"Mutation","text":"Mutation"},"weight":12},{"apiLabel":{"html":"Nuclear Proteins","text":"Nuclear Proteins"},"weight":12},{"apiLabel":{"html":"Promoter Regions, Genetic","text":"Promoter Regions, Genetic"},"weight":11},{"apiLabel":{"html":"Proteins","text":"Proteins"},"weight":11},{"apiLabel":{"html":"Retinoblastoma","text":"Retinoblastoma"},"weight":11},{"apiLabel":{"html":"Retinoblastoma Protein","text":"Retinoblastoma Protein"},"weight":18},{"apiLabel":{"html":"Retinoblastoma-Like Protein p107","text":"Retinoblastoma-Like Protein p107"},"weight":13},{"apiLabel":{"html":"Retinoblastoma-Like Protein p130","text":"Retinoblastoma-Like Protein p130"},"weight":13},{"apiLabel":{"html":"Signal Transduction","text":"Signal Transduction"},"weight":13},{"apiLabel":{"html":"Transcription Factors","text":"Transcription Factors"},"weight":11},{"apiLabel":{"html":"Tumor Suppressor Protein p53","text":"Tumor Suppressor Protein p53"},"weight":12}],"publications":[{"abstract":"Small cell lung cancer is initially highly responsive to cisplatin and etoposide but in almost every case becomes rapidly chemoresistant, leading to death within 1 year. We modeled acquired chemoresistance inÂ vivo using a series of patient-derived xenografts to generate paired chemosensitive and chemoresistant cancers. Multiple chemoresistant models demonstrated suppression of SLFN11, a factor implicated in DNA-damage repair deficiency. InÂ vivo silencing of SLFN11 was associated with marked deposition of H3K27me3, a histone modification placed by EZH2, within the gene body of SLFN11, inducing local chromatin condensation and gene silencing. Inclusion of an EZH2 inhibitor with standard cytotoxic therapies prevented emergence of acquired resistance and augmented chemotherapeutic efficacy in both chemosensitive and chemoresistant models of small cell lung cancer.","apaCitation":"Gardner, E. E., Lok, B. H., Schneeberger, V. E., Desmeules, P., Miles, L. A., Arnold, P. K., ? Poirier, J. T. (2017). Chemosensitive Relapse in Small Cell Lung Cancer Proceeds through an EZH2-SLFN11 Axis. <i>Cancer Cell</i>, <i>31</i>(2), 286?299.","capCitation":"<span class=\"title\">\n  <span>Chemosensitive Relapse in Small Cell Lung Cancer Proceeds through an EZH2-SLFN11 Axis.</span>\n  <i>Cancer cell</i>\n</span>\n<span class=\"authors\">Gardner, E. E., Lok, B. H., Schneeberger, V. E., Desmeules, P., Miles, L. A., Arnold, P. K., Ni, A., Khodos, I., de Stanchina, E., Nguyen, T., Sage, J., Campbell, J. E., Ribich, S., Rekhtman, N., Dowlati, A., Massion, P. P., Rudin, C. M., Poirier, J. T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2017</span>; <span class=\"volume\">31 (2)</span><span class=\"pages\">: 286-299</span>\n</span>","chicagoCitation":"Gardner, Eric E., Benjamin H. Lok, Valentina E. Schneeberger, Patrice Desmeules, Linde A. Miles, et al. 2017. ?Chemosensitive Relapse in Small Cell Lung Cancer Proceeds through an EZH2-SLFN11 Axis.? <i>Cancer Cell</i> 31 (2). ELSEVIER INC: 286?99.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small cell lung cancer is initially highly responsive to cisplatin and etoposide but in almost every case becomes rapidly chemoresistant, leading to death within 1 year. We modeled acquired chemoresistance inÂ vivo using a series of patient-derived xenografts to generate paired chemosensitive and chemoresistant cancers. Multiple chemoresistant models demonstrated suppression of SLFN11, a factor implicated in DNA-damage repair deficiency. InÂ vivo silencing of SLFN11 was associated with marked deposition of H3K27me3, a histone modification placed by EZH2, within the gene body of SLFN11, inducing local chromatin condensation and gene silencing. Inclusion of an EZH2 inhibitor with standard cytotoxic therapies prevented emergence of acquired resistance and augmented chemotherapeutic efficacy in both chemosensitive and chemoresistant models of small cell lung cancer.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.ccell.2017.01.006\">DOI 10.1016/j.ccell.2017.01.006</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/28196596\">PubMedID 28196596</a></span>\n  </p>","text":"Abstract:\nSmall cell lung cancer is initially highly responsive to cisplatin and etoposide but in almost every case becomes rapidly chemoresistant, leading to death within 1 year. We modeled acquired chemoresistance inÂ vivo using a series of patient-derived xenografts to generate paired chemosensitive and chemoresistant cancers. Multiple chemoresistant models demonstrated suppression of SLFN11, a factor implicated in DNA-damage repair deficiency. InÂ vivo silencing of SLFN11 was associated with marked deposition of H3K27me3, a histone modification placed by EZH2, within the gene body of SLFN11, inducing local chromatin condensation and gene silencing. Inclusion of an EZH2 inhibitor with standard cytotoxic therapies prevented emergence of acquired resistance and augmented chemotherapeutic efficacy in both chemosensitive and chemoresistant models of small cell lung cancer.\nDOI: 10.1016/j.ccell.2017.01.006, https://dx.doi.org/10.1016/j.ccell.2017.01.006\n\nPubMedID: 28196596, https://www.ncbi.nlm.nih.gov/pubmed/28196596"},"doiId":"10.1016/j.ccell.2017.01.006","doiUrl":"https://dx.doi.org/10.1016/j.ccell.2017.01.006","featured":false,"firstPublished":{"html":"Feb 13, 2017 00:00:00 AM","text":"Feb 13, 2017 00:00:00 AM","value":"2017-02-13T00:00:00.000-08:00"},"mlaCitation":"Gardner, Eric E. et al. ?Chemosensitive Relapse in Small Cell Lung Cancer Proceeds through an EZH2-SLFN11 Axis.? <i>Cancer cell</i> 31.2 (2017): 286?299. Print.","pubMedId":"28196596","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/28196596","publicationId":438256,"title":"Chemosensitive Relapse in Small Cell Lung Cancer Proceeds through an EZH2-SLFN11 Axis.","type":"article"},{"abstract":"The E2F family of transcription factors is a key determinant of cell proliferation in response to extra- and intra-cellular signals. Within this family, E2F4 is a transcriptional repressor whose activity is critical to engage and maintain cell cycle arrest in G0/G1 in conjunction with members of the retinoblastoma (RB) family. However, recent observations challenge this paradigm and indicate that E2F4 has a multitude of functions in cells besides this cell cycle regulatory role, including in embryonic and adult stem cells, during regenerative processes, and in cancer. Some of these new functions are independent of the RB family and involve direct activation of target genes. Here we review the canonical functions of E2F4 and discuss recent evidence expanding the role of this transcription factor, with a focus on cell fate decisions in tissue homeostasis and regeneration.","apaCitation":"Hsu, J., &amp; Sage, J. (2016). Novel functions for the transcription factor E2F4 in development and disease. <i>Cell Cycle </i>, 1?8.","capCitation":"<span class=\"title\">\n  <span>Novel functions for the transcription factor E2F4 in development and disease.</span>\n  <i>Cell cycle </i>\n</span>\n<span class=\"authors\">Hsu, J., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span><span class=\"pages\">: 1-8</span>\n</span>","chicagoCitation":"Hsu, Jenny, and Julien Sage. 2016. ?Novel Functions for the Transcription Factor E2F4 in Development and Disease.? <i>Cell Cycle </i>, 1?8.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The E2F family of transcription factors is a key determinant of cell proliferation in response to extra- and intra-cellular signals. Within this family, E2F4 is a transcriptional repressor whose activity is critical to engage and maintain cell cycle arrest in G0/G1 in conjunction with members of the retinoblastoma (RB) family. However, recent observations challenge this paradigm and indicate that E2F4 has a multitude of functions in cells besides this cell cycle regulatory role, including in embryonic and adult stem cells, during regenerative processes, and in cancer. Some of these new functions are independent of the RB family and involve direct activation of target genes. Here we review the canonical functions of E2F4 and discuss recent evidence expanding the role of this transcription factor, with a focus on cell fate decisions in tissue homeostasis and regeneration.</p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/27753528\">PubMedID 27753528</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5176148\">PubMedCentralID PMC5176148</a></span>\n  </p>","text":"Abstract:\nThe E2F family of transcription factors is a key determinant of cell proliferation in response to extra- and intra-cellular signals. Within this family, E2F4 is a transcriptional repressor whose activity is critical to engage and maintain cell cycle arrest in G0/G1 in conjunction with members of the retinoblastoma (RB) family. However, recent observations challenge this paradigm and indicate that E2F4 has a multitude of functions in cells besides this cell cycle regulatory role, including in embryonic and adult stem cells, during regenerative processes, and in cancer. Some of these new functions are independent of the RB family and involve direct activation of target genes. Here we review the canonical functions of E2F4 and discuss recent evidence expanding the role of this transcription factor, with a focus on cell fate decisions in tissue homeostasis and regeneration.\n\n\nPubMedID: 27753528, https://www.ncbi.nlm.nih.gov/pubmed/27753528\nPubMedCentralID: PMC5176148, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5176148"},"featured":false,"firstPublished":{"html":"Oct 18, 2016 00:00:00 AM","text":"Oct 18, 2016 00:00:00 AM","value":"2016-10-18T00:00:00.000-07:00"},"mlaCitation":"Hsu, Jenny, and Julien Sage. ?Novel Functions for the Transcription Factor E2F4 in Development and Disease.? <i>Cell cycle </i>(2016): 1?8. Print.","pubMedId":"27753528","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/27753528","publicationId":426898,"title":"Novel functions for the transcription factor E2F4 in development and disease.","type":"article"},{"abstract":"The cellular turnover of adult tissues and injury-induced repair proceed through an exquisite integration of proliferation, differentiation, and survival signals that involve stem/progenitor cell populations, their progeny, and differentiated tissues. GATA factors are DNA binding proteins that control stem cells and the development of tissues by activating or repressing transcription. Here we examined the role of GATA transcription factors in Schmidtea mediterranea, a freshwater planarian that provides an excellent model to investigate gene function in adult stem cells, regeneration, and differentiation. Smed-gata4/5/6, the homolog of the three mammalian GATA-4,-5,-6 factors is expressed at high levels in differentiated gut cells but also at lower levels in neoblast populations, the planarian stem cells. Smed-gata4/5/6 knock-down results in broad differentiation defects, especially in response to injury. These defects are not restricted to the intestinal lineage. In particular, at late time points during the response to injury, loss of Smed-gata4/5/6 leads to decreased neoblast proliferation and to gene expression changes in several neoblast subpopulations. Thus, Smed-gata4/5/6 plays a key evolutionary conserved role in intestinal differentiation in planarians. These data further support a model in which defects in the intestinal lineage can indirectly affect other differentiation pathways in planarians.","apaCitation":"Flores, N. M., Oviedo, N. J., &amp; Sage, J. (2016). Essential role for the planarian intestinal GATA transcription factor in stem cells and regeneration. <i>Developmental Biology</i>, <i>418</i>(1), 179?188.","capCitation":"<span class=\"title\">\n  <span>Essential role for the planarian intestinal GATA transcription factor in stem cells and regeneration.</span>\n  <i>Developmental biology</i>\n</span>\n<span class=\"authors\">Flores, N. M., Oviedo, N. J., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">418 (1)</span><span class=\"pages\">: 179-188</span>\n</span>","chicagoCitation":"Flores, Natasha M., NÃ©stor J. Oviedo, and Julien Sage. 2016. ?Essential Role for the Planarian Intestinal GATA Transcription Factor in Stem Cells and Regeneration.? <i>Developmental Biology</i> 418 (1). ELSEVIER INC: 179?88.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The cellular turnover of adult tissues and injury-induced repair proceed through an exquisite integration of proliferation, differentiation, and survival signals that involve stem/progenitor cell populations, their progeny, and differentiated tissues. GATA factors are DNA binding proteins that control stem cells and the development of tissues by activating or repressing transcription. Here we examined the role of GATA transcription factors in Schmidtea mediterranea, a freshwater planarian that provides an excellent model to investigate gene function in adult stem cells, regeneration, and differentiation. Smed-gata4/5/6, the homolog of the three mammalian GATA-4,-5,-6 factors is expressed at high levels in differentiated gut cells but also at lower levels in neoblast populations, the planarian stem cells. Smed-gata4/5/6 knock-down results in broad differentiation defects, especially in response to injury. These defects are not restricted to the intestinal lineage. In particular, at late time points during the response to injury, loss of Smed-gata4/5/6 leads to decreased neoblast proliferation and to gene expression changes in several neoblast subpopulations. Thus, Smed-gata4/5/6 plays a key evolutionary conserved role in intestinal differentiation in planarians. These data further support a model in which defects in the intestinal lineage can indirectly affect other differentiation pathways in planarians.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.ydbio.2016.08.015\">DOI 10.1016/j.ydbio.2016.08.015</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/27542689\">PubMedID 27542689</a></span>\n  </p>","text":"Abstract:\nThe cellular turnover of adult tissues and injury-induced repair proceed through an exquisite integration of proliferation, differentiation, and survival signals that involve stem/progenitor cell populations, their progeny, and differentiated tissues. GATA factors are DNA binding proteins that control stem cells and the development of tissues by activating or repressing transcription. Here we examined the role of GATA transcription factors in Schmidtea mediterranea, a freshwater planarian that provides an excellent model to investigate gene function in adult stem cells, regeneration, and differentiation. Smed-gata4/5/6, the homolog of the three mammalian GATA-4,-5,-6 factors is expressed at high levels in differentiated gut cells but also at lower levels in neoblast populations, the planarian stem cells. Smed-gata4/5/6 knock-down results in broad differentiation defects, especially in response to injury. These defects are not restricted to the intestinal lineage. In particular, at late time points during the response to injury, loss of Smed-gata4/5/6 leads to decreased neoblast proliferation and to gene expression changes in several neoblast subpopulations. Thus, Smed-gata4/5/6 plays a key evolutionary conserved role in intestinal differentiation in planarians. These data further support a model in which defects in the intestinal lineage can indirectly affect other differentiation pathways in planarians.\nDOI: 10.1016/j.ydbio.2016.08.015, https://dx.doi.org/10.1016/j.ydbio.2016.08.015\n\nPubMedID: 27542689, https://www.ncbi.nlm.nih.gov/pubmed/27542689"},"doiId":"10.1016/j.ydbio.2016.08.015","doiUrl":"https://dx.doi.org/10.1016/j.ydbio.2016.08.015","featured":false,"firstPublished":{"html":"Oct 1, 2016 00:00:00 AM","text":"Oct 1, 2016 00:00:00 AM","value":"2016-10-01T00:00:00.000-07:00"},"mlaCitation":"Flores, Natasha M., NÃ©stor J. Oviedo, and Julien Sage. ?Essential Role for the Planarian Intestinal GATA Transcription Factor in Stem Cells and Regeneration.? <i>Developmental biology</i> 418.1 (2016): 179?188. Print.","pubMedId":"27542689","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/27542689","publicationId":421260,"title":"Essential role for the planarian intestinal GATA transcription factor in stem cells and regeneration.","type":"article"},{"abstract":"The activity of the RAF/MEK/ERK signaling pathway is critical for the proliferation of normal and cancerous cells. Oncogenic mutations driving the development of lung adenocarcinoma often activate this signaling pathway. In contrast, pathway activity levels and their biological roles are not well established in small cell lung cancer (SCLC), a fast-growing neuroendocrine lung cancer subtype. Here we discuss the function of the RAF/MEK/ERK kinase pathway and the mechanisms leading to its activation in SCLC cells. In particular, we argue that activation of this pathway may be beneficial to the survival, proliferation, and spread of SCLC cells in response to multiple stimuli. We also consider evidence that high levels of RAF/MEK/ERK pathway activity may be detrimental to SCLC tumors, including in part byÂ interfering with their neuroendocrine fate. On the basis of these observations, we examined when small molecules targeting kinases in the RAF/MEK/ERK pathway may be useful therapeutically in patients with SCLC, including in combination with other therapeutic agents.","apaCitation":"Cristea, S., &amp; Sage, J. (2016). Is the Canonical RAF/MEK/ERK Signaling Pathway aÂ Therapeutic Target in SCLC? <i>Journal of Thoracic Oncology </i>, <i>11</i>(8), 1233?1241.","capCitation":"<span class=\"title\">\n  <span>Is the Canonical RAF/MEK/ERK Signaling Pathway aÂ Therapeutic Target in SCLC?</span>\n  <i>Journal of thoracic oncology </i>\n</span>\n<span class=\"authors\">Cristea, S., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">11 (8)</span><span class=\"pages\">: 1233-1241</span>\n</span>","chicagoCitation":"Cristea, Sandra, and Julien Sage. 2016. ?Is the Canonical RAF/MEK/ERK Signaling Pathway aÂ Therapeutic Target in SCLC?? <i>Journal of Thoracic Oncology </i>11 (8). ELSEVIER INC: 1233?41.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The activity of the RAF/MEK/ERK signaling pathway is critical for the proliferation of normal and cancerous cells. Oncogenic mutations driving the development of lung adenocarcinoma often activate this signaling pathway. In contrast, pathway activity levels and their biological roles are not well established in small cell lung cancer (SCLC), a fast-growing neuroendocrine lung cancer subtype. Here we discuss the function of the RAF/MEK/ERK kinase pathway and the mechanisms leading to its activation in SCLC cells. In particular, we argue that activation of this pathway may be beneficial to the survival, proliferation, and spread of SCLC cells in response to multiple stimuli. We also consider evidence that high levels of RAF/MEK/ERK pathway activity may be detrimental to SCLC tumors, including in part byÂ interfering with their neuroendocrine fate. On the basis of these observations, we examined when small molecules targeting kinases in the RAF/MEK/ERK pathway may be useful therapeutically in patients with SCLC, including in combination with other therapeutic agents.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.jtho.2016.04.018\">DOI 10.1016/j.jtho.2016.04.018</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/27133774\">PubMedID 27133774</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5391975\">PubMedCentralID PMC5391975</a></span>\n  </p>","text":"Abstract:\nThe activity of the RAF/MEK/ERK signaling pathway is critical for the proliferation of normal and cancerous cells. Oncogenic mutations driving the development of lung adenocarcinoma often activate this signaling pathway. In contrast, pathway activity levels and their biological roles are not well established in small cell lung cancer (SCLC), a fast-growing neuroendocrine lung cancer subtype. Here we discuss the function of the RAF/MEK/ERK kinase pathway and the mechanisms leading to its activation in SCLC cells. In particular, we argue that activation of this pathway may be beneficial to the survival, proliferation, and spread of SCLC cells in response to multiple stimuli. We also consider evidence that high levels of RAF/MEK/ERK pathway activity may be detrimental to SCLC tumors, including in part byÂ interfering with their neuroendocrine fate. On the basis of these observations, we examined when small molecules targeting kinases in the RAF/MEK/ERK pathway may be useful therapeutically in patients with SCLC, including in combination with other therapeutic agents.\nDOI: 10.1016/j.jtho.2016.04.018, https://dx.doi.org/10.1016/j.jtho.2016.04.018\n\nPubMedID: 27133774, https://www.ncbi.nlm.nih.gov/pubmed/27133774\nPubMedCentralID: PMC5391975, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5391975"},"doiId":"10.1016/j.jtho.2016.04.018","doiUrl":"https://dx.doi.org/10.1016/j.jtho.2016.04.018","featured":false,"firstPublished":{"html":"Aug 1, 2016 00:00:00 AM","text":"Aug 1, 2016 00:00:00 AM","value":"2016-08-01T00:00:00.000-07:00"},"mlaCitation":"Cristea, Sandra, and Julien Sage. ?Is the Canonical RAF/MEK/ERK Signaling Pathway aÂ Therapeutic Target in SCLC?? <i>Journal of thoracic oncology </i>11.8 (2016): 1233?1241. Print.","pubMedId":"27133774","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/27133774","publicationId":417630,"title":"Is the Canonical RAF/MEK/ERK Signaling Pathway aÂ Therapeutic Target in SCLC?","type":"article"},{"abstract":"Small cell lung cancer (SCLC) is a neuroendocrine lung cancer characterized by fast growth, early dissemination, and rapid resistance to chemotherapy. We identified a population of long-term tumor-propagating cells (TPCs) in a mouse model of SCLC. This population, marked by high levels of EpCAM and CD24, is also prevalent in human primary SCLC tumors. Murine SCLC TPCs are numerous and highly proliferative but not intrinsically chemoresistant, indicating that not all clinical features of SCLCÂ are linked to TPCs. SCLC TPCs possess a distinct transcriptional profile compared to non-TPCs, including elevated MYC activity. Genetic and pharmacological inhibition of MYC in SCLC cells to non-TPC levels inhibits long-term propagation but not short-term growth. These studies identify a highly tumorigenic population of SCLC cells in mouse models, cell lines, and patient tumors and a means to target them in this most fatal form of lung cancer.","apaCitation":"Jahchan, N. S., Lim, J. S., Bola, B., Morris, K., Seitz, G., Tran, K. Q., ? Sage, J. (2016). Identification and Targeting of Long-Term Tumor-Propagating Cells in Small Cell Lung Cancer. <i>Cell Reports</i>, <i>16</i>(3), 644?656.","capCitation":"<span class=\"title\">\n  <span>Identification and Targeting of Long-Term Tumor-Propagating Cells in Small Cell Lung Cancer.</span>\n  <i>Cell reports</i>\n</span>\n<span class=\"authors\">Jahchan, N. S., Lim, J. S., Bola, B., Morris, K., Seitz, G., Tran, K. Q., Xu, L., Trapani, F., Morrow, C. J., Cristea, S., Coles, G. L., Yang, D., Vaka, D., Kareta, M. S., George, J., Mazur, P. K., Nguyen, T., Anderson, W. C., Dylla, S. J., Blackhall, F., Peifer, M., Dive, C., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">16 (3)</span><span class=\"pages\">: 644-656</span>\n</span>","chicagoCitation":"Jahchan, Nadine S., Jing Shan Lim, Becky Bola, Karen Morris, Garrett Seitz, et al. 2016. ?Identification and Targeting of Long-Term Tumor-Propagating Cells in Small Cell Lung Cancer.? <i>Cell Reports</i> 16 (3). ELSEVIER INC: 644?56.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small cell lung cancer (SCLC) is a neuroendocrine lung cancer characterized by fast growth, early dissemination, and rapid resistance to chemotherapy. We identified a population of long-term tumor-propagating cells (TPCs) in a mouse model of SCLC. This population, marked by high levels of EpCAM and CD24, is also prevalent in human primary SCLC tumors. Murine SCLC TPCs are numerous and highly proliferative but not intrinsically chemoresistant, indicating that not all clinical features of SCLCÂ are linked to TPCs. SCLC TPCs possess a distinct transcriptional profile compared to non-TPCs, including elevated MYC activity. Genetic and pharmacological inhibition of MYC in SCLC cells to non-TPC levels inhibits long-term propagation but not short-term growth. These studies identify a highly tumorigenic population of SCLC cells in mouse models, cell lines, and patient tumors and a means to target them in this most fatal form of lung cancer.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.celrep.2016.06.021\">DOI 10.1016/j.celrep.2016.06.021</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/27373157\">PubMedID 27373157</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4956576\">PubMedCentralID PMC4956576</a></span>\n  </p>","text":"Abstract:\nSmall cell lung cancer (SCLC) is a neuroendocrine lung cancer characterized by fast growth, early dissemination, and rapid resistance to chemotherapy. We identified a population of long-term tumor-propagating cells (TPCs) in a mouse model of SCLC. This population, marked by high levels of EpCAM and CD24, is also prevalent in human primary SCLC tumors. Murine SCLC TPCs are numerous and highly proliferative but not intrinsically chemoresistant, indicating that not all clinical features of SCLCÂ are linked to TPCs. SCLC TPCs possess a distinct transcriptional profile compared to non-TPCs, including elevated MYC activity. Genetic and pharmacological inhibition of MYC in SCLC cells to non-TPC levels inhibits long-term propagation but not short-term growth. These studies identify a highly tumorigenic population of SCLC cells in mouse models, cell lines, and patient tumors and a means to target them in this most fatal form of lung cancer.\nDOI: 10.1016/j.celrep.2016.06.021, https://dx.doi.org/10.1016/j.celrep.2016.06.021\n\nPubMedID: 27373157, https://www.ncbi.nlm.nih.gov/pubmed/27373157\nPubMedCentralID: PMC4956576, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4956576"},"doiId":"10.1016/j.celrep.2016.06.021","doiUrl":"https://dx.doi.org/10.1016/j.celrep.2016.06.021","featured":true,"firstPublished":{"html":"Jul 19, 2016 00:00:00 AM","text":"Jul 19, 2016 00:00:00 AM","value":"2016-07-19T00:00:00.000-07:00"},"mlaCitation":"Jahchan, Nadine S. et al. ?Identification and Targeting of Long-Term Tumor-Propagating Cells in Small Cell Lung Cancer.? <i>Cell reports</i> 16.3 (2016): 644?656. Print.","pubMedId":"27373157","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/27373157","publicationId":416504,"title":"Identification and Targeting of Long-Term Tumor-Propagating Cells in Small Cell Lung Cancer.","type":"article"},{"abstract":"Metastases are the main cause of cancer deaths, but the mechanisms underlying metastatic progression remain poorly understood. We isolated pure populations of cancer cells from primary tumors and metastases from a genetically engineered mouse model of human small cell lung cancer (SCLC) to investigate the mechanisms that drive the metastatic spread ofÂ this lethal cancer. Genome-wide characterization of chromatin accessibility revealed the opening of large numbers of distal regulatory elements across the genome during metastatic progression. These changes correlate with copy number amplification of the Nfib locus, and differentially accessible sites were highly enriched for Nfib transcription factor binding sites. Nfib is necessary and sufficient to increase chromatin accessibility at a large subset of the intergenic regions. Nfib promotes pro-metastatic neuronal gene expression programs and drives the metastatic ability of SCLC cells. The identification of widespread chromatin changes during SCLC progression reveals an unexpected global reprogramming during metastatic progression.","apaCitation":"Denny, S. K., Yang, D., Chuang, C.-H., Brady, J. J., Lim, J. S., Gruner, B. M., ? Winslow, M. M. (2016). Nfib Promotes Metastasis through a Widespread Increase in Chromatin Accessibility. <i>CELL</i>, <i>166</i>(2), 328?342.","capCitation":"<span class=\"title\">\n  <span>Nfib Promotes Metastasis through a Widespread Increase in Chromatin Accessibility</span>\n  <i>CELL</i>\n</span>\n<span class=\"authors\">Denny, S. K., Yang, D., Chuang, C., Brady, J. J., Lim, J. S., Gruner, B. M., Chiou, S., Schep, A. N., Baral, J., Hamard, C., Antoine, M., Wislez, M., Kong, C. S., Connolly, A. J., Park, K., Sage, J., Greenleaf, W. J., Winslow, M. M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">166 (2)</span><span class=\"pages\">: 328-342</span>\n</span>","chicagoCitation":"Denny, Sarah K., Dian Yang, Chen-Hua Chuang, Jennifer J. Brady, Jing Shan Lim, et al. 2016. ?Nfib Promotes Metastasis through a Widespread Increase in Chromatin Accessibility.? <i>CELL</i> 166 (2). CELL PRESS: 328?42.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Metastases are the main cause of cancer deaths, but the mechanisms underlying metastatic progression remain poorly understood. We isolated pure populations of cancer cells from primary tumors and metastases from a genetically engineered mouse model of human small cell lung cancer (SCLC) to investigate the mechanisms that drive the metastatic spread ofÂ this lethal cancer. Genome-wide characterization of chromatin accessibility revealed the opening of large numbers of distal regulatory elements across the genome during metastatic progression. These changes correlate with copy number amplification of the Nfib locus, and differentially accessible sites were highly enriched for Nfib transcription factor binding sites. Nfib is necessary and sufficient to increase chromatin accessibility at a large subset of the intergenic regions. Nfib promotes pro-metastatic neuronal gene expression programs and drives the metastatic ability of SCLC cells. The identification of widespread chromatin changes during SCLC progression reveals an unexpected global reprogramming during metastatic progression.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.cell.2016.05.052\">DOI 10.1016/j.cell.2016.05.052</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000380255400012\">Web of Science ID 000380255400012</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/27374332\">PubMedID 27374332</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5004630\">PubMedCentralID PMC5004630</a></span>\n  </p>","text":"Abstract:\nMetastases are the main cause of cancer deaths, but the mechanisms underlying metastatic progression remain poorly understood. We isolated pure populations of cancer cells from primary tumors and metastases from a genetically engineered mouse model of human small cell lung cancer (SCLC) to investigate the mechanisms that drive the metastatic spread ofÂ this lethal cancer. Genome-wide characterization of chromatin accessibility revealed the opening of large numbers of distal regulatory elements across the genome during metastatic progression. These changes correlate with copy number amplification of the Nfib locus, and differentially accessible sites were highly enriched for Nfib transcription factor binding sites. Nfib is necessary and sufficient to increase chromatin accessibility at a large subset of the intergenic regions. Nfib promotes pro-metastatic neuronal gene expression programs and drives the metastatic ability of SCLC cells. The identification of widespread chromatin changes during SCLC progression reveals an unexpected global reprogramming during metastatic progression.\nDOI: 10.1016/j.cell.2016.05.052, https://dx.doi.org/10.1016/j.cell.2016.05.052\nWeb of Science ID: 000380255400012, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000380255400012\nPubMedID: 27374332, https://www.ncbi.nlm.nih.gov/pubmed/27374332\nPubMedCentralID: PMC5004630, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5004630"},"doiId":"10.1016/j.cell.2016.05.052","doiUrl":"https://dx.doi.org/10.1016/j.cell.2016.05.052","featured":true,"firstPublished":{"html":"Jul 14, 2016 00:00:00 AM","text":"Jul 14, 2016 00:00:00 AM","value":"2016-07-14T00:00:00.000-07:00"},"mlaCitation":"Denny, Sarah K. et al. ?Nfib Promotes Metastasis through a Widespread Increase in Chromatin Accessibility.? <i>CELL</i> 166.2 (2016): 328?342. Print.","pubMedId":"27374332","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/27374332","publicationId":413068,"title":"Nfib Promotes Metastasis through a Widespread Increase in Chromatin Accessibility","type":"article","webOfScienceId":"000380255400012","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000380255400012"},{"abstract":"Small-cell lung cancer (SCLC) is a highly aggressive subtype of lung cancer with limited treatment options. CD47 is a cell-surface molecule that promotes immune evasion by engaging signal-regulatory protein alpha (SIRP?), which serves as an inhibitory receptor on macrophages. Here, we found that CD47 is highly expressed on the surface of human SCLC cells; therefore, we investigated CD47-blocking immunotherapies as a potential approach for SCLC treatment. Disruption of the interaction of CD47 with SIRP? using anti-CD47 antibodies induced macrophage-mediated phagocytosis of human SCLC patient cells in culture. In a murine model, administration of CD47-blocking antibodies or targeted inactivation of the Cd47 gene markedly inhibited SCLC tumor growth. Furthermore, using comprehensive antibody arrays, we identified several possible therapeutic targets on the surface of SCLC cells. Antibodies to these targets, including CD56/neural cell adhesion molecule (NCAM), promoted phagocytosis in human SCLC cell lines that was enhanced when combined with CD47-blocking therapies. In light of recent clinical trials for CD47-blocking therapies in cancer treatment, these findings identify disruption of the CD47/SIRP? axis as a potential immunotherapeutic strategy for SCLC. This approach could enable personalized immunotherapeutic regimens in patients with SCLC and other cancers.","apaCitation":"Weiskopf, K., Jahchan, N. S., Schnorr, P. J., Cristea, S., Ring, A. M., Maute, R. L., ? Sage, J. (2016). CD47-blocking immunotherapies stimulate macrophage-mediated destruction of small-cell lung cancer. <i>JOURNAL OF CLINICAL INVESTIGATION</i>, <i>126</i>(7), 2610?2620.","capCitation":"<span class=\"title\">\n  <span>CD47-blocking immunotherapies stimulate macrophage-mediated destruction of small-cell lung cancer</span>\n  <i>JOURNAL OF CLINICAL INVESTIGATION</i>\n</span>\n<span class=\"authors\">Weiskopf, K., Jahchan, N. S., Schnorr, P. J., Cristea, S., Ring, A. M., Maute, R. L., Volkmer, A. K., Volkmer, J., Liu, J., Lim, J. S., Yang, D., Seitz, G., Thuyen Nguyen, T., Wu, D., Jude, K., Guerston, H., Barkal, A., Trapani, F., George, J., Poirier, J. T., Gardner, E. E., Miles, L. A., de Stanchina, E., Lofgren, S. M., Vogel, H., Winslow, M. M., Dive, C., Thomas, R. K., Rudin, C. M., van de Rijn, M., Majeti, R., Garcia, K. C., Weissman, I. L., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">126 (7)</span><span class=\"pages\">: 2610-2620</span>\n</span>","chicagoCitation":"Weiskopf, Kipp, Nadine S. Jahchan, Peter J. Schnorr, Sandra Cristea, Aaron M. Ring, et al. 2016. ?CD47-Blocking Immunotherapies Stimulate Macrophage-Mediated Destruction of Small-Cell Lung Cancer.? <i>JOURNAL OF CLINICAL INVESTIGATION</i> 126 (7). AMER SOC CLINICAL INVESTIGATION INC: 2610?20.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small-cell lung cancer (SCLC) is a highly aggressive subtype of lung cancer with limited treatment options. CD47 is a cell-surface molecule that promotes immune evasion by engaging signal-regulatory protein alpha (SIRP?), which serves as an inhibitory receptor on macrophages. Here, we found that CD47 is highly expressed on the surface of human SCLC cells; therefore, we investigated CD47-blocking immunotherapies as a potential approach for SCLC treatment. Disruption of the interaction of CD47 with SIRP? using anti-CD47 antibodies induced macrophage-mediated phagocytosis of human SCLC patient cells in culture. In a murine model, administration of CD47-blocking antibodies or targeted inactivation of the Cd47 gene markedly inhibited SCLC tumor growth. Furthermore, using comprehensive antibody arrays, we identified several possible therapeutic targets on the surface of SCLC cells. Antibodies to these targets, including CD56/neural cell adhesion molecule (NCAM), promoted phagocytosis in human SCLC cell lines that was enhanced when combined with CD47-blocking therapies. In light of recent clinical trials for CD47-blocking therapies in cancer treatment, these findings identify disruption of the CD47/SIRP? axis as a potential immunotherapeutic strategy for SCLC. This approach could enable personalized immunotherapeutic regimens in patients with SCLC and other cancers.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1172/JCI81603\">DOI 10.1172/JCI81603</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000379094800024\">Web of Science ID 000379094800024</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/27294525\">PubMedID 27294525</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4922696\">PubMedCentralID PMC4922696</a></span>\n  </p>","text":"Abstract:\nSmall-cell lung cancer (SCLC) is a highly aggressive subtype of lung cancer with limited treatment options. CD47 is a cell-surface molecule that promotes immune evasion by engaging signal-regulatory protein alpha (SIRP?), which serves as an inhibitory receptor on macrophages. Here, we found that CD47 is highly expressed on the surface of human SCLC cells; therefore, we investigated CD47-blocking immunotherapies as a potential approach for SCLC treatment. Disruption of the interaction of CD47 with SIRP? using anti-CD47 antibodies induced macrophage-mediated phagocytosis of human SCLC patient cells in culture. In a murine model, administration of CD47-blocking antibodies or targeted inactivation of the Cd47 gene markedly inhibited SCLC tumor growth. Furthermore, using comprehensive antibody arrays, we identified several possible therapeutic targets on the surface of SCLC cells. Antibodies to these targets, including CD56/neural cell adhesion molecule (NCAM), promoted phagocytosis in human SCLC cell lines that was enhanced when combined with CD47-blocking therapies. In light of recent clinical trials for CD47-blocking therapies in cancer treatment, these findings identify disruption of the CD47/SIRP? axis as a potential immunotherapeutic strategy for SCLC. This approach could enable personalized immunotherapeutic regimens in patients with SCLC and other cancers.\nDOI: 10.1172/JCI81603, https://dx.doi.org/10.1172/JCI81603\nWeb of Science ID: 000379094800024, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000379094800024\nPubMedID: 27294525, https://www.ncbi.nlm.nih.gov/pubmed/27294525\nPubMedCentralID: PMC4922696, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4922696"},"doiId":"10.1172/JCI81603","doiUrl":"https://dx.doi.org/10.1172/JCI81603","featured":true,"firstPublished":{"html":"Jul 1, 2016 00:00:00 AM","text":"Jul 1, 2016 00:00:00 AM","value":"2016-07-01T00:00:00.000-07:00"},"mlaCitation":"Weiskopf, Kipp et al. ?CD47-Blocking Immunotherapies Stimulate Macrophage-Mediated Destruction of Small-Cell Lung Cancer.? <i>JOURNAL OF CLINICAL INVESTIGATION</i> 126.7 (2016): 2610?2620. Print.","pubMedId":"27294525","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/27294525","publicationId":416393,"title":"CD47-blocking immunotherapies stimulate macrophage-mediated destruction of small-cell lung cancer","type":"article","webOfScienceId":"000379094800024","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000379094800024"},{"abstract":"Pancreatic neuroendocrine tumors (PanNETs) are a type of pancreatic cancer with limited therapeutic options. Consequently, most patients with advanced disease die from tumor progression. Current evidence indicates that a subset of cancer cells is responsible for tumor development, metastasis, and recurrence, and targeting these tumor-initiating cells is necessary to eradicate tumors. However, tumor-initiating cells and the biological processes that promote pathogenesis remain largely uncharacterized in PanNETs. Here we profile primary and metastatic tumors from an index patient and demonstrate that MET proto-oncogene activation is important for tumor growth in PanNET xenograft models. We identify a highly tumorigenic cell population within several independent surgically acquired PanNETs characterized by increased cell-surface protein CD90 expression and aldehyde dehydrogenase A1 (ALDHA1) activity, and provide in vitro and in vivo evidence for their stem-like properties. We performed proteomic profiling of 332 antigens in two cell lines and four primary tumors, and showed that CD47, a cell-surface protein that acts as a \"don't eat me\" signal co-opted by cancers to evade innate immune surveillance, is ubiquitously expressed. Moreover, CD47 coexpresses with MET and is enriched in CD90(hi)cells. Furthermore, blocking CD47 signaling promotes engulfment of tumor cells by macrophages in vitro and inhibits xenograft tumor growth, prevents metastases, and prolongs survival in vivo.","apaCitation":"Krampitz, G. W., George, B. M., Willingham, S. B., Volkmer, J.-P., Weiskopf, K., Jahchan, N., ? Weissman, I. L. (2016). Identification of tumorigenic cells and therapeutic targets in pancreatic neuroendocrine tumors. <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i>, <i>113</i>(16), 4464?4469.","capCitation":"<span class=\"title\">\n  <span>Identification of tumorigenic cells and therapeutic targets in pancreatic neuroendocrine tumors</span>\n  <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i>\n</span>\n<span class=\"authors\">Krampitz, G. W., George, B. M., Willingham, S. B., Volkmer, J., Weiskopf, K., Jahchan, N., Newman, A. M., Sahoo, D., Zemek, A. J., Yanovsky, R. L., Nguyen, J. K., Schnorr, P. J., Mazur, P. K., Sage, J., Longacre, T. A., Visser, B. C., Poultsides, G. A., Norton, J. A., Weissman, I. L.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">113 (16)</span><span class=\"pages\">: 4464-4469</span>\n</span>","chicagoCitation":"Krampitz, Geoffrey Wayne, Benson M. George, Stephen B. Willingham, Jens-Peter Volkmer, Kipp Weiskopf, et al. 2016. ?Identification of Tumorigenic Cells and Therapeutic Targets in Pancreatic Neuroendocrine Tumors.? <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i> 113 (16). NATL ACAD SCIENCES: 4464?69.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Pancreatic neuroendocrine tumors (PanNETs) are a type of pancreatic cancer with limited therapeutic options. Consequently, most patients with advanced disease die from tumor progression. Current evidence indicates that a subset of cancer cells is responsible for tumor development, metastasis, and recurrence, and targeting these tumor-initiating cells is necessary to eradicate tumors. However, tumor-initiating cells and the biological processes that promote pathogenesis remain largely uncharacterized in PanNETs. Here we profile primary and metastatic tumors from an index patient and demonstrate that MET proto-oncogene activation is important for tumor growth in PanNET xenograft models. We identify a highly tumorigenic cell population within several independent surgically acquired PanNETs characterized by increased cell-surface protein CD90 expression and aldehyde dehydrogenase A1 (ALDHA1) activity, and provide in vitro and in vivo evidence for their stem-like properties. We performed proteomic profiling of 332 antigens in two cell lines and four primary tumors, and showed that CD47, a cell-surface protein that acts as a \"don't eat me\" signal co-opted by cancers to evade innate immune surveillance, is ubiquitously expressed. Moreover, CD47 coexpresses with MET and is enriched in CD90(hi)cells. Furthermore, blocking CD47 signaling promotes engulfment of tumor cells by macrophages in vitro and inhibits xenograft tumor growth, prevents metastases, and prolongs survival in vivo.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1073/pnas.1600007113\">DOI 10.1073/pnas.1600007113</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000374393800063\">Web of Science ID 000374393800063</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/27035983\">PubMedID 27035983</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4843455\">PubMedCentralID PMC4843455</a></span>\n  </p>","text":"Abstract:\nPancreatic neuroendocrine tumors (PanNETs) are a type of pancreatic cancer with limited therapeutic options. Consequently, most patients with advanced disease die from tumor progression. Current evidence indicates that a subset of cancer cells is responsible for tumor development, metastasis, and recurrence, and targeting these tumor-initiating cells is necessary to eradicate tumors. However, tumor-initiating cells and the biological processes that promote pathogenesis remain largely uncharacterized in PanNETs. Here we profile primary and metastatic tumors from an index patient and demonstrate that MET proto-oncogene activation is important for tumor growth in PanNET xenograft models. We identify a highly tumorigenic cell population within several independent surgically acquired PanNETs characterized by increased cell-surface protein CD90 expression and aldehyde dehydrogenase A1 (ALDHA1) activity, and provide in vitro and in vivo evidence for their stem-like properties. We performed proteomic profiling of 332 antigens in two cell lines and four primary tumors, and showed that CD47, a cell-surface protein that acts as a \"don't eat me\" signal co-opted by cancers to evade innate immune surveillance, is ubiquitously expressed. Moreover, CD47 coexpresses with MET and is enriched in CD90(hi)cells. Furthermore, blocking CD47 signaling promotes engulfment of tumor cells by macrophages in vitro and inhibits xenograft tumor growth, prevents metastases, and prolongs survival in vivo.\nDOI: 10.1073/pnas.1600007113, https://dx.doi.org/10.1073/pnas.1600007113\nWeb of Science ID: 000374393800063, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000374393800063\nPubMedID: 27035983, https://www.ncbi.nlm.nih.gov/pubmed/27035983\nPubMedCentralID: PMC4843455, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4843455"},"doiId":"10.1073/pnas.1600007113","doiUrl":"https://dx.doi.org/10.1073/pnas.1600007113","featured":false,"firstPublished":{"html":"Apr 19, 2016 00:00:00 AM","text":"Apr 19, 2016 00:00:00 AM","value":"2016-04-19T00:00:00.000-07:00"},"mlaCitation":"Krampitz, Geoffrey Wayne et al. ?Identification of Tumorigenic Cells and Therapeutic Targets in Pancreatic Neuroendocrine Tumors.? <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i> 113.16 (2016): 4464?4469. Print.","pubMedId":"27035983","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/27035983","publicationId":403318,"title":"Identification of tumorigenic cells and therapeutic targets in pancreatic neuroendocrine tumors","type":"article","webOfScienceId":"000374393800063","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000374393800063"},{"apaCitation":"Reynoird, N., Mazur, P. K., Stellfeld, T., Flores, N. M., Lofgren, S. M., Carlson, S. M., ? Sage, J. (2016). Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer. <i>GENES &amp; DEVELOPMENT</i>, <i>30</i>(7), 772?785.","capCitation":"<span class=\"title\">\n  <span>Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer</span>\n  <i>GENES &amp; DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Reynoird, N., Mazur, P. K., Stellfeld, T., Flores, N. M., Lofgren, S. M., Carlson, S. M., Brambilla, E., Hainaut, P., Kaznowska, E. B., Arrowsmith, C. H., Khatri, P., Stresemann, C., Gozani, O., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">30 (7)</span><span class=\"pages\">: 772-785</span>\n</span>","chicagoCitation":"Reynoird, Nicolas, Pawel K. Mazur, Timo Stellfeld, Natasha M. Flores, Shane M. Lofgren, et al. 2016. ?Coordination of Stress Signals by the Lysine Methyltransferase SMYD2 Promotes Pancreatic Cancer.? <i>GENES &amp; DEVELOPMENT</i> 30 (7). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 772?85.","detail":{"html":"<p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.275529.115\">DOI 10.1101/gad.275529.115</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000373194300003\">Web of Science ID 000373194300003</a></span>\n  </p>","text":"DOI: 10.1101/gad.275529.115, https://dx.doi.org/10.1101/gad.275529.115\nWeb of Science ID: 000373194300003, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000373194300003"},"doiId":"10.1101/gad.275529.115","doiUrl":"https://dx.doi.org/10.1101/gad.275529.115","featured":true,"firstPublished":{"html":"Apr 1, 2016 00:00:00 AM","text":"Apr 1, 2016 00:00:00 AM","value":"2016-04-01T00:00:00.000-07:00"},"mlaCitation":"Reynoird, Nicolas et al. ?Coordination of Stress Signals by the Lysine Methyltransferase SMYD2 Promotes Pancreatic Cancer.? <i>GENES &amp; DEVELOPMENT</i> 30.7 (2016): 772?785. Print.","publicationId":405438,"title":"Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer","type":"article","webOfScienceId":"000373194300003","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000373194300003"},{"abstract":"Pancreatic ductal adenocarcinoma (PDAC) is a lethal form of cancer with few therapeutic options. We found that levels of the lysine methyltransferase SMYD2 (SET and MYND domain 2) are elevated in PDAC and that genetic and pharmacological inhibition of SMYD2 restricts PDAC growth. We further identified the stress response kinase MAPKAPK3 (MK3) as a new physiologic substrate of SMYD2 in PDAC cells. Inhibition of MAPKAPK3 impedes PDAC growth, identifying a potential new kinase target in PDAC. Finally, we show that inhibition of SMYD2 cooperates with standard chemotherapy to treat PDAC cells and tumors. These findings uncover a pivotal role for SMYD2 in promoting pancreatic cancer.","apaCitation":"Reynoird, N., Mazur, P. K., Stellfeld, T., Flores, N. M., Lofgren, S. M., Carlson, S. M., ? Sage, J. (2016). Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer. <i>Genes &amp; Development</i>, <i>30</i>(7), 772?785.","capCitation":"<span class=\"title\">\n  <span>Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer.</span>\n  <i>Genes &amp; development</i>\n</span>\n<span class=\"authors\">Reynoird, N., Mazur, P. K., Stellfeld, T., Flores, N. M., Lofgren, S. M., Carlson, S. M., Brambilla, E., Hainaut, P., Kaznowska, E. B., Arrowsmith, C. H., Khatri, P., Stresemann, C., Gozani, O., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">30 (7)</span><span class=\"pages\">: 772-785</span>\n</span>","chicagoCitation":"Reynoird, Nicolas, Pawel K. Mazur, Timo Stellfeld, Natasha M. Flores, Shane M. Lofgren, et al. 2016. ?Coordination of Stress Signals by the Lysine Methyltransferase SMYD2 Promotes Pancreatic Cancer.? <i>Genes &amp; Development</i> 30 (7). COLD SPRING HARBOR LABORATORY PRESS: 772?85.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Pancreatic ductal adenocarcinoma (PDAC) is a lethal form of cancer with few therapeutic options. We found that levels of the lysine methyltransferase SMYD2 (SET and MYND domain 2) are elevated in PDAC and that genetic and pharmacological inhibition of SMYD2 restricts PDAC growth. We further identified the stress response kinase MAPKAPK3 (MK3) as a new physiologic substrate of SMYD2 in PDAC cells. Inhibition of MAPKAPK3 impedes PDAC growth, identifying a potential new kinase target in PDAC. Finally, we show that inhibition of SMYD2 cooperates with standard chemotherapy to treat PDAC cells and tumors. These findings uncover a pivotal role for SMYD2 in promoting pancreatic cancer.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.275529.115\">DOI 10.1101/gad.275529.115</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/26988419\">PubMedID 26988419</a></span>\n  </p>","text":"Abstract:\nPancreatic ductal adenocarcinoma (PDAC) is a lethal form of cancer with few therapeutic options. We found that levels of the lysine methyltransferase SMYD2 (SET and MYND domain 2) are elevated in PDAC and that genetic and pharmacological inhibition of SMYD2 restricts PDAC growth. We further identified the stress response kinase MAPKAPK3 (MK3) as a new physiologic substrate of SMYD2 in PDAC cells. Inhibition of MAPKAPK3 impedes PDAC growth, identifying a potential new kinase target in PDAC. Finally, we show that inhibition of SMYD2 cooperates with standard chemotherapy to treat PDAC cells and tumors. These findings uncover a pivotal role for SMYD2 in promoting pancreatic cancer.\nDOI: 10.1101/gad.275529.115, https://dx.doi.org/10.1101/gad.275529.115\n\nPubMedID: 26988419, https://www.ncbi.nlm.nih.gov/pubmed/26988419"},"doiId":"10.1101/gad.275529.115","doiUrl":"https://dx.doi.org/10.1101/gad.275529.115","featured":true,"firstPublished":{"html":"Apr 1, 2016 00:00:00 AM","text":"Apr 1, 2016 00:00:00 AM","value":"2016-04-01T00:00:00.000-07:00"},"mlaCitation":"Reynoird, Nicolas et al. ?Coordination of Stress Signals by the Lysine Methyltransferase SMYD2 Promotes Pancreatic Cancer.? <i>Genes &amp; development</i> 30.7 (2016): 772?785. Print.","pubMedId":"26988419","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/26988419","publicationId":405926,"title":"Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer.","type":"article"},{"abstract":"The control of cell division is essential for normal development and the maintenance of cellular homeostasis. Abnormal cell proliferation is associated with multiple pathological states, including cancer. Although the Hippo/YAP signaling pathway was initially thought to control organ size and growth, increasing evidence indicates that this pathway also plays a major role in the control of proliferation independent of organ size control. In particular, accumulating evidence indicates that the Hippo/YAP signaling pathway functionally interacts with multiple other cellular pathways and serves as a central node in the regulation of cell division, especially in cancer cells. Here, recent observations are highlighted that connect Hippo/YAP signaling to transcription, the basic cell-cycle machinery, and the control of cell division. Furthermore, the oncogenic and tumor-suppressive attributes of YAP/TAZ are reviewed, which emphasizes the relevance of the Hippo pathway in cancer. Mol Cancer Res; 14(2); 127-40. Â©2015 AACR.","apaCitation":"Ehmer, U., &amp; Sage, J. (2016). Control of Proliferation and Cancer Growth by the Hippo Signaling Pathway. <i>Molecular Cancer Research </i>, <i>14</i>(2), 127?140.","capCitation":"<span class=\"title\">\n  <span>Control of Proliferation and Cancer Growth by the Hippo Signaling Pathway.</span>\n  <i>Molecular cancer research </i>\n</span>\n<span class=\"authors\">Ehmer, U., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">14 (2)</span><span class=\"pages\">: 127-140</span>\n</span>","chicagoCitation":"Ehmer, Ursula, and Julien Sage. 2016. ?Control of Proliferation and Cancer Growth by the Hippo Signaling Pathway.? <i>Molecular Cancer Research </i>14 (2). AMER ASSN FOR CANCER RESEARCH: 127?40.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The control of cell division is essential for normal development and the maintenance of cellular homeostasis. Abnormal cell proliferation is associated with multiple pathological states, including cancer. Although the Hippo/YAP signaling pathway was initially thought to control organ size and growth, increasing evidence indicates that this pathway also plays a major role in the control of proliferation independent of organ size control. In particular, accumulating evidence indicates that the Hippo/YAP signaling pathway functionally interacts with multiple other cellular pathways and serves as a central node in the regulation of cell division, especially in cancer cells. Here, recent observations are highlighted that connect Hippo/YAP signaling to transcription, the basic cell-cycle machinery, and the control of cell division. Furthermore, the oncogenic and tumor-suppressive attributes of YAP/TAZ are reviewed, which emphasizes the relevance of the Hippo pathway in cancer. Mol Cancer Res; 14(2); 127-40. Â©2015 AACR.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1158/1541-7786.MCR-15-0305\">DOI 10.1158/1541-7786.MCR-15-0305</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/26432795\">PubMedID 26432795</a></span>\n  </p>","text":"Abstract:\nThe control of cell division is essential for normal development and the maintenance of cellular homeostasis. Abnormal cell proliferation is associated with multiple pathological states, including cancer. Although the Hippo/YAP signaling pathway was initially thought to control organ size and growth, increasing evidence indicates that this pathway also plays a major role in the control of proliferation independent of organ size control. In particular, accumulating evidence indicates that the Hippo/YAP signaling pathway functionally interacts with multiple other cellular pathways and serves as a central node in the regulation of cell division, especially in cancer cells. Here, recent observations are highlighted that connect Hippo/YAP signaling to transcription, the basic cell-cycle machinery, and the control of cell division. Furthermore, the oncogenic and tumor-suppressive attributes of YAP/TAZ are reviewed, which emphasizes the relevance of the Hippo pathway in cancer. Mol Cancer Res; 14(2); 127-40. Â©2015 AACR.\nDOI: 10.1158/1541-7786.MCR-15-0305, https://dx.doi.org/10.1158/1541-7786.MCR-15-0305\n\nPubMedID: 26432795, https://www.ncbi.nlm.nih.gov/pubmed/26432795"},"doiId":"10.1158/1541-7786.MCR-15-0305","doiUrl":"https://dx.doi.org/10.1158/1541-7786.MCR-15-0305","featured":false,"firstPublished":{"html":"Feb 1, 2016 00:00:00 AM","text":"Feb 1, 2016 00:00:00 AM","value":"2016-02-01T00:00:00.000-08:00"},"mlaCitation":"Ehmer, Ursula, and Julien Sage. ?Control of Proliferation and Cancer Growth by the Hippo Signaling Pathway.? <i>Molecular cancer research </i>14.2 (2016): 127?140. Print.","pubMedId":"26432795","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/26432795","publicationId":407717,"title":"Control of Proliferation and Cancer Growth by the Hippo Signaling Pathway.","type":"article"},{"abstract":"The thymus is the site of T cell development and selection. In addition to lymphocytes, the thymus is composed of several types of stromal cells that are exquisitely organized to create the appropriate environment and microenvironment to support the development and selection of maturing T cells. Thymic epithelial cells (TECs) are one of the more important cell types in the thymic stroma, and they play a critical role in selecting functional T cell clones and supporting their development. In this study, we used a mouse genetics approach to investigate the consequences of deleting the Pten tumor suppressor gene in the TEC compartment of the developing thymus. We found that PTEN deficiency in TECs results in a smaller thymus with significantly disordered architecture and histology. Accordingly, loss of PTEN function also results in decreased T cells with a shift in the distribution of T cell subtypes towards CD8+ T cells. These experiments demonstrate that PTEN is critically required for the development of a functional thymic epithelium in mice. This work may help better understand the effects that certain medical conditions or clinical interventions have upon the thymus and immune function.","apaCitation":"Garfin, P. M., Nguyen, T., &amp; Sage, J. (2016). Loss of Pten Disrupts the Thymic Epithelium and Alters Thymic Function. <i>PloS One</i>, <i>11</i>(2).","capCitation":"<span class=\"title\">\n  <span>Loss of Pten Disrupts the Thymic Epithelium and Alters Thymic Function.</span>\n  <i>PloS one</i>\n</span>\n<span class=\"authors\">Garfin, P. M., Nguyen, T., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">11 (2)</span>\n</span>","chicagoCitation":"Garfin, Phillip M., Thuyen Nguyen, and Julien Sage. 2016. ?Loss of Pten Disrupts the Thymic Epithelium and Alters Thymic Function.? <i>PloS One</i> 11 (2).","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The thymus is the site of T cell development and selection. In addition to lymphocytes, the thymus is composed of several types of stromal cells that are exquisitely organized to create the appropriate environment and microenvironment to support the development and selection of maturing T cells. Thymic epithelial cells (TECs) are one of the more important cell types in the thymic stroma, and they play a critical role in selecting functional T cell clones and supporting their development. In this study, we used a mouse genetics approach to investigate the consequences of deleting the Pten tumor suppressor gene in the TEC compartment of the developing thymus. We found that PTEN deficiency in TECs results in a smaller thymus with significantly disordered architecture and histology. Accordingly, loss of PTEN function also results in decreased T cells with a shift in the distribution of T cell subtypes towards CD8+ T cells. These experiments demonstrate that PTEN is critically required for the development of a functional thymic epithelium in mice. This work may help better understand the effects that certain medical conditions or clinical interventions have upon the thymus and immune function.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1371/journal.pone.0149430\">DOI 10.1371/journal.pone.0149430</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/26914657\">PubMedID 26914657</a></span>\n  </p>","text":"Abstract:\nThe thymus is the site of T cell development and selection. In addition to lymphocytes, the thymus is composed of several types of stromal cells that are exquisitely organized to create the appropriate environment and microenvironment to support the development and selection of maturing T cells. Thymic epithelial cells (TECs) are one of the more important cell types in the thymic stroma, and they play a critical role in selecting functional T cell clones and supporting their development. In this study, we used a mouse genetics approach to investigate the consequences of deleting the Pten tumor suppressor gene in the TEC compartment of the developing thymus. We found that PTEN deficiency in TECs results in a smaller thymus with significantly disordered architecture and histology. Accordingly, loss of PTEN function also results in decreased T cells with a shift in the distribution of T cell subtypes towards CD8+ T cells. These experiments demonstrate that PTEN is critically required for the development of a functional thymic epithelium in mice. This work may help better understand the effects that certain medical conditions or clinical interventions have upon the thymus and immune function.\nDOI: 10.1371/journal.pone.0149430, https://dx.doi.org/10.1371/journal.pone.0149430\n\nPubMedID: 26914657, https://www.ncbi.nlm.nih.gov/pubmed/26914657"},"doiId":"10.1371/journal.pone.0149430","doiUrl":"https://dx.doi.org/10.1371/journal.pone.0149430","featured":false,"firstPublished":{"html":"Jan 1, 2016 00:00:00 AM","text":"Jan 1, 2016 00:00:00 AM","value":"2016-01-01T00:00:00.000-08:00"},"mlaCitation":"Garfin, Phillip M., Thuyen Nguyen, and Julien Sage. ?Loss of Pten Disrupts the Thymic Epithelium and Alters Thymic Function.? <i>PloS one</i> 11.2 (2016): n. pag. Print.","pubMedId":"26914657","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/26914657","publicationId":407718,"title":"Loss of Pten Disrupts the Thymic Epithelium and Alters Thymic Function.","type":"article"},{"abstract":"Pluripotent stem cells, defined by an unlimited self-renewal capacity and an undifferentiated state, are best typified by embryonic stem cells. These cells have a unique cell cycle compared to somatic cells as defined by a rapid progression through the cell cycle and a minimal time spent in G1. Recent reports indicate that pluripotency and cell cycle regulation are mechanistically linked. In this review, we discuss the reciprocal co-regulation of these processes, how this co-regulation may prevent differentiation, and how cellular reprogramming can re-establish the unique cell cycle regulation in induced pluripotent stem cells.","apaCitation":"Kareta, M. S., Sage, J., &amp; Wernig, M. (2015). Crosstalk between stem cell and cell cycle machineries. <i>CURRENT OPINION IN CELL BIOLOGY</i>, <i>37</i>, 68?74.","capCitation":"<span class=\"title\">\n  <span>Crosstalk between stem cell and cell cycle machineries</span>\n  <i>CURRENT OPINION IN CELL BIOLOGY</i>\n</span>\n<span class=\"authors\">Kareta, M. S., Sage, J., Wernig, M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2015</span>; <span class=\"volume\">37</span><span class=\"pages\">: 68-74</span>\n</span>","chicagoCitation":"Kareta, Michael S., Julien Sage, and Marius Wernig. 2015. ?Crosstalk between Stem Cell and Cell Cycle Machineries.? <i>CURRENT OPINION IN CELL BIOLOGY</i> 37. CURRENT BIOLOGY LTD: 68?74.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Pluripotent stem cells, defined by an unlimited self-renewal capacity and an undifferentiated state, are best typified by embryonic stem cells. These cells have a unique cell cycle compared to somatic cells as defined by a rapid progression through the cell cycle and a minimal time spent in G1. Recent reports indicate that pluripotency and cell cycle regulation are mechanistically linked. In this review, we discuss the reciprocal co-regulation of these processes, how this co-regulation may prevent differentiation, and how cellular reprogramming can re-establish the unique cell cycle regulation in induced pluripotent stem cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.ceb.2015.10.001\">DOI 10.1016/j.ceb.2015.10.001</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000367112800010\">Web of Science ID 000367112800010</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/26520682\">PubMedID 26520682</a></span>\n  </p>","text":"Abstract:\nPluripotent stem cells, defined by an unlimited self-renewal capacity and an undifferentiated state, are best typified by embryonic stem cells. These cells have a unique cell cycle compared to somatic cells as defined by a rapid progression through the cell cycle and a minimal time spent in G1. Recent reports indicate that pluripotency and cell cycle regulation are mechanistically linked. In this review, we discuss the reciprocal co-regulation of these processes, how this co-regulation may prevent differentiation, and how cellular reprogramming can re-establish the unique cell cycle regulation in induced pluripotent stem cells.\nDOI: 10.1016/j.ceb.2015.10.001, https://dx.doi.org/10.1016/j.ceb.2015.10.001\nWeb of Science ID: 000367112800010, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000367112800010\nPubMedID: 26520682, https://www.ncbi.nlm.nih.gov/pubmed/26520682"},"doiId":"10.1016/j.ceb.2015.10.001","doiUrl":"https://dx.doi.org/10.1016/j.ceb.2015.10.001","featured":false,"firstPublished":{"html":"Dec 1, 2015 00:00:00 AM","text":"Dec 1, 2015 00:00:00 AM","value":"2015-12-01T00:00:00.000-08:00"},"mlaCitation":"Kareta, Michael S., Julien Sage, and Marius Wernig. ?Crosstalk between Stem Cell and Cell Cycle Machineries.? <i>CURRENT OPINION IN CELL BIOLOGY</i> 37 (2015): 68?74. Print.","pubMedId":"26520682","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/26520682","publicationId":394915,"title":"Crosstalk between stem cell and cell cycle machineries","type":"article","webOfScienceId":"000367112800010","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000367112800010"},{"abstract":"Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal human cancers and shows resistance to any therapeutic strategy used. Here we tested small-molecule inhibitors targeting chromatin regulators as possible therapeutic agents in PDAC. We show that JQ1, an inhibitor of the bromodomain and extraterminal (BET) family of proteins, suppresses PDAC development in mice by inhibiting both MYC activity and inflammatory signals. The histone deacetylase (HDAC) inhibitor SAHA synergizes with JQ1 to augment cell death and more potently suppress advanced PDAC. Finally, using a CRISPR-Cas9-based method for gene editing directly in the mouse adult pancreas, we show that de-repression of p57 (also known as KIP2 or CDKN1C) upon combined BET and HDAC inhibition is required for the induction of combination therapy-induced cell death in PDAC. SAHA is approved for human use, and molecules similar to JQ1 are being tested in clinical trials. Thus, these studies identify a promising epigenetic-based therapeutic strategy that may be rapidly implemented in fatal human tumors.","apaCitation":"Mazur, P. K., Herner, A., Mello, S. S., Wirth, M., Hausmann, S., SÃ¡nchez-Rivera, F. J., ? Siveke, J. T. (2015). Combined inhibition of BET family proteins and histone deacetylases as a potential epigenetics-based therapy for pancreatic ductal adenocarcinoma. <i>Nature Medicine</i>, <i>21</i>(10), 1163?1171.","capCitation":"<span class=\"title\">\n  <span>Combined inhibition of BET family proteins and histone deacetylases as a potential epigenetics-based therapy for pancreatic ductal adenocarcinoma.</span>\n  <i>Nature medicine</i>\n</span>\n<span class=\"authors\">Mazur, P. K., Herner, A., Mello, S. S., Wirth, M., Hausmann, S., SÃ¡nchez-Rivera, F. J., Lofgren, S. M., Kuschma, T., Hahn, S. A., Vangala, D., Trajkovic-Arsic, M., Gupta, A., Heid, I., NoÃ«l, P. B., Braren, R., Erkan, M., Kleeff, J., Sipos, B., Sayles, L. C., Heikenwalder, M., HeÃŸmann, E., Ellenrieder, V., Esposito, I., Jacks, T., Bradner, J. E., Khatri, P., Sweet-Cordero, E. A., Attardi, L. D., Schmid, R. M., Schneider, G., Sage, J., Siveke, J. T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2015</span>; <span class=\"volume\">21 (10)</span><span class=\"pages\">: 1163-1171</span>\n</span>","chicagoCitation":"Mazur, Pawel K., Alexander Herner, Stephano S. Mello, Matthias Wirth, Simone Hausmann, et al. 2015. ?Combined Inhibition of BET Family Proteins and Histone Deacetylases as a Potential Epigenetics-Based Therapy for Pancreatic Ductal Adenocarcinoma.? <i>Nature Medicine</i> 21 (10): 1163?71.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal human cancers and shows resistance to any therapeutic strategy used. Here we tested small-molecule inhibitors targeting chromatin regulators as possible therapeutic agents in PDAC. We show that JQ1, an inhibitor of the bromodomain and extraterminal (BET) family of proteins, suppresses PDAC development in mice by inhibiting both MYC activity and inflammatory signals. The histone deacetylase (HDAC) inhibitor SAHA synergizes with JQ1 to augment cell death and more potently suppress advanced PDAC. Finally, using a CRISPR-Cas9-based method for gene editing directly in the mouse adult pancreas, we show that de-repression of p57 (also known as KIP2 or CDKN1C) upon combined BET and HDAC inhibition is required for the induction of combination therapy-induced cell death in PDAC. SAHA is approved for human use, and molecules similar to JQ1 are being tested in clinical trials. Thus, these studies identify a promising epigenetic-based therapeutic strategy that may be rapidly implemented in fatal human tumors.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nm.3952\">DOI 10.1038/nm.3952</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/26390243\">PubMedID 26390243</a></span>\n  </p>","text":"Abstract:\nPancreatic ductal adenocarcinoma (PDAC) is one of the most lethal human cancers and shows resistance to any therapeutic strategy used. Here we tested small-molecule inhibitors targeting chromatin regulators as possible therapeutic agents in PDAC. We show that JQ1, an inhibitor of the bromodomain and extraterminal (BET) family of proteins, suppresses PDAC development in mice by inhibiting both MYC activity and inflammatory signals. The histone deacetylase (HDAC) inhibitor SAHA synergizes with JQ1 to augment cell death and more potently suppress advanced PDAC. Finally, using a CRISPR-Cas9-based method for gene editing directly in the mouse adult pancreas, we show that de-repression of p57 (also known as KIP2 or CDKN1C) upon combined BET and HDAC inhibition is required for the induction of combination therapy-induced cell death in PDAC. SAHA is approved for human use, and molecules similar to JQ1 are being tested in clinical trials. Thus, these studies identify a promising epigenetic-based therapeutic strategy that may be rapidly implemented in fatal human tumors.\nDOI: 10.1038/nm.3952, https://dx.doi.org/10.1038/nm.3952\n\nPubMedID: 26390243, https://www.ncbi.nlm.nih.gov/pubmed/26390243"},"doiId":"10.1038/nm.3952","doiUrl":"https://dx.doi.org/10.1038/nm.3952","featured":true,"firstPublished":{"html":"Oct 1, 2015 00:00:00 AM","text":"Oct 1, 2015 00:00:00 AM","value":"2015-10-01T00:00:00.000-07:00"},"mlaCitation":"Mazur, Pawel K. et al. ?Combined Inhibition of BET Family Proteins and Histone Deacetylases as a Potential Epigenetics-Based Therapy for Pancreatic Ductal Adenocarcinoma.? <i>Nature medicine</i> 21.10 (2015): 1163?1171. Print.","pubMedId":"26390243","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/26390243","publicationId":398340,"title":"Combined inhibition of BET family proteins and histone deacetylases as a potential epigenetics-based therapy for pancreatic ductal adenocarcinoma.","type":"article"},{"abstract":"We have sequenced the genomes of 110 small cell lung cancers (SCLC), one of the deadliest human cancers. In nearly all the tumours analysed we found bi-allelic inactivation of TP53 and RB1, sometimes by complex genomic rearrangements. Two tumours with wild-type RB1 had evidence of chromothripsis leading to overexpression of cyclin D1 (encoded by the CCND1 gene), revealing an alternative mechanism of Rb1 deregulation. Thus, loss of the tumour suppressors TP53 and RB1 is obligatory in SCLC. We discovered somatic genomic rearrangements of TP73 that create an oncogenic version of this gene, TP73?ex2/3. In rare cases, SCLC tumours exhibited kinase gene mutations, providing a possible therapeutic opportunity for individual patients. Finally, we observed inactivating mutations in NOTCH family genes in 25% of human SCLC. Accordingly, activation of Notch signalling in a pre-clinical SCLC mouse model strikingly reduced the number of tumours and extended the survival of the mutant mice. Furthermore, neuroendocrine gene expression was abrogated by Notch activity in SCLC cells. This first comprehensive study of somatic genome alterations in SCLC uncovers several key biological processes and identifies candidate therapeutic targets in this highly lethal form of cancer.","apaCitation":"George, J., Lim, J. S., Jang, S. J., Cun, Y., Ozretic, L., Kong, G., ? Thomas, R. K. (2015). Comprehensive genomic profiles of small cell lung cancer. <i>Nature</i>, <i>524</i>(7563), 47?53.","capCitation":"<span class=\"title\">\n  <span>Comprehensive genomic profiles of small cell lung cancer.</span>\n  <i>Nature</i>\n</span>\n<span class=\"authors\">George, J., Lim, J. S., Jang, S. J., Cun, Y., Ozretic, L., Kong, G., Leenders, F., Lu, X., FernÃ¡ndez-Cuesta, L., Bosco, G., MÃ¼ller, C., Dahmen, I., Jahchan, N. S., Park, K., Yang, D., Karnezis, A. N., Vaka, D., Torres, A., Wang, M. S., Korbel, J. O., Menon, R., Chun, S., Kim, D., Wilkerson, M., Hayes, N., Engelmann, D., PÃ¼tzer, B., Bos, M., Michels, S., Vlasic, I., Seidel, D., Pinther, B., Schaub, P., Becker, C., AltmÃ¼ller, J., Yokota, J., Kohno, T., Iwakawa, R., Tsuta, K., Noguchi, M., Muley, T., Hoffmann, H., Schnabel, P. A., Petersen, I., Chen, Y., Soltermann, A., Tischler, V., Choi, C., Kim, Y., Massion, P. P., Zou, Y., Jovanovic, D., Kontic, M., Wright, G. M., Russell, P. A., Solomon, B., Koch, I., Lindner, M., Muscarella, L. A., la Torre, A., Field, J. K., Jakopovic, M., Knezevic, J., CastaÃ±os-VÃ©lez, E., Roz, L., Pastorino, U., Brustugun, O., Lund-Iversen, M., Thunnissen, E., KÃ¶hler, J., Schuler, M., Botling, J., Sandelin, M., Sanchez-Cespedes, M., Salvesen, H. B., Achter, V., Lang, U., Bogus, M., Schneider, P. M., Zander, T., AnsÃ©n, S., Hallek, M., Wolf, J., Vingron, M., Yatabe, Y., Travis, W. D., NÃ¼rnberg, P., Reinhardt, C., Perner, S., Heukamp, L., BÃ¼ttner, R., Haas, S. A., Brambilla, E., Peifer, M., Sage, J., Thomas, R. K.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2015</span>; <span class=\"volume\">524 (7563)</span><span class=\"pages\">: 47-53</span>\n</span>","chicagoCitation":"George, Julie, Jing Shan Lim, Se Jin Jang, Yupeng Cun, Luka Ozretic, et al. 2015. ?Comprehensive Genomic Profiles of Small Cell Lung Cancer.? <i>Nature</i> 524 (7563): 47?53.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">We have sequenced the genomes of 110 small cell lung cancers (SCLC), one of the deadliest human cancers. In nearly all the tumours analysed we found bi-allelic inactivation of TP53 and RB1, sometimes by complex genomic rearrangements. Two tumours with wild-type RB1 had evidence of chromothripsis leading to overexpression of cyclin D1 (encoded by the CCND1 gene), revealing an alternative mechanism of Rb1 deregulation. Thus, loss of the tumour suppressors TP53 and RB1 is obligatory in SCLC. We discovered somatic genomic rearrangements of TP73 that create an oncogenic version of this gene, TP73?ex2/3. In rare cases, SCLC tumours exhibited kinase gene mutations, providing a possible therapeutic opportunity for individual patients. Finally, we observed inactivating mutations in NOTCH family genes in 25% of human SCLC. Accordingly, activation of Notch signalling in a pre-clinical SCLC mouse model strikingly reduced the number of tumours and extended the survival of the mutant mice. Furthermore, neuroendocrine gene expression was abrogated by Notch activity in SCLC cells. This first comprehensive study of somatic genome alterations in SCLC uncovers several key biological processes and identifies candidate therapeutic targets in this highly lethal form of cancer.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nature14664\">DOI 10.1038/nature14664</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/26168399\">PubMedID 26168399</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4861069\">PubMedCentralID PMC4861069</a></span>\n  </p>","text":"Abstract:\nWe have sequenced the genomes of 110 small cell lung cancers (SCLC), one of the deadliest human cancers. In nearly all the tumours analysed we found bi-allelic inactivation of TP53 and RB1, sometimes by complex genomic rearrangements. Two tumours with wild-type RB1 had evidence of chromothripsis leading to overexpression of cyclin D1 (encoded by the CCND1 gene), revealing an alternative mechanism of Rb1 deregulation. Thus, loss of the tumour suppressors TP53 and RB1 is obligatory in SCLC. We discovered somatic genomic rearrangements of TP73 that create an oncogenic version of this gene, TP73?ex2/3. In rare cases, SCLC tumours exhibited kinase gene mutations, providing a possible therapeutic opportunity for individual patients. Finally, we observed inactivating mutations in NOTCH family genes in 25% of human SCLC. Accordingly, activation of Notch signalling in a pre-clinical SCLC mouse model strikingly reduced the number of tumours and extended the survival of the mutant mice. Furthermore, neuroendocrine gene expression was abrogated by Notch activity in SCLC cells. This first comprehensive study of somatic genome alterations in SCLC uncovers several key biological processes and identifies candidate therapeutic targets in this highly lethal form of cancer.\nDOI: 10.1038/nature14664, https://dx.doi.org/10.1038/nature14664\n\nPubMedID: 26168399, https://www.ncbi.nlm.nih.gov/pubmed/26168399\nPubMedCentralID: PMC4861069, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4861069"},"doiId":"10.1038/nature14664","doiUrl":"https://dx.doi.org/10.1038/nature14664","featured":true,"firstPublished":{"html":"Aug 6, 2015 00:00:00 AM","text":"Aug 6, 2015 00:00:00 AM","value":"2015-08-06T00:00:00.000-07:00"},"mlaCitation":"George, Julie et al. ?Comprehensive Genomic Profiles of Small Cell Lung Cancer.? <i>Nature</i> 524.7563 (2015): 47?53. Print.","pubMedId":"26168399","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/26168399","publicationId":391110,"title":"Comprehensive genomic profiles of small cell lung cancer.","type":"article"},{"apaCitation":"George, J., Lim, J. S., Jang, S. J., Cun, Y., Ozretic, L., Kong, G., ? Thomas, R. K. (2015). Comprehensive genomic profiles of small cell lung cancer. <i>NATURE</i>, <i>524</i>(7563), 47?U73.","capCitation":"<span class=\"title\">\n  <span>Comprehensive genomic profiles of small cell lung cancer</span>\n  <i>NATURE</i>\n</span>\n<span class=\"authors\">George, J., Lim, J. S., Jang, S. J., Cun, Y., Ozretic, L., Kong, G., Leenders, F., Lu, X., Fernandez-Cuesta, L., Bosco, G., Mueller, C., Dahmen, I., Jahchan, N. S., Park, K., Yang, D., Karnezis, A. N., Vaka, D., Torres, A., Wang, M. S., Korbel, J. O., Menon, R., Chun, S., Kim, D., Wilkerson, M., Hayes, N., Engelmann, D., Puetzer, B., Bos, M., Michels, S., Vlasic, I., Seidel, D., Pinther, B., Schaub, P., Becker, C., Altmueller, J., Yokota, J., Kohno, T., Iwakawa, R., Tsuta, K., Noguchi, M., Muley, T., Hoffmann, H., Schnabel, P. A., Petersen, I., Chen, Y., Soltermann, A., Tischler, V., Choi, C., Kim, Y., Massion, P. P., Zou, Y., Jovanovic, D., Kontic, M., Wright, G. M., Russell, P. A., Solomon, B., Koch, I., Lindner, M., Muscarella, L. A., la Torre, A., Field, J. K., Jakopovic, M., Knezevic, J., Castanos-Velez, E., Roz, L., Pastorino, U., Brustugun, O., Lund-Iversen, M., Thunnissen, E., Koehler, J., Schuler, M., Botling, J., Sandelin, M., Sanchez-Cespedes, M., Salvesen, H. B., Achter, V., Lang, U., Bogus, M., Schneider, P. M., Zander, T., Ansen, S., Hallek, M., Wolf, J., Vingron, M., Yatabe, Y., Travis, W. D., Nuernberg, P., Reinhardt, C., Perner, S., Heukamp, L., Buettner, R., Haas, S. A., Brambilla, E., Peifer, M., Sage, J., Thomas, R. K.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2015</span>; <span class=\"volume\">524 (7563)</span><span class=\"pages\">: 47-U73</span>\n</span>","chicagoCitation":"George, Julie, Jing Shan Lim, Se Jin Jang, Yupeng Cun, Luka Ozretic, et al. 2015. ?Comprehensive Genomic Profiles of Small Cell Lung Cancer.? <i>NATURE</i> 524 (7563). NATURE PUBLISHING GROUP: 47?U73.","detail":{"html":"<p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nature14664\">DOI 10.1038/nature14664</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000359002300029\">Web of Science ID 000359002300029</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/26168399\">PubMedID 26168399</a></span>\n  </p>","text":"DOI: 10.1038/nature14664, https://dx.doi.org/10.1038/nature14664\nWeb of Science ID: 000359002300029, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000359002300029\nPubMedID: 26168399, https://www.ncbi.nlm.nih.gov/pubmed/26168399"},"doiId":"10.1038/nature14664","doiUrl":"https://dx.doi.org/10.1038/nature14664","featured":true,"firstPublished":{"html":"Aug 6, 2015 00:00:00 AM","text":"Aug 6, 2015 00:00:00 AM","value":"2015-08-06T00:00:00.000-07:00"},"mlaCitation":"George, Julie et al. ?Comprehensive Genomic Profiles of Small Cell Lung Cancer.? <i>NATURE</i> 524.7563 (2015): 47?U73. Print.","pubMedId":"26168399","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/26168399","publicationId":391969,"title":"Comprehensive genomic profiles of small cell lung cancer","type":"article","webOfScienceId":"000359002300029","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000359002300029"},{"abstract":"Mutations in the retinoblastoma tumor suppressor gene Rb are involved in many forms of human cancer. In this study, we investigated the early consequences of inactivating Rb in the context of cellular reprogramming. We found that Rb inactivation promotes the reprogramming of differentiated cells to a pluripotent state. Unexpectedly, this effect is cell cycle independent, and instead reflects direct binding of Rb to pluripotency genes, including Sox2 and Oct4, which leads to a repressed chromatin state. More broadly, this regulation of pluripotency networks and Sox2 in particular is critical for the initiation of tumors upon loss of Rb in mice. These studies therefore identify Rb as a global transcriptional repressor of pluripotency networks, providing a molecular basis for previous reports about its involvement in cell fate pliability, and implicate misregulation of pluripotency factors such as Sox2 in tumorigenesis related to loss of Rb function.","apaCitation":"Kareta, M. S., Gorges, L. L., Hafeez, S., Benayoun, B. A., Marro, S., Zmoos, A.-F., ? Wernig, M. (2015). Inhibition of pluripotency networks by the rb tumor suppressor restricts reprogramming and tumorigenesis. <i>Cell Stem Cell</i>, <i>16</i>(1), 39?50.","capCitation":"<span class=\"title\">\n  <span>Inhibition of pluripotency networks by the rb tumor suppressor restricts reprogramming and tumorigenesis.</span>\n  <i>Cell stem cell</i>\n</span>\n<span class=\"authors\">Kareta, M. S., Gorges, L. L., Hafeez, S., Benayoun, B. A., Marro, S., Zmoos, A., Cecchini, M. J., Spacek, D., Batista, L. F., O&#39;Brien, M., Ng, Y., Ang, C. E., Vaka, D., Artandi, S. E., Dick, F. A., Brunet, A., Sage, J., Wernig, M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2015</span>; <span class=\"volume\">16 (1)</span><span class=\"pages\">: 39-50</span>\n</span>","chicagoCitation":"Kareta, Michael S., Laura L. Gorges, Sana Hafeez, BÃ©rÃ©nice A. Benayoun, Samuele Marro, et al. 2015. ?Inhibition of Pluripotency Networks by the Rb Tumor Suppressor Restricts Reprogramming and Tumorigenesis.? <i>Cell Stem Cell</i> 16 (1). ELSEVIER INC: 39?50.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Mutations in the retinoblastoma tumor suppressor gene Rb are involved in many forms of human cancer. In this study, we investigated the early consequences of inactivating Rb in the context of cellular reprogramming. We found that Rb inactivation promotes the reprogramming of differentiated cells to a pluripotent state. Unexpectedly, this effect is cell cycle independent, and instead reflects direct binding of Rb to pluripotency genes, including Sox2 and Oct4, which leads to a repressed chromatin state. More broadly, this regulation of pluripotency networks and Sox2 in particular is critical for the initiation of tumors upon loss of Rb in mice. These studies therefore identify Rb as a global transcriptional repressor of pluripotency networks, providing a molecular basis for previous reports about its involvement in cell fate pliability, and implicate misregulation of pluripotency factors such as Sox2 in tumorigenesis related to loss of Rb function.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.stem.2014.10.019\">DOI 10.1016/j.stem.2014.10.019</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/25467916\">PubMedID 25467916</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4389904\">PubMedCentralID PMC4389904</a></span>\n  </p>","text":"Abstract:\nMutations in the retinoblastoma tumor suppressor gene Rb are involved in many forms of human cancer. In this study, we investigated the early consequences of inactivating Rb in the context of cellular reprogramming. We found that Rb inactivation promotes the reprogramming of differentiated cells to a pluripotent state. Unexpectedly, this effect is cell cycle independent, and instead reflects direct binding of Rb to pluripotency genes, including Sox2 and Oct4, which leads to a repressed chromatin state. More broadly, this regulation of pluripotency networks and Sox2 in particular is critical for the initiation of tumors upon loss of Rb in mice. These studies therefore identify Rb as a global transcriptional repressor of pluripotency networks, providing a molecular basis for previous reports about its involvement in cell fate pliability, and implicate misregulation of pluripotency factors such as Sox2 in tumorigenesis related to loss of Rb function.\nDOI: 10.1016/j.stem.2014.10.019, https://dx.doi.org/10.1016/j.stem.2014.10.019\n\nPubMedID: 25467916, https://www.ncbi.nlm.nih.gov/pubmed/25467916\nPubMedCentralID: PMC4389904, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4389904"},"doiId":"10.1016/j.stem.2014.10.019","doiUrl":"https://dx.doi.org/10.1016/j.stem.2014.10.019","featured":true,"firstPublished":{"html":"Jan 8, 2015 00:00:00 AM","text":"Jan 8, 2015 00:00:00 AM","value":"2015-01-08T00:00:00.000-08:00"},"mlaCitation":"Kareta, Michael S. et al. ?Inhibition of Pluripotency Networks by the Rb Tumor Suppressor Restricts Reprogramming and Tumorigenesis.? <i>Cell stem cell</i> 16.1 (2015): 39?50. Print.","pubMedId":"25467916","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/25467916","publicationId":346411,"title":"Inhibition of pluripotency networks by the rb tumor suppressor restricts reprogramming and tumorigenesis.","type":"article"},{"abstract":"Pediatric tumors are relatively infrequent but are often associated with significant lethality and lifelong morbidity. A major goal of pediatric cancer research has been to identify key drivers of tumorigenesis to eventually develop targeted therapies to enhance cure rate and minimize acute and long-term toxic effects. Here we used genomics approaches to identify biomarkers and candidate drivers for fibrolamellar hepatocellular carcinoma (FL-HCC), a very rare subtype of pediatric liver cancer for which limited therapeutic options exist. In-depth genomics analyses of one tumor followed by immunohistochemistry validation on seven other tumors showed expression of neuroendocrine markers in FL-HCC. DNA and RNA sequencing data further showed that common cancer pathways are not visibly altered in FL-HCC but identified two novel structural variants, both resulting in fusion transcripts. The first, a 400kb deletion, results in a DNAJ1-PRKCA fusion transcript, which leads to increased PKA activity in the index tumor case and other FL-HCC cases compared to normal liver. This PKA fusion protein is oncogenic in HCC cells. The second gene fusion event, a translocation between the CLPTML1 and GLIS3 genes, generates a transcript whose product also promotes cancer phenotypes in HCC cell lines. These experiments further highlight the tumorigenic role of gene fusions in the etiology of pediatric solid tumors and identify both candidate biomarkers and possible therapeutic targets for this lethal pediatric disease.","apaCitation":"Xu, L., Hazard, F. K., Zmoos, A.-F., Jahchan, N., Chaib, H., Garfin, P. M., ? Sage, J. (2015). Genomic analysis of fibrolamellar hepatocellular carcinoma. <i>Human Molecular Genetics</i>, <i>24</i>(1), 50?63.","capCitation":"<span class=\"title\">\n  <span>Genomic analysis of fibrolamellar hepatocellular carcinoma.</span>\n  <i>Human molecular genetics</i>\n</span>\n<span class=\"authors\">Xu, L., Hazard, F. K., Zmoos, A., Jahchan, N., Chaib, H., Garfin, P. M., Rangaswami, A., Snyder, M. P., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2015</span>; <span class=\"volume\">24 (1)</span><span class=\"pages\">: 50-63</span>\n</span>","chicagoCitation":"Xu, Lei, Florette K. Hazard, Anne-Flore Zmoos, Nadine Jahchan, Hassan Chaib, et al. 2015. ?Genomic Analysis of Fibrolamellar Hepatocellular Carcinoma.? <i>Human Molecular Genetics</i> 24 (1). OXFORD UNIV PRESS: 50?63.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Pediatric tumors are relatively infrequent but are often associated with significant lethality and lifelong morbidity. A major goal of pediatric cancer research has been to identify key drivers of tumorigenesis to eventually develop targeted therapies to enhance cure rate and minimize acute and long-term toxic effects. Here we used genomics approaches to identify biomarkers and candidate drivers for fibrolamellar hepatocellular carcinoma (FL-HCC), a very rare subtype of pediatric liver cancer for which limited therapeutic options exist. In-depth genomics analyses of one tumor followed by immunohistochemistry validation on seven other tumors showed expression of neuroendocrine markers in FL-HCC. DNA and RNA sequencing data further showed that common cancer pathways are not visibly altered in FL-HCC but identified two novel structural variants, both resulting in fusion transcripts. The first, a 400kb deletion, results in a DNAJ1-PRKCA fusion transcript, which leads to increased PKA activity in the index tumor case and other FL-HCC cases compared to normal liver. This PKA fusion protein is oncogenic in HCC cells. The second gene fusion event, a translocation between the CLPTML1 and GLIS3 genes, generates a transcript whose product also promotes cancer phenotypes in HCC cell lines. These experiments further highlight the tumorigenic role of gene fusions in the etiology of pediatric solid tumors and identify both candidate biomarkers and possible therapeutic targets for this lethal pediatric disease.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1093/hmg/ddu418\">DOI 10.1093/hmg/ddu418</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/25122662\">PubMedID 25122662</a></span>\n  </p>","text":"Abstract:\nPediatric tumors are relatively infrequent but are often associated with significant lethality and lifelong morbidity. A major goal of pediatric cancer research has been to identify key drivers of tumorigenesis to eventually develop targeted therapies to enhance cure rate and minimize acute and long-term toxic effects. Here we used genomics approaches to identify biomarkers and candidate drivers for fibrolamellar hepatocellular carcinoma (FL-HCC), a very rare subtype of pediatric liver cancer for which limited therapeutic options exist. In-depth genomics analyses of one tumor followed by immunohistochemistry validation on seven other tumors showed expression of neuroendocrine markers in FL-HCC. DNA and RNA sequencing data further showed that common cancer pathways are not visibly altered in FL-HCC but identified two novel structural variants, both resulting in fusion transcripts. The first, a 400kb deletion, results in a DNAJ1-PRKCA fusion transcript, which leads to increased PKA activity in the index tumor case and other FL-HCC cases compared to normal liver. This PKA fusion protein is oncogenic in HCC cells. The second gene fusion event, a translocation between the CLPTML1 and GLIS3 genes, generates a transcript whose product also promotes cancer phenotypes in HCC cell lines. These experiments further highlight the tumorigenic role of gene fusions in the etiology of pediatric solid tumors and identify both candidate biomarkers and possible therapeutic targets for this lethal pediatric disease.\nDOI: 10.1093/hmg/ddu418, https://dx.doi.org/10.1093/hmg/ddu418\n\nPubMedID: 25122662, https://www.ncbi.nlm.nih.gov/pubmed/25122662"},"doiId":"10.1093/hmg/ddu418","doiUrl":"https://dx.doi.org/10.1093/hmg/ddu418","featured":false,"firstPublished":{"html":"Jan 1, 2015 00:00:00 AM","text":"Jan 1, 2015 00:00:00 AM","value":"2015-01-01T00:00:00.000-08:00"},"mlaCitation":"Xu, Lei et al. ?Genomic Analysis of Fibrolamellar Hepatocellular Carcinoma.? <i>Human molecular genetics</i> 24.1 (2015): 50?63. Print.","pubMedId":"25122662","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/25122662","publicationId":322199,"title":"Genomic analysis of fibrolamellar hepatocellular carcinoma.","type":"article"},{"abstract":"In mammals, a cell's decision to divide is thought toÂ be under the control of the Rb/E2F pathway. We previously found that inactivation of the Rb family of cell cycle inhibitors (Rb, p107, and p130) in quiescent liver progenitors leads to uncontrolled division and cancer initiation. Here, we show that, in contrast, deletion of the entire Rb gene family in mature hepatocytes is not sufficient for their long-term proliferation. The cell cycle block in Rb family mutant hepatocytes is independent of the Arf/p53/p21 checkpoint but can be abrogated upon decreasing liver size. At the molecular level, we identify YAP, a transcriptional regulator involved in organ size control, as a factor required for the sustained expression of cell cycle genes in hepatocytes. These experiments identify a higher level of regulation of the cell cycle inÂ vivo in which signals regulating organ size are dominant regulators of the core cell cycle machinery.","apaCitation":"Ehmer, U., Zmoos, A.-F., Auerbach, R. K., Vaka, D., Butte, A. J., Kay, M. A., &amp; Sage, J. (2014). Organ Size Control Is Dominant over Rb Family Inactivation to Restrict Proliferation InÂ Vivo. <i>Cell Reports</i>, <i>8</i>(2), 371?381.","capCitation":"<span class=\"title\">\n  <span>Organ Size Control Is Dominant over Rb Family Inactivation to Restrict Proliferation InÂ Vivo.</span>\n  <i>Cell reports</i>\n</span>\n<span class=\"authors\">Ehmer, U., Zmoos, A., Auerbach, R. K., Vaka, D., Butte, A. J., Kay, M. A., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2014</span>; <span class=\"volume\">8 (2)</span><span class=\"pages\">: 371-381</span>\n</span>","chicagoCitation":"Ehmer, Ursula, Anne-Flore Zmoos, Raymond K. Auerbach, Dedeepya Vaka, Atul J. Butte, et al. 2014. ?Organ Size Control Is Dominant over Rb Family Inactivation to Restrict Proliferation InÂ Vivo.? <i>Cell Reports</i> 8 (2). ELSEVIER INC: 371?81.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">In mammals, a cell's decision to divide is thought toÂ be under the control of the Rb/E2F pathway. We previously found that inactivation of the Rb family of cell cycle inhibitors (Rb, p107, and p130) in quiescent liver progenitors leads to uncontrolled division and cancer initiation. Here, we show that, in contrast, deletion of the entire Rb gene family in mature hepatocytes is not sufficient for their long-term proliferation. The cell cycle block in Rb family mutant hepatocytes is independent of the Arf/p53/p21 checkpoint but can be abrogated upon decreasing liver size. At the molecular level, we identify YAP, a transcriptional regulator involved in organ size control, as a factor required for the sustained expression of cell cycle genes in hepatocytes. These experiments identify a higher level of regulation of the cell cycle inÂ vivo in which signals regulating organ size are dominant regulators of the core cell cycle machinery.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.celrep.2014.06.025\">DOI 10.1016/j.celrep.2014.06.025</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/25017070\">PubMedID 25017070</a></span>\n  </p>","text":"Abstract:\nIn mammals, a cell's decision to divide is thought toÂ be under the control of the Rb/E2F pathway. We previously found that inactivation of the Rb family of cell cycle inhibitors (Rb, p107, and p130) in quiescent liver progenitors leads to uncontrolled division and cancer initiation. Here, we show that, in contrast, deletion of the entire Rb gene family in mature hepatocytes is not sufficient for their long-term proliferation. The cell cycle block in Rb family mutant hepatocytes is independent of the Arf/p53/p21 checkpoint but can be abrogated upon decreasing liver size. At the molecular level, we identify YAP, a transcriptional regulator involved in organ size control, as a factor required for the sustained expression of cell cycle genes in hepatocytes. These experiments identify a higher level of regulation of the cell cycle inÂ vivo in which signals regulating organ size are dominant regulators of the core cell cycle machinery.\nDOI: 10.1016/j.celrep.2014.06.025, https://dx.doi.org/10.1016/j.celrep.2014.06.025\n\nPubMedID: 25017070, https://www.ncbi.nlm.nih.gov/pubmed/25017070"},"doiId":"10.1016/j.celrep.2014.06.025","doiUrl":"https://dx.doi.org/10.1016/j.celrep.2014.06.025","featured":true,"firstPublished":{"html":"Jul 24, 2014 00:00:00 AM","text":"Jul 24, 2014 00:00:00 AM","value":"2014-07-24T00:00:00.000-07:00"},"mlaCitation":"Ehmer, Ursula et al. ?Organ Size Control Is Dominant over Rb Family Inactivation to Restrict Proliferation InÂ Vivo.? <i>Cell reports</i> 8.2 (2014): 371?381. Print.","pubMedId":"25017070","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/25017070","publicationId":322996,"title":"Organ Size Control Is Dominant over Rb Family Inactivation to Restrict Proliferation InÂ Vivo.","type":"article"},{"abstract":"Deregulation of lysine methylation signalling has emerged as a common aetiological factor in cancer pathogenesis, with inhibitors of several histone lysine methyltransferases (KMTs) being developed as chemotherapeutics. The largely cytoplasmic KMT SMYD3 (SET and MYND domain containing protein 3) is overexpressed in numerous human tumours. However, the molecular mechanism by which SMYD3 regulates cancer pathways and its relationship to tumorigenesis in vivo are largely unknown. Here we show that methylation of MAP3K2 by SMYD3 increases MAP kinase signalling and promotes the formation of Ras-driven carcinomas. Using mouse models for pancreatic ductal adenocarcinoma and lung adenocarcinoma, we found that abrogating SMYD3 catalytic activity inhibits tumour development in response to oncogenic Ras. We used protein array technology to identify the MAP3K2 kinase as a target of SMYD3. In cancer cell lines, SMYD3-mediated methylation of MAP3K2 at lysine 260 potentiates activation of the Ras/Raf/MEK/ERK signalling module and SMYD3 depletion synergizes with a MEK inhibitor to block Ras-driven tumorigenesis. Finally, the PP2A phosphatase complex, a key negative regulator of the MAP kinase pathway, binds to MAP3K2 and this interaction is blocked by methylation. Together, our results elucidate a new role for lysine methylation in integrating cytoplasmic kinase-signalling cascades and establish a pivotal role for SMYD3 in the regulation of oncogenic Ras signalling.","apaCitation":"Mazur, P. K., Reynoird, N., Khatri, P., Jansen, P. W. T. C., Wilkinson, A. W., Liu, S., ? Gozani, O. (2014). SMYD3 links lysine methylation of MAP3K2 to Ras-driven cancer. <i>Nature</i>, <i>510</i>(7504), 283?287.","capCitation":"<span class=\"title\">\n  <span>SMYD3 links lysine methylation of MAP3K2 to Ras-driven cancer.</span>\n  <i>Nature</i>\n</span>\n<span class=\"authors\">Mazur, P. K., Reynoird, N., Khatri, P., Jansen, P. W., Wilkinson, A. W., Liu, S., Barbash, O., Van Aller, G. S., Huddleston, M., Dhanak, D., Tummino, P. J., Kruger, R. G., Garcia, B. A., Butte, A. J., Vermeulen, M., Sage, J., Gozani, O.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2014</span>; <span class=\"volume\">510 (7504)</span><span class=\"pages\">: 283-287</span>\n</span>","chicagoCitation":"Mazur, Pawel K., Nicolas Reynoird, Purvesh Khatri, Pascal WTC Jansen, Alex W. Wilkinson, et al. 2014. ?SMYD3 Links Lysine Methylation of MAP3K2 to Ras-Driven Cancer.? <i>Nature</i> 510 (7504): 283?87.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Deregulation of lysine methylation signalling has emerged as a common aetiological factor in cancer pathogenesis, with inhibitors of several histone lysine methyltransferases (KMTs) being developed as chemotherapeutics. The largely cytoplasmic KMT SMYD3 (SET and MYND domain containing protein 3) is overexpressed in numerous human tumours. However, the molecular mechanism by which SMYD3 regulates cancer pathways and its relationship to tumorigenesis in vivo are largely unknown. Here we show that methylation of MAP3K2 by SMYD3 increases MAP kinase signalling and promotes the formation of Ras-driven carcinomas. Using mouse models for pancreatic ductal adenocarcinoma and lung adenocarcinoma, we found that abrogating SMYD3 catalytic activity inhibits tumour development in response to oncogenic Ras. We used protein array technology to identify the MAP3K2 kinase as a target of SMYD3. In cancer cell lines, SMYD3-mediated methylation of MAP3K2 at lysine 260 potentiates activation of the Ras/Raf/MEK/ERK signalling module and SMYD3 depletion synergizes with a MEK inhibitor to block Ras-driven tumorigenesis. Finally, the PP2A phosphatase complex, a key negative regulator of the MAP kinase pathway, binds to MAP3K2 and this interaction is blocked by methylation. Together, our results elucidate a new role for lysine methylation in integrating cytoplasmic kinase-signalling cascades and establish a pivotal role for SMYD3 in the regulation of oncogenic Ras signalling.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nature13320\">DOI 10.1038/nature13320</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/24847881\">PubMedID 24847881</a></span>\n  </p>","text":"Abstract:\nDeregulation of lysine methylation signalling has emerged as a common aetiological factor in cancer pathogenesis, with inhibitors of several histone lysine methyltransferases (KMTs) being developed as chemotherapeutics. The largely cytoplasmic KMT SMYD3 (SET and MYND domain containing protein 3) is overexpressed in numerous human tumours. However, the molecular mechanism by which SMYD3 regulates cancer pathways and its relationship to tumorigenesis in vivo are largely unknown. Here we show that methylation of MAP3K2 by SMYD3 increases MAP kinase signalling and promotes the formation of Ras-driven carcinomas. Using mouse models for pancreatic ductal adenocarcinoma and lung adenocarcinoma, we found that abrogating SMYD3 catalytic activity inhibits tumour development in response to oncogenic Ras. We used protein array technology to identify the MAP3K2 kinase as a target of SMYD3. In cancer cell lines, SMYD3-mediated methylation of MAP3K2 at lysine 260 potentiates activation of the Ras/Raf/MEK/ERK signalling module and SMYD3 depletion synergizes with a MEK inhibitor to block Ras-driven tumorigenesis. Finally, the PP2A phosphatase complex, a key negative regulator of the MAP kinase pathway, binds to MAP3K2 and this interaction is blocked by methylation. Together, our results elucidate a new role for lysine methylation in integrating cytoplasmic kinase-signalling cascades and establish a pivotal role for SMYD3 in the regulation of oncogenic Ras signalling.\nDOI: 10.1038/nature13320, https://dx.doi.org/10.1038/nature13320\n\nPubMedID: 24847881, https://www.ncbi.nlm.nih.gov/pubmed/24847881"},"doiId":"10.1038/nature13320","doiUrl":"https://dx.doi.org/10.1038/nature13320","featured":true,"firstPublished":{"html":"Jun 12, 2014 00:00:00 AM","text":"Jun 12, 2014 00:00:00 AM","value":"2014-06-12T00:00:00.000-07:00"},"mlaCitation":"Mazur, Pawel K. et al. ?SMYD3 Links Lysine Methylation of MAP3K2 to Ras-Driven Cancer.? <i>Nature</i> 510.7504 (2014): 283?287. Print.","pubMedId":"24847881","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/24847881","publicationId":317914,"title":"SMYD3 links lysine methylation of MAP3K2 to Ras-driven cancer.","type":"article"},{"apaCitation":"Ntziachristos, P., Lim, J. S., Sage, J., &amp; Aifantis, I. (2014). From Fly Wings to Targeted Cancer Therapies: A Centennial for Notch Signaling. <i>CANCER CELL</i>, <i>25</i>(3), 318?334.","capCitation":"<span class=\"title\">\n  <span>From Fly Wings to Targeted Cancer Therapies: A Centennial for Notch Signaling</span>\n  <i>CANCER CELL</i>\n</span>\n<span class=\"authors\">Ntziachristos, P., Lim, J. S., Sage, J., Aifantis, I.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2014</span>; <span class=\"volume\">25 (3)</span><span class=\"pages\">: 318-334</span>\n</span>","chicagoCitation":"Ntziachristos, Panagiotis, Jing Shan Lim, Julien Sage, and Iannis Aifantis. 2014. ?From Fly Wings to Targeted Cancer Therapies: A Centennial for Notch Signaling.? <i>CANCER CELL</i> 25 (3). CELL PRESS: 318?34.","detail":{"html":"<p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.ccr.2014.02.018\">DOI 10.1016/j.ccr.2014.02.018</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000333233400010\">Web of Science ID 000333233400010</a></span>\n  </p>","text":"DOI: 10.1016/j.ccr.2014.02.018, https://dx.doi.org/10.1016/j.ccr.2014.02.018\nWeb of Science ID: 000333233400010, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000333233400010"},"doiId":"10.1016/j.ccr.2014.02.018","doiUrl":"https://dx.doi.org/10.1016/j.ccr.2014.02.018","featured":false,"firstPublished":{"html":"Mar 17, 2014 00:00:00 AM","text":"Mar 17, 2014 00:00:00 AM","value":"2014-03-17T00:00:00.000-07:00"},"mlaCitation":"Ntziachristos, Panagiotis et al. ?From Fly Wings to Targeted Cancer Therapies: A Centennial for Notch Signaling.? <i>CANCER CELL</i> 25.3 (2014): 318?334. Print.","publicationId":308307,"title":"From Fly Wings to Targeted Cancer Therapies: A Centennial for Notch Signaling","type":"article","webOfScienceId":"000333233400010","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000333233400010"},{"abstract":"Small cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer with high mortality. We used a systematic drug repositioning bioinformatics approach querying a large compendium of gene expression profiles to identify candidate U.S. Food and Drug Administration (FDA)-approved drugs to treat SCLC. We found that tricyclic antidepressants and related molecules potently induce apoptosis in both chemonaÃ¯ve and chemoresistant SCLC cells in culture, in mouse and human SCLC tumors transplanted into immunocompromised mice, and in endogenous tumors from a mouse model for human SCLC. The candidate drugs activate stress pathways and induce cell death in SCLC cells, at least in part by disrupting autocrine survival signals involving neurotransmitters and their G protein-coupled receptors. The candidate drugs inhibit the growth of other neuroendocrine tumors, including pancreatic neuroendocrine tumors and Merkel cell carcinoma. These experiments identify novel targeted strategies that can be rapidly evaluated in patients with neuroendocrine tumors through the repurposing of approved drugs.Our work shows the power of bioinformatics-based drug approaches to rapidly repurpose FDA-approved drugs and identifies a novel class of molecules to treat patients with SCLC, a cancer for which no effective novel systemic treatments have been identified in several decades. In addition, our experiments highlight the importance of novel autocrine mechanisms in promoting the growth of neuroendocrine tumor cells.","apaCitation":"Jahchan, N. S., Dudley, J. T., Mazur, P. K., Flores, N., Yang, D., Palmerton, A., ? Sage, J. (2013). A Drug Repositioning Approach Identifies Tricyclic Antidepressants as Inhibitors of Small Cell Lung Cancer and Other Neuroendocrine Tumors. <i>CANCER DISCOVERY</i>, <i>3</i>(12), 1364?1377.","capCitation":"<span class=\"title\">\n  <span>A Drug Repositioning Approach Identifies Tricyclic Antidepressants as Inhibitors of Small Cell Lung Cancer and Other Neuroendocrine Tumors</span>\n  <i>CANCER DISCOVERY</i>\n</span>\n<span class=\"authors\">Jahchan, N. S., Dudley, J. T., Mazur, P. K., Flores, N., Yang, D., Palmerton, A., Zmoos, A., Vaka, D., Tran, K. Q., Zhou, M., Krasinska, K., Riess, J. W., Neal, J. W., Khatri, P., Park, K. S., Butte, A. J., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2013</span>; <span class=\"volume\">3 (12)</span><span class=\"pages\">: 1364-1377</span>\n</span>","chicagoCitation":"Jahchan, Nadine S., Joel T. Dudley, Pawel K. Mazur, Natasha Flores, Dian Yang, et al. 2013. ?A Drug Repositioning Approach Identifies Tricyclic Antidepressants as Inhibitors of Small Cell Lung Cancer and Other Neuroendocrine Tumors.? <i>CANCER DISCOVERY</i> 3 (12). AMER ASSOC CANCER RESEARCH: 1364?77.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer with high mortality. We used a systematic drug repositioning bioinformatics approach querying a large compendium of gene expression profiles to identify candidate U.S. Food and Drug Administration (FDA)-approved drugs to treat SCLC. We found that tricyclic antidepressants and related molecules potently induce apoptosis in both chemonaÃ¯ve and chemoresistant SCLC cells in culture, in mouse and human SCLC tumors transplanted into immunocompromised mice, and in endogenous tumors from a mouse model for human SCLC. The candidate drugs activate stress pathways and induce cell death in SCLC cells, at least in part by disrupting autocrine survival signals involving neurotransmitters and their G protein-coupled receptors. The candidate drugs inhibit the growth of other neuroendocrine tumors, including pancreatic neuroendocrine tumors and Merkel cell carcinoma. These experiments identify novel targeted strategies that can be rapidly evaluated in patients with neuroendocrine tumors through the repurposing of approved drugs.Our work shows the power of bioinformatics-based drug approaches to rapidly repurpose FDA-approved drugs and identifies a novel class of molecules to treat patients with SCLC, a cancer for which no effective novel systemic treatments have been identified in several decades. In addition, our experiments highlight the importance of novel autocrine mechanisms in promoting the growth of neuroendocrine tumor cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1158/2159-8290.CD-13-0183\">DOI 10.1158/2159-8290.CD-13-0183</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000328257500023\">Web of Science ID 000328257500023</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/24078773\">PubMedID 24078773</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3864571\">PubMedCentralID PMC3864571</a></span>\n  </p>","text":"Abstract:\nSmall cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer with high mortality. We used a systematic drug repositioning bioinformatics approach querying a large compendium of gene expression profiles to identify candidate U.S. Food and Drug Administration (FDA)-approved drugs to treat SCLC. We found that tricyclic antidepressants and related molecules potently induce apoptosis in both chemonaÃ¯ve and chemoresistant SCLC cells in culture, in mouse and human SCLC tumors transplanted into immunocompromised mice, and in endogenous tumors from a mouse model for human SCLC. The candidate drugs activate stress pathways and induce cell death in SCLC cells, at least in part by disrupting autocrine survival signals involving neurotransmitters and their G protein-coupled receptors. The candidate drugs inhibit the growth of other neuroendocrine tumors, including pancreatic neuroendocrine tumors and Merkel cell carcinoma. These experiments identify novel targeted strategies that can be rapidly evaluated in patients with neuroendocrine tumors through the repurposing of approved drugs.Our work shows the power of bioinformatics-based drug approaches to rapidly repurpose FDA-approved drugs and identifies a novel class of molecules to treat patients with SCLC, a cancer for which no effective novel systemic treatments have been identified in several decades. In addition, our experiments highlight the importance of novel autocrine mechanisms in promoting the growth of neuroendocrine tumor cells.\nDOI: 10.1158/2159-8290.CD-13-0183, https://dx.doi.org/10.1158/2159-8290.CD-13-0183\nWeb of Science ID: 000328257500023, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000328257500023\nPubMedID: 24078773, https://www.ncbi.nlm.nih.gov/pubmed/24078773\nPubMedCentralID: PMC3864571, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3864571"},"doiId":"10.1158/2159-8290.CD-13-0183","doiUrl":"https://dx.doi.org/10.1158/2159-8290.CD-13-0183","featured":true,"firstPublished":{"html":"Dec 1, 2013 00:00:00 AM","text":"Dec 1, 2013 00:00:00 AM","value":"2013-12-01T00:00:00.000-08:00"},"mlaCitation":"Jahchan, Nadine S. et al. ?A Drug Repositioning Approach Identifies Tricyclic Antidepressants as Inhibitors of Small Cell Lung Cancer and Other Neuroendocrine Tumors.? <i>CANCER DISCOVERY</i> 3.12 (2013): 1364?1377. Print.","pubMedId":"24078773","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/24078773","publicationId":285360,"title":"A Drug Repositioning Approach Identifies Tricyclic Antidepressants as Inhibitors of Small Cell Lung Cancer and Other Neuroendocrine Tumors","type":"article","webOfScienceId":"000328257500023","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000328257500023"},{"abstract":"Thymic involution during aging is a major cause of decreased production of T cells and reduced immunity. Here we show that inactivation of Rb family genes in young mice prevents thymic involution and results in an enlarged thymus competent for increased production of naive T cells. This phenotype originates from the expansion of functional thymic epithelial cells (TECs). In RB family mutant TECs, increased activity of E2F transcription factors drives increased expression of Foxn1, a central regulator of the thymic epithelium. Increased Foxn1 expression is required for the thymic expansion observed in Rb family mutant mice. Thus, the RB family promotes thymic involution and controls T cell production via a bone marrow-independent mechanism, identifying a novel pathway to target to increase thymic function in patients.","apaCitation":"Garfin, P. M., Min, D., Bryson, J. L., Serwold, T., Edris, B., Blackburn, C. C., ? Viatour, P. (2013). Inactivation of the RB family prevents thymus involution and promotes thymic function by direct control of Foxn1 expression. <i>Journal of Experimental Medicine</i>, <i>210</i>(6), 1087?1097.","capCitation":"<span class=\"title\">\n  <span>Inactivation of the RB family prevents thymus involution and promotes thymic function by direct control of Foxn1 expression.</span>\n  <i>journal of experimental medicine</i>\n</span>\n<span class=\"authors\">Garfin, P. M., Min, D., Bryson, J. L., Serwold, T., Edris, B., Blackburn, C. C., Richie, E. R., Weinberg, K. I., Manley, N. R., Sage, J., Viatour, P.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2013</span>; <span class=\"volume\">210 (6)</span><span class=\"pages\">: 1087-1097</span>\n</span>","chicagoCitation":"Garfin, Phillip M., Dullei Min, Jerrod L. Bryson, Thomas Serwold, Badreddin Edris, et al. 2013. ?Inactivation of the RB Family Prevents Thymus Involution and Promotes Thymic Function by Direct Control of Foxn1 Expression.? <i>Journal of Experimental Medicine</i> 210 (6): 1087?97.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Thymic involution during aging is a major cause of decreased production of T cells and reduced immunity. Here we show that inactivation of Rb family genes in young mice prevents thymic involution and results in an enlarged thymus competent for increased production of naive T cells. This phenotype originates from the expansion of functional thymic epithelial cells (TECs). In RB family mutant TECs, increased activity of E2F transcription factors drives increased expression of Foxn1, a central regulator of the thymic epithelium. Increased Foxn1 expression is required for the thymic expansion observed in Rb family mutant mice. Thus, the RB family promotes thymic involution and controls T cell production via a bone marrow-independent mechanism, identifying a novel pathway to target to increase thymic function in patients.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1084/jem.20121716\">DOI 10.1084/jem.20121716</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/23669396\">PubMedID 23669396</a></span>\n  </p>","text":"Abstract:\nThymic involution during aging is a major cause of decreased production of T cells and reduced immunity. Here we show that inactivation of Rb family genes in young mice prevents thymic involution and results in an enlarged thymus competent for increased production of naive T cells. This phenotype originates from the expansion of functional thymic epithelial cells (TECs). In RB family mutant TECs, increased activity of E2F transcription factors drives increased expression of Foxn1, a central regulator of the thymic epithelium. Increased Foxn1 expression is required for the thymic expansion observed in Rb family mutant mice. Thus, the RB family promotes thymic involution and controls T cell production via a bone marrow-independent mechanism, identifying a novel pathway to target to increase thymic function in patients.\nDOI: 10.1084/jem.20121716, https://dx.doi.org/10.1084/jem.20121716\n\nPubMedID: 23669396, https://www.ncbi.nlm.nih.gov/pubmed/23669396"},"doiId":"10.1084/jem.20121716","doiUrl":"https://dx.doi.org/10.1084/jem.20121716","featured":false,"firstPublished":{"html":"Jun 3, 2013 00:00:00 AM","text":"Jun 3, 2013 00:00:00 AM","value":"2013-06-03T00:00:00.000-07:00"},"mlaCitation":"Garfin, Phillip M. et al. ?Inactivation of the RB Family Prevents Thymus Involution and Promotes Thymic Function by Direct Control of Foxn1 Expression.? <i>journal of experimental medicine</i> 210.6 (2013): 1087?1097. Print.","pubMedId":"23669396","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/23669396","publicationId":132009,"title":"Inactivation of the RB family prevents thymus involution and promotes thymic function by direct control of Foxn1 expression.","type":"article"},{"abstract":"Upregulation of the ERK1 and ERK2 (ERK1/2) MAP kinase (MAPK) cascade occurs in >30% of cancers, often through mutational activation of receptor tyrosine kinases or other upstream genes, including KRAS and BRAF. Efforts to target endogenous MAPKs are challenged by the fact that these kinases are required for viability in mammals. Additionally, the effectiveness of new inhibitors of mutant BRAF has been diminished by acquired tumor resistance through selection for BRAF-independent mechanisms of ERK1/2 induction. Furthermore, recently identified ERK1/2-inducing mutations in MEK1 and MEK2 (MEK1/2) MAPK genes in melanoma confer resistance to emerging therapeutic MEK inhibitors, underscoring the challenges facing direct kinase inhibition in cancer. MAPK scaffolds, such as IQ motif-containing GTPase activating protein 1 (IQGAP1), assemble pathway kinases to affect signal transmission, and disrupting scaffold function therefore offers an orthogonal approach to MAPK cascade inhibition. Consistent with this, we found a requirement for IQGAP1 in RAS-driven tumorigenesis in mouse and human tissue. In addition, the ERK1/2-binding IQGAP1 WW domain peptide disrupted IQGAP1-ERK1/2 interactions, inhibited RAS- and RAF-driven tumorigenesis, bypassed acquired resistance to the BRAF inhibitor vemurafenib (PLX-4032) and acted as a systemically deliverable therapeutic to significantly increase the lifespan of tumor-bearing mice. Scaffold-kinase interaction blockade acts by a mechanism distinct from direct kinase inhibition and may be a strategy to target overactive oncogenic kinase cascades in cancer.","apaCitation":"Jameson, K. L., Mazur, P. K., Zehnder, A. M., Zhang, J., Zarnegar, B., Sage, J., &amp; Khavari, P. A. (2013). IQGAP1 scaffold-kinase interaction blockade selectively targets RAS-MAP kinase-driven tumors. <i>Nature Medicine</i>, <i>19</i>(5), 626?630.","capCitation":"<span class=\"title\">\n  <span>IQGAP1 scaffold-kinase interaction blockade selectively targets RAS-MAP kinase-driven tumors.</span>\n  <i>Nature medicine</i>\n</span>\n<span class=\"authors\">Jameson, K. L., Mazur, P. K., Zehnder, A. M., Zhang, J., Zarnegar, B., Sage, J., Khavari, P. A.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2013</span>; <span class=\"volume\">19 (5)</span><span class=\"pages\">: 626-630</span>\n</span>","chicagoCitation":"Jameson, Katherine L., Pawel K. Mazur, Ashley M. Zehnder, Jiajing Zhang, Brian Zarnegar, et al. 2013. ?IQGAP1 Scaffold-Kinase Interaction Blockade Selectively Targets RAS-MAP Kinase-Driven Tumors.? <i>Nature Medicine</i> 19 (5): 626?30.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Upregulation of the ERK1 and ERK2 (ERK1/2) MAP kinase (MAPK) cascade occurs in >30% of cancers, often through mutational activation of receptor tyrosine kinases or other upstream genes, including KRAS and BRAF. Efforts to target endogenous MAPKs are challenged by the fact that these kinases are required for viability in mammals. Additionally, the effectiveness of new inhibitors of mutant BRAF has been diminished by acquired tumor resistance through selection for BRAF-independent mechanisms of ERK1/2 induction. Furthermore, recently identified ERK1/2-inducing mutations in MEK1 and MEK2 (MEK1/2) MAPK genes in melanoma confer resistance to emerging therapeutic MEK inhibitors, underscoring the challenges facing direct kinase inhibition in cancer. MAPK scaffolds, such as IQ motif-containing GTPase activating protein 1 (IQGAP1), assemble pathway kinases to affect signal transmission, and disrupting scaffold function therefore offers an orthogonal approach to MAPK cascade inhibition. Consistent with this, we found a requirement for IQGAP1 in RAS-driven tumorigenesis in mouse and human tissue. In addition, the ERK1/2-binding IQGAP1 WW domain peptide disrupted IQGAP1-ERK1/2 interactions, inhibited RAS- and RAF-driven tumorigenesis, bypassed acquired resistance to the BRAF inhibitor vemurafenib (PLX-4032) and acted as a systemically deliverable therapeutic to significantly increase the lifespan of tumor-bearing mice. Scaffold-kinase interaction blockade acts by a mechanism distinct from direct kinase inhibition and may be a strategy to target overactive oncogenic kinase cascades in cancer.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nm.3165\">DOI 10.1038/nm.3165</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/23603816\">PubMedID 23603816</a></span>\n  </p>","text":"Abstract:\nUpregulation of the ERK1 and ERK2 (ERK1/2) MAP kinase (MAPK) cascade occurs in >30% of cancers, often through mutational activation of receptor tyrosine kinases or other upstream genes, including KRAS and BRAF. Efforts to target endogenous MAPKs are challenged by the fact that these kinases are required for viability in mammals. Additionally, the effectiveness of new inhibitors of mutant BRAF has been diminished by acquired tumor resistance through selection for BRAF-independent mechanisms of ERK1/2 induction. Furthermore, recently identified ERK1/2-inducing mutations in MEK1 and MEK2 (MEK1/2) MAPK genes in melanoma confer resistance to emerging therapeutic MEK inhibitors, underscoring the challenges facing direct kinase inhibition in cancer. MAPK scaffolds, such as IQ motif-containing GTPase activating protein 1 (IQGAP1), assemble pathway kinases to affect signal transmission, and disrupting scaffold function therefore offers an orthogonal approach to MAPK cascade inhibition. Consistent with this, we found a requirement for IQGAP1 in RAS-driven tumorigenesis in mouse and human tissue. In addition, the ERK1/2-binding IQGAP1 WW domain peptide disrupted IQGAP1-ERK1/2 interactions, inhibited RAS- and RAF-driven tumorigenesis, bypassed acquired resistance to the BRAF inhibitor vemurafenib (PLX-4032) and acted as a systemically deliverable therapeutic to significantly increase the lifespan of tumor-bearing mice. Scaffold-kinase interaction blockade acts by a mechanism distinct from direct kinase inhibition and may be a strategy to target overactive oncogenic kinase cascades in cancer.\nDOI: 10.1038/nm.3165, https://dx.doi.org/10.1038/nm.3165\n\nPubMedID: 23603816, https://www.ncbi.nlm.nih.gov/pubmed/23603816"},"doiId":"10.1038/nm.3165","doiUrl":"https://dx.doi.org/10.1038/nm.3165","featured":false,"firstPublished":{"html":"May 1, 2013 00:00:00 AM","text":"May 1, 2013 00:00:00 AM","value":"2013-05-01T00:00:00.000-07:00"},"mlaCitation":"Jameson, Katherine L. et al. ?IQGAP1 Scaffold-Kinase Interaction Blockade Selectively Targets RAS-MAP Kinase-Driven Tumors.? <i>Nature medicine</i> 19.5 (2013): 626?630. Print.","pubMedId":"23603816","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/23603816","publicationId":131405,"title":"IQGAP1 scaffold-kinase interaction blockade selectively targets RAS-MAP kinase-driven tumors.","type":"article"},{"abstract":"The retinoblastoma tumor suppressor RB is well known for its capacity to restrict cell cycle progression at the G1/S transition of the cell cycle by controlling the transcription of cell cycle genes. In this issue of Genes & Development, Hilgendorf and colleagues (pp. 1003-1015) have identified a novel tumor suppressor function for RB independent of its role as a transcriptional regulator, in which RB directly activates the apoptosis regulator Bax at the mitochondria to promote cell death.","apaCitation":"Attardi, L. D., &amp; Sage, J. (2013). RB goes mitochondrial. <i>Genes &amp; Development</i>, <i>27</i>(9), 975?979.","capCitation":"<span class=\"title\">\n  <span>RB goes mitochondrial.</span>\n  <i>Genes &amp; development</i>\n</span>\n<span class=\"authors\">Attardi, L. D., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2013</span>; <span class=\"volume\">27 (9)</span><span class=\"pages\">: 975-979</span>\n</span>","chicagoCitation":"Attardi, Laura D., and Julien Sage. 2013. ?RB Goes Mitochondrial.? <i>Genes &amp; Development</i> 27 (9): 975?79.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma tumor suppressor RB is well known for its capacity to restrict cell cycle progression at the G1/S transition of the cell cycle by controlling the transcription of cell cycle genes. In this issue of Genes & Development, Hilgendorf and colleagues (pp. 1003-1015) have identified a novel tumor suppressor function for RB independent of its role as a transcriptional regulator, in which RB directly activates the apoptosis regulator Bax at the mitochondria to promote cell death.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.219451.113\">DOI 10.1101/gad.219451.113</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/23651852\">PubMedID 23651852</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma tumor suppressor RB is well known for its capacity to restrict cell cycle progression at the G1/S transition of the cell cycle by controlling the transcription of cell cycle genes. In this issue of Genes & Development, Hilgendorf and colleagues (pp. 1003-1015) have identified a novel tumor suppressor function for RB independent of its role as a transcriptional regulator, in which RB directly activates the apoptosis regulator Bax at the mitochondria to promote cell death.\nDOI: 10.1101/gad.219451.113, https://dx.doi.org/10.1101/gad.219451.113\n\nPubMedID: 23651852, https://www.ncbi.nlm.nih.gov/pubmed/23651852"},"doiId":"10.1101/gad.219451.113","doiUrl":"https://dx.doi.org/10.1101/gad.219451.113","featured":false,"firstPublished":{"html":"May 1, 2013 00:00:00 AM","text":"May 1, 2013 00:00:00 AM","value":"2013-05-01T00:00:00.000-07:00"},"mlaCitation":"Attardi, Laura D., and Julien Sage. ?RB Goes Mitochondrial.? <i>Genes &amp; development</i> 27.9 (2013): 975?979. Print.","pubMedId":"23651852","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/23651852","publicationId":131847,"title":"RB goes mitochondrial.","type":"article"},{"abstract":"The mechanisms ensuring the long-term self-renewal of human embryonic stem cells are still only partly understood, limiting their use in cellular therapies. Here we found that increased activity of the RB cell cycle inhibitor in human embryonic stem cells induces cell cycle arrest, differentiation and cell death. Conversely, inactivation of the entire RB family (RB, p107 and p130) in human embryonic stem cells triggers G2/M arrest and cell death through functional activation of the p53 pathway and the cell cycle inhibitor p21. Differences in E2F target gene activation upon loss of RB family function between human embryonic stem cells, mouse embryonic stem cells and human fibroblasts underscore key differences in the cell cycle regulatory networks of human embryonic stem cells. Finally, loss of RB family function promotes genomic instability in both human and mouse embryonic stem cells, uncoupling cell cycle defects from chromosomal instability. These experiments indicate that a homeostatic level of RB activity is essential for the self-renewal and the survival of human embryonic stem cells.","apaCitation":"Conklin, J. F., Baker, J., &amp; Sage, J. (2012). The RB family is required for the self-renewal and survival of human embryonic stem cells. <i>NATURE COMMUNICATIONS</i>, <i>3</i>.","capCitation":"<span class=\"title\">\n  <span>The RB family is required for the self-renewal and survival of human embryonic stem cells</span>\n  <i>NATURE COMMUNICATIONS</i>\n</span>\n<span class=\"authors\">Conklin, J. F., Baker, J., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2012</span>; <span class=\"volume\">3</span>\n</span>","chicagoCitation":"Conklin, Jamie F., Julie Baker, and Julien Sage. 2012. ?The RB Family Is Required for the Self-Renewal and Survival of Human Embryonic Stem Cells.? <i>NATURE COMMUNICATIONS</i> 3. NATURE PUBLISHING GROUP.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The mechanisms ensuring the long-term self-renewal of human embryonic stem cells are still only partly understood, limiting their use in cellular therapies. Here we found that increased activity of the RB cell cycle inhibitor in human embryonic stem cells induces cell cycle arrest, differentiation and cell death. Conversely, inactivation of the entire RB family (RB, p107 and p130) in human embryonic stem cells triggers G2/M arrest and cell death through functional activation of the p53 pathway and the cell cycle inhibitor p21. Differences in E2F target gene activation upon loss of RB family function between human embryonic stem cells, mouse embryonic stem cells and human fibroblasts underscore key differences in the cell cycle regulatory networks of human embryonic stem cells. Finally, loss of RB family function promotes genomic instability in both human and mouse embryonic stem cells, uncoupling cell cycle defects from chromosomal instability. These experiments indicate that a homeostatic level of RB activity is essential for the self-renewal and the survival of human embryonic stem cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/ncomms2254\">DOI 10.1038/ncomms2254</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000316356700012\">Web of Science ID 000316356700012</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/23212373\">PubMedID 23212373</a></span>\n  </p>","text":"Abstract:\nThe mechanisms ensuring the long-term self-renewal of human embryonic stem cells are still only partly understood, limiting their use in cellular therapies. Here we found that increased activity of the RB cell cycle inhibitor in human embryonic stem cells induces cell cycle arrest, differentiation and cell death. Conversely, inactivation of the entire RB family (RB, p107 and p130) in human embryonic stem cells triggers G2/M arrest and cell death through functional activation of the p53 pathway and the cell cycle inhibitor p21. Differences in E2F target gene activation upon loss of RB family function between human embryonic stem cells, mouse embryonic stem cells and human fibroblasts underscore key differences in the cell cycle regulatory networks of human embryonic stem cells. Finally, loss of RB family function promotes genomic instability in both human and mouse embryonic stem cells, uncoupling cell cycle defects from chromosomal instability. These experiments indicate that a homeostatic level of RB activity is essential for the self-renewal and the survival of human embryonic stem cells.\nDOI: 10.1038/ncomms2254, https://dx.doi.org/10.1038/ncomms2254\nWeb of Science ID: 000316356700012, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000316356700012\nPubMedID: 23212373, https://www.ncbi.nlm.nih.gov/pubmed/23212373"},"doiId":"10.1038/ncomms2254","doiUrl":"https://dx.doi.org/10.1038/ncomms2254","featured":true,"firstPublished":{"html":"Dec 1, 2012 00:00:00 AM","text":"Dec 1, 2012 00:00:00 AM","value":"2012-12-01T00:00:00.000-08:00"},"mlaCitation":"Conklin, Jamie F., Julie Baker, and Julien Sage. ?The RB Family Is Required for the Self-Renewal and Survival of Human Embryonic Stem Cells.? <i>NATURE COMMUNICATIONS</i> 3 (2012): n. pag. Print.","pubMedId":"23212373","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/23212373","publicationId":127456,"title":"The RB family is required for the self-renewal and survival of human embryonic stem cells","type":"article","webOfScienceId":"000316356700012","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000316356700012"},{"abstract":"Human papillomavirus-16 (HPV-16) is associated etiologically with many human cervical cancers. It encodes 3 oncogenes E5, E6, and E7. Of these oncogenes, E7 has been found to be the dominant driver of cervical cancer in mice. More than 100 cellular proteins have been reported to associate with HPV-16 E7, which is thought to dysregulate the cell cycle in part by binding and inducing the degradation of pRb and its related pocket protein family members, p107 and p130. The ability of E7 to inactivate the pRb family correlates with its ability to induce head and neck cancers in mice. We previously showed that the inactivation of pRb is itself not sufficient to recapitulate the oncogenic properties of E7 in cervical carcinogenesis. In this study, we evaluated mice that were deficient in multiple pocket proteins, including mice that lacked pRb, p107, and p130. Strikingly, combined loss of two or all 3 pocket proteins resulted in development of high-grade cervical intraepithelial neoplasia, but not frank cervical carcinoma. These findings strongly argue that the oncogenic properties of HPV-16 E7 in human cervical carcinogenesis may involve disruption of E7 binding proteins beyond simply the pRb family members.","apaCitation":"Shin, M.-K., Sage, J., &amp; Lambert, P. F. (2012). Inactivating All Three Rb Family Pocket Proteins Is Insufficient to Initiate Cervical Cancer. <i>CANCER RESEARCH</i>, <i>72</i>(20), 5418?5427.","capCitation":"<span class=\"title\">\n  <span>Inactivating All Three Rb Family Pocket Proteins Is Insufficient to Initiate Cervical Cancer</span>\n  <i>CANCER RESEARCH</i>\n</span>\n<span class=\"authors\">Shin, M., Sage, J., Lambert, P. F.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2012</span>; <span class=\"volume\">72 (20)</span><span class=\"pages\">: 5418-5427</span>\n</span>","chicagoCitation":"Shin, Myeong-Kyun, Julien Sage, and Paul F. Lambert. 2012. ?Inactivating All Three Rb Family Pocket Proteins Is Insufficient to Initiate Cervical Cancer.? <i>CANCER RESEARCH</i> 72 (20). AMER ASSOC CANCER RESEARCH: 5418?27.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Human papillomavirus-16 (HPV-16) is associated etiologically with many human cervical cancers. It encodes 3 oncogenes E5, E6, and E7. Of these oncogenes, E7 has been found to be the dominant driver of cervical cancer in mice. More than 100 cellular proteins have been reported to associate with HPV-16 E7, which is thought to dysregulate the cell cycle in part by binding and inducing the degradation of pRb and its related pocket protein family members, p107 and p130. The ability of E7 to inactivate the pRb family correlates with its ability to induce head and neck cancers in mice. We previously showed that the inactivation of pRb is itself not sufficient to recapitulate the oncogenic properties of E7 in cervical carcinogenesis. In this study, we evaluated mice that were deficient in multiple pocket proteins, including mice that lacked pRb, p107, and p130. Strikingly, combined loss of two or all 3 pocket proteins resulted in development of high-grade cervical intraepithelial neoplasia, but not frank cervical carcinoma. These findings strongly argue that the oncogenic properties of HPV-16 E7 in human cervical carcinogenesis may involve disruption of E7 binding proteins beyond simply the pRb family members.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1158/0008-5472.CAN-12-2083\">DOI 10.1158/0008-5472.CAN-12-2083</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000309972700030\">Web of Science ID 000309972700030</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/22942253\">PubMedID 22942253</a></span>\n  </p>","text":"Abstract:\nHuman papillomavirus-16 (HPV-16) is associated etiologically with many human cervical cancers. It encodes 3 oncogenes E5, E6, and E7. Of these oncogenes, E7 has been found to be the dominant driver of cervical cancer in mice. More than 100 cellular proteins have been reported to associate with HPV-16 E7, which is thought to dysregulate the cell cycle in part by binding and inducing the degradation of pRb and its related pocket protein family members, p107 and p130. The ability of E7 to inactivate the pRb family correlates with its ability to induce head and neck cancers in mice. We previously showed that the inactivation of pRb is itself not sufficient to recapitulate the oncogenic properties of E7 in cervical carcinogenesis. In this study, we evaluated mice that were deficient in multiple pocket proteins, including mice that lacked pRb, p107, and p130. Strikingly, combined loss of two or all 3 pocket proteins resulted in development of high-grade cervical intraepithelial neoplasia, but not frank cervical carcinoma. These findings strongly argue that the oncogenic properties of HPV-16 E7 in human cervical carcinogenesis may involve disruption of E7 binding proteins beyond simply the pRb family members.\nDOI: 10.1158/0008-5472.CAN-12-2083, https://dx.doi.org/10.1158/0008-5472.CAN-12-2083\nWeb of Science ID: 000309972700030, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000309972700030\nPubMedID: 22942253, https://www.ncbi.nlm.nih.gov/pubmed/22942253"},"doiId":"10.1158/0008-5472.CAN-12-2083","doiUrl":"https://dx.doi.org/10.1158/0008-5472.CAN-12-2083","featured":false,"firstPublished":{"html":"Oct 15, 2012 00:00:00 AM","text":"Oct 15, 2012 00:00:00 AM","value":"2012-10-15T00:00:00.000-07:00"},"mlaCitation":"Shin, Myeong-Kyun, Julien Sage, and Paul F. Lambert. ?Inactivating All Three Rb Family Pocket Proteins Is Insufficient to Initiate Cervical Cancer.? <i>CANCER RESEARCH</i> 72.20 (2012): 5418?5427. Print.","pubMedId":"22942253","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/22942253","publicationId":242868,"title":"Inactivating All Three Rb Family Pocket Proteins Is Insufficient to Initiate Cervical Cancer","type":"article","webOfScienceId":"000309972700030","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000309972700030"},{"abstract":"Small-cell lung cancer (SCLC) is an aggressive lung tumor subtype with poor prognosis. We sequenced 29 SCLC exomes, 2 genomes and 15 transcriptomes and found an extremely high mutation rate of 7.4Â±1 protein-changing mutations per million base pairs. Therefore, we conducted integrated analyses of the various data sets to identify pathogenetically relevant mutated genes. In all cases, we found evidence for inactivation of TP53 and RB1 and identified recurrent mutations in the CREBBP, EP300 and MLL genes that encode histone modifiers. Furthermore, we observed mutations in PTEN, SLIT2 and EPHA7, as well as focal amplifications of the FGFR1 tyrosine kinase gene. Finally, we detected many of the alterations found in humans in SCLC tumors from Tp53 and Rb1 double knockout mice. Our study implicates histone modification as a major feature of SCLC, reveals potentially therapeutically tractable genomic alterations and provides a generalizable framework for the identification of biologically relevant genes in the context of high mutational background.","apaCitation":"Peifer, M., Fernandez-Cuesta, L., Sos, M. L., George, J., Seidel, D., Kasper, L. H., ? Thomas, R. K. (2012). Integrative genome analyses identify key somatic driver mutations of small-cell lung cancer. <i>NATURE GENETICS</i>, <i>44</i>(10), 1104-?","capCitation":"<span class=\"title\">\n  <span>Integrative genome analyses identify key somatic driver mutations of small-cell lung cancer</span>\n  <i>NATURE GENETICS</i>\n</span>\n<span class=\"authors\">Peifer, M., Fernandez-Cuesta, L., Sos, M. L., George, J., Seidel, D., Kasper, L. H., Plenker, D., Leenders, F., Sun, R., Zander, T., Menon, R., Koker, M., Dahmen, I., Mueller, C., Di Cerbo, V., Schildhaus, H., Altmueller, J., Baessmann, I., Becker, C., De Wilde, B., Vandesompele, J., Boehm, D., Ansen, S., Gabler, F., Wilkening, I., Heynck, S., Heuckmann, J. M., Lu, X., Carter, S. L., Cibulskis, K., Banerji, S., Getz, G., Park, K., Rauh, D., Gruetter, C., Fischer, M., Pasqualucci, L., Wright, G., Wainer, Z., Russell, P., Petersen, I., Chen, Y., Stoelben, E., Ludwig, C., Schnabel, P., Hoffmann, H., Muley, T., Brockmann, M., Engel-Riedel, W., Muscarella, L. A., Fazio, V. M., Groen, H., Timens, W., Sietsma, H., Thunnissen, E., Smit, E., Heideman, D. A., Snijders, P. J., Cappuzzo, F., Ligorio, C., Damiani, S., Field, J., Solberg, S., Brustugun, O. T., Lund-Iversen, M., Saenger, J., Clement, J. H., Soltermann, A., Moch, H., Weder, W., Solomon, B., Soria, J., Validire, P., Besse, B., Brambilla, E., Brambilla, C., Lantuejoul, S., Lorimier, P., Schneider, P. M., Hallek, M., Pao, W., Meyerson, M., Sage, J., Shendure, J., Schneider, R., Buettner, R., Wolf, J., Nuernberg, P., Perner, S., Heukamp, L. C., Brindle, P. K., Haas, S., Thomas, R. K.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2012</span>; <span class=\"volume\">44 (10)</span><span class=\"pages\">: 1104-?</span>\n</span>","chicagoCitation":"Peifer, Martin, Lynnette Fernandez-Cuesta, Martin L. Sos, Julie George, Danila Seidel, et al. 2012. ?Integrative Genome Analyses Identify Key Somatic Driver Mutations of Small-Cell Lung Cancer.? <i>NATURE GENETICS</i> 44 (10). NATURE PUBLISHING GROUP: 1104-?","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small-cell lung cancer (SCLC) is an aggressive lung tumor subtype with poor prognosis. We sequenced 29 SCLC exomes, 2 genomes and 15 transcriptomes and found an extremely high mutation rate of 7.4Â±1 protein-changing mutations per million base pairs. Therefore, we conducted integrated analyses of the various data sets to identify pathogenetically relevant mutated genes. In all cases, we found evidence for inactivation of TP53 and RB1 and identified recurrent mutations in the CREBBP, EP300 and MLL genes that encode histone modifiers. Furthermore, we observed mutations in PTEN, SLIT2 and EPHA7, as well as focal amplifications of the FGFR1 tyrosine kinase gene. Finally, we detected many of the alterations found in humans in SCLC tumors from Tp53 and Rb1 double knockout mice. Our study implicates histone modification as a major feature of SCLC, reveals potentially therapeutically tractable genomic alterations and provides a generalizable framework for the identification of biologically relevant genes in the context of high mutational background.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/ng.2396\">DOI 10.1038/ng.2396</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000309550200009\">Web of Science ID 000309550200009</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/22941188\">PubMedID 22941188</a></span>\n  </p>","text":"Abstract:\nSmall-cell lung cancer (SCLC) is an aggressive lung tumor subtype with poor prognosis. We sequenced 29 SCLC exomes, 2 genomes and 15 transcriptomes and found an extremely high mutation rate of 7.4Â±1 protein-changing mutations per million base pairs. Therefore, we conducted integrated analyses of the various data sets to identify pathogenetically relevant mutated genes. In all cases, we found evidence for inactivation of TP53 and RB1 and identified recurrent mutations in the CREBBP, EP300 and MLL genes that encode histone modifiers. Furthermore, we observed mutations in PTEN, SLIT2 and EPHA7, as well as focal amplifications of the FGFR1 tyrosine kinase gene. Finally, we detected many of the alterations found in humans in SCLC tumors from Tp53 and Rb1 double knockout mice. Our study implicates histone modification as a major feature of SCLC, reveals potentially therapeutically tractable genomic alterations and provides a generalizable framework for the identification of biologically relevant genes in the context of high mutational background.\nDOI: 10.1038/ng.2396, https://dx.doi.org/10.1038/ng.2396\nWeb of Science ID: 000309550200009, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000309550200009\nPubMedID: 22941188, https://www.ncbi.nlm.nih.gov/pubmed/22941188"},"doiId":"10.1038/ng.2396","doiUrl":"https://dx.doi.org/10.1038/ng.2396","featured":true,"firstPublished":{"html":"Oct 1, 2012 00:00:00 AM","text":"Oct 1, 2012 00:00:00 AM","value":"2012-10-01T00:00:00.000-07:00"},"mlaCitation":"Peifer, Martin et al. ?Integrative Genome Analyses Identify Key Somatic Driver Mutations of Small-Cell Lung Cancer.? <i>NATURE GENETICS</i> 44.10 (2012): 1104-? Print.","pubMedId":"22941188","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/22941188","publicationId":187872,"title":"Integrative genome analyses identify key somatic driver mutations of small-cell lung cancer","type":"article","webOfScienceId":"000309550200009","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000309550200009"},{"abstract":"Stem cells play a critical role during embryonic development and in the maintenance of homeostasis in adult individuals. A better understanding of stem cell biology, including embryonic and adult stem cells, will allow the scientific community to better comprehend a number of pathologies and possibly design novel approaches to treat patients with a variety of diseases. The retinoblastoma tumor suppressor RB controls the proliferation, differentiation, and survival of cells, and accumulating evidence points to a central role for RB activity in the biology of stem and progenitor cells. In some contexts, loss of RB function in stem or progenitor cells is a key event in the initiation of cancer and determines the subtype of cancer arising from these pluripotent cells by altering their fate. In other cases, RB inactivation is often not sufficient to initiate cancer but may still lead to some stem cell expansion, raising the possibility that strategies aimed at transiently inactivating RB might provide a novel way to expand functional stem cell populations. Future experiments dedicated to better understanding how RB and the RB pathway control a stem cell's decisions to divide, self-renew, or give rise to differentiated progeny may eventually increase our capacity to control these decisions to enhance regeneration or help prevent cancer development.","apaCitation":"Sage, J. (2012). The retinoblastoma tumor suppressor and stem cell biology. <i>GENES &amp; DEVELOPMENT</i>, <i>26</i>(13), 1409?1420.","capCitation":"<span class=\"title\">\n  <span>The retinoblastoma tumor suppressor and stem cell biology</span>\n  <i>GENES &amp; DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2012</span>; <span class=\"volume\">26 (13)</span><span class=\"pages\">: 1409-1420</span>\n</span>","chicagoCitation":"Sage, Julien. 2012. ?The Retinoblastoma Tumor Suppressor and Stem Cell Biology.? <i>GENES &amp; DEVELOPMENT</i> 26 (13). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 1409?20.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Stem cells play a critical role during embryonic development and in the maintenance of homeostasis in adult individuals. A better understanding of stem cell biology, including embryonic and adult stem cells, will allow the scientific community to better comprehend a number of pathologies and possibly design novel approaches to treat patients with a variety of diseases. The retinoblastoma tumor suppressor RB controls the proliferation, differentiation, and survival of cells, and accumulating evidence points to a central role for RB activity in the biology of stem and progenitor cells. In some contexts, loss of RB function in stem or progenitor cells is a key event in the initiation of cancer and determines the subtype of cancer arising from these pluripotent cells by altering their fate. In other cases, RB inactivation is often not sufficient to initiate cancer but may still lead to some stem cell expansion, raising the possibility that strategies aimed at transiently inactivating RB might provide a novel way to expand functional stem cell populations. Future experiments dedicated to better understanding how RB and the RB pathway control a stem cell's decisions to divide, self-renew, or give rise to differentiated progeny may eventually increase our capacity to control these decisions to enhance regeneration or help prevent cancer development.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.193730.112\">DOI 10.1101/gad.193730.112</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000306175400002\">Web of Science ID 000306175400002</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/22751497\">PubMedID 22751497</a></span>\n  </p>","text":"Abstract:\nStem cells play a critical role during embryonic development and in the maintenance of homeostasis in adult individuals. A better understanding of stem cell biology, including embryonic and adult stem cells, will allow the scientific community to better comprehend a number of pathologies and possibly design novel approaches to treat patients with a variety of diseases. The retinoblastoma tumor suppressor RB controls the proliferation, differentiation, and survival of cells, and accumulating evidence points to a central role for RB activity in the biology of stem and progenitor cells. In some contexts, loss of RB function in stem or progenitor cells is a key event in the initiation of cancer and determines the subtype of cancer arising from these pluripotent cells by altering their fate. In other cases, RB inactivation is often not sufficient to initiate cancer but may still lead to some stem cell expansion, raising the possibility that strategies aimed at transiently inactivating RB might provide a novel way to expand functional stem cell populations. Future experiments dedicated to better understanding how RB and the RB pathway control a stem cell's decisions to divide, self-renew, or give rise to differentiated progeny may eventually increase our capacity to control these decisions to enhance regeneration or help prevent cancer development.\nDOI: 10.1101/gad.193730.112, https://dx.doi.org/10.1101/gad.193730.112\nWeb of Science ID: 000306175400002, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000306175400002\nPubMedID: 22751497, https://www.ncbi.nlm.nih.gov/pubmed/22751497"},"doiId":"10.1101/gad.193730.112","doiUrl":"https://dx.doi.org/10.1101/gad.193730.112","featured":true,"firstPublished":{"html":"Jul 1, 2012 00:00:00 AM","text":"Jul 1, 2012 00:00:00 AM","value":"2012-07-01T00:00:00.000-07:00"},"mlaCitation":"Sage, Julien. ?The Retinoblastoma Tumor Suppressor and Stem Cell Biology.? <i>GENES &amp; DEVELOPMENT</i> 26.13 (2012): 1409?1420. Print.","pubMedId":"22751497","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/22751497","publicationId":122469,"title":"The retinoblastoma tumor suppressor and stem cell biology","type":"article","webOfScienceId":"000306175400002","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000306175400002"},{"abstract":"Smyd3 is a lysine methyltransferase implicated in chromatin and cancer regulation. Here we show that Smyd3 catalyzes histone H4 methylation at lysine 5 (H4K5me). This novel histone methylation mark is detected in diverse cell types and its formation is attenuated by depletion of Smyd3 protein. Further, Smyd3-driven cancer cell phenotypes require its enzymatic activity. Thus, Smyd3, via H4K5 methylation, provides a potential new link between chromatin dynamics and neoplastic disease.","apaCitation":"Van Aller, G. S., Reynoird, N., Barbash, O., Huddleston, M., Liu, S., Zmoos, A.-F., ? Kruger, R. G. (2012). Smyd3 regulates cancer cell phenotypes and catalyzes histone H4 lysine 5 methylation. <i>EPIGENETICS</i>, <i>7</i>(4), 340?343.","capCitation":"<span class=\"title\">\n  <span>Smyd3 regulates cancer cell phenotypes and catalyzes histone H4 lysine 5 methylation</span>\n  <i>EPIGENETICS</i>\n</span>\n<span class=\"authors\">Van Aller, G. S., Reynoird, N., Barbash, O., Huddleston, M., Liu, S., Zmoos, A., McDevitt, P., Sinnamon, R., Le, B., Mas, G., Annan, R., Sage, J., Garcia, B. A., Tummino, P. J., Gozani, O., Kruger, R. G.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2012</span>; <span class=\"volume\">7 (4)</span><span class=\"pages\">: 340-343</span>\n</span>","chicagoCitation":"Van Aller, Glenn S., Nicolas Reynoird, Olena Barbash, Michael Huddleston, Shichong Liu, et al. 2012. ?Smyd3 Regulates Cancer Cell Phenotypes and Catalyzes Histone H4 Lysine 5 Methylation.? <i>EPIGENETICS</i> 7 (4). LANDES BIOSCIENCE: 340?43.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Smyd3 is a lysine methyltransferase implicated in chromatin and cancer regulation. Here we show that Smyd3 catalyzes histone H4 methylation at lysine 5 (H4K5me). This novel histone methylation mark is detected in diverse cell types and its formation is attenuated by depletion of Smyd3 protein. Further, Smyd3-driven cancer cell phenotypes require its enzymatic activity. Thus, Smyd3, via H4K5 methylation, provides a potential new link between chromatin dynamics and neoplastic disease.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000302493300004\">Web of Science ID 000302493300004</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/22419068\">PubMedID 22419068</a></span>\n  </p>","text":"Abstract:\nSmyd3 is a lysine methyltransferase implicated in chromatin and cancer regulation. Here we show that Smyd3 catalyzes histone H4 methylation at lysine 5 (H4K5me). This novel histone methylation mark is detected in diverse cell types and its formation is attenuated by depletion of Smyd3 protein. Further, Smyd3-driven cancer cell phenotypes require its enzymatic activity. Thus, Smyd3, via H4K5 methylation, provides a potential new link between chromatin dynamics and neoplastic disease.\n\nWeb of Science ID: 000302493300004, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000302493300004\nPubMedID: 22419068, https://www.ncbi.nlm.nih.gov/pubmed/22419068"},"featured":true,"firstPublished":{"html":"Apr 1, 2012 00:00:00 AM","text":"Apr 1, 2012 00:00:00 AM","value":"2012-04-01T00:00:00.000-07:00"},"mlaCitation":"Van Aller, Glenn S. et al. ?Smyd3 Regulates Cancer Cell Phenotypes and Catalyzes Histone H4 Lysine 5 Methylation.? <i>EPIGENETICS</i> 7.4 (2012): 340?343. Print.","pubMedId":"22419068","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/22419068","publicationId":118651,"title":"Smyd3 regulates cancer cell phenotypes and catalyzes histone H4 lysine 5 methylation","type":"article","webOfScienceId":"000302493300004","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000302493300004"},{"apaCitation":"Sage, J., &amp; Cleary, M. L. (2012). GENOMICS The path to retinoblastoma. <i>NATURE</i>, <i>481</i>(7381), 269?270.","capCitation":"<span class=\"title\">\n  <span>GENOMICS The path to retinoblastoma</span>\n  <i>NATURE</i>\n</span>\n<span class=\"authors\">Sage, J., Cleary, M. L.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2012</span>; <span class=\"volume\">481 (7381)</span><span class=\"pages\">: 269-270</span>\n</span>","chicagoCitation":"Sage, Julien, and Michael L. Cleary. 2012. ?GENOMICS The Path to Retinoblastoma.? <i>NATURE</i> 481 (7381). NATURE PUBLISHING GROUP: 269?70.","detail":{"html":"<p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000299210600023\">Web of Science ID 000299210600023</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/22258599\">PubMedID 22258599</a></span>\n  </p>","text":"Web of Science ID: 000299210600023, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000299210600023\nPubMedID: 22258599, https://www.ncbi.nlm.nih.gov/pubmed/22258599"},"featured":false,"firstPublished":{"html":"Jan 19, 2012 00:00:00 AM","text":"Jan 19, 2012 00:00:00 AM","value":"2012-01-19T00:00:00.000-08:00"},"mlaCitation":"Sage, Julien, and Michael L. Cleary. ?GENOMICS The Path to Retinoblastoma.? <i>NATURE</i> 481.7381 (2012): 269?270. Print.","pubMedId":"22258599","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/22258599","publicationId":242869,"title":"GENOMICS The path to retinoblastoma","type":"article","webOfScienceId":"000299210600023","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000299210600023"},{"abstract":"Small-cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer for which there is no effective treatment. Using a mouse model in which deletion of Rb1 and Trp53 in the lung epithelium of adult mice induces SCLC, we found that the Hedgehog signaling pathway is activated in SCLC cells independently of the lung microenvironment. Constitutive activation of the Hedgehog signaling molecule Smoothened (Smo) promoted the clonogenicity of human SCLC in vitro and the initiation and progression of mouse SCLC in vivo. Reciprocally, deletion of Smo in Rb1 and Trp53-mutant lung epithelial cells strongly suppressed SCLC initiation and progression in mice. Furthermore, pharmacological blockade of Hedgehog signaling inhibited the growth of mouse and human SCLC, most notably following chemotherapy. These findings show a crucial cell-intrinsic role for Hedgehog signaling in the development and maintenance of SCLC and identify Hedgehog pathway inhibition as a therapeutic strategy to slow the progression of disease and delay cancer recurrence in individuals with SCLC.","apaCitation":"Park, K.-S., Martelotto, L. G., Peifer, M., Sos, M. L., Karnezis, A. N., Mahjoub, M. R., ? Sage, J. (2011). A crucial requirement for Hedgehog signaling in small cell lung cancer. <i>NATURE MEDICINE</i>, <i>17</i>(11), 1504?U1506.","capCitation":"<span class=\"title\">\n  <span>A crucial requirement for Hedgehog signaling in small cell lung cancer</span>\n  <i>NATURE MEDICINE</i>\n</span>\n<span class=\"authors\">Park, K., Martelotto, L. G., Peifer, M., Sos, M. L., Karnezis, A. N., Mahjoub, M. R., Bernard, K., Conklin, J. F., Szczepny, A., Yuan, J., Guo, R., Ospina, B., Falzon, J., Bennett, S., Brown, T. J., Markovic, A., Devereux, W. L., Ocasio, C. A., Chen, J. K., Stearns, T., Thomas, R. K., Dorsch, M., Buonamici, S., Watkins, D. N., Peacock, C. D., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">17 (11)</span><span class=\"pages\">: 1504-U1506</span>\n</span>","chicagoCitation":"Park, Kwon-Sik, Luciano G. Martelotto, Martin Peifer, Martin L. Sos, Anthony N. Karnezis, et al. 2011. ?A Crucial Requirement for Hedgehog Signaling in Small Cell Lung Cancer.? <i>NATURE MEDICINE</i> 17 (11). NATURE PUBLISHING GROUP: 1504?U1506.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small-cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer for which there is no effective treatment. Using a mouse model in which deletion of Rb1 and Trp53 in the lung epithelium of adult mice induces SCLC, we found that the Hedgehog signaling pathway is activated in SCLC cells independently of the lung microenvironment. Constitutive activation of the Hedgehog signaling molecule Smoothened (Smo) promoted the clonogenicity of human SCLC in vitro and the initiation and progression of mouse SCLC in vivo. Reciprocally, deletion of Smo in Rb1 and Trp53-mutant lung epithelial cells strongly suppressed SCLC initiation and progression in mice. Furthermore, pharmacological blockade of Hedgehog signaling inhibited the growth of mouse and human SCLC, most notably following chemotherapy. These findings show a crucial cell-intrinsic role for Hedgehog signaling in the development and maintenance of SCLC and identify Hedgehog pathway inhibition as a therapeutic strategy to slow the progression of disease and delay cancer recurrence in individuals with SCLC.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nm.2473\">DOI 10.1038/nm.2473</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000296779300043\">Web of Science ID 000296779300043</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21983857\">PubMedID 21983857</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3380617\">PubMedCentralID PMC3380617</a></span>\n  </p>","text":"Abstract:\nSmall-cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer for which there is no effective treatment. Using a mouse model in which deletion of Rb1 and Trp53 in the lung epithelium of adult mice induces SCLC, we found that the Hedgehog signaling pathway is activated in SCLC cells independently of the lung microenvironment. Constitutive activation of the Hedgehog signaling molecule Smoothened (Smo) promoted the clonogenicity of human SCLC in vitro and the initiation and progression of mouse SCLC in vivo. Reciprocally, deletion of Smo in Rb1 and Trp53-mutant lung epithelial cells strongly suppressed SCLC initiation and progression in mice. Furthermore, pharmacological blockade of Hedgehog signaling inhibited the growth of mouse and human SCLC, most notably following chemotherapy. These findings show a crucial cell-intrinsic role for Hedgehog signaling in the development and maintenance of SCLC and identify Hedgehog pathway inhibition as a therapeutic strategy to slow the progression of disease and delay cancer recurrence in individuals with SCLC.\nDOI: 10.1038/nm.2473, https://dx.doi.org/10.1038/nm.2473\nWeb of Science ID: 000296779300043, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000296779300043\nPubMedID: 21983857, https://www.ncbi.nlm.nih.gov/pubmed/21983857\nPubMedCentralID: PMC3380617, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3380617"},"doiId":"10.1038/nm.2473","doiUrl":"https://dx.doi.org/10.1038/nm.2473","featured":true,"firstPublished":{"html":"Nov 1, 2011 00:00:00 AM","text":"Nov 1, 2011 00:00:00 AM","value":"2011-11-01T00:00:00.000-07:00"},"mlaCitation":"Park, Kwon-Sik et al. ?A Crucial Requirement for Hedgehog Signaling in Small Cell Lung Cancer.? <i>NATURE MEDICINE</i> 17.11 (2011): 1504?U1506. Print.","pubMedId":"21983857","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21983857","publicationId":113515,"title":"A crucial requirement for Hedgehog signaling in small cell lung cancer","type":"article","webOfScienceId":"000296779300043","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000296779300043"},{"abstract":"Determining the signalling pathways that direct tissue expansion is a principal goal of regenerative biology. Vigorous pancreatic ?-cell replication in juvenile mice and humans declines with age, and elucidating the basis for this decay may reveal strategies for inducing ?-cell expansion, a long-sought goal for diabetes therapy. Here we show that platelet-derived growth factor receptor (Pdgfr) signalling controls age-dependent ?-cell proliferation in mouse and human pancreatic islets. With age, declining ?-cell Pdgfr levels were accompanied by reductions in ?-cell enhancer of zeste homologue 2 (Ezh2) levels and ?-cell replication. Conditional inactivation of the Pdgfra gene in ?-cells accelerated these changes, preventing mouse neonatal ?-cell expansion and adult ?-cell regeneration. Targeted human PDGFR-? activation in mouse ?-cells stimulated Erk1/2 phosphorylation, leading to Ezh2-dependent expansion of adult ?-cells. Adult human islets lack PDGF signalling competence, but exposure of juvenile human islets to PDGF-AA stimulated ?-cell proliferation. The discovery of a conserved pathway controlling age-dependent ?-cell proliferation indicates new strategies for ?-cell expansion.","apaCitation":"Chen, H., Gu, X., Liu, Y., Wang, J., Wirt, S. E., Bottino, R., ? Kim, S. K. (2011). PDGF signalling controls age-dependent proliferation in pancreatic beta-cells. <i>NATURE</i>, <i>478</i>(7369), 349-?","capCitation":"<span class=\"title\">\n  <span>PDGF signalling controls age-dependent proliferation in pancreatic beta-cells</span>\n  <i>NATURE</i>\n</span>\n<span class=\"authors\">Chen, H., Gu, X., Liu, Y., Wang, J., Wirt, S. E., Bottino, R., Schorle, H., Sage, J., Kim, S. K.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">478 (7369)</span><span class=\"pages\">: 349-?</span>\n</span>","chicagoCitation":"Chen, Hainan, Xueying Gu, Yinghua Liu, Jing Wang, Stacey E. Wirt, et al. 2011. ?PDGF Signalling Controls Age-Dependent Proliferation in Pancreatic Beta-Cells.? <i>NATURE</i> 478 (7369). NATURE PUBLISHING GROUP: 349-?","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Determining the signalling pathways that direct tissue expansion is a principal goal of regenerative biology. Vigorous pancreatic ?-cell replication in juvenile mice and humans declines with age, and elucidating the basis for this decay may reveal strategies for inducing ?-cell expansion, a long-sought goal for diabetes therapy. Here we show that platelet-derived growth factor receptor (Pdgfr) signalling controls age-dependent ?-cell proliferation in mouse and human pancreatic islets. With age, declining ?-cell Pdgfr levels were accompanied by reductions in ?-cell enhancer of zeste homologue 2 (Ezh2) levels and ?-cell replication. Conditional inactivation of the Pdgfra gene in ?-cells accelerated these changes, preventing mouse neonatal ?-cell expansion and adult ?-cell regeneration. Targeted human PDGFR-? activation in mouse ?-cells stimulated Erk1/2 phosphorylation, leading to Ezh2-dependent expansion of adult ?-cells. Adult human islets lack PDGF signalling competence, but exposure of juvenile human islets to PDGF-AA stimulated ?-cell proliferation. The discovery of a conserved pathway controlling age-dependent ?-cell proliferation indicates new strategies for ?-cell expansion.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nature10502\">DOI 10.1038/nature10502</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000296021100038\">Web of Science ID 000296021100038</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21993628\">PubMedID 21993628</a></span>\n  </p>","text":"Abstract:\nDetermining the signalling pathways that direct tissue expansion is a principal goal of regenerative biology. Vigorous pancreatic ?-cell replication in juvenile mice and humans declines with age, and elucidating the basis for this decay may reveal strategies for inducing ?-cell expansion, a long-sought goal for diabetes therapy. Here we show that platelet-derived growth factor receptor (Pdgfr) signalling controls age-dependent ?-cell proliferation in mouse and human pancreatic islets. With age, declining ?-cell Pdgfr levels were accompanied by reductions in ?-cell enhancer of zeste homologue 2 (Ezh2) levels and ?-cell replication. Conditional inactivation of the Pdgfra gene in ?-cells accelerated these changes, preventing mouse neonatal ?-cell expansion and adult ?-cell regeneration. Targeted human PDGFR-? activation in mouse ?-cells stimulated Erk1/2 phosphorylation, leading to Ezh2-dependent expansion of adult ?-cells. Adult human islets lack PDGF signalling competence, but exposure of juvenile human islets to PDGF-AA stimulated ?-cell proliferation. The discovery of a conserved pathway controlling age-dependent ?-cell proliferation indicates new strategies for ?-cell expansion.\nDOI: 10.1038/nature10502, https://dx.doi.org/10.1038/nature10502\nWeb of Science ID: 000296021100038, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000296021100038\nPubMedID: 21993628, https://www.ncbi.nlm.nih.gov/pubmed/21993628"},"doiId":"10.1038/nature10502","doiUrl":"https://dx.doi.org/10.1038/nature10502","featured":false,"firstPublished":{"html":"Oct 20, 2011 00:00:00 AM","text":"Oct 20, 2011 00:00:00 AM","value":"2011-10-20T00:00:00.000-07:00"},"mlaCitation":"Chen, Hainan et al. ?PDGF Signalling Controls Age-Dependent Proliferation in Pancreatic Beta-Cells.? <i>NATURE</i> 478.7369 (2011): 349-? Print.","pubMedId":"21993628","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21993628","publicationId":113634,"title":"PDGF signalling controls age-dependent proliferation in pancreatic beta-cells","type":"article","webOfScienceId":"000296021100038","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000296021100038"},{"apaCitation":"Sirota, M., DUDLEY, J. T., Kim, J., Chiang, A. P., Morgan, A. A., Sweet-Cordero, A., ? Butte, A. J. (2011). Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data (vol 3, 96ra77, 2011). <i>SCIENCE TRANSLATIONAL MEDICINE</i>, <i>3</i>(102).","capCitation":"<span class=\"title\">\n  <span>Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data (vol 3, 96ra77, 2011)</span>\n  <i>SCIENCE TRANSLATIONAL MEDICINE</i>\n</span>\n<span class=\"authors\">Sirota, M., DUDLEY, J. T., Kim, J., Chiang, A. P., Morgan, A. A., Sweet-Cordero, A., Sage, J., Butte, A. J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">3 (102)</span>\n</span>","chicagoCitation":"Sirota, M., J. T. DUDLEY, J. Kim, A. P. Chiang, A. A. Morgan, et al. 2011. ?Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data (Vol 3, 96ra77, 2011).? <i>SCIENCE TRANSLATIONAL MEDICINE</i> 3 (102). AMER ASSOC ADVANCEMENT SCIENCE.","detail":{"html":"<p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1126/scitranslmed.3003215\">DOI 10.1126/scitranslmed.3003215</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000295323200013\">Web of Science ID 000295323200013</a></span>\n  </p>","text":"DOI: 10.1126/scitranslmed.3003215, https://dx.doi.org/10.1126/scitranslmed.3003215\nWeb of Science ID: 000295323200013, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000295323200013"},"doiId":"10.1126/scitranslmed.3003215","doiUrl":"https://dx.doi.org/10.1126/scitranslmed.3003215","featured":false,"firstPublished":{"html":"Sep 28, 2011 00:00:00 AM","text":"Sep 28, 2011 00:00:00 AM","value":"2011-09-28T00:00:00.000-07:00"},"mlaCitation":"Sirota, M. et al. ?Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data (Vol 3, 96ra77, 2011).? <i>SCIENCE TRANSLATIONAL MEDICINE</i> 3.102 (2011): n. pag. Print.","publicationId":242870,"title":"Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data (vol 3, 96ra77, 2011)","type":"article","webOfScienceId":"000295323200013","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000295323200013"},{"abstract":"Hepatocellular carcinoma (HCC) is the third cancer killer worldwide with >600,000 deaths every year. Although the major risk factors are known, therapeutic options in patients remain limited in part because of our incomplete understanding of the cellular and molecular mechanisms influencing HCC development. Evidence indicates that the retinoblastoma (RB) pathway is functionally inactivated in most cases of HCC by genetic, epigenetic, and/or viral mechanisms. To investigate the functional relevance of this observation, we inactivated the RB pathway in the liver of adult mice by deleting the three members of the Rb (Rb1) gene family: Rb, p107, and p130. Rb family triple knockout mice develop liver tumors with histopathological features and gene expression profiles similar to human HCC. In this mouse model, cancer initiation is associated with the specific expansion of populations of liver stem/progenitor cells, indicating that the RB pathway may prevent HCC development by maintaining the quiescence of adult liver progenitor cells. In addition, we show that during tumor progression, activation of the Notch pathway via E2F transcription factors serves as a negative feedback mechanism to slow HCC growth. The level of Notch activity is also able to predict survival of HCC patients, suggesting novel means to diagnose and treat HCC.","apaCitation":"Viatour, P., Ehmer, U., Saddic, L. A., Dorrell, C., Andersen, J. B., Lin, C. W., ? Sage, J. (2011). Notch signaling inhibits hepatocellular carcinoma following inactivation of the RB pathway. <i>JOURNAL OF EXPERIMENTAL MEDICINE</i>, <i>208</i>(10), 1963?1976.","capCitation":"<span class=\"title\">\n  <span>Notch signaling inhibits hepatocellular carcinoma following inactivation of the RB pathway</span>\n  <i>JOURNAL OF EXPERIMENTAL MEDICINE</i>\n</span>\n<span class=\"authors\">Viatour, P., Ehmer, U., Saddic, L. A., Dorrell, C., Andersen, J. B., Lin, C., Zmoos, A., Mazur, P. K., Schaffer, B. E., Ostermeier, A., Vogel, H., Sylvester, K. G., Thorgeirsson, S. S., Grompe, M., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">208 (10)</span><span class=\"pages\">: 1963-1976</span>\n</span>","chicagoCitation":"Viatour, Patrick, Ursula Ehmer, Louis A. Saddic, Craig Dorrell, Jesper B. Andersen, et al. 2011. ?Notch Signaling Inhibits Hepatocellular Carcinoma Following Inactivation of the RB Pathway.? <i>JOURNAL OF EXPERIMENTAL MEDICINE</i> 208 (10). ROCKEFELLER UNIV PRESS: 1963?76.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Hepatocellular carcinoma (HCC) is the third cancer killer worldwide with >600,000 deaths every year. Although the major risk factors are known, therapeutic options in patients remain limited in part because of our incomplete understanding of the cellular and molecular mechanisms influencing HCC development. Evidence indicates that the retinoblastoma (RB) pathway is functionally inactivated in most cases of HCC by genetic, epigenetic, and/or viral mechanisms. To investigate the functional relevance of this observation, we inactivated the RB pathway in the liver of adult mice by deleting the three members of the Rb (Rb1) gene family: Rb, p107, and p130. Rb family triple knockout mice develop liver tumors with histopathological features and gene expression profiles similar to human HCC. In this mouse model, cancer initiation is associated with the specific expansion of populations of liver stem/progenitor cells, indicating that the RB pathway may prevent HCC development by maintaining the quiescence of adult liver progenitor cells. In addition, we show that during tumor progression, activation of the Notch pathway via E2F transcription factors serves as a negative feedback mechanism to slow HCC growth. The level of Notch activity is also able to predict survival of HCC patients, suggesting novel means to diagnose and treat HCC.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1084/jem.20110198\">DOI 10.1084/jem.20110198</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000295318900005\">Web of Science ID 000295318900005</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21875955\">PubMedID 21875955</a></span>\n  </p>","text":"Abstract:\nHepatocellular carcinoma (HCC) is the third cancer killer worldwide with >600,000 deaths every year. Although the major risk factors are known, therapeutic options in patients remain limited in part because of our incomplete understanding of the cellular and molecular mechanisms influencing HCC development. Evidence indicates that the retinoblastoma (RB) pathway is functionally inactivated in most cases of HCC by genetic, epigenetic, and/or viral mechanisms. To investigate the functional relevance of this observation, we inactivated the RB pathway in the liver of adult mice by deleting the three members of the Rb (Rb1) gene family: Rb, p107, and p130. Rb family triple knockout mice develop liver tumors with histopathological features and gene expression profiles similar to human HCC. In this mouse model, cancer initiation is associated with the specific expansion of populations of liver stem/progenitor cells, indicating that the RB pathway may prevent HCC development by maintaining the quiescence of adult liver progenitor cells. In addition, we show that during tumor progression, activation of the Notch pathway via E2F transcription factors serves as a negative feedback mechanism to slow HCC growth. The level of Notch activity is also able to predict survival of HCC patients, suggesting novel means to diagnose and treat HCC.\nDOI: 10.1084/jem.20110198, https://dx.doi.org/10.1084/jem.20110198\nWeb of Science ID: 000295318900005, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000295318900005\nPubMedID: 21875955, https://www.ncbi.nlm.nih.gov/pubmed/21875955"},"doiId":"10.1084/jem.20110198","doiUrl":"https://dx.doi.org/10.1084/jem.20110198","featured":true,"firstPublished":{"html":"Sep 26, 2011 00:00:00 AM","text":"Sep 26, 2011 00:00:00 AM","value":"2011-09-26T00:00:00.000-07:00"},"mlaCitation":"Viatour, Patrick et al. ?Notch Signaling Inhibits Hepatocellular Carcinoma Following Inactivation of the RB Pathway.? <i>JOURNAL OF EXPERIMENTAL MEDICINE</i> 208.10 (2011): 1963?1976. Print.","pubMedId":"21875955","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21875955","publicationId":112237,"title":"Notch signaling inhibits hepatocellular carcinoma following inactivation of the RB pathway","type":"article","webOfScienceId":"000295318900005","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000295318900005"},{"abstract":"We investigated the potential of in-depth quantitative proteomics to reveal plasma protein signatures that reflect lung tumor biology. We compared plasma protein profiles of four mouse models of lung cancer with profiles of models of pancreatic, ovarian, colon, prostate, and breast cancer and two models of inflammation. A protein signature for Titf1/Nkx2-1, a known lineage-survival oncogene in lung cancer, was found in plasmas of mouse models of lung adenocarcinoma. An EGFR signature was found in plasma of an EGFR mutant model, and a distinct plasma signature related to neuroendocrine development was uncovered in the small-cell lung cancer model. We demonstrate relevance to human lung cancer of the protein signatures identified on the basis of mouse models.","apaCitation":"Taguchi, A., Politi, K., Pitteri, S. J., Lockwood, W. W., Faca, V. M., Kelly-Spratt, K., ? Hanash, S. M. (2011). Lung Cancer Signatures in Plasma Based on Proteome Profiling of Mouse Tumor Models. <i>CANCER CELL</i>, <i>20</i>(3), 289?299.","capCitation":"<span class=\"title\">\n  <span>Lung Cancer Signatures in Plasma Based on Proteome Profiling of Mouse Tumor Models</span>\n  <i>CANCER CELL</i>\n</span>\n<span class=\"authors\">Taguchi, A., Politi, K., Pitteri, S. J., Lockwood, W. W., Faca, V. M., Kelly-Spratt, K., Wong, C., Zhang, Q., Chin, A., Park, K., Goodman, G., Gazdar, A. F., Sage, J., Dinulescu, D. M., Kucherlapati, R., DePinho, R. A., Kemp, C. J., Varmus, H. E., Hanash, S. M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">20 (3)</span><span class=\"pages\">: 289-299</span>\n</span>","chicagoCitation":"Taguchi, Ayumu, Katerina Politi, Sharon J. Pitteri, William W. Lockwood, Vitor M. Faca, et al. 2011. ?Lung Cancer Signatures in Plasma Based on Proteome Profiling of Mouse Tumor Models.? <i>CANCER CELL</i> 20 (3). CELL PRESS: 289?99.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">We investigated the potential of in-depth quantitative proteomics to reveal plasma protein signatures that reflect lung tumor biology. We compared plasma protein profiles of four mouse models of lung cancer with profiles of models of pancreatic, ovarian, colon, prostate, and breast cancer and two models of inflammation. A protein signature for Titf1/Nkx2-1, a known lineage-survival oncogene in lung cancer, was found in plasmas of mouse models of lung adenocarcinoma. An EGFR signature was found in plasma of an EGFR mutant model, and a distinct plasma signature related to neuroendocrine development was uncovered in the small-cell lung cancer model. We demonstrate relevance to human lung cancer of the protein signatures identified on the basis of mouse models.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.ccr.2011.08.007\">DOI 10.1016/j.ccr.2011.08.007</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000295205700006\">Web of Science ID 000295205700006</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21907921\">PubMedID 21907921</a></span>\n  </p>","text":"Abstract:\nWe investigated the potential of in-depth quantitative proteomics to reveal plasma protein signatures that reflect lung tumor biology. We compared plasma protein profiles of four mouse models of lung cancer with profiles of models of pancreatic, ovarian, colon, prostate, and breast cancer and two models of inflammation. A protein signature for Titf1/Nkx2-1, a known lineage-survival oncogene in lung cancer, was found in plasmas of mouse models of lung adenocarcinoma. An EGFR signature was found in plasma of an EGFR mutant model, and a distinct plasma signature related to neuroendocrine development was uncovered in the small-cell lung cancer model. We demonstrate relevance to human lung cancer of the protein signatures identified on the basis of mouse models.\nDOI: 10.1016/j.ccr.2011.08.007, https://dx.doi.org/10.1016/j.ccr.2011.08.007\nWeb of Science ID: 000295205700006, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000295205700006\nPubMedID: 21907921, https://www.ncbi.nlm.nih.gov/pubmed/21907921"},"doiId":"10.1016/j.ccr.2011.08.007","doiUrl":"https://dx.doi.org/10.1016/j.ccr.2011.08.007","featured":false,"firstPublished":{"html":"Sep 13, 2011 00:00:00 AM","text":"Sep 13, 2011 00:00:00 AM","value":"2011-09-13T00:00:00.000-07:00"},"mlaCitation":"Taguchi, Ayumu et al. ?Lung Cancer Signatures in Plasma Based on Proteome Profiling of Mouse Tumor Models.? <i>CANCER CELL</i> 20.3 (2011): 289?299. Print.","pubMedId":"21907921","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21907921","publicationId":112607,"title":"Lung Cancer Signatures in Plasma Based on Proteome Profiling of Mouse Tumor Models","type":"article","webOfScienceId":"000295205700006","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000295205700006"},{"abstract":"The retinoblastoma (RB) tumor suppressor belongs to a cellular pathway that plays a crucial role in restricting the G1-S transition of the cell cycle in response to a large number of extracellular and intracellular cues. Research in the last decade has highlighted the complexity of regulatory networks that ensure proper cell cycle progression, and has also identified multiple cellular functions beyond cell cycle regulation for RB and its two family members, p107 and p130. Here we review some of the recent evidence pointing to a role of RB as a molecular adaptor at the crossroads of multiple pathways, ensuring cellular homeostasis in different contexts. In particular, we discuss the pro- and anti-tumorigenic roles of RB during the early stages of cancer, as well as the importance of the RB pathway in stem cells and cell fate decisions.","apaCitation":"Viatour, P., &amp; Sage, J. (2011). Newly identified aspects of tumor suppression by RB. <i>DISEASE MODELS &amp; MECHANISMS</i>, <i>4</i>(5), 581?585.","capCitation":"<span class=\"title\">\n  <span>Newly identified aspects of tumor suppression by RB</span>\n  <i>DISEASE MODELS &amp; MECHANISMS</i>\n</span>\n<span class=\"authors\">Viatour, P., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">4 (5)</span><span class=\"pages\">: 581-585</span>\n</span>","chicagoCitation":"Viatour, Patrick, and Julien Sage. 2011. ?Newly Identified Aspects of Tumor Suppression by RB.? <i>DISEASE MODELS &amp; MECHANISMS</i> 4 (5). COMPANY OF BIOLOGISTS LTD: 581?85.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma (RB) tumor suppressor belongs to a cellular pathway that plays a crucial role in restricting the G1-S transition of the cell cycle in response to a large number of extracellular and intracellular cues. Research in the last decade has highlighted the complexity of regulatory networks that ensure proper cell cycle progression, and has also identified multiple cellular functions beyond cell cycle regulation for RB and its two family members, p107 and p130. Here we review some of the recent evidence pointing to a role of RB as a molecular adaptor at the crossroads of multiple pathways, ensuring cellular homeostasis in different contexts. In particular, we discuss the pro- and anti-tumorigenic roles of RB during the early stages of cancer, as well as the importance of the RB pathway in stem cells and cell fate decisions.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1242/dmm.008060\">DOI 10.1242/dmm.008060</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000294411800005\">Web of Science ID 000294411800005</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21878458\">PubMedID 21878458</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma (RB) tumor suppressor belongs to a cellular pathway that plays a crucial role in restricting the G1-S transition of the cell cycle in response to a large number of extracellular and intracellular cues. Research in the last decade has highlighted the complexity of regulatory networks that ensure proper cell cycle progression, and has also identified multiple cellular functions beyond cell cycle regulation for RB and its two family members, p107 and p130. Here we review some of the recent evidence pointing to a role of RB as a molecular adaptor at the crossroads of multiple pathways, ensuring cellular homeostasis in different contexts. In particular, we discuss the pro- and anti-tumorigenic roles of RB during the early stages of cancer, as well as the importance of the RB pathway in stem cells and cell fate decisions.\nDOI: 10.1242/dmm.008060, https://dx.doi.org/10.1242/dmm.008060\nWeb of Science ID: 000294411800005, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000294411800005\nPubMedID: 21878458, https://www.ncbi.nlm.nih.gov/pubmed/21878458"},"doiId":"10.1242/dmm.008060","doiUrl":"https://dx.doi.org/10.1242/dmm.008060","featured":false,"firstPublished":{"html":"Sep 1, 2011 00:00:00 AM","text":"Sep 1, 2011 00:00:00 AM","value":"2011-09-01T00:00:00.000-07:00"},"mlaCitation":"Viatour, Patrick, and Julien Sage. ?Newly Identified Aspects of Tumor Suppression by RB.? <i>DISEASE MODELS &amp; MECHANISMS</i> 4.5 (2011): 581?585. Print.","pubMedId":"21878458","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21878458","publicationId":112259,"title":"Newly identified aspects of tumor suppression by RB","type":"article","webOfScienceId":"000294411800005","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000294411800005"},{"abstract":"The application of established drug compounds to new therapeutic indications, known as drug repositioning, offers several advantages over traditional drug development, including reduced development costs and shorter paths to approval. Recent approaches to drug repositioning use high-throughput experimental approaches to assess a compound's potential therapeutic qualities. Here, we present a systematic computational approach to predict novel therapeutic indications on the basis of comprehensive testing of molecular signatures in drug-disease pairs. We integrated gene expression measurements from 100 diseases and gene expression measurements on 164 drug compounds, yielding predicted therapeutic potentials for these drugs. We recovered many known drug and disease relationships using computationally derived therapeutic potentials and also predict many new indications for these 164 drugs. We experimentally validated a prediction for the antiulcer drug cimetidine as a candidate therapeutic in the treatment of lung adenocarcinoma, and demonstrate its efficacy both in vitro and in vivo using mouse xenograft models. This computational method provides a systematic approach for repositioning established drugs to treat a wide range of human diseases.","apaCitation":"Sirota, M., Dudley, J. T., Kim, J., Chiang, A. P., Morgan, A. A., Sweet-Cordero, A., ? Butte, A. J. (2011). Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data. <i>SCIENCE TRANSLATIONAL MEDICINE</i>, <i>3</i>(96).","capCitation":"<span class=\"title\">\n  <span>Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data</span>\n  <i>SCIENCE TRANSLATIONAL MEDICINE</i>\n</span>\n<span class=\"authors\">Sirota, M., Dudley, J. T., Kim, J., Chiang, A. P., Morgan, A. A., Sweet-Cordero, A., Sage, J., Butte, A. J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">3 (96)</span>\n</span>","chicagoCitation":"Sirota, Marina, Joel T. Dudley, Jeewon Kim, Annie P. Chiang, Alex A. Morgan, et al. 2011. ?Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data.? <i>SCIENCE TRANSLATIONAL MEDICINE</i> 3 (96). AMER ASSOC ADVANCEMENT SCIENCE.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The application of established drug compounds to new therapeutic indications, known as drug repositioning, offers several advantages over traditional drug development, including reduced development costs and shorter paths to approval. Recent approaches to drug repositioning use high-throughput experimental approaches to assess a compound's potential therapeutic qualities. Here, we present a systematic computational approach to predict novel therapeutic indications on the basis of comprehensive testing of molecular signatures in drug-disease pairs. We integrated gene expression measurements from 100 diseases and gene expression measurements on 164 drug compounds, yielding predicted therapeutic potentials for these drugs. We recovered many known drug and disease relationships using computationally derived therapeutic potentials and also predict many new indications for these 164 drugs. We experimentally validated a prediction for the antiulcer drug cimetidine as a candidate therapeutic in the treatment of lung adenocarcinoma, and demonstrate its efficacy both in vitro and in vivo using mouse xenograft models. This computational method provides a systematic approach for repositioning established drugs to treat a wide range of human diseases.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1126/scitranslmed.3001318\">DOI 10.1126/scitranslmed.3001318</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000293953100005\">Web of Science ID 000293953100005</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21849665\">PubMedID 21849665</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3502016\">PubMedCentralID PMC3502016</a></span>\n  </p>","text":"Abstract:\nThe application of established drug compounds to new therapeutic indications, known as drug repositioning, offers several advantages over traditional drug development, including reduced development costs and shorter paths to approval. Recent approaches to drug repositioning use high-throughput experimental approaches to assess a compound's potential therapeutic qualities. Here, we present a systematic computational approach to predict novel therapeutic indications on the basis of comprehensive testing of molecular signatures in drug-disease pairs. We integrated gene expression measurements from 100 diseases and gene expression measurements on 164 drug compounds, yielding predicted therapeutic potentials for these drugs. We recovered many known drug and disease relationships using computationally derived therapeutic potentials and also predict many new indications for these 164 drugs. We experimentally validated a prediction for the antiulcer drug cimetidine as a candidate therapeutic in the treatment of lung adenocarcinoma, and demonstrate its efficacy both in vitro and in vivo using mouse xenograft models. This computational method provides a systematic approach for repositioning established drugs to treat a wide range of human diseases.\nDOI: 10.1126/scitranslmed.3001318, https://dx.doi.org/10.1126/scitranslmed.3001318\nWeb of Science ID: 000293953100005, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000293953100005\nPubMedID: 21849665, https://www.ncbi.nlm.nih.gov/pubmed/21849665\nPubMedCentralID: PMC3502016, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3502016"},"doiId":"10.1126/scitranslmed.3001318","doiUrl":"https://dx.doi.org/10.1126/scitranslmed.3001318","featured":false,"firstPublished":{"html":"Aug 17, 2011 00:00:00 AM","text":"Aug 17, 2011 00:00:00 AM","value":"2011-08-17T00:00:00.000-07:00"},"mlaCitation":"Sirota, Marina et al. ?Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data.? <i>SCIENCE TRANSLATIONAL MEDICINE</i> 3.96 (2011): n. pag. Print.","pubMedId":"21849665","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21849665","publicationId":111881,"title":"Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data","type":"article","webOfScienceId":"000293953100005","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000293953100005"},{"abstract":"It is widely believed that the molecular and cellular features of a tumor reflect its cell of origin and can thus provide clues about treatment targets. The retinoblastoma cell of origin has been debated for over a century. Here, we report that human and mouse retinoblastomas have molecular, cellular, and neurochemical features of multiple cell classes, principally amacrine/horizontal interneurons, retinal progenitor cells, and photoreceptors. Importantly, single-cell gene expression array analysis showed that these multiple cell type-specific developmental programs are coexpressed in individual retinoblastoma cells, which creates a progenitor/neuronal hybrid cell. Furthermore, neurotransmitter receptors, transporters, and biosynthetic enzymes are expressed in human retinoblastoma, and targeted disruption of these pathways reduces retinoblastoma growth inÂ vivo and inÂ vitro.","apaCitation":"McEvoy, J., Flores-Otero, J., Zhang, J., Nemeth, K., Brennan, R., Bradley, C., ? Dyer, M. A. (2011). Coexpression of Normally Incompatible Developmental Pathways in Retinoblastoma Genesis. <i>CANCER CELL</i>, <i>20</i>(2), 260?275.","capCitation":"<span class=\"title\">\n  <span>Coexpression of Normally Incompatible Developmental Pathways in Retinoblastoma Genesis</span>\n  <i>CANCER CELL</i>\n</span>\n<span class=\"authors\">McEvoy, J., Flores-Otero, J., Zhang, J., Nemeth, K., Brennan, R., Bradley, C., Krafcik, F., Rodriguez-Galindo, C., Wilson, M., Xiong, S., Lozano, G., Sage, J., Fu, L., Louhibi, L., Trimarchi, J., Pani, A., Smeyne, R., Johnson, D., Dyer, M. A.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">20 (2)</span><span class=\"pages\">: 260-275</span>\n</span>","chicagoCitation":"McEvoy, Justina, Jacqueline Flores-Otero, Jiakun Zhang, Katie Nemeth, Rachel Brennan, et al. 2011. ?Coexpression of Normally Incompatible Developmental Pathways in Retinoblastoma Genesis.? <i>CANCER CELL</i> 20 (2). CELL PRESS: 260?75.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">It is widely believed that the molecular and cellular features of a tumor reflect its cell of origin and can thus provide clues about treatment targets. The retinoblastoma cell of origin has been debated for over a century. Here, we report that human and mouse retinoblastomas have molecular, cellular, and neurochemical features of multiple cell classes, principally amacrine/horizontal interneurons, retinal progenitor cells, and photoreceptors. Importantly, single-cell gene expression array analysis showed that these multiple cell type-specific developmental programs are coexpressed in individual retinoblastoma cells, which creates a progenitor/neuronal hybrid cell. Furthermore, neurotransmitter receptors, transporters, and biosynthetic enzymes are expressed in human retinoblastoma, and targeted disruption of these pathways reduces retinoblastoma growth inÂ vivo and inÂ vitro.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.ccr.2011.07.005\">DOI 10.1016/j.ccr.2011.07.005</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000294099700013\">Web of Science ID 000294099700013</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21840489\">PubMedID 21840489</a></span>\n  </p>","text":"Abstract:\nIt is widely believed that the molecular and cellular features of a tumor reflect its cell of origin and can thus provide clues about treatment targets. The retinoblastoma cell of origin has been debated for over a century. Here, we report that human and mouse retinoblastomas have molecular, cellular, and neurochemical features of multiple cell classes, principally amacrine/horizontal interneurons, retinal progenitor cells, and photoreceptors. Importantly, single-cell gene expression array analysis showed that these multiple cell type-specific developmental programs are coexpressed in individual retinoblastoma cells, which creates a progenitor/neuronal hybrid cell. Furthermore, neurotransmitter receptors, transporters, and biosynthetic enzymes are expressed in human retinoblastoma, and targeted disruption of these pathways reduces retinoblastoma growth inÂ vivo and inÂ vitro.\nDOI: 10.1016/j.ccr.2011.07.005, https://dx.doi.org/10.1016/j.ccr.2011.07.005\nWeb of Science ID: 000294099700013, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000294099700013\nPubMedID: 21840489, https://www.ncbi.nlm.nih.gov/pubmed/21840489"},"doiId":"10.1016/j.ccr.2011.07.005","doiUrl":"https://dx.doi.org/10.1016/j.ccr.2011.07.005","featured":false,"firstPublished":{"html":"Aug 16, 2011 00:00:00 AM","text":"Aug 16, 2011 00:00:00 AM","value":"2011-08-16T00:00:00.000-07:00"},"mlaCitation":"McEvoy, Justina et al. ?Coexpression of Normally Incompatible Developmental Pathways in Retinoblastoma Genesis.? <i>CANCER CELL</i> 20.2 (2011): 260?275. Print.","pubMedId":"21840489","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21840489","publicationId":179727,"title":"Coexpression of Normally Incompatible Developmental Pathways in Retinoblastoma Genesis","type":"article","webOfScienceId":"000294099700013","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000294099700013"},{"abstract":"Retinoblastoma is a rare pediatric cancer that has served as a paradigm to investigate the mechanisms of tumorigenesis. In this issue of Genes & Development, Conkrite and colleagues (pp. 1734-1745) found high levels of the miR-17~92 and miR-106b-25 microRNAs in primary retinoblastomas and show that overexpression of miR-17~92 accelerates retinoblastoma development in mice by promoting proliferation, in part by reducing expression of the cell cycle inhibitor p21. These experiments identify the RB/miR-17~92/p21 axis as a critical regulator of retinoblastoma tumorigenesis and potentially many other cancers.","apaCitation":"Sage, J., &amp; Ventura, A. (2011). miR than meets the eye. <i>GENES &amp; DEVELOPMENT</i>, <i>25</i>(16), 1663?1667.","capCitation":"<span class=\"title\">\n  <span>miR than meets the eye</span>\n  <i>GENES &amp; DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Sage, J., Ventura, A.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">25 (16)</span><span class=\"pages\">: 1663-1667</span>\n</span>","chicagoCitation":"Sage, Julien, and Andrea Ventura. 2011. ?MiR than Meets the Eye.? <i>GENES &amp; DEVELOPMENT</i> 25 (16). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 1663?67.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Retinoblastoma is a rare pediatric cancer that has served as a paradigm to investigate the mechanisms of tumorigenesis. In this issue of Genes & Development, Conkrite and colleagues (pp. 1734-1745) found high levels of the miR-17~92 and miR-106b-25 microRNAs in primary retinoblastomas and show that overexpression of miR-17~92 accelerates retinoblastoma development in mice by promoting proliferation, in part by reducing expression of the cell cycle inhibitor p21. These experiments identify the RB/miR-17~92/p21 axis as a critical regulator of retinoblastoma tumorigenesis and potentially many other cancers.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.17454011\">DOI 10.1101/gad.17454011</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000294003800001\">Web of Science ID 000294003800001</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21852531\">PubMedID 21852531</a></span>\n  </p>","text":"Abstract:\nRetinoblastoma is a rare pediatric cancer that has served as a paradigm to investigate the mechanisms of tumorigenesis. In this issue of Genes & Development, Conkrite and colleagues (pp. 1734-1745) found high levels of the miR-17~92 and miR-106b-25 microRNAs in primary retinoblastomas and show that overexpression of miR-17~92 accelerates retinoblastoma development in mice by promoting proliferation, in part by reducing expression of the cell cycle inhibitor p21. These experiments identify the RB/miR-17~92/p21 axis as a critical regulator of retinoblastoma tumorigenesis and potentially many other cancers.\nDOI: 10.1101/gad.17454011, https://dx.doi.org/10.1101/gad.17454011\nWeb of Science ID: 000294003800001, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000294003800001\nPubMedID: 21852531, https://www.ncbi.nlm.nih.gov/pubmed/21852531"},"doiId":"10.1101/gad.17454011","doiUrl":"https://dx.doi.org/10.1101/gad.17454011","featured":true,"firstPublished":{"html":"Aug 15, 2011 00:00:00 AM","text":"Aug 15, 2011 00:00:00 AM","value":"2011-08-15T00:00:00.000-07:00"},"mlaCitation":"Sage, Julien, and Andrea Ventura. ?MiR than Meets the Eye.? <i>GENES &amp; DEVELOPMENT</i> 25.16 (2011): 1663?1667. Print.","pubMedId":"21852531","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21852531","publicationId":111910,"title":"miR than meets the eye","type":"article","webOfScienceId":"000294003800001","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000294003800001"},{"abstract":"Small cell lung carcinoma (SCLC) is a neuroendocrine subtype of lung cancer that affects more than 200,000 people worldwide every year with a very high mortality rate. Here, we used a mouse genetics approach to characterize the cell of origin for SCLC; in this mouse model, tumors are initiated by the deletion of the Rb and p53 tumor suppressor genes in the lung epithelium of adult mice. We found that mouse SCLCs often arise in the lung epithelium, where neuroendocrine cells are located, and that the majority of early lesions were composed of proliferating neuroendocrine cells. In addition, mice in which Rb and p53 are deleted in a variety of non-neuroendocrine lung epithelial cells did not develop SCLC. These data indicate that SCLC likely arises from neuroendocrine cells in the lung.","apaCitation":"Park, K.-S., Liang, M.-C., Raiser, D. M., Zamponi, R., Roach, R. R., Curtis, S. J., ? Kim, C. F. (2011). Characterization of the cell of origin for small cell lung cancer. <i>CELL CYCLE</i>, <i>10</i>(16), 2806?2815.","capCitation":"<span class=\"title\">\n  <span>Characterization of the cell of origin for small cell lung cancer</span>\n  <i>CELL CYCLE</i>\n</span>\n<span class=\"authors\">Park, K., Liang, M., Raiser, D. M., Zamponi, R., Roach, R. R., Curtis, S. J., Walton, Z., Schaffer, B. E., Roake, C. M., Zmoos, A., Kriegel, C., Wong, K., Sage, J., Kim, C. F.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">10 (16)</span><span class=\"pages\">: 2806-2815</span>\n</span>","chicagoCitation":"Park, Kwon-Sik, Mei-Chih Liang, David M. Raiser, Raffaella Zamponi, Rebecca R. Roach, et al. 2011. ?Characterization of the Cell of Origin for Small Cell Lung Cancer.? <i>CELL CYCLE</i> 10 (16). LANDES BIOSCIENCE: 2806?15.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small cell lung carcinoma (SCLC) is a neuroendocrine subtype of lung cancer that affects more than 200,000 people worldwide every year with a very high mortality rate. Here, we used a mouse genetics approach to characterize the cell of origin for SCLC; in this mouse model, tumors are initiated by the deletion of the Rb and p53 tumor suppressor genes in the lung epithelium of adult mice. We found that mouse SCLCs often arise in the lung epithelium, where neuroendocrine cells are located, and that the majority of early lesions were composed of proliferating neuroendocrine cells. In addition, mice in which Rb and p53 are deleted in a variety of non-neuroendocrine lung epithelial cells did not develop SCLC. These data indicate that SCLC likely arises from neuroendocrine cells in the lung.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.4161/cc.10.16.17012\">DOI 10.4161/cc.10.16.17012</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000294155600035\">Web of Science ID 000294155600035</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21822053\">PubMedID 21822053</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3219544\">PubMedCentralID PMC3219544</a></span>\n  </p>","text":"Abstract:\nSmall cell lung carcinoma (SCLC) is a neuroendocrine subtype of lung cancer that affects more than 200,000 people worldwide every year with a very high mortality rate. Here, we used a mouse genetics approach to characterize the cell of origin for SCLC; in this mouse model, tumors are initiated by the deletion of the Rb and p53 tumor suppressor genes in the lung epithelium of adult mice. We found that mouse SCLCs often arise in the lung epithelium, where neuroendocrine cells are located, and that the majority of early lesions were composed of proliferating neuroendocrine cells. In addition, mice in which Rb and p53 are deleted in a variety of non-neuroendocrine lung epithelial cells did not develop SCLC. These data indicate that SCLC likely arises from neuroendocrine cells in the lung.\nDOI: 10.4161/cc.10.16.17012, https://dx.doi.org/10.4161/cc.10.16.17012\nWeb of Science ID: 000294155600035, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000294155600035\nPubMedID: 21822053, https://www.ncbi.nlm.nih.gov/pubmed/21822053\nPubMedCentralID: PMC3219544, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3219544"},"doiId":"10.4161/cc.10.16.17012","doiUrl":"https://dx.doi.org/10.4161/cc.10.16.17012","featured":false,"firstPublished":{"html":"Aug 15, 2011 00:00:00 AM","text":"Aug 15, 2011 00:00:00 AM","value":"2011-08-15T00:00:00.000-07:00"},"mlaCitation":"Park, Kwon-Sik et al. ?Characterization of the Cell of Origin for Small Cell Lung Cancer.? <i>CELL CYCLE</i> 10.16 (2011): 2806?2815. Print.","pubMedId":"21822053","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21822053","publicationId":111550,"title":"Characterization of the cell of origin for small cell lung cancer","type":"article","webOfScienceId":"000294155600035","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000294155600035"},{"abstract":"Inactivation of the RB tumor suppressor and activation of the MYC family of oncogenes are frequent events in a large spectrum of human cancers. Loss of RB function and MYC activation are thought to control both overlapping and distinct cellular processes during cell cycle progression. However, how these two major cancer genes functionally interact during tumorigenesis is still unclear. Here, we sought to test whether loss of RB function would affect cancer development in a mouse model of c-MYC-induced hepatocellular carcinoma (HCC), a deadly cancer type in which RB is frequently inactivated and c-MYC often activated. We found that RB inactivation has minimal effects on the cell cycle, cell death, and differentiation features of liver tumors driven by increased levels of c-MYC. However, combined loss of RB and activation of c-MYC led to an increase in polyploidy in mature hepatocytes before the development of tumors. There was a trend for decreased survival in double mutant animals compared to mice developing c-MYC-induced tumors. Thus, loss of RB function does not provide a proliferative advantage to c-MYC-expressing HCC cells but the RB and c-MYC pathways may cooperate to control the polyploidy of mature hepatocytes.","apaCitation":"Saddic, L. A., Wirt, S., Vogel, H., Felsher, D. W., &amp; Sage, J. (2011). Functional Interactions between Retinoblastoma and c-MYC in a Mouse Model of Hepatocellular Carcinoma. <i>PLOS ONE</i>, <i>6</i>(5).","capCitation":"<span class=\"title\">\n  <span>Functional Interactions between Retinoblastoma and c-MYC in a Mouse Model of Hepatocellular Carcinoma</span>\n  <i>PLOS ONE</i>\n</span>\n<span class=\"authors\">Saddic, L. A., Wirt, S., Vogel, H., Felsher, D. W., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">6 (5)</span>\n</span>","chicagoCitation":"Saddic, Louis A., Stacey Wirt, Hannes Vogel, Dean W. Felsher, and Julien Sage. 2011. ?Functional Interactions between Retinoblastoma and c-MYC in a Mouse Model of Hepatocellular Carcinoma.? <i>PLOS ONE</i> 6 (5). PUBLIC LIBRARY SCIENCE.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Inactivation of the RB tumor suppressor and activation of the MYC family of oncogenes are frequent events in a large spectrum of human cancers. Loss of RB function and MYC activation are thought to control both overlapping and distinct cellular processes during cell cycle progression. However, how these two major cancer genes functionally interact during tumorigenesis is still unclear. Here, we sought to test whether loss of RB function would affect cancer development in a mouse model of c-MYC-induced hepatocellular carcinoma (HCC), a deadly cancer type in which RB is frequently inactivated and c-MYC often activated. We found that RB inactivation has minimal effects on the cell cycle, cell death, and differentiation features of liver tumors driven by increased levels of c-MYC. However, combined loss of RB and activation of c-MYC led to an increase in polyploidy in mature hepatocytes before the development of tumors. There was a trend for decreased survival in double mutant animals compared to mice developing c-MYC-induced tumors. Thus, loss of RB function does not provide a proliferative advantage to c-MYC-expressing HCC cells but the RB and c-MYC pathways may cooperate to control the polyploidy of mature hepatocytes.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1371/journal.pone.0019758\">DOI 10.1371/journal.pone.0019758</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000290305600045\">Web of Science ID 000290305600045</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21573126\">PubMedID 21573126</a></span>\n  </p>","text":"Abstract:\nInactivation of the RB tumor suppressor and activation of the MYC family of oncogenes are frequent events in a large spectrum of human cancers. Loss of RB function and MYC activation are thought to control both overlapping and distinct cellular processes during cell cycle progression. However, how these two major cancer genes functionally interact during tumorigenesis is still unclear. Here, we sought to test whether loss of RB function would affect cancer development in a mouse model of c-MYC-induced hepatocellular carcinoma (HCC), a deadly cancer type in which RB is frequently inactivated and c-MYC often activated. We found that RB inactivation has minimal effects on the cell cycle, cell death, and differentiation features of liver tumors driven by increased levels of c-MYC. However, combined loss of RB and activation of c-MYC led to an increase in polyploidy in mature hepatocytes before the development of tumors. There was a trend for decreased survival in double mutant animals compared to mice developing c-MYC-induced tumors. Thus, loss of RB function does not provide a proliferative advantage to c-MYC-expressing HCC cells but the RB and c-MYC pathways may cooperate to control the polyploidy of mature hepatocytes.\nDOI: 10.1371/journal.pone.0019758, https://dx.doi.org/10.1371/journal.pone.0019758\nWeb of Science ID: 000290305600045, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000290305600045\nPubMedID: 21573126, https://www.ncbi.nlm.nih.gov/pubmed/21573126"},"doiId":"10.1371/journal.pone.0019758","doiUrl":"https://dx.doi.org/10.1371/journal.pone.0019758","featured":false,"firstPublished":{"html":"May 6, 2011 00:00:00 AM","text":"May 6, 2011 00:00:00 AM","value":"2011-05-06T00:00:00.000-07:00"},"mlaCitation":"Saddic, Louis A. et al. ?Functional Interactions between Retinoblastoma and c-MYC in a Mouse Model of Hepatocellular Carcinoma.? <i>PLOS ONE</i> 6.5 (2011): n. pag. Print.","pubMedId":"21573126","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21573126","publicationId":107867,"title":"Functional Interactions between Retinoblastoma and c-MYC in a Mouse Model of Hepatocellular Carcinoma","type":"article","webOfScienceId":"000290305600045","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000290305600045"},{"abstract":"Emerging evidence suggests that microRNAs (miRNAs), an abundant class of ?22-nucleotide small regulatory RNAs, play key roles in controlling the post-transcriptional genetic programs in stem and progenitor cells. Here we systematically examined miRNA expression profiles in various adult tissue-specific stem cells and their differentiated counterparts. These analyses revealed miRNA programs that are common or unique to blood, muscle, and neural stem cell populations and miRNA signatures that mark the transitions from self-renewing and quiescent stem cells to proliferative and differentiating progenitor cells. Moreover, we identified a stem/progenitor transition miRNA (SPT-miRNA) signature that predicts the effects of genetic perturbations, such as loss of PTEN and the Rb family, AML1-ETO9a expression, and MLL-AF10 transformation, on self-renewal and proliferation potentials of mutant stem/progenitor cells. We showed that some of the SPT-miRNAs control the self-renewal of embryonic stem cells and the reconstitution potential of hematopoietic stem cells (HSCs). Finally, we demonstrated that SPT-miRNAs coordinately regulate genes that are known to play roles in controlling HSC self-renewal, such as Hoxb6 and Hoxa4. Together, these analyses reveal the miRNA programs that may control key processes in normal and aberrant stem and progenitor cells, setting the foundations for dissecting post-transcriptional regulatory networks in stem cells.","apaCitation":"Arnold, C. P., Tan, R., Zhou, B., Yue, S.-B., Schaffert, S., Biggs, J. R., ? Chen, C.-Z. (2011). MicroRNA programs in normal and aberrant stem and progenitor cells. <i>GENOME RESEARCH</i>, <i>21</i>(5), 798?810.","capCitation":"<span class=\"title\">\n  <span>MicroRNA programs in normal and aberrant stem and progenitor cells</span>\n  <i>GENOME RESEARCH</i>\n</span>\n<span class=\"authors\">Arnold, C. P., Tan, R., Zhou, B., Yue, S., Schaffert, S., Biggs, J. R., Doyonnas, R., Lo, M., Perry, J. M., Renault, V. M., Sacco, A., Somervaille, T., Viatour, P., Brunet, A., Cleary, M. L., Li, L., Sage, J., Zhang, D., Blau, H. M., Chen, C., Chen, C.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">21 (5)</span><span class=\"pages\">: 798-810</span>\n</span>","chicagoCitation":"Arnold, Christopher P., Ruoying Tan, Baiyu Zhou, Si-Biao Yue, Steven Schaffert, et al. 2011. ?MicroRNA Programs in Normal and Aberrant Stem and Progenitor Cells.? <i>GENOME RESEARCH</i> 21 (5). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 798?810.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Emerging evidence suggests that microRNAs (miRNAs), an abundant class of ?22-nucleotide small regulatory RNAs, play key roles in controlling the post-transcriptional genetic programs in stem and progenitor cells. Here we systematically examined miRNA expression profiles in various adult tissue-specific stem cells and their differentiated counterparts. These analyses revealed miRNA programs that are common or unique to blood, muscle, and neural stem cell populations and miRNA signatures that mark the transitions from self-renewing and quiescent stem cells to proliferative and differentiating progenitor cells. Moreover, we identified a stem/progenitor transition miRNA (SPT-miRNA) signature that predicts the effects of genetic perturbations, such as loss of PTEN and the Rb family, AML1-ETO9a expression, and MLL-AF10 transformation, on self-renewal and proliferation potentials of mutant stem/progenitor cells. We showed that some of the SPT-miRNAs control the self-renewal of embryonic stem cells and the reconstitution potential of hematopoietic stem cells (HSCs). Finally, we demonstrated that SPT-miRNAs coordinately regulate genes that are known to play roles in controlling HSC self-renewal, such as Hoxb6 and Hoxa4. Together, these analyses reveal the miRNA programs that may control key processes in normal and aberrant stem and progenitor cells, setting the foundations for dissecting post-transcriptional regulatory networks in stem cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gr.111385.110\">DOI 10.1101/gr.111385.110</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000290088000018\">Web of Science ID 000290088000018</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21451113\">PubMedID 21451113</a></span>\n  </p>","text":"Abstract:\nEmerging evidence suggests that microRNAs (miRNAs), an abundant class of ?22-nucleotide small regulatory RNAs, play key roles in controlling the post-transcriptional genetic programs in stem and progenitor cells. Here we systematically examined miRNA expression profiles in various adult tissue-specific stem cells and their differentiated counterparts. These analyses revealed miRNA programs that are common or unique to blood, muscle, and neural stem cell populations and miRNA signatures that mark the transitions from self-renewing and quiescent stem cells to proliferative and differentiating progenitor cells. Moreover, we identified a stem/progenitor transition miRNA (SPT-miRNA) signature that predicts the effects of genetic perturbations, such as loss of PTEN and the Rb family, AML1-ETO9a expression, and MLL-AF10 transformation, on self-renewal and proliferation potentials of mutant stem/progenitor cells. We showed that some of the SPT-miRNAs control the self-renewal of embryonic stem cells and the reconstitution potential of hematopoietic stem cells (HSCs). Finally, we demonstrated that SPT-miRNAs coordinately regulate genes that are known to play roles in controlling HSC self-renewal, such as Hoxb6 and Hoxa4. Together, these analyses reveal the miRNA programs that may control key processes in normal and aberrant stem and progenitor cells, setting the foundations for dissecting post-transcriptional regulatory networks in stem cells.\nDOI: 10.1101/gr.111385.110, https://dx.doi.org/10.1101/gr.111385.110\nWeb of Science ID: 000290088000018, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000290088000018\nPubMedID: 21451113, https://www.ncbi.nlm.nih.gov/pubmed/21451113"},"doiId":"10.1101/gr.111385.110","doiUrl":"https://dx.doi.org/10.1101/gr.111385.110","featured":false,"firstPublished":{"html":"May 1, 2011 00:00:00 AM","text":"May 1, 2011 00:00:00 AM","value":"2011-05-01T00:00:00.000-07:00"},"mlaCitation":"Arnold, Christopher P. et al. ?MicroRNA Programs in Normal and Aberrant Stem and Progenitor Cells.? <i>GENOME RESEARCH</i> 21.5 (2011): 798?810. Print.","pubMedId":"21451113","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21451113","publicationId":105633,"title":"MicroRNA programs in normal and aberrant stem and progenitor cells","type":"article","webOfScienceId":"000290088000018","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000290088000018"},{"abstract":"The integrity of the retinoblastoma tumor suppressor (RB) pathway is critical for restraining inappropriate proliferation and suppressing tumor development in a plethora of tissues. Here adenovirus-mediated RB deletion in the liver of adult mice led to DNA replication in the absence of productive mitotic condensation. The replication induced by RB loss was E2F-mediated and associated with the induction of DNA damage and a nontranscriptional G2/M checkpoint that targeted the accumulation of Cyclin B1. In the context of RB deletion or E2F activation, there was an increase in hepatocyte ploidy that was accompanied by hyperphysiological assembly of prereplication complexes. In keeping with this dysregulation, initiation of DNA replication was readily observed in hepatocytes that were phenotypically in G2/M. Under such conditions, uncoupling of replication initiation from mitotic progression led to altered genome ploidy in the liver. Interestingly, these findings in hepatocytes were not recapitulated in the basally proliferative tissues of the gastrointestinal tract, where RB deletion, while increasing DNA replication, did not lead to a profound uncoupling from mitosis. Combined, these findings demonstrate the critical role of RB in controlling cell-cycle transitions and underscore the importance of intrinsic tissue environments in resultant phenotypes.","apaCitation":"Bourgo, R. J., Ehmer, U., Sage, J., &amp; Knudsen, E. S. (2011). RB deletion disrupts coordination between DNA replication licensing and mitotic entry in vivo. <i>MOLECULAR BIOLOGY OF THE CELL</i>, <i>22</i>(7), 931?939.","capCitation":"<span class=\"title\">\n  <span>RB deletion disrupts coordination between DNA replication licensing and mitotic entry in vivo</span>\n  <i>MOLECULAR BIOLOGY OF THE CELL</i>\n</span>\n<span class=\"authors\">Bourgo, R. J., Ehmer, U., Sage, J., Knudsen, E. S.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">22 (7)</span><span class=\"pages\">: 931-939</span>\n</span>","chicagoCitation":"Bourgo, Ryan J., Ursula Ehmer, Julien Sage, and Erik S. Knudsen. 2011. ?RB Deletion Disrupts Coordination between DNA Replication Licensing and Mitotic Entry in Vivo.? <i>MOLECULAR BIOLOGY OF THE CELL</i> 22 (7). AMER SOC CELL BIOLOGY: 931?39.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The integrity of the retinoblastoma tumor suppressor (RB) pathway is critical for restraining inappropriate proliferation and suppressing tumor development in a plethora of tissues. Here adenovirus-mediated RB deletion in the liver of adult mice led to DNA replication in the absence of productive mitotic condensation. The replication induced by RB loss was E2F-mediated and associated with the induction of DNA damage and a nontranscriptional G2/M checkpoint that targeted the accumulation of Cyclin B1. In the context of RB deletion or E2F activation, there was an increase in hepatocyte ploidy that was accompanied by hyperphysiological assembly of prereplication complexes. In keeping with this dysregulation, initiation of DNA replication was readily observed in hepatocytes that were phenotypically in G2/M. Under such conditions, uncoupling of replication initiation from mitotic progression led to altered genome ploidy in the liver. Interestingly, these findings in hepatocytes were not recapitulated in the basally proliferative tissues of the gastrointestinal tract, where RB deletion, while increasing DNA replication, did not lead to a profound uncoupling from mitosis. Combined, these findings demonstrate the critical role of RB in controlling cell-cycle transitions and underscore the importance of intrinsic tissue environments in resultant phenotypes.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1091/mbc.E10-11-0895\">DOI 10.1091/mbc.E10-11-0895</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000288995800005\">Web of Science ID 000288995800005</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21289097\">PubMedID 21289097</a></span>\n  </p>","text":"Abstract:\nThe integrity of the retinoblastoma tumor suppressor (RB) pathway is critical for restraining inappropriate proliferation and suppressing tumor development in a plethora of tissues. Here adenovirus-mediated RB deletion in the liver of adult mice led to DNA replication in the absence of productive mitotic condensation. The replication induced by RB loss was E2F-mediated and associated with the induction of DNA damage and a nontranscriptional G2/M checkpoint that targeted the accumulation of Cyclin B1. In the context of RB deletion or E2F activation, there was an increase in hepatocyte ploidy that was accompanied by hyperphysiological assembly of prereplication complexes. In keeping with this dysregulation, initiation of DNA replication was readily observed in hepatocytes that were phenotypically in G2/M. Under such conditions, uncoupling of replication initiation from mitotic progression led to altered genome ploidy in the liver. Interestingly, these findings in hepatocytes were not recapitulated in the basally proliferative tissues of the gastrointestinal tract, where RB deletion, while increasing DNA replication, did not lead to a profound uncoupling from mitosis. Combined, these findings demonstrate the critical role of RB in controlling cell-cycle transitions and underscore the importance of intrinsic tissue environments in resultant phenotypes.\nDOI: 10.1091/mbc.E10-11-0895, https://dx.doi.org/10.1091/mbc.E10-11-0895\nWeb of Science ID: 000288995800005, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000288995800005\nPubMedID: 21289097, https://www.ncbi.nlm.nih.gov/pubmed/21289097"},"doiId":"10.1091/mbc.E10-11-0895","doiUrl":"https://dx.doi.org/10.1091/mbc.E10-11-0895","featured":false,"firstPublished":{"html":"Apr 1, 2011 00:00:00 AM","text":"Apr 1, 2011 00:00:00 AM","value":"2011-04-01T00:00:00.000-07:00"},"mlaCitation":"Bourgo, Ryan J. et al. ?RB Deletion Disrupts Coordination between DNA Replication Licensing and Mitotic Entry in Vivo.? <i>MOLECULAR BIOLOGY OF THE CELL</i> 22.7 (2011): 931?939. Print.","pubMedId":"21289097","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21289097","publicationId":102830,"title":"RB deletion disrupts coordination between DNA replication licensing and mitotic entry in vivo","type":"article","webOfScienceId":"000288995800005","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000288995800005"},{"abstract":"The retinoblastoma tumor suppressor (RB) is a central cell cycle regulator and tumor suppressor. RB cellular functions are known to be regulated by a diversity of post-translational modifications such as phosphorylation and acetylation, raising the possibility that RB may also be methylated in cells. Here we demonstrate that RB can be methylated by SMYD2 at lysine 860, a highly conserved and novel site of modification. This methylation event occurs in vitro and in cells, and it is regulated during cell cycle progression, cellular differentiation, and in response to DNA damage. Furthermore, we show that RB monomethylation at lysine 860 provides a direct binding site for the methyl-binding domain of the transcriptional repressor L3MBTL1. These results support the idea that a code of post-translational modifications exists for RB and helps guide its functions in mammalian cells.","apaCitation":"Saddic, L. A., West, L. E., Aslanian, A., Yates, J. R., Rubin, S. M., Gozani, O., &amp; Sage, J. (2010). Methylation of the Retinoblastoma Tumor Suppressor by SMYD2. <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i>, <i>285</i>(48), 37733?37740.","capCitation":"<span class=\"title\">\n  <span>Methylation of the Retinoblastoma Tumor Suppressor by SMYD2</span>\n  <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i>\n</span>\n<span class=\"authors\">Saddic, L. A., West, L. E., Aslanian, A., Yates, J. R., Rubin, S. M., Gozani, O., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">285 (48)</span><span class=\"pages\">: 37733-37740</span>\n</span>","chicagoCitation":"Saddic, Louis A., Lisandra E. West, Aaron Aslanian, John R. Yates, Seth M. Rubin, et al. 2010. ?Methylation of the Retinoblastoma Tumor Suppressor by SMYD2.? <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i> 285 (48). AMER SOC BIOCHEMISTRY MOLECULAR BIOLOGY INC: 37733?40.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma tumor suppressor (RB) is a central cell cycle regulator and tumor suppressor. RB cellular functions are known to be regulated by a diversity of post-translational modifications such as phosphorylation and acetylation, raising the possibility that RB may also be methylated in cells. Here we demonstrate that RB can be methylated by SMYD2 at lysine 860, a highly conserved and novel site of modification. This methylation event occurs in vitro and in cells, and it is regulated during cell cycle progression, cellular differentiation, and in response to DNA damage. Furthermore, we show that RB monomethylation at lysine 860 provides a direct binding site for the methyl-binding domain of the transcriptional repressor L3MBTL1. These results support the idea that a code of post-translational modifications exists for RB and helps guide its functions in mammalian cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1074/jbc.M110.137612\">DOI 10.1074/jbc.M110.137612</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000284424000065\">Web of Science ID 000284424000065</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20870719\">PubMedID 20870719</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma tumor suppressor (RB) is a central cell cycle regulator and tumor suppressor. RB cellular functions are known to be regulated by a diversity of post-translational modifications such as phosphorylation and acetylation, raising the possibility that RB may also be methylated in cells. Here we demonstrate that RB can be methylated by SMYD2 at lysine 860, a highly conserved and novel site of modification. This methylation event occurs in vitro and in cells, and it is regulated during cell cycle progression, cellular differentiation, and in response to DNA damage. Furthermore, we show that RB monomethylation at lysine 860 provides a direct binding site for the methyl-binding domain of the transcriptional repressor L3MBTL1. These results support the idea that a code of post-translational modifications exists for RB and helps guide its functions in mammalian cells.\nDOI: 10.1074/jbc.M110.137612, https://dx.doi.org/10.1074/jbc.M110.137612\nWeb of Science ID: 000284424000065, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000284424000065\nPubMedID: 20870719, https://www.ncbi.nlm.nih.gov/pubmed/20870719"},"doiId":"10.1074/jbc.M110.137612","doiUrl":"https://dx.doi.org/10.1074/jbc.M110.137612","featured":true,"firstPublished":{"html":"Nov 26, 2010 00:00:00 AM","text":"Nov 26, 2010 00:00:00 AM","value":"2010-11-26T00:00:00.000-08:00"},"mlaCitation":"Saddic, Louis A. et al. ?Methylation of the Retinoblastoma Tumor Suppressor by SMYD2.? <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i> 285.48 (2010): 37733?37740. Print.","pubMedId":"20870719","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20870719","publicationId":96868,"title":"Methylation of the Retinoblastoma Tumor Suppressor by SMYD2","type":"article","webOfScienceId":"000284424000065","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000284424000065"},{"abstract":"The ability of progenitor cells to exit the cell cycle is essential for proper embryonic development and homeostasis, but the mechanisms governing cell cycle exit are still not fully understood. Here, we tested the requirement for the retinoblastoma (Rb) protein and its family members p107 and p130 in G0/G1 arrest and differentiation in mammalian cells. We found that Rb family triple knockout (TKO) mouse embryos survive until days 9-11 of gestation. Strikingly, some TKO cells, including in epithelial and neural lineages, are able to exit the cell cycle in G0/G1 and differentiate in teratomas and in culture. This ability of TKO cells to arrest in G0/G1 is associated with the repression of key E2F target genes. Thus, G1 arrest is not always dependent on Rb family members, which illustrates the robustness of cell cycle regulatory networks during differentiation and allows for the identification of candidate pathways to inhibit the expansion of cancer cells with mutations in the Rb pathway.","apaCitation":"Wirt, S. E., Adler, A. S., Gebala, V., Weimann, J. M., Schaffer, B. E., Saddic, L. A., ? Sage, J. (2010). G1 arrest and differentiation can occur independently of Rb family function. <i>JOURNAL OF CELL BIOLOGY</i>, <i>191</i>(4), 809?825.","capCitation":"<span class=\"title\">\n  <span>G1 arrest and differentiation can occur independently of Rb family function</span>\n  <i>JOURNAL OF CELL BIOLOGY</i>\n</span>\n<span class=\"authors\">Wirt, S. E., Adler, A. S., Gebala, V., Weimann, J. M., Schaffer, B. E., Saddic, L. A., Viatour, P., Vogel, H., Chang, H. Y., Meissner, A., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">191 (4)</span><span class=\"pages\">: 809-825</span>\n</span>","chicagoCitation":"Wirt, Stacey E., Adam S. Adler, Veronique Gebala, James M. Weimann, Bethany E. Schaffer, et al. 2010. ?G1 Arrest and Differentiation Can Occur Independently of Rb Family Function.? <i>JOURNAL OF CELL BIOLOGY</i> 191 (4). ROCKEFELLER UNIV PRESS: 809?25.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The ability of progenitor cells to exit the cell cycle is essential for proper embryonic development and homeostasis, but the mechanisms governing cell cycle exit are still not fully understood. Here, we tested the requirement for the retinoblastoma (Rb) protein and its family members p107 and p130 in G0/G1 arrest and differentiation in mammalian cells. We found that Rb family triple knockout (TKO) mouse embryos survive until days 9-11 of gestation. Strikingly, some TKO cells, including in epithelial and neural lineages, are able to exit the cell cycle in G0/G1 and differentiate in teratomas and in culture. This ability of TKO cells to arrest in G0/G1 is associated with the repression of key E2F target genes. Thus, G1 arrest is not always dependent on Rb family members, which illustrates the robustness of cell cycle regulatory networks during differentiation and allows for the identification of candidate pathways to inhibit the expansion of cancer cells with mutations in the Rb pathway.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1083/jcb.201003048\">DOI 10.1083/jcb.201003048</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000284737200014\">Web of Science ID 000284737200014</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21059851\">PubMedID 21059851</a></span>\n  </p>","text":"Abstract:\nThe ability of progenitor cells to exit the cell cycle is essential for proper embryonic development and homeostasis, but the mechanisms governing cell cycle exit are still not fully understood. Here, we tested the requirement for the retinoblastoma (Rb) protein and its family members p107 and p130 in G0/G1 arrest and differentiation in mammalian cells. We found that Rb family triple knockout (TKO) mouse embryos survive until days 9-11 of gestation. Strikingly, some TKO cells, including in epithelial and neural lineages, are able to exit the cell cycle in G0/G1 and differentiate in teratomas and in culture. This ability of TKO cells to arrest in G0/G1 is associated with the repression of key E2F target genes. Thus, G1 arrest is not always dependent on Rb family members, which illustrates the robustness of cell cycle regulatory networks during differentiation and allows for the identification of candidate pathways to inhibit the expansion of cancer cells with mutations in the Rb pathway.\nDOI: 10.1083/jcb.201003048, https://dx.doi.org/10.1083/jcb.201003048\nWeb of Science ID: 000284737200014, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000284737200014\nPubMedID: 21059851, https://www.ncbi.nlm.nih.gov/pubmed/21059851"},"doiId":"10.1083/jcb.201003048","doiUrl":"https://dx.doi.org/10.1083/jcb.201003048","featured":true,"firstPublished":{"html":"Nov 15, 2010 00:00:00 AM","text":"Nov 15, 2010 00:00:00 AM","value":"2010-11-15T00:00:00.000-08:00"},"mlaCitation":"Wirt, Stacey E. et al. ?G1 Arrest and Differentiation Can Occur Independently of Rb Family Function.? <i>JOURNAL OF CELL BIOLOGY</i> 191.4 (2010): 809?825. Print.","pubMedId":"21059851","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21059851","publicationId":98964,"title":"G1 arrest and differentiation can occur independently of Rb family function","type":"article","webOfScienceId":"000284737200014","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000284737200014"},{"abstract":"An outstanding biological question is why tissue regeneration in mammals is limited, whereas urodele amphibians and teleost fish regenerate major structures, largely by cell cycle reentry. Upon inactivation of Rb, proliferation of postmitotic urodele skeletal muscle is induced, whereas in mammalian muscle this mechanism does not exist. We postulated that a tumor suppressor present in mammals but absent in regenerative vertebrates, the Ink4a product ARF (alternative reading frame), is a regeneration suppressor. Concomitant inactivation of Arf and Rb led to mammalian muscle cell cycle reentry, loss of differentiation properties, and upregulation of cytokinetic machinery. Single postmitotic myocytes were isolated by laser micro-dissection-catapulting, and transient suppression of Arf and Rb yielded myoblast colonies that retained the ability to differentiate and fuse into myofibers upon transplantation in vivo. These results show that differentiation of mammalian cells is reversed by inactivation of Arf and Rb and support the hypothesis that Arf evolved at the expense of regeneration.","apaCitation":"Pajcini, K. V., Corbel, S. Y., Sage, J., Pomerantz, J. H., &amp; Blau, H. M. (2010). Transient Inactivation of Rb and ARF Yields Regenerative Cells from Postmitotic Mammalian Muscle. <i>CELL STEM CELL</i>, <i>7</i>(2), 198?213.","capCitation":"<span class=\"title\">\n  <span>Transient Inactivation of Rb and ARF Yields Regenerative Cells from Postmitotic Mammalian Muscle</span>\n  <i>CELL STEM CELL</i>\n</span>\n<span class=\"authors\">Pajcini, K. V., Corbel, S. Y., Sage, J., Pomerantz, J. H., Blau, H. M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">7 (2)</span><span class=\"pages\">: 198-213</span>\n</span>","chicagoCitation":"Pajcini, Kostandin V., Stephane Y. Corbel, Julien Sage, Jason H. Pomerantz, and Helen M. Blau. 2010. ?Transient Inactivation of Rb and ARF Yields Regenerative Cells from Postmitotic Mammalian Muscle.? <i>CELL STEM CELL</i> 7 (2). CELL PRESS: 198?213.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">An outstanding biological question is why tissue regeneration in mammals is limited, whereas urodele amphibians and teleost fish regenerate major structures, largely by cell cycle reentry. Upon inactivation of Rb, proliferation of postmitotic urodele skeletal muscle is induced, whereas in mammalian muscle this mechanism does not exist. We postulated that a tumor suppressor present in mammals but absent in regenerative vertebrates, the Ink4a product ARF (alternative reading frame), is a regeneration suppressor. Concomitant inactivation of Arf and Rb led to mammalian muscle cell cycle reentry, loss of differentiation properties, and upregulation of cytokinetic machinery. Single postmitotic myocytes were isolated by laser micro-dissection-catapulting, and transient suppression of Arf and Rb yielded myoblast colonies that retained the ability to differentiate and fuse into myofibers upon transplantation in vivo. These results show that differentiation of mammalian cells is reversed by inactivation of Arf and Rb and support the hypothesis that Arf evolved at the expense of regeneration.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.stem.2010.05.022\">DOI 10.1016/j.stem.2010.05.022</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000281107400012\">Web of Science ID 000281107400012</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20682446\">PubMedID 20682446</a></span>\n  </p>","text":"Abstract:\nAn outstanding biological question is why tissue regeneration in mammals is limited, whereas urodele amphibians and teleost fish regenerate major structures, largely by cell cycle reentry. Upon inactivation of Rb, proliferation of postmitotic urodele skeletal muscle is induced, whereas in mammalian muscle this mechanism does not exist. We postulated that a tumor suppressor present in mammals but absent in regenerative vertebrates, the Ink4a product ARF (alternative reading frame), is a regeneration suppressor. Concomitant inactivation of Arf and Rb led to mammalian muscle cell cycle reentry, loss of differentiation properties, and upregulation of cytokinetic machinery. Single postmitotic myocytes were isolated by laser micro-dissection-catapulting, and transient suppression of Arf and Rb yielded myoblast colonies that retained the ability to differentiate and fuse into myofibers upon transplantation in vivo. These results show that differentiation of mammalian cells is reversed by inactivation of Arf and Rb and support the hypothesis that Arf evolved at the expense of regeneration.\nDOI: 10.1016/j.stem.2010.05.022, https://dx.doi.org/10.1016/j.stem.2010.05.022\nWeb of Science ID: 000281107400012, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000281107400012\nPubMedID: 20682446, https://www.ncbi.nlm.nih.gov/pubmed/20682446"},"doiId":"10.1016/j.stem.2010.05.022","doiUrl":"https://dx.doi.org/10.1016/j.stem.2010.05.022","featured":false,"firstPublished":{"html":"Aug 6, 2010 00:00:00 AM","text":"Aug 6, 2010 00:00:00 AM","value":"2010-08-06T00:00:00.000-07:00"},"mlaCitation":"Pajcini, Kostandin V. et al. ?Transient Inactivation of Rb and ARF Yields Regenerative Cells from Postmitotic Mammalian Muscle.? <i>CELL STEM CELL</i> 7.2 (2010): 198?213. Print.","pubMedId":"20682446","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20682446","publicationId":94994,"title":"Transient Inactivation of Rb and ARF Yields Regenerative Cells from Postmitotic Mammalian Muscle","type":"article","webOfScienceId":"000281107400012","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000281107400012"},{"abstract":"The retinoblastoma tumor suppressor RB is the downstream mediator of a cellular pathway that is thought to prevent cancer by controlling the ability of cells to enter or exit the cell cycle in G0/G1. Recently, however, accumulating evidence has suggested that RB, its family members p107 and p130, and their partners, the E2F family of transcription factors, may have important cellular functions beyond the G1/S transition of the cell cycle, including during DNA replication and at the transition into mitosis. In this issue of Genes & Development, three studies demonstrate a critical role for RB in proper chromosome condensation, centromeric function, and chromosome stability in mammalian cells, and link these cellular functions of RB to tumor suppression in mice. Here we discuss how transcriptional and post-transcriptional mechanisms under the control of the RB pathway ensure accurate progression through mitosis, thereby preventing cancer development.","apaCitation":"Sage, J., &amp; Straight, A. F. (2010). RB's original CIN? <i>GENES &amp; DEVELOPMENT</i>, <i>24</i>(13), 1329?1333.","capCitation":"<span class=\"title\">\n  <span>RB&#39;s original CIN?</span>\n  <i>GENES &amp; DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Sage, J., Straight, A. F.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">24 (13)</span><span class=\"pages\">: 1329-1333</span>\n</span>","chicagoCitation":"Sage, Julien, and Aaron F. Straight. 2010. ?RB's Original CIN?? <i>GENES &amp; DEVELOPMENT</i> 24 (13). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 1329?33.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma tumor suppressor RB is the downstream mediator of a cellular pathway that is thought to prevent cancer by controlling the ability of cells to enter or exit the cell cycle in G0/G1. Recently, however, accumulating evidence has suggested that RB, its family members p107 and p130, and their partners, the E2F family of transcription factors, may have important cellular functions beyond the G1/S transition of the cell cycle, including during DNA replication and at the transition into mitosis. In this issue of Genes & Development, three studies demonstrate a critical role for RB in proper chromosome condensation, centromeric function, and chromosome stability in mammalian cells, and link these cellular functions of RB to tumor suppression in mice. Here we discuss how transcriptional and post-transcriptional mechanisms under the control of the RB pathway ensure accurate progression through mitosis, thereby preventing cancer development.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.1948010\">DOI 10.1101/gad.1948010</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000279405000001\">Web of Science ID 000279405000001</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20551167\">PubMedID 20551167</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma tumor suppressor RB is the downstream mediator of a cellular pathway that is thought to prevent cancer by controlling the ability of cells to enter or exit the cell cycle in G0/G1. Recently, however, accumulating evidence has suggested that RB, its family members p107 and p130, and their partners, the E2F family of transcription factors, may have important cellular functions beyond the G1/S transition of the cell cycle, including during DNA replication and at the transition into mitosis. In this issue of Genes & Development, three studies demonstrate a critical role for RB in proper chromosome condensation, centromeric function, and chromosome stability in mammalian cells, and link these cellular functions of RB to tumor suppression in mice. Here we discuss how transcriptional and post-transcriptional mechanisms under the control of the RB pathway ensure accurate progression through mitosis, thereby preventing cancer development.\nDOI: 10.1101/gad.1948010, https://dx.doi.org/10.1101/gad.1948010\nWeb of Science ID: 000279405000001, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000279405000001\nPubMedID: 20551167, https://www.ncbi.nlm.nih.gov/pubmed/20551167"},"doiId":"10.1101/gad.1948010","doiUrl":"https://dx.doi.org/10.1101/gad.1948010","featured":false,"firstPublished":{"html":"Jul 1, 2010 00:00:00 AM","text":"Jul 1, 2010 00:00:00 AM","value":"2010-07-01T00:00:00.000-07:00"},"mlaCitation":"Sage, Julien, and Aaron F. Straight. ?RB's Original CIN?? <i>GENES &amp; DEVELOPMENT</i> 24.13 (2010): 1329?1333. Print.","pubMedId":"20551167","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20551167","publicationId":93482,"title":"RB's original CIN?","type":"article","webOfScienceId":"000279405000001","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000279405000001"},{"abstract":"The retinoblastoma tumor suppressor (Rb) is a potent and ubiquitously expressed cell cycle regulator, but patients with a germline Rb mutation develop a very specific tumor spectrum. This surprising observation raises the possibility that mechanisms that compensate for loss of Rb function are present or activated in many cell types. In particular, p107, a protein related to Rb, has been shown to functionally overlap for loss of Rb in several cellular contexts. To investigate the mechanisms underlying this functional redundancy between Rb and p107 in vivo, we used gene targeting in embryonic stem cells to engineer point mutations in two consensus E2F binding sites in the endogenous p107 promoter. Analysis of normal and mutant cells by gene expression and chromatin immunoprecipitation assays showed that members of the Rb and E2F families directly bound these two sites. Furthermore, we found that these two E2F sites controlled both the repression of p107 in quiescent cells and also its activation in cycling cells, as well as in Rb mutant cells. Cell cycle assays further indicated that activation of p107 transcription during S phase through the two E2F binding sites was critical for controlled cell cycle progression, uncovering a specific role for p107 to slow proliferation in mammalian cells. Direct transcriptional repression of p107 by Rb and E2F family members provides a molecular mechanism for a critical negative feedback loop during cell cycle progression and tumorigenesis. These experiments also suggest novel therapeutic strategies to increase the p107 levels in tumor cells.","apaCitation":"Burkhart, D. L., Wirt, S. E., Zmoos, A.-F., Kareta, M. S., &amp; Sage, J. (2010). Tandem E2F Binding Sites in the Promoter of the p107 Cell Cycle Regulator Control p107 Expression and Its Cellular Functions. <i>PLOS GENETICS</i>, <i>6</i>(6).","capCitation":"<span class=\"title\">\n  <span>Tandem E2F Binding Sites in the Promoter of the p107 Cell Cycle Regulator Control p107 Expression and Its Cellular Functions</span>\n  <i>PLOS GENETICS</i>\n</span>\n<span class=\"authors\">Burkhart, D. L., Wirt, S. E., Zmoos, A., Kareta, M. S., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">6 (6)</span>\n</span>","chicagoCitation":"Burkhart, Deborah L., Stacey E. Wirt, Anne-Flore Zmoos, Michael S. Kareta, and Julien Sage. 2010. ?Tandem E2F Binding Sites in the Promoter of the p107 Cell Cycle Regulator Control p107 Expression and Its Cellular Functions.? <i>PLOS GENETICS</i> 6 (6). PUBLIC LIBRARY SCIENCE.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma tumor suppressor (Rb) is a potent and ubiquitously expressed cell cycle regulator, but patients with a germline Rb mutation develop a very specific tumor spectrum. This surprising observation raises the possibility that mechanisms that compensate for loss of Rb function are present or activated in many cell types. In particular, p107, a protein related to Rb, has been shown to functionally overlap for loss of Rb in several cellular contexts. To investigate the mechanisms underlying this functional redundancy between Rb and p107 in vivo, we used gene targeting in embryonic stem cells to engineer point mutations in two consensus E2F binding sites in the endogenous p107 promoter. Analysis of normal and mutant cells by gene expression and chromatin immunoprecipitation assays showed that members of the Rb and E2F families directly bound these two sites. Furthermore, we found that these two E2F sites controlled both the repression of p107 in quiescent cells and also its activation in cycling cells, as well as in Rb mutant cells. Cell cycle assays further indicated that activation of p107 transcription during S phase through the two E2F binding sites was critical for controlled cell cycle progression, uncovering a specific role for p107 to slow proliferation in mammalian cells. Direct transcriptional repression of p107 by Rb and E2F family members provides a molecular mechanism for a critical negative feedback loop during cell cycle progression and tumorigenesis. These experiments also suggest novel therapeutic strategies to increase the p107 levels in tumor cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1371/journal.pgen.1001003\">DOI 10.1371/journal.pgen.1001003</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000279805200032\">Web of Science ID 000279805200032</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20585628\">PubMedID 20585628</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma tumor suppressor (Rb) is a potent and ubiquitously expressed cell cycle regulator, but patients with a germline Rb mutation develop a very specific tumor spectrum. This surprising observation raises the possibility that mechanisms that compensate for loss of Rb function are present or activated in many cell types. In particular, p107, a protein related to Rb, has been shown to functionally overlap for loss of Rb in several cellular contexts. To investigate the mechanisms underlying this functional redundancy between Rb and p107 in vivo, we used gene targeting in embryonic stem cells to engineer point mutations in two consensus E2F binding sites in the endogenous p107 promoter. Analysis of normal and mutant cells by gene expression and chromatin immunoprecipitation assays showed that members of the Rb and E2F families directly bound these two sites. Furthermore, we found that these two E2F sites controlled both the repression of p107 in quiescent cells and also its activation in cycling cells, as well as in Rb mutant cells. Cell cycle assays further indicated that activation of p107 transcription during S phase through the two E2F binding sites was critical for controlled cell cycle progression, uncovering a specific role for p107 to slow proliferation in mammalian cells. Direct transcriptional repression of p107 by Rb and E2F family members provides a molecular mechanism for a critical negative feedback loop during cell cycle progression and tumorigenesis. These experiments also suggest novel therapeutic strategies to increase the p107 levels in tumor cells.\nDOI: 10.1371/journal.pgen.1001003, https://dx.doi.org/10.1371/journal.pgen.1001003\nWeb of Science ID: 000279805200032, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000279805200032\nPubMedID: 20585628, https://www.ncbi.nlm.nih.gov/pubmed/20585628"},"doiId":"10.1371/journal.pgen.1001003","doiUrl":"https://dx.doi.org/10.1371/journal.pgen.1001003","featured":true,"firstPublished":{"html":"Jun 1, 2010 00:00:00 AM","text":"Jun 1, 2010 00:00:00 AM","value":"2010-06-01T00:00:00.000-07:00"},"mlaCitation":"Burkhart, Deborah L. et al. ?Tandem E2F Binding Sites in the Promoter of the p107 Cell Cycle Regulator Control p107 Expression and Its Cellular Functions.? <i>PLOS GENETICS</i> 6.6 (2010): n. pag. Print.","pubMedId":"20585628","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20585628","publicationId":93888,"title":"Tandem E2F Binding Sites in the Promoter of the p107 Cell Cycle Regulator Control p107 Expression and Its Cellular Functions","type":"article","webOfScienceId":"000279805200032","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000279805200032"},{"abstract":"Small-cell lung carcinoma (SCLC) is a neuroendocrine subtype of lung cancer. Although SCLC patients often initially respond to therapy, tumors nearly always recur, resulting in a 5-year survival rate of less than 10%. A mouse model has been developed based on the fact that the RB and p53 tumor suppressor genes are mutated in more than 90% of human SCLCs. Emerging evidence in patients and mouse models suggests that p130, a gene related to RB, may act as a tumor suppressor in SCLC cells. To test this idea, we used conditional mutant mice to delete p130 in combination with Rb and p53 in adult lung epithelial cells. We found that loss of p130 resulted in increased proliferation and significant acceleration of SCLC development in this triple-knockout mouse model. The histopathologic features of the triple-mutant mouse tumors closely resembled that of human SCLC. Genome-wide expression profiling experiments further showed that Rb/p53/p130-mutant mouse tumors were similar to human SCLC. These findings indicate that p130 plays a key tumor suppressor role in SCLC. Rb/p53/p130-mutant mice provide a novel preclinical mouse model to identify novel therapeutic targets against SCLC.","apaCitation":"Schaffer, B. E., Park, K.-S., Yiu, G., Conklin, J. F., Lin, C. W., Burkhart, D. L., ? Sage, J. (2010). Loss of p130 Accelerates Tumor Development in a Mouse Model for Human Small-Cell Lung Carcinoma. <i>CANCER RESEARCH</i>, <i>70</i>(10), 3877?3883.","capCitation":"<span class=\"title\">\n  <span>Loss of p130 Accelerates Tumor Development in a Mouse Model for Human Small-Cell Lung Carcinoma</span>\n  <i>CANCER RESEARCH</i>\n</span>\n<span class=\"authors\">Schaffer, B. E., Park, K., Yiu, G., Conklin, J. F., Lin, C., Burkhart, D. L., Karnezis, A. N., Sweet-Cordero, E. A., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">70 (10)</span><span class=\"pages\">: 3877-3883</span>\n</span>","chicagoCitation":"Schaffer, Bethany E., Kwon-Sik Park, Gloria Yiu, Jamie F. Conklin, ChenWei Lin, et al. 2010. ?Loss of p130 Accelerates Tumor Development in a Mouse Model for Human Small-Cell Lung Carcinoma.? <i>CANCER RESEARCH</i> 70 (10). AMER ASSOC CANCER RESEARCH: 3877?83.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small-cell lung carcinoma (SCLC) is a neuroendocrine subtype of lung cancer. Although SCLC patients often initially respond to therapy, tumors nearly always recur, resulting in a 5-year survival rate of less than 10%. A mouse model has been developed based on the fact that the RB and p53 tumor suppressor genes are mutated in more than 90% of human SCLCs. Emerging evidence in patients and mouse models suggests that p130, a gene related to RB, may act as a tumor suppressor in SCLC cells. To test this idea, we used conditional mutant mice to delete p130 in combination with Rb and p53 in adult lung epithelial cells. We found that loss of p130 resulted in increased proliferation and significant acceleration of SCLC development in this triple-knockout mouse model. The histopathologic features of the triple-mutant mouse tumors closely resembled that of human SCLC. Genome-wide expression profiling experiments further showed that Rb/p53/p130-mutant mouse tumors were similar to human SCLC. These findings indicate that p130 plays a key tumor suppressor role in SCLC. Rb/p53/p130-mutant mice provide a novel preclinical mouse model to identify novel therapeutic targets against SCLC.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1158/0008-5472.CAN-09-4228\">DOI 10.1158/0008-5472.CAN-09-4228</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000278486300004\">Web of Science ID 000278486300004</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20406986\">PubMedID 20406986</a></span>\n  </p>","text":"Abstract:\nSmall-cell lung carcinoma (SCLC) is a neuroendocrine subtype of lung cancer. Although SCLC patients often initially respond to therapy, tumors nearly always recur, resulting in a 5-year survival rate of less than 10%. A mouse model has been developed based on the fact that the RB and p53 tumor suppressor genes are mutated in more than 90% of human SCLCs. Emerging evidence in patients and mouse models suggests that p130, a gene related to RB, may act as a tumor suppressor in SCLC cells. To test this idea, we used conditional mutant mice to delete p130 in combination with Rb and p53 in adult lung epithelial cells. We found that loss of p130 resulted in increased proliferation and significant acceleration of SCLC development in this triple-knockout mouse model. The histopathologic features of the triple-mutant mouse tumors closely resembled that of human SCLC. Genome-wide expression profiling experiments further showed that Rb/p53/p130-mutant mouse tumors were similar to human SCLC. These findings indicate that p130 plays a key tumor suppressor role in SCLC. Rb/p53/p130-mutant mice provide a novel preclinical mouse model to identify novel therapeutic targets against SCLC.\nDOI: 10.1158/0008-5472.CAN-09-4228, https://dx.doi.org/10.1158/0008-5472.CAN-09-4228\nWeb of Science ID: 000278486300004, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000278486300004\nPubMedID: 20406986, https://www.ncbi.nlm.nih.gov/pubmed/20406986"},"doiId":"10.1158/0008-5472.CAN-09-4228","doiUrl":"https://dx.doi.org/10.1158/0008-5472.CAN-09-4228","featured":true,"firstPublished":{"html":"May 15, 2010 00:00:00 AM","text":"May 15, 2010 00:00:00 AM","value":"2010-05-15T00:00:00.000-07:00"},"mlaCitation":"Schaffer, Bethany E. et al. ?Loss of p130 Accelerates Tumor Development in a Mouse Model for Human Small-Cell Lung Carcinoma.? <i>CANCER RESEARCH</i> 70.10 (2010): 3877?3883. Print.","pubMedId":"20406986","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20406986","publicationId":91948,"title":"Loss of p130 Accelerates Tumor Development in a Mouse Model for Human Small-Cell Lung Carcinoma","type":"article","webOfScienceId":"000278486300004","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000278486300004"},{"apaCitation":"Burkhart, D. L., &amp; Sage, J. (2010). Complex transcriptional regulatory networks around the retinoblastoma tumor suppressor gene. <i>CELL CYCLE</i>, <i>9</i>(9), 1663?1664.","capCitation":"<span class=\"title\">\n  <span>Complex transcriptional regulatory networks around the retinoblastoma tumor suppressor gene</span>\n  <i>CELL CYCLE</i>\n</span>\n<span class=\"authors\">Burkhart, D. L., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">9 (9)</span><span class=\"pages\">: 1663-1664</span>\n</span>","chicagoCitation":"Burkhart, Deborah L., and Julien Sage. 2010. ?Complex Transcriptional Regulatory Networks around the Retinoblastoma Tumor Suppressor Gene.? <i>CELL CYCLE</i> 9 (9). LANDES BIOSCIENCE: 1663?64.","detail":{"html":"<p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000277402200002\">Web of Science ID 000277402200002</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20404509\">PubMedID 20404509</a></span>\n  </p>","text":"Web of Science ID: 000277402200002, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000277402200002\nPubMedID: 20404509, https://www.ncbi.nlm.nih.gov/pubmed/20404509"},"featured":false,"firstPublished":{"html":"May 1, 2010 00:00:00 AM","text":"May 1, 2010 00:00:00 AM","value":"2010-05-01T00:00:00.000-07:00"},"mlaCitation":"Burkhart, Deborah L., and Julien Sage. ?Complex Transcriptional Regulatory Networks around the Retinoblastoma Tumor Suppressor Gene.? <i>CELL CYCLE</i> 9.9 (2010): 1663?1664. Print.","pubMedId":"20404509","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20404509","publicationId":91932,"title":"Complex transcriptional regulatory networks around the retinoblastoma tumor suppressor gene","type":"article","webOfScienceId":"000277402200002","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000277402200002"},{"abstract":"p107 and its related family members Rb and p130 are critical regulators of cellular proliferation and tumorigenesis. Due to the extent of functional overlap within the Rb family, it has been difficult to assess which functions are exclusive to individual members and which are shared. Like its family members, p107 can bind a variety of cellular proteins to affect the expression of many target genes during cell cycle progression. Unlike Rb and p130, p107 is most highly expressed during the G1 to S phase transition of the cell cycle in actively dividing cells and accumulating evidence suggests a role for p107 during DNA replication. The specific roles for p107 during differentiation and development are less clear, although emerging studies suggest that it can cooperate with other Rb family members to control differentiation in multiple cell lineages. As a tumor suppressor, p107 is not as potent as Rb, yet studies in knockout mice have revealed some tumor suppressor functions in mice, depending on the context. In this review, we identify the unique and overlapping functions of p107 during the cell cycle, differentiation, and tumorigenesis.","apaCitation":"Wirt, S. E., &amp; Sage, J. (2010). p107 in the public eye: an Rb understudy and more. <i>CELL DIVISION</i>, <i>5</i>.","capCitation":"<span class=\"title\">\n  <span>p107 in the public eye: an Rb understudy and more</span>\n  <i>CELL DIVISION</i>\n</span>\n<span class=\"authors\">Wirt, S. E., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">5</span>\n</span>","chicagoCitation":"Wirt, Stacey E., and Julien Sage. 2010. ?p107 In the Public Eye: an Rb Understudy and More.? <i>CELL DIVISION</i> 5. BIOMED CENTRAL LTD.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">p107 and its related family members Rb and p130 are critical regulators of cellular proliferation and tumorigenesis. Due to the extent of functional overlap within the Rb family, it has been difficult to assess which functions are exclusive to individual members and which are shared. Like its family members, p107 can bind a variety of cellular proteins to affect the expression of many target genes during cell cycle progression. Unlike Rb and p130, p107 is most highly expressed during the G1 to S phase transition of the cell cycle in actively dividing cells and accumulating evidence suggests a role for p107 during DNA replication. The specific roles for p107 during differentiation and development are less clear, although emerging studies suggest that it can cooperate with other Rb family members to control differentiation in multiple cell lineages. As a tumor suppressor, p107 is not as potent as Rb, yet studies in knockout mice have revealed some tumor suppressor functions in mice, depending on the context. In this review, we identify the unique and overlapping functions of p107 during the cell cycle, differentiation, and tumorigenesis.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1186/1747-1028-5-9\">DOI 10.1186/1747-1028-5-9</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000283795500001\">Web of Science ID 000283795500001</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20359370\">PubMedID 20359370</a></span>\n  </p>","text":"Abstract:\np107 and its related family members Rb and p130 are critical regulators of cellular proliferation and tumorigenesis. Due to the extent of functional overlap within the Rb family, it has been difficult to assess which functions are exclusive to individual members and which are shared. Like its family members, p107 can bind a variety of cellular proteins to affect the expression of many target genes during cell cycle progression. Unlike Rb and p130, p107 is most highly expressed during the G1 to S phase transition of the cell cycle in actively dividing cells and accumulating evidence suggests a role for p107 during DNA replication. The specific roles for p107 during differentiation and development are less clear, although emerging studies suggest that it can cooperate with other Rb family members to control differentiation in multiple cell lineages. As a tumor suppressor, p107 is not as potent as Rb, yet studies in knockout mice have revealed some tumor suppressor functions in mice, depending on the context. In this review, we identify the unique and overlapping functions of p107 during the cell cycle, differentiation, and tumorigenesis.\nDOI: 10.1186/1747-1028-5-9, https://dx.doi.org/10.1186/1747-1028-5-9\nWeb of Science ID: 000283795500001, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000283795500001\nPubMedID: 20359370, https://www.ncbi.nlm.nih.gov/pubmed/20359370"},"doiId":"10.1186/1747-1028-5-9","doiUrl":"https://dx.doi.org/10.1186/1747-1028-5-9","featured":false,"firstPublished":{"html":"Apr 2, 2010 00:00:00 AM","text":"Apr 2, 2010 00:00:00 AM","value":"2010-04-02T00:00:00.000-07:00"},"mlaCitation":"Wirt, Stacey E., and Julien Sage. ?p107 In the Public Eye: an Rb Understudy and More.? <i>CELL DIVISION</i> 5 (2010): n. pag. Print.","pubMedId":"20359370","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20359370","publicationId":91257,"title":"p107 in the public eye: an Rb understudy and more","type":"article","webOfScienceId":"000283795500001","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000283795500001"},{"abstract":"In cancer cells, the retinoblastoma tumor suppressor RB is directly inactivated by mutation in the RB gene or functionally inhibited by abnormal activation of cyclin-dependent kinase activity. While variations in RB levels may also provide an important means of controlling RB function in both normal and cancer cells, little is known about the mechanisms regulating RB transcription. Here we show that members of the RB and E2F families bind directly to the RB promoter. To investigate how the RB/E2F pathway may regulate Rb transcription, we generated reporter mice carrying an eGFP transgene inserted into a bacterial artificial chromosome containing most of the Rb gene. Expression of eGFP largely parallels that of Rb in transgenic embryos and adult mice. Using these reporter mice and mutant alleles for Rb, p107, and p130, we found that RB family members modulate Rb transcription in specific cell populations in vivo and in culture. Interestingly, while Rb is a target of the RB/E2F pathway in mouse and human cells, Rb expression does not strictly correlate with the cell cycle status of these cells. These experiments identify novel regulatory feedback mechanisms within the RB pathway in mammalian cells.","apaCitation":"Burkhart, D. L., Ngai, L. K., Roake, C. M., Viatour, P., Thangavel, C., Ho, V. M., ? Sage, J. (2010). Regulation of RB Transcription In Vivo by RB Family Members. <i>MOLECULAR AND CELLULAR BIOLOGY</i>, <i>30</i>(7), 1729?1745.","capCitation":"<span class=\"title\">\n  <span>Regulation of RB Transcription In Vivo by RB Family Members</span>\n  <i>MOLECULAR AND CELLULAR BIOLOGY</i>\n</span>\n<span class=\"authors\">Burkhart, D. L., Ngai, L. K., Roake, C. M., Viatour, P., Thangavel, C., Ho, V. M., Knudsen, E. S., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">30 (7)</span><span class=\"pages\">: 1729-1745</span>\n</span>","chicagoCitation":"Burkhart, Deborah L., Lynn K. Ngai, Caitlin M. Roake, Patrick Viatour, Chellappagounder Thangavel, et al. 2010. ?Regulation of RB Transcription In Vivo by RB Family Members.? <i>MOLECULAR AND CELLULAR BIOLOGY</i> 30 (7). AMER SOC MICROBIOLOGY: 1729?45.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">In cancer cells, the retinoblastoma tumor suppressor RB is directly inactivated by mutation in the RB gene or functionally inhibited by abnormal activation of cyclin-dependent kinase activity. While variations in RB levels may also provide an important means of controlling RB function in both normal and cancer cells, little is known about the mechanisms regulating RB transcription. Here we show that members of the RB and E2F families bind directly to the RB promoter. To investigate how the RB/E2F pathway may regulate Rb transcription, we generated reporter mice carrying an eGFP transgene inserted into a bacterial artificial chromosome containing most of the Rb gene. Expression of eGFP largely parallels that of Rb in transgenic embryos and adult mice. Using these reporter mice and mutant alleles for Rb, p107, and p130, we found that RB family members modulate Rb transcription in specific cell populations in vivo and in culture. Interestingly, while Rb is a target of the RB/E2F pathway in mouse and human cells, Rb expression does not strictly correlate with the cell cycle status of these cells. These experiments identify novel regulatory feedback mechanisms within the RB pathway in mammalian cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1128/MCB.00952-09\">DOI 10.1128/MCB.00952-09</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000275302000013\">Web of Science ID 000275302000013</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20100864\">PubMedID 20100864</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2838078\">PubMedCentralID PMC2838078</a></span>\n  </p>","text":"Abstract:\nIn cancer cells, the retinoblastoma tumor suppressor RB is directly inactivated by mutation in the RB gene or functionally inhibited by abnormal activation of cyclin-dependent kinase activity. While variations in RB levels may also provide an important means of controlling RB function in both normal and cancer cells, little is known about the mechanisms regulating RB transcription. Here we show that members of the RB and E2F families bind directly to the RB promoter. To investigate how the RB/E2F pathway may regulate Rb transcription, we generated reporter mice carrying an eGFP transgene inserted into a bacterial artificial chromosome containing most of the Rb gene. Expression of eGFP largely parallels that of Rb in transgenic embryos and adult mice. Using these reporter mice and mutant alleles for Rb, p107, and p130, we found that RB family members modulate Rb transcription in specific cell populations in vivo and in culture. Interestingly, while Rb is a target of the RB/E2F pathway in mouse and human cells, Rb expression does not strictly correlate with the cell cycle status of these cells. These experiments identify novel regulatory feedback mechanisms within the RB pathway in mammalian cells.\nDOI: 10.1128/MCB.00952-09, https://dx.doi.org/10.1128/MCB.00952-09\nWeb of Science ID: 000275302000013, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000275302000013\nPubMedID: 20100864, https://www.ncbi.nlm.nih.gov/pubmed/20100864\nPubMedCentralID: PMC2838078, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2838078"},"doiId":"10.1128/MCB.00952-09","doiUrl":"https://dx.doi.org/10.1128/MCB.00952-09","featured":false,"firstPublished":{"html":"Apr 1, 2010 00:00:00 AM","text":"Apr 1, 2010 00:00:00 AM","value":"2010-04-01T00:00:00.000-07:00"},"mlaCitation":"Burkhart, Deborah L. et al. ?Regulation of RB Transcription In Vivo by RB Family Members.? <i>MOLECULAR AND CELLULAR BIOLOGY</i> 30.7 (2010): 1729?1745. Print.","pubMedId":"20100864","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20100864","publicationId":89130,"title":"Regulation of RB Transcription In Vivo by RB Family Members","type":"article","webOfScienceId":"000275302000013","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000275302000013"},{"abstract":"Human embryonic stem cells (hESCs) hold great promise in regenerative medicine. However, before the full potential of these cells is achieved, major basic biological questions need to be addressed. In particular, there are still gaps in our knowledge of the molecular mechanisms underlying the derivation of hESCs from blastocysts, the regulation of the undifferentiated, pluripotent state, and the control of differentiation into specific lineages. Furthermore, we still do not fully understand the tumorigenic potential of hESCs, limiting their use in regenerative medicine. The RB pathway is a key signaling module that controls cellular proliferation, cell survival, chromatin structure, and cellular differentiation in mammalian cells. Members of the RB pathway are important regulators of hESC biology and manipulation of the activity of this pathway may provide novel means to control the fate of hESCs. Here we review what is known about the expression and function of members of the RB pathway in hESCs and discuss areas of interest in this field.","apaCitation":"Conklin, J. F., &amp; Sage, J. (2009). Keeping an Eye on Retinoblastoma Control of Human Embryonic Stem Cells. <i>JOURNAL OF CELLULAR BIOCHEMISTRY</i>, <i>108</i>(5), 1023?1030.","capCitation":"<span class=\"title\">\n  <span>Keeping an Eye on Retinoblastoma Control of Human Embryonic Stem Cells</span>\n  <i>JOURNAL OF CELLULAR BIOCHEMISTRY</i>\n</span>\n<span class=\"authors\">Conklin, J. F., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2009</span>; <span class=\"volume\">108 (5)</span><span class=\"pages\">: 1023-1030</span>\n</span>","chicagoCitation":"Conklin, Jamie F., and Julien Sage. 2009. ?Keeping an Eye on Retinoblastoma Control of Human Embryonic Stem Cells.? <i>JOURNAL OF CELLULAR BIOCHEMISTRY</i> 108 (5). WILEY-LISS: 1023?30.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Human embryonic stem cells (hESCs) hold great promise in regenerative medicine. However, before the full potential of these cells is achieved, major basic biological questions need to be addressed. In particular, there are still gaps in our knowledge of the molecular mechanisms underlying the derivation of hESCs from blastocysts, the regulation of the undifferentiated, pluripotent state, and the control of differentiation into specific lineages. Furthermore, we still do not fully understand the tumorigenic potential of hESCs, limiting their use in regenerative medicine. The RB pathway is a key signaling module that controls cellular proliferation, cell survival, chromatin structure, and cellular differentiation in mammalian cells. Members of the RB pathway are important regulators of hESC biology and manipulation of the activity of this pathway may provide novel means to control the fate of hESCs. Here we review what is known about the expression and function of members of the RB pathway in hESCs and discuss areas of interest in this field.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1002/jcb.22342\">DOI 10.1002/jcb.22342</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000272640900001\">Web of Science ID 000272640900001</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/19760644\">PubMedID 19760644</a></span>\n  </p>","text":"Abstract:\nHuman embryonic stem cells (hESCs) hold great promise in regenerative medicine. However, before the full potential of these cells is achieved, major basic biological questions need to be addressed. In particular, there are still gaps in our knowledge of the molecular mechanisms underlying the derivation of hESCs from blastocysts, the regulation of the undifferentiated, pluripotent state, and the control of differentiation into specific lineages. Furthermore, we still do not fully understand the tumorigenic potential of hESCs, limiting their use in regenerative medicine. The RB pathway is a key signaling module that controls cellular proliferation, cell survival, chromatin structure, and cellular differentiation in mammalian cells. Members of the RB pathway are important regulators of hESC biology and manipulation of the activity of this pathway may provide novel means to control the fate of hESCs. Here we review what is known about the expression and function of members of the RB pathway in hESCs and discuss areas of interest in this field.\nDOI: 10.1002/jcb.22342, https://dx.doi.org/10.1002/jcb.22342\nWeb of Science ID: 000272640900001, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000272640900001\nPubMedID: 19760644, https://www.ncbi.nlm.nih.gov/pubmed/19760644"},"doiId":"10.1002/jcb.22342","doiUrl":"https://dx.doi.org/10.1002/jcb.22342","featured":false,"firstPublished":{"html":"Dec 1, 2009 00:00:00 AM","text":"Dec 1, 2009 00:00:00 AM","value":"2009-12-01T00:00:00.000-08:00"},"mlaCitation":"Conklin, Jamie F., and Julien Sage. ?Keeping an Eye on Retinoblastoma Control of Human Embryonic Stem Cells.? <i>JOURNAL OF CELLULAR BIOCHEMISTRY</i> 108.5 (2009): 1023?1030. Print.","pubMedId":"19760644","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/19760644","publicationId":85005,"title":"Keeping an Eye on Retinoblastoma Control of Human Embryonic Stem Cells","type":"article","webOfScienceId":"000272640900001","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000272640900001"},{"abstract":"A-type lamins are intermediate filament proteins that provide a scaffold for protein complexes regulating nuclear structure and function. Mutations in the LMNA gene are linked to a variety of degenerative disorders termed laminopathies, whereas changes in the expression of lamins are associated with tumourigenesis. The molecular pathways affected by alterations of A-type lamins and how they contribute to disease are poorly understood. Here, we show that A-type lamins have a key role in the maintenance of telomere structure, length and function, and in the stabilization of 53BP1, a component of the DNA damage response (DDR) pathway. Loss of A-type lamins alters the nuclear distribution of telomeres and results in telomere shortening, defects in telomeric heterochromatin, and increased genomic instability. In addition, A-type lamins are necessary for the processing of dysfunctional telomeres by non-homologous end joining, putatively through stabilization of 53BP1. This study shows new functions for A-type lamins in the maintenance of genomic integrity, and suggests that alterations of telomere biology and defects in DDR contribute to the pathogenesis of lamin-related diseases.","apaCitation":"Gonzalez-Suarez, I., Redwood, A. B., Perkins, S. M., Vermolen, B., Lichtensztejin, D., Grotsky, D. A., ? Gonzalo, S. (2009). Novel roles for A-type lamins in telomere biology and the DNA damage response pathway. <i>EMBO JOURNAL</i>, <i>28</i>(16), 2414?2427.","capCitation":"<span class=\"title\">\n  <span>Novel roles for A-type lamins in telomere biology and the DNA damage response pathway</span>\n  <i>EMBO JOURNAL</i>\n</span>\n<span class=\"authors\">Gonzalez-Suarez, I., Redwood, A. B., Perkins, S. M., Vermolen, B., Lichtensztejin, D., Grotsky, D. A., Morgado-Palacin, L., Gapud, E. J., Sleckman, B. P., Sullivan, T., Sage, J., Stewart, C. L., Mai, S., Gonzalo, S.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2009</span>; <span class=\"volume\">28 (16)</span><span class=\"pages\">: 2414-2427</span>\n</span>","chicagoCitation":"Gonzalez-Suarez, Ignacio, Abena B. Redwood, Stephanie M. Perkins, Bart Vermolen, Daniel Lichtensztejin, et al. 2009. ?Novel Roles for A-Type Lamins in Telomere Biology and the DNA Damage Response Pathway.? <i>EMBO JOURNAL</i> 28 (16). NATURE PUBLISHING GROUP: 2414?27.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">A-type lamins are intermediate filament proteins that provide a scaffold for protein complexes regulating nuclear structure and function. Mutations in the LMNA gene are linked to a variety of degenerative disorders termed laminopathies, whereas changes in the expression of lamins are associated with tumourigenesis. The molecular pathways affected by alterations of A-type lamins and how they contribute to disease are poorly understood. Here, we show that A-type lamins have a key role in the maintenance of telomere structure, length and function, and in the stabilization of 53BP1, a component of the DNA damage response (DDR) pathway. Loss of A-type lamins alters the nuclear distribution of telomeres and results in telomere shortening, defects in telomeric heterochromatin, and increased genomic instability. In addition, A-type lamins are necessary for the processing of dysfunctional telomeres by non-homologous end joining, putatively through stabilization of 53BP1. This study shows new functions for A-type lamins in the maintenance of genomic integrity, and suggests that alterations of telomere biology and defects in DDR contribute to the pathogenesis of lamin-related diseases.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/emboj.2009.196\">DOI 10.1038/emboj.2009.196</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000269074400010\">Web of Science ID 000269074400010</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/19629036\">PubMedID 19629036</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2735177\">PubMedCentralID PMC2735177</a></span>\n  </p>","text":"Abstract:\nA-type lamins are intermediate filament proteins that provide a scaffold for protein complexes regulating nuclear structure and function. Mutations in the LMNA gene are linked to a variety of degenerative disorders termed laminopathies, whereas changes in the expression of lamins are associated with tumourigenesis. The molecular pathways affected by alterations of A-type lamins and how they contribute to disease are poorly understood. Here, we show that A-type lamins have a key role in the maintenance of telomere structure, length and function, and in the stabilization of 53BP1, a component of the DNA damage response (DDR) pathway. Loss of A-type lamins alters the nuclear distribution of telomeres and results in telomere shortening, defects in telomeric heterochromatin, and increased genomic instability. In addition, A-type lamins are necessary for the processing of dysfunctional telomeres by non-homologous end joining, putatively through stabilization of 53BP1. This study shows new functions for A-type lamins in the maintenance of genomic integrity, and suggests that alterations of telomere biology and defects in DDR contribute to the pathogenesis of lamin-related diseases.\nDOI: 10.1038/emboj.2009.196, https://dx.doi.org/10.1038/emboj.2009.196\nWeb of Science ID: 000269074400010, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000269074400010\nPubMedID: 19629036, https://www.ncbi.nlm.nih.gov/pubmed/19629036\nPubMedCentralID: PMC2735177, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2735177"},"doiId":"10.1038/emboj.2009.196","doiUrl":"https://dx.doi.org/10.1038/emboj.2009.196","featured":false,"firstPublished":{"html":"Aug 19, 2009 00:00:00 AM","text":"Aug 19, 2009 00:00:00 AM","value":"2009-08-19T00:00:00.000-07:00"},"mlaCitation":"Gonzalez-Suarez, Ignacio et al. ?Novel Roles for A-Type Lamins in Telomere Biology and the DNA Damage Response Pathway.? <i>EMBO JOURNAL</i> 28.16 (2009): 2414?2427. Print.","pubMedId":"19629036","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/19629036","publicationId":242871,"title":"Novel roles for A-type lamins in telomere biology and the DNA damage response pathway","type":"article","webOfScienceId":"000269074400010","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000269074400010"},{"abstract":"The retinoblastoma Rb/E2F tumor suppressor pathway plays a major role in the regulation of mammalian cell cycle progression. The pRb protein, along with closely related proteins p107 and p130, exerts its anti-proliferative effects by binding to the E2F family of transcription factors known to regulate essential genes throughout the cell cycle. We sought to investigate the role of the Rb/E2F1 pathway in the lesion recognition step of nucleotide excision repair (NER) in mouse embryonic fibroblasts (MEFs). Rb-/-, p107-/-, p130-/- MEFs repaired both cyclobutane pyrimidine dimers (CPDs) and 6-4 photoproducts (6-4PPs) at higher efficiency than did wildtype cells following UV-C irradiation. The expression of damaged DNA binding gene DDB2 involved in the DNA lesion recognition step was elevated in the Rb family-deficient MEFs. To determine if the enhanced DNA repair in the absence of the Rb gene family is due to the derepression of E2F1, we assayed the ability of E2F1-deficient cells to repair damaged DNA and demonstrated that E2F1-/- MEFs are impaired for the removal of both CPDs and 6-4PPs. Furthermore, wildtype cells induced a higher expression of DDB2 and xeroderma pigmentosum gene XPC transcript levels than did E2F1-/- cells following UV-C irradiation. Using an E2F SiteScan algorithm, we uncovered a putative E2F-responsive element in the XPC promoter upstream of the transcription start site. We showed with chromatin immunoprecipitation assays the binding of E2F1 to the XPC promoter in a UV-dependent manner, suggesting that E2F1 is a transcriptional regulator of XPC. Our study identifies a novel E2F1 gene target and further supports the growing body of evidence that the Rb/E2F1 tumor suppressor pathway is involved in the regulation of the DNA lesion recognition step of nucleotide excision repair.","apaCitation":"Lin, P. S., McPherson, L. A., Chen, A. Y., Sage, J., &amp; Ford, J. M. (2009). The role of the retinoblastoma/E2F1 tumor suppressor pathway in the lesion recognition step of nucleotide excision repair. <i>DNA REPAIR</i>, <i>8</i>(7), 795?802.","capCitation":"<span class=\"title\">\n  <span>The role of the retinoblastoma/E2F1 tumor suppressor pathway in the lesion recognition step of nucleotide excision repair</span>\n  <i>DNA REPAIR</i>\n</span>\n<span class=\"authors\">Lin, P. S., McPherson, L. A., Chen, A. Y., Sage, J., Ford, J. M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2009</span>; <span class=\"volume\">8 (7)</span><span class=\"pages\">: 795-802</span>\n</span>","chicagoCitation":"Lin, Patrick S., Lisa A. McPherson, Aubrey Y. Chen, Julien Sage, and James M. Ford. 2009. ?The Role of the Retinoblastoma/E2F1 Tumor Suppressor Pathway in the Lesion Recognition Step of Nucleotide Excision Repair.? <i>DNA REPAIR</i> 8 (7). ELSEVIER SCIENCE BV: 795?802.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma Rb/E2F tumor suppressor pathway plays a major role in the regulation of mammalian cell cycle progression. The pRb protein, along with closely related proteins p107 and p130, exerts its anti-proliferative effects by binding to the E2F family of transcription factors known to regulate essential genes throughout the cell cycle. We sought to investigate the role of the Rb/E2F1 pathway in the lesion recognition step of nucleotide excision repair (NER) in mouse embryonic fibroblasts (MEFs). Rb-/-, p107-/-, p130-/- MEFs repaired both cyclobutane pyrimidine dimers (CPDs) and 6-4 photoproducts (6-4PPs) at higher efficiency than did wildtype cells following UV-C irradiation. The expression of damaged DNA binding gene DDB2 involved in the DNA lesion recognition step was elevated in the Rb family-deficient MEFs. To determine if the enhanced DNA repair in the absence of the Rb gene family is due to the derepression of E2F1, we assayed the ability of E2F1-deficient cells to repair damaged DNA and demonstrated that E2F1-/- MEFs are impaired for the removal of both CPDs and 6-4PPs. Furthermore, wildtype cells induced a higher expression of DDB2 and xeroderma pigmentosum gene XPC transcript levels than did E2F1-/- cells following UV-C irradiation. Using an E2F SiteScan algorithm, we uncovered a putative E2F-responsive element in the XPC promoter upstream of the transcription start site. We showed with chromatin immunoprecipitation assays the binding of E2F1 to the XPC promoter in a UV-dependent manner, suggesting that E2F1 is a transcriptional regulator of XPC. Our study identifies a novel E2F1 gene target and further supports the growing body of evidence that the Rb/E2F1 tumor suppressor pathway is involved in the regulation of the DNA lesion recognition step of nucleotide excision repair.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.dnarep.2009.03.003\">DOI 10.1016/j.dnarep.2009.03.003</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000268086300003\">Web of Science ID 000268086300003</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/19376752\">PubMedID 19376752</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma Rb/E2F tumor suppressor pathway plays a major role in the regulation of mammalian cell cycle progression. The pRb protein, along with closely related proteins p107 and p130, exerts its anti-proliferative effects by binding to the E2F family of transcription factors known to regulate essential genes throughout the cell cycle. We sought to investigate the role of the Rb/E2F1 pathway in the lesion recognition step of nucleotide excision repair (NER) in mouse embryonic fibroblasts (MEFs). Rb-/-, p107-/-, p130-/- MEFs repaired both cyclobutane pyrimidine dimers (CPDs) and 6-4 photoproducts (6-4PPs) at higher efficiency than did wildtype cells following UV-C irradiation. The expression of damaged DNA binding gene DDB2 involved in the DNA lesion recognition step was elevated in the Rb family-deficient MEFs. To determine if the enhanced DNA repair in the absence of the Rb gene family is due to the derepression of E2F1, we assayed the ability of E2F1-deficient cells to repair damaged DNA and demonstrated that E2F1-/- MEFs are impaired for the removal of both CPDs and 6-4PPs. Furthermore, wildtype cells induced a higher expression of DDB2 and xeroderma pigmentosum gene XPC transcript levels than did E2F1-/- cells following UV-C irradiation. Using an E2F SiteScan algorithm, we uncovered a putative E2F-responsive element in the XPC promoter upstream of the transcription start site. We showed with chromatin immunoprecipitation assays the binding of E2F1 to the XPC promoter in a UV-dependent manner, suggesting that E2F1 is a transcriptional regulator of XPC. Our study identifies a novel E2F1 gene target and further supports the growing body of evidence that the Rb/E2F1 tumor suppressor pathway is involved in the regulation of the DNA lesion recognition step of nucleotide excision repair.\nDOI: 10.1016/j.dnarep.2009.03.003, https://dx.doi.org/10.1016/j.dnarep.2009.03.003\nWeb of Science ID: 000268086300003, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000268086300003\nPubMedID: 19376752, https://www.ncbi.nlm.nih.gov/pubmed/19376752"},"doiId":"10.1016/j.dnarep.2009.03.003","doiUrl":"https://dx.doi.org/10.1016/j.dnarep.2009.03.003","featured":false,"firstPublished":{"html":"Jul 4, 2009 00:00:00 AM","text":"Jul 4, 2009 00:00:00 AM","value":"2009-07-04T00:00:00.000-07:00"},"mlaCitation":"Lin, Patrick S. et al. ?The Role of the Retinoblastoma/E2F1 Tumor Suppressor Pathway in the Lesion Recognition Step of Nucleotide Excision Repair.? <i>DNA REPAIR</i> 8.7 (2009): 795?802. Print.","pubMedId":"19376752","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/19376752","publicationId":80596,"title":"The role of the retinoblastoma/E2F1 tumor suppressor pathway in the lesion recognition step of nucleotide excision repair","type":"article","webOfScienceId":"000268086300003","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000268086300003"},{"abstract":"Mutations of the retinoblastoma tumor suppressor gene RB are frequently observed in human cancers, but rarely in non-small cell lung carcinomas (NSCLCs). Emerging evidence also suggests that the RB-related gene p130 is inactivated in a subset of human NSCLCs. To directly test the specific tumor suppressor roles of RB and p130 in NSCLC, we crossed Rb and p130 conditional mutant mice to mice carrying a conditional oncogenic K-Ras allele. In this model, controlled oncogenic K-Ras activation leads to the development of adenocarcinoma, a major subtype of NSCLC. We found that loss of p130 accelerated the death of mice, providing direct evidence in vivo that p130 is a tumor suppressor gene, albeit a weak one in this context. Loss of Rb increased the efficiency of lung cancer initiation and resulted in the development of high-grade adenocarcinomas and rapid death. Thus, despite the low frequency of RB mutations in human NSCLCs and reports that K-Ras activation and loss of RB function are rarely found in the same human tumors, loss of Rb clearly cooperates with activation of oncogenic K-Ras in lung adenocarcinoma development in mice.","apaCitation":"Ho, V. M., Schaffer, B. E., Karnezis, A. N., Park, K. S., &amp; Sage, J. (2009). The retinoblastoma gene Rb and its family member p130 suppress lung adenocarcinoma induced by oncogenic K-Ras. <i>ONCOGENE</i>, <i>28</i>(10), 1393?1399.","capCitation":"<span class=\"title\">\n  <span>The retinoblastoma gene Rb and its family member p130 suppress lung adenocarcinoma induced by oncogenic K-Ras</span>\n  <i>ONCOGENE</i>\n</span>\n<span class=\"authors\">Ho, V. M., Schaffer, B. E., Karnezis, A. N., Park, K. S., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2009</span>; <span class=\"volume\">28 (10)</span><span class=\"pages\">: 1393-1399</span>\n</span>","chicagoCitation":"Ho, V. M., B. E. Schaffer, A. N. Karnezis, K. S. Park, and J. Sage. 2009. ?The Retinoblastoma Gene Rb and Its Family Member p130 Suppress Lung Adenocarcinoma Induced by Oncogenic K-Ras.? <i>ONCOGENE</i> 28 (10). NATURE PUBLISHING GROUP: 1393?99.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Mutations of the retinoblastoma tumor suppressor gene RB are frequently observed in human cancers, but rarely in non-small cell lung carcinomas (NSCLCs). Emerging evidence also suggests that the RB-related gene p130 is inactivated in a subset of human NSCLCs. To directly test the specific tumor suppressor roles of RB and p130 in NSCLC, we crossed Rb and p130 conditional mutant mice to mice carrying a conditional oncogenic K-Ras allele. In this model, controlled oncogenic K-Ras activation leads to the development of adenocarcinoma, a major subtype of NSCLC. We found that loss of p130 accelerated the death of mice, providing direct evidence in vivo that p130 is a tumor suppressor gene, albeit a weak one in this context. Loss of Rb increased the efficiency of lung cancer initiation and resulted in the development of high-grade adenocarcinomas and rapid death. Thus, despite the low frequency of RB mutations in human NSCLCs and reports that K-Ras activation and loss of RB function are rarely found in the same human tumors, loss of Rb clearly cooperates with activation of oncogenic K-Ras in lung adenocarcinoma development in mice.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/onc.2008.491\">DOI 10.1038/onc.2008.491</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000264116000012\">Web of Science ID 000264116000012</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/19151761\">PubMedID 19151761</a></span>\n  </p>","text":"Abstract:\nMutations of the retinoblastoma tumor suppressor gene RB are frequently observed in human cancers, but rarely in non-small cell lung carcinomas (NSCLCs). Emerging evidence also suggests that the RB-related gene p130 is inactivated in a subset of human NSCLCs. To directly test the specific tumor suppressor roles of RB and p130 in NSCLC, we crossed Rb and p130 conditional mutant mice to mice carrying a conditional oncogenic K-Ras allele. In this model, controlled oncogenic K-Ras activation leads to the development of adenocarcinoma, a major subtype of NSCLC. We found that loss of p130 accelerated the death of mice, providing direct evidence in vivo that p130 is a tumor suppressor gene, albeit a weak one in this context. Loss of Rb increased the efficiency of lung cancer initiation and resulted in the development of high-grade adenocarcinomas and rapid death. Thus, despite the low frequency of RB mutations in human NSCLCs and reports that K-Ras activation and loss of RB function are rarely found in the same human tumors, loss of Rb clearly cooperates with activation of oncogenic K-Ras in lung adenocarcinoma development in mice.\nDOI: 10.1038/onc.2008.491, https://dx.doi.org/10.1038/onc.2008.491\nWeb of Science ID: 000264116000012, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000264116000012\nPubMedID: 19151761, https://www.ncbi.nlm.nih.gov/pubmed/19151761"},"doiId":"10.1038/onc.2008.491","doiUrl":"https://dx.doi.org/10.1038/onc.2008.491","featured":true,"firstPublished":{"html":"Mar 1, 2009 00:00:00 AM","text":"Mar 1, 2009 00:00:00 AM","value":"2009-03-01T00:00:00.000-08:00"},"mlaCitation":"Ho, V. M. et al. ?The Retinoblastoma Gene Rb and Its Family Member p130 Suppress Lung Adenocarcinoma Induced by Oncogenic K-Ras.? <i>ONCOGENE</i> 28.10 (2009): 1393?1399. Print.","pubMedId":"19151761","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/19151761","publicationId":77918,"title":"The retinoblastoma gene Rb and its family member p130 suppress lung adenocarcinoma induced by oncogenic K-Ras","type":"article","webOfScienceId":"000264116000012","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000264116000012"},{"abstract":"Individual members of the retinoblastoma (Rb) tumor suppressor gene family serve critical roles in the control of cellular proliferation and differentiation, but the extent of their contributions is masked by redundant and compensatory mechanisms. Here we employed a conditional knockout strategy to simultaneously inactivate all three members, Rb, p107, and p130, in adult hematopoietic stem cells (HSCs). Rb family triple knockout (TKO) mice develop a cell-intrinsic myeloproliferation that originates from hyperproliferative early hematopoietic progenitors and is accompanied by increased apoptosis in lymphoid progenitor populations. Loss of quiescence in the TKO HSC pool is associated with an expansion of these mutant stem cells but also with an enhanced mobilization and an impaired reconstitution potential upon transplantation. The presence of a single p107 allele is sufficient to largely rescue these defects. Thus, Rb family members collectively maintain HSC quiescence and the balance between lymphoid and myeloid cell fates in the hematopoietic system.","apaCitation":"Viatour, P., Somervaille, T. C., Venkatasubrahmanyam, S., Kogan, S., McLaughlin, M. E., Weissman, I. L., ? Sage, J. (2008). Hematopoietic Stem Cell Quiescence Is Maintained by Compound Contributions of the Retinoblastoma Gene Family. <i>CELL STEM CELL</i>, <i>3</i>(4), 416?428.","capCitation":"<span class=\"title\">\n  <span>Hematopoietic Stem Cell Quiescence Is Maintained by Compound Contributions of the Retinoblastoma Gene Family</span>\n  <i>CELL STEM CELL</i>\n</span>\n<span class=\"authors\">Viatour, P., Somervaille, T. C., Venkatasubrahmanyam, S., Kogan, S., McLaughlin, M. E., Weissman, I. L., Butte, A. J., Passegue, E., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2008</span>; <span class=\"volume\">3 (4)</span><span class=\"pages\">: 416-428</span>\n</span>","chicagoCitation":"Viatour, Patrick, Tim C. Somervaille, Shivkumar Venkatasubrahmanyam, Scott Kogan, Margaret E. McLaughlin, et al. 2008. ?Hematopoietic Stem Cell Quiescence Is Maintained by Compound Contributions of the Retinoblastoma Gene Family.? <i>CELL STEM CELL</i> 3 (4). CELL PRESS: 416?28.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Individual members of the retinoblastoma (Rb) tumor suppressor gene family serve critical roles in the control of cellular proliferation and differentiation, but the extent of their contributions is masked by redundant and compensatory mechanisms. Here we employed a conditional knockout strategy to simultaneously inactivate all three members, Rb, p107, and p130, in adult hematopoietic stem cells (HSCs). Rb family triple knockout (TKO) mice develop a cell-intrinsic myeloproliferation that originates from hyperproliferative early hematopoietic progenitors and is accompanied by increased apoptosis in lymphoid progenitor populations. Loss of quiescence in the TKO HSC pool is associated with an expansion of these mutant stem cells but also with an enhanced mobilization and an impaired reconstitution potential upon transplantation. The presence of a single p107 allele is sufficient to largely rescue these defects. Thus, Rb family members collectively maintain HSC quiescence and the balance between lymphoid and myeloid cell fates in the hematopoietic system.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.stem.2008.07.009\">DOI 10.1016/j.stem.2008.07.009</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000260149800012\">Web of Science ID 000260149800012</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/18940733\">PubMedID 18940733</a></span>\n  </p>","text":"Abstract:\nIndividual members of the retinoblastoma (Rb) tumor suppressor gene family serve critical roles in the control of cellular proliferation and differentiation, but the extent of their contributions is masked by redundant and compensatory mechanisms. Here we employed a conditional knockout strategy to simultaneously inactivate all three members, Rb, p107, and p130, in adult hematopoietic stem cells (HSCs). Rb family triple knockout (TKO) mice develop a cell-intrinsic myeloproliferation that originates from hyperproliferative early hematopoietic progenitors and is accompanied by increased apoptosis in lymphoid progenitor populations. Loss of quiescence in the TKO HSC pool is associated with an expansion of these mutant stem cells but also with an enhanced mobilization and an impaired reconstitution potential upon transplantation. The presence of a single p107 allele is sufficient to largely rescue these defects. Thus, Rb family members collectively maintain HSC quiescence and the balance between lymphoid and myeloid cell fates in the hematopoietic system.\nDOI: 10.1016/j.stem.2008.07.009, https://dx.doi.org/10.1016/j.stem.2008.07.009\nWeb of Science ID: 000260149800012, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000260149800012\nPubMedID: 18940733, https://www.ncbi.nlm.nih.gov/pubmed/18940733"},"doiId":"10.1016/j.stem.2008.07.009","doiUrl":"https://dx.doi.org/10.1016/j.stem.2008.07.009","featured":true,"firstPublished":{"html":"Oct 9, 2008 00:00:00 AM","text":"Oct 9, 2008 00:00:00 AM","value":"2008-10-09T00:00:00.000-07:00"},"mlaCitation":"Viatour, Patrick et al. ?Hematopoietic Stem Cell Quiescence Is Maintained by Compound Contributions of the Retinoblastoma Gene Family.? <i>CELL STEM CELL</i> 3.4 (2008): 416?428. Print.","pubMedId":"18940733","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/18940733","publicationId":75095,"title":"Hematopoietic Stem Cell Quiescence Is Maintained by Compound Contributions of the Retinoblastoma Gene Family","type":"article","webOfScienceId":"000260149800012","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000260149800012"},{"abstract":"The retinoblastoma (RB) tumour suppressor gene is functionally inactivated in a broad range of paediatric and adult cancers, and a plethora of cellular functions and partners have been identified for the RB protein. Data from human tumours and studies from mouse models indicate that loss of RB function contributes to both cancer initiation and progression. However, we still do not know the identity of the cell types in which RB normally prevents cancer initiation in vivo, and the specific functions of RB that suppress distinct aspects of the tumorigenic process are poorly understood.","apaCitation":"Burkhart, D. L., &amp; Sage, J. (2008). Cellular mechanisms of tumour suppression by the retinoblastoma gene. <i>NATURE REVIEWS CANCER</i>, <i>8</i>(9), 671?682.","capCitation":"<span class=\"title\">\n  <span>Cellular mechanisms of tumour suppression by the retinoblastoma gene</span>\n  <i>NATURE REVIEWS CANCER</i>\n</span>\n<span class=\"authors\">Burkhart, D. L., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2008</span>; <span class=\"volume\">8 (9)</span><span class=\"pages\">: 671-682</span>\n</span>","chicagoCitation":"Burkhart, Deborah L., and Julien Sage. 2008. ?Cellular Mechanisms of Tumour Suppression by the Retinoblastoma Gene.? <i>NATURE REVIEWS CANCER</i> 8 (9). NATURE PUBLISHING GROUP: 671?82.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma (RB) tumour suppressor gene is functionally inactivated in a broad range of paediatric and adult cancers, and a plethora of cellular functions and partners have been identified for the RB protein. Data from human tumours and studies from mouse models indicate that loss of RB function contributes to both cancer initiation and progression. However, we still do not know the identity of the cell types in which RB normally prevents cancer initiation in vivo, and the specific functions of RB that suppress distinct aspects of the tumorigenic process are poorly understood.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nrc2399\">DOI 10.1038/nrc2399</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000258662900012\">Web of Science ID 000258662900012</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/18650841\">PubMedID 18650841</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma (RB) tumour suppressor gene is functionally inactivated in a broad range of paediatric and adult cancers, and a plethora of cellular functions and partners have been identified for the RB protein. Data from human tumours and studies from mouse models indicate that loss of RB function contributes to both cancer initiation and progression. However, we still do not know the identity of the cell types in which RB normally prevents cancer initiation in vivo, and the specific functions of RB that suppress distinct aspects of the tumorigenic process are poorly understood.\nDOI: 10.1038/nrc2399, https://dx.doi.org/10.1038/nrc2399\nWeb of Science ID: 000258662900012, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000258662900012\nPubMedID: 18650841, https://www.ncbi.nlm.nih.gov/pubmed/18650841"},"doiId":"10.1038/nrc2399","doiUrl":"https://dx.doi.org/10.1038/nrc2399","featured":true,"firstPublished":{"html":"Sep 1, 2008 00:00:00 AM","text":"Sep 1, 2008 00:00:00 AM","value":"2008-09-01T00:00:00.000-07:00"},"mlaCitation":"Burkhart, Deborah L., and Julien Sage. ?Cellular Mechanisms of Tumour Suppression by the Retinoblastoma Gene.? <i>NATURE REVIEWS CANCER</i> 8.9 (2008): 671?682. Print.","pubMedId":"18650841","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/18650841","publicationId":72213,"title":"Cellular mechanisms of tumour suppression by the retinoblastoma gene","type":"article","webOfScienceId":"000258662900012","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000258662900012"},{"abstract":"The RB tumor suppressor gene is mutated in a broad range of human cancers, including pediatric retinoblastoma. Strikingly, however, Rb mutant mice develop tumors of the pituitary and thyroid glands, but not retinoblastoma. Mouse genetics experiments have demonstrated that p107, a protein related to pRB, is capable of preventing retinoblastoma, but not pituitary tumors, in Rb-deficient mice. Evidence suggests that the basis for this compensatory function of p107 is increased transcription of the p107 gene in response to Rb inactivation. To begin to address the context-dependency of this compensatory role of p107 and to follow p107 expression in vivo, we have generated transgenic mice carrying an enhanced GFP (eGFP) reporter inserted into a bacterial artificial chromosome (BAC) containing the mouse p107 gene. Expression of the eGFP transgene parallels that of p107 in these transgenic mice and identifies cells with a broad range of expression level for p107, even within particular organs or tissues. We also show that loss of Rb results in the upregulation of p107 transcription in specific cell populations in vivo, including subpopulations of hematopoietic cells. Thus, p107 BAC-eGFP transgenic mice serve as a useful tool to identify distinct cell types in which p107 is expressed and may have key functions in vivo, and to characterize changes in cellular networks accompanying Rb deficiency.","apaCitation":"Burkhart, D. L., Viatour, P., Ho, V. M., &amp; Sage, J. (2008). GFP reporter mice for the retinoblastoma-related cell cycle regulator p107. <i>CELL CYCLE</i>, <i>7</i>(16), 2544?2552.","capCitation":"<span class=\"title\">\n  <span>GFP reporter mice for the retinoblastoma-related cell cycle regulator p107</span>\n  <i>CELL CYCLE</i>\n</span>\n<span class=\"authors\">Burkhart, D. L., Viatour, P., Ho, V. M., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2008</span>; <span class=\"volume\">7 (16)</span><span class=\"pages\">: 2544-2552</span>\n</span>","chicagoCitation":"Burkhart, Deborah L., Patrick Viatour, Victoria M. Ho, and Julien Sage. 2008. ?GFP Reporter Mice for the Retinoblastoma-Related Cell Cycle Regulator p107.? <i>CELL CYCLE</i> 7 (16). LANDES BIOSCIENCE: 2544?52.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The RB tumor suppressor gene is mutated in a broad range of human cancers, including pediatric retinoblastoma. Strikingly, however, Rb mutant mice develop tumors of the pituitary and thyroid glands, but not retinoblastoma. Mouse genetics experiments have demonstrated that p107, a protein related to pRB, is capable of preventing retinoblastoma, but not pituitary tumors, in Rb-deficient mice. Evidence suggests that the basis for this compensatory function of p107 is increased transcription of the p107 gene in response to Rb inactivation. To begin to address the context-dependency of this compensatory role of p107 and to follow p107 expression in vivo, we have generated transgenic mice carrying an enhanced GFP (eGFP) reporter inserted into a bacterial artificial chromosome (BAC) containing the mouse p107 gene. Expression of the eGFP transgene parallels that of p107 in these transgenic mice and identifies cells with a broad range of expression level for p107, even within particular organs or tissues. We also show that loss of Rb results in the upregulation of p107 transcription in specific cell populations in vivo, including subpopulations of hematopoietic cells. Thus, p107 BAC-eGFP transgenic mice serve as a useful tool to identify distinct cell types in which p107 is expressed and may have key functions in vivo, and to characterize changes in cellular networks accompanying Rb deficiency.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000258829900016\">Web of Science ID 000258829900016</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/18719374\">PubMedID 18719374</a></span>\n  </p>","text":"Abstract:\nThe RB tumor suppressor gene is mutated in a broad range of human cancers, including pediatric retinoblastoma. Strikingly, however, Rb mutant mice develop tumors of the pituitary and thyroid glands, but not retinoblastoma. Mouse genetics experiments have demonstrated that p107, a protein related to pRB, is capable of preventing retinoblastoma, but not pituitary tumors, in Rb-deficient mice. Evidence suggests that the basis for this compensatory function of p107 is increased transcription of the p107 gene in response to Rb inactivation. To begin to address the context-dependency of this compensatory role of p107 and to follow p107 expression in vivo, we have generated transgenic mice carrying an enhanced GFP (eGFP) reporter inserted into a bacterial artificial chromosome (BAC) containing the mouse p107 gene. Expression of the eGFP transgene parallels that of p107 in these transgenic mice and identifies cells with a broad range of expression level for p107, even within particular organs or tissues. We also show that loss of Rb results in the upregulation of p107 transcription in specific cell populations in vivo, including subpopulations of hematopoietic cells. Thus, p107 BAC-eGFP transgenic mice serve as a useful tool to identify distinct cell types in which p107 is expressed and may have key functions in vivo, and to characterize changes in cellular networks accompanying Rb deficiency.\n\nWeb of Science ID: 000258829900016, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000258829900016\nPubMedID: 18719374, https://www.ncbi.nlm.nih.gov/pubmed/18719374"},"featured":false,"firstPublished":{"html":"Aug 15, 2008 00:00:00 AM","text":"Aug 15, 2008 00:00:00 AM","value":"2008-08-15T00:00:00.000-07:00"},"mlaCitation":"Burkhart, Deborah L. et al. ?GFP Reporter Mice for the Retinoblastoma-Related Cell Cycle Regulator p107.? <i>CELL CYCLE</i> 7.16 (2008): 2544?2552. Print.","pubMedId":"18719374","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/18719374","publicationId":73217,"title":"GFP reporter mice for the retinoblastoma-related cell cycle regulator p107","type":"article","webOfScienceId":"000258829900016","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000258829900016"},{"apaCitation":"Sage, J. (2007). Hope in sight for retinoblastoma. <i>NATURE MEDICINE</i>, <i>13</i>(1), 30?31.","capCitation":"<span class=\"title\">\n  <span>Hope in sight for retinoblastoma</span>\n  <i>NATURE MEDICINE</i>\n</span>\n<span class=\"authors\">Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2007</span>; <span class=\"volume\">13 (1)</span><span class=\"pages\">: 30-31</span>\n</span>","chicagoCitation":"Sage, Julien. 2007. ?Hope in Sight for Retinoblastoma.? <i>NATURE MEDICINE</i> 13 (1). NATURE PUBLISHING GROUP: 30?31.","detail":{"html":"<p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000243301800024\">Web of Science ID 000243301800024</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/17206131\">PubMedID 17206131</a></span>\n  </p>","text":"Web of Science ID: 000243301800024, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000243301800024\nPubMedID: 17206131, https://www.ncbi.nlm.nih.gov/pubmed/17206131"},"featured":false,"firstPublished":{"html":"Jan 1, 2007 00:00:00 AM","text":"Jan 1, 2007 00:00:00 AM","value":"2007-01-01T00:00:00.000-08:00"},"mlaCitation":"Sage, Julien. ?Hope in Sight for Retinoblastoma.? <i>NATURE MEDICINE</i> 13.1 (2007): 30?31. Print.","pubMedId":"17206131","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/17206131","publicationId":56229,"title":"Hope in sight for retinoblastoma","type":"article","webOfScienceId":"000243301800024","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000243301800024"},{"abstract":"Genetic studies have demonstrated that Bmi1 promotes cell proliferation and stem cell self-renewal with a correlative decrease of p16(INK4a) expression. Here, we demonstrate that Polycomb genes EZH2 and BMI1 repress p16 expression in human and mouse primary cells, but not in cells deficient for pRB protein function. The p16 locus is H3K27-methylated and bound by BMI1, RING2, and SUZ12. Inactivation of pRB family proteins abolishes H3K27 methylation and disrupts BMI1, RING2, and SUZ12 binding to the p16 locus. These results suggest a model in which pRB proteins recruit PRC2 to trimethylate p16, priming the BMI1-containing PRC1L ubiquitin ligase complex to silence p16.","apaCitation":"Kotake, Y., Cao, R., Viatour, P., Sage, J., Zhang, Y., &amp; Xiong, Y. (2007). pRB family proteins are required for H3K27 trimethylation and polycomb repression complexes binding to and silencing p16(INK4a) tumor suppressor gene. <i>GENES &amp; DEVELOPMENT</i>, <i>21</i>(1), 49?54.","capCitation":"<span class=\"title\">\n  <span>pRB family proteins are required for H3K27 trimethylation and polycomb repression complexes binding to and silencing p16(INK4a) tumor suppressor gene</span>\n  <i>GENES &amp; DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Kotake, Y., Cao, R., Viatour, P., Sage, J., Zhang, Y., Xiong, Y.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2007</span>; <span class=\"volume\">21 (1)</span><span class=\"pages\">: 49-54</span>\n</span>","chicagoCitation":"Kotake, Yojiro, Ru Cao, Patrick Viatour, Julien Sage, Yi Zhang, et al. 2007. ?PRB Family Proteins Are Required for H3K27 Trimethylation and Polycomb Repression Complexes Binding to and Silencing p16(INK4a) Tumor Suppressor Gene.? <i>GENES &amp; DEVELOPMENT</i> 21 (1). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 49?54.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Genetic studies have demonstrated that Bmi1 promotes cell proliferation and stem cell self-renewal with a correlative decrease of p16(INK4a) expression. Here, we demonstrate that Polycomb genes EZH2 and BMI1 repress p16 expression in human and mouse primary cells, but not in cells deficient for pRB protein function. The p16 locus is H3K27-methylated and bound by BMI1, RING2, and SUZ12. Inactivation of pRB family proteins abolishes H3K27 methylation and disrupts BMI1, RING2, and SUZ12 binding to the p16 locus. These results suggest a model in which pRB proteins recruit PRC2 to trimethylate p16, priming the BMI1-containing PRC1L ubiquitin ligase complex to silence p16.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.1499407\">DOI 10.1101/gad.1499407</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000243382700005\">Web of Science ID 000243382700005</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/17210787\">PubMedID 17210787</a></span>\n  </p>","text":"Abstract:\nGenetic studies have demonstrated that Bmi1 promotes cell proliferation and stem cell self-renewal with a correlative decrease of p16(INK4a) expression. Here, we demonstrate that Polycomb genes EZH2 and BMI1 repress p16 expression in human and mouse primary cells, but not in cells deficient for pRB protein function. The p16 locus is H3K27-methylated and bound by BMI1, RING2, and SUZ12. Inactivation of pRB family proteins abolishes H3K27 methylation and disrupts BMI1, RING2, and SUZ12 binding to the p16 locus. These results suggest a model in which pRB proteins recruit PRC2 to trimethylate p16, priming the BMI1-containing PRC1L ubiquitin ligase complex to silence p16.\nDOI: 10.1101/gad.1499407, https://dx.doi.org/10.1101/gad.1499407\nWeb of Science ID: 000243382700005, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000243382700005\nPubMedID: 17210787, https://www.ncbi.nlm.nih.gov/pubmed/17210787"},"doiId":"10.1101/gad.1499407","doiUrl":"https://dx.doi.org/10.1101/gad.1499407","featured":false,"firstPublished":{"html":"Jan 1, 2007 00:00:00 AM","text":"Jan 1, 2007 00:00:00 AM","value":"2007-01-01T00:00:00.000-08:00"},"mlaCitation":"Kotake, Yojiro et al. ?PRB Family Proteins Are Required for H3K27 Trimethylation and Polycomb Repression Complexes Binding to and Silencing p16(INK4a) Tumor Suppressor Gene.? <i>GENES &amp; DEVELOPMENT</i> 21.1 (2007): 49?54. Print.","pubMedId":"17210787","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/17210787","publicationId":217807,"title":"pRB family proteins are required for H3K27 trimethylation and polycomb repression complexes binding to and silencing p16(INK4a) tumor suppressor gene","type":"article","webOfScienceId":"000243382700005","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000243382700005"},{"abstract":"pRb, p107, and p130 are related proteins that play a central role in the regulation of cell cycle progression and terminal differentiation in mammalian cells. Nevertheless, it is still largely unclear how these proteins achieve this regulation in vivo. The intestinal epithelium is an ideal in vivo system in which to study the molecular pathways that regulate proliferation and differentiation because it exists in a constant state of development throughout an animal's lifetime. We studied the phenotypic effects on the intestinal epithelium of mutating Rb and p107 or p130. Although mutating these genes singly had little or no effect, loss of pRb and p107 or p130 together produced chronic hyperplasia and dysplasia of the small intestinal and colonic epithelium. In Rb/p130 double mutants this hyperplasia was associated with defects in terminal differentiation of specific cell types and was dependent on the increased proliferation seen in the epithelium of mutant animals. At the molecular level, dysregulation of the Rb pathway led to an increase in the expression of Math1, Cdx1, Cdx2, transcription factors that regulate proliferation and differentiation in the intestinal epithelium. The absence of Cdx1 function in Rb/p130 double mutant mice partially reverted the histologic phenotype by suppressing ectopic mitosis in the epithelium. These studies implicate the Rb pathway as a regulator of epithelial homeostasis in the intestine.","apaCitation":"Haigis, K., Sage, J., Glickman, J., Shafer, S., &amp; Jacks, T. (2006). The related retinoblastoma (pRb) and p130 proteins cooperate to regulate homeostasis in the intestinal epithelium. <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i>, <i>281</i>(1), 638?647.","capCitation":"<span class=\"title\">\n  <span>The related retinoblastoma (pRb) and p130 proteins cooperate to regulate homeostasis in the intestinal epithelium</span>\n  <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i>\n</span>\n<span class=\"authors\">Haigis, K., Sage, J., Glickman, J., Shafer, S., Jacks, T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2006</span>; <span class=\"volume\">281 (1)</span><span class=\"pages\">: 638-647</span>\n</span>","chicagoCitation":"Haigis, K., J. Sage, J. Glickman, S. Shafer, and T. Jacks. 2006. ?The Related Retinoblastoma (PRb) and p130 Proteins Cooperate to Regulate Homeostasis in the Intestinal Epithelium.? <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i> 281 (1). AMER SOC BIOCHEMISTRY MOLECULAR BIOLOGY INC: 638?47.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">pRb, p107, and p130 are related proteins that play a central role in the regulation of cell cycle progression and terminal differentiation in mammalian cells. Nevertheless, it is still largely unclear how these proteins achieve this regulation in vivo. The intestinal epithelium is an ideal in vivo system in which to study the molecular pathways that regulate proliferation and differentiation because it exists in a constant state of development throughout an animal's lifetime. We studied the phenotypic effects on the intestinal epithelium of mutating Rb and p107 or p130. Although mutating these genes singly had little or no effect, loss of pRb and p107 or p130 together produced chronic hyperplasia and dysplasia of the small intestinal and colonic epithelium. In Rb/p130 double mutants this hyperplasia was associated with defects in terminal differentiation of specific cell types and was dependent on the increased proliferation seen in the epithelium of mutant animals. At the molecular level, dysregulation of the Rb pathway led to an increase in the expression of Math1, Cdx1, Cdx2, transcription factors that regulate proliferation and differentiation in the intestinal epithelium. The absence of Cdx1 function in Rb/p130 double mutant mice partially reverted the histologic phenotype by suppressing ectopic mitosis in the epithelium. These studies implicate the Rb pathway as a regulator of epithelial homeostasis in the intestine.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000234307200076\">Web of Science ID 000234307200076</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/16258171\">PubMedID 16258171</a></span>\n  </p>","text":"Abstract:\npRb, p107, and p130 are related proteins that play a central role in the regulation of cell cycle progression and terminal differentiation in mammalian cells. Nevertheless, it is still largely unclear how these proteins achieve this regulation in vivo. The intestinal epithelium is an ideal in vivo system in which to study the molecular pathways that regulate proliferation and differentiation because it exists in a constant state of development throughout an animal's lifetime. We studied the phenotypic effects on the intestinal epithelium of mutating Rb and p107 or p130. Although mutating these genes singly had little or no effect, loss of pRb and p107 or p130 together produced chronic hyperplasia and dysplasia of the small intestinal and colonic epithelium. In Rb/p130 double mutants this hyperplasia was associated with defects in terminal differentiation of specific cell types and was dependent on the increased proliferation seen in the epithelium of mutant animals. At the molecular level, dysregulation of the Rb pathway led to an increase in the expression of Math1, Cdx1, Cdx2, transcription factors that regulate proliferation and differentiation in the intestinal epithelium. The absence of Cdx1 function in Rb/p130 double mutant mice partially reverted the histologic phenotype by suppressing ectopic mitosis in the epithelium. These studies implicate the Rb pathway as a regulator of epithelial homeostasis in the intestine.\n\nWeb of Science ID: 000234307200076, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000234307200076\nPubMedID: 16258171, https://www.ncbi.nlm.nih.gov/pubmed/16258171"},"featured":false,"firstPublished":{"html":"Jan 6, 2006 00:00:00 AM","text":"Jan 6, 2006 00:00:00 AM","value":"2006-01-06T00:00:00.000-08:00"},"mlaCitation":"Haigis, K. et al. ?The Related Retinoblastoma (PRb) and p130 Proteins Cooperate to Regulate Homeostasis in the Intestinal Epithelium.? <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i> 281.1 (2006): 638?647. Print.","pubMedId":"16258171","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/16258171","publicationId":242872,"title":"The related retinoblastoma (pRb) and p130 proteins cooperate to regulate homeostasis in the intestinal epithelium","type":"article","webOfScienceId":"000234307200076","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000234307200076"},{"abstract":"In primary cells, overexpression of oncogenes such as Ras(V12) induces premature senescence rather than transformation. Senescence is an irreversible form of G1 arrest that requires the p19ARF/p53 and p16INK4a/pRB pathways and may suppress tumorigenesis in vivo. Here we show that the transcription factor C/EBPbeta is required for Ras(V12)-induced senescence. C/EBPbeta-/- mouse embryo fibroblasts (MEFs) expressing Ras(V12) continued to proliferate despite unimpaired induction of p19ARF and p53, and lacked morphological features of senescent fibroblasts. Enforced C/EBPbeta expression inhibited proliferation of wild-type MEFs and also slowed proliferation of p19Arf-/- and p53-/- cells, indicating that C/EBPbeta acts downstream or independently of p19ARF/p53 to suppress growth. C/EBPbeta was unable to inhibit proliferation of MEFs lacking all three RB family proteins or wild-type cells expressing dominant negative E2F-1 and, instead, stimulated their growth. C/EBPbeta decreased expression of several E2F target genes and was associated with their promoters in chromatin immunoprecipitation assays, suggesting that C/EBPbeta functions by repressing genes required for cell cycle progression. C/EBPbeta is therefore a novel component of the RB:E2F-dependent senescence program activated by oncogenic stress in primary cells.","apaCitation":"Sebastian, T., Malik, R., Thomas, S., Sage, J., &amp; Johnson, P. F. (2005). C/EBP beta cooperates with RB : E2F to implement Ras(V12)-induced cellular senescence. <i>EMBO JOURNAL</i>, <i>24</i>(18), 3301?3312.","capCitation":"<span class=\"title\">\n  <span>C/EBP beta cooperates with RB : E2F to implement Ras(V12)-induced cellular senescence</span>\n  <i>EMBO JOURNAL</i>\n</span>\n<span class=\"authors\">Sebastian, T., Malik, R., Thomas, S., Sage, J., Johnson, P. F.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2005</span>; <span class=\"volume\">24 (18)</span><span class=\"pages\">: 3301-3312</span>\n</span>","chicagoCitation":"Sebastian, T., R. Malik, S. Thomas, J. Sage, and P. F. Johnson. 2005. ?C/EBP Beta Cooperates with RB : E2F to Implement Ras(V12)-Induced Cellular Senescence.? <i>EMBO JOURNAL</i> 24 (18). NATURE PUBLISHING GROUP: 3301?12.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">In primary cells, overexpression of oncogenes such as Ras(V12) induces premature senescence rather than transformation. Senescence is an irreversible form of G1 arrest that requires the p19ARF/p53 and p16INK4a/pRB pathways and may suppress tumorigenesis in vivo. Here we show that the transcription factor C/EBPbeta is required for Ras(V12)-induced senescence. C/EBPbeta-/- mouse embryo fibroblasts (MEFs) expressing Ras(V12) continued to proliferate despite unimpaired induction of p19ARF and p53, and lacked morphological features of senescent fibroblasts. Enforced C/EBPbeta expression inhibited proliferation of wild-type MEFs and also slowed proliferation of p19Arf-/- and p53-/- cells, indicating that C/EBPbeta acts downstream or independently of p19ARF/p53 to suppress growth. C/EBPbeta was unable to inhibit proliferation of MEFs lacking all three RB family proteins or wild-type cells expressing dominant negative E2F-1 and, instead, stimulated their growth. C/EBPbeta decreased expression of several E2F target genes and was associated with their promoters in chromatin immunoprecipitation assays, suggesting that C/EBPbeta functions by repressing genes required for cell cycle progression. C/EBPbeta is therefore a novel component of the RB:E2F-dependent senescence program activated by oncogenic stress in primary cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/sj.emboj.7600789\">DOI 10.1038/sj.emboj.7600789</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000232551600014\">Web of Science ID 000232551600014</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/16107878\">PubMedID 16107878</a></span>\n  </p>","text":"Abstract:\nIn primary cells, overexpression of oncogenes such as Ras(V12) induces premature senescence rather than transformation. Senescence is an irreversible form of G1 arrest that requires the p19ARF/p53 and p16INK4a/pRB pathways and may suppress tumorigenesis in vivo. Here we show that the transcription factor C/EBPbeta is required for Ras(V12)-induced senescence. C/EBPbeta-/- mouse embryo fibroblasts (MEFs) expressing Ras(V12) continued to proliferate despite unimpaired induction of p19ARF and p53, and lacked morphological features of senescent fibroblasts. Enforced C/EBPbeta expression inhibited proliferation of wild-type MEFs and also slowed proliferation of p19Arf-/- and p53-/- cells, indicating that C/EBPbeta acts downstream or independently of p19ARF/p53 to suppress growth. C/EBPbeta was unable to inhibit proliferation of MEFs lacking all three RB family proteins or wild-type cells expressing dominant negative E2F-1 and, instead, stimulated their growth. C/EBPbeta decreased expression of several E2F target genes and was associated with their promoters in chromatin immunoprecipitation assays, suggesting that C/EBPbeta functions by repressing genes required for cell cycle progression. C/EBPbeta is therefore a novel component of the RB:E2F-dependent senescence program activated by oncogenic stress in primary cells.\nDOI: 10.1038/sj.emboj.7600789, https://dx.doi.org/10.1038/sj.emboj.7600789\nWeb of Science ID: 000232551600014, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000232551600014\nPubMedID: 16107878, https://www.ncbi.nlm.nih.gov/pubmed/16107878"},"doiId":"10.1038/sj.emboj.7600789","doiUrl":"https://dx.doi.org/10.1038/sj.emboj.7600789","featured":false,"firstPublished":{"html":"Sep 21, 2005 00:00:00 AM","text":"Sep 21, 2005 00:00:00 AM","value":"2005-09-21T00:00:00.000-07:00"},"mlaCitation":"Sebastian, T. et al. ?C/EBP Beta Cooperates with RB : E2F to Implement Ras(V12)-Induced Cellular Senescence.? <i>EMBO JOURNAL</i> 24.18 (2005): 3301?3312. Print.","pubMedId":"16107878","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/16107878","publicationId":44096,"title":"C/EBP beta cooperates with RB : E2F to implement Ras(V12)-induced cellular senescence","type":"article","webOfScienceId":"000232551600014","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000232551600014"},{"abstract":"As the population of industrial nations ages, the incidence of cancer and cancer mortality is increasing. Intuitively, older organisms may be less able to cope with accumulated damage and thus be more prone to develop cancer. However, so far, the links between aging and cancer have been only partially explored. Strikingly, four recent studies now indicate that premature senescence accompanied by cell cycle arrest occurs in tumors initiated by an oncogenic mutation. Thus, senescence may act as a key tumor suppressor mechanism in young tumors in vivo.","apaCitation":"Sage, J. (2005). Making young tumors old: a new weapon against cancer? <i>Science of Aging Knowledge Environment : SAGE KE</i>, <i>2005</i>(33), pe25-?","capCitation":"<span class=\"title\">\n  <span>Making young tumors old: a new weapon against cancer?</span>\n  <i>Science of aging knowledge environment : SAGE KE</i>\n</span>\n<span class=\"authors\">Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2005</span>; <span class=\"volume\">2005 (33)</span><span class=\"pages\">: pe25-?</span>\n</span>","chicagoCitation":"Sage, Julien. 2005. ?Making Young Tumors Old: a New Weapon against Cancer?? <i>Science of Aging Knowledge Environment : SAGE KE</i> 2005 (33): pe25-?","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">As the population of industrial nations ages, the incidence of cancer and cancer mortality is increasing. Intuitively, older organisms may be less able to cope with accumulated damage and thus be more prone to develop cancer. However, so far, the links between aging and cancer have been only partially explored. Strikingly, four recent studies now indicate that premature senescence accompanied by cell cycle arrest occurs in tumors initiated by an oncogenic mutation. Thus, senescence may act as a key tumor suppressor mechanism in young tumors in vivo.</p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/16107660\">PubMedID 16107660</a></span>\n  </p>","text":"Abstract:\nAs the population of industrial nations ages, the incidence of cancer and cancer mortality is increasing. Intuitively, older organisms may be less able to cope with accumulated damage and thus be more prone to develop cancer. However, so far, the links between aging and cancer have been only partially explored. Strikingly, four recent studies now indicate that premature senescence accompanied by cell cycle arrest occurs in tumors initiated by an oncogenic mutation. Thus, senescence may act as a key tumor suppressor mechanism in young tumors in vivo.\n\n\nPubMedID: 16107660, https://www.ncbi.nlm.nih.gov/pubmed/16107660"},"featured":false,"firstPublished":{"html":"Aug 17, 2005 00:00:00 AM","text":"Aug 17, 2005 00:00:00 AM","value":"2005-08-17T00:00:00.000-07:00"},"mlaCitation":"Sage, Julien. ?Making Young Tumors Old: a New Weapon against Cancer?? <i>Science of aging knowledge environment : SAGE KE</i> 2005.33 (2005): pe25-? Print.","pubMedId":"16107660","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/16107660","publicationId":44091,"title":"Making young tumors old: a new weapon against cancer?","type":"article"},{"abstract":"Certain cells of the human retina are extremely sensitive to loss of function of the retinoblastoma tumor suppressor gene RB. Retinoblastomas develop early in life and at high frequency in individuals heterozygous for a germ-line RB mutation, and sporadic retinoblastomas invariably have somatic mutation in the RB gene. In contrast, retinoblastomas do not develop in Rb+/- mice. Although retinoblastoma is thought to have developmental origins, the function of Rb in retinal development has not been fully characterized. Here we studied the role of Rb in normal retinal development and in retinoblastoma using conditional Rb mutations in the mouse. In late embryogenesis, Rb-deficient retinas exhibited ectopic S-phase and high levels of p53-independent apoptosis, particularly in the differentiating retinal ganglion cell layer. During postnatal retinal development, loss of Rb led to more widespread retinal apoptosis, and adults showed loss of photoreceptors and bipolar cells. Conditional Rb mutation in the retina did not result in retinoblastoma formation even in a p53-mutant background. However, on a p107- or p130-deficient background, Rb mutation in the retina caused retinal dysplasia or retinoblastoma.","apaCitation":"Macpherson, D., Sage, J., Kim, T., Ho, D., McLaughlin, M. E., &amp; Jacks, T. (2004). Cell type-specific effects of Rb deletion in the murine retina. <i>GENES &amp; DEVELOPMENT</i>, <i>18</i>(14), 1681?1694.","capCitation":"<span class=\"title\">\n  <span>Cell type-specific effects of Rb deletion in the murine retina</span>\n  <i>GENES &amp; DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Macpherson, D., Sage, J., Kim, T., Ho, D., McLaughlin, M. E., Jacks, T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2004</span>; <span class=\"volume\">18 (14)</span><span class=\"pages\">: 1681-1694</span>\n</span>","chicagoCitation":"Macpherson, D., J. Sage, T. Kim, D. Ho, M. E. McLaughlin, et al. 2004. ?Cell Type-Specific Effects of Rb Deletion in the Murine Retina.? <i>GENES &amp; DEVELOPMENT</i> 18 (14). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 1681?94.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Certain cells of the human retina are extremely sensitive to loss of function of the retinoblastoma tumor suppressor gene RB. Retinoblastomas develop early in life and at high frequency in individuals heterozygous for a germ-line RB mutation, and sporadic retinoblastomas invariably have somatic mutation in the RB gene. In contrast, retinoblastomas do not develop in Rb+/- mice. Although retinoblastoma is thought to have developmental origins, the function of Rb in retinal development has not been fully characterized. Here we studied the role of Rb in normal retinal development and in retinoblastoma using conditional Rb mutations in the mouse. In late embryogenesis, Rb-deficient retinas exhibited ectopic S-phase and high levels of p53-independent apoptosis, particularly in the differentiating retinal ganglion cell layer. During postnatal retinal development, loss of Rb led to more widespread retinal apoptosis, and adults showed loss of photoreceptors and bipolar cells. Conditional Rb mutation in the retina did not result in retinoblastoma formation even in a p53-mutant background. However, on a p107- or p130-deficient background, Rb mutation in the retina caused retinal dysplasia or retinoblastoma.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.1203304\">DOI 10.1101/gad.1203304</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000222742100005\">Web of Science ID 000222742100005</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/15231717\">PubMedID 15231717</a></span>\n  </p>","text":"Abstract:\nCertain cells of the human retina are extremely sensitive to loss of function of the retinoblastoma tumor suppressor gene RB. Retinoblastomas develop early in life and at high frequency in individuals heterozygous for a germ-line RB mutation, and sporadic retinoblastomas invariably have somatic mutation in the RB gene. In contrast, retinoblastomas do not develop in Rb+/- mice. Although retinoblastoma is thought to have developmental origins, the function of Rb in retinal development has not been fully characterized. Here we studied the role of Rb in normal retinal development and in retinoblastoma using conditional Rb mutations in the mouse. In late embryogenesis, Rb-deficient retinas exhibited ectopic S-phase and high levels of p53-independent apoptosis, particularly in the differentiating retinal ganglion cell layer. During postnatal retinal development, loss of Rb led to more widespread retinal apoptosis, and adults showed loss of photoreceptors and bipolar cells. Conditional Rb mutation in the retina did not result in retinoblastoma formation even in a p53-mutant background. However, on a p107- or p130-deficient background, Rb mutation in the retina caused retinal dysplasia or retinoblastoma.\nDOI: 10.1101/gad.1203304, https://dx.doi.org/10.1101/gad.1203304\nWeb of Science ID: 000222742100005, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000222742100005\nPubMedID: 15231717, https://www.ncbi.nlm.nih.gov/pubmed/15231717"},"doiId":"10.1101/gad.1203304","doiUrl":"https://dx.doi.org/10.1101/gad.1203304","featured":false,"firstPublished":{"html":"Jul 15, 2004 00:00:00 AM","text":"Jul 15, 2004 00:00:00 AM","value":"2004-07-15T00:00:00.000-07:00"},"mlaCitation":"Macpherson, D. et al. ?Cell Type-Specific Effects of Rb Deletion in the Murine Retina.? <i>GENES &amp; DEVELOPMENT</i> 18.14 (2004): 1681?1694. Print.","pubMedId":"15231717","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/15231717","publicationId":34410,"title":"Cell type-specific effects of Rb deletion in the murine retina","type":"article","webOfScienceId":"000222742100005","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000222742100005"},{"abstract":"The retinoblastoma (RB) tumor suppressor has been proposed to function as a key mediator of cell cycle checkpoints induced by chemotherapeutic agents. However, these prior studies have relied on embryonic fibroblasts harboring chronic loss of RB, a condition under which compensation of RB functions is known to occur. Here we utilized primary adult fibroblasts derived from mice harboring loxP sites flanking exon 3 of the Rb gene to delineate the action of RB in the chemotherapeutic response. In this system we find that targeted disruption of Rb leads to little overt change in cell cycle distribution. However, these cells exhibited deregulation of RB/E2F target genes and became aneuploid following culture in the absence of RB. When challenged with both DNA damaging and antimetabolite chemotherapeutics, RB was required for primary adult cells to undergo DNA damage checkpoint responses and loss of RB resulted in enhanced aneuploidy following challenge. In contrast, following spontaneous immortalization and the loss of functional p53 signaling, the antimetabolite 5-fluorouracil (5-FU) failed to induce arrest despite the presence of RB. In these immortal cultures RB/E2F targets were deregulated in a complex, gene-specific manner and RB was required for the checkpoint response to camptothecin (CPT). Mechanistic analyses of the checkpoint responses in primary cells indicated that loss of RB leads to increased p53 signaling and decreased viability following both CPT and 5-FU treatment. However, the mechanism through which these agents act to facilitate cell cycle inhibition through RB were distinct. These studies underscore the critical role of RB in DNA-damage checkpoint signaling and demonstrate that RB mediates chemotherapeutic-induced cell cycle inhibition in adult fibroblasts by distinct mechanisms.","apaCitation":"Mayhew, C. N., Perkin, L. M., Zhang, X. P., Sage, J., Jacks, T., &amp; Knudsen, E. S. (2004). Discrete signaling pathways participate in RB-dependent responses to chemotherapeutic agents. <i>ONCOGENE</i>, <i>23</i>(23), 4107?4120.","capCitation":"<span class=\"title\">\n  <span>Discrete signaling pathways participate in RB-dependent responses to chemotherapeutic agents</span>\n  <i>ONCOGENE</i>\n</span>\n<span class=\"authors\">Mayhew, C. N., Perkin, L. M., Zhang, X. P., Sage, J., Jacks, T., Knudsen, E. S.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2004</span>; <span class=\"volume\">23 (23)</span><span class=\"pages\">: 4107-4120</span>\n</span>","chicagoCitation":"Mayhew, C. N., L. M. Perkin, X. P. Zhang, J. Sage, T. Jacks, et al. 2004. ?Discrete Signaling Pathways Participate in RB-Dependent Responses to Chemotherapeutic Agents.? <i>ONCOGENE</i> 23 (23). NATURE PUBLISHING GROUP: 4107?20.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma (RB) tumor suppressor has been proposed to function as a key mediator of cell cycle checkpoints induced by chemotherapeutic agents. However, these prior studies have relied on embryonic fibroblasts harboring chronic loss of RB, a condition under which compensation of RB functions is known to occur. Here we utilized primary adult fibroblasts derived from mice harboring loxP sites flanking exon 3 of the Rb gene to delineate the action of RB in the chemotherapeutic response. In this system we find that targeted disruption of Rb leads to little overt change in cell cycle distribution. However, these cells exhibited deregulation of RB/E2F target genes and became aneuploid following culture in the absence of RB. When challenged with both DNA damaging and antimetabolite chemotherapeutics, RB was required for primary adult cells to undergo DNA damage checkpoint responses and loss of RB resulted in enhanced aneuploidy following challenge. In contrast, following spontaneous immortalization and the loss of functional p53 signaling, the antimetabolite 5-fluorouracil (5-FU) failed to induce arrest despite the presence of RB. In these immortal cultures RB/E2F targets were deregulated in a complex, gene-specific manner and RB was required for the checkpoint response to camptothecin (CPT). Mechanistic analyses of the checkpoint responses in primary cells indicated that loss of RB leads to increased p53 signaling and decreased viability following both CPT and 5-FU treatment. However, the mechanism through which these agents act to facilitate cell cycle inhibition through RB were distinct. These studies underscore the critical role of RB in DNA-damage checkpoint signaling and demonstrate that RB mediates chemotherapeutic-induced cell cycle inhibition in adult fibroblasts by distinct mechanisms.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/sj.onc.1207503\">DOI 10.1038/sj.onc.1207503</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000221520200009\">Web of Science ID 000221520200009</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/15064736\">PubMedID 15064736</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma (RB) tumor suppressor has been proposed to function as a key mediator of cell cycle checkpoints induced by chemotherapeutic agents. However, these prior studies have relied on embryonic fibroblasts harboring chronic loss of RB, a condition under which compensation of RB functions is known to occur. Here we utilized primary adult fibroblasts derived from mice harboring loxP sites flanking exon 3 of the Rb gene to delineate the action of RB in the chemotherapeutic response. In this system we find that targeted disruption of Rb leads to little overt change in cell cycle distribution. However, these cells exhibited deregulation of RB/E2F target genes and became aneuploid following culture in the absence of RB. When challenged with both DNA damaging and antimetabolite chemotherapeutics, RB was required for primary adult cells to undergo DNA damage checkpoint responses and loss of RB resulted in enhanced aneuploidy following challenge. In contrast, following spontaneous immortalization and the loss of functional p53 signaling, the antimetabolite 5-fluorouracil (5-FU) failed to induce arrest despite the presence of RB. In these immortal cultures RB/E2F targets were deregulated in a complex, gene-specific manner and RB was required for the checkpoint response to camptothecin (CPT). Mechanistic analyses of the checkpoint responses in primary cells indicated that loss of RB leads to increased p53 signaling and decreased viability following both CPT and 5-FU treatment. However, the mechanism through which these agents act to facilitate cell cycle inhibition through RB were distinct. These studies underscore the critical role of RB in DNA-damage checkpoint signaling and demonstrate that RB mediates chemotherapeutic-induced cell cycle inhibition in adult fibroblasts by distinct mechanisms.\nDOI: 10.1038/sj.onc.1207503, https://dx.doi.org/10.1038/sj.onc.1207503\nWeb of Science ID: 000221520200009, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000221520200009\nPubMedID: 15064736, https://www.ncbi.nlm.nih.gov/pubmed/15064736"},"doiId":"10.1038/sj.onc.1207503","doiUrl":"https://dx.doi.org/10.1038/sj.onc.1207503","featured":false,"firstPublished":{"html":"May 20, 2004 00:00:00 AM","text":"May 20, 2004 00:00:00 AM","value":"2004-05-20T00:00:00.000-07:00"},"mlaCitation":"Mayhew, C. N. et al. ?Discrete Signaling Pathways Participate in RB-Dependent Responses to Chemotherapeutic Agents.? <i>ONCOGENE</i> 23.23 (2004): 4107?4120. Print.","pubMedId":"15064736","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/15064736","publicationId":242873,"title":"Discrete signaling pathways participate in RB-dependent responses to chemotherapeutic agents","type":"article","webOfScienceId":"000221520200009","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000221520200009"},{"abstract":"From yeast to humans, cell cycle progression is orchestrated by the oscillation of kinase activities associated with cyclins. In an article published recently in Cell, Ren and Rollins investigate mechanisms controlling the G0/G1 transition in quiescent cells and identify new cyclin C/Cdk3 complexes as key regulators of cell cycle reentry in human cells.","apaCitation":"Sage, J. (2004). Cyclin C makes an entry into the cell cycle. <i>DEVELOPMENTAL CELL</i>, <i>6</i>(5), 607?608.","capCitation":"<span class=\"title\">\n  <span>Cyclin C makes an entry into the cell cycle</span>\n  <i>DEVELOPMENTAL CELL</i>\n</span>\n<span class=\"authors\">Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2004</span>; <span class=\"volume\">6 (5)</span><span class=\"pages\">: 607-608</span>\n</span>","chicagoCitation":"Sage, J. 2004. ?Cyclin C Makes an Entry into the Cell Cycle.? <i>DEVELOPMENTAL CELL</i> 6 (5). CELL PRESS: 607?8.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">From yeast to humans, cell cycle progression is orchestrated by the oscillation of kinase activities associated with cyclins. In an article published recently in Cell, Ren and Rollins investigate mechanisms controlling the G0/G1 transition in quiescent cells and identify new cyclin C/Cdk3 complexes as key regulators of cell cycle reentry in human cells.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000222443100001\">Web of Science ID 000222443100001</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/15130482\">PubMedID 15130482</a></span>\n  </p>","text":"Abstract:\nFrom yeast to humans, cell cycle progression is orchestrated by the oscillation of kinase activities associated with cyclins. In an article published recently in Cell, Ren and Rollins investigate mechanisms controlling the G0/G1 transition in quiescent cells and identify new cyclin C/Cdk3 complexes as key regulators of cell cycle reentry in human cells.\n\nWeb of Science ID: 000222443100001, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000222443100001\nPubMedID: 15130482, https://www.ncbi.nlm.nih.gov/pubmed/15130482"},"featured":false,"firstPublished":{"html":"May 1, 2004 00:00:00 AM","text":"May 1, 2004 00:00:00 AM","value":"2004-05-01T00:00:00.000-07:00"},"mlaCitation":"Sage, J. ?Cyclin C Makes an Entry into the Cell Cycle.? <i>DEVELOPMENTAL CELL</i> 6.5 (2004): 607?608. Print.","pubMedId":"15130482","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/15130482","publicationId":33256,"title":"Cyclin C makes an entry into the cell cycle","type":"article","webOfScienceId":"000222443100001","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000222443100001"},{"abstract":"Cell cycle checkpoints induced by DNA damage play an integral role in preservation of genomic stability by allowing cells to limit the propagation of deleterious mutations. The retinoblastoma tumor suppressor (RB) is crucial for the maintenance of the DNA damage checkpoint function because it elicits cell cycle arrest in response to a variety of genotoxic stresses. Although sporadic loss of RB is characteristic of most cancers and results in the bypass of the DNA damage checkpoint, the consequence of RB loss upon chemotherapeutic responsiveness has been largely uninvestigated. Here, we employed a conditional knockout approach to ablate RB in adult fibroblasts. This system enabled us to examine the DNA damage response of adult cells following acute RB deletion. Using this system, we demonstrated that loss of RB disrupted the DNA damage checkpoint elicited by either cisplatin or camptothecin exposure. Strikingly, this bypass was not associated with enhanced repair, but rather the accumulation of phosphorylated H2AX (gammaH2AX) foci, which indicate DNA double-strand breaks. The formation of gammaH2AX foci was due to ongoing replication following chemotherapeutic treatment in the RB-deficient cells. Additionally, peak gammaH2AX accumulation occurred in S-phase cells undergoing DNA replication in the presence of damage, and these gammaH2AX foci co-localized with replication foci. These results demonstrate that acute RB loss abrogates DNA damage-induced cell cycle arrest to induce gammaH2AX foci formation. Thus, secondary genetic lesions induced by RB loss have implications for the chemotherapeutic response and the development of genetic instability.","apaCitation":"Bosco, E. E., Mayhew, C. N., Hennigan, R. F., Sage, J., Jacks, T., &amp; Knudsen, E. S. (2004). RB signaling prevents replication-dependent DNA double-strand breaks following genotoxic insult. <i>NUCLEIC ACIDS RESEARCH</i>, <i>32</i>(1), 25?34.","capCitation":"<span class=\"title\">\n  <span>RB signaling prevents replication-dependent DNA double-strand breaks following genotoxic insult</span>\n  <i>NUCLEIC ACIDS RESEARCH</i>\n</span>\n<span class=\"authors\">Bosco, E. E., Mayhew, C. N., Hennigan, R. F., Sage, J., Jacks, T., Knudsen, E. S.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2004</span>; <span class=\"volume\">32 (1)</span><span class=\"pages\">: 25-34</span>\n</span>","chicagoCitation":"Bosco, E. E., C. N. Mayhew, R. F. Hennigan, J. Sage, T. Jacks, et al. 2004. ?RB Signaling Prevents Replication-Dependent DNA Double-Strand Breaks Following Genotoxic Insult.? <i>NUCLEIC ACIDS RESEARCH</i> 32 (1). OXFORD UNIV PRESS: 25?34.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Cell cycle checkpoints induced by DNA damage play an integral role in preservation of genomic stability by allowing cells to limit the propagation of deleterious mutations. The retinoblastoma tumor suppressor (RB) is crucial for the maintenance of the DNA damage checkpoint function because it elicits cell cycle arrest in response to a variety of genotoxic stresses. Although sporadic loss of RB is characteristic of most cancers and results in the bypass of the DNA damage checkpoint, the consequence of RB loss upon chemotherapeutic responsiveness has been largely uninvestigated. Here, we employed a conditional knockout approach to ablate RB in adult fibroblasts. This system enabled us to examine the DNA damage response of adult cells following acute RB deletion. Using this system, we demonstrated that loss of RB disrupted the DNA damage checkpoint elicited by either cisplatin or camptothecin exposure. Strikingly, this bypass was not associated with enhanced repair, but rather the accumulation of phosphorylated H2AX (gammaH2AX) foci, which indicate DNA double-strand breaks. The formation of gammaH2AX foci was due to ongoing replication following chemotherapeutic treatment in the RB-deficient cells. Additionally, peak gammaH2AX accumulation occurred in S-phase cells undergoing DNA replication in the presence of damage, and these gammaH2AX foci co-localized with replication foci. These results demonstrate that acute RB loss abrogates DNA damage-induced cell cycle arrest to induce gammaH2AX foci formation. Thus, secondary genetic lesions induced by RB loss have implications for the chemotherapeutic response and the development of genetic instability.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1093/nar/gkg919\">DOI 10.1093/nar/gkg919</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000188988700018\">Web of Science ID 000188988700018</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/14704340\">PubMedID 14704340</a></span>\n  </p>","text":"Abstract:\nCell cycle checkpoints induced by DNA damage play an integral role in preservation of genomic stability by allowing cells to limit the propagation of deleterious mutations. The retinoblastoma tumor suppressor (RB) is crucial for the maintenance of the DNA damage checkpoint function because it elicits cell cycle arrest in response to a variety of genotoxic stresses. Although sporadic loss of RB is characteristic of most cancers and results in the bypass of the DNA damage checkpoint, the consequence of RB loss upon chemotherapeutic responsiveness has been largely uninvestigated. Here, we employed a conditional knockout approach to ablate RB in adult fibroblasts. This system enabled us to examine the DNA damage response of adult cells following acute RB deletion. Using this system, we demonstrated that loss of RB disrupted the DNA damage checkpoint elicited by either cisplatin or camptothecin exposure. Strikingly, this bypass was not associated with enhanced repair, but rather the accumulation of phosphorylated H2AX (gammaH2AX) foci, which indicate DNA double-strand breaks. The formation of gammaH2AX foci was due to ongoing replication following chemotherapeutic treatment in the RB-deficient cells. Additionally, peak gammaH2AX accumulation occurred in S-phase cells undergoing DNA replication in the presence of damage, and these gammaH2AX foci co-localized with replication foci. These results demonstrate that acute RB loss abrogates DNA damage-induced cell cycle arrest to induce gammaH2AX foci formation. Thus, secondary genetic lesions induced by RB loss have implications for the chemotherapeutic response and the development of genetic instability.\nDOI: 10.1093/nar/gkg919, https://dx.doi.org/10.1093/nar/gkg919\nWeb of Science ID: 000188988700018, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000188988700018\nPubMedID: 14704340, https://www.ncbi.nlm.nih.gov/pubmed/14704340"},"doiId":"10.1093/nar/gkg919","doiUrl":"https://dx.doi.org/10.1093/nar/gkg919","featured":true,"firstPublished":{"html":"Jan 1, 2004 00:00:00 AM","text":"Jan 1, 2004 00:00:00 AM","value":"2004-01-01T00:00:00.000-08:00"},"mlaCitation":"Bosco, E. E. et al. ?RB Signaling Prevents Replication-Dependent DNA Double-Strand Breaks Following Genotoxic Insult.? <i>NUCLEIC ACIDS RESEARCH</i> 32.1 (2004): 25?34. Print.","pubMedId":"14704340","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/14704340","publicationId":242874,"title":"RB signaling prevents replication-dependent DNA double-strand breaks following genotoxic insult","type":"article","webOfScienceId":"000188988700018","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000188988700018"},{"abstract":"Although the human papillomavirus (HPV) E7 oncogene is known to contribute to the development of human cervical cancer, the mechanisms of its carcinogenesis are poorly understood. The first identified and most recognized function of E7 is its binding to and inactivation of the retinoblastoma tumor suppressor (pRb), but at least 18 other biological activities have also been reported for E7. Thus, it remains unclear which of these many activities contribute to the oncogenic potential of E7. We used a Cre-lox system to abolish pRb expression in the epidermis of transgenic mice and compared the outcome with the effects of E7 expression in the same tissue at early ages. Mice lacking pRb in epidermis showed epithelial hyperplasia, aberrant DNA synthesis, and improper differentiation. In addition, Rb-deleted epidermis (i.e., epidermis composed of cells with Rb deleted) exhibited centrosomal abnormalities and failed to arrest the cell cycle in response to ionizing radiation. Transgenic mice expressing E7 in skin display the same range of phenotypes. In sum, few differences were detected between Rb-deleted epidermis and E7-expressing epidermis in young mice. However, when both E7 was expressed and Rb was deleted in the same tissue, increased hyperplasia and dysplasia were observed. These findings indicate that inactivation of the Rb pathway can largely account for E7's phenotypes at an early age, but that pRb-independent activities of E7 are detectable in vivo.","apaCitation":"Balsitis, S. J., Sage, J., Duensing, S., Munger, K., Jacks, T., &amp; Lambert, P. F. (2003). Recapitulation of the effects of the human papillomavirus type 16 E7 oncogene on mouse epithelium by somatic Rb deletion and detection of pRb-independent effects of E7 in vivo. <i>MOLECULAR AND CELLULAR BIOLOGY</i>, <i>23</i>(24), 9094?9103.","capCitation":"<span class=\"title\">\n  <span>Recapitulation of the effects of the human papillomavirus type 16 E7 oncogene on mouse epithelium by somatic Rb deletion and detection of pRb-independent effects of E7 in vivo</span>\n  <i>MOLECULAR AND CELLULAR BIOLOGY</i>\n</span>\n<span class=\"authors\">Balsitis, S. J., Sage, J., Duensing, S., Munger, K., Jacks, T., Lambert, P. F.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2003</span>; <span class=\"volume\">23 (24)</span><span class=\"pages\">: 9094-9103</span>\n</span>","chicagoCitation":"Balsitis, S. J., J. Sage, S. Duensing, K. Munger, T. Jacks, et al. 2003. ?Recapitulation of the Effects of the Human Papillomavirus Type 16 E7 Oncogene on Mouse Epithelium by Somatic Rb Deletion and Detection of PRb-Independent Effects of E7 in Vivo.? <i>MOLECULAR AND CELLULAR BIOLOGY</i> 23 (24). AMER SOC MICROBIOLOGY: 9094?9103.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Although the human papillomavirus (HPV) E7 oncogene is known to contribute to the development of human cervical cancer, the mechanisms of its carcinogenesis are poorly understood. The first identified and most recognized function of E7 is its binding to and inactivation of the retinoblastoma tumor suppressor (pRb), but at least 18 other biological activities have also been reported for E7. Thus, it remains unclear which of these many activities contribute to the oncogenic potential of E7. We used a Cre-lox system to abolish pRb expression in the epidermis of transgenic mice and compared the outcome with the effects of E7 expression in the same tissue at early ages. Mice lacking pRb in epidermis showed epithelial hyperplasia, aberrant DNA synthesis, and improper differentiation. In addition, Rb-deleted epidermis (i.e., epidermis composed of cells with Rb deleted) exhibited centrosomal abnormalities and failed to arrest the cell cycle in response to ionizing radiation. Transgenic mice expressing E7 in skin display the same range of phenotypes. In sum, few differences were detected between Rb-deleted epidermis and E7-expressing epidermis in young mice. However, when both E7 was expressed and Rb was deleted in the same tissue, increased hyperplasia and dysplasia were observed. These findings indicate that inactivation of the Rb pathway can largely account for E7's phenotypes at an early age, but that pRb-independent activities of E7 are detectable in vivo.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1128/MBC.23.24.9094-9103.2003\">DOI 10.1128/MBC.23.24.9094-9103.2003</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000187040400017\">Web of Science ID 000187040400017</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/14645521\">PubMedID 14645521</a></span>\n  </p>","text":"Abstract:\nAlthough the human papillomavirus (HPV) E7 oncogene is known to contribute to the development of human cervical cancer, the mechanisms of its carcinogenesis are poorly understood. The first identified and most recognized function of E7 is its binding to and inactivation of the retinoblastoma tumor suppressor (pRb), but at least 18 other biological activities have also been reported for E7. Thus, it remains unclear which of these many activities contribute to the oncogenic potential of E7. We used a Cre-lox system to abolish pRb expression in the epidermis of transgenic mice and compared the outcome with the effects of E7 expression in the same tissue at early ages. Mice lacking pRb in epidermis showed epithelial hyperplasia, aberrant DNA synthesis, and improper differentiation. In addition, Rb-deleted epidermis (i.e., epidermis composed of cells with Rb deleted) exhibited centrosomal abnormalities and failed to arrest the cell cycle in response to ionizing radiation. Transgenic mice expressing E7 in skin display the same range of phenotypes. In sum, few differences were detected between Rb-deleted epidermis and E7-expressing epidermis in young mice. However, when both E7 was expressed and Rb was deleted in the same tissue, increased hyperplasia and dysplasia were observed. These findings indicate that inactivation of the Rb pathway can largely account for E7's phenotypes at an early age, but that pRb-independent activities of E7 are detectable in vivo.\nDOI: 10.1128/MBC.23.24.9094-9103.2003, https://dx.doi.org/10.1128/MBC.23.24.9094-9103.2003\nWeb of Science ID: 000187040400017, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000187040400017\nPubMedID: 14645521, https://www.ncbi.nlm.nih.gov/pubmed/14645521"},"doiId":"10.1128/MBC.23.24.9094-9103.2003","doiUrl":"https://dx.doi.org/10.1128/MBC.23.24.9094-9103.2003","featured":false,"firstPublished":{"html":"Dec 1, 2003 00:00:00 AM","text":"Dec 1, 2003 00:00:00 AM","value":"2003-12-01T00:00:00.000-08:00"},"mlaCitation":"Balsitis, S. J. et al. ?Recapitulation of the Effects of the Human Papillomavirus Type 16 E7 Oncogene on Mouse Epithelium by Somatic Rb Deletion and Detection of PRb-Independent Effects of E7 in Vivo.? <i>MOLECULAR AND CELLULAR BIOLOGY</i> 23.24 (2003): 9094?9103. Print.","pubMedId":"14645521","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/14645521","publicationId":242875,"title":"Recapitulation of the effects of the human papillomavirus type 16 E7 oncogene on mouse epithelium by somatic Rb deletion and detection of pRb-independent effects of E7 in vivo","type":"article","webOfScienceId":"000187040400017","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000187040400017"},{"abstract":"The induction of apoptosis by the p53 protein is critical for its activity as a tumor suppressor. Although it is clear that p53 induces apoptosis at least in part by direct transcriptional activation of target genes, the set of p53 target genes that mediate p53 function in apoptosis in vivo remains to be well defined. The Perp (p53 apoptosis effector related to PMP-22) gene is highly expressed in cells undergoing p53-dependent apoptosis as compared to cells undergoing p53-dependent G1 arrest. Perp is a direct p53 target, and its overexpression is sufficient to induce cell death in fibroblasts, implicating it as an important component of p53 apoptotic function. Here, through the generation of Perp-deficient mice, we analyze the role of Perp in the p53 apoptosis pathway in multiple primary cell types by comparing the cell death responses of Perp null cells to those of wild-type and p53 null cells. These experiments demonstrate the involvement of Perp in p53-mediated cell death in thymocytes and neurons but not in E1A-expressing MEFs, indicating a cell type-specific role for Perp in the p53 cell death pathway. In addition, we show that Perp is not required for proliferation-associated functions of p53. Thus, Perp selectively mediates the p53 apoptotic response, and the requirement for Perp is dictated by cellular context.","apaCitation":"Ihrie, R. A., Reczek, E., Horner, J. S., Khachatrian, L., Sage, J., Jacks, T., &amp; Attardi, L. D. (2003). Perp is a mediator of p53-dependent apoptosis in diverse cell types. <i>CURRENT BIOLOGY</i>, <i>13</i>(22), 1985?1990.","capCitation":"<span class=\"title\">\n  <span>Perp is a mediator of p53-dependent apoptosis in diverse cell types</span>\n  <i>CURRENT BIOLOGY</i>\n</span>\n<span class=\"authors\">Ihrie, R. A., Reczek, E., Horner, J. S., Khachatrian, L., Sage, J., Jacks, T., Attardi, L. D.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2003</span>; <span class=\"volume\">13 (22)</span><span class=\"pages\">: 1985-1990</span>\n</span>","chicagoCitation":"Ihrie, R. A., E. Reczek, J. S. Horner, L. Khachatrian, J. Sage, et al. 2003. ?Perp Is a Mediator of p53-Dependent Apoptosis in Diverse Cell Types.? <i>CURRENT BIOLOGY</i> 13 (22). CELL PRESS: 1985?90.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The induction of apoptosis by the p53 protein is critical for its activity as a tumor suppressor. Although it is clear that p53 induces apoptosis at least in part by direct transcriptional activation of target genes, the set of p53 target genes that mediate p53 function in apoptosis in vivo remains to be well defined. The Perp (p53 apoptosis effector related to PMP-22) gene is highly expressed in cells undergoing p53-dependent apoptosis as compared to cells undergoing p53-dependent G1 arrest. Perp is a direct p53 target, and its overexpression is sufficient to induce cell death in fibroblasts, implicating it as an important component of p53 apoptotic function. Here, through the generation of Perp-deficient mice, we analyze the role of Perp in the p53 apoptosis pathway in multiple primary cell types by comparing the cell death responses of Perp null cells to those of wild-type and p53 null cells. These experiments demonstrate the involvement of Perp in p53-mediated cell death in thymocytes and neurons but not in E1A-expressing MEFs, indicating a cell type-specific role for Perp in the p53 cell death pathway. In addition, we show that Perp is not required for proliferation-associated functions of p53. Thus, Perp selectively mediates the p53 apoptotic response, and the requirement for Perp is dictated by cellular context.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.cub.2003.10.055\">DOI 10.1016/j.cub.2003.10.055</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000186558100026\">Web of Science ID 000186558100026</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/14614825\">PubMedID 14614825</a></span>\n  </p>","text":"Abstract:\nThe induction of apoptosis by the p53 protein is critical for its activity as a tumor suppressor. Although it is clear that p53 induces apoptosis at least in part by direct transcriptional activation of target genes, the set of p53 target genes that mediate p53 function in apoptosis in vivo remains to be well defined. The Perp (p53 apoptosis effector related to PMP-22) gene is highly expressed in cells undergoing p53-dependent apoptosis as compared to cells undergoing p53-dependent G1 arrest. Perp is a direct p53 target, and its overexpression is sufficient to induce cell death in fibroblasts, implicating it as an important component of p53 apoptotic function. Here, through the generation of Perp-deficient mice, we analyze the role of Perp in the p53 apoptosis pathway in multiple primary cell types by comparing the cell death responses of Perp null cells to those of wild-type and p53 null cells. These experiments demonstrate the involvement of Perp in p53-mediated cell death in thymocytes and neurons but not in E1A-expressing MEFs, indicating a cell type-specific role for Perp in the p53 cell death pathway. In addition, we show that Perp is not required for proliferation-associated functions of p53. Thus, Perp selectively mediates the p53 apoptotic response, and the requirement for Perp is dictated by cellular context.\nDOI: 10.1016/j.cub.2003.10.055, https://dx.doi.org/10.1016/j.cub.2003.10.055\nWeb of Science ID: 000186558100026, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000186558100026\nPubMedID: 14614825, https://www.ncbi.nlm.nih.gov/pubmed/14614825"},"doiId":"10.1016/j.cub.2003.10.055","doiUrl":"https://dx.doi.org/10.1016/j.cub.2003.10.055","featured":false,"firstPublished":{"html":"Nov 11, 2003 00:00:00 AM","text":"Nov 11, 2003 00:00:00 AM","value":"2003-11-11T00:00:00.000-08:00"},"mlaCitation":"Ihrie, R. A. et al. ?Perp Is a Mediator of p53-Dependent Apoptosis in Diverse Cell Types.? <i>CURRENT BIOLOGY</i> 13.22 (2003): 1985?1990. Print.","pubMedId":"14614825","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/14614825","publicationId":29469,"title":"Perp is a mediator of p53-dependent apoptosis in diverse cell types","type":"article","webOfScienceId":"000186558100026","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000186558100026"},{"abstract":"Cancer cells arise from normal cells through the acquisition of a series of mutations in oncogenes and tumour suppressor genes. Mouse models of human cancer often rely on germline alterations that activate or inactivate genes of interest. One limitation of this approach is that germline mutations might have effects other than somatic mutations, owing to developmental compensation. To model sporadic cancers associated with inactivation of the retinoblastoma (RB) tumour suppressor gene in humans, we have produced a conditional allele of the mouse Rb gene. We show here that acute loss of Rb in primary quiescent cells is sufficient for cell cycle entry and has phenotypic consequences different from germline loss of Rb function. This difference is explained in part by functional compensation by the Rb-related gene p107. We also show that acute loss of Rb in senescent cells leads to reversal of the cellular senescence programme. Thus, the use of conditional knockout strategies might refine our understanding of gene function and help to model human cancer more accurately.","apaCitation":"Sage, J., Miller, A. L., Perez-Mancera, P. A., Wysocki, J. M., &amp; Jacks, T. (2003). Acute mutation of retinoblastoma gene function is sufficient for cell cycle re-entry. <i>NATURE</i>, <i>424</i>(6945), 223?228.","capCitation":"<span class=\"title\">\n  <span>Acute mutation of retinoblastoma gene function is sufficient for cell cycle re-entry</span>\n  <i>NATURE</i>\n</span>\n<span class=\"authors\">Sage, J., Miller, A. L., Perez-Mancera, P. A., Wysocki, J. M., Jacks, T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2003</span>; <span class=\"volume\">424 (6945)</span><span class=\"pages\">: 223-228</span>\n</span>","chicagoCitation":"Sage, J., A. L. Miller, P. A. Perez-Mancera, J. M. Wysocki, and T. Jacks. 2003. ?Acute Mutation of Retinoblastoma Gene Function Is Sufficient for Cell Cycle Re-Entry.? <i>NATURE</i> 424 (6945). NATURE PUBLISHING GROUP: 223?28.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Cancer cells arise from normal cells through the acquisition of a series of mutations in oncogenes and tumour suppressor genes. Mouse models of human cancer often rely on germline alterations that activate or inactivate genes of interest. One limitation of this approach is that germline mutations might have effects other than somatic mutations, owing to developmental compensation. To model sporadic cancers associated with inactivation of the retinoblastoma (RB) tumour suppressor gene in humans, we have produced a conditional allele of the mouse Rb gene. We show here that acute loss of Rb in primary quiescent cells is sufficient for cell cycle entry and has phenotypic consequences different from germline loss of Rb function. This difference is explained in part by functional compensation by the Rb-related gene p107. We also show that acute loss of Rb in senescent cells leads to reversal of the cellular senescence programme. Thus, the use of conditional knockout strategies might refine our understanding of gene function and help to model human cancer more accurately.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nature01764\">DOI 10.1038/nature01764</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000184032700049\">Web of Science ID 000184032700049</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/12853964\">PubMedID 12853964</a></span>\n  </p>","text":"Abstract:\nCancer cells arise from normal cells through the acquisition of a series of mutations in oncogenes and tumour suppressor genes. Mouse models of human cancer often rely on germline alterations that activate or inactivate genes of interest. One limitation of this approach is that germline mutations might have effects other than somatic mutations, owing to developmental compensation. To model sporadic cancers associated with inactivation of the retinoblastoma (RB) tumour suppressor gene in humans, we have produced a conditional allele of the mouse Rb gene. We show here that acute loss of Rb in primary quiescent cells is sufficient for cell cycle entry and has phenotypic consequences different from germline loss of Rb function. This difference is explained in part by functional compensation by the Rb-related gene p107. We also show that acute loss of Rb in senescent cells leads to reversal of the cellular senescence programme. Thus, the use of conditional knockout strategies might refine our understanding of gene function and help to model human cancer more accurately.\nDOI: 10.1038/nature01764, https://dx.doi.org/10.1038/nature01764\nWeb of Science ID: 000184032700049, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000184032700049\nPubMedID: 12853964, https://www.ncbi.nlm.nih.gov/pubmed/12853964"},"doiId":"10.1038/nature01764","doiUrl":"https://dx.doi.org/10.1038/nature01764","featured":true,"firstPublished":{"html":"Jul 10, 2003 00:00:00 AM","text":"Jul 10, 2003 00:00:00 AM","value":"2003-07-10T00:00:00.000-07:00"},"mlaCitation":"Sage, J. et al. ?Acute Mutation of Retinoblastoma Gene Function Is Sufficient for Cell Cycle Re-Entry.? <i>NATURE</i> 424.6945 (2003): 223?228. Print.","pubMedId":"12853964","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/12853964","publicationId":24327,"title":"Acute mutation of retinoblastoma gene function is sufficient for cell cycle re-entry","type":"article","webOfScienceId":"000184032700049","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000184032700049"},{"abstract":"Targeted disruption of the retinoblastoma gene in mice leads to embryonic lethality in midgestation accompanied by defective erythropoiesis. Rb(-/-) embryos also exhibit inappropriate cell cycle activity and apoptosis in the central nervous system (CNS), peripheral nervous system (PNS), and ocular lens. Loss of p53 can prevent the apoptosis in the CNS and lens; however, the specific signals leading to p53 activation have not been determined. Here we test the hypothesis that hypoxia caused by defective erythropoiesis in Rb-null embryos contributes to p53-dependent apoptosis. We show evidence of hypoxia in CNS tissue from Rb(-/-) embryos. The Cre-loxP system was then used to generate embryos in which Rb was deleted in the CNS, PNS and lens, in the presence of normal erythropoiesis. In contrast to the massive CNS apoptosis in Rb-null embryos at embryonic day 13.5 (E13.5), conditional mutants did not have elevated apoptosis in this tissue. There was still significant apoptosis in the PNS and lens, however. Rb(-/-) cells in the CNS, PNS, and lens underwent inappropriate S-phase entry in the conditional mutants at E13.5. By E18.5, conditional mutants had increased brain size and weight as well as defects in skeletal muscle development. These data support a model in which hypoxia is a necessary cofactor in the death of CNS neurons in the developing Rb mutant embryo.","apaCitation":"Macpherson, D., Sage, J., Crowley, D., Trumpp, A., Bronson, R. T., &amp; Jacks, T. (2003). Conditional mutation of Rb causes cell cycle defects without apoptosis in the central nervous system. <i>MOLECULAR AND CELLULAR BIOLOGY</i>, <i>23</i>(3), 1044?1053.","capCitation":"<span class=\"title\">\n  <span>Conditional mutation of Rb causes cell cycle defects without apoptosis in the central nervous system</span>\n  <i>MOLECULAR AND CELLULAR BIOLOGY</i>\n</span>\n<span class=\"authors\">Macpherson, D., Sage, J., Crowley, D., Trumpp, A., Bronson, R. T., Jacks, T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2003</span>; <span class=\"volume\">23 (3)</span><span class=\"pages\">: 1044-1053</span>\n</span>","chicagoCitation":"Macpherson, D., J. Sage, D. Crowley, A. Trumpp, R. T. Bronson, et al. 2003. ?Conditional Mutation of Rb Causes Cell Cycle Defects without Apoptosis in the Central Nervous System.? <i>MOLECULAR AND CELLULAR BIOLOGY</i> 23 (3). AMER SOC MICROBIOLOGY: 1044?53.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Targeted disruption of the retinoblastoma gene in mice leads to embryonic lethality in midgestation accompanied by defective erythropoiesis. Rb(-/-) embryos also exhibit inappropriate cell cycle activity and apoptosis in the central nervous system (CNS), peripheral nervous system (PNS), and ocular lens. Loss of p53 can prevent the apoptosis in the CNS and lens; however, the specific signals leading to p53 activation have not been determined. Here we test the hypothesis that hypoxia caused by defective erythropoiesis in Rb-null embryos contributes to p53-dependent apoptosis. We show evidence of hypoxia in CNS tissue from Rb(-/-) embryos. The Cre-loxP system was then used to generate embryos in which Rb was deleted in the CNS, PNS and lens, in the presence of normal erythropoiesis. In contrast to the massive CNS apoptosis in Rb-null embryos at embryonic day 13.5 (E13.5), conditional mutants did not have elevated apoptosis in this tissue. There was still significant apoptosis in the PNS and lens, however. Rb(-/-) cells in the CNS, PNS, and lens underwent inappropriate S-phase entry in the conditional mutants at E13.5. By E18.5, conditional mutants had increased brain size and weight as well as defects in skeletal muscle development. These data support a model in which hypoxia is a necessary cofactor in the death of CNS neurons in the developing Rb mutant embryo.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1128/MCB.23.3.1044-1053.2003\">DOI 10.1128/MCB.23.3.1044-1053.2003</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000180673600026\">Web of Science ID 000180673600026</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/12529408\">PubMedID 12529408</a></span>\n  </p>","text":"Abstract:\nTargeted disruption of the retinoblastoma gene in mice leads to embryonic lethality in midgestation accompanied by defective erythropoiesis. Rb(-/-) embryos also exhibit inappropriate cell cycle activity and apoptosis in the central nervous system (CNS), peripheral nervous system (PNS), and ocular lens. Loss of p53 can prevent the apoptosis in the CNS and lens; however, the specific signals leading to p53 activation have not been determined. Here we test the hypothesis that hypoxia caused by defective erythropoiesis in Rb-null embryos contributes to p53-dependent apoptosis. We show evidence of hypoxia in CNS tissue from Rb(-/-) embryos. The Cre-loxP system was then used to generate embryos in which Rb was deleted in the CNS, PNS and lens, in the presence of normal erythropoiesis. In contrast to the massive CNS apoptosis in Rb-null embryos at embryonic day 13.5 (E13.5), conditional mutants did not have elevated apoptosis in this tissue. There was still significant apoptosis in the PNS and lens, however. Rb(-/-) cells in the CNS, PNS, and lens underwent inappropriate S-phase entry in the conditional mutants at E13.5. By E18.5, conditional mutants had increased brain size and weight as well as defects in skeletal muscle development. These data support a model in which hypoxia is a necessary cofactor in the death of CNS neurons in the developing Rb mutant embryo.\nDOI: 10.1128/MCB.23.3.1044-1053.2003, https://dx.doi.org/10.1128/MCB.23.3.1044-1053.2003\nWeb of Science ID: 000180673600026, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000180673600026\nPubMedID: 12529408, https://www.ncbi.nlm.nih.gov/pubmed/12529408"},"doiId":"10.1128/MCB.23.3.1044-1053.2003","doiUrl":"https://dx.doi.org/10.1128/MCB.23.3.1044-1053.2003","featured":false,"firstPublished":{"html":"Feb 1, 2003 00:00:00 AM","text":"Feb 1, 2003 00:00:00 AM","value":"2003-02-01T00:00:00.000-08:00"},"mlaCitation":"Macpherson, D. et al. ?Conditional Mutation of Rb Causes Cell Cycle Defects without Apoptosis in the Central Nervous System.? <i>MOLECULAR AND CELLULAR BIOLOGY</i> 23.3 (2003): 1044?1053. Print.","pubMedId":"12529408","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/12529408","publicationId":21084,"title":"Conditional mutation of Rb causes cell cycle defects without apoptosis in the central nervous system","type":"article","webOfScienceId":"000180673600026","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000180673600026"},{"abstract":"Defining the molecular mechanisms that coordinately regulate proliferation and differentiation is a central issue in development. Here, we describe a mechanism in which induction of the Ets repressor METS/PE1 links terminal differentiation to cell cycle arrest. Using macrophages as a model, we provide evidence that METS/PE1 blocks Ras-dependent proliferation without inhibiting Ras-dependent expression of cell type-specific genes by selectively replacing Ets activators on the promoters of cell cycle control genes. Antiproliferative effects of METS require its interaction with DP103, a DEAD box-containing protein that assembles a novel corepressor complex. Functional interactions between the METS/DP103 complex and E2F/ pRB family proteins are also necessary for inhibition of cellular proliferation, suggesting a combinatorial code that directs permanent cell cycle exit during terminal differentiation.","apaCitation":"Klappacher, G. W., Lunyak, V. V., Sykes, D. B., Sawka-Verhelle, D., Sage, J., Brard, G., ? Glass, C. K. (2002). An induced Ets repressor complex regulates growth arrest during terminal macrophage differentiation. <i>CELL</i>, <i>109</i>(2), 169?180.","capCitation":"<span class=\"title\">\n  <span>An induced Ets repressor complex regulates growth arrest during terminal macrophage differentiation</span>\n  <i>CELL</i>\n</span>\n<span class=\"authors\">Klappacher, G. W., Lunyak, V. V., Sykes, D. B., Sawka-Verhelle, D., Sage, J., Brard, G., Ngo, S. D., Gangadharan, D., Jacks, T., Kamps, M. P., Rose, D. W., Rosenfeld, M. G., Glass, C. K.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2002</span>; <span class=\"volume\">109 (2)</span><span class=\"pages\">: 169-180</span>\n</span>","chicagoCitation":"Klappacher, G. W., V. V. Lunyak, D. B. Sykes, D. Sawka-Verhelle, J. Sage, et al. 2002. ?An Induced Ets Repressor Complex Regulates Growth Arrest during Terminal Macrophage Differentiation.? <i>CELL</i> 109 (2). CELL PRESS: 169?80.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Defining the molecular mechanisms that coordinately regulate proliferation and differentiation is a central issue in development. Here, we describe a mechanism in which induction of the Ets repressor METS/PE1 links terminal differentiation to cell cycle arrest. Using macrophages as a model, we provide evidence that METS/PE1 blocks Ras-dependent proliferation without inhibiting Ras-dependent expression of cell type-specific genes by selectively replacing Ets activators on the promoters of cell cycle control genes. Antiproliferative effects of METS require its interaction with DP103, a DEAD box-containing protein that assembles a novel corepressor complex. Functional interactions between the METS/DP103 complex and E2F/ pRB family proteins are also necessary for inhibition of cellular proliferation, suggesting a combinatorial code that directs permanent cell cycle exit during terminal differentiation.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000175082600007\">Web of Science ID 000175082600007</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/12007404\">PubMedID 12007404</a></span>\n  </p>","text":"Abstract:\nDefining the molecular mechanisms that coordinately regulate proliferation and differentiation is a central issue in development. Here, we describe a mechanism in which induction of the Ets repressor METS/PE1 links terminal differentiation to cell cycle arrest. Using macrophages as a model, we provide evidence that METS/PE1 blocks Ras-dependent proliferation without inhibiting Ras-dependent expression of cell type-specific genes by selectively replacing Ets activators on the promoters of cell cycle control genes. Antiproliferative effects of METS require its interaction with DP103, a DEAD box-containing protein that assembles a novel corepressor complex. Functional interactions between the METS/DP103 complex and E2F/ pRB family proteins are also necessary for inhibition of cellular proliferation, suggesting a combinatorial code that directs permanent cell cycle exit during terminal differentiation.\n\nWeb of Science ID: 000175082600007, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000175082600007\nPubMedID: 12007404, https://www.ncbi.nlm.nih.gov/pubmed/12007404"},"featured":false,"firstPublished":{"html":"Apr 19, 2002 00:00:00 AM","text":"Apr 19, 2002 00:00:00 AM","value":"2002-04-19T00:00:00.000-07:00"},"mlaCitation":"Klappacher, G. W. et al. ?An Induced Ets Repressor Complex Regulates Growth Arrest during Terminal Macrophage Differentiation.? <i>CELL</i> 109.2 (2002): 169?180. Print.","pubMedId":"12007404","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/12007404","publicationId":242876,"title":"An induced Ets repressor complex regulates growth arrest during terminal macrophage differentiation","type":"article","webOfScienceId":"000175082600007","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000175082600007"},{"abstract":"The p53 tumor suppressor gene is the most frequently mutated gene in human cancers, and germ-line p53 mutations cause a familial predisposition for cancer. Germ-line or sporadic p53 mutations are usually missense and typically affect the central DNA-binding domain of the protein. Because p53 functions as a tetrameric transcription factor, mutant p53 is thought to inhibit the function of wild-type p53 protein. Here, we studied the possible dominant-negative inhibition of wild-type p53 protein by two different, frequently occurring point mutations. The R270H and P275S mutations were targeted into the genome of mouse embryonic stem cells to allow the analysis of the effects of the mutant proteins expressed in normal cells at single-copy levels. In embryonic stem cells, the presence of a heterozygous point-mutated allele resulted in delayed transcriptional activation of several p53 downstream target genes on exposure to gamma irradiation. Doxorubicin-induced apoptosis was severely affected in the mutant embryonic stem cells compared with wild-type cells. Heterozygous mutant thymocytes had a severe defect in p53-dependent apoptotic pathways after treatment with gamma irradiation or doxorubicin, whereas p53-independent apoptotic pathways were intact. Together these data demonstrate that physiological expression of point-mutated p53 can strongly limit overall cellular p53 function, supporting the dominant-negative action of such mutants. Also, cells heterozygous for such mutations may be compromised in terms of tumor suppression and response to chemotherapeutic agents.","apaCitation":"de Vries, A., Flores, E. R., Miranda, B., Hsieh, H. M., van Oostrom, C. T. M., Sage, J., &amp; Jacks, T. (2002). Targeted point mutations of p53 lead to dominant-negative inhibition of wild-type p53 function. <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i>, <i>99</i>(5), 2948?2953.","capCitation":"<span class=\"title\">\n  <span>Targeted point mutations of p53 lead to dominant-negative inhibition of wild-type p53 function</span>\n  <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i>\n</span>\n<span class=\"authors\">de Vries, A., Flores, E. R., Miranda, B., Hsieh, H. M., van Oostrom, C. T., Sage, J., Jacks, T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2002</span>; <span class=\"volume\">99 (5)</span><span class=\"pages\">: 2948-2953</span>\n</span>","chicagoCitation":"de Vries, A., E. R. Flores, B. Miranda, H. M. Hsieh, C. TM van Oostrom, et al. 2002. ?Targeted Point Mutations of p53 Lead to Dominant-Negative Inhibition of Wild-Type p53 Function.? <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i> 99 (5). NATL ACAD SCIENCES: 2948?53.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The p53 tumor suppressor gene is the most frequently mutated gene in human cancers, and germ-line p53 mutations cause a familial predisposition for cancer. Germ-line or sporadic p53 mutations are usually missense and typically affect the central DNA-binding domain of the protein. Because p53 functions as a tetrameric transcription factor, mutant p53 is thought to inhibit the function of wild-type p53 protein. Here, we studied the possible dominant-negative inhibition of wild-type p53 protein by two different, frequently occurring point mutations. The R270H and P275S mutations were targeted into the genome of mouse embryonic stem cells to allow the analysis of the effects of the mutant proteins expressed in normal cells at single-copy levels. In embryonic stem cells, the presence of a heterozygous point-mutated allele resulted in delayed transcriptional activation of several p53 downstream target genes on exposure to gamma irradiation. Doxorubicin-induced apoptosis was severely affected in the mutant embryonic stem cells compared with wild-type cells. Heterozygous mutant thymocytes had a severe defect in p53-dependent apoptotic pathways after treatment with gamma irradiation or doxorubicin, whereas p53-independent apoptotic pathways were intact. Together these data demonstrate that physiological expression of point-mutated p53 can strongly limit overall cellular p53 function, supporting the dominant-negative action of such mutants. Also, cells heterozygous for such mutations may be compromised in terms of tumor suppression and response to chemotherapeutic agents.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1073/pnas.052713099\">DOI 10.1073/pnas.052713099</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000174284600065\">Web of Science ID 000174284600065</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/11867759\">PubMedID 11867759</a></span>\n  </p>","text":"Abstract:\nThe p53 tumor suppressor gene is the most frequently mutated gene in human cancers, and germ-line p53 mutations cause a familial predisposition for cancer. Germ-line or sporadic p53 mutations are usually missense and typically affect the central DNA-binding domain of the protein. Because p53 functions as a tetrameric transcription factor, mutant p53 is thought to inhibit the function of wild-type p53 protein. Here, we studied the possible dominant-negative inhibition of wild-type p53 protein by two different, frequently occurring point mutations. The R270H and P275S mutations were targeted into the genome of mouse embryonic stem cells to allow the analysis of the effects of the mutant proteins expressed in normal cells at single-copy levels. In embryonic stem cells, the presence of a heterozygous point-mutated allele resulted in delayed transcriptional activation of several p53 downstream target genes on exposure to gamma irradiation. Doxorubicin-induced apoptosis was severely affected in the mutant embryonic stem cells compared with wild-type cells. Heterozygous mutant thymocytes had a severe defect in p53-dependent apoptotic pathways after treatment with gamma irradiation or doxorubicin, whereas p53-independent apoptotic pathways were intact. Together these data demonstrate that physiological expression of point-mutated p53 can strongly limit overall cellular p53 function, supporting the dominant-negative action of such mutants. Also, cells heterozygous for such mutations may be compromised in terms of tumor suppression and response to chemotherapeutic agents.\nDOI: 10.1073/pnas.052713099, https://dx.doi.org/10.1073/pnas.052713099\nWeb of Science ID: 000174284600065, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000174284600065\nPubMedID: 11867759, https://www.ncbi.nlm.nih.gov/pubmed/11867759"},"doiId":"10.1073/pnas.052713099","doiUrl":"https://dx.doi.org/10.1073/pnas.052713099","featured":false,"firstPublished":{"html":"Mar 5, 2002 00:00:00 AM","text":"Mar 5, 2002 00:00:00 AM","value":"2002-03-05T00:00:00.000-08:00"},"mlaCitation":"de Vries, A. et al. ?Targeted Point Mutations of p53 Lead to Dominant-Negative Inhibition of Wild-Type p53 Function.? <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i> 99.5 (2002): 2948?2953. Print.","pubMedId":"11867759","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/11867759","publicationId":242877,"title":"Targeted point mutations of p53 lead to dominant-negative inhibition of wild-type p53 function","type":"article","webOfScienceId":"000174284600065","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000174284600065"},{"apaCitation":"Sage, J. (2001). The RB gene family and control of the cell cycle. <i>BULLETIN DU CANCER</i>, <i>88</i>(6), 541?543.","capCitation":"<span class=\"title\">\n  <span>The RB gene family and control of the cell cycle</span>\n  <i>BULLETIN DU CANCER</i>\n</span>\n<span class=\"authors\">Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2001</span>; <span class=\"volume\">88 (6)</span><span class=\"pages\">: 541-543</span>\n</span>","chicagoCitation":"Sage, J. 2001. ?The RB Gene Family and Control of the Cell Cycle.? <i>BULLETIN DU CANCER</i> 88 (6). JOHN LIBBEY EUROTEXT LTD: 541?43.","detail":{"html":"<p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000169993400001\">Web of Science ID 000169993400001</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/11459699\">PubMedID 11459699</a></span>\n  </p>","text":"Web of Science ID: 000169993400001, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000169993400001\nPubMedID: 11459699, https://www.ncbi.nlm.nih.gov/pubmed/11459699"},"featured":false,"firstPublished":{"html":"Jun 1, 2001 00:00:00 AM","text":"Jun 1, 2001 00:00:00 AM","value":"2001-06-01T00:00:00.000-07:00"},"mlaCitation":"Sage, J. ?The RB Gene Family and Control of the Cell Cycle.? <i>BULLETIN DU CANCER</i> 88.6 (2001): 541?543. Print.","pubMedId":"11459699","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/11459699","publicationId":242878,"title":"The RB gene family and control of the cell cycle","type":"article","webOfScienceId":"000169993400001","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000169993400001"},{"abstract":"The retinoblastoma protein, pRB, and the closely related proteins p107 and p130 are important regulators of the mammalian cell cycle. Biochemical and genetic studies have demonstrated overlapping as well as distinct functions for the three proteins in cell cycle control and mouse development. However, the role of the pRB family as a whole in the regulation of cell proliferation, cell death, or cell differentiation is not known. We generated embryonic stem (ES) cells and other cell types mutant for all three genes. Triple knock-out mouse embryonic fibroblasts (TKO MEFs) had a shorter cell cycle than wild-type, single, or double knock-out control cells. TKO cells were resistant to G(1) arrest following DNA damage, despite retaining functional p53 activity. They were also insensitive to G(1) arrest signals following contact inhibition or serum starvation. Finally, TKO MEFs did not undergo senescence in culture and do possess some characteristics of transformed cells. Our results confirm the essential role of the Rb family in the control of the G(1)/S transition, place the three Rb family members downstream of multiple cell cycle control pathways, and further the link between loss of cell cycle control and tumorigenesis.","apaCitation":"Sage, J., Mulligan, G. J., Attardi, L. D., Miller, A., Chen, S. Q., Williams, B., ? Jacks, T. (2000). Targeted disruption of the three Rb-related genes leads to loss of G(1) control and immortalization. <i>GENES &amp; DEVELOPMENT</i>, <i>14</i>(23), 3037?3050.","capCitation":"<span class=\"title\">\n  <span>Targeted disruption of the three Rb-related genes leads to loss of G(1) control and immortalization</span>\n  <i>GENES &amp; DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Sage, J., Mulligan, G. J., Attardi, L. D., Miller, A., Chen, S. Q., Williams, B., Theodorou, E., Jacks, T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2000</span>; <span class=\"volume\">14 (23)</span><span class=\"pages\">: 3037-3050</span>\n</span>","chicagoCitation":"Sage, J., G. J. Mulligan, L. D. Attardi, A. Miller, S. Q. Chen, et al. 2000. ?Targeted Disruption of the Three Rb-Related Genes Leads to Loss of G(1) Control and Immortalization.? <i>GENES &amp; DEVELOPMENT</i> 14 (23). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 3037?50.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma protein, pRB, and the closely related proteins p107 and p130 are important regulators of the mammalian cell cycle. Biochemical and genetic studies have demonstrated overlapping as well as distinct functions for the three proteins in cell cycle control and mouse development. However, the role of the pRB family as a whole in the regulation of cell proliferation, cell death, or cell differentiation is not known. We generated embryonic stem (ES) cells and other cell types mutant for all three genes. Triple knock-out mouse embryonic fibroblasts (TKO MEFs) had a shorter cell cycle than wild-type, single, or double knock-out control cells. TKO cells were resistant to G(1) arrest following DNA damage, despite retaining functional p53 activity. They were also insensitive to G(1) arrest signals following contact inhibition or serum starvation. Finally, TKO MEFs did not undergo senescence in culture and do possess some characteristics of transformed cells. Our results confirm the essential role of the Rb family in the control of the G(1)/S transition, place the three Rb family members downstream of multiple cell cycle control pathways, and further the link between loss of cell cycle control and tumorigenesis.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000165788600011\">Web of Science ID 000165788600011</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/11114892\">PubMedID 11114892</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma protein, pRB, and the closely related proteins p107 and p130 are important regulators of the mammalian cell cycle. Biochemical and genetic studies have demonstrated overlapping as well as distinct functions for the three proteins in cell cycle control and mouse development. However, the role of the pRB family as a whole in the regulation of cell proliferation, cell death, or cell differentiation is not known. We generated embryonic stem (ES) cells and other cell types mutant for all three genes. Triple knock-out mouse embryonic fibroblasts (TKO MEFs) had a shorter cell cycle than wild-type, single, or double knock-out control cells. TKO cells were resistant to G(1) arrest following DNA damage, despite retaining functional p53 activity. They were also insensitive to G(1) arrest signals following contact inhibition or serum starvation. Finally, TKO MEFs did not undergo senescence in culture and do possess some characteristics of transformed cells. Our results confirm the essential role of the Rb family in the control of the G(1)/S transition, place the three Rb family members downstream of multiple cell cycle control pathways, and further the link between loss of cell cycle control and tumorigenesis.\n\nWeb of Science ID: 000165788600011, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000165788600011\nPubMedID: 11114892, https://www.ncbi.nlm.nih.gov/pubmed/11114892"},"featured":true,"firstPublished":{"html":"Dec 1, 2000 00:00:00 AM","text":"Dec 1, 2000 00:00:00 AM","value":"2000-12-01T00:00:00.000-08:00"},"mlaCitation":"Sage, J. et al. ?Targeted Disruption of the Three Rb-Related Genes Leads to Loss of G(1) Control and Immortalization.? <i>GENES &amp; DEVELOPMENT</i> 14.23 (2000): 3037?3050. Print.","pubMedId":"11114892","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/11114892","publicationId":8961,"title":"Targeted disruption of the three Rb-related genes leads to loss of G(1) control and immortalization","type":"article","webOfScienceId":"000165788600011","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000165788600011"},{"abstract":"The retinoblastoma (Rb) gene product is a prototypic tumor suppressor. Mice lacking the Rb gene are not viable and die in utero at approximately 13 days of gestation. In this study, we have rescued Rb-/- prostates by grafting pelvic organ rudiments from Rb-/- mouse embryos under the renal capsule of adult male nude mouse hosts. Grafts of embryonic pelvic organs developed into functional prostatic tissue. Some of the prostatic tissue generated was further used to construct chimeric prostatic tissue recombinants by combining wild-type rat urogenital mesenchyme (rUGM) with Rb-/- and Rb+/+ prostatic epithelium (PRE). The tissue recombinants were grown as subcapsular renal grafts and treated from the time of grafting with Silastic capsules containing 25 mg of testosterone plus 2.5 mg of estradiol. During 5-8 weeks of hormone treatment, rUGM+Rb+/+PRE tissue recombinants developed prostatic hyperplasia, whereas PRE in rUGM+Rb-/-PRE tissue recombinants developed hyperplasia, atypical hyperplasia, and carcinoma. During carcinogenesis in rUGM+Rb-/-PRE tissue recombinants, prostatic epithelial cells of the basal lineage disappeared, whereas the luminal cells underwent carcinogenesis. Epithelial E-cadherin almost totally disappeared. In all cases, epithelial PCNA labeling was elevated in tissue recombinants containing Rb-/- versus Rb+/+ epithelium. These epithelial changes were associated with almost total loss of smooth muscle cells in the stroma. In contrast, in untreated hosts rUGM+Rb+/+PRE tissue recombinants developed normally, and rUGM+Rb-/-PRE tissue recombinants developed mild epithelial hyperplasia. The results of this study demonstrate that Rb-/- prostatic tissue can be rescued from embryonic lethal mice and used to test its susceptibility to hormonal carcinogenesis. Deletion of the Rb gene predisposes prostatic epithelium to hyperplasia and increases proliferative activity Susceptibility to hormonal carcinogenesis in response to exogenous testosterone + estradiol is manifested in the progression from atypica hyperplasia to carcinoma. Thus, these findings demonstrate that the absence of the Rb tumor suppressor gene may predispose prostatic epithelial cells to carcinogenesis. Rescue of organs from Rb-/- embryos not only provides an opportunity to analyze the Rb gene pathway in the development and progression of prostate cancer but also provides an opportunity for specifically evaluating the role of the Rb pathway in development and carcinogenesis in other organs, such as the mammary gland and colon. Because rUGM greatly stimulates prostatic epithelial proliferation, the tissue recombinant model is a particularly useful tool for assessing the functional role of other genes in prostatic carcinogenesis through use of the appropriate transgenic or gene knockout mice.","apaCitation":"Wang, Y. H., Hayward, S. W., Donjacour, A. A., Young, P., Jacks, T., Sage, J., ? Cunha, G. R. (2000). Sex hormone-induced carcinogenesis in Rb-deficient prostate tissue. <i>CANCER RESEARCH</i>, <i>60</i>(21), 6008?6017.","capCitation":"<span class=\"title\">\n  <span>Sex hormone-induced carcinogenesis in Rb-deficient prostate tissue</span>\n  <i>CANCER RESEARCH</i>\n</span>\n<span class=\"authors\">Wang, Y. H., Hayward, S. W., Donjacour, A. A., Young, P., Jacks, T., Sage, J., Dahiya, R., Cardiff, R. D., Day, M. L., Cunha, G. R.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2000</span>; <span class=\"volume\">60 (21)</span><span class=\"pages\">: 6008-6017</span>\n</span>","chicagoCitation":"Wang, Y. H., S. W. Hayward, A. A. Donjacour, P. Young, T. Jacks, et al. 2000. ?Sex Hormone-Induced Carcinogenesis in Rb-Deficient Prostate Tissue.? <i>CANCER RESEARCH</i> 60 (21). AMER ASSOC CANCER RESEARCH: 6008?17.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma (Rb) gene product is a prototypic tumor suppressor. Mice lacking the Rb gene are not viable and die in utero at approximately 13 days of gestation. In this study, we have rescued Rb-/- prostates by grafting pelvic organ rudiments from Rb-/- mouse embryos under the renal capsule of adult male nude mouse hosts. Grafts of embryonic pelvic organs developed into functional prostatic tissue. Some of the prostatic tissue generated was further used to construct chimeric prostatic tissue recombinants by combining wild-type rat urogenital mesenchyme (rUGM) with Rb-/- and Rb+/+ prostatic epithelium (PRE). The tissue recombinants were grown as subcapsular renal grafts and treated from the time of grafting with Silastic capsules containing 25 mg of testosterone plus 2.5 mg of estradiol. During 5-8 weeks of hormone treatment, rUGM+Rb+/+PRE tissue recombinants developed prostatic hyperplasia, whereas PRE in rUGM+Rb-/-PRE tissue recombinants developed hyperplasia, atypical hyperplasia, and carcinoma. During carcinogenesis in rUGM+Rb-/-PRE tissue recombinants, prostatic epithelial cells of the basal lineage disappeared, whereas the luminal cells underwent carcinogenesis. Epithelial E-cadherin almost totally disappeared. In all cases, epithelial PCNA labeling was elevated in tissue recombinants containing Rb-/- versus Rb+/+ epithelium. These epithelial changes were associated with almost total loss of smooth muscle cells in the stroma. In contrast, in untreated hosts rUGM+Rb+/+PRE tissue recombinants developed normally, and rUGM+Rb-/-PRE tissue recombinants developed mild epithelial hyperplasia. The results of this study demonstrate that Rb-/- prostatic tissue can be rescued from embryonic lethal mice and used to test its susceptibility to hormonal carcinogenesis. Deletion of the Rb gene predisposes prostatic epithelium to hyperplasia and increases proliferative activity Susceptibility to hormonal carcinogenesis in response to exogenous testosterone + estradiol is manifested in the progression from atypica hyperplasia to carcinoma. Thus, these findings demonstrate that the absence of the Rb tumor suppressor gene may predispose prostatic epithelial cells to carcinogenesis. Rescue of organs from Rb-/- embryos not only provides an opportunity to analyze the Rb gene pathway in the development and progression of prostate cancer but also provides an opportunity for specifically evaluating the role of the Rb pathway in development and carcinogenesis in other organs, such as the mammary gland and colon. Because rUGM greatly stimulates prostatic epithelial proliferation, the tissue recombinant model is a particularly useful tool for assessing the functional role of other genes in prostatic carcinogenesis through use of the appropriate transgenic or gene knockout mice.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000165230300021\">Web of Science ID 000165230300021</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/11085521\">PubMedID 11085521</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma (Rb) gene product is a prototypic tumor suppressor. Mice lacking the Rb gene are not viable and die in utero at approximately 13 days of gestation. In this study, we have rescued Rb-/- prostates by grafting pelvic organ rudiments from Rb-/- mouse embryos under the renal capsule of adult male nude mouse hosts. Grafts of embryonic pelvic organs developed into functional prostatic tissue. Some of the prostatic tissue generated was further used to construct chimeric prostatic tissue recombinants by combining wild-type rat urogenital mesenchyme (rUGM) with Rb-/- and Rb+/+ prostatic epithelium (PRE). The tissue recombinants were grown as subcapsular renal grafts and treated from the time of grafting with Silastic capsules containing 25 mg of testosterone plus 2.5 mg of estradiol. During 5-8 weeks of hormone treatment, rUGM+Rb+/+PRE tissue recombinants developed prostatic hyperplasia, whereas PRE in rUGM+Rb-/-PRE tissue recombinants developed hyperplasia, atypical hyperplasia, and carcinoma. During carcinogenesis in rUGM+Rb-/-PRE tissue recombinants, prostatic epithelial cells of the basal lineage disappeared, whereas the luminal cells underwent carcinogenesis. Epithelial E-cadherin almost totally disappeared. In all cases, epithelial PCNA labeling was elevated in tissue recombinants containing Rb-/- versus Rb+/+ epithelium. These epithelial changes were associated with almost total loss of smooth muscle cells in the stroma. In contrast, in untreated hosts rUGM+Rb+/+PRE tissue recombinants developed normally, and rUGM+Rb-/-PRE tissue recombinants developed mild epithelial hyperplasia. The results of this study demonstrate that Rb-/- prostatic tissue can be rescued from embryonic lethal mice and used to test its susceptibility to hormonal carcinogenesis. Deletion of the Rb gene predisposes prostatic epithelium to hyperplasia and increases proliferative activity Susceptibility to hormonal carcinogenesis in response to exogenous testosterone + estradiol is manifested in the progression from atypica hyperplasia to carcinoma. Thus, these findings demonstrate that the absence of the Rb tumor suppressor gene may predispose prostatic epithelial cells to carcinogenesis. Rescue of organs from Rb-/- embryos not only provides an opportunity to analyze the Rb gene pathway in the development and progression of prostate cancer but also provides an opportunity for specifically evaluating the role of the Rb pathway in development and carcinogenesis in other organs, such as the mammary gland and colon. Because rUGM greatly stimulates prostatic epithelial proliferation, the tissue recombinant model is a particularly useful tool for assessing the functional role of other genes in prostatic carcinogenesis through use of the appropriate transgenic or gene knockout mice.\n\nWeb of Science ID: 000165230300021, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000165230300021\nPubMedID: 11085521, https://www.ncbi.nlm.nih.gov/pubmed/11085521"},"featured":false,"firstPublished":{"html":"Nov 1, 2000 00:00:00 AM","text":"Nov 1, 2000 00:00:00 AM","value":"2000-11-01T00:00:00.000-08:00"},"mlaCitation":"Wang, Y. H. et al. ?Sex Hormone-Induced Carcinogenesis in Rb-Deficient Prostate Tissue.? <i>CANCER RESEARCH</i> 60.21 (2000): 6008?6017. Print.","pubMedId":"11085521","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/11085521","publicationId":242879,"title":"Sex hormone-induced carcinogenesis in Rb-deficient prostate tissue","type":"article","webOfScienceId":"000165230300021","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000165230300021"},{"abstract":"To analyze NF-kappa B activity in the testis, we used murine transgenic lines carrying a LacZ reporter gene under the control of a NF-kappa B-responsive promoter (Schmidt-Ullrich et al. [1996] Dev 122:2117-2128). We constructed three independent lines containing the promoter of the gene encoding p105, the precursor of the p50 subunit. This promoter contains three NF-kappa B-binding sites in its proximal part. Our results show that in adult mice, the beta-galactosidase activity which reflects nuclear NF-kappa B activity, is first detected in spermatocytes at the pachytene stage and remains activated in the following steps of germ cell differentiation and maturation. Using transgenic mice carrying a p105nlslacZ construct with the 3 NF-kappa B sites mutated in the p105 promoter, we found a significant reduction in the transgene activity, confirming the important role of NF-kappa B in the activation of the transgene. To confirm the stage of induction during spermatogenesis, we analysed the beta-galactosidase activity in the testes from prepuberal mice in which cells synchrouneously enter meiosis. We detected the transgene activity at 18 days after birth, corresponding to the pachytene stage in spermatocytes. In nuclear extracts prepared from prepuberal mice, we found a peak of NF-kappa B DNA-binding activity made of p50 and p65 subunits at day 18 after birth, which remains high in the later stages. Further analysis showed that I kappa B alpha and beta, but not epsilon are expressed in the testes. Altogether, these data suggest that NF-kappa B factors are stage specifically controlled and may play a role during the development of sperm cells.","apaCitation":"Lilienbaum, A., Sage, J., Memet, S., Rassoulzadegan, M., Cuzin, F., &amp; Israel, A. (2000). NF-kappa B is developmentally regulated during spermatogenesis in mice. <i>DEVELOPMENTAL DYNAMICS</i>, <i>219</i>(3), 333?340.","capCitation":"<span class=\"title\">\n  <span>NF-kappa B is developmentally regulated during spermatogenesis in mice</span>\n  <i>DEVELOPMENTAL DYNAMICS</i>\n</span>\n<span class=\"authors\">Lilienbaum, A., Sage, J., Memet, S., Rassoulzadegan, M., Cuzin, F., Israel, A.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2000</span>; <span class=\"volume\">219 (3)</span><span class=\"pages\">: 333-340</span>\n</span>","chicagoCitation":"Lilienbaum, A., J. Sage, S. Memet, M. Rassoulzadegan, F. Cuzin, et al. 2000. ?NF-Kappa B Is Developmentally Regulated during Spermatogenesis in Mice.? <i>DEVELOPMENTAL DYNAMICS</i> 219 (3). WILEY-LISS: 333?40.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">To analyze NF-kappa B activity in the testis, we used murine transgenic lines carrying a LacZ reporter gene under the control of a NF-kappa B-responsive promoter (Schmidt-Ullrich et al. [1996] Dev 122:2117-2128). We constructed three independent lines containing the promoter of the gene encoding p105, the precursor of the p50 subunit. This promoter contains three NF-kappa B-binding sites in its proximal part. Our results show that in adult mice, the beta-galactosidase activity which reflects nuclear NF-kappa B activity, is first detected in spermatocytes at the pachytene stage and remains activated in the following steps of germ cell differentiation and maturation. Using transgenic mice carrying a p105nlslacZ construct with the 3 NF-kappa B sites mutated in the p105 promoter, we found a significant reduction in the transgene activity, confirming the important role of NF-kappa B in the activation of the transgene. To confirm the stage of induction during spermatogenesis, we analysed the beta-galactosidase activity in the testes from prepuberal mice in which cells synchrouneously enter meiosis. We detected the transgene activity at 18 days after birth, corresponding to the pachytene stage in spermatocytes. In nuclear extracts prepared from prepuberal mice, we found a peak of NF-kappa B DNA-binding activity made of p50 and p65 subunits at day 18 after birth, which remains high in the later stages. Further analysis showed that I kappa B alpha and beta, but not epsilon are expressed in the testes. Altogether, these data suggest that NF-kappa B factors are stage specifically controlled and may play a role during the development of sperm cells.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000165122200005\">Web of Science ID 000165122200005</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/11066090\">PubMedID 11066090</a></span>\n  </p>","text":"Abstract:\nTo analyze NF-kappa B activity in the testis, we used murine transgenic lines carrying a LacZ reporter gene under the control of a NF-kappa B-responsive promoter (Schmidt-Ullrich et al. [1996] Dev 122:2117-2128). We constructed three independent lines containing the promoter of the gene encoding p105, the precursor of the p50 subunit. This promoter contains three NF-kappa B-binding sites in its proximal part. Our results show that in adult mice, the beta-galactosidase activity which reflects nuclear NF-kappa B activity, is first detected in spermatocytes at the pachytene stage and remains activated in the following steps of germ cell differentiation and maturation. Using transgenic mice carrying a p105nlslacZ construct with the 3 NF-kappa B sites mutated in the p105 promoter, we found a significant reduction in the transgene activity, confirming the important role of NF-kappa B in the activation of the transgene. To confirm the stage of induction during spermatogenesis, we analysed the beta-galactosidase activity in the testes from prepuberal mice in which cells synchrouneously enter meiosis. We detected the transgene activity at 18 days after birth, corresponding to the pachytene stage in spermatocytes. In nuclear extracts prepared from prepuberal mice, we found a peak of NF-kappa B DNA-binding activity made of p50 and p65 subunits at day 18 after birth, which remains high in the later stages. Further analysis showed that I kappa B alpha and beta, but not epsilon are expressed in the testes. Altogether, these data suggest that NF-kappa B factors are stage specifically controlled and may play a role during the development of sperm cells.\n\nWeb of Science ID: 000165122200005, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000165122200005\nPubMedID: 11066090, https://www.ncbi.nlm.nih.gov/pubmed/11066090"},"featured":false,"firstPublished":{"html":"Nov 1, 2000 00:00:00 AM","text":"Nov 1, 2000 00:00:00 AM","value":"2000-11-01T00:00:00.000-08:00"},"mlaCitation":"Lilienbaum, A. et al. ?NF-Kappa B Is Developmentally Regulated during Spermatogenesis in Mice.? <i>DEVELOPMENTAL DYNAMICS</i> 219.3 (2000): 333?340. Print.","pubMedId":"11066090","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/11066090","publicationId":242880,"title":"NF-kappa B is developmentally regulated during spermatogenesis in mice","type":"article","webOfScienceId":"000165122200005","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000165122200005"},{"abstract":"Angiogenesis has been demonstrated to be essential for tumor growth and metastasis, and inhibition of angiogenesis is emerging as a promising strategy for treating cancer. Among the most potent inhibitors of angiogenesis is the fumagillin family of natural products. An analog of fumagillin, known as TNP-470 or AGM-1470, has been undergoing clinical trials for treating a variety of cancers. TNP-470 has been shown to block endothelial cell cycle progression in the late G(1) phase. Although the direct molecular target for TNP-470 has been identified as the type 2 methionine aminopeptidase (MetAP2), how inhibition of this enzyme leads to cell cycle arrest has remained unclear. We report that treatment of endothelial and other drug-sensitive cell types leads to the activation of the p53 pathway, causing an accumulation of the G(1) cyclin-dependent kinase inhibitor p21(WAF1/CIP1). The requirement of p53 and p21(WAF1/CIP1) for the cell cycle inhibition by TNP-470 is underscored by the observation that cells deficient in p53 and p21(WAF1/CIP1) are resistant to TNP-470. These results shed significant light on the mechanism of cell cycle inhibition by TNP-470 and suggest an alternative method of activating p53 in endothelial cells to halt angiogenesis and tumor progression.","apaCitation":"Zhang, Y., Griffith, E. C., Sage, J., Jacks, T., &amp; Liu, J. O. (2000). Cell cycle inhibition by the anti-angiogenic agent TNP-470 is mediated by p53 and p21(WAF1/CIP1). <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i>, <i>97</i>(12), 6427?6432.","capCitation":"<span class=\"title\">\n  <span>Cell cycle inhibition by the anti-angiogenic agent TNP-470 is mediated by p53 and p21(WAF1/CIP1)</span>\n  <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i>\n</span>\n<span class=\"authors\">Zhang, Y., Griffith, E. C., Sage, J., Jacks, T., Liu, J. O.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2000</span>; <span class=\"volume\">97 (12)</span><span class=\"pages\">: 6427-6432</span>\n</span>","chicagoCitation":"Zhang, Y., E. C. Griffith, J. Sage, T. Jacks, and J. O. Liu. 2000. ?Cell Cycle Inhibition by the Anti-Angiogenic Agent TNP-470 Is Mediated by p53 and p21(WAF1/CIP1).? <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i> 97 (12). NATL ACAD SCIENCES: 6427?32.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Angiogenesis has been demonstrated to be essential for tumor growth and metastasis, and inhibition of angiogenesis is emerging as a promising strategy for treating cancer. Among the most potent inhibitors of angiogenesis is the fumagillin family of natural products. An analog of fumagillin, known as TNP-470 or AGM-1470, has been undergoing clinical trials for treating a variety of cancers. TNP-470 has been shown to block endothelial cell cycle progression in the late G(1) phase. Although the direct molecular target for TNP-470 has been identified as the type 2 methionine aminopeptidase (MetAP2), how inhibition of this enzyme leads to cell cycle arrest has remained unclear. We report that treatment of endothelial and other drug-sensitive cell types leads to the activation of the p53 pathway, causing an accumulation of the G(1) cyclin-dependent kinase inhibitor p21(WAF1/CIP1). The requirement of p53 and p21(WAF1/CIP1) for the cell cycle inhibition by TNP-470 is underscored by the observation that cells deficient in p53 and p21(WAF1/CIP1) are resistant to TNP-470. These results shed significant light on the mechanism of cell cycle inhibition by TNP-470 and suggest an alternative method of activating p53 in endothelial cells to halt angiogenesis and tumor progression.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000087526300037\">Web of Science ID 000087526300037</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/10841547\">PubMedID 10841547</a></span>\n  </p>","text":"Abstract:\nAngiogenesis has been demonstrated to be essential for tumor growth and metastasis, and inhibition of angiogenesis is emerging as a promising strategy for treating cancer. Among the most potent inhibitors of angiogenesis is the fumagillin family of natural products. An analog of fumagillin, known as TNP-470 or AGM-1470, has been undergoing clinical trials for treating a variety of cancers. TNP-470 has been shown to block endothelial cell cycle progression in the late G(1) phase. Although the direct molecular target for TNP-470 has been identified as the type 2 methionine aminopeptidase (MetAP2), how inhibition of this enzyme leads to cell cycle arrest has remained unclear. We report that treatment of endothelial and other drug-sensitive cell types leads to the activation of the p53 pathway, causing an accumulation of the G(1) cyclin-dependent kinase inhibitor p21(WAF1/CIP1). The requirement of p53 and p21(WAF1/CIP1) for the cell cycle inhibition by TNP-470 is underscored by the observation that cells deficient in p53 and p21(WAF1/CIP1) are resistant to TNP-470. These results shed significant light on the mechanism of cell cycle inhibition by TNP-470 and suggest an alternative method of activating p53 in endothelial cells to halt angiogenesis and tumor progression.\n\nWeb of Science ID: 000087526300037, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000087526300037\nPubMedID: 10841547, https://www.ncbi.nlm.nih.gov/pubmed/10841547"},"featured":false,"firstPublished":{"html":"Jun 6, 2000 00:00:00 AM","text":"Jun 6, 2000 00:00:00 AM","value":"2000-06-06T00:00:00.000-07:00"},"mlaCitation":"Zhang, Y. et al. ?Cell Cycle Inhibition by the Anti-Angiogenic Agent TNP-470 Is Mediated by p53 and p21(WAF1/CIP1).? <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i> 97.12 (2000): 6427?6432. Print.","pubMedId":"10841547","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/10841547","publicationId":242881,"title":"Cell cycle inhibition by the anti-angiogenic agent TNP-470 is mediated by p53 and p21(WAF1/CIP1)","type":"article","webOfScienceId":"000087526300037","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000087526300037"},{"abstract":"Transcription controls active at the initial stages of meiosis are clearly key elements in the regulation of germinal differentiation. Transcription of the Sycp1 gene (synaptonemal complex protein 1) starts as early as the leptotene and zygotene stages. Constructs with Sycp1 5' upstream sequences directed the expression of reporter genes to pachytene spermatocytes in transgenic mice. A short fragment encompassing the transcription start (n.t. -54 to +102) was sufficient for stage-specific expression in the adult male and for temporal regulation during development. Upstream enhancer element(s) quantitatively regulating expression were localized in the region between -54 and -260. The gene is normally expressed both in the male and female gonads, but none of the promoter sequences active in the testis allowed the expression of reporter genes during meiosis in the ovary.","apaCitation":"Sage, J., Martin, L., Meuwissen, R., Heyting, C., Cuzin, F., &amp; Rassoulzadegan, M. (1999). Temporal and spatial control of the Sycp1 gene transcription in the mouse meiosis: regulatory elements active in the male are not sufficient for expression in the female gonad. <i>MECHANISMS OF DEVELOPMENT</i>, <i>80</i>(1), 29?39.","capCitation":"<span class=\"title\">\n  <span>Temporal and spatial control of the Sycp1 gene transcription in the mouse meiosis: regulatory elements active in the male are not sufficient for expression in the female gonad</span>\n  <i>MECHANISMS OF DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Sage, J., Martin, L., Meuwissen, R., Heyting, C., Cuzin, F., Rassoulzadegan, M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">1999</span>; <span class=\"volume\">80 (1)</span><span class=\"pages\">: 29-39</span>\n</span>","chicagoCitation":"Sage, J., L. Martin, R. Meuwissen, C. Heyting, F. Cuzin, et al. 1999. ?Temporal and Spatial Control of the Sycp1 Gene Transcription in the Mouse Meiosis: Regulatory Elements Active in the Male Are Not Sufficient for Expression in the Female Gonad.? <i>MECHANISMS OF DEVELOPMENT</i> 80 (1). ELSEVIER SCIENCE BV: 29?39.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Transcription controls active at the initial stages of meiosis are clearly key elements in the regulation of germinal differentiation. Transcription of the Sycp1 gene (synaptonemal complex protein 1) starts as early as the leptotene and zygotene stages. Constructs with Sycp1 5' upstream sequences directed the expression of reporter genes to pachytene spermatocytes in transgenic mice. A short fragment encompassing the transcription start (n.t. -54 to +102) was sufficient for stage-specific expression in the adult male and for temporal regulation during development. Upstream enhancer element(s) quantitatively regulating expression were localized in the region between -54 and -260. The gene is normally expressed both in the male and female gonads, but none of the promoter sequences active in the testis allowed the expression of reporter genes during meiosis in the ovary.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000078473700003\">Web of Science ID 000078473700003</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/10096061\">PubMedID 10096061</a></span>\n  </p>","text":"Abstract:\nTranscription controls active at the initial stages of meiosis are clearly key elements in the regulation of germinal differentiation. Transcription of the Sycp1 gene (synaptonemal complex protein 1) starts as early as the leptotene and zygotene stages. Constructs with Sycp1 5' upstream sequences directed the expression of reporter genes to pachytene spermatocytes in transgenic mice. A short fragment encompassing the transcription start (n.t. -54 to +102) was sufficient for stage-specific expression in the adult male and for temporal regulation during development. Upstream enhancer element(s) quantitatively regulating expression were localized in the region between -54 and -260. The gene is normally expressed both in the male and female gonads, but none of the promoter sequences active in the testis allowed the expression of reporter genes during meiosis in the ovary.\n\nWeb of Science ID: 000078473700003, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000078473700003\nPubMedID: 10096061, https://www.ncbi.nlm.nih.gov/pubmed/10096061"},"featured":false,"firstPublished":{"html":"Jan 1, 1999 00:00:00 AM","text":"Jan 1, 1999 00:00:00 AM","value":"1999-01-01T00:00:00.000-08:00"},"mlaCitation":"Sage, J. et al. ?Temporal and Spatial Control of the Sycp1 Gene Transcription in the Mouse Meiosis: Regulatory Elements Active in the Male Are Not Sufficient for Expression in the Female Gonad.? <i>MECHANISMS OF DEVELOPMENT</i> 80.1 (1999): 29?39. Print.","pubMedId":"10096061","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/10096061","publicationId":242883,"title":"Temporal and spatial control of the Sycp1 gene transcription in the mouse meiosis: regulatory elements active in the male are not sufficient for expression in the female gonad","type":"article","webOfScienceId":"000078473700003","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000078473700003"},{"abstract":"Transgenic mice were generated expressing a testicular Cre recombinase driven by promoter sequences derived from the gene encoding Synaptonemal Complex Protein 1 (Sycp1), expressed at an early stage of the male meiosis (leptotene to zygotene). Recombination at target LoxP sites was examined during germinal differentiation in mice harboring Sycp1-Cre and a second transgene where LoxP sites flank either the beta geo coding region, the Pgk1 promoter, or a tk-neo cassette inserted into the Rxr alpha locus. The LoxP-flanked transgenes were stably maintained in the somatic tissues of the double transgenic animals, as well as in the progeny of the females. Mice born after mating the double-transgenic males with normal females showed extensive deletions of the LoxP-flanked sequences. When the males were hemizygous for the Sycp1-Cre transgene, the deletions were observed even in the fraction of the offspring which had not inherited the Cre gene, thus demonstrating that expression occurred in the male parent during spermatogenesis. The high efficiency of excision at the LoxP sites makes the Sycp1-Cre transgenic males suitable for evaluating the role of defined gene functions in the germinal differentiation process.","apaCitation":"Vidal, F., Sage, J., Cuzin, F., &amp; Rassoulzadegan, M. (1998). Cre expression in primary spermatocytes: A tool for genetic engineering of the germ line. <i>MOLECULAR REPRODUCTION AND DEVELOPMENT</i>, <i>51</i>(3), 274?280.","capCitation":"<span class=\"title\">\n  <span>Cre expression in primary spermatocytes: A tool for genetic engineering of the germ line</span>\n  <i>MOLECULAR REPRODUCTION AND DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Vidal, F., Sage, J., Cuzin, F., Rassoulzadegan, M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">1998</span>; <span class=\"volume\">51 (3)</span><span class=\"pages\">: 274-280</span>\n</span>","chicagoCitation":"Vidal, F., J. Sage, F. Cuzin, and M. Rassoulzadegan. 1998. ?Cre Expression in Primary Spermatocytes: A Tool for Genetic Engineering of the Germ Line.? <i>MOLECULAR REPRODUCTION AND DEVELOPMENT</i> 51 (3). WILEY-LISS: 274?80.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Transgenic mice were generated expressing a testicular Cre recombinase driven by promoter sequences derived from the gene encoding Synaptonemal Complex Protein 1 (Sycp1), expressed at an early stage of the male meiosis (leptotene to zygotene). Recombination at target LoxP sites was examined during germinal differentiation in mice harboring Sycp1-Cre and a second transgene where LoxP sites flank either the beta geo coding region, the Pgk1 promoter, or a tk-neo cassette inserted into the Rxr alpha locus. The LoxP-flanked transgenes were stably maintained in the somatic tissues of the double transgenic animals, as well as in the progeny of the females. Mice born after mating the double-transgenic males with normal females showed extensive deletions of the LoxP-flanked sequences. When the males were hemizygous for the Sycp1-Cre transgene, the deletions were observed even in the fraction of the offspring which had not inherited the Cre gene, thus demonstrating that expression occurred in the male parent during spermatogenesis. The high efficiency of excision at the LoxP sites makes the Sycp1-Cre transgenic males suitable for evaluating the role of defined gene functions in the germinal differentiation process.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000076145500006\">Web of Science ID 000076145500006</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/9771647\">PubMedID 9771647</a></span>\n  </p>","text":"Abstract:\nTransgenic mice were generated expressing a testicular Cre recombinase driven by promoter sequences derived from the gene encoding Synaptonemal Complex Protein 1 (Sycp1), expressed at an early stage of the male meiosis (leptotene to zygotene). Recombination at target LoxP sites was examined during germinal differentiation in mice harboring Sycp1-Cre and a second transgene where LoxP sites flank either the beta geo coding region, the Pgk1 promoter, or a tk-neo cassette inserted into the Rxr alpha locus. The LoxP-flanked transgenes were stably maintained in the somatic tissues of the double transgenic animals, as well as in the progeny of the females. Mice born after mating the double-transgenic males with normal females showed extensive deletions of the LoxP-flanked sequences. When the males were hemizygous for the Sycp1-Cre transgene, the deletions were observed even in the fraction of the offspring which had not inherited the Cre gene, thus demonstrating that expression occurred in the male parent during spermatogenesis. The high efficiency of excision at the LoxP sites makes the Sycp1-Cre transgenic males suitable for evaluating the role of defined gene functions in the germinal differentiation process.\n\nWeb of Science ID: 000076145500006, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000076145500006\nPubMedID: 9771647, https://www.ncbi.nlm.nih.gov/pubmed/9771647"},"featured":false,"firstPublished":{"html":"Nov 1, 1998 00:00:00 AM","text":"Nov 1, 1998 00:00:00 AM","value":"1998-11-01T00:00:00.000-08:00"},"mlaCitation":"Vidal, F. et al. ?Cre Expression in Primary Spermatocytes: A Tool for Genetic Engineering of the Germ Line.? <i>MOLECULAR REPRODUCTION AND DEVELOPMENT</i> 51.3 (1998): 274?280. Print.","pubMedId":"9771647","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/9771647","publicationId":242886,"title":"Cre expression in primary spermatocytes: A tool for genetic engineering of the germ line","type":"article","webOfScienceId":"000076145500006","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000076145500006"},{"abstract":"The Kit receptor and its ligand KL, which together constitute an essential effector at various stages of embryonic development, are both present during adult gametogenesis. In the testis, KL is expressed in Sertoli cells, and Kit in germ cells, starting at the premeiotic stages. A series of observations indicated previously a role in spermatogonia survival, without excluding a possible function at later stages. We identified a complex pattern of expression of the two components in the adult murine testis, suggestive of a role in the meiotic progression of spermatocytes. At stages VII-VIII of the cycle of the seminiferous epithelium, the time when spermatocytes enter meiosis, the membrane-associated form of KL extends on the Sertoli cell from the peripheral to the adluminal compartment of the tubule. We also found that the receptor is present on the surface of germ cells up to the pachytene stage. The availability of differentiated Sertoli cell lines, which express the KL protein and support part of the maturation of germ cells in coculture, allowed us to ask whether, in the in vitro reconstructed system, transit of spermatocytes through meiosis requires the Kit-KL interaction. Addition of a blocking monoclonal antibody against the Kit receptor (ACK2) inhibited extensively the appearance of haploid cells and the expression of a haploid-phase-specific gene (Prm1). Recognition of the supporting Sertoli cell by germ cells was not affected, indicating a requirement for the activity of the receptor for either entering or completing meiosis. Involvement of the membrane-associated form of the ligand was suggested by the observation that addition of the soluble form of KL was equally inhibitory.","apaCitation":"Vincent, S., Segretain, D., Nishikawa, S., Nishikawa, S., Sage, J., Cuzin, F., &amp; Rassoulzadegan, M. (1998). Stage-specific expression of the Kit receptor and its ligand (KL) during male gametogenesis in the mouse: a Kit-KL interaction critical for meiosis. <i>DEVELOPMENT</i>, <i>125</i>(22), 4585?4593.","capCitation":"<span class=\"title\">\n  <span>Stage-specific expression of the Kit receptor and its ligand (KL) during male gametogenesis in the mouse: a Kit-KL interaction critical for meiosis</span>\n  <i>DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Vincent, S., Segretain, D., Nishikawa, S., Nishikawa, S., Sage, J., Cuzin, F., Rassoulzadegan, M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">1998</span>; <span class=\"volume\">125 (22)</span><span class=\"pages\">: 4585-4593</span>\n</span>","chicagoCitation":"Vincent, S., D. Segretain, S. Nishikawa, S. Nishikawa, J. Sage, et al. 1998. ?Stage-Specific Expression of the Kit Receptor and Its Ligand (KL) during Male Gametogenesis in the Mouse: a Kit-KL Interaction Critical for Meiosis.? <i>DEVELOPMENT</i> 125 (22). COMPANY OF BIOLOGISTS LTD: 4585?93.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The Kit receptor and its ligand KL, which together constitute an essential effector at various stages of embryonic development, are both present during adult gametogenesis. In the testis, KL is expressed in Sertoli cells, and Kit in germ cells, starting at the premeiotic stages. A series of observations indicated previously a role in spermatogonia survival, without excluding a possible function at later stages. We identified a complex pattern of expression of the two components in the adult murine testis, suggestive of a role in the meiotic progression of spermatocytes. At stages VII-VIII of the cycle of the seminiferous epithelium, the time when spermatocytes enter meiosis, the membrane-associated form of KL extends on the Sertoli cell from the peripheral to the adluminal compartment of the tubule. We also found that the receptor is present on the surface of germ cells up to the pachytene stage. The availability of differentiated Sertoli cell lines, which express the KL protein and support part of the maturation of germ cells in coculture, allowed us to ask whether, in the in vitro reconstructed system, transit of spermatocytes through meiosis requires the Kit-KL interaction. Addition of a blocking monoclonal antibody against the Kit receptor (ACK2) inhibited extensively the appearance of haploid cells and the expression of a haploid-phase-specific gene (Prm1). Recognition of the supporting Sertoli cell by germ cells was not affected, indicating a requirement for the activity of the receptor for either entering or completing meiosis. Involvement of the membrane-associated form of the ligand was suggested by the observation that addition of the soluble form of KL was equally inhibitory.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000077649200022\">Web of Science ID 000077649200022</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/9778516\">PubMedID 9778516</a></span>\n  </p>","text":"Abstract:\nThe Kit receptor and its ligand KL, which together constitute an essential effector at various stages of embryonic development, are both present during adult gametogenesis. In the testis, KL is expressed in Sertoli cells, and Kit in germ cells, starting at the premeiotic stages. A series of observations indicated previously a role in spermatogonia survival, without excluding a possible function at later stages. We identified a complex pattern of expression of the two components in the adult murine testis, suggestive of a role in the meiotic progression of spermatocytes. At stages VII-VIII of the cycle of the seminiferous epithelium, the time when spermatocytes enter meiosis, the membrane-associated form of KL extends on the Sertoli cell from the peripheral to the adluminal compartment of the tubule. We also found that the receptor is present on the surface of germ cells up to the pachytene stage. The availability of differentiated Sertoli cell lines, which express the KL protein and support part of the maturation of germ cells in coculture, allowed us to ask whether, in the in vitro reconstructed system, transit of spermatocytes through meiosis requires the Kit-KL interaction. Addition of a blocking monoclonal antibody against the Kit receptor (ACK2) inhibited extensively the appearance of haploid cells and the expression of a haploid-phase-specific gene (Prm1). Recognition of the supporting Sertoli cell by germ cells was not affected, indicating a requirement for the activity of the receptor for either entering or completing meiosis. Involvement of the membrane-associated form of the ligand was suggested by the observation that addition of the soluble form of KL was equally inhibitory.\n\nWeb of Science ID: 000077649200022, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000077649200022\nPubMedID: 9778516, https://www.ncbi.nlm.nih.gov/pubmed/9778516"},"featured":true,"firstPublished":{"html":"Nov 1, 1998 00:00:00 AM","text":"Nov 1, 1998 00:00:00 AM","value":"1998-11-01T00:00:00.000-08:00"},"mlaCitation":"Vincent, S. et al. ?Stage-Specific Expression of the Kit Receptor and Its Ligand (KL) during Male Gametogenesis in the Mouse: a Kit-KL Interaction Critical for Meiosis.? <i>DEVELOPMENT</i> 125.22 (1998): 4585?4593. Print.","pubMedId":"9778516","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/9778516","publicationId":242885,"title":"Stage-specific expression of the Kit receptor and its ligand (KL) during male gametogenesis in the mouse: a Kit-KL interaction critical for meiosis","type":"article","webOfScienceId":"000077649200022","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000077649200022"},{"abstract":"Differentiation of male germ cells requires a continuous cross-talk with their somatic support, the Sertoli cell. An in vitro model of Sertoli cells was recently provided by established cell lines which maintain Sertoli-specific characteristics, among which is a regulated phagocytic capacity. In vivo, Sertoli cells take up the residual cytoplasm expelled from the maturing sperm, a process restricted to a limited period of germinal maturation, and they also eliminate abnormally differentiated germ cells in case of hormonal deficiency. Cells of the Sertoli line efficiently take up latex beads, as well as dead cells in the cultures. A semiquantitative assay of phagocytosis was developed, based on the uptake of fluorescent latex beads. 15P-1 cultures were found to contain a minor fraction of active phagocytes. After addition of a defined fraction of germ cells, however, all the cells internalized beads as efficiently as macrophages. The inducing cell was identified as the pachytene spermatocyte, a cell type which, in vivo, is associated with Sertoli cells when they express their phagocytic potential. These inducing meiotic cells were not internalized themselves. Rather, they interacted with Sertoli cells via a surface signal that was resistant to formaldehyde fixation. The whole induction process does not involve changes in Sertoli gene expression, since it occurs even in the presence of high doses of cycloheximide. After the required initial contact, further maintenance of the activity was dependent on factor(s) secreted in the medium of the activated culture. Phagocytosis was, on the other hand, abrogated in the presence of factor(s) secreted by a distinct fraction of germ cells, enriched in the late stages (second division) of meiosis.","apaCitation":"Grandjean, V., Sage, J., Ranc, F., Cuzin, F., &amp; Rassoulzadegan, M. (1997). Stage-specific signals in germ line differentiation: Control of Sertoli cell phagocytic activity by spermatogenic cells. <i>DEVELOPMENTAL BIOLOGY</i>, <i>184</i>(1), 165?174.","capCitation":"<span class=\"title\">\n  <span>Stage-specific signals in germ line differentiation: Control of Sertoli cell phagocytic activity by spermatogenic cells</span>\n  <i>DEVELOPMENTAL BIOLOGY</i>\n</span>\n<span class=\"authors\">Grandjean, V., Sage, J., Ranc, F., Cuzin, F., Rassoulzadegan, M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">1997</span>; <span class=\"volume\">184 (1)</span><span class=\"pages\">: 165-174</span>\n</span>","chicagoCitation":"Grandjean, V., J. Sage, F. Ranc, F. Cuzin, and M. Rassoulzadegan. 1997. ?Stage-Specific Signals in Germ Line Differentiation: Control of Sertoli Cell Phagocytic Activity by Spermatogenic Cells.? <i>DEVELOPMENTAL BIOLOGY</i> 184 (1). ACADEMIC PRESS INC JNL-COMP SUBSCRIPTIONS: 165?74.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Differentiation of male germ cells requires a continuous cross-talk with their somatic support, the Sertoli cell. An in vitro model of Sertoli cells was recently provided by established cell lines which maintain Sertoli-specific characteristics, among which is a regulated phagocytic capacity. In vivo, Sertoli cells take up the residual cytoplasm expelled from the maturing sperm, a process restricted to a limited period of germinal maturation, and they also eliminate abnormally differentiated germ cells in case of hormonal deficiency. Cells of the Sertoli line efficiently take up latex beads, as well as dead cells in the cultures. A semiquantitative assay of phagocytosis was developed, based on the uptake of fluorescent latex beads. 15P-1 cultures were found to contain a minor fraction of active phagocytes. After addition of a defined fraction of germ cells, however, all the cells internalized beads as efficiently as macrophages. The inducing cell was identified as the pachytene spermatocyte, a cell type which, in vivo, is associated with Sertoli cells when they express their phagocytic potential. These inducing meiotic cells were not internalized themselves. Rather, they interacted with Sertoli cells via a surface signal that was resistant to formaldehyde fixation. The whole induction process does not involve changes in Sertoli gene expression, since it occurs even in the presence of high doses of cycloheximide. After the required initial contact, further maintenance of the activity was dependent on factor(s) secreted in the medium of the activated culture. Phagocytosis was, on the other hand, abrogated in the presence of factor(s) secreted by a distinct fraction of germ cells, enriched in the late stages (second division) of meiosis.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/A1997WV15600015\">Web of Science ID A1997WV15600015</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/9142992\">PubMedID 9142992</a></span>\n  </p>","text":"Abstract:\nDifferentiation of male germ cells requires a continuous cross-talk with their somatic support, the Sertoli cell. An in vitro model of Sertoli cells was recently provided by established cell lines which maintain Sertoli-specific characteristics, among which is a regulated phagocytic capacity. In vivo, Sertoli cells take up the residual cytoplasm expelled from the maturing sperm, a process restricted to a limited period of germinal maturation, and they also eliminate abnormally differentiated germ cells in case of hormonal deficiency. Cells of the Sertoli line efficiently take up latex beads, as well as dead cells in the cultures. A semiquantitative assay of phagocytosis was developed, based on the uptake of fluorescent latex beads. 15P-1 cultures were found to contain a minor fraction of active phagocytes. After addition of a defined fraction of germ cells, however, all the cells internalized beads as efficiently as macrophages. The inducing cell was identified as the pachytene spermatocyte, a cell type which, in vivo, is associated with Sertoli cells when they express their phagocytic potential. These inducing meiotic cells were not internalized themselves. Rather, they interacted with Sertoli cells via a surface signal that was resistant to formaldehyde fixation. The whole induction process does not involve changes in Sertoli gene expression, since it occurs even in the presence of high doses of cycloheximide. After the required initial contact, further maintenance of the activity was dependent on factor(s) secreted in the medium of the activated culture. Phagocytosis was, on the other hand, abrogated in the presence of factor(s) secreted by a distinct fraction of germ cells, enriched in the late stages (second division) of meiosis.\n\nWeb of Science ID: A1997WV15600015, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/A1997WV15600015\nPubMedID: 9142992, https://www.ncbi.nlm.nih.gov/pubmed/9142992"},"featured":false,"firstPublished":{"html":"Apr 1, 1997 00:00:00 AM","text":"Apr 1, 1997 00:00:00 AM","value":"1997-04-01T00:00:00.000-08:00"},"mlaCitation":"Grandjean, V. et al. ?Stage-Specific Signals in Germ Line Differentiation: Control of Sertoli Cell Phagocytic Activity by Spermatogenic Cells.? <i>DEVELOPMENTAL BIOLOGY</i> 184.1 (1997): 165?174. Print.","pubMedId":"9142992","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/9142992","publicationId":242889,"title":"Stage-specific signals in germ line differentiation: Control of Sertoli cell phagocytic activity by spermatogenic cells","type":"article","webOfScienceId":"A1997WV15600015","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/A1997WV15600015"},{"abstract":"A cell culture system that supports the differentiation of male germ cells through meiosis is described. It takes advantage of the properties of a cell line, 15P-1, established from testicular cells of transgenic mice that express the large T protein of polyoma virus in the seminiferous epithelium. This line exhibits features characteristics of Sertoli cells, including transcription of the Wilms' tumor (WT1) and Steel genes. Cells of the 15P-1 type support the meiotic and postmeiotic differentiation in cocultures of diploid premeiotic germ cells into haploid spermatids expressing the protamine (Prm-1) gene. When cocultured with 15P-1 cells, testicular cells explanted from immature 9-day-old animals, before the onset of the first meiosis, generated tetrads of haploid cells with the morphology of round spermatids and initiated protamine transcription.","apaCitation":"Rassoulzadegan, M., PAQUISFLUCKLINGER, V., Bertino, B., Sage, J., Jasin, M., Miyagawa, K., ? Cuzin, F. (1993). TRANSMEIOTIC DIFFERENTIATION OF MALE GERM-CELLS IN CULTURE. <i>CELL</i>, <i>75</i>(5), 997?1006.","capCitation":"<span class=\"title\">\n  <span>TRANSMEIOTIC DIFFERENTIATION OF MALE GERM-CELLS IN CULTURE</span>\n  <i>CELL</i>\n</span>\n<span class=\"authors\">Rassoulzadegan, M., PAQUISFLUCKLINGER, V., Bertino, B., Sage, J., Jasin, M., Miyagawa, K., VANHEYNINGEN, V., Besmer, P., Cuzin, F.</span>\n\n<span class=\"details\">\n   <span class=\"year\">1993</span>; <span class=\"volume\">75 (5)</span><span class=\"pages\">: 997-1006</span>\n</span>","chicagoCitation":"Rassoulzadegan, M., V. PAQUISFLUCKLINGER, B. Bertino, J. Sage, M. Jasin, et al. 1993. ?TRANSMEIOTIC DIFFERENTIATION OF MALE GERM-CELLS IN CULTURE.? <i>CELL</i> 75 (5). CELL PRESS: 997?1006.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">A cell culture system that supports the differentiation of male germ cells through meiosis is described. It takes advantage of the properties of a cell line, 15P-1, established from testicular cells of transgenic mice that express the large T protein of polyoma virus in the seminiferous epithelium. This line exhibits features characteristics of Sertoli cells, including transcription of the Wilms' tumor (WT1) and Steel genes. Cells of the 15P-1 type support the meiotic and postmeiotic differentiation in cocultures of diploid premeiotic germ cells into haploid spermatids expressing the protamine (Prm-1) gene. When cocultured with 15P-1 cells, testicular cells explanted from immature 9-day-old animals, before the onset of the first meiosis, generated tetrads of haploid cells with the morphology of round spermatids and initiated protamine transcription.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/A1993MK96600019\">Web of Science ID A1993MK96600019</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/7504588\">PubMedID 7504588</a></span>\n  </p>","text":"Abstract:\nA cell culture system that supports the differentiation of male germ cells through meiosis is described. It takes advantage of the properties of a cell line, 15P-1, established from testicular cells of transgenic mice that express the large T protein of polyoma virus in the seminiferous epithelium. This line exhibits features characteristics of Sertoli cells, including transcription of the Wilms' tumor (WT1) and Steel genes. Cells of the 15P-1 type support the meiotic and postmeiotic differentiation in cocultures of diploid premeiotic germ cells into haploid spermatids expressing the protamine (Prm-1) gene. When cocultured with 15P-1 cells, testicular cells explanted from immature 9-day-old animals, before the onset of the first meiosis, generated tetrads of haploid cells with the morphology of round spermatids and initiated protamine transcription.\n\nWeb of Science ID: A1993MK96600019, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/A1993MK96600019\nPubMedID: 7504588, https://www.ncbi.nlm.nih.gov/pubmed/7504588"},"featured":true,"firstPublished":{"html":"Dec 3, 1993 00:00:00 AM","text":"Dec 3, 1993 00:00:00 AM","value":"1993-12-03T00:00:00.000-08:00"},"mlaCitation":"Rassoulzadegan, M. et al. ?TRANSMEIOTIC DIFFERENTIATION OF MALE GERM-CELLS IN CULTURE.? <i>CELL</i> 75.5 (1993): 997?1006. Print.","pubMedId":"7504588","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/7504588","publicationId":242891,"title":"TRANSMEIOTIC DIFFERENTIATION OF MALE GERM-CELLS IN CULTURE","type":"article","webOfScienceId":"A1993MK96600019","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/A1993MK96600019"}],"registryId":"86efbfdcf61311d2ae660800209269ec","searchTokens":["julsage@stanford.edu","julsage@stanford"],"shortTitle":{"academicInstituteDisplay":false,"affiliation":"capFaculty","bannerDisplay":true,"label":{"html":"Professor of Pediatrics (Hematology/Oncology) and of Genetics","text":"Professor of Pediatrics (Hematology/Oncology) and of Genetics"},"organization":{"interDepartmentalProgram":false,"orgCode":"NULL"},"title":"Professor of Pediatrics (Hematology/Oncology) and of Genetics","type":"faculty"},"similarProfiles":[{"profileId":19907,"type":"similarPublications"},{"profileId":5931,"type":"similarPublications"},{"profileId":3851,"type":"similarPublications"},{"profileId":72452,"type":"similarPublications"},{"profileId":4726,"type":"similarPublications"},{"profileId":4517,"type":"similarPublications"},{"profileId":7991,"type":"similarPublications"},{"profileId":3892,"type":"similarPublications"},{"profileId":10445,"type":"similarPublications"},{"profileId":4658,"type":"similarPublications"},{"profileId":4713,"type":"similarPublications"},{"profileId":6012,"type":"similarPublications"},{"profileId":6066,"type":"similarPublications"},{"profileId":6423,"type":"similarPublications"},{"profileId":18937,"type":"similarPublications"},{"profileId":37666,"type":"similarPublications"},{"profileId":73156,"type":"similarPublications"},{"profileId":8514,"type":"similarPublications"},{"profileId":4159,"type":"similarPublications"},{"profileId":9405,"type":"similarPublications"},{"profileId":19524,"type":"similarPublications"}],"titles":[{"academicInstituteDisplay":true,"affiliation":"capFaculty","appointmentType":"pr","bannerDisplay":false,"jobCode":"1119","label":{"html":"Professor, <a href='http://pedhemonc.stanford.edu/'> Pediatrics - Hematology & Oncology</a>","text":"Professor, Pediatrics - Hematology & Oncology"},"organization":{"interDepartmentalProgram":false,"label":{"html":"<a href='http://pedhemonc.stanford.edu/'>Pediatrics - Hematology & Oncology</a>","text":"Pediatrics - Hematology & Oncology"},"org":"Pediatrics - Hematology & Oncology","orgCode":"XIXC","orgUrl":"http://pedhemonc.stanford.edu/"},"title":"Professor","type":"faculty"},{"academicInstituteDisplay":true,"affiliation":"capFaculty","appointmentType":"s0","bannerDisplay":false,"jobCode":"1119","label":{"html":"Professor, <a href='http://genetics.stanford.edu/'> Genetics</a>","text":"Professor, Genetics"},"organization":{"interDepartmentalProgram":false,"label":{"html":"<a href='http://genetics.stanford.edu/'>Genetics</a>","text":"Genetics"},"org":"Genetics","orgCode":"WAZC","orgUrl":"http://genetics.stanford.edu/"},"title":"Professor","type":"faculty"},{"academicInstituteDisplay":true,"affiliation":"capFaculty","appointmentType":"mb","bannerDisplay":false,"jobCode":"1234","label":{"html":"Member, <a href='http://biox.stanford.edu/'> Bio-X</a>","text":"Member, Bio-X"},"organization":{"interDepartmentalProgram":false,"label":{"html":"<a href='http://biox.stanford.edu/'>Bio-X</a>","text":"Bio-X"},"org":"Bio-X","orgCode":"BIOX","orgUrl":"http://biox.stanford.edu/"},"title":"Member","type":"faculty"},{"academicInstituteDisplay":true,"affiliation":"capFaculty","appointmentType":"mb","bannerDisplay":false,"jobCode":"1234","label":{"html":"Member, <a href='http://chri.stanford.edu/'> Child Health Research Institute</a>","text":"Member, Child Health Research Institute"},"organization":{"interDepartmentalProgram":false,"label":{"html":"<a href='http://chri.stanford.edu/'>Child Health Research Institute</a>","text":"Child Health Research Institute"},"org":"Child Health Research Institute","orgCode":"I001","orgUrl":"http://chri.stanford.edu/"},"title":"Member","type":"faculty"},{"academicInstituteDisplay":true,"affiliation":"capFaculty","appointmentType":"mb","bannerDisplay":false,"jobCode":"1234","label":{"html":"Member, <a href='http://cancer.stanford.edu'> Stanford Cancer Institute</a>","text":"Member, Stanford Cancer Institute"},"organization":{"interDepartmentalProgram":false,"label":{"html":"<a href='http://cancer.stanford.edu'>Stanford Cancer Institute</a>","text":"Stanford Cancer Institute"},"org":"Stanford Cancer Institute","orgCode":"SCC3","orgUrl":"http://cancer.stanford.edu"},"title":"Member","type":"faculty"},{"academicInstituteDisplay":true,"affiliation":"capFaculty","appointmentType":"mb","bannerDisplay":false,"jobCode":"C622","label":{"html":"Faculty Fellow, <a href='https://chemh.stanford.edu/'> Stanford ChEM-H</a>","text":"Faculty Fellow, Stanford ChEM-H"},"organization":{"interDepartmentalProgram":false,"label":{"html":"<a href='https://chemh.stanford.edu/'>Stanford ChEM-H</a>","text":"Stanford ChEM-H"},"org":"Stanford ChEM-H","orgCode":"NOQ1","orgUrl":"https://chemh.stanford.edu/"},"title":"Faculty Fellow","type":"faculty"}],"uid":"julsage","universityId":"05348051"},"ui":{"bioTabActive":true,"bioTabUrl":"/profiles/hesc/julien-sage?tab=bio","clinicalTrialResults":{"count":2,"trials":[{"briefSummary":"\n      Many children with the childhood cancer, Retinoblastoma, have surgery to remove the tumor\n      and sometimes the entire eye. The purpose of this study is to collect the extra tissue from\n      patients who undergo tumor removal for laboratory experiments that will help us understand\n      not only what occurs in retinoblastoma cells but also how cells normally function. Some of\n      these studies will include an evaluation of how cells control the way that genes are\n      expressed, how cells \"know\" to become retinal cells, how cells remain retinal cells, how\n      cells lose their identity as retinal cells, what changes make retinoblastoma cells different\n      from normal retinal cells, and what changes make some retinoblastomas worse than others.\n    ","clinicalTrialUrl":"https://med.stanford.edu/clinicaltrials/trials/NCT01642823","contactEmail":"julsage@stanford.edu","contactName":"Julien Sage","contactPhone":"650-723-5113","formattedContactPhoneNumber":"650-723-5113","hasContactInfo":true,"id":"NCT01642823","investigatorApiUrl":"/profiles/proxy/api/cap/profiles?ids=6009,8178","investigators":[{"displayName":"JulienÂ Sage","firstName":"Julien","lastName":"Sage","principalInvestigator":true,"profileId":6009},{"displayName":"DouglasÂ Fredrick, MD","firstName":"Douglas","lastName":"Fredrick","middleName":"R","principalInvestigator":false,"profileId":8178}],"recruiting":false,"stanfordRecruitingStatus":"NOT_RECRUITING","studyTitle":"Cancer Biology of Retinoblastoma","trialsApiUrl":"/profiles/proxy/api/cap/trials/NCT01642823"},{"briefSummary":"\n      Intrapatient dose escalation study of desipramine in subjects with small cell lung cancer\n      (SCLC) and other high-grade neuroendocrine tumors.\n    ","clinicalTrialUrl":"https://med.stanford.edu/clinicaltrials/trials/NCT01719861","contactEmail":"ccto-office@stanford.edu","contactName":"CCTO","contactPhone":"650-498-7061","formattedContactPhoneNumber":"650-498-7061","hasContactInfo":true,"id":"NCT01719861","investigatorApiUrl":"/profiles/proxy/api/cap/profiles?ids=6009,19045,4194,6035,6058","investigators":[{"displayName":"JulienÂ Sage","firstName":"Julien","lastName":"Sage","principalInvestigator":false,"profileId":6009},{"displayName":"Joel Neal, MD, PhD","firstName":"Joel","lastName":"Neal","middleName":"William","principalInvestigator":true,"profileId":19045},{"displayName":"Alan F. Schatzberg","firstName":"Alan","lastName":"Schatzberg","middleName":"Frederic","principalInvestigator":false,"profileId":4194},{"displayName":"Pamela L. Kunz","firstName":"Pamela","lastName":"Kunz","middleName":"Lyn","principalInvestigator":false,"profileId":6035},{"displayName":"HeatherÂ Wakelee, MD","firstName":"Heather","lastName":"Wakelee","middleName":"Ann","principalInvestigator":false,"profileId":6058}],"recruiting":false,"stanfordRecruitingStatus":"NOT_RECRUITING","studyTitle":"Phase 2a Desipramine in Small Cell Lung Cancer and Other High-Grade Neuroendocrine Tumors","trialsApiUrl":"/profiles/proxy/api/cap/trials/NCT01719861"}]},"currentResearchInterests":"Our research program focuses on the mechanisms that control the proliferation of mammalian cells under normal and pathological conditions (regeneration, cancer), with a particular emphasis on stem cells and gene regulatory networks. We combine genetic, genomics, and proteomics approaches to identify and investigate genes and pathways involved in cancer initiation and progression. We use genome-editing strategies to develop and study genetically-engineered mouse models for human cancers, including lung cancer, pancreatic cancer, and liver cancer. Our work spans the investigation of fundamental biological processes to the implementation of clinical trials based on our findings in pre-clinical models.","emailBody":"I would like to share the profile of JulienÂ Sage with you. https://profiles.stanford.edu/julien-sage","emailSubject":"JulienÂ Sage's Profile","hasBioTab":true,"hasProfessionalTab":false,"hasPublicationsTab":true,"hasResearchAndScholarshipTab":true,"hasTeachingTab":true,"headerAcademicAppointments":"Academic Appointments","headerProfessionalEducation":"Professional Education","headerProfessionalOrganizations":"Boards, Advisory Committees, Professional Organizations","hideTabs":false,"independentStudiesCount":13,"intranetUrl":"https://cap.stanford.edu/profiles/auth/frdActionServlet?choiceId=facProfile&profileId=6009","pdfUrl":"https://cap.stanford.edu/profiles/frdActionServlet?choiceId=printerprofile&profileversion=full&profileId=6009","primaryOrganizationName":"Pediatrics - Hematology & Oncology","priorYearCourses":[],"professionalTabUrl":"/profiles/hesc/julien-sage?tab=professional","publicUrl":"https://profiles.stanford.edu/julien-sage","publicationSummary":{"all":{"count":85},"articles":{"count":85,"publications":[{"abstract":"Small cell lung cancer is initially highly responsive to cisplatin and etoposide but in almost every case becomes rapidly chemoresistant, leading to death within 1 year. We modeled acquired chemoresistance inÂ vivo using a series of patient-derived xenografts to generate paired chemosensitive and chemoresistant cancers. Multiple chemoresistant models demonstrated suppression of SLFN11, a factor implicated in DNA-damage repair deficiency. InÂ vivo silencing of SLFN11 was associated with marked deposition of H3K27me3, a histone modification placed by EZH2, within the gene body of SLFN11, inducing local chromatin condensation and gene silencing. Inclusion of an EZH2 inhibitor with standard cytotoxic therapies prevented emergence of acquired resistance and augmented chemotherapeutic efficacy in both chemosensitive and chemoresistant models of small cell lung cancer.","apaCitation":"Gardner, E. E., Lok, B. H., Schneeberger, V. E., Desmeules, P., Miles, L. A., Arnold, P. K., ? Poirier, J. T. (2017). Chemosensitive Relapse in Small Cell Lung Cancer Proceeds through an EZH2-SLFN11 Axis. <i>Cancer Cell</i>, <i>31</i>(2), 286?299.","capCitation":"<span class=\"title\">\n  <span>Chemosensitive Relapse in Small Cell Lung Cancer Proceeds through an EZH2-SLFN11 Axis.</span>\n  <i>Cancer cell</i>\n</span>\n<span class=\"authors\">Gardner, E. E., Lok, B. H., Schneeberger, V. E., Desmeules, P., Miles, L. A., Arnold, P. K., Ni, A., Khodos, I., de Stanchina, E., Nguyen, T., Sage, J., Campbell, J. E., Ribich, S., Rekhtman, N., Dowlati, A., Massion, P. P., Rudin, C. M., Poirier, J. T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2017</span>; <span class=\"volume\">31 (2)</span><span class=\"pages\">: 286-299</span>\n</span>","chicagoCitation":"Gardner, Eric E., Benjamin H. Lok, Valentina E. Schneeberger, Patrice Desmeules, Linde A. Miles, et al. 2017. ?Chemosensitive Relapse in Small Cell Lung Cancer Proceeds through an EZH2-SLFN11 Axis.? <i>Cancer Cell</i> 31 (2). ELSEVIER INC: 286?99.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small cell lung cancer is initially highly responsive to cisplatin and etoposide but in almost every case becomes rapidly chemoresistant, leading to death within 1 year. We modeled acquired chemoresistance inÂ vivo using a series of patient-derived xenografts to generate paired chemosensitive and chemoresistant cancers. Multiple chemoresistant models demonstrated suppression of SLFN11, a factor implicated in DNA-damage repair deficiency. InÂ vivo silencing of SLFN11 was associated with marked deposition of H3K27me3, a histone modification placed by EZH2, within the gene body of SLFN11, inducing local chromatin condensation and gene silencing. Inclusion of an EZH2 inhibitor with standard cytotoxic therapies prevented emergence of acquired resistance and augmented chemotherapeutic efficacy in both chemosensitive and chemoresistant models of small cell lung cancer.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.ccell.2017.01.006\">DOI 10.1016/j.ccell.2017.01.006</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/28196596\">PubMedID 28196596</a></span>\n  </p>","text":"Abstract:\nSmall cell lung cancer is initially highly responsive to cisplatin and etoposide but in almost every case becomes rapidly chemoresistant, leading to death within 1 year. We modeled acquired chemoresistance inÂ vivo using a series of patient-derived xenografts to generate paired chemosensitive and chemoresistant cancers. Multiple chemoresistant models demonstrated suppression of SLFN11, a factor implicated in DNA-damage repair deficiency. InÂ vivo silencing of SLFN11 was associated with marked deposition of H3K27me3, a histone modification placed by EZH2, within the gene body of SLFN11, inducing local chromatin condensation and gene silencing. Inclusion of an EZH2 inhibitor with standard cytotoxic therapies prevented emergence of acquired resistance and augmented chemotherapeutic efficacy in both chemosensitive and chemoresistant models of small cell lung cancer.\nDOI: 10.1016/j.ccell.2017.01.006, https://dx.doi.org/10.1016/j.ccell.2017.01.006\n\nPubMedID: 28196596, https://www.ncbi.nlm.nih.gov/pubmed/28196596"},"doiId":"10.1016/j.ccell.2017.01.006","doiUrl":"https://dx.doi.org/10.1016/j.ccell.2017.01.006","featured":false,"firstPublished":{"html":"Feb 13, 2017 00:00:00 AM","text":"Feb 13, 2017 00:00:00 AM","value":"2017-02-13T00:00:00.000-08:00"},"mlaCitation":"Gardner, Eric E. et al. ?Chemosensitive Relapse in Small Cell Lung Cancer Proceeds through an EZH2-SLFN11 Axis.? <i>Cancer cell</i> 31.2 (2017): 286?299. Print.","pubMedId":"28196596","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/28196596","publicationId":438256,"title":"Chemosensitive Relapse in Small Cell Lung Cancer Proceeds through an EZH2-SLFN11 Axis.","type":"article"},{"abstract":"The E2F family of transcription factors is a key determinant of cell proliferation in response to extra- and intra-cellular signals. Within this family, E2F4 is a transcriptional repressor whose activity is critical to engage and maintain cell cycle arrest in G0/G1 in conjunction with members of the retinoblastoma (RB) family. However, recent observations challenge this paradigm and indicate that E2F4 has a multitude of functions in cells besides this cell cycle regulatory role, including in embryonic and adult stem cells, during regenerative processes, and in cancer. Some of these new functions are independent of the RB family and involve direct activation of target genes. Here we review the canonical functions of E2F4 and discuss recent evidence expanding the role of this transcription factor, with a focus on cell fate decisions in tissue homeostasis and regeneration.","apaCitation":"Hsu, J., &amp; Sage, J. (2016). Novel functions for the transcription factor E2F4 in development and disease. <i>Cell Cycle </i>, 1?8.","capCitation":"<span class=\"title\">\n  <span>Novel functions for the transcription factor E2F4 in development and disease.</span>\n  <i>Cell cycle </i>\n</span>\n<span class=\"authors\">Hsu, J., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span><span class=\"pages\">: 1-8</span>\n</span>","chicagoCitation":"Hsu, Jenny, and Julien Sage. 2016. ?Novel Functions for the Transcription Factor E2F4 in Development and Disease.? <i>Cell Cycle </i>, 1?8.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The E2F family of transcription factors is a key determinant of cell proliferation in response to extra- and intra-cellular signals. Within this family, E2F4 is a transcriptional repressor whose activity is critical to engage and maintain cell cycle arrest in G0/G1 in conjunction with members of the retinoblastoma (RB) family. However, recent observations challenge this paradigm and indicate that E2F4 has a multitude of functions in cells besides this cell cycle regulatory role, including in embryonic and adult stem cells, during regenerative processes, and in cancer. Some of these new functions are independent of the RB family and involve direct activation of target genes. Here we review the canonical functions of E2F4 and discuss recent evidence expanding the role of this transcription factor, with a focus on cell fate decisions in tissue homeostasis and regeneration.</p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/27753528\">PubMedID 27753528</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5176148\">PubMedCentralID PMC5176148</a></span>\n  </p>","text":"Abstract:\nThe E2F family of transcription factors is a key determinant of cell proliferation in response to extra- and intra-cellular signals. Within this family, E2F4 is a transcriptional repressor whose activity is critical to engage and maintain cell cycle arrest in G0/G1 in conjunction with members of the retinoblastoma (RB) family. However, recent observations challenge this paradigm and indicate that E2F4 has a multitude of functions in cells besides this cell cycle regulatory role, including in embryonic and adult stem cells, during regenerative processes, and in cancer. Some of these new functions are independent of the RB family and involve direct activation of target genes. Here we review the canonical functions of E2F4 and discuss recent evidence expanding the role of this transcription factor, with a focus on cell fate decisions in tissue homeostasis and regeneration.\n\n\nPubMedID: 27753528, https://www.ncbi.nlm.nih.gov/pubmed/27753528\nPubMedCentralID: PMC5176148, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5176148"},"featured":false,"firstPublished":{"html":"Oct 18, 2016 00:00:00 AM","text":"Oct 18, 2016 00:00:00 AM","value":"2016-10-18T00:00:00.000-07:00"},"mlaCitation":"Hsu, Jenny, and Julien Sage. ?Novel Functions for the Transcription Factor E2F4 in Development and Disease.? <i>Cell cycle </i>(2016): 1?8. Print.","pubMedId":"27753528","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/27753528","publicationId":426898,"title":"Novel functions for the transcription factor E2F4 in development and disease.","type":"article"},{"abstract":"The cellular turnover of adult tissues and injury-induced repair proceed through an exquisite integration of proliferation, differentiation, and survival signals that involve stem/progenitor cell populations, their progeny, and differentiated tissues. GATA factors are DNA binding proteins that control stem cells and the development of tissues by activating or repressing transcription. Here we examined the role of GATA transcription factors in Schmidtea mediterranea, a freshwater planarian that provides an excellent model to investigate gene function in adult stem cells, regeneration, and differentiation. Smed-gata4/5/6, the homolog of the three mammalian GATA-4,-5,-6 factors is expressed at high levels in differentiated gut cells but also at lower levels in neoblast populations, the planarian stem cells. Smed-gata4/5/6 knock-down results in broad differentiation defects, especially in response to injury. These defects are not restricted to the intestinal lineage. In particular, at late time points during the response to injury, loss of Smed-gata4/5/6 leads to decreased neoblast proliferation and to gene expression changes in several neoblast subpopulations. Thus, Smed-gata4/5/6 plays a key evolutionary conserved role in intestinal differentiation in planarians. These data further support a model in which defects in the intestinal lineage can indirectly affect other differentiation pathways in planarians.","apaCitation":"Flores, N. M., Oviedo, N. J., &amp; Sage, J. (2016). Essential role for the planarian intestinal GATA transcription factor in stem cells and regeneration. <i>Developmental Biology</i>, <i>418</i>(1), 179?188.","capCitation":"<span class=\"title\">\n  <span>Essential role for the planarian intestinal GATA transcription factor in stem cells and regeneration.</span>\n  <i>Developmental biology</i>\n</span>\n<span class=\"authors\">Flores, N. M., Oviedo, N. J., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">418 (1)</span><span class=\"pages\">: 179-188</span>\n</span>","chicagoCitation":"Flores, Natasha M., NÃ©stor J. Oviedo, and Julien Sage. 2016. ?Essential Role for the Planarian Intestinal GATA Transcription Factor in Stem Cells and Regeneration.? <i>Developmental Biology</i> 418 (1). ELSEVIER INC: 179?88.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The cellular turnover of adult tissues and injury-induced repair proceed through an exquisite integration of proliferation, differentiation, and survival signals that involve stem/progenitor cell populations, their progeny, and differentiated tissues. GATA factors are DNA binding proteins that control stem cells and the development of tissues by activating or repressing transcription. Here we examined the role of GATA transcription factors in Schmidtea mediterranea, a freshwater planarian that provides an excellent model to investigate gene function in adult stem cells, regeneration, and differentiation. Smed-gata4/5/6, the homolog of the three mammalian GATA-4,-5,-6 factors is expressed at high levels in differentiated gut cells but also at lower levels in neoblast populations, the planarian stem cells. Smed-gata4/5/6 knock-down results in broad differentiation defects, especially in response to injury. These defects are not restricted to the intestinal lineage. In particular, at late time points during the response to injury, loss of Smed-gata4/5/6 leads to decreased neoblast proliferation and to gene expression changes in several neoblast subpopulations. Thus, Smed-gata4/5/6 plays a key evolutionary conserved role in intestinal differentiation in planarians. These data further support a model in which defects in the intestinal lineage can indirectly affect other differentiation pathways in planarians.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.ydbio.2016.08.015\">DOI 10.1016/j.ydbio.2016.08.015</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/27542689\">PubMedID 27542689</a></span>\n  </p>","text":"Abstract:\nThe cellular turnover of adult tissues and injury-induced repair proceed through an exquisite integration of proliferation, differentiation, and survival signals that involve stem/progenitor cell populations, their progeny, and differentiated tissues. GATA factors are DNA binding proteins that control stem cells and the development of tissues by activating or repressing transcription. Here we examined the role of GATA transcription factors in Schmidtea mediterranea, a freshwater planarian that provides an excellent model to investigate gene function in adult stem cells, regeneration, and differentiation. Smed-gata4/5/6, the homolog of the three mammalian GATA-4,-5,-6 factors is expressed at high levels in differentiated gut cells but also at lower levels in neoblast populations, the planarian stem cells. Smed-gata4/5/6 knock-down results in broad differentiation defects, especially in response to injury. These defects are not restricted to the intestinal lineage. In particular, at late time points during the response to injury, loss of Smed-gata4/5/6 leads to decreased neoblast proliferation and to gene expression changes in several neoblast subpopulations. Thus, Smed-gata4/5/6 plays a key evolutionary conserved role in intestinal differentiation in planarians. These data further support a model in which defects in the intestinal lineage can indirectly affect other differentiation pathways in planarians.\nDOI: 10.1016/j.ydbio.2016.08.015, https://dx.doi.org/10.1016/j.ydbio.2016.08.015\n\nPubMedID: 27542689, https://www.ncbi.nlm.nih.gov/pubmed/27542689"},"doiId":"10.1016/j.ydbio.2016.08.015","doiUrl":"https://dx.doi.org/10.1016/j.ydbio.2016.08.015","featured":false,"firstPublished":{"html":"Oct 1, 2016 00:00:00 AM","text":"Oct 1, 2016 00:00:00 AM","value":"2016-10-01T00:00:00.000-07:00"},"mlaCitation":"Flores, Natasha M., NÃ©stor J. Oviedo, and Julien Sage. ?Essential Role for the Planarian Intestinal GATA Transcription Factor in Stem Cells and Regeneration.? <i>Developmental biology</i> 418.1 (2016): 179?188. Print.","pubMedId":"27542689","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/27542689","publicationId":421260,"title":"Essential role for the planarian intestinal GATA transcription factor in stem cells and regeneration.","type":"article"},{"abstract":"The activity of the RAF/MEK/ERK signaling pathway is critical for the proliferation of normal and cancerous cells. Oncogenic mutations driving the development of lung adenocarcinoma often activate this signaling pathway. In contrast, pathway activity levels and their biological roles are not well established in small cell lung cancer (SCLC), a fast-growing neuroendocrine lung cancer subtype. Here we discuss the function of the RAF/MEK/ERK kinase pathway and the mechanisms leading to its activation in SCLC cells. In particular, we argue that activation of this pathway may be beneficial to the survival, proliferation, and spread of SCLC cells in response to multiple stimuli. We also consider evidence that high levels of RAF/MEK/ERK pathway activity may be detrimental to SCLC tumors, including in part byÂ interfering with their neuroendocrine fate. On the basis of these observations, we examined when small molecules targeting kinases in the RAF/MEK/ERK pathway may be useful therapeutically in patients with SCLC, including in combination with other therapeutic agents.","apaCitation":"Cristea, S., &amp; Sage, J. (2016). Is the Canonical RAF/MEK/ERK Signaling Pathway aÂ Therapeutic Target in SCLC? <i>Journal of Thoracic Oncology </i>, <i>11</i>(8), 1233?1241.","capCitation":"<span class=\"title\">\n  <span>Is the Canonical RAF/MEK/ERK Signaling Pathway aÂ Therapeutic Target in SCLC?</span>\n  <i>Journal of thoracic oncology </i>\n</span>\n<span class=\"authors\">Cristea, S., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">11 (8)</span><span class=\"pages\">: 1233-1241</span>\n</span>","chicagoCitation":"Cristea, Sandra, and Julien Sage. 2016. ?Is the Canonical RAF/MEK/ERK Signaling Pathway aÂ Therapeutic Target in SCLC?? <i>Journal of Thoracic Oncology </i>11 (8). ELSEVIER INC: 1233?41.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The activity of the RAF/MEK/ERK signaling pathway is critical for the proliferation of normal and cancerous cells. Oncogenic mutations driving the development of lung adenocarcinoma often activate this signaling pathway. In contrast, pathway activity levels and their biological roles are not well established in small cell lung cancer (SCLC), a fast-growing neuroendocrine lung cancer subtype. Here we discuss the function of the RAF/MEK/ERK kinase pathway and the mechanisms leading to its activation in SCLC cells. In particular, we argue that activation of this pathway may be beneficial to the survival, proliferation, and spread of SCLC cells in response to multiple stimuli. We also consider evidence that high levels of RAF/MEK/ERK pathway activity may be detrimental to SCLC tumors, including in part byÂ interfering with their neuroendocrine fate. On the basis of these observations, we examined when small molecules targeting kinases in the RAF/MEK/ERK pathway may be useful therapeutically in patients with SCLC, including in combination with other therapeutic agents.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.jtho.2016.04.018\">DOI 10.1016/j.jtho.2016.04.018</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/27133774\">PubMedID 27133774</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5391975\">PubMedCentralID PMC5391975</a></span>\n  </p>","text":"Abstract:\nThe activity of the RAF/MEK/ERK signaling pathway is critical for the proliferation of normal and cancerous cells. Oncogenic mutations driving the development of lung adenocarcinoma often activate this signaling pathway. In contrast, pathway activity levels and their biological roles are not well established in small cell lung cancer (SCLC), a fast-growing neuroendocrine lung cancer subtype. Here we discuss the function of the RAF/MEK/ERK kinase pathway and the mechanisms leading to its activation in SCLC cells. In particular, we argue that activation of this pathway may be beneficial to the survival, proliferation, and spread of SCLC cells in response to multiple stimuli. We also consider evidence that high levels of RAF/MEK/ERK pathway activity may be detrimental to SCLC tumors, including in part byÂ interfering with their neuroendocrine fate. On the basis of these observations, we examined when small molecules targeting kinases in the RAF/MEK/ERK pathway may be useful therapeutically in patients with SCLC, including in combination with other therapeutic agents.\nDOI: 10.1016/j.jtho.2016.04.018, https://dx.doi.org/10.1016/j.jtho.2016.04.018\n\nPubMedID: 27133774, https://www.ncbi.nlm.nih.gov/pubmed/27133774\nPubMedCentralID: PMC5391975, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5391975"},"doiId":"10.1016/j.jtho.2016.04.018","doiUrl":"https://dx.doi.org/10.1016/j.jtho.2016.04.018","featured":false,"firstPublished":{"html":"Aug 1, 2016 00:00:00 AM","text":"Aug 1, 2016 00:00:00 AM","value":"2016-08-01T00:00:00.000-07:00"},"mlaCitation":"Cristea, Sandra, and Julien Sage. ?Is the Canonical RAF/MEK/ERK Signaling Pathway aÂ Therapeutic Target in SCLC?? <i>Journal of thoracic oncology </i>11.8 (2016): 1233?1241. Print.","pubMedId":"27133774","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/27133774","publicationId":417630,"title":"Is the Canonical RAF/MEK/ERK Signaling Pathway aÂ Therapeutic Target in SCLC?","type":"article"},{"abstract":"Small cell lung cancer (SCLC) is a neuroendocrine lung cancer characterized by fast growth, early dissemination, and rapid resistance to chemotherapy. We identified a population of long-term tumor-propagating cells (TPCs) in a mouse model of SCLC. This population, marked by high levels of EpCAM and CD24, is also prevalent in human primary SCLC tumors. Murine SCLC TPCs are numerous and highly proliferative but not intrinsically chemoresistant, indicating that not all clinical features of SCLCÂ are linked to TPCs. SCLC TPCs possess a distinct transcriptional profile compared to non-TPCs, including elevated MYC activity. Genetic and pharmacological inhibition of MYC in SCLC cells to non-TPC levels inhibits long-term propagation but not short-term growth. These studies identify a highly tumorigenic population of SCLC cells in mouse models, cell lines, and patient tumors and a means to target them in this most fatal form of lung cancer.","apaCitation":"Jahchan, N. S., Lim, J. S., Bola, B., Morris, K., Seitz, G., Tran, K. Q., ? Sage, J. (2016). Identification and Targeting of Long-Term Tumor-Propagating Cells in Small Cell Lung Cancer. <i>Cell Reports</i>, <i>16</i>(3), 644?656.","capCitation":"<span class=\"title\">\n  <span>Identification and Targeting of Long-Term Tumor-Propagating Cells in Small Cell Lung Cancer.</span>\n  <i>Cell reports</i>\n</span>\n<span class=\"authors\">Jahchan, N. S., Lim, J. S., Bola, B., Morris, K., Seitz, G., Tran, K. Q., Xu, L., Trapani, F., Morrow, C. J., Cristea, S., Coles, G. L., Yang, D., Vaka, D., Kareta, M. S., George, J., Mazur, P. K., Nguyen, T., Anderson, W. C., Dylla, S. J., Blackhall, F., Peifer, M., Dive, C., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">16 (3)</span><span class=\"pages\">: 644-656</span>\n</span>","chicagoCitation":"Jahchan, Nadine S., Jing Shan Lim, Becky Bola, Karen Morris, Garrett Seitz, et al. 2016. ?Identification and Targeting of Long-Term Tumor-Propagating Cells in Small Cell Lung Cancer.? <i>Cell Reports</i> 16 (3). ELSEVIER INC: 644?56.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small cell lung cancer (SCLC) is a neuroendocrine lung cancer characterized by fast growth, early dissemination, and rapid resistance to chemotherapy. We identified a population of long-term tumor-propagating cells (TPCs) in a mouse model of SCLC. This population, marked by high levels of EpCAM and CD24, is also prevalent in human primary SCLC tumors. Murine SCLC TPCs are numerous and highly proliferative but not intrinsically chemoresistant, indicating that not all clinical features of SCLCÂ are linked to TPCs. SCLC TPCs possess a distinct transcriptional profile compared to non-TPCs, including elevated MYC activity. Genetic and pharmacological inhibition of MYC in SCLC cells to non-TPC levels inhibits long-term propagation but not short-term growth. These studies identify a highly tumorigenic population of SCLC cells in mouse models, cell lines, and patient tumors and a means to target them in this most fatal form of lung cancer.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.celrep.2016.06.021\">DOI 10.1016/j.celrep.2016.06.021</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/27373157\">PubMedID 27373157</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4956576\">PubMedCentralID PMC4956576</a></span>\n  </p>","text":"Abstract:\nSmall cell lung cancer (SCLC) is a neuroendocrine lung cancer characterized by fast growth, early dissemination, and rapid resistance to chemotherapy. We identified a population of long-term tumor-propagating cells (TPCs) in a mouse model of SCLC. This population, marked by high levels of EpCAM and CD24, is also prevalent in human primary SCLC tumors. Murine SCLC TPCs are numerous and highly proliferative but not intrinsically chemoresistant, indicating that not all clinical features of SCLCÂ are linked to TPCs. SCLC TPCs possess a distinct transcriptional profile compared to non-TPCs, including elevated MYC activity. Genetic and pharmacological inhibition of MYC in SCLC cells to non-TPC levels inhibits long-term propagation but not short-term growth. These studies identify a highly tumorigenic population of SCLC cells in mouse models, cell lines, and patient tumors and a means to target them in this most fatal form of lung cancer.\nDOI: 10.1016/j.celrep.2016.06.021, https://dx.doi.org/10.1016/j.celrep.2016.06.021\n\nPubMedID: 27373157, https://www.ncbi.nlm.nih.gov/pubmed/27373157\nPubMedCentralID: PMC4956576, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4956576"},"doiId":"10.1016/j.celrep.2016.06.021","doiUrl":"https://dx.doi.org/10.1016/j.celrep.2016.06.021","featured":true,"firstPublished":{"html":"Jul 19, 2016 00:00:00 AM","text":"Jul 19, 2016 00:00:00 AM","value":"2016-07-19T00:00:00.000-07:00"},"mlaCitation":"Jahchan, Nadine S. et al. ?Identification and Targeting of Long-Term Tumor-Propagating Cells in Small Cell Lung Cancer.? <i>Cell reports</i> 16.3 (2016): 644?656. Print.","pubMedId":"27373157","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/27373157","publicationId":416504,"title":"Identification and Targeting of Long-Term Tumor-Propagating Cells in Small Cell Lung Cancer.","type":"article"},{"abstract":"Metastases are the main cause of cancer deaths, but the mechanisms underlying metastatic progression remain poorly understood. We isolated pure populations of cancer cells from primary tumors and metastases from a genetically engineered mouse model of human small cell lung cancer (SCLC) to investigate the mechanisms that drive the metastatic spread ofÂ this lethal cancer. Genome-wide characterization of chromatin accessibility revealed the opening of large numbers of distal regulatory elements across the genome during metastatic progression. These changes correlate with copy number amplification of the Nfib locus, and differentially accessible sites were highly enriched for Nfib transcription factor binding sites. Nfib is necessary and sufficient to increase chromatin accessibility at a large subset of the intergenic regions. Nfib promotes pro-metastatic neuronal gene expression programs and drives the metastatic ability of SCLC cells. The identification of widespread chromatin changes during SCLC progression reveals an unexpected global reprogramming during metastatic progression.","apaCitation":"Denny, S. K., Yang, D., Chuang, C.-H., Brady, J. J., Lim, J. S., Gruner, B. M., ? Winslow, M. M. (2016). Nfib Promotes Metastasis through a Widespread Increase in Chromatin Accessibility. <i>CELL</i>, <i>166</i>(2), 328?342.","capCitation":"<span class=\"title\">\n  <span>Nfib Promotes Metastasis through a Widespread Increase in Chromatin Accessibility</span>\n  <i>CELL</i>\n</span>\n<span class=\"authors\">Denny, S. K., Yang, D., Chuang, C., Brady, J. J., Lim, J. S., Gruner, B. M., Chiou, S., Schep, A. N., Baral, J., Hamard, C., Antoine, M., Wislez, M., Kong, C. S., Connolly, A. J., Park, K., Sage, J., Greenleaf, W. J., Winslow, M. M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">166 (2)</span><span class=\"pages\">: 328-342</span>\n</span>","chicagoCitation":"Denny, Sarah K., Dian Yang, Chen-Hua Chuang, Jennifer J. Brady, Jing Shan Lim, et al. 2016. ?Nfib Promotes Metastasis through a Widespread Increase in Chromatin Accessibility.? <i>CELL</i> 166 (2). CELL PRESS: 328?42.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Metastases are the main cause of cancer deaths, but the mechanisms underlying metastatic progression remain poorly understood. We isolated pure populations of cancer cells from primary tumors and metastases from a genetically engineered mouse model of human small cell lung cancer (SCLC) to investigate the mechanisms that drive the metastatic spread ofÂ this lethal cancer. Genome-wide characterization of chromatin accessibility revealed the opening of large numbers of distal regulatory elements across the genome during metastatic progression. These changes correlate with copy number amplification of the Nfib locus, and differentially accessible sites were highly enriched for Nfib transcription factor binding sites. Nfib is necessary and sufficient to increase chromatin accessibility at a large subset of the intergenic regions. Nfib promotes pro-metastatic neuronal gene expression programs and drives the metastatic ability of SCLC cells. The identification of widespread chromatin changes during SCLC progression reveals an unexpected global reprogramming during metastatic progression.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.cell.2016.05.052\">DOI 10.1016/j.cell.2016.05.052</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000380255400012\">Web of Science ID 000380255400012</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/27374332\">PubMedID 27374332</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5004630\">PubMedCentralID PMC5004630</a></span>\n  </p>","text":"Abstract:\nMetastases are the main cause of cancer deaths, but the mechanisms underlying metastatic progression remain poorly understood. We isolated pure populations of cancer cells from primary tumors and metastases from a genetically engineered mouse model of human small cell lung cancer (SCLC) to investigate the mechanisms that drive the metastatic spread ofÂ this lethal cancer. Genome-wide characterization of chromatin accessibility revealed the opening of large numbers of distal regulatory elements across the genome during metastatic progression. These changes correlate with copy number amplification of the Nfib locus, and differentially accessible sites were highly enriched for Nfib transcription factor binding sites. Nfib is necessary and sufficient to increase chromatin accessibility at a large subset of the intergenic regions. Nfib promotes pro-metastatic neuronal gene expression programs and drives the metastatic ability of SCLC cells. The identification of widespread chromatin changes during SCLC progression reveals an unexpected global reprogramming during metastatic progression.\nDOI: 10.1016/j.cell.2016.05.052, https://dx.doi.org/10.1016/j.cell.2016.05.052\nWeb of Science ID: 000380255400012, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000380255400012\nPubMedID: 27374332, https://www.ncbi.nlm.nih.gov/pubmed/27374332\nPubMedCentralID: PMC5004630, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5004630"},"doiId":"10.1016/j.cell.2016.05.052","doiUrl":"https://dx.doi.org/10.1016/j.cell.2016.05.052","featured":true,"firstPublished":{"html":"Jul 14, 2016 00:00:00 AM","text":"Jul 14, 2016 00:00:00 AM","value":"2016-07-14T00:00:00.000-07:00"},"mlaCitation":"Denny, Sarah K. et al. ?Nfib Promotes Metastasis through a Widespread Increase in Chromatin Accessibility.? <i>CELL</i> 166.2 (2016): 328?342. Print.","pubMedId":"27374332","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/27374332","publicationId":413068,"title":"Nfib Promotes Metastasis through a Widespread Increase in Chromatin Accessibility","type":"article","webOfScienceId":"000380255400012","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000380255400012"},{"abstract":"Small-cell lung cancer (SCLC) is a highly aggressive subtype of lung cancer with limited treatment options. CD47 is a cell-surface molecule that promotes immune evasion by engaging signal-regulatory protein alpha (SIRP?), which serves as an inhibitory receptor on macrophages. Here, we found that CD47 is highly expressed on the surface of human SCLC cells; therefore, we investigated CD47-blocking immunotherapies as a potential approach for SCLC treatment. Disruption of the interaction of CD47 with SIRP? using anti-CD47 antibodies induced macrophage-mediated phagocytosis of human SCLC patient cells in culture. In a murine model, administration of CD47-blocking antibodies or targeted inactivation of the Cd47 gene markedly inhibited SCLC tumor growth. Furthermore, using comprehensive antibody arrays, we identified several possible therapeutic targets on the surface of SCLC cells. Antibodies to these targets, including CD56/neural cell adhesion molecule (NCAM), promoted phagocytosis in human SCLC cell lines that was enhanced when combined with CD47-blocking therapies. In light of recent clinical trials for CD47-blocking therapies in cancer treatment, these findings identify disruption of the CD47/SIRP? axis as a potential immunotherapeutic strategy for SCLC. This approach could enable personalized immunotherapeutic regimens in patients with SCLC and other cancers.","apaCitation":"Weiskopf, K., Jahchan, N. S., Schnorr, P. J., Cristea, S., Ring, A. M., Maute, R. L., ? Sage, J. (2016). CD47-blocking immunotherapies stimulate macrophage-mediated destruction of small-cell lung cancer. <i>JOURNAL OF CLINICAL INVESTIGATION</i>, <i>126</i>(7), 2610?2620.","capCitation":"<span class=\"title\">\n  <span>CD47-blocking immunotherapies stimulate macrophage-mediated destruction of small-cell lung cancer</span>\n  <i>JOURNAL OF CLINICAL INVESTIGATION</i>\n</span>\n<span class=\"authors\">Weiskopf, K., Jahchan, N. S., Schnorr, P. J., Cristea, S., Ring, A. M., Maute, R. L., Volkmer, A. K., Volkmer, J., Liu, J., Lim, J. S., Yang, D., Seitz, G., Thuyen Nguyen, T., Wu, D., Jude, K., Guerston, H., Barkal, A., Trapani, F., George, J., Poirier, J. T., Gardner, E. E., Miles, L. A., de Stanchina, E., Lofgren, S. M., Vogel, H., Winslow, M. M., Dive, C., Thomas, R. K., Rudin, C. M., van de Rijn, M., Majeti, R., Garcia, K. C., Weissman, I. L., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">126 (7)</span><span class=\"pages\">: 2610-2620</span>\n</span>","chicagoCitation":"Weiskopf, Kipp, Nadine S. Jahchan, Peter J. Schnorr, Sandra Cristea, Aaron M. Ring, et al. 2016. ?CD47-Blocking Immunotherapies Stimulate Macrophage-Mediated Destruction of Small-Cell Lung Cancer.? <i>JOURNAL OF CLINICAL INVESTIGATION</i> 126 (7). AMER SOC CLINICAL INVESTIGATION INC: 2610?20.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small-cell lung cancer (SCLC) is a highly aggressive subtype of lung cancer with limited treatment options. CD47 is a cell-surface molecule that promotes immune evasion by engaging signal-regulatory protein alpha (SIRP?), which serves as an inhibitory receptor on macrophages. Here, we found that CD47 is highly expressed on the surface of human SCLC cells; therefore, we investigated CD47-blocking immunotherapies as a potential approach for SCLC treatment. Disruption of the interaction of CD47 with SIRP? using anti-CD47 antibodies induced macrophage-mediated phagocytosis of human SCLC patient cells in culture. In a murine model, administration of CD47-blocking antibodies or targeted inactivation of the Cd47 gene markedly inhibited SCLC tumor growth. Furthermore, using comprehensive antibody arrays, we identified several possible therapeutic targets on the surface of SCLC cells. Antibodies to these targets, including CD56/neural cell adhesion molecule (NCAM), promoted phagocytosis in human SCLC cell lines that was enhanced when combined with CD47-blocking therapies. In light of recent clinical trials for CD47-blocking therapies in cancer treatment, these findings identify disruption of the CD47/SIRP? axis as a potential immunotherapeutic strategy for SCLC. This approach could enable personalized immunotherapeutic regimens in patients with SCLC and other cancers.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1172/JCI81603\">DOI 10.1172/JCI81603</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000379094800024\">Web of Science ID 000379094800024</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/27294525\">PubMedID 27294525</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4922696\">PubMedCentralID PMC4922696</a></span>\n  </p>","text":"Abstract:\nSmall-cell lung cancer (SCLC) is a highly aggressive subtype of lung cancer with limited treatment options. CD47 is a cell-surface molecule that promotes immune evasion by engaging signal-regulatory protein alpha (SIRP?), which serves as an inhibitory receptor on macrophages. Here, we found that CD47 is highly expressed on the surface of human SCLC cells; therefore, we investigated CD47-blocking immunotherapies as a potential approach for SCLC treatment. Disruption of the interaction of CD47 with SIRP? using anti-CD47 antibodies induced macrophage-mediated phagocytosis of human SCLC patient cells in culture. In a murine model, administration of CD47-blocking antibodies or targeted inactivation of the Cd47 gene markedly inhibited SCLC tumor growth. Furthermore, using comprehensive antibody arrays, we identified several possible therapeutic targets on the surface of SCLC cells. Antibodies to these targets, including CD56/neural cell adhesion molecule (NCAM), promoted phagocytosis in human SCLC cell lines that was enhanced when combined with CD47-blocking therapies. In light of recent clinical trials for CD47-blocking therapies in cancer treatment, these findings identify disruption of the CD47/SIRP? axis as a potential immunotherapeutic strategy for SCLC. This approach could enable personalized immunotherapeutic regimens in patients with SCLC and other cancers.\nDOI: 10.1172/JCI81603, https://dx.doi.org/10.1172/JCI81603\nWeb of Science ID: 000379094800024, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000379094800024\nPubMedID: 27294525, https://www.ncbi.nlm.nih.gov/pubmed/27294525\nPubMedCentralID: PMC4922696, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4922696"},"doiId":"10.1172/JCI81603","doiUrl":"https://dx.doi.org/10.1172/JCI81603","featured":true,"firstPublished":{"html":"Jul 1, 2016 00:00:00 AM","text":"Jul 1, 2016 00:00:00 AM","value":"2016-07-01T00:00:00.000-07:00"},"mlaCitation":"Weiskopf, Kipp et al. ?CD47-Blocking Immunotherapies Stimulate Macrophage-Mediated Destruction of Small-Cell Lung Cancer.? <i>JOURNAL OF CLINICAL INVESTIGATION</i> 126.7 (2016): 2610?2620. Print.","pubMedId":"27294525","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/27294525","publicationId":416393,"title":"CD47-blocking immunotherapies stimulate macrophage-mediated destruction of small-cell lung cancer","type":"article","webOfScienceId":"000379094800024","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000379094800024"},{"abstract":"Pancreatic neuroendocrine tumors (PanNETs) are a type of pancreatic cancer with limited therapeutic options. Consequently, most patients with advanced disease die from tumor progression. Current evidence indicates that a subset of cancer cells is responsible for tumor development, metastasis, and recurrence, and targeting these tumor-initiating cells is necessary to eradicate tumors. However, tumor-initiating cells and the biological processes that promote pathogenesis remain largely uncharacterized in PanNETs. Here we profile primary and metastatic tumors from an index patient and demonstrate that MET proto-oncogene activation is important for tumor growth in PanNET xenograft models. We identify a highly tumorigenic cell population within several independent surgically acquired PanNETs characterized by increased cell-surface protein CD90 expression and aldehyde dehydrogenase A1 (ALDHA1) activity, and provide in vitro and in vivo evidence for their stem-like properties. We performed proteomic profiling of 332 antigens in two cell lines and four primary tumors, and showed that CD47, a cell-surface protein that acts as a \"don't eat me\" signal co-opted by cancers to evade innate immune surveillance, is ubiquitously expressed. Moreover, CD47 coexpresses with MET and is enriched in CD90(hi)cells. Furthermore, blocking CD47 signaling promotes engulfment of tumor cells by macrophages in vitro and inhibits xenograft tumor growth, prevents metastases, and prolongs survival in vivo.","apaCitation":"Krampitz, G. W., George, B. M., Willingham, S. B., Volkmer, J.-P., Weiskopf, K., Jahchan, N., ? Weissman, I. L. (2016). Identification of tumorigenic cells and therapeutic targets in pancreatic neuroendocrine tumors. <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i>, <i>113</i>(16), 4464?4469.","capCitation":"<span class=\"title\">\n  <span>Identification of tumorigenic cells and therapeutic targets in pancreatic neuroendocrine tumors</span>\n  <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i>\n</span>\n<span class=\"authors\">Krampitz, G. W., George, B. M., Willingham, S. B., Volkmer, J., Weiskopf, K., Jahchan, N., Newman, A. M., Sahoo, D., Zemek, A. J., Yanovsky, R. L., Nguyen, J. K., Schnorr, P. J., Mazur, P. K., Sage, J., Longacre, T. A., Visser, B. C., Poultsides, G. A., Norton, J. A., Weissman, I. L.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">113 (16)</span><span class=\"pages\">: 4464-4469</span>\n</span>","chicagoCitation":"Krampitz, Geoffrey Wayne, Benson M. George, Stephen B. Willingham, Jens-Peter Volkmer, Kipp Weiskopf, et al. 2016. ?Identification of Tumorigenic Cells and Therapeutic Targets in Pancreatic Neuroendocrine Tumors.? <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i> 113 (16). NATL ACAD SCIENCES: 4464?69.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Pancreatic neuroendocrine tumors (PanNETs) are a type of pancreatic cancer with limited therapeutic options. Consequently, most patients with advanced disease die from tumor progression. Current evidence indicates that a subset of cancer cells is responsible for tumor development, metastasis, and recurrence, and targeting these tumor-initiating cells is necessary to eradicate tumors. However, tumor-initiating cells and the biological processes that promote pathogenesis remain largely uncharacterized in PanNETs. Here we profile primary and metastatic tumors from an index patient and demonstrate that MET proto-oncogene activation is important for tumor growth in PanNET xenograft models. We identify a highly tumorigenic cell population within several independent surgically acquired PanNETs characterized by increased cell-surface protein CD90 expression and aldehyde dehydrogenase A1 (ALDHA1) activity, and provide in vitro and in vivo evidence for their stem-like properties. We performed proteomic profiling of 332 antigens in two cell lines and four primary tumors, and showed that CD47, a cell-surface protein that acts as a \"don't eat me\" signal co-opted by cancers to evade innate immune surveillance, is ubiquitously expressed. Moreover, CD47 coexpresses with MET and is enriched in CD90(hi)cells. Furthermore, blocking CD47 signaling promotes engulfment of tumor cells by macrophages in vitro and inhibits xenograft tumor growth, prevents metastases, and prolongs survival in vivo.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1073/pnas.1600007113\">DOI 10.1073/pnas.1600007113</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000374393800063\">Web of Science ID 000374393800063</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/27035983\">PubMedID 27035983</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4843455\">PubMedCentralID PMC4843455</a></span>\n  </p>","text":"Abstract:\nPancreatic neuroendocrine tumors (PanNETs) are a type of pancreatic cancer with limited therapeutic options. Consequently, most patients with advanced disease die from tumor progression. Current evidence indicates that a subset of cancer cells is responsible for tumor development, metastasis, and recurrence, and targeting these tumor-initiating cells is necessary to eradicate tumors. However, tumor-initiating cells and the biological processes that promote pathogenesis remain largely uncharacterized in PanNETs. Here we profile primary and metastatic tumors from an index patient and demonstrate that MET proto-oncogene activation is important for tumor growth in PanNET xenograft models. We identify a highly tumorigenic cell population within several independent surgically acquired PanNETs characterized by increased cell-surface protein CD90 expression and aldehyde dehydrogenase A1 (ALDHA1) activity, and provide in vitro and in vivo evidence for their stem-like properties. We performed proteomic profiling of 332 antigens in two cell lines and four primary tumors, and showed that CD47, a cell-surface protein that acts as a \"don't eat me\" signal co-opted by cancers to evade innate immune surveillance, is ubiquitously expressed. Moreover, CD47 coexpresses with MET and is enriched in CD90(hi)cells. Furthermore, blocking CD47 signaling promotes engulfment of tumor cells by macrophages in vitro and inhibits xenograft tumor growth, prevents metastases, and prolongs survival in vivo.\nDOI: 10.1073/pnas.1600007113, https://dx.doi.org/10.1073/pnas.1600007113\nWeb of Science ID: 000374393800063, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000374393800063\nPubMedID: 27035983, https://www.ncbi.nlm.nih.gov/pubmed/27035983\nPubMedCentralID: PMC4843455, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4843455"},"doiId":"10.1073/pnas.1600007113","doiUrl":"https://dx.doi.org/10.1073/pnas.1600007113","featured":false,"firstPublished":{"html":"Apr 19, 2016 00:00:00 AM","text":"Apr 19, 2016 00:00:00 AM","value":"2016-04-19T00:00:00.000-07:00"},"mlaCitation":"Krampitz, Geoffrey Wayne et al. ?Identification of Tumorigenic Cells and Therapeutic Targets in Pancreatic Neuroendocrine Tumors.? <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i> 113.16 (2016): 4464?4469. Print.","pubMedId":"27035983","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/27035983","publicationId":403318,"title":"Identification of tumorigenic cells and therapeutic targets in pancreatic neuroendocrine tumors","type":"article","webOfScienceId":"000374393800063","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000374393800063"},{"apaCitation":"Reynoird, N., Mazur, P. K., Stellfeld, T., Flores, N. M., Lofgren, S. M., Carlson, S. M., ? Sage, J. (2016). Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer. <i>GENES &amp; DEVELOPMENT</i>, <i>30</i>(7), 772?785.","capCitation":"<span class=\"title\">\n  <span>Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer</span>\n  <i>GENES &amp; DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Reynoird, N., Mazur, P. K., Stellfeld, T., Flores, N. M., Lofgren, S. M., Carlson, S. M., Brambilla, E., Hainaut, P., Kaznowska, E. B., Arrowsmith, C. H., Khatri, P., Stresemann, C., Gozani, O., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">30 (7)</span><span class=\"pages\">: 772-785</span>\n</span>","chicagoCitation":"Reynoird, Nicolas, Pawel K. Mazur, Timo Stellfeld, Natasha M. Flores, Shane M. Lofgren, et al. 2016. ?Coordination of Stress Signals by the Lysine Methyltransferase SMYD2 Promotes Pancreatic Cancer.? <i>GENES &amp; DEVELOPMENT</i> 30 (7). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 772?85.","detail":{"html":"<p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.275529.115\">DOI 10.1101/gad.275529.115</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000373194300003\">Web of Science ID 000373194300003</a></span>\n  </p>","text":"DOI: 10.1101/gad.275529.115, https://dx.doi.org/10.1101/gad.275529.115\nWeb of Science ID: 000373194300003, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000373194300003"},"doiId":"10.1101/gad.275529.115","doiUrl":"https://dx.doi.org/10.1101/gad.275529.115","featured":true,"firstPublished":{"html":"Apr 1, 2016 00:00:00 AM","text":"Apr 1, 2016 00:00:00 AM","value":"2016-04-01T00:00:00.000-07:00"},"mlaCitation":"Reynoird, Nicolas et al. ?Coordination of Stress Signals by the Lysine Methyltransferase SMYD2 Promotes Pancreatic Cancer.? <i>GENES &amp; DEVELOPMENT</i> 30.7 (2016): 772?785. Print.","publicationId":405438,"title":"Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer","type":"article","webOfScienceId":"000373194300003","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000373194300003"},{"abstract":"Pancreatic ductal adenocarcinoma (PDAC) is a lethal form of cancer with few therapeutic options. We found that levels of the lysine methyltransferase SMYD2 (SET and MYND domain 2) are elevated in PDAC and that genetic and pharmacological inhibition of SMYD2 restricts PDAC growth. We further identified the stress response kinase MAPKAPK3 (MK3) as a new physiologic substrate of SMYD2 in PDAC cells. Inhibition of MAPKAPK3 impedes PDAC growth, identifying a potential new kinase target in PDAC. Finally, we show that inhibition of SMYD2 cooperates with standard chemotherapy to treat PDAC cells and tumors. These findings uncover a pivotal role for SMYD2 in promoting pancreatic cancer.","apaCitation":"Reynoird, N., Mazur, P. K., Stellfeld, T., Flores, N. M., Lofgren, S. M., Carlson, S. M., ? Sage, J. (2016). Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer. <i>Genes &amp; Development</i>, <i>30</i>(7), 772?785.","capCitation":"<span class=\"title\">\n  <span>Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer.</span>\n  <i>Genes &amp; development</i>\n</span>\n<span class=\"authors\">Reynoird, N., Mazur, P. K., Stellfeld, T., Flores, N. M., Lofgren, S. M., Carlson, S. M., Brambilla, E., Hainaut, P., Kaznowska, E. B., Arrowsmith, C. H., Khatri, P., Stresemann, C., Gozani, O., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">30 (7)</span><span class=\"pages\">: 772-785</span>\n</span>","chicagoCitation":"Reynoird, Nicolas, Pawel K. Mazur, Timo Stellfeld, Natasha M. Flores, Shane M. Lofgren, et al. 2016. ?Coordination of Stress Signals by the Lysine Methyltransferase SMYD2 Promotes Pancreatic Cancer.? <i>Genes &amp; Development</i> 30 (7). COLD SPRING HARBOR LABORATORY PRESS: 772?85.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Pancreatic ductal adenocarcinoma (PDAC) is a lethal form of cancer with few therapeutic options. We found that levels of the lysine methyltransferase SMYD2 (SET and MYND domain 2) are elevated in PDAC and that genetic and pharmacological inhibition of SMYD2 restricts PDAC growth. We further identified the stress response kinase MAPKAPK3 (MK3) as a new physiologic substrate of SMYD2 in PDAC cells. Inhibition of MAPKAPK3 impedes PDAC growth, identifying a potential new kinase target in PDAC. Finally, we show that inhibition of SMYD2 cooperates with standard chemotherapy to treat PDAC cells and tumors. These findings uncover a pivotal role for SMYD2 in promoting pancreatic cancer.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.275529.115\">DOI 10.1101/gad.275529.115</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/26988419\">PubMedID 26988419</a></span>\n  </p>","text":"Abstract:\nPancreatic ductal adenocarcinoma (PDAC) is a lethal form of cancer with few therapeutic options. We found that levels of the lysine methyltransferase SMYD2 (SET and MYND domain 2) are elevated in PDAC and that genetic and pharmacological inhibition of SMYD2 restricts PDAC growth. We further identified the stress response kinase MAPKAPK3 (MK3) as a new physiologic substrate of SMYD2 in PDAC cells. Inhibition of MAPKAPK3 impedes PDAC growth, identifying a potential new kinase target in PDAC. Finally, we show that inhibition of SMYD2 cooperates with standard chemotherapy to treat PDAC cells and tumors. These findings uncover a pivotal role for SMYD2 in promoting pancreatic cancer.\nDOI: 10.1101/gad.275529.115, https://dx.doi.org/10.1101/gad.275529.115\n\nPubMedID: 26988419, https://www.ncbi.nlm.nih.gov/pubmed/26988419"},"doiId":"10.1101/gad.275529.115","doiUrl":"https://dx.doi.org/10.1101/gad.275529.115","featured":true,"firstPublished":{"html":"Apr 1, 2016 00:00:00 AM","text":"Apr 1, 2016 00:00:00 AM","value":"2016-04-01T00:00:00.000-07:00"},"mlaCitation":"Reynoird, Nicolas et al. ?Coordination of Stress Signals by the Lysine Methyltransferase SMYD2 Promotes Pancreatic Cancer.? <i>Genes &amp; development</i> 30.7 (2016): 772?785. Print.","pubMedId":"26988419","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/26988419","publicationId":405926,"title":"Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer.","type":"article"},{"abstract":"The control of cell division is essential for normal development and the maintenance of cellular homeostasis. Abnormal cell proliferation is associated with multiple pathological states, including cancer. Although the Hippo/YAP signaling pathway was initially thought to control organ size and growth, increasing evidence indicates that this pathway also plays a major role in the control of proliferation independent of organ size control. In particular, accumulating evidence indicates that the Hippo/YAP signaling pathway functionally interacts with multiple other cellular pathways and serves as a central node in the regulation of cell division, especially in cancer cells. Here, recent observations are highlighted that connect Hippo/YAP signaling to transcription, the basic cell-cycle machinery, and the control of cell division. Furthermore, the oncogenic and tumor-suppressive attributes of YAP/TAZ are reviewed, which emphasizes the relevance of the Hippo pathway in cancer. Mol Cancer Res; 14(2); 127-40. Â©2015 AACR.","apaCitation":"Ehmer, U., &amp; Sage, J. (2016). Control of Proliferation and Cancer Growth by the Hippo Signaling Pathway. <i>Molecular Cancer Research </i>, <i>14</i>(2), 127?140.","capCitation":"<span class=\"title\">\n  <span>Control of Proliferation and Cancer Growth by the Hippo Signaling Pathway.</span>\n  <i>Molecular cancer research </i>\n</span>\n<span class=\"authors\">Ehmer, U., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">14 (2)</span><span class=\"pages\">: 127-140</span>\n</span>","chicagoCitation":"Ehmer, Ursula, and Julien Sage. 2016. ?Control of Proliferation and Cancer Growth by the Hippo Signaling Pathway.? <i>Molecular Cancer Research </i>14 (2). AMER ASSN FOR CANCER RESEARCH: 127?40.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The control of cell division is essential for normal development and the maintenance of cellular homeostasis. Abnormal cell proliferation is associated with multiple pathological states, including cancer. Although the Hippo/YAP signaling pathway was initially thought to control organ size and growth, increasing evidence indicates that this pathway also plays a major role in the control of proliferation independent of organ size control. In particular, accumulating evidence indicates that the Hippo/YAP signaling pathway functionally interacts with multiple other cellular pathways and serves as a central node in the regulation of cell division, especially in cancer cells. Here, recent observations are highlighted that connect Hippo/YAP signaling to transcription, the basic cell-cycle machinery, and the control of cell division. Furthermore, the oncogenic and tumor-suppressive attributes of YAP/TAZ are reviewed, which emphasizes the relevance of the Hippo pathway in cancer. Mol Cancer Res; 14(2); 127-40. Â©2015 AACR.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1158/1541-7786.MCR-15-0305\">DOI 10.1158/1541-7786.MCR-15-0305</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/26432795\">PubMedID 26432795</a></span>\n  </p>","text":"Abstract:\nThe control of cell division is essential for normal development and the maintenance of cellular homeostasis. Abnormal cell proliferation is associated with multiple pathological states, including cancer. Although the Hippo/YAP signaling pathway was initially thought to control organ size and growth, increasing evidence indicates that this pathway also plays a major role in the control of proliferation independent of organ size control. In particular, accumulating evidence indicates that the Hippo/YAP signaling pathway functionally interacts with multiple other cellular pathways and serves as a central node in the regulation of cell division, especially in cancer cells. Here, recent observations are highlighted that connect Hippo/YAP signaling to transcription, the basic cell-cycle machinery, and the control of cell division. Furthermore, the oncogenic and tumor-suppressive attributes of YAP/TAZ are reviewed, which emphasizes the relevance of the Hippo pathway in cancer. Mol Cancer Res; 14(2); 127-40. Â©2015 AACR.\nDOI: 10.1158/1541-7786.MCR-15-0305, https://dx.doi.org/10.1158/1541-7786.MCR-15-0305\n\nPubMedID: 26432795, https://www.ncbi.nlm.nih.gov/pubmed/26432795"},"doiId":"10.1158/1541-7786.MCR-15-0305","doiUrl":"https://dx.doi.org/10.1158/1541-7786.MCR-15-0305","featured":false,"firstPublished":{"html":"Feb 1, 2016 00:00:00 AM","text":"Feb 1, 2016 00:00:00 AM","value":"2016-02-01T00:00:00.000-08:00"},"mlaCitation":"Ehmer, Ursula, and Julien Sage. ?Control of Proliferation and Cancer Growth by the Hippo Signaling Pathway.? <i>Molecular cancer research </i>14.2 (2016): 127?140. Print.","pubMedId":"26432795","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/26432795","publicationId":407717,"title":"Control of Proliferation and Cancer Growth by the Hippo Signaling Pathway.","type":"article"},{"abstract":"The thymus is the site of T cell development and selection. In addition to lymphocytes, the thymus is composed of several types of stromal cells that are exquisitely organized to create the appropriate environment and microenvironment to support the development and selection of maturing T cells. Thymic epithelial cells (TECs) are one of the more important cell types in the thymic stroma, and they play a critical role in selecting functional T cell clones and supporting their development. In this study, we used a mouse genetics approach to investigate the consequences of deleting the Pten tumor suppressor gene in the TEC compartment of the developing thymus. We found that PTEN deficiency in TECs results in a smaller thymus with significantly disordered architecture and histology. Accordingly, loss of PTEN function also results in decreased T cells with a shift in the distribution of T cell subtypes towards CD8+ T cells. These experiments demonstrate that PTEN is critically required for the development of a functional thymic epithelium in mice. This work may help better understand the effects that certain medical conditions or clinical interventions have upon the thymus and immune function.","apaCitation":"Garfin, P. M., Nguyen, T., &amp; Sage, J. (2016). Loss of Pten Disrupts the Thymic Epithelium and Alters Thymic Function. <i>PloS One</i>, <i>11</i>(2).","capCitation":"<span class=\"title\">\n  <span>Loss of Pten Disrupts the Thymic Epithelium and Alters Thymic Function.</span>\n  <i>PloS one</i>\n</span>\n<span class=\"authors\">Garfin, P. M., Nguyen, T., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">11 (2)</span>\n</span>","chicagoCitation":"Garfin, Phillip M., Thuyen Nguyen, and Julien Sage. 2016. ?Loss of Pten Disrupts the Thymic Epithelium and Alters Thymic Function.? <i>PloS One</i> 11 (2).","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The thymus is the site of T cell development and selection. In addition to lymphocytes, the thymus is composed of several types of stromal cells that are exquisitely organized to create the appropriate environment and microenvironment to support the development and selection of maturing T cells. Thymic epithelial cells (TECs) are one of the more important cell types in the thymic stroma, and they play a critical role in selecting functional T cell clones and supporting their development. In this study, we used a mouse genetics approach to investigate the consequences of deleting the Pten tumor suppressor gene in the TEC compartment of the developing thymus. We found that PTEN deficiency in TECs results in a smaller thymus with significantly disordered architecture and histology. Accordingly, loss of PTEN function also results in decreased T cells with a shift in the distribution of T cell subtypes towards CD8+ T cells. These experiments demonstrate that PTEN is critically required for the development of a functional thymic epithelium in mice. This work may help better understand the effects that certain medical conditions or clinical interventions have upon the thymus and immune function.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1371/journal.pone.0149430\">DOI 10.1371/journal.pone.0149430</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/26914657\">PubMedID 26914657</a></span>\n  </p>","text":"Abstract:\nThe thymus is the site of T cell development and selection. In addition to lymphocytes, the thymus is composed of several types of stromal cells that are exquisitely organized to create the appropriate environment and microenvironment to support the development and selection of maturing T cells. Thymic epithelial cells (TECs) are one of the more important cell types in the thymic stroma, and they play a critical role in selecting functional T cell clones and supporting their development. In this study, we used a mouse genetics approach to investigate the consequences of deleting the Pten tumor suppressor gene in the TEC compartment of the developing thymus. We found that PTEN deficiency in TECs results in a smaller thymus with significantly disordered architecture and histology. Accordingly, loss of PTEN function also results in decreased T cells with a shift in the distribution of T cell subtypes towards CD8+ T cells. These experiments demonstrate that PTEN is critically required for the development of a functional thymic epithelium in mice. This work may help better understand the effects that certain medical conditions or clinical interventions have upon the thymus and immune function.\nDOI: 10.1371/journal.pone.0149430, https://dx.doi.org/10.1371/journal.pone.0149430\n\nPubMedID: 26914657, https://www.ncbi.nlm.nih.gov/pubmed/26914657"},"doiId":"10.1371/journal.pone.0149430","doiUrl":"https://dx.doi.org/10.1371/journal.pone.0149430","featured":false,"firstPublished":{"html":"Jan 1, 2016 00:00:00 AM","text":"Jan 1, 2016 00:00:00 AM","value":"2016-01-01T00:00:00.000-08:00"},"mlaCitation":"Garfin, Phillip M., Thuyen Nguyen, and Julien Sage. ?Loss of Pten Disrupts the Thymic Epithelium and Alters Thymic Function.? <i>PloS one</i> 11.2 (2016): n. pag. Print.","pubMedId":"26914657","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/26914657","publicationId":407718,"title":"Loss of Pten Disrupts the Thymic Epithelium and Alters Thymic Function.","type":"article"},{"abstract":"Pluripotent stem cells, defined by an unlimited self-renewal capacity and an undifferentiated state, are best typified by embryonic stem cells. These cells have a unique cell cycle compared to somatic cells as defined by a rapid progression through the cell cycle and a minimal time spent in G1. Recent reports indicate that pluripotency and cell cycle regulation are mechanistically linked. In this review, we discuss the reciprocal co-regulation of these processes, how this co-regulation may prevent differentiation, and how cellular reprogramming can re-establish the unique cell cycle regulation in induced pluripotent stem cells.","apaCitation":"Kareta, M. S., Sage, J., &amp; Wernig, M. (2015). Crosstalk between stem cell and cell cycle machineries. <i>CURRENT OPINION IN CELL BIOLOGY</i>, <i>37</i>, 68?74.","capCitation":"<span class=\"title\">\n  <span>Crosstalk between stem cell and cell cycle machineries</span>\n  <i>CURRENT OPINION IN CELL BIOLOGY</i>\n</span>\n<span class=\"authors\">Kareta, M. S., Sage, J., Wernig, M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2015</span>; <span class=\"volume\">37</span><span class=\"pages\">: 68-74</span>\n</span>","chicagoCitation":"Kareta, Michael S., Julien Sage, and Marius Wernig. 2015. ?Crosstalk between Stem Cell and Cell Cycle Machineries.? <i>CURRENT OPINION IN CELL BIOLOGY</i> 37. CURRENT BIOLOGY LTD: 68?74.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Pluripotent stem cells, defined by an unlimited self-renewal capacity and an undifferentiated state, are best typified by embryonic stem cells. These cells have a unique cell cycle compared to somatic cells as defined by a rapid progression through the cell cycle and a minimal time spent in G1. Recent reports indicate that pluripotency and cell cycle regulation are mechanistically linked. In this review, we discuss the reciprocal co-regulation of these processes, how this co-regulation may prevent differentiation, and how cellular reprogramming can re-establish the unique cell cycle regulation in induced pluripotent stem cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.ceb.2015.10.001\">DOI 10.1016/j.ceb.2015.10.001</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000367112800010\">Web of Science ID 000367112800010</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/26520682\">PubMedID 26520682</a></span>\n  </p>","text":"Abstract:\nPluripotent stem cells, defined by an unlimited self-renewal capacity and an undifferentiated state, are best typified by embryonic stem cells. These cells have a unique cell cycle compared to somatic cells as defined by a rapid progression through the cell cycle and a minimal time spent in G1. Recent reports indicate that pluripotency and cell cycle regulation are mechanistically linked. In this review, we discuss the reciprocal co-regulation of these processes, how this co-regulation may prevent differentiation, and how cellular reprogramming can re-establish the unique cell cycle regulation in induced pluripotent stem cells.\nDOI: 10.1016/j.ceb.2015.10.001, https://dx.doi.org/10.1016/j.ceb.2015.10.001\nWeb of Science ID: 000367112800010, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000367112800010\nPubMedID: 26520682, https://www.ncbi.nlm.nih.gov/pubmed/26520682"},"doiId":"10.1016/j.ceb.2015.10.001","doiUrl":"https://dx.doi.org/10.1016/j.ceb.2015.10.001","featured":false,"firstPublished":{"html":"Dec 1, 2015 00:00:00 AM","text":"Dec 1, 2015 00:00:00 AM","value":"2015-12-01T00:00:00.000-08:00"},"mlaCitation":"Kareta, Michael S., Julien Sage, and Marius Wernig. ?Crosstalk between Stem Cell and Cell Cycle Machineries.? <i>CURRENT OPINION IN CELL BIOLOGY</i> 37 (2015): 68?74. Print.","pubMedId":"26520682","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/26520682","publicationId":394915,"title":"Crosstalk between stem cell and cell cycle machineries","type":"article","webOfScienceId":"000367112800010","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000367112800010"},{"abstract":"Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal human cancers and shows resistance to any therapeutic strategy used. Here we tested small-molecule inhibitors targeting chromatin regulators as possible therapeutic agents in PDAC. We show that JQ1, an inhibitor of the bromodomain and extraterminal (BET) family of proteins, suppresses PDAC development in mice by inhibiting both MYC activity and inflammatory signals. The histone deacetylase (HDAC) inhibitor SAHA synergizes with JQ1 to augment cell death and more potently suppress advanced PDAC. Finally, using a CRISPR-Cas9-based method for gene editing directly in the mouse adult pancreas, we show that de-repression of p57 (also known as KIP2 or CDKN1C) upon combined BET and HDAC inhibition is required for the induction of combination therapy-induced cell death in PDAC. SAHA is approved for human use, and molecules similar to JQ1 are being tested in clinical trials. Thus, these studies identify a promising epigenetic-based therapeutic strategy that may be rapidly implemented in fatal human tumors.","apaCitation":"Mazur, P. K., Herner, A., Mello, S. S., Wirth, M., Hausmann, S., SÃ¡nchez-Rivera, F. J., ? Siveke, J. T. (2015). Combined inhibition of BET family proteins and histone deacetylases as a potential epigenetics-based therapy for pancreatic ductal adenocarcinoma. <i>Nature Medicine</i>, <i>21</i>(10), 1163?1171.","capCitation":"<span class=\"title\">\n  <span>Combined inhibition of BET family proteins and histone deacetylases as a potential epigenetics-based therapy for pancreatic ductal adenocarcinoma.</span>\n  <i>Nature medicine</i>\n</span>\n<span class=\"authors\">Mazur, P. K., Herner, A., Mello, S. S., Wirth, M., Hausmann, S., SÃ¡nchez-Rivera, F. J., Lofgren, S. M., Kuschma, T., Hahn, S. A., Vangala, D., Trajkovic-Arsic, M., Gupta, A., Heid, I., NoÃ«l, P. B., Braren, R., Erkan, M., Kleeff, J., Sipos, B., Sayles, L. C., Heikenwalder, M., HeÃŸmann, E., Ellenrieder, V., Esposito, I., Jacks, T., Bradner, J. E., Khatri, P., Sweet-Cordero, E. A., Attardi, L. D., Schmid, R. M., Schneider, G., Sage, J., Siveke, J. T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2015</span>; <span class=\"volume\">21 (10)</span><span class=\"pages\">: 1163-1171</span>\n</span>","chicagoCitation":"Mazur, Pawel K., Alexander Herner, Stephano S. Mello, Matthias Wirth, Simone Hausmann, et al. 2015. ?Combined Inhibition of BET Family Proteins and Histone Deacetylases as a Potential Epigenetics-Based Therapy for Pancreatic Ductal Adenocarcinoma.? <i>Nature Medicine</i> 21 (10): 1163?71.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal human cancers and shows resistance to any therapeutic strategy used. Here we tested small-molecule inhibitors targeting chromatin regulators as possible therapeutic agents in PDAC. We show that JQ1, an inhibitor of the bromodomain and extraterminal (BET) family of proteins, suppresses PDAC development in mice by inhibiting both MYC activity and inflammatory signals. The histone deacetylase (HDAC) inhibitor SAHA synergizes with JQ1 to augment cell death and more potently suppress advanced PDAC. Finally, using a CRISPR-Cas9-based method for gene editing directly in the mouse adult pancreas, we show that de-repression of p57 (also known as KIP2 or CDKN1C) upon combined BET and HDAC inhibition is required for the induction of combination therapy-induced cell death in PDAC. SAHA is approved for human use, and molecules similar to JQ1 are being tested in clinical trials. Thus, these studies identify a promising epigenetic-based therapeutic strategy that may be rapidly implemented in fatal human tumors.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nm.3952\">DOI 10.1038/nm.3952</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/26390243\">PubMedID 26390243</a></span>\n  </p>","text":"Abstract:\nPancreatic ductal adenocarcinoma (PDAC) is one of the most lethal human cancers and shows resistance to any therapeutic strategy used. Here we tested small-molecule inhibitors targeting chromatin regulators as possible therapeutic agents in PDAC. We show that JQ1, an inhibitor of the bromodomain and extraterminal (BET) family of proteins, suppresses PDAC development in mice by inhibiting both MYC activity and inflammatory signals. The histone deacetylase (HDAC) inhibitor SAHA synergizes with JQ1 to augment cell death and more potently suppress advanced PDAC. Finally, using a CRISPR-Cas9-based method for gene editing directly in the mouse adult pancreas, we show that de-repression of p57 (also known as KIP2 or CDKN1C) upon combined BET and HDAC inhibition is required for the induction of combination therapy-induced cell death in PDAC. SAHA is approved for human use, and molecules similar to JQ1 are being tested in clinical trials. Thus, these studies identify a promising epigenetic-based therapeutic strategy that may be rapidly implemented in fatal human tumors.\nDOI: 10.1038/nm.3952, https://dx.doi.org/10.1038/nm.3952\n\nPubMedID: 26390243, https://www.ncbi.nlm.nih.gov/pubmed/26390243"},"doiId":"10.1038/nm.3952","doiUrl":"https://dx.doi.org/10.1038/nm.3952","featured":true,"firstPublished":{"html":"Oct 1, 2015 00:00:00 AM","text":"Oct 1, 2015 00:00:00 AM","value":"2015-10-01T00:00:00.000-07:00"},"mlaCitation":"Mazur, Pawel K. et al. ?Combined Inhibition of BET Family Proteins and Histone Deacetylases as a Potential Epigenetics-Based Therapy for Pancreatic Ductal Adenocarcinoma.? <i>Nature medicine</i> 21.10 (2015): 1163?1171. Print.","pubMedId":"26390243","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/26390243","publicationId":398340,"title":"Combined inhibition of BET family proteins and histone deacetylases as a potential epigenetics-based therapy for pancreatic ductal adenocarcinoma.","type":"article"},{"abstract":"We have sequenced the genomes of 110 small cell lung cancers (SCLC), one of the deadliest human cancers. In nearly all the tumours analysed we found bi-allelic inactivation of TP53 and RB1, sometimes by complex genomic rearrangements. Two tumours with wild-type RB1 had evidence of chromothripsis leading to overexpression of cyclin D1 (encoded by the CCND1 gene), revealing an alternative mechanism of Rb1 deregulation. Thus, loss of the tumour suppressors TP53 and RB1 is obligatory in SCLC. We discovered somatic genomic rearrangements of TP73 that create an oncogenic version of this gene, TP73?ex2/3. In rare cases, SCLC tumours exhibited kinase gene mutations, providing a possible therapeutic opportunity for individual patients. Finally, we observed inactivating mutations in NOTCH family genes in 25% of human SCLC. Accordingly, activation of Notch signalling in a pre-clinical SCLC mouse model strikingly reduced the number of tumours and extended the survival of the mutant mice. Furthermore, neuroendocrine gene expression was abrogated by Notch activity in SCLC cells. This first comprehensive study of somatic genome alterations in SCLC uncovers several key biological processes and identifies candidate therapeutic targets in this highly lethal form of cancer.","apaCitation":"George, J., Lim, J. S., Jang, S. J., Cun, Y., Ozretic, L., Kong, G., ? Thomas, R. K. (2015). Comprehensive genomic profiles of small cell lung cancer. <i>Nature</i>, <i>524</i>(7563), 47?53.","capCitation":"<span class=\"title\">\n  <span>Comprehensive genomic profiles of small cell lung cancer.</span>\n  <i>Nature</i>\n</span>\n<span class=\"authors\">George, J., Lim, J. S., Jang, S. J., Cun, Y., Ozretic, L., Kong, G., Leenders, F., Lu, X., FernÃ¡ndez-Cuesta, L., Bosco, G., MÃ¼ller, C., Dahmen, I., Jahchan, N. S., Park, K., Yang, D., Karnezis, A. N., Vaka, D., Torres, A., Wang, M. S., Korbel, J. O., Menon, R., Chun, S., Kim, D., Wilkerson, M., Hayes, N., Engelmann, D., PÃ¼tzer, B., Bos, M., Michels, S., Vlasic, I., Seidel, D., Pinther, B., Schaub, P., Becker, C., AltmÃ¼ller, J., Yokota, J., Kohno, T., Iwakawa, R., Tsuta, K., Noguchi, M., Muley, T., Hoffmann, H., Schnabel, P. A., Petersen, I., Chen, Y., Soltermann, A., Tischler, V., Choi, C., Kim, Y., Massion, P. P., Zou, Y., Jovanovic, D., Kontic, M., Wright, G. M., Russell, P. A., Solomon, B., Koch, I., Lindner, M., Muscarella, L. A., la Torre, A., Field, J. K., Jakopovic, M., Knezevic, J., CastaÃ±os-VÃ©lez, E., Roz, L., Pastorino, U., Brustugun, O., Lund-Iversen, M., Thunnissen, E., KÃ¶hler, J., Schuler, M., Botling, J., Sandelin, M., Sanchez-Cespedes, M., Salvesen, H. B., Achter, V., Lang, U., Bogus, M., Schneider, P. M., Zander, T., AnsÃ©n, S., Hallek, M., Wolf, J., Vingron, M., Yatabe, Y., Travis, W. D., NÃ¼rnberg, P., Reinhardt, C., Perner, S., Heukamp, L., BÃ¼ttner, R., Haas, S. A., Brambilla, E., Peifer, M., Sage, J., Thomas, R. K.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2015</span>; <span class=\"volume\">524 (7563)</span><span class=\"pages\">: 47-53</span>\n</span>","chicagoCitation":"George, Julie, Jing Shan Lim, Se Jin Jang, Yupeng Cun, Luka Ozretic, et al. 2015. ?Comprehensive Genomic Profiles of Small Cell Lung Cancer.? <i>Nature</i> 524 (7563): 47?53.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">We have sequenced the genomes of 110 small cell lung cancers (SCLC), one of the deadliest human cancers. In nearly all the tumours analysed we found bi-allelic inactivation of TP53 and RB1, sometimes by complex genomic rearrangements. Two tumours with wild-type RB1 had evidence of chromothripsis leading to overexpression of cyclin D1 (encoded by the CCND1 gene), revealing an alternative mechanism of Rb1 deregulation. Thus, loss of the tumour suppressors TP53 and RB1 is obligatory in SCLC. We discovered somatic genomic rearrangements of TP73 that create an oncogenic version of this gene, TP73?ex2/3. In rare cases, SCLC tumours exhibited kinase gene mutations, providing a possible therapeutic opportunity for individual patients. Finally, we observed inactivating mutations in NOTCH family genes in 25% of human SCLC. Accordingly, activation of Notch signalling in a pre-clinical SCLC mouse model strikingly reduced the number of tumours and extended the survival of the mutant mice. Furthermore, neuroendocrine gene expression was abrogated by Notch activity in SCLC cells. This first comprehensive study of somatic genome alterations in SCLC uncovers several key biological processes and identifies candidate therapeutic targets in this highly lethal form of cancer.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nature14664\">DOI 10.1038/nature14664</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/26168399\">PubMedID 26168399</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4861069\">PubMedCentralID PMC4861069</a></span>\n  </p>","text":"Abstract:\nWe have sequenced the genomes of 110 small cell lung cancers (SCLC), one of the deadliest human cancers. In nearly all the tumours analysed we found bi-allelic inactivation of TP53 and RB1, sometimes by complex genomic rearrangements. Two tumours with wild-type RB1 had evidence of chromothripsis leading to overexpression of cyclin D1 (encoded by the CCND1 gene), revealing an alternative mechanism of Rb1 deregulation. Thus, loss of the tumour suppressors TP53 and RB1 is obligatory in SCLC. We discovered somatic genomic rearrangements of TP73 that create an oncogenic version of this gene, TP73?ex2/3. In rare cases, SCLC tumours exhibited kinase gene mutations, providing a possible therapeutic opportunity for individual patients. Finally, we observed inactivating mutations in NOTCH family genes in 25% of human SCLC. Accordingly, activation of Notch signalling in a pre-clinical SCLC mouse model strikingly reduced the number of tumours and extended the survival of the mutant mice. Furthermore, neuroendocrine gene expression was abrogated by Notch activity in SCLC cells. This first comprehensive study of somatic genome alterations in SCLC uncovers several key biological processes and identifies candidate therapeutic targets in this highly lethal form of cancer.\nDOI: 10.1038/nature14664, https://dx.doi.org/10.1038/nature14664\n\nPubMedID: 26168399, https://www.ncbi.nlm.nih.gov/pubmed/26168399\nPubMedCentralID: PMC4861069, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4861069"},"doiId":"10.1038/nature14664","doiUrl":"https://dx.doi.org/10.1038/nature14664","featured":true,"firstPublished":{"html":"Aug 6, 2015 00:00:00 AM","text":"Aug 6, 2015 00:00:00 AM","value":"2015-08-06T00:00:00.000-07:00"},"mlaCitation":"George, Julie et al. ?Comprehensive Genomic Profiles of Small Cell Lung Cancer.? <i>Nature</i> 524.7563 (2015): 47?53. Print.","pubMedId":"26168399","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/26168399","publicationId":391110,"title":"Comprehensive genomic profiles of small cell lung cancer.","type":"article"},{"apaCitation":"George, J., Lim, J. S., Jang, S. J., Cun, Y., Ozretic, L., Kong, G., ? Thomas, R. K. (2015). Comprehensive genomic profiles of small cell lung cancer. <i>NATURE</i>, <i>524</i>(7563), 47?U73.","capCitation":"<span class=\"title\">\n  <span>Comprehensive genomic profiles of small cell lung cancer</span>\n  <i>NATURE</i>\n</span>\n<span class=\"authors\">George, J., Lim, J. S., Jang, S. J., Cun, Y., Ozretic, L., Kong, G., Leenders, F., Lu, X., Fernandez-Cuesta, L., Bosco, G., Mueller, C., Dahmen, I., Jahchan, N. S., Park, K., Yang, D., Karnezis, A. N., Vaka, D., Torres, A., Wang, M. S., Korbel, J. O., Menon, R., Chun, S., Kim, D., Wilkerson, M., Hayes, N., Engelmann, D., Puetzer, B., Bos, M., Michels, S., Vlasic, I., Seidel, D., Pinther, B., Schaub, P., Becker, C., Altmueller, J., Yokota, J., Kohno, T., Iwakawa, R., Tsuta, K., Noguchi, M., Muley, T., Hoffmann, H., Schnabel, P. A., Petersen, I., Chen, Y., Soltermann, A., Tischler, V., Choi, C., Kim, Y., Massion, P. P., Zou, Y., Jovanovic, D., Kontic, M., Wright, G. M., Russell, P. A., Solomon, B., Koch, I., Lindner, M., Muscarella, L. A., la Torre, A., Field, J. K., Jakopovic, M., Knezevic, J., Castanos-Velez, E., Roz, L., Pastorino, U., Brustugun, O., Lund-Iversen, M., Thunnissen, E., Koehler, J., Schuler, M., Botling, J., Sandelin, M., Sanchez-Cespedes, M., Salvesen, H. B., Achter, V., Lang, U., Bogus, M., Schneider, P. M., Zander, T., Ansen, S., Hallek, M., Wolf, J., Vingron, M., Yatabe, Y., Travis, W. D., Nuernberg, P., Reinhardt, C., Perner, S., Heukamp, L., Buettner, R., Haas, S. A., Brambilla, E., Peifer, M., Sage, J., Thomas, R. K.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2015</span>; <span class=\"volume\">524 (7563)</span><span class=\"pages\">: 47-U73</span>\n</span>","chicagoCitation":"George, Julie, Jing Shan Lim, Se Jin Jang, Yupeng Cun, Luka Ozretic, et al. 2015. ?Comprehensive Genomic Profiles of Small Cell Lung Cancer.? <i>NATURE</i> 524 (7563). NATURE PUBLISHING GROUP: 47?U73.","detail":{"html":"<p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nature14664\">DOI 10.1038/nature14664</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000359002300029\">Web of Science ID 000359002300029</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/26168399\">PubMedID 26168399</a></span>\n  </p>","text":"DOI: 10.1038/nature14664, https://dx.doi.org/10.1038/nature14664\nWeb of Science ID: 000359002300029, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000359002300029\nPubMedID: 26168399, https://www.ncbi.nlm.nih.gov/pubmed/26168399"},"doiId":"10.1038/nature14664","doiUrl":"https://dx.doi.org/10.1038/nature14664","featured":true,"firstPublished":{"html":"Aug 6, 2015 00:00:00 AM","text":"Aug 6, 2015 00:00:00 AM","value":"2015-08-06T00:00:00.000-07:00"},"mlaCitation":"George, Julie et al. ?Comprehensive Genomic Profiles of Small Cell Lung Cancer.? <i>NATURE</i> 524.7563 (2015): 47?U73. Print.","pubMedId":"26168399","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/26168399","publicationId":391969,"title":"Comprehensive genomic profiles of small cell lung cancer","type":"article","webOfScienceId":"000359002300029","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000359002300029"},{"abstract":"Mutations in the retinoblastoma tumor suppressor gene Rb are involved in many forms of human cancer. In this study, we investigated the early consequences of inactivating Rb in the context of cellular reprogramming. We found that Rb inactivation promotes the reprogramming of differentiated cells to a pluripotent state. Unexpectedly, this effect is cell cycle independent, and instead reflects direct binding of Rb to pluripotency genes, including Sox2 and Oct4, which leads to a repressed chromatin state. More broadly, this regulation of pluripotency networks and Sox2 in particular is critical for the initiation of tumors upon loss of Rb in mice. These studies therefore identify Rb as a global transcriptional repressor of pluripotency networks, providing a molecular basis for previous reports about its involvement in cell fate pliability, and implicate misregulation of pluripotency factors such as Sox2 in tumorigenesis related to loss of Rb function.","apaCitation":"Kareta, M. S., Gorges, L. L., Hafeez, S., Benayoun, B. A., Marro, S., Zmoos, A.-F., ? Wernig, M. (2015). Inhibition of pluripotency networks by the rb tumor suppressor restricts reprogramming and tumorigenesis. <i>Cell Stem Cell</i>, <i>16</i>(1), 39?50.","capCitation":"<span class=\"title\">\n  <span>Inhibition of pluripotency networks by the rb tumor suppressor restricts reprogramming and tumorigenesis.</span>\n  <i>Cell stem cell</i>\n</span>\n<span class=\"authors\">Kareta, M. S., Gorges, L. L., Hafeez, S., Benayoun, B. A., Marro, S., Zmoos, A., Cecchini, M. J., Spacek, D., Batista, L. F., O&#39;Brien, M., Ng, Y., Ang, C. E., Vaka, D., Artandi, S. E., Dick, F. A., Brunet, A., Sage, J., Wernig, M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2015</span>; <span class=\"volume\">16 (1)</span><span class=\"pages\">: 39-50</span>\n</span>","chicagoCitation":"Kareta, Michael S., Laura L. Gorges, Sana Hafeez, BÃ©rÃ©nice A. Benayoun, Samuele Marro, et al. 2015. ?Inhibition of Pluripotency Networks by the Rb Tumor Suppressor Restricts Reprogramming and Tumorigenesis.? <i>Cell Stem Cell</i> 16 (1). ELSEVIER INC: 39?50.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Mutations in the retinoblastoma tumor suppressor gene Rb are involved in many forms of human cancer. In this study, we investigated the early consequences of inactivating Rb in the context of cellular reprogramming. We found that Rb inactivation promotes the reprogramming of differentiated cells to a pluripotent state. Unexpectedly, this effect is cell cycle independent, and instead reflects direct binding of Rb to pluripotency genes, including Sox2 and Oct4, which leads to a repressed chromatin state. More broadly, this regulation of pluripotency networks and Sox2 in particular is critical for the initiation of tumors upon loss of Rb in mice. These studies therefore identify Rb as a global transcriptional repressor of pluripotency networks, providing a molecular basis for previous reports about its involvement in cell fate pliability, and implicate misregulation of pluripotency factors such as Sox2 in tumorigenesis related to loss of Rb function.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.stem.2014.10.019\">DOI 10.1016/j.stem.2014.10.019</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/25467916\">PubMedID 25467916</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4389904\">PubMedCentralID PMC4389904</a></span>\n  </p>","text":"Abstract:\nMutations in the retinoblastoma tumor suppressor gene Rb are involved in many forms of human cancer. In this study, we investigated the early consequences of inactivating Rb in the context of cellular reprogramming. We found that Rb inactivation promotes the reprogramming of differentiated cells to a pluripotent state. Unexpectedly, this effect is cell cycle independent, and instead reflects direct binding of Rb to pluripotency genes, including Sox2 and Oct4, which leads to a repressed chromatin state. More broadly, this regulation of pluripotency networks and Sox2 in particular is critical for the initiation of tumors upon loss of Rb in mice. These studies therefore identify Rb as a global transcriptional repressor of pluripotency networks, providing a molecular basis for previous reports about its involvement in cell fate pliability, and implicate misregulation of pluripotency factors such as Sox2 in tumorigenesis related to loss of Rb function.\nDOI: 10.1016/j.stem.2014.10.019, https://dx.doi.org/10.1016/j.stem.2014.10.019\n\nPubMedID: 25467916, https://www.ncbi.nlm.nih.gov/pubmed/25467916\nPubMedCentralID: PMC4389904, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4389904"},"doiId":"10.1016/j.stem.2014.10.019","doiUrl":"https://dx.doi.org/10.1016/j.stem.2014.10.019","featured":true,"firstPublished":{"html":"Jan 8, 2015 00:00:00 AM","text":"Jan 8, 2015 00:00:00 AM","value":"2015-01-08T00:00:00.000-08:00"},"mlaCitation":"Kareta, Michael S. et al. ?Inhibition of Pluripotency Networks by the Rb Tumor Suppressor Restricts Reprogramming and Tumorigenesis.? <i>Cell stem cell</i> 16.1 (2015): 39?50. Print.","pubMedId":"25467916","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/25467916","publicationId":346411,"title":"Inhibition of pluripotency networks by the rb tumor suppressor restricts reprogramming and tumorigenesis.","type":"article"},{"abstract":"Pediatric tumors are relatively infrequent but are often associated with significant lethality and lifelong morbidity. A major goal of pediatric cancer research has been to identify key drivers of tumorigenesis to eventually develop targeted therapies to enhance cure rate and minimize acute and long-term toxic effects. Here we used genomics approaches to identify biomarkers and candidate drivers for fibrolamellar hepatocellular carcinoma (FL-HCC), a very rare subtype of pediatric liver cancer for which limited therapeutic options exist. In-depth genomics analyses of one tumor followed by immunohistochemistry validation on seven other tumors showed expression of neuroendocrine markers in FL-HCC. DNA and RNA sequencing data further showed that common cancer pathways are not visibly altered in FL-HCC but identified two novel structural variants, both resulting in fusion transcripts. The first, a 400kb deletion, results in a DNAJ1-PRKCA fusion transcript, which leads to increased PKA activity in the index tumor case and other FL-HCC cases compared to normal liver. This PKA fusion protein is oncogenic in HCC cells. The second gene fusion event, a translocation between the CLPTML1 and GLIS3 genes, generates a transcript whose product also promotes cancer phenotypes in HCC cell lines. These experiments further highlight the tumorigenic role of gene fusions in the etiology of pediatric solid tumors and identify both candidate biomarkers and possible therapeutic targets for this lethal pediatric disease.","apaCitation":"Xu, L., Hazard, F. K., Zmoos, A.-F., Jahchan, N., Chaib, H., Garfin, P. M., ? Sage, J. (2015). Genomic analysis of fibrolamellar hepatocellular carcinoma. <i>Human Molecular Genetics</i>, <i>24</i>(1), 50?63.","capCitation":"<span class=\"title\">\n  <span>Genomic analysis of fibrolamellar hepatocellular carcinoma.</span>\n  <i>Human molecular genetics</i>\n</span>\n<span class=\"authors\">Xu, L., Hazard, F. K., Zmoos, A., Jahchan, N., Chaib, H., Garfin, P. M., Rangaswami, A., Snyder, M. P., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2015</span>; <span class=\"volume\">24 (1)</span><span class=\"pages\">: 50-63</span>\n</span>","chicagoCitation":"Xu, Lei, Florette K. Hazard, Anne-Flore Zmoos, Nadine Jahchan, Hassan Chaib, et al. 2015. ?Genomic Analysis of Fibrolamellar Hepatocellular Carcinoma.? <i>Human Molecular Genetics</i> 24 (1). OXFORD UNIV PRESS: 50?63.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Pediatric tumors are relatively infrequent but are often associated with significant lethality and lifelong morbidity. A major goal of pediatric cancer research has been to identify key drivers of tumorigenesis to eventually develop targeted therapies to enhance cure rate and minimize acute and long-term toxic effects. Here we used genomics approaches to identify biomarkers and candidate drivers for fibrolamellar hepatocellular carcinoma (FL-HCC), a very rare subtype of pediatric liver cancer for which limited therapeutic options exist. In-depth genomics analyses of one tumor followed by immunohistochemistry validation on seven other tumors showed expression of neuroendocrine markers in FL-HCC. DNA and RNA sequencing data further showed that common cancer pathways are not visibly altered in FL-HCC but identified two novel structural variants, both resulting in fusion transcripts. The first, a 400kb deletion, results in a DNAJ1-PRKCA fusion transcript, which leads to increased PKA activity in the index tumor case and other FL-HCC cases compared to normal liver. This PKA fusion protein is oncogenic in HCC cells. The second gene fusion event, a translocation between the CLPTML1 and GLIS3 genes, generates a transcript whose product also promotes cancer phenotypes in HCC cell lines. These experiments further highlight the tumorigenic role of gene fusions in the etiology of pediatric solid tumors and identify both candidate biomarkers and possible therapeutic targets for this lethal pediatric disease.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1093/hmg/ddu418\">DOI 10.1093/hmg/ddu418</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/25122662\">PubMedID 25122662</a></span>\n  </p>","text":"Abstract:\nPediatric tumors are relatively infrequent but are often associated with significant lethality and lifelong morbidity. A major goal of pediatric cancer research has been to identify key drivers of tumorigenesis to eventually develop targeted therapies to enhance cure rate and minimize acute and long-term toxic effects. Here we used genomics approaches to identify biomarkers and candidate drivers for fibrolamellar hepatocellular carcinoma (FL-HCC), a very rare subtype of pediatric liver cancer for which limited therapeutic options exist. In-depth genomics analyses of one tumor followed by immunohistochemistry validation on seven other tumors showed expression of neuroendocrine markers in FL-HCC. DNA and RNA sequencing data further showed that common cancer pathways are not visibly altered in FL-HCC but identified two novel structural variants, both resulting in fusion transcripts. The first, a 400kb deletion, results in a DNAJ1-PRKCA fusion transcript, which leads to increased PKA activity in the index tumor case and other FL-HCC cases compared to normal liver. This PKA fusion protein is oncogenic in HCC cells. The second gene fusion event, a translocation between the CLPTML1 and GLIS3 genes, generates a transcript whose product also promotes cancer phenotypes in HCC cell lines. These experiments further highlight the tumorigenic role of gene fusions in the etiology of pediatric solid tumors and identify both candidate biomarkers and possible therapeutic targets for this lethal pediatric disease.\nDOI: 10.1093/hmg/ddu418, https://dx.doi.org/10.1093/hmg/ddu418\n\nPubMedID: 25122662, https://www.ncbi.nlm.nih.gov/pubmed/25122662"},"doiId":"10.1093/hmg/ddu418","doiUrl":"https://dx.doi.org/10.1093/hmg/ddu418","featured":false,"firstPublished":{"html":"Jan 1, 2015 00:00:00 AM","text":"Jan 1, 2015 00:00:00 AM","value":"2015-01-01T00:00:00.000-08:00"},"mlaCitation":"Xu, Lei et al. ?Genomic Analysis of Fibrolamellar Hepatocellular Carcinoma.? <i>Human molecular genetics</i> 24.1 (2015): 50?63. Print.","pubMedId":"25122662","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/25122662","publicationId":322199,"title":"Genomic analysis of fibrolamellar hepatocellular carcinoma.","type":"article"},{"abstract":"In mammals, a cell's decision to divide is thought toÂ be under the control of the Rb/E2F pathway. We previously found that inactivation of the Rb family of cell cycle inhibitors (Rb, p107, and p130) in quiescent liver progenitors leads to uncontrolled division and cancer initiation. Here, we show that, in contrast, deletion of the entire Rb gene family in mature hepatocytes is not sufficient for their long-term proliferation. The cell cycle block in Rb family mutant hepatocytes is independent of the Arf/p53/p21 checkpoint but can be abrogated upon decreasing liver size. At the molecular level, we identify YAP, a transcriptional regulator involved in organ size control, as a factor required for the sustained expression of cell cycle genes in hepatocytes. These experiments identify a higher level of regulation of the cell cycle inÂ vivo in which signals regulating organ size are dominant regulators of the core cell cycle machinery.","apaCitation":"Ehmer, U., Zmoos, A.-F., Auerbach, R. K., Vaka, D., Butte, A. J., Kay, M. A., &amp; Sage, J. (2014). Organ Size Control Is Dominant over Rb Family Inactivation to Restrict Proliferation InÂ Vivo. <i>Cell Reports</i>, <i>8</i>(2), 371?381.","capCitation":"<span class=\"title\">\n  <span>Organ Size Control Is Dominant over Rb Family Inactivation to Restrict Proliferation InÂ Vivo.</span>\n  <i>Cell reports</i>\n</span>\n<span class=\"authors\">Ehmer, U., Zmoos, A., Auerbach, R. K., Vaka, D., Butte, A. J., Kay, M. A., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2014</span>; <span class=\"volume\">8 (2)</span><span class=\"pages\">: 371-381</span>\n</span>","chicagoCitation":"Ehmer, Ursula, Anne-Flore Zmoos, Raymond K. Auerbach, Dedeepya Vaka, Atul J. Butte, et al. 2014. ?Organ Size Control Is Dominant over Rb Family Inactivation to Restrict Proliferation InÂ Vivo.? <i>Cell Reports</i> 8 (2). ELSEVIER INC: 371?81.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">In mammals, a cell's decision to divide is thought toÂ be under the control of the Rb/E2F pathway. We previously found that inactivation of the Rb family of cell cycle inhibitors (Rb, p107, and p130) in quiescent liver progenitors leads to uncontrolled division and cancer initiation. Here, we show that, in contrast, deletion of the entire Rb gene family in mature hepatocytes is not sufficient for their long-term proliferation. The cell cycle block in Rb family mutant hepatocytes is independent of the Arf/p53/p21 checkpoint but can be abrogated upon decreasing liver size. At the molecular level, we identify YAP, a transcriptional regulator involved in organ size control, as a factor required for the sustained expression of cell cycle genes in hepatocytes. These experiments identify a higher level of regulation of the cell cycle inÂ vivo in which signals regulating organ size are dominant regulators of the core cell cycle machinery.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.celrep.2014.06.025\">DOI 10.1016/j.celrep.2014.06.025</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/25017070\">PubMedID 25017070</a></span>\n  </p>","text":"Abstract:\nIn mammals, a cell's decision to divide is thought toÂ be under the control of the Rb/E2F pathway. We previously found that inactivation of the Rb family of cell cycle inhibitors (Rb, p107, and p130) in quiescent liver progenitors leads to uncontrolled division and cancer initiation. Here, we show that, in contrast, deletion of the entire Rb gene family in mature hepatocytes is not sufficient for their long-term proliferation. The cell cycle block in Rb family mutant hepatocytes is independent of the Arf/p53/p21 checkpoint but can be abrogated upon decreasing liver size. At the molecular level, we identify YAP, a transcriptional regulator involved in organ size control, as a factor required for the sustained expression of cell cycle genes in hepatocytes. These experiments identify a higher level of regulation of the cell cycle inÂ vivo in which signals regulating organ size are dominant regulators of the core cell cycle machinery.\nDOI: 10.1016/j.celrep.2014.06.025, https://dx.doi.org/10.1016/j.celrep.2014.06.025\n\nPubMedID: 25017070, https://www.ncbi.nlm.nih.gov/pubmed/25017070"},"doiId":"10.1016/j.celrep.2014.06.025","doiUrl":"https://dx.doi.org/10.1016/j.celrep.2014.06.025","featured":true,"firstPublished":{"html":"Jul 24, 2014 00:00:00 AM","text":"Jul 24, 2014 00:00:00 AM","value":"2014-07-24T00:00:00.000-07:00"},"mlaCitation":"Ehmer, Ursula et al. ?Organ Size Control Is Dominant over Rb Family Inactivation to Restrict Proliferation InÂ Vivo.? <i>Cell reports</i> 8.2 (2014): 371?381. Print.","pubMedId":"25017070","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/25017070","publicationId":322996,"title":"Organ Size Control Is Dominant over Rb Family Inactivation to Restrict Proliferation InÂ Vivo.","type":"article"},{"abstract":"Deregulation of lysine methylation signalling has emerged as a common aetiological factor in cancer pathogenesis, with inhibitors of several histone lysine methyltransferases (KMTs) being developed as chemotherapeutics. The largely cytoplasmic KMT SMYD3 (SET and MYND domain containing protein 3) is overexpressed in numerous human tumours. However, the molecular mechanism by which SMYD3 regulates cancer pathways and its relationship to tumorigenesis in vivo are largely unknown. Here we show that methylation of MAP3K2 by SMYD3 increases MAP kinase signalling and promotes the formation of Ras-driven carcinomas. Using mouse models for pancreatic ductal adenocarcinoma and lung adenocarcinoma, we found that abrogating SMYD3 catalytic activity inhibits tumour development in response to oncogenic Ras. We used protein array technology to identify the MAP3K2 kinase as a target of SMYD3. In cancer cell lines, SMYD3-mediated methylation of MAP3K2 at lysine 260 potentiates activation of the Ras/Raf/MEK/ERK signalling module and SMYD3 depletion synergizes with a MEK inhibitor to block Ras-driven tumorigenesis. Finally, the PP2A phosphatase complex, a key negative regulator of the MAP kinase pathway, binds to MAP3K2 and this interaction is blocked by methylation. Together, our results elucidate a new role for lysine methylation in integrating cytoplasmic kinase-signalling cascades and establish a pivotal role for SMYD3 in the regulation of oncogenic Ras signalling.","apaCitation":"Mazur, P. K., Reynoird, N., Khatri, P., Jansen, P. W. T. C., Wilkinson, A. W., Liu, S., ? Gozani, O. (2014). SMYD3 links lysine methylation of MAP3K2 to Ras-driven cancer. <i>Nature</i>, <i>510</i>(7504), 283?287.","capCitation":"<span class=\"title\">\n  <span>SMYD3 links lysine methylation of MAP3K2 to Ras-driven cancer.</span>\n  <i>Nature</i>\n</span>\n<span class=\"authors\">Mazur, P. K., Reynoird, N., Khatri, P., Jansen, P. W., Wilkinson, A. W., Liu, S., Barbash, O., Van Aller, G. S., Huddleston, M., Dhanak, D., Tummino, P. J., Kruger, R. G., Garcia, B. A., Butte, A. J., Vermeulen, M., Sage, J., Gozani, O.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2014</span>; <span class=\"volume\">510 (7504)</span><span class=\"pages\">: 283-287</span>\n</span>","chicagoCitation":"Mazur, Pawel K., Nicolas Reynoird, Purvesh Khatri, Pascal WTC Jansen, Alex W. Wilkinson, et al. 2014. ?SMYD3 Links Lysine Methylation of MAP3K2 to Ras-Driven Cancer.? <i>Nature</i> 510 (7504): 283?87.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Deregulation of lysine methylation signalling has emerged as a common aetiological factor in cancer pathogenesis, with inhibitors of several histone lysine methyltransferases (KMTs) being developed as chemotherapeutics. The largely cytoplasmic KMT SMYD3 (SET and MYND domain containing protein 3) is overexpressed in numerous human tumours. However, the molecular mechanism by which SMYD3 regulates cancer pathways and its relationship to tumorigenesis in vivo are largely unknown. Here we show that methylation of MAP3K2 by SMYD3 increases MAP kinase signalling and promotes the formation of Ras-driven carcinomas. Using mouse models for pancreatic ductal adenocarcinoma and lung adenocarcinoma, we found that abrogating SMYD3 catalytic activity inhibits tumour development in response to oncogenic Ras. We used protein array technology to identify the MAP3K2 kinase as a target of SMYD3. In cancer cell lines, SMYD3-mediated methylation of MAP3K2 at lysine 260 potentiates activation of the Ras/Raf/MEK/ERK signalling module and SMYD3 depletion synergizes with a MEK inhibitor to block Ras-driven tumorigenesis. Finally, the PP2A phosphatase complex, a key negative regulator of the MAP kinase pathway, binds to MAP3K2 and this interaction is blocked by methylation. Together, our results elucidate a new role for lysine methylation in integrating cytoplasmic kinase-signalling cascades and establish a pivotal role for SMYD3 in the regulation of oncogenic Ras signalling.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nature13320\">DOI 10.1038/nature13320</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/24847881\">PubMedID 24847881</a></span>\n  </p>","text":"Abstract:\nDeregulation of lysine methylation signalling has emerged as a common aetiological factor in cancer pathogenesis, with inhibitors of several histone lysine methyltransferases (KMTs) being developed as chemotherapeutics. The largely cytoplasmic KMT SMYD3 (SET and MYND domain containing protein 3) is overexpressed in numerous human tumours. However, the molecular mechanism by which SMYD3 regulates cancer pathways and its relationship to tumorigenesis in vivo are largely unknown. Here we show that methylation of MAP3K2 by SMYD3 increases MAP kinase signalling and promotes the formation of Ras-driven carcinomas. Using mouse models for pancreatic ductal adenocarcinoma and lung adenocarcinoma, we found that abrogating SMYD3 catalytic activity inhibits tumour development in response to oncogenic Ras. We used protein array technology to identify the MAP3K2 kinase as a target of SMYD3. In cancer cell lines, SMYD3-mediated methylation of MAP3K2 at lysine 260 potentiates activation of the Ras/Raf/MEK/ERK signalling module and SMYD3 depletion synergizes with a MEK inhibitor to block Ras-driven tumorigenesis. Finally, the PP2A phosphatase complex, a key negative regulator of the MAP kinase pathway, binds to MAP3K2 and this interaction is blocked by methylation. Together, our results elucidate a new role for lysine methylation in integrating cytoplasmic kinase-signalling cascades and establish a pivotal role for SMYD3 in the regulation of oncogenic Ras signalling.\nDOI: 10.1038/nature13320, https://dx.doi.org/10.1038/nature13320\n\nPubMedID: 24847881, https://www.ncbi.nlm.nih.gov/pubmed/24847881"},"doiId":"10.1038/nature13320","doiUrl":"https://dx.doi.org/10.1038/nature13320","featured":true,"firstPublished":{"html":"Jun 12, 2014 00:00:00 AM","text":"Jun 12, 2014 00:00:00 AM","value":"2014-06-12T00:00:00.000-07:00"},"mlaCitation":"Mazur, Pawel K. et al. ?SMYD3 Links Lysine Methylation of MAP3K2 to Ras-Driven Cancer.? <i>Nature</i> 510.7504 (2014): 283?287. Print.","pubMedId":"24847881","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/24847881","publicationId":317914,"title":"SMYD3 links lysine methylation of MAP3K2 to Ras-driven cancer.","type":"article"},{"apaCitation":"Ntziachristos, P., Lim, J. S., Sage, J., &amp; Aifantis, I. (2014). From Fly Wings to Targeted Cancer Therapies: A Centennial for Notch Signaling. <i>CANCER CELL</i>, <i>25</i>(3), 318?334.","capCitation":"<span class=\"title\">\n  <span>From Fly Wings to Targeted Cancer Therapies: A Centennial for Notch Signaling</span>\n  <i>CANCER CELL</i>\n</span>\n<span class=\"authors\">Ntziachristos, P., Lim, J. S., Sage, J., Aifantis, I.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2014</span>; <span class=\"volume\">25 (3)</span><span class=\"pages\">: 318-334</span>\n</span>","chicagoCitation":"Ntziachristos, Panagiotis, Jing Shan Lim, Julien Sage, and Iannis Aifantis. 2014. ?From Fly Wings to Targeted Cancer Therapies: A Centennial for Notch Signaling.? <i>CANCER CELL</i> 25 (3). CELL PRESS: 318?34.","detail":{"html":"<p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.ccr.2014.02.018\">DOI 10.1016/j.ccr.2014.02.018</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000333233400010\">Web of Science ID 000333233400010</a></span>\n  </p>","text":"DOI: 10.1016/j.ccr.2014.02.018, https://dx.doi.org/10.1016/j.ccr.2014.02.018\nWeb of Science ID: 000333233400010, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000333233400010"},"doiId":"10.1016/j.ccr.2014.02.018","doiUrl":"https://dx.doi.org/10.1016/j.ccr.2014.02.018","featured":false,"firstPublished":{"html":"Mar 17, 2014 00:00:00 AM","text":"Mar 17, 2014 00:00:00 AM","value":"2014-03-17T00:00:00.000-07:00"},"mlaCitation":"Ntziachristos, Panagiotis et al. ?From Fly Wings to Targeted Cancer Therapies: A Centennial for Notch Signaling.? <i>CANCER CELL</i> 25.3 (2014): 318?334. Print.","publicationId":308307,"title":"From Fly Wings to Targeted Cancer Therapies: A Centennial for Notch Signaling","type":"article","webOfScienceId":"000333233400010","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000333233400010"},{"abstract":"Small cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer with high mortality. We used a systematic drug repositioning bioinformatics approach querying a large compendium of gene expression profiles to identify candidate U.S. Food and Drug Administration (FDA)-approved drugs to treat SCLC. We found that tricyclic antidepressants and related molecules potently induce apoptosis in both chemonaÃ¯ve and chemoresistant SCLC cells in culture, in mouse and human SCLC tumors transplanted into immunocompromised mice, and in endogenous tumors from a mouse model for human SCLC. The candidate drugs activate stress pathways and induce cell death in SCLC cells, at least in part by disrupting autocrine survival signals involving neurotransmitters and their G protein-coupled receptors. The candidate drugs inhibit the growth of other neuroendocrine tumors, including pancreatic neuroendocrine tumors and Merkel cell carcinoma. These experiments identify novel targeted strategies that can be rapidly evaluated in patients with neuroendocrine tumors through the repurposing of approved drugs.Our work shows the power of bioinformatics-based drug approaches to rapidly repurpose FDA-approved drugs and identifies a novel class of molecules to treat patients with SCLC, a cancer for which no effective novel systemic treatments have been identified in several decades. In addition, our experiments highlight the importance of novel autocrine mechanisms in promoting the growth of neuroendocrine tumor cells.","apaCitation":"Jahchan, N. S., Dudley, J. T., Mazur, P. K., Flores, N., Yang, D., Palmerton, A., ? Sage, J. (2013). A Drug Repositioning Approach Identifies Tricyclic Antidepressants as Inhibitors of Small Cell Lung Cancer and Other Neuroendocrine Tumors. <i>CANCER DISCOVERY</i>, <i>3</i>(12), 1364?1377.","capCitation":"<span class=\"title\">\n  <span>A Drug Repositioning Approach Identifies Tricyclic Antidepressants as Inhibitors of Small Cell Lung Cancer and Other Neuroendocrine Tumors</span>\n  <i>CANCER DISCOVERY</i>\n</span>\n<span class=\"authors\">Jahchan, N. S., Dudley, J. T., Mazur, P. K., Flores, N., Yang, D., Palmerton, A., Zmoos, A., Vaka, D., Tran, K. Q., Zhou, M., Krasinska, K., Riess, J. W., Neal, J. W., Khatri, P., Park, K. S., Butte, A. J., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2013</span>; <span class=\"volume\">3 (12)</span><span class=\"pages\">: 1364-1377</span>\n</span>","chicagoCitation":"Jahchan, Nadine S., Joel T. Dudley, Pawel K. Mazur, Natasha Flores, Dian Yang, et al. 2013. ?A Drug Repositioning Approach Identifies Tricyclic Antidepressants as Inhibitors of Small Cell Lung Cancer and Other Neuroendocrine Tumors.? <i>CANCER DISCOVERY</i> 3 (12). AMER ASSOC CANCER RESEARCH: 1364?77.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer with high mortality. We used a systematic drug repositioning bioinformatics approach querying a large compendium of gene expression profiles to identify candidate U.S. Food and Drug Administration (FDA)-approved drugs to treat SCLC. We found that tricyclic antidepressants and related molecules potently induce apoptosis in both chemonaÃ¯ve and chemoresistant SCLC cells in culture, in mouse and human SCLC tumors transplanted into immunocompromised mice, and in endogenous tumors from a mouse model for human SCLC. The candidate drugs activate stress pathways and induce cell death in SCLC cells, at least in part by disrupting autocrine survival signals involving neurotransmitters and their G protein-coupled receptors. The candidate drugs inhibit the growth of other neuroendocrine tumors, including pancreatic neuroendocrine tumors and Merkel cell carcinoma. These experiments identify novel targeted strategies that can be rapidly evaluated in patients with neuroendocrine tumors through the repurposing of approved drugs.Our work shows the power of bioinformatics-based drug approaches to rapidly repurpose FDA-approved drugs and identifies a novel class of molecules to treat patients with SCLC, a cancer for which no effective novel systemic treatments have been identified in several decades. In addition, our experiments highlight the importance of novel autocrine mechanisms in promoting the growth of neuroendocrine tumor cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1158/2159-8290.CD-13-0183\">DOI 10.1158/2159-8290.CD-13-0183</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000328257500023\">Web of Science ID 000328257500023</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/24078773\">PubMedID 24078773</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3864571\">PubMedCentralID PMC3864571</a></span>\n  </p>","text":"Abstract:\nSmall cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer with high mortality. We used a systematic drug repositioning bioinformatics approach querying a large compendium of gene expression profiles to identify candidate U.S. Food and Drug Administration (FDA)-approved drugs to treat SCLC. We found that tricyclic antidepressants and related molecules potently induce apoptosis in both chemonaÃ¯ve and chemoresistant SCLC cells in culture, in mouse and human SCLC tumors transplanted into immunocompromised mice, and in endogenous tumors from a mouse model for human SCLC. The candidate drugs activate stress pathways and induce cell death in SCLC cells, at least in part by disrupting autocrine survival signals involving neurotransmitters and their G protein-coupled receptors. The candidate drugs inhibit the growth of other neuroendocrine tumors, including pancreatic neuroendocrine tumors and Merkel cell carcinoma. These experiments identify novel targeted strategies that can be rapidly evaluated in patients with neuroendocrine tumors through the repurposing of approved drugs.Our work shows the power of bioinformatics-based drug approaches to rapidly repurpose FDA-approved drugs and identifies a novel class of molecules to treat patients with SCLC, a cancer for which no effective novel systemic treatments have been identified in several decades. In addition, our experiments highlight the importance of novel autocrine mechanisms in promoting the growth of neuroendocrine tumor cells.\nDOI: 10.1158/2159-8290.CD-13-0183, https://dx.doi.org/10.1158/2159-8290.CD-13-0183\nWeb of Science ID: 000328257500023, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000328257500023\nPubMedID: 24078773, https://www.ncbi.nlm.nih.gov/pubmed/24078773\nPubMedCentralID: PMC3864571, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3864571"},"doiId":"10.1158/2159-8290.CD-13-0183","doiUrl":"https://dx.doi.org/10.1158/2159-8290.CD-13-0183","featured":true,"firstPublished":{"html":"Dec 1, 2013 00:00:00 AM","text":"Dec 1, 2013 00:00:00 AM","value":"2013-12-01T00:00:00.000-08:00"},"mlaCitation":"Jahchan, Nadine S. et al. ?A Drug Repositioning Approach Identifies Tricyclic Antidepressants as Inhibitors of Small Cell Lung Cancer and Other Neuroendocrine Tumors.? <i>CANCER DISCOVERY</i> 3.12 (2013): 1364?1377. Print.","pubMedId":"24078773","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/24078773","publicationId":285360,"title":"A Drug Repositioning Approach Identifies Tricyclic Antidepressants as Inhibitors of Small Cell Lung Cancer and Other Neuroendocrine Tumors","type":"article","webOfScienceId":"000328257500023","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000328257500023"},{"abstract":"Thymic involution during aging is a major cause of decreased production of T cells and reduced immunity. Here we show that inactivation of Rb family genes in young mice prevents thymic involution and results in an enlarged thymus competent for increased production of naive T cells. This phenotype originates from the expansion of functional thymic epithelial cells (TECs). In RB family mutant TECs, increased activity of E2F transcription factors drives increased expression of Foxn1, a central regulator of the thymic epithelium. Increased Foxn1 expression is required for the thymic expansion observed in Rb family mutant mice. Thus, the RB family promotes thymic involution and controls T cell production via a bone marrow-independent mechanism, identifying a novel pathway to target to increase thymic function in patients.","apaCitation":"Garfin, P. M., Min, D., Bryson, J. L., Serwold, T., Edris, B., Blackburn, C. C., ? Viatour, P. (2013). Inactivation of the RB family prevents thymus involution and promotes thymic function by direct control of Foxn1 expression. <i>Journal of Experimental Medicine</i>, <i>210</i>(6), 1087?1097.","capCitation":"<span class=\"title\">\n  <span>Inactivation of the RB family prevents thymus involution and promotes thymic function by direct control of Foxn1 expression.</span>\n  <i>journal of experimental medicine</i>\n</span>\n<span class=\"authors\">Garfin, P. M., Min, D., Bryson, J. L., Serwold, T., Edris, B., Blackburn, C. C., Richie, E. R., Weinberg, K. I., Manley, N. R., Sage, J., Viatour, P.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2013</span>; <span class=\"volume\">210 (6)</span><span class=\"pages\">: 1087-1097</span>\n</span>","chicagoCitation":"Garfin, Phillip M., Dullei Min, Jerrod L. Bryson, Thomas Serwold, Badreddin Edris, et al. 2013. ?Inactivation of the RB Family Prevents Thymus Involution and Promotes Thymic Function by Direct Control of Foxn1 Expression.? <i>Journal of Experimental Medicine</i> 210 (6): 1087?97.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Thymic involution during aging is a major cause of decreased production of T cells and reduced immunity. Here we show that inactivation of Rb family genes in young mice prevents thymic involution and results in an enlarged thymus competent for increased production of naive T cells. This phenotype originates from the expansion of functional thymic epithelial cells (TECs). In RB family mutant TECs, increased activity of E2F transcription factors drives increased expression of Foxn1, a central regulator of the thymic epithelium. Increased Foxn1 expression is required for the thymic expansion observed in Rb family mutant mice. Thus, the RB family promotes thymic involution and controls T cell production via a bone marrow-independent mechanism, identifying a novel pathway to target to increase thymic function in patients.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1084/jem.20121716\">DOI 10.1084/jem.20121716</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/23669396\">PubMedID 23669396</a></span>\n  </p>","text":"Abstract:\nThymic involution during aging is a major cause of decreased production of T cells and reduced immunity. Here we show that inactivation of Rb family genes in young mice prevents thymic involution and results in an enlarged thymus competent for increased production of naive T cells. This phenotype originates from the expansion of functional thymic epithelial cells (TECs). In RB family mutant TECs, increased activity of E2F transcription factors drives increased expression of Foxn1, a central regulator of the thymic epithelium. Increased Foxn1 expression is required for the thymic expansion observed in Rb family mutant mice. Thus, the RB family promotes thymic involution and controls T cell production via a bone marrow-independent mechanism, identifying a novel pathway to target to increase thymic function in patients.\nDOI: 10.1084/jem.20121716, https://dx.doi.org/10.1084/jem.20121716\n\nPubMedID: 23669396, https://www.ncbi.nlm.nih.gov/pubmed/23669396"},"doiId":"10.1084/jem.20121716","doiUrl":"https://dx.doi.org/10.1084/jem.20121716","featured":false,"firstPublished":{"html":"Jun 3, 2013 00:00:00 AM","text":"Jun 3, 2013 00:00:00 AM","value":"2013-06-03T00:00:00.000-07:00"},"mlaCitation":"Garfin, Phillip M. et al. ?Inactivation of the RB Family Prevents Thymus Involution and Promotes Thymic Function by Direct Control of Foxn1 Expression.? <i>journal of experimental medicine</i> 210.6 (2013): 1087?1097. Print.","pubMedId":"23669396","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/23669396","publicationId":132009,"title":"Inactivation of the RB family prevents thymus involution and promotes thymic function by direct control of Foxn1 expression.","type":"article"},{"abstract":"Upregulation of the ERK1 and ERK2 (ERK1/2) MAP kinase (MAPK) cascade occurs in >30% of cancers, often through mutational activation of receptor tyrosine kinases or other upstream genes, including KRAS and BRAF. Efforts to target endogenous MAPKs are challenged by the fact that these kinases are required for viability in mammals. Additionally, the effectiveness of new inhibitors of mutant BRAF has been diminished by acquired tumor resistance through selection for BRAF-independent mechanisms of ERK1/2 induction. Furthermore, recently identified ERK1/2-inducing mutations in MEK1 and MEK2 (MEK1/2) MAPK genes in melanoma confer resistance to emerging therapeutic MEK inhibitors, underscoring the challenges facing direct kinase inhibition in cancer. MAPK scaffolds, such as IQ motif-containing GTPase activating protein 1 (IQGAP1), assemble pathway kinases to affect signal transmission, and disrupting scaffold function therefore offers an orthogonal approach to MAPK cascade inhibition. Consistent with this, we found a requirement for IQGAP1 in RAS-driven tumorigenesis in mouse and human tissue. In addition, the ERK1/2-binding IQGAP1 WW domain peptide disrupted IQGAP1-ERK1/2 interactions, inhibited RAS- and RAF-driven tumorigenesis, bypassed acquired resistance to the BRAF inhibitor vemurafenib (PLX-4032) and acted as a systemically deliverable therapeutic to significantly increase the lifespan of tumor-bearing mice. Scaffold-kinase interaction blockade acts by a mechanism distinct from direct kinase inhibition and may be a strategy to target overactive oncogenic kinase cascades in cancer.","apaCitation":"Jameson, K. L., Mazur, P. K., Zehnder, A. M., Zhang, J., Zarnegar, B., Sage, J., &amp; Khavari, P. A. (2013). IQGAP1 scaffold-kinase interaction blockade selectively targets RAS-MAP kinase-driven tumors. <i>Nature Medicine</i>, <i>19</i>(5), 626?630.","capCitation":"<span class=\"title\">\n  <span>IQGAP1 scaffold-kinase interaction blockade selectively targets RAS-MAP kinase-driven tumors.</span>\n  <i>Nature medicine</i>\n</span>\n<span class=\"authors\">Jameson, K. L., Mazur, P. K., Zehnder, A. M., Zhang, J., Zarnegar, B., Sage, J., Khavari, P. A.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2013</span>; <span class=\"volume\">19 (5)</span><span class=\"pages\">: 626-630</span>\n</span>","chicagoCitation":"Jameson, Katherine L., Pawel K. Mazur, Ashley M. Zehnder, Jiajing Zhang, Brian Zarnegar, et al. 2013. ?IQGAP1 Scaffold-Kinase Interaction Blockade Selectively Targets RAS-MAP Kinase-Driven Tumors.? <i>Nature Medicine</i> 19 (5): 626?30.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Upregulation of the ERK1 and ERK2 (ERK1/2) MAP kinase (MAPK) cascade occurs in >30% of cancers, often through mutational activation of receptor tyrosine kinases or other upstream genes, including KRAS and BRAF. Efforts to target endogenous MAPKs are challenged by the fact that these kinases are required for viability in mammals. Additionally, the effectiveness of new inhibitors of mutant BRAF has been diminished by acquired tumor resistance through selection for BRAF-independent mechanisms of ERK1/2 induction. Furthermore, recently identified ERK1/2-inducing mutations in MEK1 and MEK2 (MEK1/2) MAPK genes in melanoma confer resistance to emerging therapeutic MEK inhibitors, underscoring the challenges facing direct kinase inhibition in cancer. MAPK scaffolds, such as IQ motif-containing GTPase activating protein 1 (IQGAP1), assemble pathway kinases to affect signal transmission, and disrupting scaffold function therefore offers an orthogonal approach to MAPK cascade inhibition. Consistent with this, we found a requirement for IQGAP1 in RAS-driven tumorigenesis in mouse and human tissue. In addition, the ERK1/2-binding IQGAP1 WW domain peptide disrupted IQGAP1-ERK1/2 interactions, inhibited RAS- and RAF-driven tumorigenesis, bypassed acquired resistance to the BRAF inhibitor vemurafenib (PLX-4032) and acted as a systemically deliverable therapeutic to significantly increase the lifespan of tumor-bearing mice. Scaffold-kinase interaction blockade acts by a mechanism distinct from direct kinase inhibition and may be a strategy to target overactive oncogenic kinase cascades in cancer.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nm.3165\">DOI 10.1038/nm.3165</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/23603816\">PubMedID 23603816</a></span>\n  </p>","text":"Abstract:\nUpregulation of the ERK1 and ERK2 (ERK1/2) MAP kinase (MAPK) cascade occurs in >30% of cancers, often through mutational activation of receptor tyrosine kinases or other upstream genes, including KRAS and BRAF. Efforts to target endogenous MAPKs are challenged by the fact that these kinases are required for viability in mammals. Additionally, the effectiveness of new inhibitors of mutant BRAF has been diminished by acquired tumor resistance through selection for BRAF-independent mechanisms of ERK1/2 induction. Furthermore, recently identified ERK1/2-inducing mutations in MEK1 and MEK2 (MEK1/2) MAPK genes in melanoma confer resistance to emerging therapeutic MEK inhibitors, underscoring the challenges facing direct kinase inhibition in cancer. MAPK scaffolds, such as IQ motif-containing GTPase activating protein 1 (IQGAP1), assemble pathway kinases to affect signal transmission, and disrupting scaffold function therefore offers an orthogonal approach to MAPK cascade inhibition. Consistent with this, we found a requirement for IQGAP1 in RAS-driven tumorigenesis in mouse and human tissue. In addition, the ERK1/2-binding IQGAP1 WW domain peptide disrupted IQGAP1-ERK1/2 interactions, inhibited RAS- and RAF-driven tumorigenesis, bypassed acquired resistance to the BRAF inhibitor vemurafenib (PLX-4032) and acted as a systemically deliverable therapeutic to significantly increase the lifespan of tumor-bearing mice. Scaffold-kinase interaction blockade acts by a mechanism distinct from direct kinase inhibition and may be a strategy to target overactive oncogenic kinase cascades in cancer.\nDOI: 10.1038/nm.3165, https://dx.doi.org/10.1038/nm.3165\n\nPubMedID: 23603816, https://www.ncbi.nlm.nih.gov/pubmed/23603816"},"doiId":"10.1038/nm.3165","doiUrl":"https://dx.doi.org/10.1038/nm.3165","featured":false,"firstPublished":{"html":"May 1, 2013 00:00:00 AM","text":"May 1, 2013 00:00:00 AM","value":"2013-05-01T00:00:00.000-07:00"},"mlaCitation":"Jameson, Katherine L. et al. ?IQGAP1 Scaffold-Kinase Interaction Blockade Selectively Targets RAS-MAP Kinase-Driven Tumors.? <i>Nature medicine</i> 19.5 (2013): 626?630. Print.","pubMedId":"23603816","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/23603816","publicationId":131405,"title":"IQGAP1 scaffold-kinase interaction blockade selectively targets RAS-MAP kinase-driven tumors.","type":"article"},{"abstract":"The retinoblastoma tumor suppressor RB is well known for its capacity to restrict cell cycle progression at the G1/S transition of the cell cycle by controlling the transcription of cell cycle genes. In this issue of Genes & Development, Hilgendorf and colleagues (pp. 1003-1015) have identified a novel tumor suppressor function for RB independent of its role as a transcriptional regulator, in which RB directly activates the apoptosis regulator Bax at the mitochondria to promote cell death.","apaCitation":"Attardi, L. D., &amp; Sage, J. (2013). RB goes mitochondrial. <i>Genes &amp; Development</i>, <i>27</i>(9), 975?979.","capCitation":"<span class=\"title\">\n  <span>RB goes mitochondrial.</span>\n  <i>Genes &amp; development</i>\n</span>\n<span class=\"authors\">Attardi, L. D., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2013</span>; <span class=\"volume\">27 (9)</span><span class=\"pages\">: 975-979</span>\n</span>","chicagoCitation":"Attardi, Laura D., and Julien Sage. 2013. ?RB Goes Mitochondrial.? <i>Genes &amp; Development</i> 27 (9): 975?79.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma tumor suppressor RB is well known for its capacity to restrict cell cycle progression at the G1/S transition of the cell cycle by controlling the transcription of cell cycle genes. In this issue of Genes & Development, Hilgendorf and colleagues (pp. 1003-1015) have identified a novel tumor suppressor function for RB independent of its role as a transcriptional regulator, in which RB directly activates the apoptosis regulator Bax at the mitochondria to promote cell death.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.219451.113\">DOI 10.1101/gad.219451.113</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/23651852\">PubMedID 23651852</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma tumor suppressor RB is well known for its capacity to restrict cell cycle progression at the G1/S transition of the cell cycle by controlling the transcription of cell cycle genes. In this issue of Genes & Development, Hilgendorf and colleagues (pp. 1003-1015) have identified a novel tumor suppressor function for RB independent of its role as a transcriptional regulator, in which RB directly activates the apoptosis regulator Bax at the mitochondria to promote cell death.\nDOI: 10.1101/gad.219451.113, https://dx.doi.org/10.1101/gad.219451.113\n\nPubMedID: 23651852, https://www.ncbi.nlm.nih.gov/pubmed/23651852"},"doiId":"10.1101/gad.219451.113","doiUrl":"https://dx.doi.org/10.1101/gad.219451.113","featured":false,"firstPublished":{"html":"May 1, 2013 00:00:00 AM","text":"May 1, 2013 00:00:00 AM","value":"2013-05-01T00:00:00.000-07:00"},"mlaCitation":"Attardi, Laura D., and Julien Sage. ?RB Goes Mitochondrial.? <i>Genes &amp; development</i> 27.9 (2013): 975?979. Print.","pubMedId":"23651852","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/23651852","publicationId":131847,"title":"RB goes mitochondrial.","type":"article"},{"abstract":"The mechanisms ensuring the long-term self-renewal of human embryonic stem cells are still only partly understood, limiting their use in cellular therapies. Here we found that increased activity of the RB cell cycle inhibitor in human embryonic stem cells induces cell cycle arrest, differentiation and cell death. Conversely, inactivation of the entire RB family (RB, p107 and p130) in human embryonic stem cells triggers G2/M arrest and cell death through functional activation of the p53 pathway and the cell cycle inhibitor p21. Differences in E2F target gene activation upon loss of RB family function between human embryonic stem cells, mouse embryonic stem cells and human fibroblasts underscore key differences in the cell cycle regulatory networks of human embryonic stem cells. Finally, loss of RB family function promotes genomic instability in both human and mouse embryonic stem cells, uncoupling cell cycle defects from chromosomal instability. These experiments indicate that a homeostatic level of RB activity is essential for the self-renewal and the survival of human embryonic stem cells.","apaCitation":"Conklin, J. F., Baker, J., &amp; Sage, J. (2012). The RB family is required for the self-renewal and survival of human embryonic stem cells. <i>NATURE COMMUNICATIONS</i>, <i>3</i>.","capCitation":"<span class=\"title\">\n  <span>The RB family is required for the self-renewal and survival of human embryonic stem cells</span>\n  <i>NATURE COMMUNICATIONS</i>\n</span>\n<span class=\"authors\">Conklin, J. F., Baker, J., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2012</span>; <span class=\"volume\">3</span>\n</span>","chicagoCitation":"Conklin, Jamie F., Julie Baker, and Julien Sage. 2012. ?The RB Family Is Required for the Self-Renewal and Survival of Human Embryonic Stem Cells.? <i>NATURE COMMUNICATIONS</i> 3. NATURE PUBLISHING GROUP.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The mechanisms ensuring the long-term self-renewal of human embryonic stem cells are still only partly understood, limiting their use in cellular therapies. Here we found that increased activity of the RB cell cycle inhibitor in human embryonic stem cells induces cell cycle arrest, differentiation and cell death. Conversely, inactivation of the entire RB family (RB, p107 and p130) in human embryonic stem cells triggers G2/M arrest and cell death through functional activation of the p53 pathway and the cell cycle inhibitor p21. Differences in E2F target gene activation upon loss of RB family function between human embryonic stem cells, mouse embryonic stem cells and human fibroblasts underscore key differences in the cell cycle regulatory networks of human embryonic stem cells. Finally, loss of RB family function promotes genomic instability in both human and mouse embryonic stem cells, uncoupling cell cycle defects from chromosomal instability. These experiments indicate that a homeostatic level of RB activity is essential for the self-renewal and the survival of human embryonic stem cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/ncomms2254\">DOI 10.1038/ncomms2254</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000316356700012\">Web of Science ID 000316356700012</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/23212373\">PubMedID 23212373</a></span>\n  </p>","text":"Abstract:\nThe mechanisms ensuring the long-term self-renewal of human embryonic stem cells are still only partly understood, limiting their use in cellular therapies. Here we found that increased activity of the RB cell cycle inhibitor in human embryonic stem cells induces cell cycle arrest, differentiation and cell death. Conversely, inactivation of the entire RB family (RB, p107 and p130) in human embryonic stem cells triggers G2/M arrest and cell death through functional activation of the p53 pathway and the cell cycle inhibitor p21. Differences in E2F target gene activation upon loss of RB family function between human embryonic stem cells, mouse embryonic stem cells and human fibroblasts underscore key differences in the cell cycle regulatory networks of human embryonic stem cells. Finally, loss of RB family function promotes genomic instability in both human and mouse embryonic stem cells, uncoupling cell cycle defects from chromosomal instability. These experiments indicate that a homeostatic level of RB activity is essential for the self-renewal and the survival of human embryonic stem cells.\nDOI: 10.1038/ncomms2254, https://dx.doi.org/10.1038/ncomms2254\nWeb of Science ID: 000316356700012, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000316356700012\nPubMedID: 23212373, https://www.ncbi.nlm.nih.gov/pubmed/23212373"},"doiId":"10.1038/ncomms2254","doiUrl":"https://dx.doi.org/10.1038/ncomms2254","featured":true,"firstPublished":{"html":"Dec 1, 2012 00:00:00 AM","text":"Dec 1, 2012 00:00:00 AM","value":"2012-12-01T00:00:00.000-08:00"},"mlaCitation":"Conklin, Jamie F., Julie Baker, and Julien Sage. ?The RB Family Is Required for the Self-Renewal and Survival of Human Embryonic Stem Cells.? <i>NATURE COMMUNICATIONS</i> 3 (2012): n. pag. Print.","pubMedId":"23212373","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/23212373","publicationId":127456,"title":"The RB family is required for the self-renewal and survival of human embryonic stem cells","type":"article","webOfScienceId":"000316356700012","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000316356700012"},{"abstract":"Human papillomavirus-16 (HPV-16) is associated etiologically with many human cervical cancers. It encodes 3 oncogenes E5, E6, and E7. Of these oncogenes, E7 has been found to be the dominant driver of cervical cancer in mice. More than 100 cellular proteins have been reported to associate with HPV-16 E7, which is thought to dysregulate the cell cycle in part by binding and inducing the degradation of pRb and its related pocket protein family members, p107 and p130. The ability of E7 to inactivate the pRb family correlates with its ability to induce head and neck cancers in mice. We previously showed that the inactivation of pRb is itself not sufficient to recapitulate the oncogenic properties of E7 in cervical carcinogenesis. In this study, we evaluated mice that were deficient in multiple pocket proteins, including mice that lacked pRb, p107, and p130. Strikingly, combined loss of two or all 3 pocket proteins resulted in development of high-grade cervical intraepithelial neoplasia, but not frank cervical carcinoma. These findings strongly argue that the oncogenic properties of HPV-16 E7 in human cervical carcinogenesis may involve disruption of E7 binding proteins beyond simply the pRb family members.","apaCitation":"Shin, M.-K., Sage, J., &amp; Lambert, P. F. (2012). Inactivating All Three Rb Family Pocket Proteins Is Insufficient to Initiate Cervical Cancer. <i>CANCER RESEARCH</i>, <i>72</i>(20), 5418?5427.","capCitation":"<span class=\"title\">\n  <span>Inactivating All Three Rb Family Pocket Proteins Is Insufficient to Initiate Cervical Cancer</span>\n  <i>CANCER RESEARCH</i>\n</span>\n<span class=\"authors\">Shin, M., Sage, J., Lambert, P. F.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2012</span>; <span class=\"volume\">72 (20)</span><span class=\"pages\">: 5418-5427</span>\n</span>","chicagoCitation":"Shin, Myeong-Kyun, Julien Sage, and Paul F. Lambert. 2012. ?Inactivating All Three Rb Family Pocket Proteins Is Insufficient to Initiate Cervical Cancer.? <i>CANCER RESEARCH</i> 72 (20). AMER ASSOC CANCER RESEARCH: 5418?27.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Human papillomavirus-16 (HPV-16) is associated etiologically with many human cervical cancers. It encodes 3 oncogenes E5, E6, and E7. Of these oncogenes, E7 has been found to be the dominant driver of cervical cancer in mice. More than 100 cellular proteins have been reported to associate with HPV-16 E7, which is thought to dysregulate the cell cycle in part by binding and inducing the degradation of pRb and its related pocket protein family members, p107 and p130. The ability of E7 to inactivate the pRb family correlates with its ability to induce head and neck cancers in mice. We previously showed that the inactivation of pRb is itself not sufficient to recapitulate the oncogenic properties of E7 in cervical carcinogenesis. In this study, we evaluated mice that were deficient in multiple pocket proteins, including mice that lacked pRb, p107, and p130. Strikingly, combined loss of two or all 3 pocket proteins resulted in development of high-grade cervical intraepithelial neoplasia, but not frank cervical carcinoma. These findings strongly argue that the oncogenic properties of HPV-16 E7 in human cervical carcinogenesis may involve disruption of E7 binding proteins beyond simply the pRb family members.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1158/0008-5472.CAN-12-2083\">DOI 10.1158/0008-5472.CAN-12-2083</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000309972700030\">Web of Science ID 000309972700030</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/22942253\">PubMedID 22942253</a></span>\n  </p>","text":"Abstract:\nHuman papillomavirus-16 (HPV-16) is associated etiologically with many human cervical cancers. It encodes 3 oncogenes E5, E6, and E7. Of these oncogenes, E7 has been found to be the dominant driver of cervical cancer in mice. More than 100 cellular proteins have been reported to associate with HPV-16 E7, which is thought to dysregulate the cell cycle in part by binding and inducing the degradation of pRb and its related pocket protein family members, p107 and p130. The ability of E7 to inactivate the pRb family correlates with its ability to induce head and neck cancers in mice. We previously showed that the inactivation of pRb is itself not sufficient to recapitulate the oncogenic properties of E7 in cervical carcinogenesis. In this study, we evaluated mice that were deficient in multiple pocket proteins, including mice that lacked pRb, p107, and p130. Strikingly, combined loss of two or all 3 pocket proteins resulted in development of high-grade cervical intraepithelial neoplasia, but not frank cervical carcinoma. These findings strongly argue that the oncogenic properties of HPV-16 E7 in human cervical carcinogenesis may involve disruption of E7 binding proteins beyond simply the pRb family members.\nDOI: 10.1158/0008-5472.CAN-12-2083, https://dx.doi.org/10.1158/0008-5472.CAN-12-2083\nWeb of Science ID: 000309972700030, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000309972700030\nPubMedID: 22942253, https://www.ncbi.nlm.nih.gov/pubmed/22942253"},"doiId":"10.1158/0008-5472.CAN-12-2083","doiUrl":"https://dx.doi.org/10.1158/0008-5472.CAN-12-2083","featured":false,"firstPublished":{"html":"Oct 15, 2012 00:00:00 AM","text":"Oct 15, 2012 00:00:00 AM","value":"2012-10-15T00:00:00.000-07:00"},"mlaCitation":"Shin, Myeong-Kyun, Julien Sage, and Paul F. Lambert. ?Inactivating All Three Rb Family Pocket Proteins Is Insufficient to Initiate Cervical Cancer.? <i>CANCER RESEARCH</i> 72.20 (2012): 5418?5427. Print.","pubMedId":"22942253","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/22942253","publicationId":242868,"title":"Inactivating All Three Rb Family Pocket Proteins Is Insufficient to Initiate Cervical Cancer","type":"article","webOfScienceId":"000309972700030","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000309972700030"},{"abstract":"Small-cell lung cancer (SCLC) is an aggressive lung tumor subtype with poor prognosis. We sequenced 29 SCLC exomes, 2 genomes and 15 transcriptomes and found an extremely high mutation rate of 7.4Â±1 protein-changing mutations per million base pairs. Therefore, we conducted integrated analyses of the various data sets to identify pathogenetically relevant mutated genes. In all cases, we found evidence for inactivation of TP53 and RB1 and identified recurrent mutations in the CREBBP, EP300 and MLL genes that encode histone modifiers. Furthermore, we observed mutations in PTEN, SLIT2 and EPHA7, as well as focal amplifications of the FGFR1 tyrosine kinase gene. Finally, we detected many of the alterations found in humans in SCLC tumors from Tp53 and Rb1 double knockout mice. Our study implicates histone modification as a major feature of SCLC, reveals potentially therapeutically tractable genomic alterations and provides a generalizable framework for the identification of biologically relevant genes in the context of high mutational background.","apaCitation":"Peifer, M., Fernandez-Cuesta, L., Sos, M. L., George, J., Seidel, D., Kasper, L. H., ? Thomas, R. K. (2012). Integrative genome analyses identify key somatic driver mutations of small-cell lung cancer. <i>NATURE GENETICS</i>, <i>44</i>(10), 1104-?","capCitation":"<span class=\"title\">\n  <span>Integrative genome analyses identify key somatic driver mutations of small-cell lung cancer</span>\n  <i>NATURE GENETICS</i>\n</span>\n<span class=\"authors\">Peifer, M., Fernandez-Cuesta, L., Sos, M. L., George, J., Seidel, D., Kasper, L. H., Plenker, D., Leenders, F., Sun, R., Zander, T., Menon, R., Koker, M., Dahmen, I., Mueller, C., Di Cerbo, V., Schildhaus, H., Altmueller, J., Baessmann, I., Becker, C., De Wilde, B., Vandesompele, J., Boehm, D., Ansen, S., Gabler, F., Wilkening, I., Heynck, S., Heuckmann, J. M., Lu, X., Carter, S. L., Cibulskis, K., Banerji, S., Getz, G., Park, K., Rauh, D., Gruetter, C., Fischer, M., Pasqualucci, L., Wright, G., Wainer, Z., Russell, P., Petersen, I., Chen, Y., Stoelben, E., Ludwig, C., Schnabel, P., Hoffmann, H., Muley, T., Brockmann, M., Engel-Riedel, W., Muscarella, L. A., Fazio, V. M., Groen, H., Timens, W., Sietsma, H., Thunnissen, E., Smit, E., Heideman, D. A., Snijders, P. J., Cappuzzo, F., Ligorio, C., Damiani, S., Field, J., Solberg, S., Brustugun, O. T., Lund-Iversen, M., Saenger, J., Clement, J. H., Soltermann, A., Moch, H., Weder, W., Solomon, B., Soria, J., Validire, P., Besse, B., Brambilla, E., Brambilla, C., Lantuejoul, S., Lorimier, P., Schneider, P. M., Hallek, M., Pao, W., Meyerson, M., Sage, J., Shendure, J., Schneider, R., Buettner, R., Wolf, J., Nuernberg, P., Perner, S., Heukamp, L. C., Brindle, P. K., Haas, S., Thomas, R. K.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2012</span>; <span class=\"volume\">44 (10)</span><span class=\"pages\">: 1104-?</span>\n</span>","chicagoCitation":"Peifer, Martin, Lynnette Fernandez-Cuesta, Martin L. Sos, Julie George, Danila Seidel, et al. 2012. ?Integrative Genome Analyses Identify Key Somatic Driver Mutations of Small-Cell Lung Cancer.? <i>NATURE GENETICS</i> 44 (10). NATURE PUBLISHING GROUP: 1104-?","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small-cell lung cancer (SCLC) is an aggressive lung tumor subtype with poor prognosis. We sequenced 29 SCLC exomes, 2 genomes and 15 transcriptomes and found an extremely high mutation rate of 7.4Â±1 protein-changing mutations per million base pairs. Therefore, we conducted integrated analyses of the various data sets to identify pathogenetically relevant mutated genes. In all cases, we found evidence for inactivation of TP53 and RB1 and identified recurrent mutations in the CREBBP, EP300 and MLL genes that encode histone modifiers. Furthermore, we observed mutations in PTEN, SLIT2 and EPHA7, as well as focal amplifications of the FGFR1 tyrosine kinase gene. Finally, we detected many of the alterations found in humans in SCLC tumors from Tp53 and Rb1 double knockout mice. Our study implicates histone modification as a major feature of SCLC, reveals potentially therapeutically tractable genomic alterations and provides a generalizable framework for the identification of biologically relevant genes in the context of high mutational background.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/ng.2396\">DOI 10.1038/ng.2396</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000309550200009\">Web of Science ID 000309550200009</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/22941188\">PubMedID 22941188</a></span>\n  </p>","text":"Abstract:\nSmall-cell lung cancer (SCLC) is an aggressive lung tumor subtype with poor prognosis. We sequenced 29 SCLC exomes, 2 genomes and 15 transcriptomes and found an extremely high mutation rate of 7.4Â±1 protein-changing mutations per million base pairs. Therefore, we conducted integrated analyses of the various data sets to identify pathogenetically relevant mutated genes. In all cases, we found evidence for inactivation of TP53 and RB1 and identified recurrent mutations in the CREBBP, EP300 and MLL genes that encode histone modifiers. Furthermore, we observed mutations in PTEN, SLIT2 and EPHA7, as well as focal amplifications of the FGFR1 tyrosine kinase gene. Finally, we detected many of the alterations found in humans in SCLC tumors from Tp53 and Rb1 double knockout mice. Our study implicates histone modification as a major feature of SCLC, reveals potentially therapeutically tractable genomic alterations and provides a generalizable framework for the identification of biologically relevant genes in the context of high mutational background.\nDOI: 10.1038/ng.2396, https://dx.doi.org/10.1038/ng.2396\nWeb of Science ID: 000309550200009, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000309550200009\nPubMedID: 22941188, https://www.ncbi.nlm.nih.gov/pubmed/22941188"},"doiId":"10.1038/ng.2396","doiUrl":"https://dx.doi.org/10.1038/ng.2396","featured":true,"firstPublished":{"html":"Oct 1, 2012 00:00:00 AM","text":"Oct 1, 2012 00:00:00 AM","value":"2012-10-01T00:00:00.000-07:00"},"mlaCitation":"Peifer, Martin et al. ?Integrative Genome Analyses Identify Key Somatic Driver Mutations of Small-Cell Lung Cancer.? <i>NATURE GENETICS</i> 44.10 (2012): 1104-? Print.","pubMedId":"22941188","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/22941188","publicationId":187872,"title":"Integrative genome analyses identify key somatic driver mutations of small-cell lung cancer","type":"article","webOfScienceId":"000309550200009","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000309550200009"},{"abstract":"Stem cells play a critical role during embryonic development and in the maintenance of homeostasis in adult individuals. A better understanding of stem cell biology, including embryonic and adult stem cells, will allow the scientific community to better comprehend a number of pathologies and possibly design novel approaches to treat patients with a variety of diseases. The retinoblastoma tumor suppressor RB controls the proliferation, differentiation, and survival of cells, and accumulating evidence points to a central role for RB activity in the biology of stem and progenitor cells. In some contexts, loss of RB function in stem or progenitor cells is a key event in the initiation of cancer and determines the subtype of cancer arising from these pluripotent cells by altering their fate. In other cases, RB inactivation is often not sufficient to initiate cancer but may still lead to some stem cell expansion, raising the possibility that strategies aimed at transiently inactivating RB might provide a novel way to expand functional stem cell populations. Future experiments dedicated to better understanding how RB and the RB pathway control a stem cell's decisions to divide, self-renew, or give rise to differentiated progeny may eventually increase our capacity to control these decisions to enhance regeneration or help prevent cancer development.","apaCitation":"Sage, J. (2012). The retinoblastoma tumor suppressor and stem cell biology. <i>GENES &amp; DEVELOPMENT</i>, <i>26</i>(13), 1409?1420.","capCitation":"<span class=\"title\">\n  <span>The retinoblastoma tumor suppressor and stem cell biology</span>\n  <i>GENES &amp; DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2012</span>; <span class=\"volume\">26 (13)</span><span class=\"pages\">: 1409-1420</span>\n</span>","chicagoCitation":"Sage, Julien. 2012. ?The Retinoblastoma Tumor Suppressor and Stem Cell Biology.? <i>GENES &amp; DEVELOPMENT</i> 26 (13). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 1409?20.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Stem cells play a critical role during embryonic development and in the maintenance of homeostasis in adult individuals. A better understanding of stem cell biology, including embryonic and adult stem cells, will allow the scientific community to better comprehend a number of pathologies and possibly design novel approaches to treat patients with a variety of diseases. The retinoblastoma tumor suppressor RB controls the proliferation, differentiation, and survival of cells, and accumulating evidence points to a central role for RB activity in the biology of stem and progenitor cells. In some contexts, loss of RB function in stem or progenitor cells is a key event in the initiation of cancer and determines the subtype of cancer arising from these pluripotent cells by altering their fate. In other cases, RB inactivation is often not sufficient to initiate cancer but may still lead to some stem cell expansion, raising the possibility that strategies aimed at transiently inactivating RB might provide a novel way to expand functional stem cell populations. Future experiments dedicated to better understanding how RB and the RB pathway control a stem cell's decisions to divide, self-renew, or give rise to differentiated progeny may eventually increase our capacity to control these decisions to enhance regeneration or help prevent cancer development.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.193730.112\">DOI 10.1101/gad.193730.112</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000306175400002\">Web of Science ID 000306175400002</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/22751497\">PubMedID 22751497</a></span>\n  </p>","text":"Abstract:\nStem cells play a critical role during embryonic development and in the maintenance of homeostasis in adult individuals. A better understanding of stem cell biology, including embryonic and adult stem cells, will allow the scientific community to better comprehend a number of pathologies and possibly design novel approaches to treat patients with a variety of diseases. The retinoblastoma tumor suppressor RB controls the proliferation, differentiation, and survival of cells, and accumulating evidence points to a central role for RB activity in the biology of stem and progenitor cells. In some contexts, loss of RB function in stem or progenitor cells is a key event in the initiation of cancer and determines the subtype of cancer arising from these pluripotent cells by altering their fate. In other cases, RB inactivation is often not sufficient to initiate cancer but may still lead to some stem cell expansion, raising the possibility that strategies aimed at transiently inactivating RB might provide a novel way to expand functional stem cell populations. Future experiments dedicated to better understanding how RB and the RB pathway control a stem cell's decisions to divide, self-renew, or give rise to differentiated progeny may eventually increase our capacity to control these decisions to enhance regeneration or help prevent cancer development.\nDOI: 10.1101/gad.193730.112, https://dx.doi.org/10.1101/gad.193730.112\nWeb of Science ID: 000306175400002, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000306175400002\nPubMedID: 22751497, https://www.ncbi.nlm.nih.gov/pubmed/22751497"},"doiId":"10.1101/gad.193730.112","doiUrl":"https://dx.doi.org/10.1101/gad.193730.112","featured":true,"firstPublished":{"html":"Jul 1, 2012 00:00:00 AM","text":"Jul 1, 2012 00:00:00 AM","value":"2012-07-01T00:00:00.000-07:00"},"mlaCitation":"Sage, Julien. ?The Retinoblastoma Tumor Suppressor and Stem Cell Biology.? <i>GENES &amp; DEVELOPMENT</i> 26.13 (2012): 1409?1420. Print.","pubMedId":"22751497","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/22751497","publicationId":122469,"title":"The retinoblastoma tumor suppressor and stem cell biology","type":"article","webOfScienceId":"000306175400002","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000306175400002"},{"abstract":"Smyd3 is a lysine methyltransferase implicated in chromatin and cancer regulation. Here we show that Smyd3 catalyzes histone H4 methylation at lysine 5 (H4K5me). This novel histone methylation mark is detected in diverse cell types and its formation is attenuated by depletion of Smyd3 protein. Further, Smyd3-driven cancer cell phenotypes require its enzymatic activity. Thus, Smyd3, via H4K5 methylation, provides a potential new link between chromatin dynamics and neoplastic disease.","apaCitation":"Van Aller, G. S., Reynoird, N., Barbash, O., Huddleston, M., Liu, S., Zmoos, A.-F., ? Kruger, R. G. (2012). Smyd3 regulates cancer cell phenotypes and catalyzes histone H4 lysine 5 methylation. <i>EPIGENETICS</i>, <i>7</i>(4), 340?343.","capCitation":"<span class=\"title\">\n  <span>Smyd3 regulates cancer cell phenotypes and catalyzes histone H4 lysine 5 methylation</span>\n  <i>EPIGENETICS</i>\n</span>\n<span class=\"authors\">Van Aller, G. S., Reynoird, N., Barbash, O., Huddleston, M., Liu, S., Zmoos, A., McDevitt, P., Sinnamon, R., Le, B., Mas, G., Annan, R., Sage, J., Garcia, B. A., Tummino, P. J., Gozani, O., Kruger, R. G.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2012</span>; <span class=\"volume\">7 (4)</span><span class=\"pages\">: 340-343</span>\n</span>","chicagoCitation":"Van Aller, Glenn S., Nicolas Reynoird, Olena Barbash, Michael Huddleston, Shichong Liu, et al. 2012. ?Smyd3 Regulates Cancer Cell Phenotypes and Catalyzes Histone H4 Lysine 5 Methylation.? <i>EPIGENETICS</i> 7 (4). LANDES BIOSCIENCE: 340?43.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Smyd3 is a lysine methyltransferase implicated in chromatin and cancer regulation. Here we show that Smyd3 catalyzes histone H4 methylation at lysine 5 (H4K5me). This novel histone methylation mark is detected in diverse cell types and its formation is attenuated by depletion of Smyd3 protein. Further, Smyd3-driven cancer cell phenotypes require its enzymatic activity. Thus, Smyd3, via H4K5 methylation, provides a potential new link between chromatin dynamics and neoplastic disease.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000302493300004\">Web of Science ID 000302493300004</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/22419068\">PubMedID 22419068</a></span>\n  </p>","text":"Abstract:\nSmyd3 is a lysine methyltransferase implicated in chromatin and cancer regulation. Here we show that Smyd3 catalyzes histone H4 methylation at lysine 5 (H4K5me). This novel histone methylation mark is detected in diverse cell types and its formation is attenuated by depletion of Smyd3 protein. Further, Smyd3-driven cancer cell phenotypes require its enzymatic activity. Thus, Smyd3, via H4K5 methylation, provides a potential new link between chromatin dynamics and neoplastic disease.\n\nWeb of Science ID: 000302493300004, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000302493300004\nPubMedID: 22419068, https://www.ncbi.nlm.nih.gov/pubmed/22419068"},"featured":true,"firstPublished":{"html":"Apr 1, 2012 00:00:00 AM","text":"Apr 1, 2012 00:00:00 AM","value":"2012-04-01T00:00:00.000-07:00"},"mlaCitation":"Van Aller, Glenn S. et al. ?Smyd3 Regulates Cancer Cell Phenotypes and Catalyzes Histone H4 Lysine 5 Methylation.? <i>EPIGENETICS</i> 7.4 (2012): 340?343. Print.","pubMedId":"22419068","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/22419068","publicationId":118651,"title":"Smyd3 regulates cancer cell phenotypes and catalyzes histone H4 lysine 5 methylation","type":"article","webOfScienceId":"000302493300004","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000302493300004"},{"apaCitation":"Sage, J., &amp; Cleary, M. L. (2012). GENOMICS The path to retinoblastoma. <i>NATURE</i>, <i>481</i>(7381), 269?270.","capCitation":"<span class=\"title\">\n  <span>GENOMICS The path to retinoblastoma</span>\n  <i>NATURE</i>\n</span>\n<span class=\"authors\">Sage, J., Cleary, M. L.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2012</span>; <span class=\"volume\">481 (7381)</span><span class=\"pages\">: 269-270</span>\n</span>","chicagoCitation":"Sage, Julien, and Michael L. Cleary. 2012. ?GENOMICS The Path to Retinoblastoma.? <i>NATURE</i> 481 (7381). NATURE PUBLISHING GROUP: 269?70.","detail":{"html":"<p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000299210600023\">Web of Science ID 000299210600023</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/22258599\">PubMedID 22258599</a></span>\n  </p>","text":"Web of Science ID: 000299210600023, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000299210600023\nPubMedID: 22258599, https://www.ncbi.nlm.nih.gov/pubmed/22258599"},"featured":false,"firstPublished":{"html":"Jan 19, 2012 00:00:00 AM","text":"Jan 19, 2012 00:00:00 AM","value":"2012-01-19T00:00:00.000-08:00"},"mlaCitation":"Sage, Julien, and Michael L. Cleary. ?GENOMICS The Path to Retinoblastoma.? <i>NATURE</i> 481.7381 (2012): 269?270. Print.","pubMedId":"22258599","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/22258599","publicationId":242869,"title":"GENOMICS The path to retinoblastoma","type":"article","webOfScienceId":"000299210600023","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000299210600023"},{"abstract":"Small-cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer for which there is no effective treatment. Using a mouse model in which deletion of Rb1 and Trp53 in the lung epithelium of adult mice induces SCLC, we found that the Hedgehog signaling pathway is activated in SCLC cells independently of the lung microenvironment. Constitutive activation of the Hedgehog signaling molecule Smoothened (Smo) promoted the clonogenicity of human SCLC in vitro and the initiation and progression of mouse SCLC in vivo. Reciprocally, deletion of Smo in Rb1 and Trp53-mutant lung epithelial cells strongly suppressed SCLC initiation and progression in mice. Furthermore, pharmacological blockade of Hedgehog signaling inhibited the growth of mouse and human SCLC, most notably following chemotherapy. These findings show a crucial cell-intrinsic role for Hedgehog signaling in the development and maintenance of SCLC and identify Hedgehog pathway inhibition as a therapeutic strategy to slow the progression of disease and delay cancer recurrence in individuals with SCLC.","apaCitation":"Park, K.-S., Martelotto, L. G., Peifer, M., Sos, M. L., Karnezis, A. N., Mahjoub, M. R., ? Sage, J. (2011). A crucial requirement for Hedgehog signaling in small cell lung cancer. <i>NATURE MEDICINE</i>, <i>17</i>(11), 1504?U1506.","capCitation":"<span class=\"title\">\n  <span>A crucial requirement for Hedgehog signaling in small cell lung cancer</span>\n  <i>NATURE MEDICINE</i>\n</span>\n<span class=\"authors\">Park, K., Martelotto, L. G., Peifer, M., Sos, M. L., Karnezis, A. N., Mahjoub, M. R., Bernard, K., Conklin, J. F., Szczepny, A., Yuan, J., Guo, R., Ospina, B., Falzon, J., Bennett, S., Brown, T. J., Markovic, A., Devereux, W. L., Ocasio, C. A., Chen, J. K., Stearns, T., Thomas, R. K., Dorsch, M., Buonamici, S., Watkins, D. N., Peacock, C. D., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">17 (11)</span><span class=\"pages\">: 1504-U1506</span>\n</span>","chicagoCitation":"Park, Kwon-Sik, Luciano G. Martelotto, Martin Peifer, Martin L. Sos, Anthony N. Karnezis, et al. 2011. ?A Crucial Requirement for Hedgehog Signaling in Small Cell Lung Cancer.? <i>NATURE MEDICINE</i> 17 (11). NATURE PUBLISHING GROUP: 1504?U1506.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small-cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer for which there is no effective treatment. Using a mouse model in which deletion of Rb1 and Trp53 in the lung epithelium of adult mice induces SCLC, we found that the Hedgehog signaling pathway is activated in SCLC cells independently of the lung microenvironment. Constitutive activation of the Hedgehog signaling molecule Smoothened (Smo) promoted the clonogenicity of human SCLC in vitro and the initiation and progression of mouse SCLC in vivo. Reciprocally, deletion of Smo in Rb1 and Trp53-mutant lung epithelial cells strongly suppressed SCLC initiation and progression in mice. Furthermore, pharmacological blockade of Hedgehog signaling inhibited the growth of mouse and human SCLC, most notably following chemotherapy. These findings show a crucial cell-intrinsic role for Hedgehog signaling in the development and maintenance of SCLC and identify Hedgehog pathway inhibition as a therapeutic strategy to slow the progression of disease and delay cancer recurrence in individuals with SCLC.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nm.2473\">DOI 10.1038/nm.2473</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000296779300043\">Web of Science ID 000296779300043</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21983857\">PubMedID 21983857</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3380617\">PubMedCentralID PMC3380617</a></span>\n  </p>","text":"Abstract:\nSmall-cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer for which there is no effective treatment. Using a mouse model in which deletion of Rb1 and Trp53 in the lung epithelium of adult mice induces SCLC, we found that the Hedgehog signaling pathway is activated in SCLC cells independently of the lung microenvironment. Constitutive activation of the Hedgehog signaling molecule Smoothened (Smo) promoted the clonogenicity of human SCLC in vitro and the initiation and progression of mouse SCLC in vivo. Reciprocally, deletion of Smo in Rb1 and Trp53-mutant lung epithelial cells strongly suppressed SCLC initiation and progression in mice. Furthermore, pharmacological blockade of Hedgehog signaling inhibited the growth of mouse and human SCLC, most notably following chemotherapy. These findings show a crucial cell-intrinsic role for Hedgehog signaling in the development and maintenance of SCLC and identify Hedgehog pathway inhibition as a therapeutic strategy to slow the progression of disease and delay cancer recurrence in individuals with SCLC.\nDOI: 10.1038/nm.2473, https://dx.doi.org/10.1038/nm.2473\nWeb of Science ID: 000296779300043, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000296779300043\nPubMedID: 21983857, https://www.ncbi.nlm.nih.gov/pubmed/21983857\nPubMedCentralID: PMC3380617, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3380617"},"doiId":"10.1038/nm.2473","doiUrl":"https://dx.doi.org/10.1038/nm.2473","featured":true,"firstPublished":{"html":"Nov 1, 2011 00:00:00 AM","text":"Nov 1, 2011 00:00:00 AM","value":"2011-11-01T00:00:00.000-07:00"},"mlaCitation":"Park, Kwon-Sik et al. ?A Crucial Requirement for Hedgehog Signaling in Small Cell Lung Cancer.? <i>NATURE MEDICINE</i> 17.11 (2011): 1504?U1506. Print.","pubMedId":"21983857","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21983857","publicationId":113515,"title":"A crucial requirement for Hedgehog signaling in small cell lung cancer","type":"article","webOfScienceId":"000296779300043","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000296779300043"},{"abstract":"Determining the signalling pathways that direct tissue expansion is a principal goal of regenerative biology. Vigorous pancreatic ?-cell replication in juvenile mice and humans declines with age, and elucidating the basis for this decay may reveal strategies for inducing ?-cell expansion, a long-sought goal for diabetes therapy. Here we show that platelet-derived growth factor receptor (Pdgfr) signalling controls age-dependent ?-cell proliferation in mouse and human pancreatic islets. With age, declining ?-cell Pdgfr levels were accompanied by reductions in ?-cell enhancer of zeste homologue 2 (Ezh2) levels and ?-cell replication. Conditional inactivation of the Pdgfra gene in ?-cells accelerated these changes, preventing mouse neonatal ?-cell expansion and adult ?-cell regeneration. Targeted human PDGFR-? activation in mouse ?-cells stimulated Erk1/2 phosphorylation, leading to Ezh2-dependent expansion of adult ?-cells. Adult human islets lack PDGF signalling competence, but exposure of juvenile human islets to PDGF-AA stimulated ?-cell proliferation. The discovery of a conserved pathway controlling age-dependent ?-cell proliferation indicates new strategies for ?-cell expansion.","apaCitation":"Chen, H., Gu, X., Liu, Y., Wang, J., Wirt, S. E., Bottino, R., ? Kim, S. K. (2011). PDGF signalling controls age-dependent proliferation in pancreatic beta-cells. <i>NATURE</i>, <i>478</i>(7369), 349-?","capCitation":"<span class=\"title\">\n  <span>PDGF signalling controls age-dependent proliferation in pancreatic beta-cells</span>\n  <i>NATURE</i>\n</span>\n<span class=\"authors\">Chen, H., Gu, X., Liu, Y., Wang, J., Wirt, S. E., Bottino, R., Schorle, H., Sage, J., Kim, S. K.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">478 (7369)</span><span class=\"pages\">: 349-?</span>\n</span>","chicagoCitation":"Chen, Hainan, Xueying Gu, Yinghua Liu, Jing Wang, Stacey E. Wirt, et al. 2011. ?PDGF Signalling Controls Age-Dependent Proliferation in Pancreatic Beta-Cells.? <i>NATURE</i> 478 (7369). NATURE PUBLISHING GROUP: 349-?","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Determining the signalling pathways that direct tissue expansion is a principal goal of regenerative biology. Vigorous pancreatic ?-cell replication in juvenile mice and humans declines with age, and elucidating the basis for this decay may reveal strategies for inducing ?-cell expansion, a long-sought goal for diabetes therapy. Here we show that platelet-derived growth factor receptor (Pdgfr) signalling controls age-dependent ?-cell proliferation in mouse and human pancreatic islets. With age, declining ?-cell Pdgfr levels were accompanied by reductions in ?-cell enhancer of zeste homologue 2 (Ezh2) levels and ?-cell replication. Conditional inactivation of the Pdgfra gene in ?-cells accelerated these changes, preventing mouse neonatal ?-cell expansion and adult ?-cell regeneration. Targeted human PDGFR-? activation in mouse ?-cells stimulated Erk1/2 phosphorylation, leading to Ezh2-dependent expansion of adult ?-cells. Adult human islets lack PDGF signalling competence, but exposure of juvenile human islets to PDGF-AA stimulated ?-cell proliferation. The discovery of a conserved pathway controlling age-dependent ?-cell proliferation indicates new strategies for ?-cell expansion.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nature10502\">DOI 10.1038/nature10502</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000296021100038\">Web of Science ID 000296021100038</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21993628\">PubMedID 21993628</a></span>\n  </p>","text":"Abstract:\nDetermining the signalling pathways that direct tissue expansion is a principal goal of regenerative biology. Vigorous pancreatic ?-cell replication in juvenile mice and humans declines with age, and elucidating the basis for this decay may reveal strategies for inducing ?-cell expansion, a long-sought goal for diabetes therapy. Here we show that platelet-derived growth factor receptor (Pdgfr) signalling controls age-dependent ?-cell proliferation in mouse and human pancreatic islets. With age, declining ?-cell Pdgfr levels were accompanied by reductions in ?-cell enhancer of zeste homologue 2 (Ezh2) levels and ?-cell replication. Conditional inactivation of the Pdgfra gene in ?-cells accelerated these changes, preventing mouse neonatal ?-cell expansion and adult ?-cell regeneration. Targeted human PDGFR-? activation in mouse ?-cells stimulated Erk1/2 phosphorylation, leading to Ezh2-dependent expansion of adult ?-cells. Adult human islets lack PDGF signalling competence, but exposure of juvenile human islets to PDGF-AA stimulated ?-cell proliferation. The discovery of a conserved pathway controlling age-dependent ?-cell proliferation indicates new strategies for ?-cell expansion.\nDOI: 10.1038/nature10502, https://dx.doi.org/10.1038/nature10502\nWeb of Science ID: 000296021100038, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000296021100038\nPubMedID: 21993628, https://www.ncbi.nlm.nih.gov/pubmed/21993628"},"doiId":"10.1038/nature10502","doiUrl":"https://dx.doi.org/10.1038/nature10502","featured":false,"firstPublished":{"html":"Oct 20, 2011 00:00:00 AM","text":"Oct 20, 2011 00:00:00 AM","value":"2011-10-20T00:00:00.000-07:00"},"mlaCitation":"Chen, Hainan et al. ?PDGF Signalling Controls Age-Dependent Proliferation in Pancreatic Beta-Cells.? <i>NATURE</i> 478.7369 (2011): 349-? Print.","pubMedId":"21993628","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21993628","publicationId":113634,"title":"PDGF signalling controls age-dependent proliferation in pancreatic beta-cells","type":"article","webOfScienceId":"000296021100038","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000296021100038"},{"apaCitation":"Sirota, M., DUDLEY, J. T., Kim, J., Chiang, A. P., Morgan, A. A., Sweet-Cordero, A., ? Butte, A. J. (2011). Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data (vol 3, 96ra77, 2011). <i>SCIENCE TRANSLATIONAL MEDICINE</i>, <i>3</i>(102).","capCitation":"<span class=\"title\">\n  <span>Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data (vol 3, 96ra77, 2011)</span>\n  <i>SCIENCE TRANSLATIONAL MEDICINE</i>\n</span>\n<span class=\"authors\">Sirota, M., DUDLEY, J. T., Kim, J., Chiang, A. P., Morgan, A. A., Sweet-Cordero, A., Sage, J., Butte, A. J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">3 (102)</span>\n</span>","chicagoCitation":"Sirota, M., J. T. DUDLEY, J. Kim, A. P. Chiang, A. A. Morgan, et al. 2011. ?Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data (Vol 3, 96ra77, 2011).? <i>SCIENCE TRANSLATIONAL MEDICINE</i> 3 (102). AMER ASSOC ADVANCEMENT SCIENCE.","detail":{"html":"<p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1126/scitranslmed.3003215\">DOI 10.1126/scitranslmed.3003215</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000295323200013\">Web of Science ID 000295323200013</a></span>\n  </p>","text":"DOI: 10.1126/scitranslmed.3003215, https://dx.doi.org/10.1126/scitranslmed.3003215\nWeb of Science ID: 000295323200013, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000295323200013"},"doiId":"10.1126/scitranslmed.3003215","doiUrl":"https://dx.doi.org/10.1126/scitranslmed.3003215","featured":false,"firstPublished":{"html":"Sep 28, 2011 00:00:00 AM","text":"Sep 28, 2011 00:00:00 AM","value":"2011-09-28T00:00:00.000-07:00"},"mlaCitation":"Sirota, M. et al. ?Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data (Vol 3, 96ra77, 2011).? <i>SCIENCE TRANSLATIONAL MEDICINE</i> 3.102 (2011): n. pag. Print.","publicationId":242870,"title":"Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data (vol 3, 96ra77, 2011)","type":"article","webOfScienceId":"000295323200013","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000295323200013"},{"abstract":"Hepatocellular carcinoma (HCC) is the third cancer killer worldwide with >600,000 deaths every year. Although the major risk factors are known, therapeutic options in patients remain limited in part because of our incomplete understanding of the cellular and molecular mechanisms influencing HCC development. Evidence indicates that the retinoblastoma (RB) pathway is functionally inactivated in most cases of HCC by genetic, epigenetic, and/or viral mechanisms. To investigate the functional relevance of this observation, we inactivated the RB pathway in the liver of adult mice by deleting the three members of the Rb (Rb1) gene family: Rb, p107, and p130. Rb family triple knockout mice develop liver tumors with histopathological features and gene expression profiles similar to human HCC. In this mouse model, cancer initiation is associated with the specific expansion of populations of liver stem/progenitor cells, indicating that the RB pathway may prevent HCC development by maintaining the quiescence of adult liver progenitor cells. In addition, we show that during tumor progression, activation of the Notch pathway via E2F transcription factors serves as a negative feedback mechanism to slow HCC growth. The level of Notch activity is also able to predict survival of HCC patients, suggesting novel means to diagnose and treat HCC.","apaCitation":"Viatour, P., Ehmer, U., Saddic, L. A., Dorrell, C., Andersen, J. B., Lin, C. W., ? Sage, J. (2011). Notch signaling inhibits hepatocellular carcinoma following inactivation of the RB pathway. <i>JOURNAL OF EXPERIMENTAL MEDICINE</i>, <i>208</i>(10), 1963?1976.","capCitation":"<span class=\"title\">\n  <span>Notch signaling inhibits hepatocellular carcinoma following inactivation of the RB pathway</span>\n  <i>JOURNAL OF EXPERIMENTAL MEDICINE</i>\n</span>\n<span class=\"authors\">Viatour, P., Ehmer, U., Saddic, L. A., Dorrell, C., Andersen, J. B., Lin, C., Zmoos, A., Mazur, P. K., Schaffer, B. E., Ostermeier, A., Vogel, H., Sylvester, K. G., Thorgeirsson, S. S., Grompe, M., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">208 (10)</span><span class=\"pages\">: 1963-1976</span>\n</span>","chicagoCitation":"Viatour, Patrick, Ursula Ehmer, Louis A. Saddic, Craig Dorrell, Jesper B. Andersen, et al. 2011. ?Notch Signaling Inhibits Hepatocellular Carcinoma Following Inactivation of the RB Pathway.? <i>JOURNAL OF EXPERIMENTAL MEDICINE</i> 208 (10). ROCKEFELLER UNIV PRESS: 1963?76.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Hepatocellular carcinoma (HCC) is the third cancer killer worldwide with >600,000 deaths every year. Although the major risk factors are known, therapeutic options in patients remain limited in part because of our incomplete understanding of the cellular and molecular mechanisms influencing HCC development. Evidence indicates that the retinoblastoma (RB) pathway is functionally inactivated in most cases of HCC by genetic, epigenetic, and/or viral mechanisms. To investigate the functional relevance of this observation, we inactivated the RB pathway in the liver of adult mice by deleting the three members of the Rb (Rb1) gene family: Rb, p107, and p130. Rb family triple knockout mice develop liver tumors with histopathological features and gene expression profiles similar to human HCC. In this mouse model, cancer initiation is associated with the specific expansion of populations of liver stem/progenitor cells, indicating that the RB pathway may prevent HCC development by maintaining the quiescence of adult liver progenitor cells. In addition, we show that during tumor progression, activation of the Notch pathway via E2F transcription factors serves as a negative feedback mechanism to slow HCC growth. The level of Notch activity is also able to predict survival of HCC patients, suggesting novel means to diagnose and treat HCC.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1084/jem.20110198\">DOI 10.1084/jem.20110198</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000295318900005\">Web of Science ID 000295318900005</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21875955\">PubMedID 21875955</a></span>\n  </p>","text":"Abstract:\nHepatocellular carcinoma (HCC) is the third cancer killer worldwide with >600,000 deaths every year. Although the major risk factors are known, therapeutic options in patients remain limited in part because of our incomplete understanding of the cellular and molecular mechanisms influencing HCC development. Evidence indicates that the retinoblastoma (RB) pathway is functionally inactivated in most cases of HCC by genetic, epigenetic, and/or viral mechanisms. To investigate the functional relevance of this observation, we inactivated the RB pathway in the liver of adult mice by deleting the three members of the Rb (Rb1) gene family: Rb, p107, and p130. Rb family triple knockout mice develop liver tumors with histopathological features and gene expression profiles similar to human HCC. In this mouse model, cancer initiation is associated with the specific expansion of populations of liver stem/progenitor cells, indicating that the RB pathway may prevent HCC development by maintaining the quiescence of adult liver progenitor cells. In addition, we show that during tumor progression, activation of the Notch pathway via E2F transcription factors serves as a negative feedback mechanism to slow HCC growth. The level of Notch activity is also able to predict survival of HCC patients, suggesting novel means to diagnose and treat HCC.\nDOI: 10.1084/jem.20110198, https://dx.doi.org/10.1084/jem.20110198\nWeb of Science ID: 000295318900005, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000295318900005\nPubMedID: 21875955, https://www.ncbi.nlm.nih.gov/pubmed/21875955"},"doiId":"10.1084/jem.20110198","doiUrl":"https://dx.doi.org/10.1084/jem.20110198","featured":true,"firstPublished":{"html":"Sep 26, 2011 00:00:00 AM","text":"Sep 26, 2011 00:00:00 AM","value":"2011-09-26T00:00:00.000-07:00"},"mlaCitation":"Viatour, Patrick et al. ?Notch Signaling Inhibits Hepatocellular Carcinoma Following Inactivation of the RB Pathway.? <i>JOURNAL OF EXPERIMENTAL MEDICINE</i> 208.10 (2011): 1963?1976. Print.","pubMedId":"21875955","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21875955","publicationId":112237,"title":"Notch signaling inhibits hepatocellular carcinoma following inactivation of the RB pathway","type":"article","webOfScienceId":"000295318900005","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000295318900005"},{"abstract":"We investigated the potential of in-depth quantitative proteomics to reveal plasma protein signatures that reflect lung tumor biology. We compared plasma protein profiles of four mouse models of lung cancer with profiles of models of pancreatic, ovarian, colon, prostate, and breast cancer and two models of inflammation. A protein signature for Titf1/Nkx2-1, a known lineage-survival oncogene in lung cancer, was found in plasmas of mouse models of lung adenocarcinoma. An EGFR signature was found in plasma of an EGFR mutant model, and a distinct plasma signature related to neuroendocrine development was uncovered in the small-cell lung cancer model. We demonstrate relevance to human lung cancer of the protein signatures identified on the basis of mouse models.","apaCitation":"Taguchi, A., Politi, K., Pitteri, S. J., Lockwood, W. W., Faca, V. M., Kelly-Spratt, K., ? Hanash, S. M. (2011). Lung Cancer Signatures in Plasma Based on Proteome Profiling of Mouse Tumor Models. <i>CANCER CELL</i>, <i>20</i>(3), 289?299.","capCitation":"<span class=\"title\">\n  <span>Lung Cancer Signatures in Plasma Based on Proteome Profiling of Mouse Tumor Models</span>\n  <i>CANCER CELL</i>\n</span>\n<span class=\"authors\">Taguchi, A., Politi, K., Pitteri, S. J., Lockwood, W. W., Faca, V. M., Kelly-Spratt, K., Wong, C., Zhang, Q., Chin, A., Park, K., Goodman, G., Gazdar, A. F., Sage, J., Dinulescu, D. M., Kucherlapati, R., DePinho, R. A., Kemp, C. J., Varmus, H. E., Hanash, S. M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">20 (3)</span><span class=\"pages\">: 289-299</span>\n</span>","chicagoCitation":"Taguchi, Ayumu, Katerina Politi, Sharon J. Pitteri, William W. Lockwood, Vitor M. Faca, et al. 2011. ?Lung Cancer Signatures in Plasma Based on Proteome Profiling of Mouse Tumor Models.? <i>CANCER CELL</i> 20 (3). CELL PRESS: 289?99.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">We investigated the potential of in-depth quantitative proteomics to reveal plasma protein signatures that reflect lung tumor biology. We compared plasma protein profiles of four mouse models of lung cancer with profiles of models of pancreatic, ovarian, colon, prostate, and breast cancer and two models of inflammation. A protein signature for Titf1/Nkx2-1, a known lineage-survival oncogene in lung cancer, was found in plasmas of mouse models of lung adenocarcinoma. An EGFR signature was found in plasma of an EGFR mutant model, and a distinct plasma signature related to neuroendocrine development was uncovered in the small-cell lung cancer model. We demonstrate relevance to human lung cancer of the protein signatures identified on the basis of mouse models.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.ccr.2011.08.007\">DOI 10.1016/j.ccr.2011.08.007</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000295205700006\">Web of Science ID 000295205700006</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21907921\">PubMedID 21907921</a></span>\n  </p>","text":"Abstract:\nWe investigated the potential of in-depth quantitative proteomics to reveal plasma protein signatures that reflect lung tumor biology. We compared plasma protein profiles of four mouse models of lung cancer with profiles of models of pancreatic, ovarian, colon, prostate, and breast cancer and two models of inflammation. A protein signature for Titf1/Nkx2-1, a known lineage-survival oncogene in lung cancer, was found in plasmas of mouse models of lung adenocarcinoma. An EGFR signature was found in plasma of an EGFR mutant model, and a distinct plasma signature related to neuroendocrine development was uncovered in the small-cell lung cancer model. We demonstrate relevance to human lung cancer of the protein signatures identified on the basis of mouse models.\nDOI: 10.1016/j.ccr.2011.08.007, https://dx.doi.org/10.1016/j.ccr.2011.08.007\nWeb of Science ID: 000295205700006, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000295205700006\nPubMedID: 21907921, https://www.ncbi.nlm.nih.gov/pubmed/21907921"},"doiId":"10.1016/j.ccr.2011.08.007","doiUrl":"https://dx.doi.org/10.1016/j.ccr.2011.08.007","featured":false,"firstPublished":{"html":"Sep 13, 2011 00:00:00 AM","text":"Sep 13, 2011 00:00:00 AM","value":"2011-09-13T00:00:00.000-07:00"},"mlaCitation":"Taguchi, Ayumu et al. ?Lung Cancer Signatures in Plasma Based on Proteome Profiling of Mouse Tumor Models.? <i>CANCER CELL</i> 20.3 (2011): 289?299. Print.","pubMedId":"21907921","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21907921","publicationId":112607,"title":"Lung Cancer Signatures in Plasma Based on Proteome Profiling of Mouse Tumor Models","type":"article","webOfScienceId":"000295205700006","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000295205700006"},{"abstract":"The retinoblastoma (RB) tumor suppressor belongs to a cellular pathway that plays a crucial role in restricting the G1-S transition of the cell cycle in response to a large number of extracellular and intracellular cues. Research in the last decade has highlighted the complexity of regulatory networks that ensure proper cell cycle progression, and has also identified multiple cellular functions beyond cell cycle regulation for RB and its two family members, p107 and p130. Here we review some of the recent evidence pointing to a role of RB as a molecular adaptor at the crossroads of multiple pathways, ensuring cellular homeostasis in different contexts. In particular, we discuss the pro- and anti-tumorigenic roles of RB during the early stages of cancer, as well as the importance of the RB pathway in stem cells and cell fate decisions.","apaCitation":"Viatour, P., &amp; Sage, J. (2011). Newly identified aspects of tumor suppression by RB. <i>DISEASE MODELS &amp; MECHANISMS</i>, <i>4</i>(5), 581?585.","capCitation":"<span class=\"title\">\n  <span>Newly identified aspects of tumor suppression by RB</span>\n  <i>DISEASE MODELS &amp; MECHANISMS</i>\n</span>\n<span class=\"authors\">Viatour, P., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">4 (5)</span><span class=\"pages\">: 581-585</span>\n</span>","chicagoCitation":"Viatour, Patrick, and Julien Sage. 2011. ?Newly Identified Aspects of Tumor Suppression by RB.? <i>DISEASE MODELS &amp; MECHANISMS</i> 4 (5). COMPANY OF BIOLOGISTS LTD: 581?85.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma (RB) tumor suppressor belongs to a cellular pathway that plays a crucial role in restricting the G1-S transition of the cell cycle in response to a large number of extracellular and intracellular cues. Research in the last decade has highlighted the complexity of regulatory networks that ensure proper cell cycle progression, and has also identified multiple cellular functions beyond cell cycle regulation for RB and its two family members, p107 and p130. Here we review some of the recent evidence pointing to a role of RB as a molecular adaptor at the crossroads of multiple pathways, ensuring cellular homeostasis in different contexts. In particular, we discuss the pro- and anti-tumorigenic roles of RB during the early stages of cancer, as well as the importance of the RB pathway in stem cells and cell fate decisions.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1242/dmm.008060\">DOI 10.1242/dmm.008060</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000294411800005\">Web of Science ID 000294411800005</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21878458\">PubMedID 21878458</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma (RB) tumor suppressor belongs to a cellular pathway that plays a crucial role in restricting the G1-S transition of the cell cycle in response to a large number of extracellular and intracellular cues. Research in the last decade has highlighted the complexity of regulatory networks that ensure proper cell cycle progression, and has also identified multiple cellular functions beyond cell cycle regulation for RB and its two family members, p107 and p130. Here we review some of the recent evidence pointing to a role of RB as a molecular adaptor at the crossroads of multiple pathways, ensuring cellular homeostasis in different contexts. In particular, we discuss the pro- and anti-tumorigenic roles of RB during the early stages of cancer, as well as the importance of the RB pathway in stem cells and cell fate decisions.\nDOI: 10.1242/dmm.008060, https://dx.doi.org/10.1242/dmm.008060\nWeb of Science ID: 000294411800005, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000294411800005\nPubMedID: 21878458, https://www.ncbi.nlm.nih.gov/pubmed/21878458"},"doiId":"10.1242/dmm.008060","doiUrl":"https://dx.doi.org/10.1242/dmm.008060","featured":false,"firstPublished":{"html":"Sep 1, 2011 00:00:00 AM","text":"Sep 1, 2011 00:00:00 AM","value":"2011-09-01T00:00:00.000-07:00"},"mlaCitation":"Viatour, Patrick, and Julien Sage. ?Newly Identified Aspects of Tumor Suppression by RB.? <i>DISEASE MODELS &amp; MECHANISMS</i> 4.5 (2011): 581?585. Print.","pubMedId":"21878458","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21878458","publicationId":112259,"title":"Newly identified aspects of tumor suppression by RB","type":"article","webOfScienceId":"000294411800005","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000294411800005"},{"abstract":"The application of established drug compounds to new therapeutic indications, known as drug repositioning, offers several advantages over traditional drug development, including reduced development costs and shorter paths to approval. Recent approaches to drug repositioning use high-throughput experimental approaches to assess a compound's potential therapeutic qualities. Here, we present a systematic computational approach to predict novel therapeutic indications on the basis of comprehensive testing of molecular signatures in drug-disease pairs. We integrated gene expression measurements from 100 diseases and gene expression measurements on 164 drug compounds, yielding predicted therapeutic potentials for these drugs. We recovered many known drug and disease relationships using computationally derived therapeutic potentials and also predict many new indications for these 164 drugs. We experimentally validated a prediction for the antiulcer drug cimetidine as a candidate therapeutic in the treatment of lung adenocarcinoma, and demonstrate its efficacy both in vitro and in vivo using mouse xenograft models. This computational method provides a systematic approach for repositioning established drugs to treat a wide range of human diseases.","apaCitation":"Sirota, M., Dudley, J. T., Kim, J., Chiang, A. P., Morgan, A. A., Sweet-Cordero, A., ? Butte, A. J. (2011). Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data. <i>SCIENCE TRANSLATIONAL MEDICINE</i>, <i>3</i>(96).","capCitation":"<span class=\"title\">\n  <span>Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data</span>\n  <i>SCIENCE TRANSLATIONAL MEDICINE</i>\n</span>\n<span class=\"authors\">Sirota, M., Dudley, J. T., Kim, J., Chiang, A. P., Morgan, A. A., Sweet-Cordero, A., Sage, J., Butte, A. J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">3 (96)</span>\n</span>","chicagoCitation":"Sirota, Marina, Joel T. Dudley, Jeewon Kim, Annie P. Chiang, Alex A. Morgan, et al. 2011. ?Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data.? <i>SCIENCE TRANSLATIONAL MEDICINE</i> 3 (96). AMER ASSOC ADVANCEMENT SCIENCE.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The application of established drug compounds to new therapeutic indications, known as drug repositioning, offers several advantages over traditional drug development, including reduced development costs and shorter paths to approval. Recent approaches to drug repositioning use high-throughput experimental approaches to assess a compound's potential therapeutic qualities. Here, we present a systematic computational approach to predict novel therapeutic indications on the basis of comprehensive testing of molecular signatures in drug-disease pairs. We integrated gene expression measurements from 100 diseases and gene expression measurements on 164 drug compounds, yielding predicted therapeutic potentials for these drugs. We recovered many known drug and disease relationships using computationally derived therapeutic potentials and also predict many new indications for these 164 drugs. We experimentally validated a prediction for the antiulcer drug cimetidine as a candidate therapeutic in the treatment of lung adenocarcinoma, and demonstrate its efficacy both in vitro and in vivo using mouse xenograft models. This computational method provides a systematic approach for repositioning established drugs to treat a wide range of human diseases.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1126/scitranslmed.3001318\">DOI 10.1126/scitranslmed.3001318</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000293953100005\">Web of Science ID 000293953100005</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21849665\">PubMedID 21849665</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3502016\">PubMedCentralID PMC3502016</a></span>\n  </p>","text":"Abstract:\nThe application of established drug compounds to new therapeutic indications, known as drug repositioning, offers several advantages over traditional drug development, including reduced development costs and shorter paths to approval. Recent approaches to drug repositioning use high-throughput experimental approaches to assess a compound's potential therapeutic qualities. Here, we present a systematic computational approach to predict novel therapeutic indications on the basis of comprehensive testing of molecular signatures in drug-disease pairs. We integrated gene expression measurements from 100 diseases and gene expression measurements on 164 drug compounds, yielding predicted therapeutic potentials for these drugs. We recovered many known drug and disease relationships using computationally derived therapeutic potentials and also predict many new indications for these 164 drugs. We experimentally validated a prediction for the antiulcer drug cimetidine as a candidate therapeutic in the treatment of lung adenocarcinoma, and demonstrate its efficacy both in vitro and in vivo using mouse xenograft models. This computational method provides a systematic approach for repositioning established drugs to treat a wide range of human diseases.\nDOI: 10.1126/scitranslmed.3001318, https://dx.doi.org/10.1126/scitranslmed.3001318\nWeb of Science ID: 000293953100005, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000293953100005\nPubMedID: 21849665, https://www.ncbi.nlm.nih.gov/pubmed/21849665\nPubMedCentralID: PMC3502016, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3502016"},"doiId":"10.1126/scitranslmed.3001318","doiUrl":"https://dx.doi.org/10.1126/scitranslmed.3001318","featured":false,"firstPublished":{"html":"Aug 17, 2011 00:00:00 AM","text":"Aug 17, 2011 00:00:00 AM","value":"2011-08-17T00:00:00.000-07:00"},"mlaCitation":"Sirota, Marina et al. ?Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data.? <i>SCIENCE TRANSLATIONAL MEDICINE</i> 3.96 (2011): n. pag. Print.","pubMedId":"21849665","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21849665","publicationId":111881,"title":"Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data","type":"article","webOfScienceId":"000293953100005","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000293953100005"},{"abstract":"It is widely believed that the molecular and cellular features of a tumor reflect its cell of origin and can thus provide clues about treatment targets. The retinoblastoma cell of origin has been debated for over a century. Here, we report that human and mouse retinoblastomas have molecular, cellular, and neurochemical features of multiple cell classes, principally amacrine/horizontal interneurons, retinal progenitor cells, and photoreceptors. Importantly, single-cell gene expression array analysis showed that these multiple cell type-specific developmental programs are coexpressed in individual retinoblastoma cells, which creates a progenitor/neuronal hybrid cell. Furthermore, neurotransmitter receptors, transporters, and biosynthetic enzymes are expressed in human retinoblastoma, and targeted disruption of these pathways reduces retinoblastoma growth inÂ vivo and inÂ vitro.","apaCitation":"McEvoy, J., Flores-Otero, J., Zhang, J., Nemeth, K., Brennan, R., Bradley, C., ? Dyer, M. A. (2011). Coexpression of Normally Incompatible Developmental Pathways in Retinoblastoma Genesis. <i>CANCER CELL</i>, <i>20</i>(2), 260?275.","capCitation":"<span class=\"title\">\n  <span>Coexpression of Normally Incompatible Developmental Pathways in Retinoblastoma Genesis</span>\n  <i>CANCER CELL</i>\n</span>\n<span class=\"authors\">McEvoy, J., Flores-Otero, J., Zhang, J., Nemeth, K., Brennan, R., Bradley, C., Krafcik, F., Rodriguez-Galindo, C., Wilson, M., Xiong, S., Lozano, G., Sage, J., Fu, L., Louhibi, L., Trimarchi, J., Pani, A., Smeyne, R., Johnson, D., Dyer, M. A.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">20 (2)</span><span class=\"pages\">: 260-275</span>\n</span>","chicagoCitation":"McEvoy, Justina, Jacqueline Flores-Otero, Jiakun Zhang, Katie Nemeth, Rachel Brennan, et al. 2011. ?Coexpression of Normally Incompatible Developmental Pathways in Retinoblastoma Genesis.? <i>CANCER CELL</i> 20 (2). CELL PRESS: 260?75.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">It is widely believed that the molecular and cellular features of a tumor reflect its cell of origin and can thus provide clues about treatment targets. The retinoblastoma cell of origin has been debated for over a century. Here, we report that human and mouse retinoblastomas have molecular, cellular, and neurochemical features of multiple cell classes, principally amacrine/horizontal interneurons, retinal progenitor cells, and photoreceptors. Importantly, single-cell gene expression array analysis showed that these multiple cell type-specific developmental programs are coexpressed in individual retinoblastoma cells, which creates a progenitor/neuronal hybrid cell. Furthermore, neurotransmitter receptors, transporters, and biosynthetic enzymes are expressed in human retinoblastoma, and targeted disruption of these pathways reduces retinoblastoma growth inÂ vivo and inÂ vitro.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.ccr.2011.07.005\">DOI 10.1016/j.ccr.2011.07.005</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000294099700013\">Web of Science ID 000294099700013</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21840489\">PubMedID 21840489</a></span>\n  </p>","text":"Abstract:\nIt is widely believed that the molecular and cellular features of a tumor reflect its cell of origin and can thus provide clues about treatment targets. The retinoblastoma cell of origin has been debated for over a century. Here, we report that human and mouse retinoblastomas have molecular, cellular, and neurochemical features of multiple cell classes, principally amacrine/horizontal interneurons, retinal progenitor cells, and photoreceptors. Importantly, single-cell gene expression array analysis showed that these multiple cell type-specific developmental programs are coexpressed in individual retinoblastoma cells, which creates a progenitor/neuronal hybrid cell. Furthermore, neurotransmitter receptors, transporters, and biosynthetic enzymes are expressed in human retinoblastoma, and targeted disruption of these pathways reduces retinoblastoma growth inÂ vivo and inÂ vitro.\nDOI: 10.1016/j.ccr.2011.07.005, https://dx.doi.org/10.1016/j.ccr.2011.07.005\nWeb of Science ID: 000294099700013, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000294099700013\nPubMedID: 21840489, https://www.ncbi.nlm.nih.gov/pubmed/21840489"},"doiId":"10.1016/j.ccr.2011.07.005","doiUrl":"https://dx.doi.org/10.1016/j.ccr.2011.07.005","featured":false,"firstPublished":{"html":"Aug 16, 2011 00:00:00 AM","text":"Aug 16, 2011 00:00:00 AM","value":"2011-08-16T00:00:00.000-07:00"},"mlaCitation":"McEvoy, Justina et al. ?Coexpression of Normally Incompatible Developmental Pathways in Retinoblastoma Genesis.? <i>CANCER CELL</i> 20.2 (2011): 260?275. Print.","pubMedId":"21840489","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21840489","publicationId":179727,"title":"Coexpression of Normally Incompatible Developmental Pathways in Retinoblastoma Genesis","type":"article","webOfScienceId":"000294099700013","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000294099700013"},{"abstract":"Retinoblastoma is a rare pediatric cancer that has served as a paradigm to investigate the mechanisms of tumorigenesis. In this issue of Genes & Development, Conkrite and colleagues (pp. 1734-1745) found high levels of the miR-17~92 and miR-106b-25 microRNAs in primary retinoblastomas and show that overexpression of miR-17~92 accelerates retinoblastoma development in mice by promoting proliferation, in part by reducing expression of the cell cycle inhibitor p21. These experiments identify the RB/miR-17~92/p21 axis as a critical regulator of retinoblastoma tumorigenesis and potentially many other cancers.","apaCitation":"Sage, J., &amp; Ventura, A. (2011). miR than meets the eye. <i>GENES &amp; DEVELOPMENT</i>, <i>25</i>(16), 1663?1667.","capCitation":"<span class=\"title\">\n  <span>miR than meets the eye</span>\n  <i>GENES &amp; DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Sage, J., Ventura, A.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">25 (16)</span><span class=\"pages\">: 1663-1667</span>\n</span>","chicagoCitation":"Sage, Julien, and Andrea Ventura. 2011. ?MiR than Meets the Eye.? <i>GENES &amp; DEVELOPMENT</i> 25 (16). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 1663?67.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Retinoblastoma is a rare pediatric cancer that has served as a paradigm to investigate the mechanisms of tumorigenesis. In this issue of Genes & Development, Conkrite and colleagues (pp. 1734-1745) found high levels of the miR-17~92 and miR-106b-25 microRNAs in primary retinoblastomas and show that overexpression of miR-17~92 accelerates retinoblastoma development in mice by promoting proliferation, in part by reducing expression of the cell cycle inhibitor p21. These experiments identify the RB/miR-17~92/p21 axis as a critical regulator of retinoblastoma tumorigenesis and potentially many other cancers.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.17454011\">DOI 10.1101/gad.17454011</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000294003800001\">Web of Science ID 000294003800001</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21852531\">PubMedID 21852531</a></span>\n  </p>","text":"Abstract:\nRetinoblastoma is a rare pediatric cancer that has served as a paradigm to investigate the mechanisms of tumorigenesis. In this issue of Genes & Development, Conkrite and colleagues (pp. 1734-1745) found high levels of the miR-17~92 and miR-106b-25 microRNAs in primary retinoblastomas and show that overexpression of miR-17~92 accelerates retinoblastoma development in mice by promoting proliferation, in part by reducing expression of the cell cycle inhibitor p21. These experiments identify the RB/miR-17~92/p21 axis as a critical regulator of retinoblastoma tumorigenesis and potentially many other cancers.\nDOI: 10.1101/gad.17454011, https://dx.doi.org/10.1101/gad.17454011\nWeb of Science ID: 000294003800001, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000294003800001\nPubMedID: 21852531, https://www.ncbi.nlm.nih.gov/pubmed/21852531"},"doiId":"10.1101/gad.17454011","doiUrl":"https://dx.doi.org/10.1101/gad.17454011","featured":true,"firstPublished":{"html":"Aug 15, 2011 00:00:00 AM","text":"Aug 15, 2011 00:00:00 AM","value":"2011-08-15T00:00:00.000-07:00"},"mlaCitation":"Sage, Julien, and Andrea Ventura. ?MiR than Meets the Eye.? <i>GENES &amp; DEVELOPMENT</i> 25.16 (2011): 1663?1667. Print.","pubMedId":"21852531","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21852531","publicationId":111910,"title":"miR than meets the eye","type":"article","webOfScienceId":"000294003800001","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000294003800001"},{"abstract":"Small cell lung carcinoma (SCLC) is a neuroendocrine subtype of lung cancer that affects more than 200,000 people worldwide every year with a very high mortality rate. Here, we used a mouse genetics approach to characterize the cell of origin for SCLC; in this mouse model, tumors are initiated by the deletion of the Rb and p53 tumor suppressor genes in the lung epithelium of adult mice. We found that mouse SCLCs often arise in the lung epithelium, where neuroendocrine cells are located, and that the majority of early lesions were composed of proliferating neuroendocrine cells. In addition, mice in which Rb and p53 are deleted in a variety of non-neuroendocrine lung epithelial cells did not develop SCLC. These data indicate that SCLC likely arises from neuroendocrine cells in the lung.","apaCitation":"Park, K.-S., Liang, M.-C., Raiser, D. M., Zamponi, R., Roach, R. R., Curtis, S. J., ? Kim, C. F. (2011). Characterization of the cell of origin for small cell lung cancer. <i>CELL CYCLE</i>, <i>10</i>(16), 2806?2815.","capCitation":"<span class=\"title\">\n  <span>Characterization of the cell of origin for small cell lung cancer</span>\n  <i>CELL CYCLE</i>\n</span>\n<span class=\"authors\">Park, K., Liang, M., Raiser, D. M., Zamponi, R., Roach, R. R., Curtis, S. J., Walton, Z., Schaffer, B. E., Roake, C. M., Zmoos, A., Kriegel, C., Wong, K., Sage, J., Kim, C. F.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">10 (16)</span><span class=\"pages\">: 2806-2815</span>\n</span>","chicagoCitation":"Park, Kwon-Sik, Mei-Chih Liang, David M. Raiser, Raffaella Zamponi, Rebecca R. Roach, et al. 2011. ?Characterization of the Cell of Origin for Small Cell Lung Cancer.? <i>CELL CYCLE</i> 10 (16). LANDES BIOSCIENCE: 2806?15.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small cell lung carcinoma (SCLC) is a neuroendocrine subtype of lung cancer that affects more than 200,000 people worldwide every year with a very high mortality rate. Here, we used a mouse genetics approach to characterize the cell of origin for SCLC; in this mouse model, tumors are initiated by the deletion of the Rb and p53 tumor suppressor genes in the lung epithelium of adult mice. We found that mouse SCLCs often arise in the lung epithelium, where neuroendocrine cells are located, and that the majority of early lesions were composed of proliferating neuroendocrine cells. In addition, mice in which Rb and p53 are deleted in a variety of non-neuroendocrine lung epithelial cells did not develop SCLC. These data indicate that SCLC likely arises from neuroendocrine cells in the lung.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.4161/cc.10.16.17012\">DOI 10.4161/cc.10.16.17012</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000294155600035\">Web of Science ID 000294155600035</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21822053\">PubMedID 21822053</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3219544\">PubMedCentralID PMC3219544</a></span>\n  </p>","text":"Abstract:\nSmall cell lung carcinoma (SCLC) is a neuroendocrine subtype of lung cancer that affects more than 200,000 people worldwide every year with a very high mortality rate. Here, we used a mouse genetics approach to characterize the cell of origin for SCLC; in this mouse model, tumors are initiated by the deletion of the Rb and p53 tumor suppressor genes in the lung epithelium of adult mice. We found that mouse SCLCs often arise in the lung epithelium, where neuroendocrine cells are located, and that the majority of early lesions were composed of proliferating neuroendocrine cells. In addition, mice in which Rb and p53 are deleted in a variety of non-neuroendocrine lung epithelial cells did not develop SCLC. These data indicate that SCLC likely arises from neuroendocrine cells in the lung.\nDOI: 10.4161/cc.10.16.17012, https://dx.doi.org/10.4161/cc.10.16.17012\nWeb of Science ID: 000294155600035, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000294155600035\nPubMedID: 21822053, https://www.ncbi.nlm.nih.gov/pubmed/21822053\nPubMedCentralID: PMC3219544, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3219544"},"doiId":"10.4161/cc.10.16.17012","doiUrl":"https://dx.doi.org/10.4161/cc.10.16.17012","featured":false,"firstPublished":{"html":"Aug 15, 2011 00:00:00 AM","text":"Aug 15, 2011 00:00:00 AM","value":"2011-08-15T00:00:00.000-07:00"},"mlaCitation":"Park, Kwon-Sik et al. ?Characterization of the Cell of Origin for Small Cell Lung Cancer.? <i>CELL CYCLE</i> 10.16 (2011): 2806?2815. Print.","pubMedId":"21822053","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21822053","publicationId":111550,"title":"Characterization of the cell of origin for small cell lung cancer","type":"article","webOfScienceId":"000294155600035","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000294155600035"},{"abstract":"Inactivation of the RB tumor suppressor and activation of the MYC family of oncogenes are frequent events in a large spectrum of human cancers. Loss of RB function and MYC activation are thought to control both overlapping and distinct cellular processes during cell cycle progression. However, how these two major cancer genes functionally interact during tumorigenesis is still unclear. Here, we sought to test whether loss of RB function would affect cancer development in a mouse model of c-MYC-induced hepatocellular carcinoma (HCC), a deadly cancer type in which RB is frequently inactivated and c-MYC often activated. We found that RB inactivation has minimal effects on the cell cycle, cell death, and differentiation features of liver tumors driven by increased levels of c-MYC. However, combined loss of RB and activation of c-MYC led to an increase in polyploidy in mature hepatocytes before the development of tumors. There was a trend for decreased survival in double mutant animals compared to mice developing c-MYC-induced tumors. Thus, loss of RB function does not provide a proliferative advantage to c-MYC-expressing HCC cells but the RB and c-MYC pathways may cooperate to control the polyploidy of mature hepatocytes.","apaCitation":"Saddic, L. A., Wirt, S., Vogel, H., Felsher, D. W., &amp; Sage, J. (2011). Functional Interactions between Retinoblastoma and c-MYC in a Mouse Model of Hepatocellular Carcinoma. <i>PLOS ONE</i>, <i>6</i>(5).","capCitation":"<span class=\"title\">\n  <span>Functional Interactions between Retinoblastoma and c-MYC in a Mouse Model of Hepatocellular Carcinoma</span>\n  <i>PLOS ONE</i>\n</span>\n<span class=\"authors\">Saddic, L. A., Wirt, S., Vogel, H., Felsher, D. W., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">6 (5)</span>\n</span>","chicagoCitation":"Saddic, Louis A., Stacey Wirt, Hannes Vogel, Dean W. Felsher, and Julien Sage. 2011. ?Functional Interactions between Retinoblastoma and c-MYC in a Mouse Model of Hepatocellular Carcinoma.? <i>PLOS ONE</i> 6 (5). PUBLIC LIBRARY SCIENCE.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Inactivation of the RB tumor suppressor and activation of the MYC family of oncogenes are frequent events in a large spectrum of human cancers. Loss of RB function and MYC activation are thought to control both overlapping and distinct cellular processes during cell cycle progression. However, how these two major cancer genes functionally interact during tumorigenesis is still unclear. Here, we sought to test whether loss of RB function would affect cancer development in a mouse model of c-MYC-induced hepatocellular carcinoma (HCC), a deadly cancer type in which RB is frequently inactivated and c-MYC often activated. We found that RB inactivation has minimal effects on the cell cycle, cell death, and differentiation features of liver tumors driven by increased levels of c-MYC. However, combined loss of RB and activation of c-MYC led to an increase in polyploidy in mature hepatocytes before the development of tumors. There was a trend for decreased survival in double mutant animals compared to mice developing c-MYC-induced tumors. Thus, loss of RB function does not provide a proliferative advantage to c-MYC-expressing HCC cells but the RB and c-MYC pathways may cooperate to control the polyploidy of mature hepatocytes.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1371/journal.pone.0019758\">DOI 10.1371/journal.pone.0019758</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000290305600045\">Web of Science ID 000290305600045</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21573126\">PubMedID 21573126</a></span>\n  </p>","text":"Abstract:\nInactivation of the RB tumor suppressor and activation of the MYC family of oncogenes are frequent events in a large spectrum of human cancers. Loss of RB function and MYC activation are thought to control both overlapping and distinct cellular processes during cell cycle progression. However, how these two major cancer genes functionally interact during tumorigenesis is still unclear. Here, we sought to test whether loss of RB function would affect cancer development in a mouse model of c-MYC-induced hepatocellular carcinoma (HCC), a deadly cancer type in which RB is frequently inactivated and c-MYC often activated. We found that RB inactivation has minimal effects on the cell cycle, cell death, and differentiation features of liver tumors driven by increased levels of c-MYC. However, combined loss of RB and activation of c-MYC led to an increase in polyploidy in mature hepatocytes before the development of tumors. There was a trend for decreased survival in double mutant animals compared to mice developing c-MYC-induced tumors. Thus, loss of RB function does not provide a proliferative advantage to c-MYC-expressing HCC cells but the RB and c-MYC pathways may cooperate to control the polyploidy of mature hepatocytes.\nDOI: 10.1371/journal.pone.0019758, https://dx.doi.org/10.1371/journal.pone.0019758\nWeb of Science ID: 000290305600045, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000290305600045\nPubMedID: 21573126, https://www.ncbi.nlm.nih.gov/pubmed/21573126"},"doiId":"10.1371/journal.pone.0019758","doiUrl":"https://dx.doi.org/10.1371/journal.pone.0019758","featured":false,"firstPublished":{"html":"May 6, 2011 00:00:00 AM","text":"May 6, 2011 00:00:00 AM","value":"2011-05-06T00:00:00.000-07:00"},"mlaCitation":"Saddic, Louis A. et al. ?Functional Interactions between Retinoblastoma and c-MYC in a Mouse Model of Hepatocellular Carcinoma.? <i>PLOS ONE</i> 6.5 (2011): n. pag. Print.","pubMedId":"21573126","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21573126","publicationId":107867,"title":"Functional Interactions between Retinoblastoma and c-MYC in a Mouse Model of Hepatocellular Carcinoma","type":"article","webOfScienceId":"000290305600045","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000290305600045"},{"abstract":"Emerging evidence suggests that microRNAs (miRNAs), an abundant class of ?22-nucleotide small regulatory RNAs, play key roles in controlling the post-transcriptional genetic programs in stem and progenitor cells. Here we systematically examined miRNA expression profiles in various adult tissue-specific stem cells and their differentiated counterparts. These analyses revealed miRNA programs that are common or unique to blood, muscle, and neural stem cell populations and miRNA signatures that mark the transitions from self-renewing and quiescent stem cells to proliferative and differentiating progenitor cells. Moreover, we identified a stem/progenitor transition miRNA (SPT-miRNA) signature that predicts the effects of genetic perturbations, such as loss of PTEN and the Rb family, AML1-ETO9a expression, and MLL-AF10 transformation, on self-renewal and proliferation potentials of mutant stem/progenitor cells. We showed that some of the SPT-miRNAs control the self-renewal of embryonic stem cells and the reconstitution potential of hematopoietic stem cells (HSCs). Finally, we demonstrated that SPT-miRNAs coordinately regulate genes that are known to play roles in controlling HSC self-renewal, such as Hoxb6 and Hoxa4. Together, these analyses reveal the miRNA programs that may control key processes in normal and aberrant stem and progenitor cells, setting the foundations for dissecting post-transcriptional regulatory networks in stem cells.","apaCitation":"Arnold, C. P., Tan, R., Zhou, B., Yue, S.-B., Schaffert, S., Biggs, J. R., ? Chen, C.-Z. (2011). MicroRNA programs in normal and aberrant stem and progenitor cells. <i>GENOME RESEARCH</i>, <i>21</i>(5), 798?810.","capCitation":"<span class=\"title\">\n  <span>MicroRNA programs in normal and aberrant stem and progenitor cells</span>\n  <i>GENOME RESEARCH</i>\n</span>\n<span class=\"authors\">Arnold, C. P., Tan, R., Zhou, B., Yue, S., Schaffert, S., Biggs, J. R., Doyonnas, R., Lo, M., Perry, J. M., Renault, V. M., Sacco, A., Somervaille, T., Viatour, P., Brunet, A., Cleary, M. L., Li, L., Sage, J., Zhang, D., Blau, H. M., Chen, C., Chen, C.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">21 (5)</span><span class=\"pages\">: 798-810</span>\n</span>","chicagoCitation":"Arnold, Christopher P., Ruoying Tan, Baiyu Zhou, Si-Biao Yue, Steven Schaffert, et al. 2011. ?MicroRNA Programs in Normal and Aberrant Stem and Progenitor Cells.? <i>GENOME RESEARCH</i> 21 (5). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 798?810.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Emerging evidence suggests that microRNAs (miRNAs), an abundant class of ?22-nucleotide small regulatory RNAs, play key roles in controlling the post-transcriptional genetic programs in stem and progenitor cells. Here we systematically examined miRNA expression profiles in various adult tissue-specific stem cells and their differentiated counterparts. These analyses revealed miRNA programs that are common or unique to blood, muscle, and neural stem cell populations and miRNA signatures that mark the transitions from self-renewing and quiescent stem cells to proliferative and differentiating progenitor cells. Moreover, we identified a stem/progenitor transition miRNA (SPT-miRNA) signature that predicts the effects of genetic perturbations, such as loss of PTEN and the Rb family, AML1-ETO9a expression, and MLL-AF10 transformation, on self-renewal and proliferation potentials of mutant stem/progenitor cells. We showed that some of the SPT-miRNAs control the self-renewal of embryonic stem cells and the reconstitution potential of hematopoietic stem cells (HSCs). Finally, we demonstrated that SPT-miRNAs coordinately regulate genes that are known to play roles in controlling HSC self-renewal, such as Hoxb6 and Hoxa4. Together, these analyses reveal the miRNA programs that may control key processes in normal and aberrant stem and progenitor cells, setting the foundations for dissecting post-transcriptional regulatory networks in stem cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gr.111385.110\">DOI 10.1101/gr.111385.110</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000290088000018\">Web of Science ID 000290088000018</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21451113\">PubMedID 21451113</a></span>\n  </p>","text":"Abstract:\nEmerging evidence suggests that microRNAs (miRNAs), an abundant class of ?22-nucleotide small regulatory RNAs, play key roles in controlling the post-transcriptional genetic programs in stem and progenitor cells. Here we systematically examined miRNA expression profiles in various adult tissue-specific stem cells and their differentiated counterparts. These analyses revealed miRNA programs that are common or unique to blood, muscle, and neural stem cell populations and miRNA signatures that mark the transitions from self-renewing and quiescent stem cells to proliferative and differentiating progenitor cells. Moreover, we identified a stem/progenitor transition miRNA (SPT-miRNA) signature that predicts the effects of genetic perturbations, such as loss of PTEN and the Rb family, AML1-ETO9a expression, and MLL-AF10 transformation, on self-renewal and proliferation potentials of mutant stem/progenitor cells. We showed that some of the SPT-miRNAs control the self-renewal of embryonic stem cells and the reconstitution potential of hematopoietic stem cells (HSCs). Finally, we demonstrated that SPT-miRNAs coordinately regulate genes that are known to play roles in controlling HSC self-renewal, such as Hoxb6 and Hoxa4. Together, these analyses reveal the miRNA programs that may control key processes in normal and aberrant stem and progenitor cells, setting the foundations for dissecting post-transcriptional regulatory networks in stem cells.\nDOI: 10.1101/gr.111385.110, https://dx.doi.org/10.1101/gr.111385.110\nWeb of Science ID: 000290088000018, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000290088000018\nPubMedID: 21451113, https://www.ncbi.nlm.nih.gov/pubmed/21451113"},"doiId":"10.1101/gr.111385.110","doiUrl":"https://dx.doi.org/10.1101/gr.111385.110","featured":false,"firstPublished":{"html":"May 1, 2011 00:00:00 AM","text":"May 1, 2011 00:00:00 AM","value":"2011-05-01T00:00:00.000-07:00"},"mlaCitation":"Arnold, Christopher P. et al. ?MicroRNA Programs in Normal and Aberrant Stem and Progenitor Cells.? <i>GENOME RESEARCH</i> 21.5 (2011): 798?810. Print.","pubMedId":"21451113","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21451113","publicationId":105633,"title":"MicroRNA programs in normal and aberrant stem and progenitor cells","type":"article","webOfScienceId":"000290088000018","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000290088000018"},{"abstract":"The integrity of the retinoblastoma tumor suppressor (RB) pathway is critical for restraining inappropriate proliferation and suppressing tumor development in a plethora of tissues. Here adenovirus-mediated RB deletion in the liver of adult mice led to DNA replication in the absence of productive mitotic condensation. The replication induced by RB loss was E2F-mediated and associated with the induction of DNA damage and a nontranscriptional G2/M checkpoint that targeted the accumulation of Cyclin B1. In the context of RB deletion or E2F activation, there was an increase in hepatocyte ploidy that was accompanied by hyperphysiological assembly of prereplication complexes. In keeping with this dysregulation, initiation of DNA replication was readily observed in hepatocytes that were phenotypically in G2/M. Under such conditions, uncoupling of replication initiation from mitotic progression led to altered genome ploidy in the liver. Interestingly, these findings in hepatocytes were not recapitulated in the basally proliferative tissues of the gastrointestinal tract, where RB deletion, while increasing DNA replication, did not lead to a profound uncoupling from mitosis. Combined, these findings demonstrate the critical role of RB in controlling cell-cycle transitions and underscore the importance of intrinsic tissue environments in resultant phenotypes.","apaCitation":"Bourgo, R. J., Ehmer, U., Sage, J., &amp; Knudsen, E. S. (2011). RB deletion disrupts coordination between DNA replication licensing and mitotic entry in vivo. <i>MOLECULAR BIOLOGY OF THE CELL</i>, <i>22</i>(7), 931?939.","capCitation":"<span class=\"title\">\n  <span>RB deletion disrupts coordination between DNA replication licensing and mitotic entry in vivo</span>\n  <i>MOLECULAR BIOLOGY OF THE CELL</i>\n</span>\n<span class=\"authors\">Bourgo, R. J., Ehmer, U., Sage, J., Knudsen, E. S.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">22 (7)</span><span class=\"pages\">: 931-939</span>\n</span>","chicagoCitation":"Bourgo, Ryan J., Ursula Ehmer, Julien Sage, and Erik S. Knudsen. 2011. ?RB Deletion Disrupts Coordination between DNA Replication Licensing and Mitotic Entry in Vivo.? <i>MOLECULAR BIOLOGY OF THE CELL</i> 22 (7). AMER SOC CELL BIOLOGY: 931?39.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The integrity of the retinoblastoma tumor suppressor (RB) pathway is critical for restraining inappropriate proliferation and suppressing tumor development in a plethora of tissues. Here adenovirus-mediated RB deletion in the liver of adult mice led to DNA replication in the absence of productive mitotic condensation. The replication induced by RB loss was E2F-mediated and associated with the induction of DNA damage and a nontranscriptional G2/M checkpoint that targeted the accumulation of Cyclin B1. In the context of RB deletion or E2F activation, there was an increase in hepatocyte ploidy that was accompanied by hyperphysiological assembly of prereplication complexes. In keeping with this dysregulation, initiation of DNA replication was readily observed in hepatocytes that were phenotypically in G2/M. Under such conditions, uncoupling of replication initiation from mitotic progression led to altered genome ploidy in the liver. Interestingly, these findings in hepatocytes were not recapitulated in the basally proliferative tissues of the gastrointestinal tract, where RB deletion, while increasing DNA replication, did not lead to a profound uncoupling from mitosis. Combined, these findings demonstrate the critical role of RB in controlling cell-cycle transitions and underscore the importance of intrinsic tissue environments in resultant phenotypes.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1091/mbc.E10-11-0895\">DOI 10.1091/mbc.E10-11-0895</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000288995800005\">Web of Science ID 000288995800005</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21289097\">PubMedID 21289097</a></span>\n  </p>","text":"Abstract:\nThe integrity of the retinoblastoma tumor suppressor (RB) pathway is critical for restraining inappropriate proliferation and suppressing tumor development in a plethora of tissues. Here adenovirus-mediated RB deletion in the liver of adult mice led to DNA replication in the absence of productive mitotic condensation. The replication induced by RB loss was E2F-mediated and associated with the induction of DNA damage and a nontranscriptional G2/M checkpoint that targeted the accumulation of Cyclin B1. In the context of RB deletion or E2F activation, there was an increase in hepatocyte ploidy that was accompanied by hyperphysiological assembly of prereplication complexes. In keeping with this dysregulation, initiation of DNA replication was readily observed in hepatocytes that were phenotypically in G2/M. Under such conditions, uncoupling of replication initiation from mitotic progression led to altered genome ploidy in the liver. Interestingly, these findings in hepatocytes were not recapitulated in the basally proliferative tissues of the gastrointestinal tract, where RB deletion, while increasing DNA replication, did not lead to a profound uncoupling from mitosis. Combined, these findings demonstrate the critical role of RB in controlling cell-cycle transitions and underscore the importance of intrinsic tissue environments in resultant phenotypes.\nDOI: 10.1091/mbc.E10-11-0895, https://dx.doi.org/10.1091/mbc.E10-11-0895\nWeb of Science ID: 000288995800005, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000288995800005\nPubMedID: 21289097, https://www.ncbi.nlm.nih.gov/pubmed/21289097"},"doiId":"10.1091/mbc.E10-11-0895","doiUrl":"https://dx.doi.org/10.1091/mbc.E10-11-0895","featured":false,"firstPublished":{"html":"Apr 1, 2011 00:00:00 AM","text":"Apr 1, 2011 00:00:00 AM","value":"2011-04-01T00:00:00.000-07:00"},"mlaCitation":"Bourgo, Ryan J. et al. ?RB Deletion Disrupts Coordination between DNA Replication Licensing and Mitotic Entry in Vivo.? <i>MOLECULAR BIOLOGY OF THE CELL</i> 22.7 (2011): 931?939. Print.","pubMedId":"21289097","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21289097","publicationId":102830,"title":"RB deletion disrupts coordination between DNA replication licensing and mitotic entry in vivo","type":"article","webOfScienceId":"000288995800005","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000288995800005"},{"abstract":"The retinoblastoma tumor suppressor (RB) is a central cell cycle regulator and tumor suppressor. RB cellular functions are known to be regulated by a diversity of post-translational modifications such as phosphorylation and acetylation, raising the possibility that RB may also be methylated in cells. Here we demonstrate that RB can be methylated by SMYD2 at lysine 860, a highly conserved and novel site of modification. This methylation event occurs in vitro and in cells, and it is regulated during cell cycle progression, cellular differentiation, and in response to DNA damage. Furthermore, we show that RB monomethylation at lysine 860 provides a direct binding site for the methyl-binding domain of the transcriptional repressor L3MBTL1. These results support the idea that a code of post-translational modifications exists for RB and helps guide its functions in mammalian cells.","apaCitation":"Saddic, L. A., West, L. E., Aslanian, A., Yates, J. R., Rubin, S. M., Gozani, O., &amp; Sage, J. (2010). Methylation of the Retinoblastoma Tumor Suppressor by SMYD2. <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i>, <i>285</i>(48), 37733?37740.","capCitation":"<span class=\"title\">\n  <span>Methylation of the Retinoblastoma Tumor Suppressor by SMYD2</span>\n  <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i>\n</span>\n<span class=\"authors\">Saddic, L. A., West, L. E., Aslanian, A., Yates, J. R., Rubin, S. M., Gozani, O., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">285 (48)</span><span class=\"pages\">: 37733-37740</span>\n</span>","chicagoCitation":"Saddic, Louis A., Lisandra E. West, Aaron Aslanian, John R. Yates, Seth M. Rubin, et al. 2010. ?Methylation of the Retinoblastoma Tumor Suppressor by SMYD2.? <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i> 285 (48). AMER SOC BIOCHEMISTRY MOLECULAR BIOLOGY INC: 37733?40.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma tumor suppressor (RB) is a central cell cycle regulator and tumor suppressor. RB cellular functions are known to be regulated by a diversity of post-translational modifications such as phosphorylation and acetylation, raising the possibility that RB may also be methylated in cells. Here we demonstrate that RB can be methylated by SMYD2 at lysine 860, a highly conserved and novel site of modification. This methylation event occurs in vitro and in cells, and it is regulated during cell cycle progression, cellular differentiation, and in response to DNA damage. Furthermore, we show that RB monomethylation at lysine 860 provides a direct binding site for the methyl-binding domain of the transcriptional repressor L3MBTL1. These results support the idea that a code of post-translational modifications exists for RB and helps guide its functions in mammalian cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1074/jbc.M110.137612\">DOI 10.1074/jbc.M110.137612</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000284424000065\">Web of Science ID 000284424000065</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20870719\">PubMedID 20870719</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma tumor suppressor (RB) is a central cell cycle regulator and tumor suppressor. RB cellular functions are known to be regulated by a diversity of post-translational modifications such as phosphorylation and acetylation, raising the possibility that RB may also be methylated in cells. Here we demonstrate that RB can be methylated by SMYD2 at lysine 860, a highly conserved and novel site of modification. This methylation event occurs in vitro and in cells, and it is regulated during cell cycle progression, cellular differentiation, and in response to DNA damage. Furthermore, we show that RB monomethylation at lysine 860 provides a direct binding site for the methyl-binding domain of the transcriptional repressor L3MBTL1. These results support the idea that a code of post-translational modifications exists for RB and helps guide its functions in mammalian cells.\nDOI: 10.1074/jbc.M110.137612, https://dx.doi.org/10.1074/jbc.M110.137612\nWeb of Science ID: 000284424000065, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000284424000065\nPubMedID: 20870719, https://www.ncbi.nlm.nih.gov/pubmed/20870719"},"doiId":"10.1074/jbc.M110.137612","doiUrl":"https://dx.doi.org/10.1074/jbc.M110.137612","featured":true,"firstPublished":{"html":"Nov 26, 2010 00:00:00 AM","text":"Nov 26, 2010 00:00:00 AM","value":"2010-11-26T00:00:00.000-08:00"},"mlaCitation":"Saddic, Louis A. et al. ?Methylation of the Retinoblastoma Tumor Suppressor by SMYD2.? <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i> 285.48 (2010): 37733?37740. Print.","pubMedId":"20870719","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20870719","publicationId":96868,"title":"Methylation of the Retinoblastoma Tumor Suppressor by SMYD2","type":"article","webOfScienceId":"000284424000065","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000284424000065"},{"abstract":"The ability of progenitor cells to exit the cell cycle is essential for proper embryonic development and homeostasis, but the mechanisms governing cell cycle exit are still not fully understood. Here, we tested the requirement for the retinoblastoma (Rb) protein and its family members p107 and p130 in G0/G1 arrest and differentiation in mammalian cells. We found that Rb family triple knockout (TKO) mouse embryos survive until days 9-11 of gestation. Strikingly, some TKO cells, including in epithelial and neural lineages, are able to exit the cell cycle in G0/G1 and differentiate in teratomas and in culture. This ability of TKO cells to arrest in G0/G1 is associated with the repression of key E2F target genes. Thus, G1 arrest is not always dependent on Rb family members, which illustrates the robustness of cell cycle regulatory networks during differentiation and allows for the identification of candidate pathways to inhibit the expansion of cancer cells with mutations in the Rb pathway.","apaCitation":"Wirt, S. E., Adler, A. S., Gebala, V., Weimann, J. M., Schaffer, B. E., Saddic, L. A., ? Sage, J. (2010). G1 arrest and differentiation can occur independently of Rb family function. <i>JOURNAL OF CELL BIOLOGY</i>, <i>191</i>(4), 809?825.","capCitation":"<span class=\"title\">\n  <span>G1 arrest and differentiation can occur independently of Rb family function</span>\n  <i>JOURNAL OF CELL BIOLOGY</i>\n</span>\n<span class=\"authors\">Wirt, S. E., Adler, A. S., Gebala, V., Weimann, J. M., Schaffer, B. E., Saddic, L. A., Viatour, P., Vogel, H., Chang, H. Y., Meissner, A., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">191 (4)</span><span class=\"pages\">: 809-825</span>\n</span>","chicagoCitation":"Wirt, Stacey E., Adam S. Adler, Veronique Gebala, James M. Weimann, Bethany E. Schaffer, et al. 2010. ?G1 Arrest and Differentiation Can Occur Independently of Rb Family Function.? <i>JOURNAL OF CELL BIOLOGY</i> 191 (4). ROCKEFELLER UNIV PRESS: 809?25.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The ability of progenitor cells to exit the cell cycle is essential for proper embryonic development and homeostasis, but the mechanisms governing cell cycle exit are still not fully understood. Here, we tested the requirement for the retinoblastoma (Rb) protein and its family members p107 and p130 in G0/G1 arrest and differentiation in mammalian cells. We found that Rb family triple knockout (TKO) mouse embryos survive until days 9-11 of gestation. Strikingly, some TKO cells, including in epithelial and neural lineages, are able to exit the cell cycle in G0/G1 and differentiate in teratomas and in culture. This ability of TKO cells to arrest in G0/G1 is associated with the repression of key E2F target genes. Thus, G1 arrest is not always dependent on Rb family members, which illustrates the robustness of cell cycle regulatory networks during differentiation and allows for the identification of candidate pathways to inhibit the expansion of cancer cells with mutations in the Rb pathway.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1083/jcb.201003048\">DOI 10.1083/jcb.201003048</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000284737200014\">Web of Science ID 000284737200014</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21059851\">PubMedID 21059851</a></span>\n  </p>","text":"Abstract:\nThe ability of progenitor cells to exit the cell cycle is essential for proper embryonic development and homeostasis, but the mechanisms governing cell cycle exit are still not fully understood. Here, we tested the requirement for the retinoblastoma (Rb) protein and its family members p107 and p130 in G0/G1 arrest and differentiation in mammalian cells. We found that Rb family triple knockout (TKO) mouse embryos survive until days 9-11 of gestation. Strikingly, some TKO cells, including in epithelial and neural lineages, are able to exit the cell cycle in G0/G1 and differentiate in teratomas and in culture. This ability of TKO cells to arrest in G0/G1 is associated with the repression of key E2F target genes. Thus, G1 arrest is not always dependent on Rb family members, which illustrates the robustness of cell cycle regulatory networks during differentiation and allows for the identification of candidate pathways to inhibit the expansion of cancer cells with mutations in the Rb pathway.\nDOI: 10.1083/jcb.201003048, https://dx.doi.org/10.1083/jcb.201003048\nWeb of Science ID: 000284737200014, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000284737200014\nPubMedID: 21059851, https://www.ncbi.nlm.nih.gov/pubmed/21059851"},"doiId":"10.1083/jcb.201003048","doiUrl":"https://dx.doi.org/10.1083/jcb.201003048","featured":true,"firstPublished":{"html":"Nov 15, 2010 00:00:00 AM","text":"Nov 15, 2010 00:00:00 AM","value":"2010-11-15T00:00:00.000-08:00"},"mlaCitation":"Wirt, Stacey E. et al. ?G1 Arrest and Differentiation Can Occur Independently of Rb Family Function.? <i>JOURNAL OF CELL BIOLOGY</i> 191.4 (2010): 809?825. Print.","pubMedId":"21059851","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21059851","publicationId":98964,"title":"G1 arrest and differentiation can occur independently of Rb family function","type":"article","webOfScienceId":"000284737200014","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000284737200014"},{"abstract":"An outstanding biological question is why tissue regeneration in mammals is limited, whereas urodele amphibians and teleost fish regenerate major structures, largely by cell cycle reentry. Upon inactivation of Rb, proliferation of postmitotic urodele skeletal muscle is induced, whereas in mammalian muscle this mechanism does not exist. We postulated that a tumor suppressor present in mammals but absent in regenerative vertebrates, the Ink4a product ARF (alternative reading frame), is a regeneration suppressor. Concomitant inactivation of Arf and Rb led to mammalian muscle cell cycle reentry, loss of differentiation properties, and upregulation of cytokinetic machinery. Single postmitotic myocytes were isolated by laser micro-dissection-catapulting, and transient suppression of Arf and Rb yielded myoblast colonies that retained the ability to differentiate and fuse into myofibers upon transplantation in vivo. These results show that differentiation of mammalian cells is reversed by inactivation of Arf and Rb and support the hypothesis that Arf evolved at the expense of regeneration.","apaCitation":"Pajcini, K. V., Corbel, S. Y., Sage, J., Pomerantz, J. H., &amp; Blau, H. M. (2010). Transient Inactivation of Rb and ARF Yields Regenerative Cells from Postmitotic Mammalian Muscle. <i>CELL STEM CELL</i>, <i>7</i>(2), 198?213.","capCitation":"<span class=\"title\">\n  <span>Transient Inactivation of Rb and ARF Yields Regenerative Cells from Postmitotic Mammalian Muscle</span>\n  <i>CELL STEM CELL</i>\n</span>\n<span class=\"authors\">Pajcini, K. V., Corbel, S. Y., Sage, J., Pomerantz, J. H., Blau, H. M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">7 (2)</span><span class=\"pages\">: 198-213</span>\n</span>","chicagoCitation":"Pajcini, Kostandin V., Stephane Y. Corbel, Julien Sage, Jason H. Pomerantz, and Helen M. Blau. 2010. ?Transient Inactivation of Rb and ARF Yields Regenerative Cells from Postmitotic Mammalian Muscle.? <i>CELL STEM CELL</i> 7 (2). CELL PRESS: 198?213.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">An outstanding biological question is why tissue regeneration in mammals is limited, whereas urodele amphibians and teleost fish regenerate major structures, largely by cell cycle reentry. Upon inactivation of Rb, proliferation of postmitotic urodele skeletal muscle is induced, whereas in mammalian muscle this mechanism does not exist. We postulated that a tumor suppressor present in mammals but absent in regenerative vertebrates, the Ink4a product ARF (alternative reading frame), is a regeneration suppressor. Concomitant inactivation of Arf and Rb led to mammalian muscle cell cycle reentry, loss of differentiation properties, and upregulation of cytokinetic machinery. Single postmitotic myocytes were isolated by laser micro-dissection-catapulting, and transient suppression of Arf and Rb yielded myoblast colonies that retained the ability to differentiate and fuse into myofibers upon transplantation in vivo. These results show that differentiation of mammalian cells is reversed by inactivation of Arf and Rb and support the hypothesis that Arf evolved at the expense of regeneration.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.stem.2010.05.022\">DOI 10.1016/j.stem.2010.05.022</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000281107400012\">Web of Science ID 000281107400012</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20682446\">PubMedID 20682446</a></span>\n  </p>","text":"Abstract:\nAn outstanding biological question is why tissue regeneration in mammals is limited, whereas urodele amphibians and teleost fish regenerate major structures, largely by cell cycle reentry. Upon inactivation of Rb, proliferation of postmitotic urodele skeletal muscle is induced, whereas in mammalian muscle this mechanism does not exist. We postulated that a tumor suppressor present in mammals but absent in regenerative vertebrates, the Ink4a product ARF (alternative reading frame), is a regeneration suppressor. Concomitant inactivation of Arf and Rb led to mammalian muscle cell cycle reentry, loss of differentiation properties, and upregulation of cytokinetic machinery. Single postmitotic myocytes were isolated by laser micro-dissection-catapulting, and transient suppression of Arf and Rb yielded myoblast colonies that retained the ability to differentiate and fuse into myofibers upon transplantation in vivo. These results show that differentiation of mammalian cells is reversed by inactivation of Arf and Rb and support the hypothesis that Arf evolved at the expense of regeneration.\nDOI: 10.1016/j.stem.2010.05.022, https://dx.doi.org/10.1016/j.stem.2010.05.022\nWeb of Science ID: 000281107400012, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000281107400012\nPubMedID: 20682446, https://www.ncbi.nlm.nih.gov/pubmed/20682446"},"doiId":"10.1016/j.stem.2010.05.022","doiUrl":"https://dx.doi.org/10.1016/j.stem.2010.05.022","featured":false,"firstPublished":{"html":"Aug 6, 2010 00:00:00 AM","text":"Aug 6, 2010 00:00:00 AM","value":"2010-08-06T00:00:00.000-07:00"},"mlaCitation":"Pajcini, Kostandin V. et al. ?Transient Inactivation of Rb and ARF Yields Regenerative Cells from Postmitotic Mammalian Muscle.? <i>CELL STEM CELL</i> 7.2 (2010): 198?213. Print.","pubMedId":"20682446","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20682446","publicationId":94994,"title":"Transient Inactivation of Rb and ARF Yields Regenerative Cells from Postmitotic Mammalian Muscle","type":"article","webOfScienceId":"000281107400012","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000281107400012"},{"abstract":"The retinoblastoma tumor suppressor RB is the downstream mediator of a cellular pathway that is thought to prevent cancer by controlling the ability of cells to enter or exit the cell cycle in G0/G1. Recently, however, accumulating evidence has suggested that RB, its family members p107 and p130, and their partners, the E2F family of transcription factors, may have important cellular functions beyond the G1/S transition of the cell cycle, including during DNA replication and at the transition into mitosis. In this issue of Genes & Development, three studies demonstrate a critical role for RB in proper chromosome condensation, centromeric function, and chromosome stability in mammalian cells, and link these cellular functions of RB to tumor suppression in mice. Here we discuss how transcriptional and post-transcriptional mechanisms under the control of the RB pathway ensure accurate progression through mitosis, thereby preventing cancer development.","apaCitation":"Sage, J., &amp; Straight, A. F. (2010). RB's original CIN? <i>GENES &amp; DEVELOPMENT</i>, <i>24</i>(13), 1329?1333.","capCitation":"<span class=\"title\">\n  <span>RB&#39;s original CIN?</span>\n  <i>GENES &amp; DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Sage, J., Straight, A. F.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">24 (13)</span><span class=\"pages\">: 1329-1333</span>\n</span>","chicagoCitation":"Sage, Julien, and Aaron F. Straight. 2010. ?RB's Original CIN?? <i>GENES &amp; DEVELOPMENT</i> 24 (13). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 1329?33.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma tumor suppressor RB is the downstream mediator of a cellular pathway that is thought to prevent cancer by controlling the ability of cells to enter or exit the cell cycle in G0/G1. Recently, however, accumulating evidence has suggested that RB, its family members p107 and p130, and their partners, the E2F family of transcription factors, may have important cellular functions beyond the G1/S transition of the cell cycle, including during DNA replication and at the transition into mitosis. In this issue of Genes & Development, three studies demonstrate a critical role for RB in proper chromosome condensation, centromeric function, and chromosome stability in mammalian cells, and link these cellular functions of RB to tumor suppression in mice. Here we discuss how transcriptional and post-transcriptional mechanisms under the control of the RB pathway ensure accurate progression through mitosis, thereby preventing cancer development.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.1948010\">DOI 10.1101/gad.1948010</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000279405000001\">Web of Science ID 000279405000001</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20551167\">PubMedID 20551167</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma tumor suppressor RB is the downstream mediator of a cellular pathway that is thought to prevent cancer by controlling the ability of cells to enter or exit the cell cycle in G0/G1. Recently, however, accumulating evidence has suggested that RB, its family members p107 and p130, and their partners, the E2F family of transcription factors, may have important cellular functions beyond the G1/S transition of the cell cycle, including during DNA replication and at the transition into mitosis. In this issue of Genes & Development, three studies demonstrate a critical role for RB in proper chromosome condensation, centromeric function, and chromosome stability in mammalian cells, and link these cellular functions of RB to tumor suppression in mice. Here we discuss how transcriptional and post-transcriptional mechanisms under the control of the RB pathway ensure accurate progression through mitosis, thereby preventing cancer development.\nDOI: 10.1101/gad.1948010, https://dx.doi.org/10.1101/gad.1948010\nWeb of Science ID: 000279405000001, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000279405000001\nPubMedID: 20551167, https://www.ncbi.nlm.nih.gov/pubmed/20551167"},"doiId":"10.1101/gad.1948010","doiUrl":"https://dx.doi.org/10.1101/gad.1948010","featured":false,"firstPublished":{"html":"Jul 1, 2010 00:00:00 AM","text":"Jul 1, 2010 00:00:00 AM","value":"2010-07-01T00:00:00.000-07:00"},"mlaCitation":"Sage, Julien, and Aaron F. Straight. ?RB's Original CIN?? <i>GENES &amp; DEVELOPMENT</i> 24.13 (2010): 1329?1333. Print.","pubMedId":"20551167","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20551167","publicationId":93482,"title":"RB's original CIN?","type":"article","webOfScienceId":"000279405000001","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000279405000001"},{"abstract":"The retinoblastoma tumor suppressor (Rb) is a potent and ubiquitously expressed cell cycle regulator, but patients with a germline Rb mutation develop a very specific tumor spectrum. This surprising observation raises the possibility that mechanisms that compensate for loss of Rb function are present or activated in many cell types. In particular, p107, a protein related to Rb, has been shown to functionally overlap for loss of Rb in several cellular contexts. To investigate the mechanisms underlying this functional redundancy between Rb and p107 in vivo, we used gene targeting in embryonic stem cells to engineer point mutations in two consensus E2F binding sites in the endogenous p107 promoter. Analysis of normal and mutant cells by gene expression and chromatin immunoprecipitation assays showed that members of the Rb and E2F families directly bound these two sites. Furthermore, we found that these two E2F sites controlled both the repression of p107 in quiescent cells and also its activation in cycling cells, as well as in Rb mutant cells. Cell cycle assays further indicated that activation of p107 transcription during S phase through the two E2F binding sites was critical for controlled cell cycle progression, uncovering a specific role for p107 to slow proliferation in mammalian cells. Direct transcriptional repression of p107 by Rb and E2F family members provides a molecular mechanism for a critical negative feedback loop during cell cycle progression and tumorigenesis. These experiments also suggest novel therapeutic strategies to increase the p107 levels in tumor cells.","apaCitation":"Burkhart, D. L., Wirt, S. E., Zmoos, A.-F., Kareta, M. S., &amp; Sage, J. (2010). Tandem E2F Binding Sites in the Promoter of the p107 Cell Cycle Regulator Control p107 Expression and Its Cellular Functions. <i>PLOS GENETICS</i>, <i>6</i>(6).","capCitation":"<span class=\"title\">\n  <span>Tandem E2F Binding Sites in the Promoter of the p107 Cell Cycle Regulator Control p107 Expression and Its Cellular Functions</span>\n  <i>PLOS GENETICS</i>\n</span>\n<span class=\"authors\">Burkhart, D. L., Wirt, S. E., Zmoos, A., Kareta, M. S., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">6 (6)</span>\n</span>","chicagoCitation":"Burkhart, Deborah L., Stacey E. Wirt, Anne-Flore Zmoos, Michael S. Kareta, and Julien Sage. 2010. ?Tandem E2F Binding Sites in the Promoter of the p107 Cell Cycle Regulator Control p107 Expression and Its Cellular Functions.? <i>PLOS GENETICS</i> 6 (6). PUBLIC LIBRARY SCIENCE.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma tumor suppressor (Rb) is a potent and ubiquitously expressed cell cycle regulator, but patients with a germline Rb mutation develop a very specific tumor spectrum. This surprising observation raises the possibility that mechanisms that compensate for loss of Rb function are present or activated in many cell types. In particular, p107, a protein related to Rb, has been shown to functionally overlap for loss of Rb in several cellular contexts. To investigate the mechanisms underlying this functional redundancy between Rb and p107 in vivo, we used gene targeting in embryonic stem cells to engineer point mutations in two consensus E2F binding sites in the endogenous p107 promoter. Analysis of normal and mutant cells by gene expression and chromatin immunoprecipitation assays showed that members of the Rb and E2F families directly bound these two sites. Furthermore, we found that these two E2F sites controlled both the repression of p107 in quiescent cells and also its activation in cycling cells, as well as in Rb mutant cells. Cell cycle assays further indicated that activation of p107 transcription during S phase through the two E2F binding sites was critical for controlled cell cycle progression, uncovering a specific role for p107 to slow proliferation in mammalian cells. Direct transcriptional repression of p107 by Rb and E2F family members provides a molecular mechanism for a critical negative feedback loop during cell cycle progression and tumorigenesis. These experiments also suggest novel therapeutic strategies to increase the p107 levels in tumor cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1371/journal.pgen.1001003\">DOI 10.1371/journal.pgen.1001003</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000279805200032\">Web of Science ID 000279805200032</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20585628\">PubMedID 20585628</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma tumor suppressor (Rb) is a potent and ubiquitously expressed cell cycle regulator, but patients with a germline Rb mutation develop a very specific tumor spectrum. This surprising observation raises the possibility that mechanisms that compensate for loss of Rb function are present or activated in many cell types. In particular, p107, a protein related to Rb, has been shown to functionally overlap for loss of Rb in several cellular contexts. To investigate the mechanisms underlying this functional redundancy between Rb and p107 in vivo, we used gene targeting in embryonic stem cells to engineer point mutations in two consensus E2F binding sites in the endogenous p107 promoter. Analysis of normal and mutant cells by gene expression and chromatin immunoprecipitation assays showed that members of the Rb and E2F families directly bound these two sites. Furthermore, we found that these two E2F sites controlled both the repression of p107 in quiescent cells and also its activation in cycling cells, as well as in Rb mutant cells. Cell cycle assays further indicated that activation of p107 transcription during S phase through the two E2F binding sites was critical for controlled cell cycle progression, uncovering a specific role for p107 to slow proliferation in mammalian cells. Direct transcriptional repression of p107 by Rb and E2F family members provides a molecular mechanism for a critical negative feedback loop during cell cycle progression and tumorigenesis. These experiments also suggest novel therapeutic strategies to increase the p107 levels in tumor cells.\nDOI: 10.1371/journal.pgen.1001003, https://dx.doi.org/10.1371/journal.pgen.1001003\nWeb of Science ID: 000279805200032, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000279805200032\nPubMedID: 20585628, https://www.ncbi.nlm.nih.gov/pubmed/20585628"},"doiId":"10.1371/journal.pgen.1001003","doiUrl":"https://dx.doi.org/10.1371/journal.pgen.1001003","featured":true,"firstPublished":{"html":"Jun 1, 2010 00:00:00 AM","text":"Jun 1, 2010 00:00:00 AM","value":"2010-06-01T00:00:00.000-07:00"},"mlaCitation":"Burkhart, Deborah L. et al. ?Tandem E2F Binding Sites in the Promoter of the p107 Cell Cycle Regulator Control p107 Expression and Its Cellular Functions.? <i>PLOS GENETICS</i> 6.6 (2010): n. pag. Print.","pubMedId":"20585628","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20585628","publicationId":93888,"title":"Tandem E2F Binding Sites in the Promoter of the p107 Cell Cycle Regulator Control p107 Expression and Its Cellular Functions","type":"article","webOfScienceId":"000279805200032","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000279805200032"},{"abstract":"Small-cell lung carcinoma (SCLC) is a neuroendocrine subtype of lung cancer. Although SCLC patients often initially respond to therapy, tumors nearly always recur, resulting in a 5-year survival rate of less than 10%. A mouse model has been developed based on the fact that the RB and p53 tumor suppressor genes are mutated in more than 90% of human SCLCs. Emerging evidence in patients and mouse models suggests that p130, a gene related to RB, may act as a tumor suppressor in SCLC cells. To test this idea, we used conditional mutant mice to delete p130 in combination with Rb and p53 in adult lung epithelial cells. We found that loss of p130 resulted in increased proliferation and significant acceleration of SCLC development in this triple-knockout mouse model. The histopathologic features of the triple-mutant mouse tumors closely resembled that of human SCLC. Genome-wide expression profiling experiments further showed that Rb/p53/p130-mutant mouse tumors were similar to human SCLC. These findings indicate that p130 plays a key tumor suppressor role in SCLC. Rb/p53/p130-mutant mice provide a novel preclinical mouse model to identify novel therapeutic targets against SCLC.","apaCitation":"Schaffer, B. E., Park, K.-S., Yiu, G., Conklin, J. F., Lin, C. W., Burkhart, D. L., ? Sage, J. (2010). Loss of p130 Accelerates Tumor Development in a Mouse Model for Human Small-Cell Lung Carcinoma. <i>CANCER RESEARCH</i>, <i>70</i>(10), 3877?3883.","capCitation":"<span class=\"title\">\n  <span>Loss of p130 Accelerates Tumor Development in a Mouse Model for Human Small-Cell Lung Carcinoma</span>\n  <i>CANCER RESEARCH</i>\n</span>\n<span class=\"authors\">Schaffer, B. E., Park, K., Yiu, G., Conklin, J. F., Lin, C., Burkhart, D. L., Karnezis, A. N., Sweet-Cordero, E. A., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">70 (10)</span><span class=\"pages\">: 3877-3883</span>\n</span>","chicagoCitation":"Schaffer, Bethany E., Kwon-Sik Park, Gloria Yiu, Jamie F. Conklin, ChenWei Lin, et al. 2010. ?Loss of p130 Accelerates Tumor Development in a Mouse Model for Human Small-Cell Lung Carcinoma.? <i>CANCER RESEARCH</i> 70 (10). AMER ASSOC CANCER RESEARCH: 3877?83.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small-cell lung carcinoma (SCLC) is a neuroendocrine subtype of lung cancer. Although SCLC patients often initially respond to therapy, tumors nearly always recur, resulting in a 5-year survival rate of less than 10%. A mouse model has been developed based on the fact that the RB and p53 tumor suppressor genes are mutated in more than 90% of human SCLCs. Emerging evidence in patients and mouse models suggests that p130, a gene related to RB, may act as a tumor suppressor in SCLC cells. To test this idea, we used conditional mutant mice to delete p130 in combination with Rb and p53 in adult lung epithelial cells. We found that loss of p130 resulted in increased proliferation and significant acceleration of SCLC development in this triple-knockout mouse model. The histopathologic features of the triple-mutant mouse tumors closely resembled that of human SCLC. Genome-wide expression profiling experiments further showed that Rb/p53/p130-mutant mouse tumors were similar to human SCLC. These findings indicate that p130 plays a key tumor suppressor role in SCLC. Rb/p53/p130-mutant mice provide a novel preclinical mouse model to identify novel therapeutic targets against SCLC.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1158/0008-5472.CAN-09-4228\">DOI 10.1158/0008-5472.CAN-09-4228</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000278486300004\">Web of Science ID 000278486300004</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20406986\">PubMedID 20406986</a></span>\n  </p>","text":"Abstract:\nSmall-cell lung carcinoma (SCLC) is a neuroendocrine subtype of lung cancer. Although SCLC patients often initially respond to therapy, tumors nearly always recur, resulting in a 5-year survival rate of less than 10%. A mouse model has been developed based on the fact that the RB and p53 tumor suppressor genes are mutated in more than 90% of human SCLCs. Emerging evidence in patients and mouse models suggests that p130, a gene related to RB, may act as a tumor suppressor in SCLC cells. To test this idea, we used conditional mutant mice to delete p130 in combination with Rb and p53 in adult lung epithelial cells. We found that loss of p130 resulted in increased proliferation and significant acceleration of SCLC development in this triple-knockout mouse model. The histopathologic features of the triple-mutant mouse tumors closely resembled that of human SCLC. Genome-wide expression profiling experiments further showed that Rb/p53/p130-mutant mouse tumors were similar to human SCLC. These findings indicate that p130 plays a key tumor suppressor role in SCLC. Rb/p53/p130-mutant mice provide a novel preclinical mouse model to identify novel therapeutic targets against SCLC.\nDOI: 10.1158/0008-5472.CAN-09-4228, https://dx.doi.org/10.1158/0008-5472.CAN-09-4228\nWeb of Science ID: 000278486300004, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000278486300004\nPubMedID: 20406986, https://www.ncbi.nlm.nih.gov/pubmed/20406986"},"doiId":"10.1158/0008-5472.CAN-09-4228","doiUrl":"https://dx.doi.org/10.1158/0008-5472.CAN-09-4228","featured":true,"firstPublished":{"html":"May 15, 2010 00:00:00 AM","text":"May 15, 2010 00:00:00 AM","value":"2010-05-15T00:00:00.000-07:00"},"mlaCitation":"Schaffer, Bethany E. et al. ?Loss of p130 Accelerates Tumor Development in a Mouse Model for Human Small-Cell Lung Carcinoma.? <i>CANCER RESEARCH</i> 70.10 (2010): 3877?3883. Print.","pubMedId":"20406986","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20406986","publicationId":91948,"title":"Loss of p130 Accelerates Tumor Development in a Mouse Model for Human Small-Cell Lung Carcinoma","type":"article","webOfScienceId":"000278486300004","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000278486300004"},{"apaCitation":"Burkhart, D. L., &amp; Sage, J. (2010). Complex transcriptional regulatory networks around the retinoblastoma tumor suppressor gene. <i>CELL CYCLE</i>, <i>9</i>(9), 1663?1664.","capCitation":"<span class=\"title\">\n  <span>Complex transcriptional regulatory networks around the retinoblastoma tumor suppressor gene</span>\n  <i>CELL CYCLE</i>\n</span>\n<span class=\"authors\">Burkhart, D. L., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">9 (9)</span><span class=\"pages\">: 1663-1664</span>\n</span>","chicagoCitation":"Burkhart, Deborah L., and Julien Sage. 2010. ?Complex Transcriptional Regulatory Networks around the Retinoblastoma Tumor Suppressor Gene.? <i>CELL CYCLE</i> 9 (9). LANDES BIOSCIENCE: 1663?64.","detail":{"html":"<p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000277402200002\">Web of Science ID 000277402200002</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20404509\">PubMedID 20404509</a></span>\n  </p>","text":"Web of Science ID: 000277402200002, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000277402200002\nPubMedID: 20404509, https://www.ncbi.nlm.nih.gov/pubmed/20404509"},"featured":false,"firstPublished":{"html":"May 1, 2010 00:00:00 AM","text":"May 1, 2010 00:00:00 AM","value":"2010-05-01T00:00:00.000-07:00"},"mlaCitation":"Burkhart, Deborah L., and Julien Sage. ?Complex Transcriptional Regulatory Networks around the Retinoblastoma Tumor Suppressor Gene.? <i>CELL CYCLE</i> 9.9 (2010): 1663?1664. Print.","pubMedId":"20404509","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20404509","publicationId":91932,"title":"Complex transcriptional regulatory networks around the retinoblastoma tumor suppressor gene","type":"article","webOfScienceId":"000277402200002","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000277402200002"},{"abstract":"p107 and its related family members Rb and p130 are critical regulators of cellular proliferation and tumorigenesis. Due to the extent of functional overlap within the Rb family, it has been difficult to assess which functions are exclusive to individual members and which are shared. Like its family members, p107 can bind a variety of cellular proteins to affect the expression of many target genes during cell cycle progression. Unlike Rb and p130, p107 is most highly expressed during the G1 to S phase transition of the cell cycle in actively dividing cells and accumulating evidence suggests a role for p107 during DNA replication. The specific roles for p107 during differentiation and development are less clear, although emerging studies suggest that it can cooperate with other Rb family members to control differentiation in multiple cell lineages. As a tumor suppressor, p107 is not as potent as Rb, yet studies in knockout mice have revealed some tumor suppressor functions in mice, depending on the context. In this review, we identify the unique and overlapping functions of p107 during the cell cycle, differentiation, and tumorigenesis.","apaCitation":"Wirt, S. E., &amp; Sage, J. (2010). p107 in the public eye: an Rb understudy and more. <i>CELL DIVISION</i>, <i>5</i>.","capCitation":"<span class=\"title\">\n  <span>p107 in the public eye: an Rb understudy and more</span>\n  <i>CELL DIVISION</i>\n</span>\n<span class=\"authors\">Wirt, S. E., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">5</span>\n</span>","chicagoCitation":"Wirt, Stacey E., and Julien Sage. 2010. ?p107 In the Public Eye: an Rb Understudy and More.? <i>CELL DIVISION</i> 5. BIOMED CENTRAL LTD.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">p107 and its related family members Rb and p130 are critical regulators of cellular proliferation and tumorigenesis. Due to the extent of functional overlap within the Rb family, it has been difficult to assess which functions are exclusive to individual members and which are shared. Like its family members, p107 can bind a variety of cellular proteins to affect the expression of many target genes during cell cycle progression. Unlike Rb and p130, p107 is most highly expressed during the G1 to S phase transition of the cell cycle in actively dividing cells and accumulating evidence suggests a role for p107 during DNA replication. The specific roles for p107 during differentiation and development are less clear, although emerging studies suggest that it can cooperate with other Rb family members to control differentiation in multiple cell lineages. As a tumor suppressor, p107 is not as potent as Rb, yet studies in knockout mice have revealed some tumor suppressor functions in mice, depending on the context. In this review, we identify the unique and overlapping functions of p107 during the cell cycle, differentiation, and tumorigenesis.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1186/1747-1028-5-9\">DOI 10.1186/1747-1028-5-9</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000283795500001\">Web of Science ID 000283795500001</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20359370\">PubMedID 20359370</a></span>\n  </p>","text":"Abstract:\np107 and its related family members Rb and p130 are critical regulators of cellular proliferation and tumorigenesis. Due to the extent of functional overlap within the Rb family, it has been difficult to assess which functions are exclusive to individual members and which are shared. Like its family members, p107 can bind a variety of cellular proteins to affect the expression of many target genes during cell cycle progression. Unlike Rb and p130, p107 is most highly expressed during the G1 to S phase transition of the cell cycle in actively dividing cells and accumulating evidence suggests a role for p107 during DNA replication. The specific roles for p107 during differentiation and development are less clear, although emerging studies suggest that it can cooperate with other Rb family members to control differentiation in multiple cell lineages. As a tumor suppressor, p107 is not as potent as Rb, yet studies in knockout mice have revealed some tumor suppressor functions in mice, depending on the context. In this review, we identify the unique and overlapping functions of p107 during the cell cycle, differentiation, and tumorigenesis.\nDOI: 10.1186/1747-1028-5-9, https://dx.doi.org/10.1186/1747-1028-5-9\nWeb of Science ID: 000283795500001, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000283795500001\nPubMedID: 20359370, https://www.ncbi.nlm.nih.gov/pubmed/20359370"},"doiId":"10.1186/1747-1028-5-9","doiUrl":"https://dx.doi.org/10.1186/1747-1028-5-9","featured":false,"firstPublished":{"html":"Apr 2, 2010 00:00:00 AM","text":"Apr 2, 2010 00:00:00 AM","value":"2010-04-02T00:00:00.000-07:00"},"mlaCitation":"Wirt, Stacey E., and Julien Sage. ?p107 In the Public Eye: an Rb Understudy and More.? <i>CELL DIVISION</i> 5 (2010): n. pag. Print.","pubMedId":"20359370","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20359370","publicationId":91257,"title":"p107 in the public eye: an Rb understudy and more","type":"article","webOfScienceId":"000283795500001","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000283795500001"},{"abstract":"In cancer cells, the retinoblastoma tumor suppressor RB is directly inactivated by mutation in the RB gene or functionally inhibited by abnormal activation of cyclin-dependent kinase activity. While variations in RB levels may also provide an important means of controlling RB function in both normal and cancer cells, little is known about the mechanisms regulating RB transcription. Here we show that members of the RB and E2F families bind directly to the RB promoter. To investigate how the RB/E2F pathway may regulate Rb transcription, we generated reporter mice carrying an eGFP transgene inserted into a bacterial artificial chromosome containing most of the Rb gene. Expression of eGFP largely parallels that of Rb in transgenic embryos and adult mice. Using these reporter mice and mutant alleles for Rb, p107, and p130, we found that RB family members modulate Rb transcription in specific cell populations in vivo and in culture. Interestingly, while Rb is a target of the RB/E2F pathway in mouse and human cells, Rb expression does not strictly correlate with the cell cycle status of these cells. These experiments identify novel regulatory feedback mechanisms within the RB pathway in mammalian cells.","apaCitation":"Burkhart, D. L., Ngai, L. K., Roake, C. M., Viatour, P., Thangavel, C., Ho, V. M., ? Sage, J. (2010). Regulation of RB Transcription In Vivo by RB Family Members. <i>MOLECULAR AND CELLULAR BIOLOGY</i>, <i>30</i>(7), 1729?1745.","capCitation":"<span class=\"title\">\n  <span>Regulation of RB Transcription In Vivo by RB Family Members</span>\n  <i>MOLECULAR AND CELLULAR BIOLOGY</i>\n</span>\n<span class=\"authors\">Burkhart, D. L., Ngai, L. K., Roake, C. M., Viatour, P., Thangavel, C., Ho, V. M., Knudsen, E. S., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">30 (7)</span><span class=\"pages\">: 1729-1745</span>\n</span>","chicagoCitation":"Burkhart, Deborah L., Lynn K. Ngai, Caitlin M. Roake, Patrick Viatour, Chellappagounder Thangavel, et al. 2010. ?Regulation of RB Transcription In Vivo by RB Family Members.? <i>MOLECULAR AND CELLULAR BIOLOGY</i> 30 (7). AMER SOC MICROBIOLOGY: 1729?45.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">In cancer cells, the retinoblastoma tumor suppressor RB is directly inactivated by mutation in the RB gene or functionally inhibited by abnormal activation of cyclin-dependent kinase activity. While variations in RB levels may also provide an important means of controlling RB function in both normal and cancer cells, little is known about the mechanisms regulating RB transcription. Here we show that members of the RB and E2F families bind directly to the RB promoter. To investigate how the RB/E2F pathway may regulate Rb transcription, we generated reporter mice carrying an eGFP transgene inserted into a bacterial artificial chromosome containing most of the Rb gene. Expression of eGFP largely parallels that of Rb in transgenic embryos and adult mice. Using these reporter mice and mutant alleles for Rb, p107, and p130, we found that RB family members modulate Rb transcription in specific cell populations in vivo and in culture. Interestingly, while Rb is a target of the RB/E2F pathway in mouse and human cells, Rb expression does not strictly correlate with the cell cycle status of these cells. These experiments identify novel regulatory feedback mechanisms within the RB pathway in mammalian cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1128/MCB.00952-09\">DOI 10.1128/MCB.00952-09</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000275302000013\">Web of Science ID 000275302000013</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20100864\">PubMedID 20100864</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2838078\">PubMedCentralID PMC2838078</a></span>\n  </p>","text":"Abstract:\nIn cancer cells, the retinoblastoma tumor suppressor RB is directly inactivated by mutation in the RB gene or functionally inhibited by abnormal activation of cyclin-dependent kinase activity. While variations in RB levels may also provide an important means of controlling RB function in both normal and cancer cells, little is known about the mechanisms regulating RB transcription. Here we show that members of the RB and E2F families bind directly to the RB promoter. To investigate how the RB/E2F pathway may regulate Rb transcription, we generated reporter mice carrying an eGFP transgene inserted into a bacterial artificial chromosome containing most of the Rb gene. Expression of eGFP largely parallels that of Rb in transgenic embryos and adult mice. Using these reporter mice and mutant alleles for Rb, p107, and p130, we found that RB family members modulate Rb transcription in specific cell populations in vivo and in culture. Interestingly, while Rb is a target of the RB/E2F pathway in mouse and human cells, Rb expression does not strictly correlate with the cell cycle status of these cells. These experiments identify novel regulatory feedback mechanisms within the RB pathway in mammalian cells.\nDOI: 10.1128/MCB.00952-09, https://dx.doi.org/10.1128/MCB.00952-09\nWeb of Science ID: 000275302000013, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000275302000013\nPubMedID: 20100864, https://www.ncbi.nlm.nih.gov/pubmed/20100864\nPubMedCentralID: PMC2838078, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2838078"},"doiId":"10.1128/MCB.00952-09","doiUrl":"https://dx.doi.org/10.1128/MCB.00952-09","featured":false,"firstPublished":{"html":"Apr 1, 2010 00:00:00 AM","text":"Apr 1, 2010 00:00:00 AM","value":"2010-04-01T00:00:00.000-07:00"},"mlaCitation":"Burkhart, Deborah L. et al. ?Regulation of RB Transcription In Vivo by RB Family Members.? <i>MOLECULAR AND CELLULAR BIOLOGY</i> 30.7 (2010): 1729?1745. Print.","pubMedId":"20100864","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20100864","publicationId":89130,"title":"Regulation of RB Transcription In Vivo by RB Family Members","type":"article","webOfScienceId":"000275302000013","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000275302000013"},{"abstract":"Human embryonic stem cells (hESCs) hold great promise in regenerative medicine. However, before the full potential of these cells is achieved, major basic biological questions need to be addressed. In particular, there are still gaps in our knowledge of the molecular mechanisms underlying the derivation of hESCs from blastocysts, the regulation of the undifferentiated, pluripotent state, and the control of differentiation into specific lineages. Furthermore, we still do not fully understand the tumorigenic potential of hESCs, limiting their use in regenerative medicine. The RB pathway is a key signaling module that controls cellular proliferation, cell survival, chromatin structure, and cellular differentiation in mammalian cells. Members of the RB pathway are important regulators of hESC biology and manipulation of the activity of this pathway may provide novel means to control the fate of hESCs. Here we review what is known about the expression and function of members of the RB pathway in hESCs and discuss areas of interest in this field.","apaCitation":"Conklin, J. F., &amp; Sage, J. (2009). Keeping an Eye on Retinoblastoma Control of Human Embryonic Stem Cells. <i>JOURNAL OF CELLULAR BIOCHEMISTRY</i>, <i>108</i>(5), 1023?1030.","capCitation":"<span class=\"title\">\n  <span>Keeping an Eye on Retinoblastoma Control of Human Embryonic Stem Cells</span>\n  <i>JOURNAL OF CELLULAR BIOCHEMISTRY</i>\n</span>\n<span class=\"authors\">Conklin, J. F., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2009</span>; <span class=\"volume\">108 (5)</span><span class=\"pages\">: 1023-1030</span>\n</span>","chicagoCitation":"Conklin, Jamie F., and Julien Sage. 2009. ?Keeping an Eye on Retinoblastoma Control of Human Embryonic Stem Cells.? <i>JOURNAL OF CELLULAR BIOCHEMISTRY</i> 108 (5). WILEY-LISS: 1023?30.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Human embryonic stem cells (hESCs) hold great promise in regenerative medicine. However, before the full potential of these cells is achieved, major basic biological questions need to be addressed. In particular, there are still gaps in our knowledge of the molecular mechanisms underlying the derivation of hESCs from blastocysts, the regulation of the undifferentiated, pluripotent state, and the control of differentiation into specific lineages. Furthermore, we still do not fully understand the tumorigenic potential of hESCs, limiting their use in regenerative medicine. The RB pathway is a key signaling module that controls cellular proliferation, cell survival, chromatin structure, and cellular differentiation in mammalian cells. Members of the RB pathway are important regulators of hESC biology and manipulation of the activity of this pathway may provide novel means to control the fate of hESCs. Here we review what is known about the expression and function of members of the RB pathway in hESCs and discuss areas of interest in this field.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1002/jcb.22342\">DOI 10.1002/jcb.22342</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000272640900001\">Web of Science ID 000272640900001</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/19760644\">PubMedID 19760644</a></span>\n  </p>","text":"Abstract:\nHuman embryonic stem cells (hESCs) hold great promise in regenerative medicine. However, before the full potential of these cells is achieved, major basic biological questions need to be addressed. In particular, there are still gaps in our knowledge of the molecular mechanisms underlying the derivation of hESCs from blastocysts, the regulation of the undifferentiated, pluripotent state, and the control of differentiation into specific lineages. Furthermore, we still do not fully understand the tumorigenic potential of hESCs, limiting their use in regenerative medicine. The RB pathway is a key signaling module that controls cellular proliferation, cell survival, chromatin structure, and cellular differentiation in mammalian cells. Members of the RB pathway are important regulators of hESC biology and manipulation of the activity of this pathway may provide novel means to control the fate of hESCs. Here we review what is known about the expression and function of members of the RB pathway in hESCs and discuss areas of interest in this field.\nDOI: 10.1002/jcb.22342, https://dx.doi.org/10.1002/jcb.22342\nWeb of Science ID: 000272640900001, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000272640900001\nPubMedID: 19760644, https://www.ncbi.nlm.nih.gov/pubmed/19760644"},"doiId":"10.1002/jcb.22342","doiUrl":"https://dx.doi.org/10.1002/jcb.22342","featured":false,"firstPublished":{"html":"Dec 1, 2009 00:00:00 AM","text":"Dec 1, 2009 00:00:00 AM","value":"2009-12-01T00:00:00.000-08:00"},"mlaCitation":"Conklin, Jamie F., and Julien Sage. ?Keeping an Eye on Retinoblastoma Control of Human Embryonic Stem Cells.? <i>JOURNAL OF CELLULAR BIOCHEMISTRY</i> 108.5 (2009): 1023?1030. Print.","pubMedId":"19760644","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/19760644","publicationId":85005,"title":"Keeping an Eye on Retinoblastoma Control of Human Embryonic Stem Cells","type":"article","webOfScienceId":"000272640900001","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000272640900001"},{"abstract":"A-type lamins are intermediate filament proteins that provide a scaffold for protein complexes regulating nuclear structure and function. Mutations in the LMNA gene are linked to a variety of degenerative disorders termed laminopathies, whereas changes in the expression of lamins are associated with tumourigenesis. The molecular pathways affected by alterations of A-type lamins and how they contribute to disease are poorly understood. Here, we show that A-type lamins have a key role in the maintenance of telomere structure, length and function, and in the stabilization of 53BP1, a component of the DNA damage response (DDR) pathway. Loss of A-type lamins alters the nuclear distribution of telomeres and results in telomere shortening, defects in telomeric heterochromatin, and increased genomic instability. In addition, A-type lamins are necessary for the processing of dysfunctional telomeres by non-homologous end joining, putatively through stabilization of 53BP1. This study shows new functions for A-type lamins in the maintenance of genomic integrity, and suggests that alterations of telomere biology and defects in DDR contribute to the pathogenesis of lamin-related diseases.","apaCitation":"Gonzalez-Suarez, I., Redwood, A. B., Perkins, S. M., Vermolen, B., Lichtensztejin, D., Grotsky, D. A., ? Gonzalo, S. (2009). Novel roles for A-type lamins in telomere biology and the DNA damage response pathway. <i>EMBO JOURNAL</i>, <i>28</i>(16), 2414?2427.","capCitation":"<span class=\"title\">\n  <span>Novel roles for A-type lamins in telomere biology and the DNA damage response pathway</span>\n  <i>EMBO JOURNAL</i>\n</span>\n<span class=\"authors\">Gonzalez-Suarez, I., Redwood, A. B., Perkins, S. M., Vermolen, B., Lichtensztejin, D., Grotsky, D. A., Morgado-Palacin, L., Gapud, E. J., Sleckman, B. P., Sullivan, T., Sage, J., Stewart, C. L., Mai, S., Gonzalo, S.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2009</span>; <span class=\"volume\">28 (16)</span><span class=\"pages\">: 2414-2427</span>\n</span>","chicagoCitation":"Gonzalez-Suarez, Ignacio, Abena B. Redwood, Stephanie M. Perkins, Bart Vermolen, Daniel Lichtensztejin, et al. 2009. ?Novel Roles for A-Type Lamins in Telomere Biology and the DNA Damage Response Pathway.? <i>EMBO JOURNAL</i> 28 (16). NATURE PUBLISHING GROUP: 2414?27.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">A-type lamins are intermediate filament proteins that provide a scaffold for protein complexes regulating nuclear structure and function. Mutations in the LMNA gene are linked to a variety of degenerative disorders termed laminopathies, whereas changes in the expression of lamins are associated with tumourigenesis. The molecular pathways affected by alterations of A-type lamins and how they contribute to disease are poorly understood. Here, we show that A-type lamins have a key role in the maintenance of telomere structure, length and function, and in the stabilization of 53BP1, a component of the DNA damage response (DDR) pathway. Loss of A-type lamins alters the nuclear distribution of telomeres and results in telomere shortening, defects in telomeric heterochromatin, and increased genomic instability. In addition, A-type lamins are necessary for the processing of dysfunctional telomeres by non-homologous end joining, putatively through stabilization of 53BP1. This study shows new functions for A-type lamins in the maintenance of genomic integrity, and suggests that alterations of telomere biology and defects in DDR contribute to the pathogenesis of lamin-related diseases.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/emboj.2009.196\">DOI 10.1038/emboj.2009.196</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000269074400010\">Web of Science ID 000269074400010</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/19629036\">PubMedID 19629036</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2735177\">PubMedCentralID PMC2735177</a></span>\n  </p>","text":"Abstract:\nA-type lamins are intermediate filament proteins that provide a scaffold for protein complexes regulating nuclear structure and function. Mutations in the LMNA gene are linked to a variety of degenerative disorders termed laminopathies, whereas changes in the expression of lamins are associated with tumourigenesis. The molecular pathways affected by alterations of A-type lamins and how they contribute to disease are poorly understood. Here, we show that A-type lamins have a key role in the maintenance of telomere structure, length and function, and in the stabilization of 53BP1, a component of the DNA damage response (DDR) pathway. Loss of A-type lamins alters the nuclear distribution of telomeres and results in telomere shortening, defects in telomeric heterochromatin, and increased genomic instability. In addition, A-type lamins are necessary for the processing of dysfunctional telomeres by non-homologous end joining, putatively through stabilization of 53BP1. This study shows new functions for A-type lamins in the maintenance of genomic integrity, and suggests that alterations of telomere biology and defects in DDR contribute to the pathogenesis of lamin-related diseases.\nDOI: 10.1038/emboj.2009.196, https://dx.doi.org/10.1038/emboj.2009.196\nWeb of Science ID: 000269074400010, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000269074400010\nPubMedID: 19629036, https://www.ncbi.nlm.nih.gov/pubmed/19629036\nPubMedCentralID: PMC2735177, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2735177"},"doiId":"10.1038/emboj.2009.196","doiUrl":"https://dx.doi.org/10.1038/emboj.2009.196","featured":false,"firstPublished":{"html":"Aug 19, 2009 00:00:00 AM","text":"Aug 19, 2009 00:00:00 AM","value":"2009-08-19T00:00:00.000-07:00"},"mlaCitation":"Gonzalez-Suarez, Ignacio et al. ?Novel Roles for A-Type Lamins in Telomere Biology and the DNA Damage Response Pathway.? <i>EMBO JOURNAL</i> 28.16 (2009): 2414?2427. Print.","pubMedId":"19629036","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/19629036","publicationId":242871,"title":"Novel roles for A-type lamins in telomere biology and the DNA damage response pathway","type":"article","webOfScienceId":"000269074400010","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000269074400010"},{"abstract":"The retinoblastoma Rb/E2F tumor suppressor pathway plays a major role in the regulation of mammalian cell cycle progression. The pRb protein, along with closely related proteins p107 and p130, exerts its anti-proliferative effects by binding to the E2F family of transcription factors known to regulate essential genes throughout the cell cycle. We sought to investigate the role of the Rb/E2F1 pathway in the lesion recognition step of nucleotide excision repair (NER) in mouse embryonic fibroblasts (MEFs). Rb-/-, p107-/-, p130-/- MEFs repaired both cyclobutane pyrimidine dimers (CPDs) and 6-4 photoproducts (6-4PPs) at higher efficiency than did wildtype cells following UV-C irradiation. The expression of damaged DNA binding gene DDB2 involved in the DNA lesion recognition step was elevated in the Rb family-deficient MEFs. To determine if the enhanced DNA repair in the absence of the Rb gene family is due to the derepression of E2F1, we assayed the ability of E2F1-deficient cells to repair damaged DNA and demonstrated that E2F1-/- MEFs are impaired for the removal of both CPDs and 6-4PPs. Furthermore, wildtype cells induced a higher expression of DDB2 and xeroderma pigmentosum gene XPC transcript levels than did E2F1-/- cells following UV-C irradiation. Using an E2F SiteScan algorithm, we uncovered a putative E2F-responsive element in the XPC promoter upstream of the transcription start site. We showed with chromatin immunoprecipitation assays the binding of E2F1 to the XPC promoter in a UV-dependent manner, suggesting that E2F1 is a transcriptional regulator of XPC. Our study identifies a novel E2F1 gene target and further supports the growing body of evidence that the Rb/E2F1 tumor suppressor pathway is involved in the regulation of the DNA lesion recognition step of nucleotide excision repair.","apaCitation":"Lin, P. S., McPherson, L. A., Chen, A. Y., Sage, J., &amp; Ford, J. M. (2009). The role of the retinoblastoma/E2F1 tumor suppressor pathway in the lesion recognition step of nucleotide excision repair. <i>DNA REPAIR</i>, <i>8</i>(7), 795?802.","capCitation":"<span class=\"title\">\n  <span>The role of the retinoblastoma/E2F1 tumor suppressor pathway in the lesion recognition step of nucleotide excision repair</span>\n  <i>DNA REPAIR</i>\n</span>\n<span class=\"authors\">Lin, P. S., McPherson, L. A., Chen, A. Y., Sage, J., Ford, J. M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2009</span>; <span class=\"volume\">8 (7)</span><span class=\"pages\">: 795-802</span>\n</span>","chicagoCitation":"Lin, Patrick S., Lisa A. McPherson, Aubrey Y. Chen, Julien Sage, and James M. Ford. 2009. ?The Role of the Retinoblastoma/E2F1 Tumor Suppressor Pathway in the Lesion Recognition Step of Nucleotide Excision Repair.? <i>DNA REPAIR</i> 8 (7). ELSEVIER SCIENCE BV: 795?802.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma Rb/E2F tumor suppressor pathway plays a major role in the regulation of mammalian cell cycle progression. The pRb protein, along with closely related proteins p107 and p130, exerts its anti-proliferative effects by binding to the E2F family of transcription factors known to regulate essential genes throughout the cell cycle. We sought to investigate the role of the Rb/E2F1 pathway in the lesion recognition step of nucleotide excision repair (NER) in mouse embryonic fibroblasts (MEFs). Rb-/-, p107-/-, p130-/- MEFs repaired both cyclobutane pyrimidine dimers (CPDs) and 6-4 photoproducts (6-4PPs) at higher efficiency than did wildtype cells following UV-C irradiation. The expression of damaged DNA binding gene DDB2 involved in the DNA lesion recognition step was elevated in the Rb family-deficient MEFs. To determine if the enhanced DNA repair in the absence of the Rb gene family is due to the derepression of E2F1, we assayed the ability of E2F1-deficient cells to repair damaged DNA and demonstrated that E2F1-/- MEFs are impaired for the removal of both CPDs and 6-4PPs. Furthermore, wildtype cells induced a higher expression of DDB2 and xeroderma pigmentosum gene XPC transcript levels than did E2F1-/- cells following UV-C irradiation. Using an E2F SiteScan algorithm, we uncovered a putative E2F-responsive element in the XPC promoter upstream of the transcription start site. We showed with chromatin immunoprecipitation assays the binding of E2F1 to the XPC promoter in a UV-dependent manner, suggesting that E2F1 is a transcriptional regulator of XPC. Our study identifies a novel E2F1 gene target and further supports the growing body of evidence that the Rb/E2F1 tumor suppressor pathway is involved in the regulation of the DNA lesion recognition step of nucleotide excision repair.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.dnarep.2009.03.003\">DOI 10.1016/j.dnarep.2009.03.003</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000268086300003\">Web of Science ID 000268086300003</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/19376752\">PubMedID 19376752</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma Rb/E2F tumor suppressor pathway plays a major role in the regulation of mammalian cell cycle progression. The pRb protein, along with closely related proteins p107 and p130, exerts its anti-proliferative effects by binding to the E2F family of transcription factors known to regulate essential genes throughout the cell cycle. We sought to investigate the role of the Rb/E2F1 pathway in the lesion recognition step of nucleotide excision repair (NER) in mouse embryonic fibroblasts (MEFs). Rb-/-, p107-/-, p130-/- MEFs repaired both cyclobutane pyrimidine dimers (CPDs) and 6-4 photoproducts (6-4PPs) at higher efficiency than did wildtype cells following UV-C irradiation. The expression of damaged DNA binding gene DDB2 involved in the DNA lesion recognition step was elevated in the Rb family-deficient MEFs. To determine if the enhanced DNA repair in the absence of the Rb gene family is due to the derepression of E2F1, we assayed the ability of E2F1-deficient cells to repair damaged DNA and demonstrated that E2F1-/- MEFs are impaired for the removal of both CPDs and 6-4PPs. Furthermore, wildtype cells induced a higher expression of DDB2 and xeroderma pigmentosum gene XPC transcript levels than did E2F1-/- cells following UV-C irradiation. Using an E2F SiteScan algorithm, we uncovered a putative E2F-responsive element in the XPC promoter upstream of the transcription start site. We showed with chromatin immunoprecipitation assays the binding of E2F1 to the XPC promoter in a UV-dependent manner, suggesting that E2F1 is a transcriptional regulator of XPC. Our study identifies a novel E2F1 gene target and further supports the growing body of evidence that the Rb/E2F1 tumor suppressor pathway is involved in the regulation of the DNA lesion recognition step of nucleotide excision repair.\nDOI: 10.1016/j.dnarep.2009.03.003, https://dx.doi.org/10.1016/j.dnarep.2009.03.003\nWeb of Science ID: 000268086300003, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000268086300003\nPubMedID: 19376752, https://www.ncbi.nlm.nih.gov/pubmed/19376752"},"doiId":"10.1016/j.dnarep.2009.03.003","doiUrl":"https://dx.doi.org/10.1016/j.dnarep.2009.03.003","featured":false,"firstPublished":{"html":"Jul 4, 2009 00:00:00 AM","text":"Jul 4, 2009 00:00:00 AM","value":"2009-07-04T00:00:00.000-07:00"},"mlaCitation":"Lin, Patrick S. et al. ?The Role of the Retinoblastoma/E2F1 Tumor Suppressor Pathway in the Lesion Recognition Step of Nucleotide Excision Repair.? <i>DNA REPAIR</i> 8.7 (2009): 795?802. Print.","pubMedId":"19376752","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/19376752","publicationId":80596,"title":"The role of the retinoblastoma/E2F1 tumor suppressor pathway in the lesion recognition step of nucleotide excision repair","type":"article","webOfScienceId":"000268086300003","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000268086300003"},{"abstract":"Mutations of the retinoblastoma tumor suppressor gene RB are frequently observed in human cancers, but rarely in non-small cell lung carcinomas (NSCLCs). Emerging evidence also suggests that the RB-related gene p130 is inactivated in a subset of human NSCLCs. To directly test the specific tumor suppressor roles of RB and p130 in NSCLC, we crossed Rb and p130 conditional mutant mice to mice carrying a conditional oncogenic K-Ras allele. In this model, controlled oncogenic K-Ras activation leads to the development of adenocarcinoma, a major subtype of NSCLC. We found that loss of p130 accelerated the death of mice, providing direct evidence in vivo that p130 is a tumor suppressor gene, albeit a weak one in this context. Loss of Rb increased the efficiency of lung cancer initiation and resulted in the development of high-grade adenocarcinomas and rapid death. Thus, despite the low frequency of RB mutations in human NSCLCs and reports that K-Ras activation and loss of RB function are rarely found in the same human tumors, loss of Rb clearly cooperates with activation of oncogenic K-Ras in lung adenocarcinoma development in mice.","apaCitation":"Ho, V. M., Schaffer, B. E., Karnezis, A. N., Park, K. S., &amp; Sage, J. (2009). The retinoblastoma gene Rb and its family member p130 suppress lung adenocarcinoma induced by oncogenic K-Ras. <i>ONCOGENE</i>, <i>28</i>(10), 1393?1399.","capCitation":"<span class=\"title\">\n  <span>The retinoblastoma gene Rb and its family member p130 suppress lung adenocarcinoma induced by oncogenic K-Ras</span>\n  <i>ONCOGENE</i>\n</span>\n<span class=\"authors\">Ho, V. M., Schaffer, B. E., Karnezis, A. N., Park, K. S., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2009</span>; <span class=\"volume\">28 (10)</span><span class=\"pages\">: 1393-1399</span>\n</span>","chicagoCitation":"Ho, V. M., B. E. Schaffer, A. N. Karnezis, K. S. Park, and J. Sage. 2009. ?The Retinoblastoma Gene Rb and Its Family Member p130 Suppress Lung Adenocarcinoma Induced by Oncogenic K-Ras.? <i>ONCOGENE</i> 28 (10). NATURE PUBLISHING GROUP: 1393?99.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Mutations of the retinoblastoma tumor suppressor gene RB are frequently observed in human cancers, but rarely in non-small cell lung carcinomas (NSCLCs). Emerging evidence also suggests that the RB-related gene p130 is inactivated in a subset of human NSCLCs. To directly test the specific tumor suppressor roles of RB and p130 in NSCLC, we crossed Rb and p130 conditional mutant mice to mice carrying a conditional oncogenic K-Ras allele. In this model, controlled oncogenic K-Ras activation leads to the development of adenocarcinoma, a major subtype of NSCLC. We found that loss of p130 accelerated the death of mice, providing direct evidence in vivo that p130 is a tumor suppressor gene, albeit a weak one in this context. Loss of Rb increased the efficiency of lung cancer initiation and resulted in the development of high-grade adenocarcinomas and rapid death. Thus, despite the low frequency of RB mutations in human NSCLCs and reports that K-Ras activation and loss of RB function are rarely found in the same human tumors, loss of Rb clearly cooperates with activation of oncogenic K-Ras in lung adenocarcinoma development in mice.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/onc.2008.491\">DOI 10.1038/onc.2008.491</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000264116000012\">Web of Science ID 000264116000012</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/19151761\">PubMedID 19151761</a></span>\n  </p>","text":"Abstract:\nMutations of the retinoblastoma tumor suppressor gene RB are frequently observed in human cancers, but rarely in non-small cell lung carcinomas (NSCLCs). Emerging evidence also suggests that the RB-related gene p130 is inactivated in a subset of human NSCLCs. To directly test the specific tumor suppressor roles of RB and p130 in NSCLC, we crossed Rb and p130 conditional mutant mice to mice carrying a conditional oncogenic K-Ras allele. In this model, controlled oncogenic K-Ras activation leads to the development of adenocarcinoma, a major subtype of NSCLC. We found that loss of p130 accelerated the death of mice, providing direct evidence in vivo that p130 is a tumor suppressor gene, albeit a weak one in this context. Loss of Rb increased the efficiency of lung cancer initiation and resulted in the development of high-grade adenocarcinomas and rapid death. Thus, despite the low frequency of RB mutations in human NSCLCs and reports that K-Ras activation and loss of RB function are rarely found in the same human tumors, loss of Rb clearly cooperates with activation of oncogenic K-Ras in lung adenocarcinoma development in mice.\nDOI: 10.1038/onc.2008.491, https://dx.doi.org/10.1038/onc.2008.491\nWeb of Science ID: 000264116000012, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000264116000012\nPubMedID: 19151761, https://www.ncbi.nlm.nih.gov/pubmed/19151761"},"doiId":"10.1038/onc.2008.491","doiUrl":"https://dx.doi.org/10.1038/onc.2008.491","featured":true,"firstPublished":{"html":"Mar 1, 2009 00:00:00 AM","text":"Mar 1, 2009 00:00:00 AM","value":"2009-03-01T00:00:00.000-08:00"},"mlaCitation":"Ho, V. M. et al. ?The Retinoblastoma Gene Rb and Its Family Member p130 Suppress Lung Adenocarcinoma Induced by Oncogenic K-Ras.? <i>ONCOGENE</i> 28.10 (2009): 1393?1399. Print.","pubMedId":"19151761","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/19151761","publicationId":77918,"title":"The retinoblastoma gene Rb and its family member p130 suppress lung adenocarcinoma induced by oncogenic K-Ras","type":"article","webOfScienceId":"000264116000012","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000264116000012"},{"abstract":"Individual members of the retinoblastoma (Rb) tumor suppressor gene family serve critical roles in the control of cellular proliferation and differentiation, but the extent of their contributions is masked by redundant and compensatory mechanisms. Here we employed a conditional knockout strategy to simultaneously inactivate all three members, Rb, p107, and p130, in adult hematopoietic stem cells (HSCs). Rb family triple knockout (TKO) mice develop a cell-intrinsic myeloproliferation that originates from hyperproliferative early hematopoietic progenitors and is accompanied by increased apoptosis in lymphoid progenitor populations. Loss of quiescence in the TKO HSC pool is associated with an expansion of these mutant stem cells but also with an enhanced mobilization and an impaired reconstitution potential upon transplantation. The presence of a single p107 allele is sufficient to largely rescue these defects. Thus, Rb family members collectively maintain HSC quiescence and the balance between lymphoid and myeloid cell fates in the hematopoietic system.","apaCitation":"Viatour, P., Somervaille, T. C., Venkatasubrahmanyam, S., Kogan, S., McLaughlin, M. E., Weissman, I. L., ? Sage, J. (2008). Hematopoietic Stem Cell Quiescence Is Maintained by Compound Contributions of the Retinoblastoma Gene Family. <i>CELL STEM CELL</i>, <i>3</i>(4), 416?428.","capCitation":"<span class=\"title\">\n  <span>Hematopoietic Stem Cell Quiescence Is Maintained by Compound Contributions of the Retinoblastoma Gene Family</span>\n  <i>CELL STEM CELL</i>\n</span>\n<span class=\"authors\">Viatour, P., Somervaille, T. C., Venkatasubrahmanyam, S., Kogan, S., McLaughlin, M. E., Weissman, I. L., Butte, A. J., Passegue, E., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2008</span>; <span class=\"volume\">3 (4)</span><span class=\"pages\">: 416-428</span>\n</span>","chicagoCitation":"Viatour, Patrick, Tim C. Somervaille, Shivkumar Venkatasubrahmanyam, Scott Kogan, Margaret E. McLaughlin, et al. 2008. ?Hematopoietic Stem Cell Quiescence Is Maintained by Compound Contributions of the Retinoblastoma Gene Family.? <i>CELL STEM CELL</i> 3 (4). CELL PRESS: 416?28.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Individual members of the retinoblastoma (Rb) tumor suppressor gene family serve critical roles in the control of cellular proliferation and differentiation, but the extent of their contributions is masked by redundant and compensatory mechanisms. Here we employed a conditional knockout strategy to simultaneously inactivate all three members, Rb, p107, and p130, in adult hematopoietic stem cells (HSCs). Rb family triple knockout (TKO) mice develop a cell-intrinsic myeloproliferation that originates from hyperproliferative early hematopoietic progenitors and is accompanied by increased apoptosis in lymphoid progenitor populations. Loss of quiescence in the TKO HSC pool is associated with an expansion of these mutant stem cells but also with an enhanced mobilization and an impaired reconstitution potential upon transplantation. The presence of a single p107 allele is sufficient to largely rescue these defects. Thus, Rb family members collectively maintain HSC quiescence and the balance between lymphoid and myeloid cell fates in the hematopoietic system.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.stem.2008.07.009\">DOI 10.1016/j.stem.2008.07.009</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000260149800012\">Web of Science ID 000260149800012</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/18940733\">PubMedID 18940733</a></span>\n  </p>","text":"Abstract:\nIndividual members of the retinoblastoma (Rb) tumor suppressor gene family serve critical roles in the control of cellular proliferation and differentiation, but the extent of their contributions is masked by redundant and compensatory mechanisms. Here we employed a conditional knockout strategy to simultaneously inactivate all three members, Rb, p107, and p130, in adult hematopoietic stem cells (HSCs). Rb family triple knockout (TKO) mice develop a cell-intrinsic myeloproliferation that originates from hyperproliferative early hematopoietic progenitors and is accompanied by increased apoptosis in lymphoid progenitor populations. Loss of quiescence in the TKO HSC pool is associated with an expansion of these mutant stem cells but also with an enhanced mobilization and an impaired reconstitution potential upon transplantation. The presence of a single p107 allele is sufficient to largely rescue these defects. Thus, Rb family members collectively maintain HSC quiescence and the balance between lymphoid and myeloid cell fates in the hematopoietic system.\nDOI: 10.1016/j.stem.2008.07.009, https://dx.doi.org/10.1016/j.stem.2008.07.009\nWeb of Science ID: 000260149800012, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000260149800012\nPubMedID: 18940733, https://www.ncbi.nlm.nih.gov/pubmed/18940733"},"doiId":"10.1016/j.stem.2008.07.009","doiUrl":"https://dx.doi.org/10.1016/j.stem.2008.07.009","featured":true,"firstPublished":{"html":"Oct 9, 2008 00:00:00 AM","text":"Oct 9, 2008 00:00:00 AM","value":"2008-10-09T00:00:00.000-07:00"},"mlaCitation":"Viatour, Patrick et al. ?Hematopoietic Stem Cell Quiescence Is Maintained by Compound Contributions of the Retinoblastoma Gene Family.? <i>CELL STEM CELL</i> 3.4 (2008): 416?428. Print.","pubMedId":"18940733","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/18940733","publicationId":75095,"title":"Hematopoietic Stem Cell Quiescence Is Maintained by Compound Contributions of the Retinoblastoma Gene Family","type":"article","webOfScienceId":"000260149800012","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000260149800012"},{"abstract":"The retinoblastoma (RB) tumour suppressor gene is functionally inactivated in a broad range of paediatric and adult cancers, and a plethora of cellular functions and partners have been identified for the RB protein. Data from human tumours and studies from mouse models indicate that loss of RB function contributes to both cancer initiation and progression. However, we still do not know the identity of the cell types in which RB normally prevents cancer initiation in vivo, and the specific functions of RB that suppress distinct aspects of the tumorigenic process are poorly understood.","apaCitation":"Burkhart, D. L., &amp; Sage, J. (2008). Cellular mechanisms of tumour suppression by the retinoblastoma gene. <i>NATURE REVIEWS CANCER</i>, <i>8</i>(9), 671?682.","capCitation":"<span class=\"title\">\n  <span>Cellular mechanisms of tumour suppression by the retinoblastoma gene</span>\n  <i>NATURE REVIEWS CANCER</i>\n</span>\n<span class=\"authors\">Burkhart, D. L., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2008</span>; <span class=\"volume\">8 (9)</span><span class=\"pages\">: 671-682</span>\n</span>","chicagoCitation":"Burkhart, Deborah L., and Julien Sage. 2008. ?Cellular Mechanisms of Tumour Suppression by the Retinoblastoma Gene.? <i>NATURE REVIEWS CANCER</i> 8 (9). NATURE PUBLISHING GROUP: 671?82.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma (RB) tumour suppressor gene is functionally inactivated in a broad range of paediatric and adult cancers, and a plethora of cellular functions and partners have been identified for the RB protein. Data from human tumours and studies from mouse models indicate that loss of RB function contributes to both cancer initiation and progression. However, we still do not know the identity of the cell types in which RB normally prevents cancer initiation in vivo, and the specific functions of RB that suppress distinct aspects of the tumorigenic process are poorly understood.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nrc2399\">DOI 10.1038/nrc2399</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000258662900012\">Web of Science ID 000258662900012</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/18650841\">PubMedID 18650841</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma (RB) tumour suppressor gene is functionally inactivated in a broad range of paediatric and adult cancers, and a plethora of cellular functions and partners have been identified for the RB protein. Data from human tumours and studies from mouse models indicate that loss of RB function contributes to both cancer initiation and progression. However, we still do not know the identity of the cell types in which RB normally prevents cancer initiation in vivo, and the specific functions of RB that suppress distinct aspects of the tumorigenic process are poorly understood.\nDOI: 10.1038/nrc2399, https://dx.doi.org/10.1038/nrc2399\nWeb of Science ID: 000258662900012, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000258662900012\nPubMedID: 18650841, https://www.ncbi.nlm.nih.gov/pubmed/18650841"},"doiId":"10.1038/nrc2399","doiUrl":"https://dx.doi.org/10.1038/nrc2399","featured":true,"firstPublished":{"html":"Sep 1, 2008 00:00:00 AM","text":"Sep 1, 2008 00:00:00 AM","value":"2008-09-01T00:00:00.000-07:00"},"mlaCitation":"Burkhart, Deborah L., and Julien Sage. ?Cellular Mechanisms of Tumour Suppression by the Retinoblastoma Gene.? <i>NATURE REVIEWS CANCER</i> 8.9 (2008): 671?682. Print.","pubMedId":"18650841","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/18650841","publicationId":72213,"title":"Cellular mechanisms of tumour suppression by the retinoblastoma gene","type":"article","webOfScienceId":"000258662900012","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000258662900012"},{"abstract":"The RB tumor suppressor gene is mutated in a broad range of human cancers, including pediatric retinoblastoma. Strikingly, however, Rb mutant mice develop tumors of the pituitary and thyroid glands, but not retinoblastoma. Mouse genetics experiments have demonstrated that p107, a protein related to pRB, is capable of preventing retinoblastoma, but not pituitary tumors, in Rb-deficient mice. Evidence suggests that the basis for this compensatory function of p107 is increased transcription of the p107 gene in response to Rb inactivation. To begin to address the context-dependency of this compensatory role of p107 and to follow p107 expression in vivo, we have generated transgenic mice carrying an enhanced GFP (eGFP) reporter inserted into a bacterial artificial chromosome (BAC) containing the mouse p107 gene. Expression of the eGFP transgene parallels that of p107 in these transgenic mice and identifies cells with a broad range of expression level for p107, even within particular organs or tissues. We also show that loss of Rb results in the upregulation of p107 transcription in specific cell populations in vivo, including subpopulations of hematopoietic cells. Thus, p107 BAC-eGFP transgenic mice serve as a useful tool to identify distinct cell types in which p107 is expressed and may have key functions in vivo, and to characterize changes in cellular networks accompanying Rb deficiency.","apaCitation":"Burkhart, D. L., Viatour, P., Ho, V. M., &amp; Sage, J. (2008). GFP reporter mice for the retinoblastoma-related cell cycle regulator p107. <i>CELL CYCLE</i>, <i>7</i>(16), 2544?2552.","capCitation":"<span class=\"title\">\n  <span>GFP reporter mice for the retinoblastoma-related cell cycle regulator p107</span>\n  <i>CELL CYCLE</i>\n</span>\n<span class=\"authors\">Burkhart, D. L., Viatour, P., Ho, V. M., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2008</span>; <span class=\"volume\">7 (16)</span><span class=\"pages\">: 2544-2552</span>\n</span>","chicagoCitation":"Burkhart, Deborah L., Patrick Viatour, Victoria M. Ho, and Julien Sage. 2008. ?GFP Reporter Mice for the Retinoblastoma-Related Cell Cycle Regulator p107.? <i>CELL CYCLE</i> 7 (16). LANDES BIOSCIENCE: 2544?52.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The RB tumor suppressor gene is mutated in a broad range of human cancers, including pediatric retinoblastoma. Strikingly, however, Rb mutant mice develop tumors of the pituitary and thyroid glands, but not retinoblastoma. Mouse genetics experiments have demonstrated that p107, a protein related to pRB, is capable of preventing retinoblastoma, but not pituitary tumors, in Rb-deficient mice. Evidence suggests that the basis for this compensatory function of p107 is increased transcription of the p107 gene in response to Rb inactivation. To begin to address the context-dependency of this compensatory role of p107 and to follow p107 expression in vivo, we have generated transgenic mice carrying an enhanced GFP (eGFP) reporter inserted into a bacterial artificial chromosome (BAC) containing the mouse p107 gene. Expression of the eGFP transgene parallels that of p107 in these transgenic mice and identifies cells with a broad range of expression level for p107, even within particular organs or tissues. We also show that loss of Rb results in the upregulation of p107 transcription in specific cell populations in vivo, including subpopulations of hematopoietic cells. Thus, p107 BAC-eGFP transgenic mice serve as a useful tool to identify distinct cell types in which p107 is expressed and may have key functions in vivo, and to characterize changes in cellular networks accompanying Rb deficiency.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000258829900016\">Web of Science ID 000258829900016</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/18719374\">PubMedID 18719374</a></span>\n  </p>","text":"Abstract:\nThe RB tumor suppressor gene is mutated in a broad range of human cancers, including pediatric retinoblastoma. Strikingly, however, Rb mutant mice develop tumors of the pituitary and thyroid glands, but not retinoblastoma. Mouse genetics experiments have demonstrated that p107, a protein related to pRB, is capable of preventing retinoblastoma, but not pituitary tumors, in Rb-deficient mice. Evidence suggests that the basis for this compensatory function of p107 is increased transcription of the p107 gene in response to Rb inactivation. To begin to address the context-dependency of this compensatory role of p107 and to follow p107 expression in vivo, we have generated transgenic mice carrying an enhanced GFP (eGFP) reporter inserted into a bacterial artificial chromosome (BAC) containing the mouse p107 gene. Expression of the eGFP transgene parallels that of p107 in these transgenic mice and identifies cells with a broad range of expression level for p107, even within particular organs or tissues. We also show that loss of Rb results in the upregulation of p107 transcription in specific cell populations in vivo, including subpopulations of hematopoietic cells. Thus, p107 BAC-eGFP transgenic mice serve as a useful tool to identify distinct cell types in which p107 is expressed and may have key functions in vivo, and to characterize changes in cellular networks accompanying Rb deficiency.\n\nWeb of Science ID: 000258829900016, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000258829900016\nPubMedID: 18719374, https://www.ncbi.nlm.nih.gov/pubmed/18719374"},"featured":false,"firstPublished":{"html":"Aug 15, 2008 00:00:00 AM","text":"Aug 15, 2008 00:00:00 AM","value":"2008-08-15T00:00:00.000-07:00"},"mlaCitation":"Burkhart, Deborah L. et al. ?GFP Reporter Mice for the Retinoblastoma-Related Cell Cycle Regulator p107.? <i>CELL CYCLE</i> 7.16 (2008): 2544?2552. Print.","pubMedId":"18719374","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/18719374","publicationId":73217,"title":"GFP reporter mice for the retinoblastoma-related cell cycle regulator p107","type":"article","webOfScienceId":"000258829900016","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000258829900016"},{"apaCitation":"Sage, J. (2007). Hope in sight for retinoblastoma. <i>NATURE MEDICINE</i>, <i>13</i>(1), 30?31.","capCitation":"<span class=\"title\">\n  <span>Hope in sight for retinoblastoma</span>\n  <i>NATURE MEDICINE</i>\n</span>\n<span class=\"authors\">Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2007</span>; <span class=\"volume\">13 (1)</span><span class=\"pages\">: 30-31</span>\n</span>","chicagoCitation":"Sage, Julien. 2007. ?Hope in Sight for Retinoblastoma.? <i>NATURE MEDICINE</i> 13 (1). NATURE PUBLISHING GROUP: 30?31.","detail":{"html":"<p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000243301800024\">Web of Science ID 000243301800024</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/17206131\">PubMedID 17206131</a></span>\n  </p>","text":"Web of Science ID: 000243301800024, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000243301800024\nPubMedID: 17206131, https://www.ncbi.nlm.nih.gov/pubmed/17206131"},"featured":false,"firstPublished":{"html":"Jan 1, 2007 00:00:00 AM","text":"Jan 1, 2007 00:00:00 AM","value":"2007-01-01T00:00:00.000-08:00"},"mlaCitation":"Sage, Julien. ?Hope in Sight for Retinoblastoma.? <i>NATURE MEDICINE</i> 13.1 (2007): 30?31. Print.","pubMedId":"17206131","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/17206131","publicationId":56229,"title":"Hope in sight for retinoblastoma","type":"article","webOfScienceId":"000243301800024","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000243301800024"},{"abstract":"Genetic studies have demonstrated that Bmi1 promotes cell proliferation and stem cell self-renewal with a correlative decrease of p16(INK4a) expression. Here, we demonstrate that Polycomb genes EZH2 and BMI1 repress p16 expression in human and mouse primary cells, but not in cells deficient for pRB protein function. The p16 locus is H3K27-methylated and bound by BMI1, RING2, and SUZ12. Inactivation of pRB family proteins abolishes H3K27 methylation and disrupts BMI1, RING2, and SUZ12 binding to the p16 locus. These results suggest a model in which pRB proteins recruit PRC2 to trimethylate p16, priming the BMI1-containing PRC1L ubiquitin ligase complex to silence p16.","apaCitation":"Kotake, Y., Cao, R., Viatour, P., Sage, J., Zhang, Y., &amp; Xiong, Y. (2007). pRB family proteins are required for H3K27 trimethylation and polycomb repression complexes binding to and silencing p16(INK4a) tumor suppressor gene. <i>GENES &amp; DEVELOPMENT</i>, <i>21</i>(1), 49?54.","capCitation":"<span class=\"title\">\n  <span>pRB family proteins are required for H3K27 trimethylation and polycomb repression complexes binding to and silencing p16(INK4a) tumor suppressor gene</span>\n  <i>GENES &amp; DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Kotake, Y., Cao, R., Viatour, P., Sage, J., Zhang, Y., Xiong, Y.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2007</span>; <span class=\"volume\">21 (1)</span><span class=\"pages\">: 49-54</span>\n</span>","chicagoCitation":"Kotake, Yojiro, Ru Cao, Patrick Viatour, Julien Sage, Yi Zhang, et al. 2007. ?PRB Family Proteins Are Required for H3K27 Trimethylation and Polycomb Repression Complexes Binding to and Silencing p16(INK4a) Tumor Suppressor Gene.? <i>GENES &amp; DEVELOPMENT</i> 21 (1). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 49?54.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Genetic studies have demonstrated that Bmi1 promotes cell proliferation and stem cell self-renewal with a correlative decrease of p16(INK4a) expression. Here, we demonstrate that Polycomb genes EZH2 and BMI1 repress p16 expression in human and mouse primary cells, but not in cells deficient for pRB protein function. The p16 locus is H3K27-methylated and bound by BMI1, RING2, and SUZ12. Inactivation of pRB family proteins abolishes H3K27 methylation and disrupts BMI1, RING2, and SUZ12 binding to the p16 locus. These results suggest a model in which pRB proteins recruit PRC2 to trimethylate p16, priming the BMI1-containing PRC1L ubiquitin ligase complex to silence p16.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.1499407\">DOI 10.1101/gad.1499407</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000243382700005\">Web of Science ID 000243382700005</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/17210787\">PubMedID 17210787</a></span>\n  </p>","text":"Abstract:\nGenetic studies have demonstrated that Bmi1 promotes cell proliferation and stem cell self-renewal with a correlative decrease of p16(INK4a) expression. Here, we demonstrate that Polycomb genes EZH2 and BMI1 repress p16 expression in human and mouse primary cells, but not in cells deficient for pRB protein function. The p16 locus is H3K27-methylated and bound by BMI1, RING2, and SUZ12. Inactivation of pRB family proteins abolishes H3K27 methylation and disrupts BMI1, RING2, and SUZ12 binding to the p16 locus. These results suggest a model in which pRB proteins recruit PRC2 to trimethylate p16, priming the BMI1-containing PRC1L ubiquitin ligase complex to silence p16.\nDOI: 10.1101/gad.1499407, https://dx.doi.org/10.1101/gad.1499407\nWeb of Science ID: 000243382700005, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000243382700005\nPubMedID: 17210787, https://www.ncbi.nlm.nih.gov/pubmed/17210787"},"doiId":"10.1101/gad.1499407","doiUrl":"https://dx.doi.org/10.1101/gad.1499407","featured":false,"firstPublished":{"html":"Jan 1, 2007 00:00:00 AM","text":"Jan 1, 2007 00:00:00 AM","value":"2007-01-01T00:00:00.000-08:00"},"mlaCitation":"Kotake, Yojiro et al. ?PRB Family Proteins Are Required for H3K27 Trimethylation and Polycomb Repression Complexes Binding to and Silencing p16(INK4a) Tumor Suppressor Gene.? <i>GENES &amp; DEVELOPMENT</i> 21.1 (2007): 49?54. Print.","pubMedId":"17210787","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/17210787","publicationId":217807,"title":"pRB family proteins are required for H3K27 trimethylation and polycomb repression complexes binding to and silencing p16(INK4a) tumor suppressor gene","type":"article","webOfScienceId":"000243382700005","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000243382700005"},{"abstract":"pRb, p107, and p130 are related proteins that play a central role in the regulation of cell cycle progression and terminal differentiation in mammalian cells. Nevertheless, it is still largely unclear how these proteins achieve this regulation in vivo. The intestinal epithelium is an ideal in vivo system in which to study the molecular pathways that regulate proliferation and differentiation because it exists in a constant state of development throughout an animal's lifetime. We studied the phenotypic effects on the intestinal epithelium of mutating Rb and p107 or p130. Although mutating these genes singly had little or no effect, loss of pRb and p107 or p130 together produced chronic hyperplasia and dysplasia of the small intestinal and colonic epithelium. In Rb/p130 double mutants this hyperplasia was associated with defects in terminal differentiation of specific cell types and was dependent on the increased proliferation seen in the epithelium of mutant animals. At the molecular level, dysregulation of the Rb pathway led to an increase in the expression of Math1, Cdx1, Cdx2, transcription factors that regulate proliferation and differentiation in the intestinal epithelium. The absence of Cdx1 function in Rb/p130 double mutant mice partially reverted the histologic phenotype by suppressing ectopic mitosis in the epithelium. These studies implicate the Rb pathway as a regulator of epithelial homeostasis in the intestine.","apaCitation":"Haigis, K., Sage, J., Glickman, J., Shafer, S., &amp; Jacks, T. (2006). The related retinoblastoma (pRb) and p130 proteins cooperate to regulate homeostasis in the intestinal epithelium. <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i>, <i>281</i>(1), 638?647.","capCitation":"<span class=\"title\">\n  <span>The related retinoblastoma (pRb) and p130 proteins cooperate to regulate homeostasis in the intestinal epithelium</span>\n  <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i>\n</span>\n<span class=\"authors\">Haigis, K., Sage, J., Glickman, J., Shafer, S., Jacks, T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2006</span>; <span class=\"volume\">281 (1)</span><span class=\"pages\">: 638-647</span>\n</span>","chicagoCitation":"Haigis, K., J. Sage, J. Glickman, S. Shafer, and T. Jacks. 2006. ?The Related Retinoblastoma (PRb) and p130 Proteins Cooperate to Regulate Homeostasis in the Intestinal Epithelium.? <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i> 281 (1). AMER SOC BIOCHEMISTRY MOLECULAR BIOLOGY INC: 638?47.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">pRb, p107, and p130 are related proteins that play a central role in the regulation of cell cycle progression and terminal differentiation in mammalian cells. Nevertheless, it is still largely unclear how these proteins achieve this regulation in vivo. The intestinal epithelium is an ideal in vivo system in which to study the molecular pathways that regulate proliferation and differentiation because it exists in a constant state of development throughout an animal's lifetime. We studied the phenotypic effects on the intestinal epithelium of mutating Rb and p107 or p130. Although mutating these genes singly had little or no effect, loss of pRb and p107 or p130 together produced chronic hyperplasia and dysplasia of the small intestinal and colonic epithelium. In Rb/p130 double mutants this hyperplasia was associated with defects in terminal differentiation of specific cell types and was dependent on the increased proliferation seen in the epithelium of mutant animals. At the molecular level, dysregulation of the Rb pathway led to an increase in the expression of Math1, Cdx1, Cdx2, transcription factors that regulate proliferation and differentiation in the intestinal epithelium. The absence of Cdx1 function in Rb/p130 double mutant mice partially reverted the histologic phenotype by suppressing ectopic mitosis in the epithelium. These studies implicate the Rb pathway as a regulator of epithelial homeostasis in the intestine.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000234307200076\">Web of Science ID 000234307200076</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/16258171\">PubMedID 16258171</a></span>\n  </p>","text":"Abstract:\npRb, p107, and p130 are related proteins that play a central role in the regulation of cell cycle progression and terminal differentiation in mammalian cells. Nevertheless, it is still largely unclear how these proteins achieve this regulation in vivo. The intestinal epithelium is an ideal in vivo system in which to study the molecular pathways that regulate proliferation and differentiation because it exists in a constant state of development throughout an animal's lifetime. We studied the phenotypic effects on the intestinal epithelium of mutating Rb and p107 or p130. Although mutating these genes singly had little or no effect, loss of pRb and p107 or p130 together produced chronic hyperplasia and dysplasia of the small intestinal and colonic epithelium. In Rb/p130 double mutants this hyperplasia was associated with defects in terminal differentiation of specific cell types and was dependent on the increased proliferation seen in the epithelium of mutant animals. At the molecular level, dysregulation of the Rb pathway led to an increase in the expression of Math1, Cdx1, Cdx2, transcription factors that regulate proliferation and differentiation in the intestinal epithelium. The absence of Cdx1 function in Rb/p130 double mutant mice partially reverted the histologic phenotype by suppressing ectopic mitosis in the epithelium. These studies implicate the Rb pathway as a regulator of epithelial homeostasis in the intestine.\n\nWeb of Science ID: 000234307200076, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000234307200076\nPubMedID: 16258171, https://www.ncbi.nlm.nih.gov/pubmed/16258171"},"featured":false,"firstPublished":{"html":"Jan 6, 2006 00:00:00 AM","text":"Jan 6, 2006 00:00:00 AM","value":"2006-01-06T00:00:00.000-08:00"},"mlaCitation":"Haigis, K. et al. ?The Related Retinoblastoma (PRb) and p130 Proteins Cooperate to Regulate Homeostasis in the Intestinal Epithelium.? <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i> 281.1 (2006): 638?647. Print.","pubMedId":"16258171","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/16258171","publicationId":242872,"title":"The related retinoblastoma (pRb) and p130 proteins cooperate to regulate homeostasis in the intestinal epithelium","type":"article","webOfScienceId":"000234307200076","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000234307200076"},{"abstract":"In primary cells, overexpression of oncogenes such as Ras(V12) induces premature senescence rather than transformation. Senescence is an irreversible form of G1 arrest that requires the p19ARF/p53 and p16INK4a/pRB pathways and may suppress tumorigenesis in vivo. Here we show that the transcription factor C/EBPbeta is required for Ras(V12)-induced senescence. C/EBPbeta-/- mouse embryo fibroblasts (MEFs) expressing Ras(V12) continued to proliferate despite unimpaired induction of p19ARF and p53, and lacked morphological features of senescent fibroblasts. Enforced C/EBPbeta expression inhibited proliferation of wild-type MEFs and also slowed proliferation of p19Arf-/- and p53-/- cells, indicating that C/EBPbeta acts downstream or independently of p19ARF/p53 to suppress growth. C/EBPbeta was unable to inhibit proliferation of MEFs lacking all three RB family proteins or wild-type cells expressing dominant negative E2F-1 and, instead, stimulated their growth. C/EBPbeta decreased expression of several E2F target genes and was associated with their promoters in chromatin immunoprecipitation assays, suggesting that C/EBPbeta functions by repressing genes required for cell cycle progression. C/EBPbeta is therefore a novel component of the RB:E2F-dependent senescence program activated by oncogenic stress in primary cells.","apaCitation":"Sebastian, T., Malik, R., Thomas, S., Sage, J., &amp; Johnson, P. F. (2005). C/EBP beta cooperates with RB : E2F to implement Ras(V12)-induced cellular senescence. <i>EMBO JOURNAL</i>, <i>24</i>(18), 3301?3312.","capCitation":"<span class=\"title\">\n  <span>C/EBP beta cooperates with RB : E2F to implement Ras(V12)-induced cellular senescence</span>\n  <i>EMBO JOURNAL</i>\n</span>\n<span class=\"authors\">Sebastian, T., Malik, R., Thomas, S., Sage, J., Johnson, P. F.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2005</span>; <span class=\"volume\">24 (18)</span><span class=\"pages\">: 3301-3312</span>\n</span>","chicagoCitation":"Sebastian, T., R. Malik, S. Thomas, J. Sage, and P. F. Johnson. 2005. ?C/EBP Beta Cooperates with RB : E2F to Implement Ras(V12)-Induced Cellular Senescence.? <i>EMBO JOURNAL</i> 24 (18). NATURE PUBLISHING GROUP: 3301?12.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">In primary cells, overexpression of oncogenes such as Ras(V12) induces premature senescence rather than transformation. Senescence is an irreversible form of G1 arrest that requires the p19ARF/p53 and p16INK4a/pRB pathways and may suppress tumorigenesis in vivo. Here we show that the transcription factor C/EBPbeta is required for Ras(V12)-induced senescence. C/EBPbeta-/- mouse embryo fibroblasts (MEFs) expressing Ras(V12) continued to proliferate despite unimpaired induction of p19ARF and p53, and lacked morphological features of senescent fibroblasts. Enforced C/EBPbeta expression inhibited proliferation of wild-type MEFs and also slowed proliferation of p19Arf-/- and p53-/- cells, indicating that C/EBPbeta acts downstream or independently of p19ARF/p53 to suppress growth. C/EBPbeta was unable to inhibit proliferation of MEFs lacking all three RB family proteins or wild-type cells expressing dominant negative E2F-1 and, instead, stimulated their growth. C/EBPbeta decreased expression of several E2F target genes and was associated with their promoters in chromatin immunoprecipitation assays, suggesting that C/EBPbeta functions by repressing genes required for cell cycle progression. C/EBPbeta is therefore a novel component of the RB:E2F-dependent senescence program activated by oncogenic stress in primary cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/sj.emboj.7600789\">DOI 10.1038/sj.emboj.7600789</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000232551600014\">Web of Science ID 000232551600014</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/16107878\">PubMedID 16107878</a></span>\n  </p>","text":"Abstract:\nIn primary cells, overexpression of oncogenes such as Ras(V12) induces premature senescence rather than transformation. Senescence is an irreversible form of G1 arrest that requires the p19ARF/p53 and p16INK4a/pRB pathways and may suppress tumorigenesis in vivo. Here we show that the transcription factor C/EBPbeta is required for Ras(V12)-induced senescence. C/EBPbeta-/- mouse embryo fibroblasts (MEFs) expressing Ras(V12) continued to proliferate despite unimpaired induction of p19ARF and p53, and lacked morphological features of senescent fibroblasts. Enforced C/EBPbeta expression inhibited proliferation of wild-type MEFs and also slowed proliferation of p19Arf-/- and p53-/- cells, indicating that C/EBPbeta acts downstream or independently of p19ARF/p53 to suppress growth. C/EBPbeta was unable to inhibit proliferation of MEFs lacking all three RB family proteins or wild-type cells expressing dominant negative E2F-1 and, instead, stimulated their growth. C/EBPbeta decreased expression of several E2F target genes and was associated with their promoters in chromatin immunoprecipitation assays, suggesting that C/EBPbeta functions by repressing genes required for cell cycle progression. C/EBPbeta is therefore a novel component of the RB:E2F-dependent senescence program activated by oncogenic stress in primary cells.\nDOI: 10.1038/sj.emboj.7600789, https://dx.doi.org/10.1038/sj.emboj.7600789\nWeb of Science ID: 000232551600014, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000232551600014\nPubMedID: 16107878, https://www.ncbi.nlm.nih.gov/pubmed/16107878"},"doiId":"10.1038/sj.emboj.7600789","doiUrl":"https://dx.doi.org/10.1038/sj.emboj.7600789","featured":false,"firstPublished":{"html":"Sep 21, 2005 00:00:00 AM","text":"Sep 21, 2005 00:00:00 AM","value":"2005-09-21T00:00:00.000-07:00"},"mlaCitation":"Sebastian, T. et al. ?C/EBP Beta Cooperates with RB : E2F to Implement Ras(V12)-Induced Cellular Senescence.? <i>EMBO JOURNAL</i> 24.18 (2005): 3301?3312. Print.","pubMedId":"16107878","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/16107878","publicationId":44096,"title":"C/EBP beta cooperates with RB : E2F to implement Ras(V12)-induced cellular senescence","type":"article","webOfScienceId":"000232551600014","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000232551600014"},{"abstract":"As the population of industrial nations ages, the incidence of cancer and cancer mortality is increasing. Intuitively, older organisms may be less able to cope with accumulated damage and thus be more prone to develop cancer. However, so far, the links between aging and cancer have been only partially explored. Strikingly, four recent studies now indicate that premature senescence accompanied by cell cycle arrest occurs in tumors initiated by an oncogenic mutation. Thus, senescence may act as a key tumor suppressor mechanism in young tumors in vivo.","apaCitation":"Sage, J. (2005). Making young tumors old: a new weapon against cancer? <i>Science of Aging Knowledge Environment : SAGE KE</i>, <i>2005</i>(33), pe25-?","capCitation":"<span class=\"title\">\n  <span>Making young tumors old: a new weapon against cancer?</span>\n  <i>Science of aging knowledge environment : SAGE KE</i>\n</span>\n<span class=\"authors\">Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2005</span>; <span class=\"volume\">2005 (33)</span><span class=\"pages\">: pe25-?</span>\n</span>","chicagoCitation":"Sage, Julien. 2005. ?Making Young Tumors Old: a New Weapon against Cancer?? <i>Science of Aging Knowledge Environment : SAGE KE</i> 2005 (33): pe25-?","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">As the population of industrial nations ages, the incidence of cancer and cancer mortality is increasing. Intuitively, older organisms may be less able to cope with accumulated damage and thus be more prone to develop cancer. However, so far, the links between aging and cancer have been only partially explored. Strikingly, four recent studies now indicate that premature senescence accompanied by cell cycle arrest occurs in tumors initiated by an oncogenic mutation. Thus, senescence may act as a key tumor suppressor mechanism in young tumors in vivo.</p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/16107660\">PubMedID 16107660</a></span>\n  </p>","text":"Abstract:\nAs the population of industrial nations ages, the incidence of cancer and cancer mortality is increasing. Intuitively, older organisms may be less able to cope with accumulated damage and thus be more prone to develop cancer. However, so far, the links between aging and cancer have been only partially explored. Strikingly, four recent studies now indicate that premature senescence accompanied by cell cycle arrest occurs in tumors initiated by an oncogenic mutation. Thus, senescence may act as a key tumor suppressor mechanism in young tumors in vivo.\n\n\nPubMedID: 16107660, https://www.ncbi.nlm.nih.gov/pubmed/16107660"},"featured":false,"firstPublished":{"html":"Aug 17, 2005 00:00:00 AM","text":"Aug 17, 2005 00:00:00 AM","value":"2005-08-17T00:00:00.000-07:00"},"mlaCitation":"Sage, Julien. ?Making Young Tumors Old: a New Weapon against Cancer?? <i>Science of aging knowledge environment : SAGE KE</i> 2005.33 (2005): pe25-? Print.","pubMedId":"16107660","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/16107660","publicationId":44091,"title":"Making young tumors old: a new weapon against cancer?","type":"article"},{"abstract":"Certain cells of the human retina are extremely sensitive to loss of function of the retinoblastoma tumor suppressor gene RB. Retinoblastomas develop early in life and at high frequency in individuals heterozygous for a germ-line RB mutation, and sporadic retinoblastomas invariably have somatic mutation in the RB gene. In contrast, retinoblastomas do not develop in Rb+/- mice. Although retinoblastoma is thought to have developmental origins, the function of Rb in retinal development has not been fully characterized. Here we studied the role of Rb in normal retinal development and in retinoblastoma using conditional Rb mutations in the mouse. In late embryogenesis, Rb-deficient retinas exhibited ectopic S-phase and high levels of p53-independent apoptosis, particularly in the differentiating retinal ganglion cell layer. During postnatal retinal development, loss of Rb led to more widespread retinal apoptosis, and adults showed loss of photoreceptors and bipolar cells. Conditional Rb mutation in the retina did not result in retinoblastoma formation even in a p53-mutant background. However, on a p107- or p130-deficient background, Rb mutation in the retina caused retinal dysplasia or retinoblastoma.","apaCitation":"Macpherson, D., Sage, J., Kim, T., Ho, D., McLaughlin, M. E., &amp; Jacks, T. (2004). Cell type-specific effects of Rb deletion in the murine retina. <i>GENES &amp; DEVELOPMENT</i>, <i>18</i>(14), 1681?1694.","capCitation":"<span class=\"title\">\n  <span>Cell type-specific effects of Rb deletion in the murine retina</span>\n  <i>GENES &amp; DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Macpherson, D., Sage, J., Kim, T., Ho, D., McLaughlin, M. E., Jacks, T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2004</span>; <span class=\"volume\">18 (14)</span><span class=\"pages\">: 1681-1694</span>\n</span>","chicagoCitation":"Macpherson, D., J. Sage, T. Kim, D. Ho, M. E. McLaughlin, et al. 2004. ?Cell Type-Specific Effects of Rb Deletion in the Murine Retina.? <i>GENES &amp; DEVELOPMENT</i> 18 (14). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 1681?94.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Certain cells of the human retina are extremely sensitive to loss of function of the retinoblastoma tumor suppressor gene RB. Retinoblastomas develop early in life and at high frequency in individuals heterozygous for a germ-line RB mutation, and sporadic retinoblastomas invariably have somatic mutation in the RB gene. In contrast, retinoblastomas do not develop in Rb+/- mice. Although retinoblastoma is thought to have developmental origins, the function of Rb in retinal development has not been fully characterized. Here we studied the role of Rb in normal retinal development and in retinoblastoma using conditional Rb mutations in the mouse. In late embryogenesis, Rb-deficient retinas exhibited ectopic S-phase and high levels of p53-independent apoptosis, particularly in the differentiating retinal ganglion cell layer. During postnatal retinal development, loss of Rb led to more widespread retinal apoptosis, and adults showed loss of photoreceptors and bipolar cells. Conditional Rb mutation in the retina did not result in retinoblastoma formation even in a p53-mutant background. However, on a p107- or p130-deficient background, Rb mutation in the retina caused retinal dysplasia or retinoblastoma.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.1203304\">DOI 10.1101/gad.1203304</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000222742100005\">Web of Science ID 000222742100005</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/15231717\">PubMedID 15231717</a></span>\n  </p>","text":"Abstract:\nCertain cells of the human retina are extremely sensitive to loss of function of the retinoblastoma tumor suppressor gene RB. Retinoblastomas develop early in life and at high frequency in individuals heterozygous for a germ-line RB mutation, and sporadic retinoblastomas invariably have somatic mutation in the RB gene. In contrast, retinoblastomas do not develop in Rb+/- mice. Although retinoblastoma is thought to have developmental origins, the function of Rb in retinal development has not been fully characterized. Here we studied the role of Rb in normal retinal development and in retinoblastoma using conditional Rb mutations in the mouse. In late embryogenesis, Rb-deficient retinas exhibited ectopic S-phase and high levels of p53-independent apoptosis, particularly in the differentiating retinal ganglion cell layer. During postnatal retinal development, loss of Rb led to more widespread retinal apoptosis, and adults showed loss of photoreceptors and bipolar cells. Conditional Rb mutation in the retina did not result in retinoblastoma formation even in a p53-mutant background. However, on a p107- or p130-deficient background, Rb mutation in the retina caused retinal dysplasia or retinoblastoma.\nDOI: 10.1101/gad.1203304, https://dx.doi.org/10.1101/gad.1203304\nWeb of Science ID: 000222742100005, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000222742100005\nPubMedID: 15231717, https://www.ncbi.nlm.nih.gov/pubmed/15231717"},"doiId":"10.1101/gad.1203304","doiUrl":"https://dx.doi.org/10.1101/gad.1203304","featured":false,"firstPublished":{"html":"Jul 15, 2004 00:00:00 AM","text":"Jul 15, 2004 00:00:00 AM","value":"2004-07-15T00:00:00.000-07:00"},"mlaCitation":"Macpherson, D. et al. ?Cell Type-Specific Effects of Rb Deletion in the Murine Retina.? <i>GENES &amp; DEVELOPMENT</i> 18.14 (2004): 1681?1694. Print.","pubMedId":"15231717","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/15231717","publicationId":34410,"title":"Cell type-specific effects of Rb deletion in the murine retina","type":"article","webOfScienceId":"000222742100005","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000222742100005"},{"abstract":"The retinoblastoma (RB) tumor suppressor has been proposed to function as a key mediator of cell cycle checkpoints induced by chemotherapeutic agents. However, these prior studies have relied on embryonic fibroblasts harboring chronic loss of RB, a condition under which compensation of RB functions is known to occur. Here we utilized primary adult fibroblasts derived from mice harboring loxP sites flanking exon 3 of the Rb gene to delineate the action of RB in the chemotherapeutic response. In this system we find that targeted disruption of Rb leads to little overt change in cell cycle distribution. However, these cells exhibited deregulation of RB/E2F target genes and became aneuploid following culture in the absence of RB. When challenged with both DNA damaging and antimetabolite chemotherapeutics, RB was required for primary adult cells to undergo DNA damage checkpoint responses and loss of RB resulted in enhanced aneuploidy following challenge. In contrast, following spontaneous immortalization and the loss of functional p53 signaling, the antimetabolite 5-fluorouracil (5-FU) failed to induce arrest despite the presence of RB. In these immortal cultures RB/E2F targets were deregulated in a complex, gene-specific manner and RB was required for the checkpoint response to camptothecin (CPT). Mechanistic analyses of the checkpoint responses in primary cells indicated that loss of RB leads to increased p53 signaling and decreased viability following both CPT and 5-FU treatment. However, the mechanism through which these agents act to facilitate cell cycle inhibition through RB were distinct. These studies underscore the critical role of RB in DNA-damage checkpoint signaling and demonstrate that RB mediates chemotherapeutic-induced cell cycle inhibition in adult fibroblasts by distinct mechanisms.","apaCitation":"Mayhew, C. N., Perkin, L. M., Zhang, X. P., Sage, J., Jacks, T., &amp; Knudsen, E. S. (2004). Discrete signaling pathways participate in RB-dependent responses to chemotherapeutic agents. <i>ONCOGENE</i>, <i>23</i>(23), 4107?4120.","capCitation":"<span class=\"title\">\n  <span>Discrete signaling pathways participate in RB-dependent responses to chemotherapeutic agents</span>\n  <i>ONCOGENE</i>\n</span>\n<span class=\"authors\">Mayhew, C. N., Perkin, L. M., Zhang, X. P., Sage, J., Jacks, T., Knudsen, E. S.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2004</span>; <span class=\"volume\">23 (23)</span><span class=\"pages\">: 4107-4120</span>\n</span>","chicagoCitation":"Mayhew, C. N., L. M. Perkin, X. P. Zhang, J. Sage, T. Jacks, et al. 2004. ?Discrete Signaling Pathways Participate in RB-Dependent Responses to Chemotherapeutic Agents.? <i>ONCOGENE</i> 23 (23). NATURE PUBLISHING GROUP: 4107?20.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma (RB) tumor suppressor has been proposed to function as a key mediator of cell cycle checkpoints induced by chemotherapeutic agents. However, these prior studies have relied on embryonic fibroblasts harboring chronic loss of RB, a condition under which compensation of RB functions is known to occur. Here we utilized primary adult fibroblasts derived from mice harboring loxP sites flanking exon 3 of the Rb gene to delineate the action of RB in the chemotherapeutic response. In this system we find that targeted disruption of Rb leads to little overt change in cell cycle distribution. However, these cells exhibited deregulation of RB/E2F target genes and became aneuploid following culture in the absence of RB. When challenged with both DNA damaging and antimetabolite chemotherapeutics, RB was required for primary adult cells to undergo DNA damage checkpoint responses and loss of RB resulted in enhanced aneuploidy following challenge. In contrast, following spontaneous immortalization and the loss of functional p53 signaling, the antimetabolite 5-fluorouracil (5-FU) failed to induce arrest despite the presence of RB. In these immortal cultures RB/E2F targets were deregulated in a complex, gene-specific manner and RB was required for the checkpoint response to camptothecin (CPT). Mechanistic analyses of the checkpoint responses in primary cells indicated that loss of RB leads to increased p53 signaling and decreased viability following both CPT and 5-FU treatment. However, the mechanism through which these agents act to facilitate cell cycle inhibition through RB were distinct. These studies underscore the critical role of RB in DNA-damage checkpoint signaling and demonstrate that RB mediates chemotherapeutic-induced cell cycle inhibition in adult fibroblasts by distinct mechanisms.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/sj.onc.1207503\">DOI 10.1038/sj.onc.1207503</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000221520200009\">Web of Science ID 000221520200009</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/15064736\">PubMedID 15064736</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma (RB) tumor suppressor has been proposed to function as a key mediator of cell cycle checkpoints induced by chemotherapeutic agents. However, these prior studies have relied on embryonic fibroblasts harboring chronic loss of RB, a condition under which compensation of RB functions is known to occur. Here we utilized primary adult fibroblasts derived from mice harboring loxP sites flanking exon 3 of the Rb gene to delineate the action of RB in the chemotherapeutic response. In this system we find that targeted disruption of Rb leads to little overt change in cell cycle distribution. However, these cells exhibited deregulation of RB/E2F target genes and became aneuploid following culture in the absence of RB. When challenged with both DNA damaging and antimetabolite chemotherapeutics, RB was required for primary adult cells to undergo DNA damage checkpoint responses and loss of RB resulted in enhanced aneuploidy following challenge. In contrast, following spontaneous immortalization and the loss of functional p53 signaling, the antimetabolite 5-fluorouracil (5-FU) failed to induce arrest despite the presence of RB. In these immortal cultures RB/E2F targets were deregulated in a complex, gene-specific manner and RB was required for the checkpoint response to camptothecin (CPT). Mechanistic analyses of the checkpoint responses in primary cells indicated that loss of RB leads to increased p53 signaling and decreased viability following both CPT and 5-FU treatment. However, the mechanism through which these agents act to facilitate cell cycle inhibition through RB were distinct. These studies underscore the critical role of RB in DNA-damage checkpoint signaling and demonstrate that RB mediates chemotherapeutic-induced cell cycle inhibition in adult fibroblasts by distinct mechanisms.\nDOI: 10.1038/sj.onc.1207503, https://dx.doi.org/10.1038/sj.onc.1207503\nWeb of Science ID: 000221520200009, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000221520200009\nPubMedID: 15064736, https://www.ncbi.nlm.nih.gov/pubmed/15064736"},"doiId":"10.1038/sj.onc.1207503","doiUrl":"https://dx.doi.org/10.1038/sj.onc.1207503","featured":false,"firstPublished":{"html":"May 20, 2004 00:00:00 AM","text":"May 20, 2004 00:00:00 AM","value":"2004-05-20T00:00:00.000-07:00"},"mlaCitation":"Mayhew, C. N. et al. ?Discrete Signaling Pathways Participate in RB-Dependent Responses to Chemotherapeutic Agents.? <i>ONCOGENE</i> 23.23 (2004): 4107?4120. Print.","pubMedId":"15064736","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/15064736","publicationId":242873,"title":"Discrete signaling pathways participate in RB-dependent responses to chemotherapeutic agents","type":"article","webOfScienceId":"000221520200009","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000221520200009"},{"abstract":"From yeast to humans, cell cycle progression is orchestrated by the oscillation of kinase activities associated with cyclins. In an article published recently in Cell, Ren and Rollins investigate mechanisms controlling the G0/G1 transition in quiescent cells and identify new cyclin C/Cdk3 complexes as key regulators of cell cycle reentry in human cells.","apaCitation":"Sage, J. (2004). Cyclin C makes an entry into the cell cycle. <i>DEVELOPMENTAL CELL</i>, <i>6</i>(5), 607?608.","capCitation":"<span class=\"title\">\n  <span>Cyclin C makes an entry into the cell cycle</span>\n  <i>DEVELOPMENTAL CELL</i>\n</span>\n<span class=\"authors\">Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2004</span>; <span class=\"volume\">6 (5)</span><span class=\"pages\">: 607-608</span>\n</span>","chicagoCitation":"Sage, J. 2004. ?Cyclin C Makes an Entry into the Cell Cycle.? <i>DEVELOPMENTAL CELL</i> 6 (5). CELL PRESS: 607?8.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">From yeast to humans, cell cycle progression is orchestrated by the oscillation of kinase activities associated with cyclins. In an article published recently in Cell, Ren and Rollins investigate mechanisms controlling the G0/G1 transition in quiescent cells and identify new cyclin C/Cdk3 complexes as key regulators of cell cycle reentry in human cells.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000222443100001\">Web of Science ID 000222443100001</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/15130482\">PubMedID 15130482</a></span>\n  </p>","text":"Abstract:\nFrom yeast to humans, cell cycle progression is orchestrated by the oscillation of kinase activities associated with cyclins. In an article published recently in Cell, Ren and Rollins investigate mechanisms controlling the G0/G1 transition in quiescent cells and identify new cyclin C/Cdk3 complexes as key regulators of cell cycle reentry in human cells.\n\nWeb of Science ID: 000222443100001, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000222443100001\nPubMedID: 15130482, https://www.ncbi.nlm.nih.gov/pubmed/15130482"},"featured":false,"firstPublished":{"html":"May 1, 2004 00:00:00 AM","text":"May 1, 2004 00:00:00 AM","value":"2004-05-01T00:00:00.000-07:00"},"mlaCitation":"Sage, J. ?Cyclin C Makes an Entry into the Cell Cycle.? <i>DEVELOPMENTAL CELL</i> 6.5 (2004): 607?608. Print.","pubMedId":"15130482","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/15130482","publicationId":33256,"title":"Cyclin C makes an entry into the cell cycle","type":"article","webOfScienceId":"000222443100001","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000222443100001"},{"abstract":"Cell cycle checkpoints induced by DNA damage play an integral role in preservation of genomic stability by allowing cells to limit the propagation of deleterious mutations. The retinoblastoma tumor suppressor (RB) is crucial for the maintenance of the DNA damage checkpoint function because it elicits cell cycle arrest in response to a variety of genotoxic stresses. Although sporadic loss of RB is characteristic of most cancers and results in the bypass of the DNA damage checkpoint, the consequence of RB loss upon chemotherapeutic responsiveness has been largely uninvestigated. Here, we employed a conditional knockout approach to ablate RB in adult fibroblasts. This system enabled us to examine the DNA damage response of adult cells following acute RB deletion. Using this system, we demonstrated that loss of RB disrupted the DNA damage checkpoint elicited by either cisplatin or camptothecin exposure. Strikingly, this bypass was not associated with enhanced repair, but rather the accumulation of phosphorylated H2AX (gammaH2AX) foci, which indicate DNA double-strand breaks. The formation of gammaH2AX foci was due to ongoing replication following chemotherapeutic treatment in the RB-deficient cells. Additionally, peak gammaH2AX accumulation occurred in S-phase cells undergoing DNA replication in the presence of damage, and these gammaH2AX foci co-localized with replication foci. These results demonstrate that acute RB loss abrogates DNA damage-induced cell cycle arrest to induce gammaH2AX foci formation. Thus, secondary genetic lesions induced by RB loss have implications for the chemotherapeutic response and the development of genetic instability.","apaCitation":"Bosco, E. E., Mayhew, C. N., Hennigan, R. F., Sage, J., Jacks, T., &amp; Knudsen, E. S. (2004). RB signaling prevents replication-dependent DNA double-strand breaks following genotoxic insult. <i>NUCLEIC ACIDS RESEARCH</i>, <i>32</i>(1), 25?34.","capCitation":"<span class=\"title\">\n  <span>RB signaling prevents replication-dependent DNA double-strand breaks following genotoxic insult</span>\n  <i>NUCLEIC ACIDS RESEARCH</i>\n</span>\n<span class=\"authors\">Bosco, E. E., Mayhew, C. N., Hennigan, R. F., Sage, J., Jacks, T., Knudsen, E. S.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2004</span>; <span class=\"volume\">32 (1)</span><span class=\"pages\">: 25-34</span>\n</span>","chicagoCitation":"Bosco, E. E., C. N. Mayhew, R. F. Hennigan, J. Sage, T. Jacks, et al. 2004. ?RB Signaling Prevents Replication-Dependent DNA Double-Strand Breaks Following Genotoxic Insult.? <i>NUCLEIC ACIDS RESEARCH</i> 32 (1). OXFORD UNIV PRESS: 25?34.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Cell cycle checkpoints induced by DNA damage play an integral role in preservation of genomic stability by allowing cells to limit the propagation of deleterious mutations. The retinoblastoma tumor suppressor (RB) is crucial for the maintenance of the DNA damage checkpoint function because it elicits cell cycle arrest in response to a variety of genotoxic stresses. Although sporadic loss of RB is characteristic of most cancers and results in the bypass of the DNA damage checkpoint, the consequence of RB loss upon chemotherapeutic responsiveness has been largely uninvestigated. Here, we employed a conditional knockout approach to ablate RB in adult fibroblasts. This system enabled us to examine the DNA damage response of adult cells following acute RB deletion. Using this system, we demonstrated that loss of RB disrupted the DNA damage checkpoint elicited by either cisplatin or camptothecin exposure. Strikingly, this bypass was not associated with enhanced repair, but rather the accumulation of phosphorylated H2AX (gammaH2AX) foci, which indicate DNA double-strand breaks. The formation of gammaH2AX foci was due to ongoing replication following chemotherapeutic treatment in the RB-deficient cells. Additionally, peak gammaH2AX accumulation occurred in S-phase cells undergoing DNA replication in the presence of damage, and these gammaH2AX foci co-localized with replication foci. These results demonstrate that acute RB loss abrogates DNA damage-induced cell cycle arrest to induce gammaH2AX foci formation. Thus, secondary genetic lesions induced by RB loss have implications for the chemotherapeutic response and the development of genetic instability.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1093/nar/gkg919\">DOI 10.1093/nar/gkg919</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000188988700018\">Web of Science ID 000188988700018</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/14704340\">PubMedID 14704340</a></span>\n  </p>","text":"Abstract:\nCell cycle checkpoints induced by DNA damage play an integral role in preservation of genomic stability by allowing cells to limit the propagation of deleterious mutations. The retinoblastoma tumor suppressor (RB) is crucial for the maintenance of the DNA damage checkpoint function because it elicits cell cycle arrest in response to a variety of genotoxic stresses. Although sporadic loss of RB is characteristic of most cancers and results in the bypass of the DNA damage checkpoint, the consequence of RB loss upon chemotherapeutic responsiveness has been largely uninvestigated. Here, we employed a conditional knockout approach to ablate RB in adult fibroblasts. This system enabled us to examine the DNA damage response of adult cells following acute RB deletion. Using this system, we demonstrated that loss of RB disrupted the DNA damage checkpoint elicited by either cisplatin or camptothecin exposure. Strikingly, this bypass was not associated with enhanced repair, but rather the accumulation of phosphorylated H2AX (gammaH2AX) foci, which indicate DNA double-strand breaks. The formation of gammaH2AX foci was due to ongoing replication following chemotherapeutic treatment in the RB-deficient cells. Additionally, peak gammaH2AX accumulation occurred in S-phase cells undergoing DNA replication in the presence of damage, and these gammaH2AX foci co-localized with replication foci. These results demonstrate that acute RB loss abrogates DNA damage-induced cell cycle arrest to induce gammaH2AX foci formation. Thus, secondary genetic lesions induced by RB loss have implications for the chemotherapeutic response and the development of genetic instability.\nDOI: 10.1093/nar/gkg919, https://dx.doi.org/10.1093/nar/gkg919\nWeb of Science ID: 000188988700018, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000188988700018\nPubMedID: 14704340, https://www.ncbi.nlm.nih.gov/pubmed/14704340"},"doiId":"10.1093/nar/gkg919","doiUrl":"https://dx.doi.org/10.1093/nar/gkg919","featured":true,"firstPublished":{"html":"Jan 1, 2004 00:00:00 AM","text":"Jan 1, 2004 00:00:00 AM","value":"2004-01-01T00:00:00.000-08:00"},"mlaCitation":"Bosco, E. E. et al. ?RB Signaling Prevents Replication-Dependent DNA Double-Strand Breaks Following Genotoxic Insult.? <i>NUCLEIC ACIDS RESEARCH</i> 32.1 (2004): 25?34. Print.","pubMedId":"14704340","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/14704340","publicationId":242874,"title":"RB signaling prevents replication-dependent DNA double-strand breaks following genotoxic insult","type":"article","webOfScienceId":"000188988700018","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000188988700018"},{"abstract":"Although the human papillomavirus (HPV) E7 oncogene is known to contribute to the development of human cervical cancer, the mechanisms of its carcinogenesis are poorly understood. The first identified and most recognized function of E7 is its binding to and inactivation of the retinoblastoma tumor suppressor (pRb), but at least 18 other biological activities have also been reported for E7. Thus, it remains unclear which of these many activities contribute to the oncogenic potential of E7. We used a Cre-lox system to abolish pRb expression in the epidermis of transgenic mice and compared the outcome with the effects of E7 expression in the same tissue at early ages. Mice lacking pRb in epidermis showed epithelial hyperplasia, aberrant DNA synthesis, and improper differentiation. In addition, Rb-deleted epidermis (i.e., epidermis composed of cells with Rb deleted) exhibited centrosomal abnormalities and failed to arrest the cell cycle in response to ionizing radiation. Transgenic mice expressing E7 in skin display the same range of phenotypes. In sum, few differences were detected between Rb-deleted epidermis and E7-expressing epidermis in young mice. However, when both E7 was expressed and Rb was deleted in the same tissue, increased hyperplasia and dysplasia were observed. These findings indicate that inactivation of the Rb pathway can largely account for E7's phenotypes at an early age, but that pRb-independent activities of E7 are detectable in vivo.","apaCitation":"Balsitis, S. J., Sage, J., Duensing, S., Munger, K., Jacks, T., &amp; Lambert, P. F. (2003). Recapitulation of the effects of the human papillomavirus type 16 E7 oncogene on mouse epithelium by somatic Rb deletion and detection of pRb-independent effects of E7 in vivo. <i>MOLECULAR AND CELLULAR BIOLOGY</i>, <i>23</i>(24), 9094?9103.","capCitation":"<span class=\"title\">\n  <span>Recapitulation of the effects of the human papillomavirus type 16 E7 oncogene on mouse epithelium by somatic Rb deletion and detection of pRb-independent effects of E7 in vivo</span>\n  <i>MOLECULAR AND CELLULAR BIOLOGY</i>\n</span>\n<span class=\"authors\">Balsitis, S. J., Sage, J., Duensing, S., Munger, K., Jacks, T., Lambert, P. F.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2003</span>; <span class=\"volume\">23 (24)</span><span class=\"pages\">: 9094-9103</span>\n</span>","chicagoCitation":"Balsitis, S. J., J. Sage, S. Duensing, K. Munger, T. Jacks, et al. 2003. ?Recapitulation of the Effects of the Human Papillomavirus Type 16 E7 Oncogene on Mouse Epithelium by Somatic Rb Deletion and Detection of PRb-Independent Effects of E7 in Vivo.? <i>MOLECULAR AND CELLULAR BIOLOGY</i> 23 (24). AMER SOC MICROBIOLOGY: 9094?9103.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Although the human papillomavirus (HPV) E7 oncogene is known to contribute to the development of human cervical cancer, the mechanisms of its carcinogenesis are poorly understood. The first identified and most recognized function of E7 is its binding to and inactivation of the retinoblastoma tumor suppressor (pRb), but at least 18 other biological activities have also been reported for E7. Thus, it remains unclear which of these many activities contribute to the oncogenic potential of E7. We used a Cre-lox system to abolish pRb expression in the epidermis of transgenic mice and compared the outcome with the effects of E7 expression in the same tissue at early ages. Mice lacking pRb in epidermis showed epithelial hyperplasia, aberrant DNA synthesis, and improper differentiation. In addition, Rb-deleted epidermis (i.e., epidermis composed of cells with Rb deleted) exhibited centrosomal abnormalities and failed to arrest the cell cycle in response to ionizing radiation. Transgenic mice expressing E7 in skin display the same range of phenotypes. In sum, few differences were detected between Rb-deleted epidermis and E7-expressing epidermis in young mice. However, when both E7 was expressed and Rb was deleted in the same tissue, increased hyperplasia and dysplasia were observed. These findings indicate that inactivation of the Rb pathway can largely account for E7's phenotypes at an early age, but that pRb-independent activities of E7 are detectable in vivo.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1128/MBC.23.24.9094-9103.2003\">DOI 10.1128/MBC.23.24.9094-9103.2003</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000187040400017\">Web of Science ID 000187040400017</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/14645521\">PubMedID 14645521</a></span>\n  </p>","text":"Abstract:\nAlthough the human papillomavirus (HPV) E7 oncogene is known to contribute to the development of human cervical cancer, the mechanisms of its carcinogenesis are poorly understood. The first identified and most recognized function of E7 is its binding to and inactivation of the retinoblastoma tumor suppressor (pRb), but at least 18 other biological activities have also been reported for E7. Thus, it remains unclear which of these many activities contribute to the oncogenic potential of E7. We used a Cre-lox system to abolish pRb expression in the epidermis of transgenic mice and compared the outcome with the effects of E7 expression in the same tissue at early ages. Mice lacking pRb in epidermis showed epithelial hyperplasia, aberrant DNA synthesis, and improper differentiation. In addition, Rb-deleted epidermis (i.e., epidermis composed of cells with Rb deleted) exhibited centrosomal abnormalities and failed to arrest the cell cycle in response to ionizing radiation. Transgenic mice expressing E7 in skin display the same range of phenotypes. In sum, few differences were detected between Rb-deleted epidermis and E7-expressing epidermis in young mice. However, when both E7 was expressed and Rb was deleted in the same tissue, increased hyperplasia and dysplasia were observed. These findings indicate that inactivation of the Rb pathway can largely account for E7's phenotypes at an early age, but that pRb-independent activities of E7 are detectable in vivo.\nDOI: 10.1128/MBC.23.24.9094-9103.2003, https://dx.doi.org/10.1128/MBC.23.24.9094-9103.2003\nWeb of Science ID: 000187040400017, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000187040400017\nPubMedID: 14645521, https://www.ncbi.nlm.nih.gov/pubmed/14645521"},"doiId":"10.1128/MBC.23.24.9094-9103.2003","doiUrl":"https://dx.doi.org/10.1128/MBC.23.24.9094-9103.2003","featured":false,"firstPublished":{"html":"Dec 1, 2003 00:00:00 AM","text":"Dec 1, 2003 00:00:00 AM","value":"2003-12-01T00:00:00.000-08:00"},"mlaCitation":"Balsitis, S. J. et al. ?Recapitulation of the Effects of the Human Papillomavirus Type 16 E7 Oncogene on Mouse Epithelium by Somatic Rb Deletion and Detection of PRb-Independent Effects of E7 in Vivo.? <i>MOLECULAR AND CELLULAR BIOLOGY</i> 23.24 (2003): 9094?9103. Print.","pubMedId":"14645521","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/14645521","publicationId":242875,"title":"Recapitulation of the effects of the human papillomavirus type 16 E7 oncogene on mouse epithelium by somatic Rb deletion and detection of pRb-independent effects of E7 in vivo","type":"article","webOfScienceId":"000187040400017","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000187040400017"},{"abstract":"The induction of apoptosis by the p53 protein is critical for its activity as a tumor suppressor. Although it is clear that p53 induces apoptosis at least in part by direct transcriptional activation of target genes, the set of p53 target genes that mediate p53 function in apoptosis in vivo remains to be well defined. The Perp (p53 apoptosis effector related to PMP-22) gene is highly expressed in cells undergoing p53-dependent apoptosis as compared to cells undergoing p53-dependent G1 arrest. Perp is a direct p53 target, and its overexpression is sufficient to induce cell death in fibroblasts, implicating it as an important component of p53 apoptotic function. Here, through the generation of Perp-deficient mice, we analyze the role of Perp in the p53 apoptosis pathway in multiple primary cell types by comparing the cell death responses of Perp null cells to those of wild-type and p53 null cells. These experiments demonstrate the involvement of Perp in p53-mediated cell death in thymocytes and neurons but not in E1A-expressing MEFs, indicating a cell type-specific role for Perp in the p53 cell death pathway. In addition, we show that Perp is not required for proliferation-associated functions of p53. Thus, Perp selectively mediates the p53 apoptotic response, and the requirement for Perp is dictated by cellular context.","apaCitation":"Ihrie, R. A., Reczek, E., Horner, J. S., Khachatrian, L., Sage, J., Jacks, T., &amp; Attardi, L. D. (2003). Perp is a mediator of p53-dependent apoptosis in diverse cell types. <i>CURRENT BIOLOGY</i>, <i>13</i>(22), 1985?1990.","capCitation":"<span class=\"title\">\n  <span>Perp is a mediator of p53-dependent apoptosis in diverse cell types</span>\n  <i>CURRENT BIOLOGY</i>\n</span>\n<span class=\"authors\">Ihrie, R. A., Reczek, E., Horner, J. S., Khachatrian, L., Sage, J., Jacks, T., Attardi, L. D.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2003</span>; <span class=\"volume\">13 (22)</span><span class=\"pages\">: 1985-1990</span>\n</span>","chicagoCitation":"Ihrie, R. A., E. Reczek, J. S. Horner, L. Khachatrian, J. Sage, et al. 2003. ?Perp Is a Mediator of p53-Dependent Apoptosis in Diverse Cell Types.? <i>CURRENT BIOLOGY</i> 13 (22). CELL PRESS: 1985?90.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The induction of apoptosis by the p53 protein is critical for its activity as a tumor suppressor. Although it is clear that p53 induces apoptosis at least in part by direct transcriptional activation of target genes, the set of p53 target genes that mediate p53 function in apoptosis in vivo remains to be well defined. The Perp (p53 apoptosis effector related to PMP-22) gene is highly expressed in cells undergoing p53-dependent apoptosis as compared to cells undergoing p53-dependent G1 arrest. Perp is a direct p53 target, and its overexpression is sufficient to induce cell death in fibroblasts, implicating it as an important component of p53 apoptotic function. Here, through the generation of Perp-deficient mice, we analyze the role of Perp in the p53 apoptosis pathway in multiple primary cell types by comparing the cell death responses of Perp null cells to those of wild-type and p53 null cells. These experiments demonstrate the involvement of Perp in p53-mediated cell death in thymocytes and neurons but not in E1A-expressing MEFs, indicating a cell type-specific role for Perp in the p53 cell death pathway. In addition, we show that Perp is not required for proliferation-associated functions of p53. Thus, Perp selectively mediates the p53 apoptotic response, and the requirement for Perp is dictated by cellular context.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.cub.2003.10.055\">DOI 10.1016/j.cub.2003.10.055</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000186558100026\">Web of Science ID 000186558100026</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/14614825\">PubMedID 14614825</a></span>\n  </p>","text":"Abstract:\nThe induction of apoptosis by the p53 protein is critical for its activity as a tumor suppressor. Although it is clear that p53 induces apoptosis at least in part by direct transcriptional activation of target genes, the set of p53 target genes that mediate p53 function in apoptosis in vivo remains to be well defined. The Perp (p53 apoptosis effector related to PMP-22) gene is highly expressed in cells undergoing p53-dependent apoptosis as compared to cells undergoing p53-dependent G1 arrest. Perp is a direct p53 target, and its overexpression is sufficient to induce cell death in fibroblasts, implicating it as an important component of p53 apoptotic function. Here, through the generation of Perp-deficient mice, we analyze the role of Perp in the p53 apoptosis pathway in multiple primary cell types by comparing the cell death responses of Perp null cells to those of wild-type and p53 null cells. These experiments demonstrate the involvement of Perp in p53-mediated cell death in thymocytes and neurons but not in E1A-expressing MEFs, indicating a cell type-specific role for Perp in the p53 cell death pathway. In addition, we show that Perp is not required for proliferation-associated functions of p53. Thus, Perp selectively mediates the p53 apoptotic response, and the requirement for Perp is dictated by cellular context.\nDOI: 10.1016/j.cub.2003.10.055, https://dx.doi.org/10.1016/j.cub.2003.10.055\nWeb of Science ID: 000186558100026, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000186558100026\nPubMedID: 14614825, https://www.ncbi.nlm.nih.gov/pubmed/14614825"},"doiId":"10.1016/j.cub.2003.10.055","doiUrl":"https://dx.doi.org/10.1016/j.cub.2003.10.055","featured":false,"firstPublished":{"html":"Nov 11, 2003 00:00:00 AM","text":"Nov 11, 2003 00:00:00 AM","value":"2003-11-11T00:00:00.000-08:00"},"mlaCitation":"Ihrie, R. A. et al. ?Perp Is a Mediator of p53-Dependent Apoptosis in Diverse Cell Types.? <i>CURRENT BIOLOGY</i> 13.22 (2003): 1985?1990. Print.","pubMedId":"14614825","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/14614825","publicationId":29469,"title":"Perp is a mediator of p53-dependent apoptosis in diverse cell types","type":"article","webOfScienceId":"000186558100026","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000186558100026"},{"abstract":"Cancer cells arise from normal cells through the acquisition of a series of mutations in oncogenes and tumour suppressor genes. Mouse models of human cancer often rely on germline alterations that activate or inactivate genes of interest. One limitation of this approach is that germline mutations might have effects other than somatic mutations, owing to developmental compensation. To model sporadic cancers associated with inactivation of the retinoblastoma (RB) tumour suppressor gene in humans, we have produced a conditional allele of the mouse Rb gene. We show here that acute loss of Rb in primary quiescent cells is sufficient for cell cycle entry and has phenotypic consequences different from germline loss of Rb function. This difference is explained in part by functional compensation by the Rb-related gene p107. We also show that acute loss of Rb in senescent cells leads to reversal of the cellular senescence programme. Thus, the use of conditional knockout strategies might refine our understanding of gene function and help to model human cancer more accurately.","apaCitation":"Sage, J., Miller, A. L., Perez-Mancera, P. A., Wysocki, J. M., &amp; Jacks, T. (2003). Acute mutation of retinoblastoma gene function is sufficient for cell cycle re-entry. <i>NATURE</i>, <i>424</i>(6945), 223?228.","capCitation":"<span class=\"title\">\n  <span>Acute mutation of retinoblastoma gene function is sufficient for cell cycle re-entry</span>\n  <i>NATURE</i>\n</span>\n<span class=\"authors\">Sage, J., Miller, A. L., Perez-Mancera, P. A., Wysocki, J. M., Jacks, T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2003</span>; <span class=\"volume\">424 (6945)</span><span class=\"pages\">: 223-228</span>\n</span>","chicagoCitation":"Sage, J., A. L. Miller, P. A. Perez-Mancera, J. M. Wysocki, and T. Jacks. 2003. ?Acute Mutation of Retinoblastoma Gene Function Is Sufficient for Cell Cycle Re-Entry.? <i>NATURE</i> 424 (6945). NATURE PUBLISHING GROUP: 223?28.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Cancer cells arise from normal cells through the acquisition of a series of mutations in oncogenes and tumour suppressor genes. Mouse models of human cancer often rely on germline alterations that activate or inactivate genes of interest. One limitation of this approach is that germline mutations might have effects other than somatic mutations, owing to developmental compensation. To model sporadic cancers associated with inactivation of the retinoblastoma (RB) tumour suppressor gene in humans, we have produced a conditional allele of the mouse Rb gene. We show here that acute loss of Rb in primary quiescent cells is sufficient for cell cycle entry and has phenotypic consequences different from germline loss of Rb function. This difference is explained in part by functional compensation by the Rb-related gene p107. We also show that acute loss of Rb in senescent cells leads to reversal of the cellular senescence programme. Thus, the use of conditional knockout strategies might refine our understanding of gene function and help to model human cancer more accurately.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nature01764\">DOI 10.1038/nature01764</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000184032700049\">Web of Science ID 000184032700049</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/12853964\">PubMedID 12853964</a></span>\n  </p>","text":"Abstract:\nCancer cells arise from normal cells through the acquisition of a series of mutations in oncogenes and tumour suppressor genes. Mouse models of human cancer often rely on germline alterations that activate or inactivate genes of interest. One limitation of this approach is that germline mutations might have effects other than somatic mutations, owing to developmental compensation. To model sporadic cancers associated with inactivation of the retinoblastoma (RB) tumour suppressor gene in humans, we have produced a conditional allele of the mouse Rb gene. We show here that acute loss of Rb in primary quiescent cells is sufficient for cell cycle entry and has phenotypic consequences different from germline loss of Rb function. This difference is explained in part by functional compensation by the Rb-related gene p107. We also show that acute loss of Rb in senescent cells leads to reversal of the cellular senescence programme. Thus, the use of conditional knockout strategies might refine our understanding of gene function and help to model human cancer more accurately.\nDOI: 10.1038/nature01764, https://dx.doi.org/10.1038/nature01764\nWeb of Science ID: 000184032700049, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000184032700049\nPubMedID: 12853964, https://www.ncbi.nlm.nih.gov/pubmed/12853964"},"doiId":"10.1038/nature01764","doiUrl":"https://dx.doi.org/10.1038/nature01764","featured":true,"firstPublished":{"html":"Jul 10, 2003 00:00:00 AM","text":"Jul 10, 2003 00:00:00 AM","value":"2003-07-10T00:00:00.000-07:00"},"mlaCitation":"Sage, J. et al. ?Acute Mutation of Retinoblastoma Gene Function Is Sufficient for Cell Cycle Re-Entry.? <i>NATURE</i> 424.6945 (2003): 223?228. Print.","pubMedId":"12853964","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/12853964","publicationId":24327,"title":"Acute mutation of retinoblastoma gene function is sufficient for cell cycle re-entry","type":"article","webOfScienceId":"000184032700049","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000184032700049"},{"abstract":"Targeted disruption of the retinoblastoma gene in mice leads to embryonic lethality in midgestation accompanied by defective erythropoiesis. Rb(-/-) embryos also exhibit inappropriate cell cycle activity and apoptosis in the central nervous system (CNS), peripheral nervous system (PNS), and ocular lens. Loss of p53 can prevent the apoptosis in the CNS and lens; however, the specific signals leading to p53 activation have not been determined. Here we test the hypothesis that hypoxia caused by defective erythropoiesis in Rb-null embryos contributes to p53-dependent apoptosis. We show evidence of hypoxia in CNS tissue from Rb(-/-) embryos. The Cre-loxP system was then used to generate embryos in which Rb was deleted in the CNS, PNS and lens, in the presence of normal erythropoiesis. In contrast to the massive CNS apoptosis in Rb-null embryos at embryonic day 13.5 (E13.5), conditional mutants did not have elevated apoptosis in this tissue. There was still significant apoptosis in the PNS and lens, however. Rb(-/-) cells in the CNS, PNS, and lens underwent inappropriate S-phase entry in the conditional mutants at E13.5. By E18.5, conditional mutants had increased brain size and weight as well as defects in skeletal muscle development. These data support a model in which hypoxia is a necessary cofactor in the death of CNS neurons in the developing Rb mutant embryo.","apaCitation":"Macpherson, D., Sage, J., Crowley, D., Trumpp, A., Bronson, R. T., &amp; Jacks, T. (2003). Conditional mutation of Rb causes cell cycle defects without apoptosis in the central nervous system. <i>MOLECULAR AND CELLULAR BIOLOGY</i>, <i>23</i>(3), 1044?1053.","capCitation":"<span class=\"title\">\n  <span>Conditional mutation of Rb causes cell cycle defects without apoptosis in the central nervous system</span>\n  <i>MOLECULAR AND CELLULAR BIOLOGY</i>\n</span>\n<span class=\"authors\">Macpherson, D., Sage, J., Crowley, D., Trumpp, A., Bronson, R. T., Jacks, T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2003</span>; <span class=\"volume\">23 (3)</span><span class=\"pages\">: 1044-1053</span>\n</span>","chicagoCitation":"Macpherson, D., J. Sage, D. Crowley, A. Trumpp, R. T. Bronson, et al. 2003. ?Conditional Mutation of Rb Causes Cell Cycle Defects without Apoptosis in the Central Nervous System.? <i>MOLECULAR AND CELLULAR BIOLOGY</i> 23 (3). AMER SOC MICROBIOLOGY: 1044?53.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Targeted disruption of the retinoblastoma gene in mice leads to embryonic lethality in midgestation accompanied by defective erythropoiesis. Rb(-/-) embryos also exhibit inappropriate cell cycle activity and apoptosis in the central nervous system (CNS), peripheral nervous system (PNS), and ocular lens. Loss of p53 can prevent the apoptosis in the CNS and lens; however, the specific signals leading to p53 activation have not been determined. Here we test the hypothesis that hypoxia caused by defective erythropoiesis in Rb-null embryos contributes to p53-dependent apoptosis. We show evidence of hypoxia in CNS tissue from Rb(-/-) embryos. The Cre-loxP system was then used to generate embryos in which Rb was deleted in the CNS, PNS and lens, in the presence of normal erythropoiesis. In contrast to the massive CNS apoptosis in Rb-null embryos at embryonic day 13.5 (E13.5), conditional mutants did not have elevated apoptosis in this tissue. There was still significant apoptosis in the PNS and lens, however. Rb(-/-) cells in the CNS, PNS, and lens underwent inappropriate S-phase entry in the conditional mutants at E13.5. By E18.5, conditional mutants had increased brain size and weight as well as defects in skeletal muscle development. These data support a model in which hypoxia is a necessary cofactor in the death of CNS neurons in the developing Rb mutant embryo.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1128/MCB.23.3.1044-1053.2003\">DOI 10.1128/MCB.23.3.1044-1053.2003</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000180673600026\">Web of Science ID 000180673600026</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/12529408\">PubMedID 12529408</a></span>\n  </p>","text":"Abstract:\nTargeted disruption of the retinoblastoma gene in mice leads to embryonic lethality in midgestation accompanied by defective erythropoiesis. Rb(-/-) embryos also exhibit inappropriate cell cycle activity and apoptosis in the central nervous system (CNS), peripheral nervous system (PNS), and ocular lens. Loss of p53 can prevent the apoptosis in the CNS and lens; however, the specific signals leading to p53 activation have not been determined. Here we test the hypothesis that hypoxia caused by defective erythropoiesis in Rb-null embryos contributes to p53-dependent apoptosis. We show evidence of hypoxia in CNS tissue from Rb(-/-) embryos. The Cre-loxP system was then used to generate embryos in which Rb was deleted in the CNS, PNS and lens, in the presence of normal erythropoiesis. In contrast to the massive CNS apoptosis in Rb-null embryos at embryonic day 13.5 (E13.5), conditional mutants did not have elevated apoptosis in this tissue. There was still significant apoptosis in the PNS and lens, however. Rb(-/-) cells in the CNS, PNS, and lens underwent inappropriate S-phase entry in the conditional mutants at E13.5. By E18.5, conditional mutants had increased brain size and weight as well as defects in skeletal muscle development. These data support a model in which hypoxia is a necessary cofactor in the death of CNS neurons in the developing Rb mutant embryo.\nDOI: 10.1128/MCB.23.3.1044-1053.2003, https://dx.doi.org/10.1128/MCB.23.3.1044-1053.2003\nWeb of Science ID: 000180673600026, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000180673600026\nPubMedID: 12529408, https://www.ncbi.nlm.nih.gov/pubmed/12529408"},"doiId":"10.1128/MCB.23.3.1044-1053.2003","doiUrl":"https://dx.doi.org/10.1128/MCB.23.3.1044-1053.2003","featured":false,"firstPublished":{"html":"Feb 1, 2003 00:00:00 AM","text":"Feb 1, 2003 00:00:00 AM","value":"2003-02-01T00:00:00.000-08:00"},"mlaCitation":"Macpherson, D. et al. ?Conditional Mutation of Rb Causes Cell Cycle Defects without Apoptosis in the Central Nervous System.? <i>MOLECULAR AND CELLULAR BIOLOGY</i> 23.3 (2003): 1044?1053. Print.","pubMedId":"12529408","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/12529408","publicationId":21084,"title":"Conditional mutation of Rb causes cell cycle defects without apoptosis in the central nervous system","type":"article","webOfScienceId":"000180673600026","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000180673600026"},{"abstract":"Defining the molecular mechanisms that coordinately regulate proliferation and differentiation is a central issue in development. Here, we describe a mechanism in which induction of the Ets repressor METS/PE1 links terminal differentiation to cell cycle arrest. Using macrophages as a model, we provide evidence that METS/PE1 blocks Ras-dependent proliferation without inhibiting Ras-dependent expression of cell type-specific genes by selectively replacing Ets activators on the promoters of cell cycle control genes. Antiproliferative effects of METS require its interaction with DP103, a DEAD box-containing protein that assembles a novel corepressor complex. Functional interactions between the METS/DP103 complex and E2F/ pRB family proteins are also necessary for inhibition of cellular proliferation, suggesting a combinatorial code that directs permanent cell cycle exit during terminal differentiation.","apaCitation":"Klappacher, G. W., Lunyak, V. V., Sykes, D. B., Sawka-Verhelle, D., Sage, J., Brard, G., ? Glass, C. K. (2002). An induced Ets repressor complex regulates growth arrest during terminal macrophage differentiation. <i>CELL</i>, <i>109</i>(2), 169?180.","capCitation":"<span class=\"title\">\n  <span>An induced Ets repressor complex regulates growth arrest during terminal macrophage differentiation</span>\n  <i>CELL</i>\n</span>\n<span class=\"authors\">Klappacher, G. W., Lunyak, V. V., Sykes, D. B., Sawka-Verhelle, D., Sage, J., Brard, G., Ngo, S. D., Gangadharan, D., Jacks, T., Kamps, M. P., Rose, D. W., Rosenfeld, M. G., Glass, C. K.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2002</span>; <span class=\"volume\">109 (2)</span><span class=\"pages\">: 169-180</span>\n</span>","chicagoCitation":"Klappacher, G. W., V. V. Lunyak, D. B. Sykes, D. Sawka-Verhelle, J. Sage, et al. 2002. ?An Induced Ets Repressor Complex Regulates Growth Arrest during Terminal Macrophage Differentiation.? <i>CELL</i> 109 (2). CELL PRESS: 169?80.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Defining the molecular mechanisms that coordinately regulate proliferation and differentiation is a central issue in development. Here, we describe a mechanism in which induction of the Ets repressor METS/PE1 links terminal differentiation to cell cycle arrest. Using macrophages as a model, we provide evidence that METS/PE1 blocks Ras-dependent proliferation without inhibiting Ras-dependent expression of cell type-specific genes by selectively replacing Ets activators on the promoters of cell cycle control genes. Antiproliferative effects of METS require its interaction with DP103, a DEAD box-containing protein that assembles a novel corepressor complex. Functional interactions between the METS/DP103 complex and E2F/ pRB family proteins are also necessary for inhibition of cellular proliferation, suggesting a combinatorial code that directs permanent cell cycle exit during terminal differentiation.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000175082600007\">Web of Science ID 000175082600007</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/12007404\">PubMedID 12007404</a></span>\n  </p>","text":"Abstract:\nDefining the molecular mechanisms that coordinately regulate proliferation and differentiation is a central issue in development. Here, we describe a mechanism in which induction of the Ets repressor METS/PE1 links terminal differentiation to cell cycle arrest. Using macrophages as a model, we provide evidence that METS/PE1 blocks Ras-dependent proliferation without inhibiting Ras-dependent expression of cell type-specific genes by selectively replacing Ets activators on the promoters of cell cycle control genes. Antiproliferative effects of METS require its interaction with DP103, a DEAD box-containing protein that assembles a novel corepressor complex. Functional interactions between the METS/DP103 complex and E2F/ pRB family proteins are also necessary for inhibition of cellular proliferation, suggesting a combinatorial code that directs permanent cell cycle exit during terminal differentiation.\n\nWeb of Science ID: 000175082600007, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000175082600007\nPubMedID: 12007404, https://www.ncbi.nlm.nih.gov/pubmed/12007404"},"featured":false,"firstPublished":{"html":"Apr 19, 2002 00:00:00 AM","text":"Apr 19, 2002 00:00:00 AM","value":"2002-04-19T00:00:00.000-07:00"},"mlaCitation":"Klappacher, G. W. et al. ?An Induced Ets Repressor Complex Regulates Growth Arrest during Terminal Macrophage Differentiation.? <i>CELL</i> 109.2 (2002): 169?180. Print.","pubMedId":"12007404","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/12007404","publicationId":242876,"title":"An induced Ets repressor complex regulates growth arrest during terminal macrophage differentiation","type":"article","webOfScienceId":"000175082600007","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000175082600007"},{"abstract":"The p53 tumor suppressor gene is the most frequently mutated gene in human cancers, and germ-line p53 mutations cause a familial predisposition for cancer. Germ-line or sporadic p53 mutations are usually missense and typically affect the central DNA-binding domain of the protein. Because p53 functions as a tetrameric transcription factor, mutant p53 is thought to inhibit the function of wild-type p53 protein. Here, we studied the possible dominant-negative inhibition of wild-type p53 protein by two different, frequently occurring point mutations. The R270H and P275S mutations were targeted into the genome of mouse embryonic stem cells to allow the analysis of the effects of the mutant proteins expressed in normal cells at single-copy levels. In embryonic stem cells, the presence of a heterozygous point-mutated allele resulted in delayed transcriptional activation of several p53 downstream target genes on exposure to gamma irradiation. Doxorubicin-induced apoptosis was severely affected in the mutant embryonic stem cells compared with wild-type cells. Heterozygous mutant thymocytes had a severe defect in p53-dependent apoptotic pathways after treatment with gamma irradiation or doxorubicin, whereas p53-independent apoptotic pathways were intact. Together these data demonstrate that physiological expression of point-mutated p53 can strongly limit overall cellular p53 function, supporting the dominant-negative action of such mutants. Also, cells heterozygous for such mutations may be compromised in terms of tumor suppression and response to chemotherapeutic agents.","apaCitation":"de Vries, A., Flores, E. R., Miranda, B., Hsieh, H. M., van Oostrom, C. T. M., Sage, J., &amp; Jacks, T. (2002). Targeted point mutations of p53 lead to dominant-negative inhibition of wild-type p53 function. <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i>, <i>99</i>(5), 2948?2953.","capCitation":"<span class=\"title\">\n  <span>Targeted point mutations of p53 lead to dominant-negative inhibition of wild-type p53 function</span>\n  <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i>\n</span>\n<span class=\"authors\">de Vries, A., Flores, E. R., Miranda, B., Hsieh, H. M., van Oostrom, C. T., Sage, J., Jacks, T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2002</span>; <span class=\"volume\">99 (5)</span><span class=\"pages\">: 2948-2953</span>\n</span>","chicagoCitation":"de Vries, A., E. R. Flores, B. Miranda, H. M. Hsieh, C. TM van Oostrom, et al. 2002. ?Targeted Point Mutations of p53 Lead to Dominant-Negative Inhibition of Wild-Type p53 Function.? <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i> 99 (5). NATL ACAD SCIENCES: 2948?53.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The p53 tumor suppressor gene is the most frequently mutated gene in human cancers, and germ-line p53 mutations cause a familial predisposition for cancer. Germ-line or sporadic p53 mutations are usually missense and typically affect the central DNA-binding domain of the protein. Because p53 functions as a tetrameric transcription factor, mutant p53 is thought to inhibit the function of wild-type p53 protein. Here, we studied the possible dominant-negative inhibition of wild-type p53 protein by two different, frequently occurring point mutations. The R270H and P275S mutations were targeted into the genome of mouse embryonic stem cells to allow the analysis of the effects of the mutant proteins expressed in normal cells at single-copy levels. In embryonic stem cells, the presence of a heterozygous point-mutated allele resulted in delayed transcriptional activation of several p53 downstream target genes on exposure to gamma irradiation. Doxorubicin-induced apoptosis was severely affected in the mutant embryonic stem cells compared with wild-type cells. Heterozygous mutant thymocytes had a severe defect in p53-dependent apoptotic pathways after treatment with gamma irradiation or doxorubicin, whereas p53-independent apoptotic pathways were intact. Together these data demonstrate that physiological expression of point-mutated p53 can strongly limit overall cellular p53 function, supporting the dominant-negative action of such mutants. Also, cells heterozygous for such mutations may be compromised in terms of tumor suppression and response to chemotherapeutic agents.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1073/pnas.052713099\">DOI 10.1073/pnas.052713099</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000174284600065\">Web of Science ID 000174284600065</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/11867759\">PubMedID 11867759</a></span>\n  </p>","text":"Abstract:\nThe p53 tumor suppressor gene is the most frequently mutated gene in human cancers, and germ-line p53 mutations cause a familial predisposition for cancer. Germ-line or sporadic p53 mutations are usually missense and typically affect the central DNA-binding domain of the protein. Because p53 functions as a tetrameric transcription factor, mutant p53 is thought to inhibit the function of wild-type p53 protein. Here, we studied the possible dominant-negative inhibition of wild-type p53 protein by two different, frequently occurring point mutations. The R270H and P275S mutations were targeted into the genome of mouse embryonic stem cells to allow the analysis of the effects of the mutant proteins expressed in normal cells at single-copy levels. In embryonic stem cells, the presence of a heterozygous point-mutated allele resulted in delayed transcriptional activation of several p53 downstream target genes on exposure to gamma irradiation. Doxorubicin-induced apoptosis was severely affected in the mutant embryonic stem cells compared with wild-type cells. Heterozygous mutant thymocytes had a severe defect in p53-dependent apoptotic pathways after treatment with gamma irradiation or doxorubicin, whereas p53-independent apoptotic pathways were intact. Together these data demonstrate that physiological expression of point-mutated p53 can strongly limit overall cellular p53 function, supporting the dominant-negative action of such mutants. Also, cells heterozygous for such mutations may be compromised in terms of tumor suppression and response to chemotherapeutic agents.\nDOI: 10.1073/pnas.052713099, https://dx.doi.org/10.1073/pnas.052713099\nWeb of Science ID: 000174284600065, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000174284600065\nPubMedID: 11867759, https://www.ncbi.nlm.nih.gov/pubmed/11867759"},"doiId":"10.1073/pnas.052713099","doiUrl":"https://dx.doi.org/10.1073/pnas.052713099","featured":false,"firstPublished":{"html":"Mar 5, 2002 00:00:00 AM","text":"Mar 5, 2002 00:00:00 AM","value":"2002-03-05T00:00:00.000-08:00"},"mlaCitation":"de Vries, A. et al. ?Targeted Point Mutations of p53 Lead to Dominant-Negative Inhibition of Wild-Type p53 Function.? <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i> 99.5 (2002): 2948?2953. Print.","pubMedId":"11867759","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/11867759","publicationId":242877,"title":"Targeted point mutations of p53 lead to dominant-negative inhibition of wild-type p53 function","type":"article","webOfScienceId":"000174284600065","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000174284600065"},{"apaCitation":"Sage, J. (2001). The RB gene family and control of the cell cycle. <i>BULLETIN DU CANCER</i>, <i>88</i>(6), 541?543.","capCitation":"<span class=\"title\">\n  <span>The RB gene family and control of the cell cycle</span>\n  <i>BULLETIN DU CANCER</i>\n</span>\n<span class=\"authors\">Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2001</span>; <span class=\"volume\">88 (6)</span><span class=\"pages\">: 541-543</span>\n</span>","chicagoCitation":"Sage, J. 2001. ?The RB Gene Family and Control of the Cell Cycle.? <i>BULLETIN DU CANCER</i> 88 (6). JOHN LIBBEY EUROTEXT LTD: 541?43.","detail":{"html":"<p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000169993400001\">Web of Science ID 000169993400001</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/11459699\">PubMedID 11459699</a></span>\n  </p>","text":"Web of Science ID: 000169993400001, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000169993400001\nPubMedID: 11459699, https://www.ncbi.nlm.nih.gov/pubmed/11459699"},"featured":false,"firstPublished":{"html":"Jun 1, 2001 00:00:00 AM","text":"Jun 1, 2001 00:00:00 AM","value":"2001-06-01T00:00:00.000-07:00"},"mlaCitation":"Sage, J. ?The RB Gene Family and Control of the Cell Cycle.? <i>BULLETIN DU CANCER</i> 88.6 (2001): 541?543. Print.","pubMedId":"11459699","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/11459699","publicationId":242878,"title":"The RB gene family and control of the cell cycle","type":"article","webOfScienceId":"000169993400001","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000169993400001"},{"abstract":"The retinoblastoma protein, pRB, and the closely related proteins p107 and p130 are important regulators of the mammalian cell cycle. Biochemical and genetic studies have demonstrated overlapping as well as distinct functions for the three proteins in cell cycle control and mouse development. However, the role of the pRB family as a whole in the regulation of cell proliferation, cell death, or cell differentiation is not known. We generated embryonic stem (ES) cells and other cell types mutant for all three genes. Triple knock-out mouse embryonic fibroblasts (TKO MEFs) had a shorter cell cycle than wild-type, single, or double knock-out control cells. TKO cells were resistant to G(1) arrest following DNA damage, despite retaining functional p53 activity. They were also insensitive to G(1) arrest signals following contact inhibition or serum starvation. Finally, TKO MEFs did not undergo senescence in culture and do possess some characteristics of transformed cells. Our results confirm the essential role of the Rb family in the control of the G(1)/S transition, place the three Rb family members downstream of multiple cell cycle control pathways, and further the link between loss of cell cycle control and tumorigenesis.","apaCitation":"Sage, J., Mulligan, G. J., Attardi, L. D., Miller, A., Chen, S. Q., Williams, B., ? Jacks, T. (2000). Targeted disruption of the three Rb-related genes leads to loss of G(1) control and immortalization. <i>GENES &amp; DEVELOPMENT</i>, <i>14</i>(23), 3037?3050.","capCitation":"<span class=\"title\">\n  <span>Targeted disruption of the three Rb-related genes leads to loss of G(1) control and immortalization</span>\n  <i>GENES &amp; DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Sage, J., Mulligan, G. J., Attardi, L. D., Miller, A., Chen, S. Q., Williams, B., Theodorou, E., Jacks, T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2000</span>; <span class=\"volume\">14 (23)</span><span class=\"pages\">: 3037-3050</span>\n</span>","chicagoCitation":"Sage, J., G. J. Mulligan, L. D. Attardi, A. Miller, S. Q. Chen, et al. 2000. ?Targeted Disruption of the Three Rb-Related Genes Leads to Loss of G(1) Control and Immortalization.? <i>GENES &amp; DEVELOPMENT</i> 14 (23). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 3037?50.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma protein, pRB, and the closely related proteins p107 and p130 are important regulators of the mammalian cell cycle. Biochemical and genetic studies have demonstrated overlapping as well as distinct functions for the three proteins in cell cycle control and mouse development. However, the role of the pRB family as a whole in the regulation of cell proliferation, cell death, or cell differentiation is not known. We generated embryonic stem (ES) cells and other cell types mutant for all three genes. Triple knock-out mouse embryonic fibroblasts (TKO MEFs) had a shorter cell cycle than wild-type, single, or double knock-out control cells. TKO cells were resistant to G(1) arrest following DNA damage, despite retaining functional p53 activity. They were also insensitive to G(1) arrest signals following contact inhibition or serum starvation. Finally, TKO MEFs did not undergo senescence in culture and do possess some characteristics of transformed cells. Our results confirm the essential role of the Rb family in the control of the G(1)/S transition, place the three Rb family members downstream of multiple cell cycle control pathways, and further the link between loss of cell cycle control and tumorigenesis.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000165788600011\">Web of Science ID 000165788600011</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/11114892\">PubMedID 11114892</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma protein, pRB, and the closely related proteins p107 and p130 are important regulators of the mammalian cell cycle. Biochemical and genetic studies have demonstrated overlapping as well as distinct functions for the three proteins in cell cycle control and mouse development. However, the role of the pRB family as a whole in the regulation of cell proliferation, cell death, or cell differentiation is not known. We generated embryonic stem (ES) cells and other cell types mutant for all three genes. Triple knock-out mouse embryonic fibroblasts (TKO MEFs) had a shorter cell cycle than wild-type, single, or double knock-out control cells. TKO cells were resistant to G(1) arrest following DNA damage, despite retaining functional p53 activity. They were also insensitive to G(1) arrest signals following contact inhibition or serum starvation. Finally, TKO MEFs did not undergo senescence in culture and do possess some characteristics of transformed cells. Our results confirm the essential role of the Rb family in the control of the G(1)/S transition, place the three Rb family members downstream of multiple cell cycle control pathways, and further the link between loss of cell cycle control and tumorigenesis.\n\nWeb of Science ID: 000165788600011, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000165788600011\nPubMedID: 11114892, https://www.ncbi.nlm.nih.gov/pubmed/11114892"},"featured":true,"firstPublished":{"html":"Dec 1, 2000 00:00:00 AM","text":"Dec 1, 2000 00:00:00 AM","value":"2000-12-01T00:00:00.000-08:00"},"mlaCitation":"Sage, J. et al. ?Targeted Disruption of the Three Rb-Related Genes Leads to Loss of G(1) Control and Immortalization.? <i>GENES &amp; DEVELOPMENT</i> 14.23 (2000): 3037?3050. Print.","pubMedId":"11114892","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/11114892","publicationId":8961,"title":"Targeted disruption of the three Rb-related genes leads to loss of G(1) control and immortalization","type":"article","webOfScienceId":"000165788600011","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000165788600011"},{"abstract":"The retinoblastoma (Rb) gene product is a prototypic tumor suppressor. Mice lacking the Rb gene are not viable and die in utero at approximately 13 days of gestation. In this study, we have rescued Rb-/- prostates by grafting pelvic organ rudiments from Rb-/- mouse embryos under the renal capsule of adult male nude mouse hosts. Grafts of embryonic pelvic organs developed into functional prostatic tissue. Some of the prostatic tissue generated was further used to construct chimeric prostatic tissue recombinants by combining wild-type rat urogenital mesenchyme (rUGM) with Rb-/- and Rb+/+ prostatic epithelium (PRE). The tissue recombinants were grown as subcapsular renal grafts and treated from the time of grafting with Silastic capsules containing 25 mg of testosterone plus 2.5 mg of estradiol. During 5-8 weeks of hormone treatment, rUGM+Rb+/+PRE tissue recombinants developed prostatic hyperplasia, whereas PRE in rUGM+Rb-/-PRE tissue recombinants developed hyperplasia, atypical hyperplasia, and carcinoma. During carcinogenesis in rUGM+Rb-/-PRE tissue recombinants, prostatic epithelial cells of the basal lineage disappeared, whereas the luminal cells underwent carcinogenesis. Epithelial E-cadherin almost totally disappeared. In all cases, epithelial PCNA labeling was elevated in tissue recombinants containing Rb-/- versus Rb+/+ epithelium. These epithelial changes were associated with almost total loss of smooth muscle cells in the stroma. In contrast, in untreated hosts rUGM+Rb+/+PRE tissue recombinants developed normally, and rUGM+Rb-/-PRE tissue recombinants developed mild epithelial hyperplasia. The results of this study demonstrate that Rb-/- prostatic tissue can be rescued from embryonic lethal mice and used to test its susceptibility to hormonal carcinogenesis. Deletion of the Rb gene predisposes prostatic epithelium to hyperplasia and increases proliferative activity Susceptibility to hormonal carcinogenesis in response to exogenous testosterone + estradiol is manifested in the progression from atypica hyperplasia to carcinoma. Thus, these findings demonstrate that the absence of the Rb tumor suppressor gene may predispose prostatic epithelial cells to carcinogenesis. Rescue of organs from Rb-/- embryos not only provides an opportunity to analyze the Rb gene pathway in the development and progression of prostate cancer but also provides an opportunity for specifically evaluating the role of the Rb pathway in development and carcinogenesis in other organs, such as the mammary gland and colon. Because rUGM greatly stimulates prostatic epithelial proliferation, the tissue recombinant model is a particularly useful tool for assessing the functional role of other genes in prostatic carcinogenesis through use of the appropriate transgenic or gene knockout mice.","apaCitation":"Wang, Y. H., Hayward, S. W., Donjacour, A. A., Young, P., Jacks, T., Sage, J., ? Cunha, G. R. (2000). Sex hormone-induced carcinogenesis in Rb-deficient prostate tissue. <i>CANCER RESEARCH</i>, <i>60</i>(21), 6008?6017.","capCitation":"<span class=\"title\">\n  <span>Sex hormone-induced carcinogenesis in Rb-deficient prostate tissue</span>\n  <i>CANCER RESEARCH</i>\n</span>\n<span class=\"authors\">Wang, Y. H., Hayward, S. W., Donjacour, A. A., Young, P., Jacks, T., Sage, J., Dahiya, R., Cardiff, R. D., Day, M. L., Cunha, G. R.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2000</span>; <span class=\"volume\">60 (21)</span><span class=\"pages\">: 6008-6017</span>\n</span>","chicagoCitation":"Wang, Y. H., S. W. Hayward, A. A. Donjacour, P. Young, T. Jacks, et al. 2000. ?Sex Hormone-Induced Carcinogenesis in Rb-Deficient Prostate Tissue.? <i>CANCER RESEARCH</i> 60 (21). AMER ASSOC CANCER RESEARCH: 6008?17.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma (Rb) gene product is a prototypic tumor suppressor. Mice lacking the Rb gene are not viable and die in utero at approximately 13 days of gestation. In this study, we have rescued Rb-/- prostates by grafting pelvic organ rudiments from Rb-/- mouse embryos under the renal capsule of adult male nude mouse hosts. Grafts of embryonic pelvic organs developed into functional prostatic tissue. Some of the prostatic tissue generated was further used to construct chimeric prostatic tissue recombinants by combining wild-type rat urogenital mesenchyme (rUGM) with Rb-/- and Rb+/+ prostatic epithelium (PRE). The tissue recombinants were grown as subcapsular renal grafts and treated from the time of grafting with Silastic capsules containing 25 mg of testosterone plus 2.5 mg of estradiol. During 5-8 weeks of hormone treatment, rUGM+Rb+/+PRE tissue recombinants developed prostatic hyperplasia, whereas PRE in rUGM+Rb-/-PRE tissue recombinants developed hyperplasia, atypical hyperplasia, and carcinoma. During carcinogenesis in rUGM+Rb-/-PRE tissue recombinants, prostatic epithelial cells of the basal lineage disappeared, whereas the luminal cells underwent carcinogenesis. Epithelial E-cadherin almost totally disappeared. In all cases, epithelial PCNA labeling was elevated in tissue recombinants containing Rb-/- versus Rb+/+ epithelium. These epithelial changes were associated with almost total loss of smooth muscle cells in the stroma. In contrast, in untreated hosts rUGM+Rb+/+PRE tissue recombinants developed normally, and rUGM+Rb-/-PRE tissue recombinants developed mild epithelial hyperplasia. The results of this study demonstrate that Rb-/- prostatic tissue can be rescued from embryonic lethal mice and used to test its susceptibility to hormonal carcinogenesis. Deletion of the Rb gene predisposes prostatic epithelium to hyperplasia and increases proliferative activity Susceptibility to hormonal carcinogenesis in response to exogenous testosterone + estradiol is manifested in the progression from atypica hyperplasia to carcinoma. Thus, these findings demonstrate that the absence of the Rb tumor suppressor gene may predispose prostatic epithelial cells to carcinogenesis. Rescue of organs from Rb-/- embryos not only provides an opportunity to analyze the Rb gene pathway in the development and progression of prostate cancer but also provides an opportunity for specifically evaluating the role of the Rb pathway in development and carcinogenesis in other organs, such as the mammary gland and colon. Because rUGM greatly stimulates prostatic epithelial proliferation, the tissue recombinant model is a particularly useful tool for assessing the functional role of other genes in prostatic carcinogenesis through use of the appropriate transgenic or gene knockout mice.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000165230300021\">Web of Science ID 000165230300021</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/11085521\">PubMedID 11085521</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma (Rb) gene product is a prototypic tumor suppressor. Mice lacking the Rb gene are not viable and die in utero at approximately 13 days of gestation. In this study, we have rescued Rb-/- prostates by grafting pelvic organ rudiments from Rb-/- mouse embryos under the renal capsule of adult male nude mouse hosts. Grafts of embryonic pelvic organs developed into functional prostatic tissue. Some of the prostatic tissue generated was further used to construct chimeric prostatic tissue recombinants by combining wild-type rat urogenital mesenchyme (rUGM) with Rb-/- and Rb+/+ prostatic epithelium (PRE). The tissue recombinants were grown as subcapsular renal grafts and treated from the time of grafting with Silastic capsules containing 25 mg of testosterone plus 2.5 mg of estradiol. During 5-8 weeks of hormone treatment, rUGM+Rb+/+PRE tissue recombinants developed prostatic hyperplasia, whereas PRE in rUGM+Rb-/-PRE tissue recombinants developed hyperplasia, atypical hyperplasia, and carcinoma. During carcinogenesis in rUGM+Rb-/-PRE tissue recombinants, prostatic epithelial cells of the basal lineage disappeared, whereas the luminal cells underwent carcinogenesis. Epithelial E-cadherin almost totally disappeared. In all cases, epithelial PCNA labeling was elevated in tissue recombinants containing Rb-/- versus Rb+/+ epithelium. These epithelial changes were associated with almost total loss of smooth muscle cells in the stroma. In contrast, in untreated hosts rUGM+Rb+/+PRE tissue recombinants developed normally, and rUGM+Rb-/-PRE tissue recombinants developed mild epithelial hyperplasia. The results of this study demonstrate that Rb-/- prostatic tissue can be rescued from embryonic lethal mice and used to test its susceptibility to hormonal carcinogenesis. Deletion of the Rb gene predisposes prostatic epithelium to hyperplasia and increases proliferative activity Susceptibility to hormonal carcinogenesis in response to exogenous testosterone + estradiol is manifested in the progression from atypica hyperplasia to carcinoma. Thus, these findings demonstrate that the absence of the Rb tumor suppressor gene may predispose prostatic epithelial cells to carcinogenesis. Rescue of organs from Rb-/- embryos not only provides an opportunity to analyze the Rb gene pathway in the development and progression of prostate cancer but also provides an opportunity for specifically evaluating the role of the Rb pathway in development and carcinogenesis in other organs, such as the mammary gland and colon. Because rUGM greatly stimulates prostatic epithelial proliferation, the tissue recombinant model is a particularly useful tool for assessing the functional role of other genes in prostatic carcinogenesis through use of the appropriate transgenic or gene knockout mice.\n\nWeb of Science ID: 000165230300021, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000165230300021\nPubMedID: 11085521, https://www.ncbi.nlm.nih.gov/pubmed/11085521"},"featured":false,"firstPublished":{"html":"Nov 1, 2000 00:00:00 AM","text":"Nov 1, 2000 00:00:00 AM","value":"2000-11-01T00:00:00.000-08:00"},"mlaCitation":"Wang, Y. H. et al. ?Sex Hormone-Induced Carcinogenesis in Rb-Deficient Prostate Tissue.? <i>CANCER RESEARCH</i> 60.21 (2000): 6008?6017. Print.","pubMedId":"11085521","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/11085521","publicationId":242879,"title":"Sex hormone-induced carcinogenesis in Rb-deficient prostate tissue","type":"article","webOfScienceId":"000165230300021","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000165230300021"},{"abstract":"To analyze NF-kappa B activity in the testis, we used murine transgenic lines carrying a LacZ reporter gene under the control of a NF-kappa B-responsive promoter (Schmidt-Ullrich et al. [1996] Dev 122:2117-2128). We constructed three independent lines containing the promoter of the gene encoding p105, the precursor of the p50 subunit. This promoter contains three NF-kappa B-binding sites in its proximal part. Our results show that in adult mice, the beta-galactosidase activity which reflects nuclear NF-kappa B activity, is first detected in spermatocytes at the pachytene stage and remains activated in the following steps of germ cell differentiation and maturation. Using transgenic mice carrying a p105nlslacZ construct with the 3 NF-kappa B sites mutated in the p105 promoter, we found a significant reduction in the transgene activity, confirming the important role of NF-kappa B in the activation of the transgene. To confirm the stage of induction during spermatogenesis, we analysed the beta-galactosidase activity in the testes from prepuberal mice in which cells synchrouneously enter meiosis. We detected the transgene activity at 18 days after birth, corresponding to the pachytene stage in spermatocytes. In nuclear extracts prepared from prepuberal mice, we found a peak of NF-kappa B DNA-binding activity made of p50 and p65 subunits at day 18 after birth, which remains high in the later stages. Further analysis showed that I kappa B alpha and beta, but not epsilon are expressed in the testes. Altogether, these data suggest that NF-kappa B factors are stage specifically controlled and may play a role during the development of sperm cells.","apaCitation":"Lilienbaum, A., Sage, J., Memet, S., Rassoulzadegan, M., Cuzin, F., &amp; Israel, A. (2000). NF-kappa B is developmentally regulated during spermatogenesis in mice. <i>DEVELOPMENTAL DYNAMICS</i>, <i>219</i>(3), 333?340.","capCitation":"<span class=\"title\">\n  <span>NF-kappa B is developmentally regulated during spermatogenesis in mice</span>\n  <i>DEVELOPMENTAL DYNAMICS</i>\n</span>\n<span class=\"authors\">Lilienbaum, A., Sage, J., Memet, S., Rassoulzadegan, M., Cuzin, F., Israel, A.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2000</span>; <span class=\"volume\">219 (3)</span><span class=\"pages\">: 333-340</span>\n</span>","chicagoCitation":"Lilienbaum, A., J. Sage, S. Memet, M. Rassoulzadegan, F. Cuzin, et al. 2000. ?NF-Kappa B Is Developmentally Regulated during Spermatogenesis in Mice.? <i>DEVELOPMENTAL DYNAMICS</i> 219 (3). WILEY-LISS: 333?40.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">To analyze NF-kappa B activity in the testis, we used murine transgenic lines carrying a LacZ reporter gene under the control of a NF-kappa B-responsive promoter (Schmidt-Ullrich et al. [1996] Dev 122:2117-2128). We constructed three independent lines containing the promoter of the gene encoding p105, the precursor of the p50 subunit. This promoter contains three NF-kappa B-binding sites in its proximal part. Our results show that in adult mice, the beta-galactosidase activity which reflects nuclear NF-kappa B activity, is first detected in spermatocytes at the pachytene stage and remains activated in the following steps of germ cell differentiation and maturation. Using transgenic mice carrying a p105nlslacZ construct with the 3 NF-kappa B sites mutated in the p105 promoter, we found a significant reduction in the transgene activity, confirming the important role of NF-kappa B in the activation of the transgene. To confirm the stage of induction during spermatogenesis, we analysed the beta-galactosidase activity in the testes from prepuberal mice in which cells synchrouneously enter meiosis. We detected the transgene activity at 18 days after birth, corresponding to the pachytene stage in spermatocytes. In nuclear extracts prepared from prepuberal mice, we found a peak of NF-kappa B DNA-binding activity made of p50 and p65 subunits at day 18 after birth, which remains high in the later stages. Further analysis showed that I kappa B alpha and beta, but not epsilon are expressed in the testes. Altogether, these data suggest that NF-kappa B factors are stage specifically controlled and may play a role during the development of sperm cells.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000165122200005\">Web of Science ID 000165122200005</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/11066090\">PubMedID 11066090</a></span>\n  </p>","text":"Abstract:\nTo analyze NF-kappa B activity in the testis, we used murine transgenic lines carrying a LacZ reporter gene under the control of a NF-kappa B-responsive promoter (Schmidt-Ullrich et al. [1996] Dev 122:2117-2128). We constructed three independent lines containing the promoter of the gene encoding p105, the precursor of the p50 subunit. This promoter contains three NF-kappa B-binding sites in its proximal part. Our results show that in adult mice, the beta-galactosidase activity which reflects nuclear NF-kappa B activity, is first detected in spermatocytes at the pachytene stage and remains activated in the following steps of germ cell differentiation and maturation. Using transgenic mice carrying a p105nlslacZ construct with the 3 NF-kappa B sites mutated in the p105 promoter, we found a significant reduction in the transgene activity, confirming the important role of NF-kappa B in the activation of the transgene. To confirm the stage of induction during spermatogenesis, we analysed the beta-galactosidase activity in the testes from prepuberal mice in which cells synchrouneously enter meiosis. We detected the transgene activity at 18 days after birth, corresponding to the pachytene stage in spermatocytes. In nuclear extracts prepared from prepuberal mice, we found a peak of NF-kappa B DNA-binding activity made of p50 and p65 subunits at day 18 after birth, which remains high in the later stages. Further analysis showed that I kappa B alpha and beta, but not epsilon are expressed in the testes. Altogether, these data suggest that NF-kappa B factors are stage specifically controlled and may play a role during the development of sperm cells.\n\nWeb of Science ID: 000165122200005, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000165122200005\nPubMedID: 11066090, https://www.ncbi.nlm.nih.gov/pubmed/11066090"},"featured":false,"firstPublished":{"html":"Nov 1, 2000 00:00:00 AM","text":"Nov 1, 2000 00:00:00 AM","value":"2000-11-01T00:00:00.000-08:00"},"mlaCitation":"Lilienbaum, A. et al. ?NF-Kappa B Is Developmentally Regulated during Spermatogenesis in Mice.? <i>DEVELOPMENTAL DYNAMICS</i> 219.3 (2000): 333?340. Print.","pubMedId":"11066090","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/11066090","publicationId":242880,"title":"NF-kappa B is developmentally regulated during spermatogenesis in mice","type":"article","webOfScienceId":"000165122200005","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000165122200005"},{"abstract":"Angiogenesis has been demonstrated to be essential for tumor growth and metastasis, and inhibition of angiogenesis is emerging as a promising strategy for treating cancer. Among the most potent inhibitors of angiogenesis is the fumagillin family of natural products. An analog of fumagillin, known as TNP-470 or AGM-1470, has been undergoing clinical trials for treating a variety of cancers. TNP-470 has been shown to block endothelial cell cycle progression in the late G(1) phase. Although the direct molecular target for TNP-470 has been identified as the type 2 methionine aminopeptidase (MetAP2), how inhibition of this enzyme leads to cell cycle arrest has remained unclear. We report that treatment of endothelial and other drug-sensitive cell types leads to the activation of the p53 pathway, causing an accumulation of the G(1) cyclin-dependent kinase inhibitor p21(WAF1/CIP1). The requirement of p53 and p21(WAF1/CIP1) for the cell cycle inhibition by TNP-470 is underscored by the observation that cells deficient in p53 and p21(WAF1/CIP1) are resistant to TNP-470. These results shed significant light on the mechanism of cell cycle inhibition by TNP-470 and suggest an alternative method of activating p53 in endothelial cells to halt angiogenesis and tumor progression.","apaCitation":"Zhang, Y., Griffith, E. C., Sage, J., Jacks, T., &amp; Liu, J. O. (2000). Cell cycle inhibition by the anti-angiogenic agent TNP-470 is mediated by p53 and p21(WAF1/CIP1). <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i>, <i>97</i>(12), 6427?6432.","capCitation":"<span class=\"title\">\n  <span>Cell cycle inhibition by the anti-angiogenic agent TNP-470 is mediated by p53 and p21(WAF1/CIP1)</span>\n  <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i>\n</span>\n<span class=\"authors\">Zhang, Y., Griffith, E. C., Sage, J., Jacks, T., Liu, J. O.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2000</span>; <span class=\"volume\">97 (12)</span><span class=\"pages\">: 6427-6432</span>\n</span>","chicagoCitation":"Zhang, Y., E. C. Griffith, J. Sage, T. Jacks, and J. O. Liu. 2000. ?Cell Cycle Inhibition by the Anti-Angiogenic Agent TNP-470 Is Mediated by p53 and p21(WAF1/CIP1).? <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i> 97 (12). NATL ACAD SCIENCES: 6427?32.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Angiogenesis has been demonstrated to be essential for tumor growth and metastasis, and inhibition of angiogenesis is emerging as a promising strategy for treating cancer. Among the most potent inhibitors of angiogenesis is the fumagillin family of natural products. An analog of fumagillin, known as TNP-470 or AGM-1470, has been undergoing clinical trials for treating a variety of cancers. TNP-470 has been shown to block endothelial cell cycle progression in the late G(1) phase. Although the direct molecular target for TNP-470 has been identified as the type 2 methionine aminopeptidase (MetAP2), how inhibition of this enzyme leads to cell cycle arrest has remained unclear. We report that treatment of endothelial and other drug-sensitive cell types leads to the activation of the p53 pathway, causing an accumulation of the G(1) cyclin-dependent kinase inhibitor p21(WAF1/CIP1). The requirement of p53 and p21(WAF1/CIP1) for the cell cycle inhibition by TNP-470 is underscored by the observation that cells deficient in p53 and p21(WAF1/CIP1) are resistant to TNP-470. These results shed significant light on the mechanism of cell cycle inhibition by TNP-470 and suggest an alternative method of activating p53 in endothelial cells to halt angiogenesis and tumor progression.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000087526300037\">Web of Science ID 000087526300037</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/10841547\">PubMedID 10841547</a></span>\n  </p>","text":"Abstract:\nAngiogenesis has been demonstrated to be essential for tumor growth and metastasis, and inhibition of angiogenesis is emerging as a promising strategy for treating cancer. Among the most potent inhibitors of angiogenesis is the fumagillin family of natural products. An analog of fumagillin, known as TNP-470 or AGM-1470, has been undergoing clinical trials for treating a variety of cancers. TNP-470 has been shown to block endothelial cell cycle progression in the late G(1) phase. Although the direct molecular target for TNP-470 has been identified as the type 2 methionine aminopeptidase (MetAP2), how inhibition of this enzyme leads to cell cycle arrest has remained unclear. We report that treatment of endothelial and other drug-sensitive cell types leads to the activation of the p53 pathway, causing an accumulation of the G(1) cyclin-dependent kinase inhibitor p21(WAF1/CIP1). The requirement of p53 and p21(WAF1/CIP1) for the cell cycle inhibition by TNP-470 is underscored by the observation that cells deficient in p53 and p21(WAF1/CIP1) are resistant to TNP-470. These results shed significant light on the mechanism of cell cycle inhibition by TNP-470 and suggest an alternative method of activating p53 in endothelial cells to halt angiogenesis and tumor progression.\n\nWeb of Science ID: 000087526300037, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000087526300037\nPubMedID: 10841547, https://www.ncbi.nlm.nih.gov/pubmed/10841547"},"featured":false,"firstPublished":{"html":"Jun 6, 2000 00:00:00 AM","text":"Jun 6, 2000 00:00:00 AM","value":"2000-06-06T00:00:00.000-07:00"},"mlaCitation":"Zhang, Y. et al. ?Cell Cycle Inhibition by the Anti-Angiogenic Agent TNP-470 Is Mediated by p53 and p21(WAF1/CIP1).? <i>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</i> 97.12 (2000): 6427?6432. Print.","pubMedId":"10841547","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/10841547","publicationId":242881,"title":"Cell cycle inhibition by the anti-angiogenic agent TNP-470 is mediated by p53 and p21(WAF1/CIP1)","type":"article","webOfScienceId":"000087526300037","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000087526300037"},{"abstract":"Transcription controls active at the initial stages of meiosis are clearly key elements in the regulation of germinal differentiation. Transcription of the Sycp1 gene (synaptonemal complex protein 1) starts as early as the leptotene and zygotene stages. Constructs with Sycp1 5' upstream sequences directed the expression of reporter genes to pachytene spermatocytes in transgenic mice. A short fragment encompassing the transcription start (n.t. -54 to +102) was sufficient for stage-specific expression in the adult male and for temporal regulation during development. Upstream enhancer element(s) quantitatively regulating expression were localized in the region between -54 and -260. The gene is normally expressed both in the male and female gonads, but none of the promoter sequences active in the testis allowed the expression of reporter genes during meiosis in the ovary.","apaCitation":"Sage, J., Martin, L., Meuwissen, R., Heyting, C., Cuzin, F., &amp; Rassoulzadegan, M. (1999). Temporal and spatial control of the Sycp1 gene transcription in the mouse meiosis: regulatory elements active in the male are not sufficient for expression in the female gonad. <i>MECHANISMS OF DEVELOPMENT</i>, <i>80</i>(1), 29?39.","capCitation":"<span class=\"title\">\n  <span>Temporal and spatial control of the Sycp1 gene transcription in the mouse meiosis: regulatory elements active in the male are not sufficient for expression in the female gonad</span>\n  <i>MECHANISMS OF DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Sage, J., Martin, L., Meuwissen, R., Heyting, C., Cuzin, F., Rassoulzadegan, M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">1999</span>; <span class=\"volume\">80 (1)</span><span class=\"pages\">: 29-39</span>\n</span>","chicagoCitation":"Sage, J., L. Martin, R. Meuwissen, C. Heyting, F. Cuzin, et al. 1999. ?Temporal and Spatial Control of the Sycp1 Gene Transcription in the Mouse Meiosis: Regulatory Elements Active in the Male Are Not Sufficient for Expression in the Female Gonad.? <i>MECHANISMS OF DEVELOPMENT</i> 80 (1). ELSEVIER SCIENCE BV: 29?39.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Transcription controls active at the initial stages of meiosis are clearly key elements in the regulation of germinal differentiation. Transcription of the Sycp1 gene (synaptonemal complex protein 1) starts as early as the leptotene and zygotene stages. Constructs with Sycp1 5' upstream sequences directed the expression of reporter genes to pachytene spermatocytes in transgenic mice. A short fragment encompassing the transcription start (n.t. -54 to +102) was sufficient for stage-specific expression in the adult male and for temporal regulation during development. Upstream enhancer element(s) quantitatively regulating expression were localized in the region between -54 and -260. The gene is normally expressed both in the male and female gonads, but none of the promoter sequences active in the testis allowed the expression of reporter genes during meiosis in the ovary.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000078473700003\">Web of Science ID 000078473700003</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/10096061\">PubMedID 10096061</a></span>\n  </p>","text":"Abstract:\nTranscription controls active at the initial stages of meiosis are clearly key elements in the regulation of germinal differentiation. Transcription of the Sycp1 gene (synaptonemal complex protein 1) starts as early as the leptotene and zygotene stages. Constructs with Sycp1 5' upstream sequences directed the expression of reporter genes to pachytene spermatocytes in transgenic mice. A short fragment encompassing the transcription start (n.t. -54 to +102) was sufficient for stage-specific expression in the adult male and for temporal regulation during development. Upstream enhancer element(s) quantitatively regulating expression were localized in the region between -54 and -260. The gene is normally expressed both in the male and female gonads, but none of the promoter sequences active in the testis allowed the expression of reporter genes during meiosis in the ovary.\n\nWeb of Science ID: 000078473700003, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000078473700003\nPubMedID: 10096061, https://www.ncbi.nlm.nih.gov/pubmed/10096061"},"featured":false,"firstPublished":{"html":"Jan 1, 1999 00:00:00 AM","text":"Jan 1, 1999 00:00:00 AM","value":"1999-01-01T00:00:00.000-08:00"},"mlaCitation":"Sage, J. et al. ?Temporal and Spatial Control of the Sycp1 Gene Transcription in the Mouse Meiosis: Regulatory Elements Active in the Male Are Not Sufficient for Expression in the Female Gonad.? <i>MECHANISMS OF DEVELOPMENT</i> 80.1 (1999): 29?39. Print.","pubMedId":"10096061","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/10096061","publicationId":242883,"title":"Temporal and spatial control of the Sycp1 gene transcription in the mouse meiosis: regulatory elements active in the male are not sufficient for expression in the female gonad","type":"article","webOfScienceId":"000078473700003","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000078473700003"},{"abstract":"Transgenic mice were generated expressing a testicular Cre recombinase driven by promoter sequences derived from the gene encoding Synaptonemal Complex Protein 1 (Sycp1), expressed at an early stage of the male meiosis (leptotene to zygotene). Recombination at target LoxP sites was examined during germinal differentiation in mice harboring Sycp1-Cre and a second transgene where LoxP sites flank either the beta geo coding region, the Pgk1 promoter, or a tk-neo cassette inserted into the Rxr alpha locus. The LoxP-flanked transgenes were stably maintained in the somatic tissues of the double transgenic animals, as well as in the progeny of the females. Mice born after mating the double-transgenic males with normal females showed extensive deletions of the LoxP-flanked sequences. When the males were hemizygous for the Sycp1-Cre transgene, the deletions were observed even in the fraction of the offspring which had not inherited the Cre gene, thus demonstrating that expression occurred in the male parent during spermatogenesis. The high efficiency of excision at the LoxP sites makes the Sycp1-Cre transgenic males suitable for evaluating the role of defined gene functions in the germinal differentiation process.","apaCitation":"Vidal, F., Sage, J., Cuzin, F., &amp; Rassoulzadegan, M. (1998). Cre expression in primary spermatocytes: A tool for genetic engineering of the germ line. <i>MOLECULAR REPRODUCTION AND DEVELOPMENT</i>, <i>51</i>(3), 274?280.","capCitation":"<span class=\"title\">\n  <span>Cre expression in primary spermatocytes: A tool for genetic engineering of the germ line</span>\n  <i>MOLECULAR REPRODUCTION AND DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Vidal, F., Sage, J., Cuzin, F., Rassoulzadegan, M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">1998</span>; <span class=\"volume\">51 (3)</span><span class=\"pages\">: 274-280</span>\n</span>","chicagoCitation":"Vidal, F., J. Sage, F. Cuzin, and M. Rassoulzadegan. 1998. ?Cre Expression in Primary Spermatocytes: A Tool for Genetic Engineering of the Germ Line.? <i>MOLECULAR REPRODUCTION AND DEVELOPMENT</i> 51 (3). WILEY-LISS: 274?80.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Transgenic mice were generated expressing a testicular Cre recombinase driven by promoter sequences derived from the gene encoding Synaptonemal Complex Protein 1 (Sycp1), expressed at an early stage of the male meiosis (leptotene to zygotene). Recombination at target LoxP sites was examined during germinal differentiation in mice harboring Sycp1-Cre and a second transgene where LoxP sites flank either the beta geo coding region, the Pgk1 promoter, or a tk-neo cassette inserted into the Rxr alpha locus. The LoxP-flanked transgenes were stably maintained in the somatic tissues of the double transgenic animals, as well as in the progeny of the females. Mice born after mating the double-transgenic males with normal females showed extensive deletions of the LoxP-flanked sequences. When the males were hemizygous for the Sycp1-Cre transgene, the deletions were observed even in the fraction of the offspring which had not inherited the Cre gene, thus demonstrating that expression occurred in the male parent during spermatogenesis. The high efficiency of excision at the LoxP sites makes the Sycp1-Cre transgenic males suitable for evaluating the role of defined gene functions in the germinal differentiation process.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000076145500006\">Web of Science ID 000076145500006</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/9771647\">PubMedID 9771647</a></span>\n  </p>","text":"Abstract:\nTransgenic mice were generated expressing a testicular Cre recombinase driven by promoter sequences derived from the gene encoding Synaptonemal Complex Protein 1 (Sycp1), expressed at an early stage of the male meiosis (leptotene to zygotene). Recombination at target LoxP sites was examined during germinal differentiation in mice harboring Sycp1-Cre and a second transgene where LoxP sites flank either the beta geo coding region, the Pgk1 promoter, or a tk-neo cassette inserted into the Rxr alpha locus. The LoxP-flanked transgenes were stably maintained in the somatic tissues of the double transgenic animals, as well as in the progeny of the females. Mice born after mating the double-transgenic males with normal females showed extensive deletions of the LoxP-flanked sequences. When the males were hemizygous for the Sycp1-Cre transgene, the deletions were observed even in the fraction of the offspring which had not inherited the Cre gene, thus demonstrating that expression occurred in the male parent during spermatogenesis. The high efficiency of excision at the LoxP sites makes the Sycp1-Cre transgenic males suitable for evaluating the role of defined gene functions in the germinal differentiation process.\n\nWeb of Science ID: 000076145500006, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000076145500006\nPubMedID: 9771647, https://www.ncbi.nlm.nih.gov/pubmed/9771647"},"featured":false,"firstPublished":{"html":"Nov 1, 1998 00:00:00 AM","text":"Nov 1, 1998 00:00:00 AM","value":"1998-11-01T00:00:00.000-08:00"},"mlaCitation":"Vidal, F. et al. ?Cre Expression in Primary Spermatocytes: A Tool for Genetic Engineering of the Germ Line.? <i>MOLECULAR REPRODUCTION AND DEVELOPMENT</i> 51.3 (1998): 274?280. Print.","pubMedId":"9771647","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/9771647","publicationId":242886,"title":"Cre expression in primary spermatocytes: A tool for genetic engineering of the germ line","type":"article","webOfScienceId":"000076145500006","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000076145500006"},{"abstract":"The Kit receptor and its ligand KL, which together constitute an essential effector at various stages of embryonic development, are both present during adult gametogenesis. In the testis, KL is expressed in Sertoli cells, and Kit in germ cells, starting at the premeiotic stages. A series of observations indicated previously a role in spermatogonia survival, without excluding a possible function at later stages. We identified a complex pattern of expression of the two components in the adult murine testis, suggestive of a role in the meiotic progression of spermatocytes. At stages VII-VIII of the cycle of the seminiferous epithelium, the time when spermatocytes enter meiosis, the membrane-associated form of KL extends on the Sertoli cell from the peripheral to the adluminal compartment of the tubule. We also found that the receptor is present on the surface of germ cells up to the pachytene stage. The availability of differentiated Sertoli cell lines, which express the KL protein and support part of the maturation of germ cells in coculture, allowed us to ask whether, in the in vitro reconstructed system, transit of spermatocytes through meiosis requires the Kit-KL interaction. Addition of a blocking monoclonal antibody against the Kit receptor (ACK2) inhibited extensively the appearance of haploid cells and the expression of a haploid-phase-specific gene (Prm1). Recognition of the supporting Sertoli cell by germ cells was not affected, indicating a requirement for the activity of the receptor for either entering or completing meiosis. Involvement of the membrane-associated form of the ligand was suggested by the observation that addition of the soluble form of KL was equally inhibitory.","apaCitation":"Vincent, S., Segretain, D., Nishikawa, S., Nishikawa, S., Sage, J., Cuzin, F., &amp; Rassoulzadegan, M. (1998). Stage-specific expression of the Kit receptor and its ligand (KL) during male gametogenesis in the mouse: a Kit-KL interaction critical for meiosis. <i>DEVELOPMENT</i>, <i>125</i>(22), 4585?4593.","capCitation":"<span class=\"title\">\n  <span>Stage-specific expression of the Kit receptor and its ligand (KL) during male gametogenesis in the mouse: a Kit-KL interaction critical for meiosis</span>\n  <i>DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Vincent, S., Segretain, D., Nishikawa, S., Nishikawa, S., Sage, J., Cuzin, F., Rassoulzadegan, M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">1998</span>; <span class=\"volume\">125 (22)</span><span class=\"pages\">: 4585-4593</span>\n</span>","chicagoCitation":"Vincent, S., D. Segretain, S. Nishikawa, S. Nishikawa, J. Sage, et al. 1998. ?Stage-Specific Expression of the Kit Receptor and Its Ligand (KL) during Male Gametogenesis in the Mouse: a Kit-KL Interaction Critical for Meiosis.? <i>DEVELOPMENT</i> 125 (22). COMPANY OF BIOLOGISTS LTD: 4585?93.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The Kit receptor and its ligand KL, which together constitute an essential effector at various stages of embryonic development, are both present during adult gametogenesis. In the testis, KL is expressed in Sertoli cells, and Kit in germ cells, starting at the premeiotic stages. A series of observations indicated previously a role in spermatogonia survival, without excluding a possible function at later stages. We identified a complex pattern of expression of the two components in the adult murine testis, suggestive of a role in the meiotic progression of spermatocytes. At stages VII-VIII of the cycle of the seminiferous epithelium, the time when spermatocytes enter meiosis, the membrane-associated form of KL extends on the Sertoli cell from the peripheral to the adluminal compartment of the tubule. We also found that the receptor is present on the surface of germ cells up to the pachytene stage. The availability of differentiated Sertoli cell lines, which express the KL protein and support part of the maturation of germ cells in coculture, allowed us to ask whether, in the in vitro reconstructed system, transit of spermatocytes through meiosis requires the Kit-KL interaction. Addition of a blocking monoclonal antibody against the Kit receptor (ACK2) inhibited extensively the appearance of haploid cells and the expression of a haploid-phase-specific gene (Prm1). Recognition of the supporting Sertoli cell by germ cells was not affected, indicating a requirement for the activity of the receptor for either entering or completing meiosis. Involvement of the membrane-associated form of the ligand was suggested by the observation that addition of the soluble form of KL was equally inhibitory.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000077649200022\">Web of Science ID 000077649200022</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/9778516\">PubMedID 9778516</a></span>\n  </p>","text":"Abstract:\nThe Kit receptor and its ligand KL, which together constitute an essential effector at various stages of embryonic development, are both present during adult gametogenesis. In the testis, KL is expressed in Sertoli cells, and Kit in germ cells, starting at the premeiotic stages. A series of observations indicated previously a role in spermatogonia survival, without excluding a possible function at later stages. We identified a complex pattern of expression of the two components in the adult murine testis, suggestive of a role in the meiotic progression of spermatocytes. At stages VII-VIII of the cycle of the seminiferous epithelium, the time when spermatocytes enter meiosis, the membrane-associated form of KL extends on the Sertoli cell from the peripheral to the adluminal compartment of the tubule. We also found that the receptor is present on the surface of germ cells up to the pachytene stage. The availability of differentiated Sertoli cell lines, which express the KL protein and support part of the maturation of germ cells in coculture, allowed us to ask whether, in the in vitro reconstructed system, transit of spermatocytes through meiosis requires the Kit-KL interaction. Addition of a blocking monoclonal antibody against the Kit receptor (ACK2) inhibited extensively the appearance of haploid cells and the expression of a haploid-phase-specific gene (Prm1). Recognition of the supporting Sertoli cell by germ cells was not affected, indicating a requirement for the activity of the receptor for either entering or completing meiosis. Involvement of the membrane-associated form of the ligand was suggested by the observation that addition of the soluble form of KL was equally inhibitory.\n\nWeb of Science ID: 000077649200022, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000077649200022\nPubMedID: 9778516, https://www.ncbi.nlm.nih.gov/pubmed/9778516"},"featured":true,"firstPublished":{"html":"Nov 1, 1998 00:00:00 AM","text":"Nov 1, 1998 00:00:00 AM","value":"1998-11-01T00:00:00.000-08:00"},"mlaCitation":"Vincent, S. et al. ?Stage-Specific Expression of the Kit Receptor and Its Ligand (KL) during Male Gametogenesis in the Mouse: a Kit-KL Interaction Critical for Meiosis.? <i>DEVELOPMENT</i> 125.22 (1998): 4585?4593. Print.","pubMedId":"9778516","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/9778516","publicationId":242885,"title":"Stage-specific expression of the Kit receptor and its ligand (KL) during male gametogenesis in the mouse: a Kit-KL interaction critical for meiosis","type":"article","webOfScienceId":"000077649200022","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000077649200022"},{"abstract":"Differentiation of male germ cells requires a continuous cross-talk with their somatic support, the Sertoli cell. An in vitro model of Sertoli cells was recently provided by established cell lines which maintain Sertoli-specific characteristics, among which is a regulated phagocytic capacity. In vivo, Sertoli cells take up the residual cytoplasm expelled from the maturing sperm, a process restricted to a limited period of germinal maturation, and they also eliminate abnormally differentiated germ cells in case of hormonal deficiency. Cells of the Sertoli line efficiently take up latex beads, as well as dead cells in the cultures. A semiquantitative assay of phagocytosis was developed, based on the uptake of fluorescent latex beads. 15P-1 cultures were found to contain a minor fraction of active phagocytes. After addition of a defined fraction of germ cells, however, all the cells internalized beads as efficiently as macrophages. The inducing cell was identified as the pachytene spermatocyte, a cell type which, in vivo, is associated with Sertoli cells when they express their phagocytic potential. These inducing meiotic cells were not internalized themselves. Rather, they interacted with Sertoli cells via a surface signal that was resistant to formaldehyde fixation. The whole induction process does not involve changes in Sertoli gene expression, since it occurs even in the presence of high doses of cycloheximide. After the required initial contact, further maintenance of the activity was dependent on factor(s) secreted in the medium of the activated culture. Phagocytosis was, on the other hand, abrogated in the presence of factor(s) secreted by a distinct fraction of germ cells, enriched in the late stages (second division) of meiosis.","apaCitation":"Grandjean, V., Sage, J., Ranc, F., Cuzin, F., &amp; Rassoulzadegan, M. (1997). Stage-specific signals in germ line differentiation: Control of Sertoli cell phagocytic activity by spermatogenic cells. <i>DEVELOPMENTAL BIOLOGY</i>, <i>184</i>(1), 165?174.","capCitation":"<span class=\"title\">\n  <span>Stage-specific signals in germ line differentiation: Control of Sertoli cell phagocytic activity by spermatogenic cells</span>\n  <i>DEVELOPMENTAL BIOLOGY</i>\n</span>\n<span class=\"authors\">Grandjean, V., Sage, J., Ranc, F., Cuzin, F., Rassoulzadegan, M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">1997</span>; <span class=\"volume\">184 (1)</span><span class=\"pages\">: 165-174</span>\n</span>","chicagoCitation":"Grandjean, V., J. Sage, F. Ranc, F. Cuzin, and M. Rassoulzadegan. 1997. ?Stage-Specific Signals in Germ Line Differentiation: Control of Sertoli Cell Phagocytic Activity by Spermatogenic Cells.? <i>DEVELOPMENTAL BIOLOGY</i> 184 (1). ACADEMIC PRESS INC JNL-COMP SUBSCRIPTIONS: 165?74.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Differentiation of male germ cells requires a continuous cross-talk with their somatic support, the Sertoli cell. An in vitro model of Sertoli cells was recently provided by established cell lines which maintain Sertoli-specific characteristics, among which is a regulated phagocytic capacity. In vivo, Sertoli cells take up the residual cytoplasm expelled from the maturing sperm, a process restricted to a limited period of germinal maturation, and they also eliminate abnormally differentiated germ cells in case of hormonal deficiency. Cells of the Sertoli line efficiently take up latex beads, as well as dead cells in the cultures. A semiquantitative assay of phagocytosis was developed, based on the uptake of fluorescent latex beads. 15P-1 cultures were found to contain a minor fraction of active phagocytes. After addition of a defined fraction of germ cells, however, all the cells internalized beads as efficiently as macrophages. The inducing cell was identified as the pachytene spermatocyte, a cell type which, in vivo, is associated with Sertoli cells when they express their phagocytic potential. These inducing meiotic cells were not internalized themselves. Rather, they interacted with Sertoli cells via a surface signal that was resistant to formaldehyde fixation. The whole induction process does not involve changes in Sertoli gene expression, since it occurs even in the presence of high doses of cycloheximide. After the required initial contact, further maintenance of the activity was dependent on factor(s) secreted in the medium of the activated culture. Phagocytosis was, on the other hand, abrogated in the presence of factor(s) secreted by a distinct fraction of germ cells, enriched in the late stages (second division) of meiosis.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/A1997WV15600015\">Web of Science ID A1997WV15600015</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/9142992\">PubMedID 9142992</a></span>\n  </p>","text":"Abstract:\nDifferentiation of male germ cells requires a continuous cross-talk with their somatic support, the Sertoli cell. An in vitro model of Sertoli cells was recently provided by established cell lines which maintain Sertoli-specific characteristics, among which is a regulated phagocytic capacity. In vivo, Sertoli cells take up the residual cytoplasm expelled from the maturing sperm, a process restricted to a limited period of germinal maturation, and they also eliminate abnormally differentiated germ cells in case of hormonal deficiency. Cells of the Sertoli line efficiently take up latex beads, as well as dead cells in the cultures. A semiquantitative assay of phagocytosis was developed, based on the uptake of fluorescent latex beads. 15P-1 cultures were found to contain a minor fraction of active phagocytes. After addition of a defined fraction of germ cells, however, all the cells internalized beads as efficiently as macrophages. The inducing cell was identified as the pachytene spermatocyte, a cell type which, in vivo, is associated with Sertoli cells when they express their phagocytic potential. These inducing meiotic cells were not internalized themselves. Rather, they interacted with Sertoli cells via a surface signal that was resistant to formaldehyde fixation. The whole induction process does not involve changes in Sertoli gene expression, since it occurs even in the presence of high doses of cycloheximide. After the required initial contact, further maintenance of the activity was dependent on factor(s) secreted in the medium of the activated culture. Phagocytosis was, on the other hand, abrogated in the presence of factor(s) secreted by a distinct fraction of germ cells, enriched in the late stages (second division) of meiosis.\n\nWeb of Science ID: A1997WV15600015, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/A1997WV15600015\nPubMedID: 9142992, https://www.ncbi.nlm.nih.gov/pubmed/9142992"},"featured":false,"firstPublished":{"html":"Apr 1, 1997 00:00:00 AM","text":"Apr 1, 1997 00:00:00 AM","value":"1997-04-01T00:00:00.000-08:00"},"mlaCitation":"Grandjean, V. et al. ?Stage-Specific Signals in Germ Line Differentiation: Control of Sertoli Cell Phagocytic Activity by Spermatogenic Cells.? <i>DEVELOPMENTAL BIOLOGY</i> 184.1 (1997): 165?174. Print.","pubMedId":"9142992","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/9142992","publicationId":242889,"title":"Stage-specific signals in germ line differentiation: Control of Sertoli cell phagocytic activity by spermatogenic cells","type":"article","webOfScienceId":"A1997WV15600015","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/A1997WV15600015"},{"abstract":"A cell culture system that supports the differentiation of male germ cells through meiosis is described. It takes advantage of the properties of a cell line, 15P-1, established from testicular cells of transgenic mice that express the large T protein of polyoma virus in the seminiferous epithelium. This line exhibits features characteristics of Sertoli cells, including transcription of the Wilms' tumor (WT1) and Steel genes. Cells of the 15P-1 type support the meiotic and postmeiotic differentiation in cocultures of diploid premeiotic germ cells into haploid spermatids expressing the protamine (Prm-1) gene. When cocultured with 15P-1 cells, testicular cells explanted from immature 9-day-old animals, before the onset of the first meiosis, generated tetrads of haploid cells with the morphology of round spermatids and initiated protamine transcription.","apaCitation":"Rassoulzadegan, M., PAQUISFLUCKLINGER, V., Bertino, B., Sage, J., Jasin, M., Miyagawa, K., ? Cuzin, F. (1993). TRANSMEIOTIC DIFFERENTIATION OF MALE GERM-CELLS IN CULTURE. <i>CELL</i>, <i>75</i>(5), 997?1006.","capCitation":"<span class=\"title\">\n  <span>TRANSMEIOTIC DIFFERENTIATION OF MALE GERM-CELLS IN CULTURE</span>\n  <i>CELL</i>\n</span>\n<span class=\"authors\">Rassoulzadegan, M., PAQUISFLUCKLINGER, V., Bertino, B., Sage, J., Jasin, M., Miyagawa, K., VANHEYNINGEN, V., Besmer, P., Cuzin, F.</span>\n\n<span class=\"details\">\n   <span class=\"year\">1993</span>; <span class=\"volume\">75 (5)</span><span class=\"pages\">: 997-1006</span>\n</span>","chicagoCitation":"Rassoulzadegan, M., V. PAQUISFLUCKLINGER, B. Bertino, J. Sage, M. Jasin, et al. 1993. ?TRANSMEIOTIC DIFFERENTIATION OF MALE GERM-CELLS IN CULTURE.? <i>CELL</i> 75 (5). CELL PRESS: 997?1006.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">A cell culture system that supports the differentiation of male germ cells through meiosis is described. It takes advantage of the properties of a cell line, 15P-1, established from testicular cells of transgenic mice that express the large T protein of polyoma virus in the seminiferous epithelium. This line exhibits features characteristics of Sertoli cells, including transcription of the Wilms' tumor (WT1) and Steel genes. Cells of the 15P-1 type support the meiotic and postmeiotic differentiation in cocultures of diploid premeiotic germ cells into haploid spermatids expressing the protamine (Prm-1) gene. When cocultured with 15P-1 cells, testicular cells explanted from immature 9-day-old animals, before the onset of the first meiosis, generated tetrads of haploid cells with the morphology of round spermatids and initiated protamine transcription.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/A1993MK96600019\">Web of Science ID A1993MK96600019</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/7504588\">PubMedID 7504588</a></span>\n  </p>","text":"Abstract:\nA cell culture system that supports the differentiation of male germ cells through meiosis is described. It takes advantage of the properties of a cell line, 15P-1, established from testicular cells of transgenic mice that express the large T protein of polyoma virus in the seminiferous epithelium. This line exhibits features characteristics of Sertoli cells, including transcription of the Wilms' tumor (WT1) and Steel genes. Cells of the 15P-1 type support the meiotic and postmeiotic differentiation in cocultures of diploid premeiotic germ cells into haploid spermatids expressing the protamine (Prm-1) gene. When cocultured with 15P-1 cells, testicular cells explanted from immature 9-day-old animals, before the onset of the first meiosis, generated tetrads of haploid cells with the morphology of round spermatids and initiated protamine transcription.\n\nWeb of Science ID: A1993MK96600019, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/A1993MK96600019\nPubMedID: 7504588, https://www.ncbi.nlm.nih.gov/pubmed/7504588"},"featured":true,"firstPublished":{"html":"Dec 3, 1993 00:00:00 AM","text":"Dec 3, 1993 00:00:00 AM","value":"1993-12-03T00:00:00.000-08:00"},"mlaCitation":"Rassoulzadegan, M. et al. ?TRANSMEIOTIC DIFFERENTIATION OF MALE GERM-CELLS IN CULTURE.? <i>CELL</i> 75.5 (1993): 997?1006. Print.","pubMedId":"7504588","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/7504588","publicationId":242891,"title":"TRANSMEIOTIC DIFFERENTIATION OF MALE GERM-CELLS IN CULTURE","type":"article","webOfScienceId":"A1993MK96600019","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/A1993MK96600019"}]},"featured":{"count":31,"publications":[{"abstract":"Small cell lung cancer (SCLC) is a neuroendocrine lung cancer characterized by fast growth, early dissemination, and rapid resistance to chemotherapy. We identified a population of long-term tumor-propagating cells (TPCs) in a mouse model of SCLC. This population, marked by high levels of EpCAM and CD24, is also prevalent in human primary SCLC tumors. Murine SCLC TPCs are numerous and highly proliferative but not intrinsically chemoresistant, indicating that not all clinical features of SCLCÂ are linked to TPCs. SCLC TPCs possess a distinct transcriptional profile compared to non-TPCs, including elevated MYC activity. Genetic and pharmacological inhibition of MYC in SCLC cells to non-TPC levels inhibits long-term propagation but not short-term growth. These studies identify a highly tumorigenic population of SCLC cells in mouse models, cell lines, and patient tumors and a means to target them in this most fatal form of lung cancer.","apaCitation":"Jahchan, N. S., Lim, J. S., Bola, B., Morris, K., Seitz, G., Tran, K. Q., ? Sage, J. (2016). Identification and Targeting of Long-Term Tumor-Propagating Cells in Small Cell Lung Cancer. <i>Cell Reports</i>, <i>16</i>(3), 644?656.","capCitation":"<span class=\"title\">\n  <span>Identification and Targeting of Long-Term Tumor-Propagating Cells in Small Cell Lung Cancer.</span>\n  <i>Cell reports</i>\n</span>\n<span class=\"authors\">Jahchan, N. S., Lim, J. S., Bola, B., Morris, K., Seitz, G., Tran, K. Q., Xu, L., Trapani, F., Morrow, C. J., Cristea, S., Coles, G. L., Yang, D., Vaka, D., Kareta, M. S., George, J., Mazur, P. K., Nguyen, T., Anderson, W. C., Dylla, S. J., Blackhall, F., Peifer, M., Dive, C., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">16 (3)</span><span class=\"pages\">: 644-656</span>\n</span>","chicagoCitation":"Jahchan, Nadine S., Jing Shan Lim, Becky Bola, Karen Morris, Garrett Seitz, et al. 2016. ?Identification and Targeting of Long-Term Tumor-Propagating Cells in Small Cell Lung Cancer.? <i>Cell Reports</i> 16 (3). ELSEVIER INC: 644?56.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small cell lung cancer (SCLC) is a neuroendocrine lung cancer characterized by fast growth, early dissemination, and rapid resistance to chemotherapy. We identified a population of long-term tumor-propagating cells (TPCs) in a mouse model of SCLC. This population, marked by high levels of EpCAM and CD24, is also prevalent in human primary SCLC tumors. Murine SCLC TPCs are numerous and highly proliferative but not intrinsically chemoresistant, indicating that not all clinical features of SCLCÂ are linked to TPCs. SCLC TPCs possess a distinct transcriptional profile compared to non-TPCs, including elevated MYC activity. Genetic and pharmacological inhibition of MYC in SCLC cells to non-TPC levels inhibits long-term propagation but not short-term growth. These studies identify a highly tumorigenic population of SCLC cells in mouse models, cell lines, and patient tumors and a means to target them in this most fatal form of lung cancer.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.celrep.2016.06.021\">DOI 10.1016/j.celrep.2016.06.021</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/27373157\">PubMedID 27373157</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4956576\">PubMedCentralID PMC4956576</a></span>\n  </p>","text":"Abstract:\nSmall cell lung cancer (SCLC) is a neuroendocrine lung cancer characterized by fast growth, early dissemination, and rapid resistance to chemotherapy. We identified a population of long-term tumor-propagating cells (TPCs) in a mouse model of SCLC. This population, marked by high levels of EpCAM and CD24, is also prevalent in human primary SCLC tumors. Murine SCLC TPCs are numerous and highly proliferative but not intrinsically chemoresistant, indicating that not all clinical features of SCLCÂ are linked to TPCs. SCLC TPCs possess a distinct transcriptional profile compared to non-TPCs, including elevated MYC activity. Genetic and pharmacological inhibition of MYC in SCLC cells to non-TPC levels inhibits long-term propagation but not short-term growth. These studies identify a highly tumorigenic population of SCLC cells in mouse models, cell lines, and patient tumors and a means to target them in this most fatal form of lung cancer.\nDOI: 10.1016/j.celrep.2016.06.021, https://dx.doi.org/10.1016/j.celrep.2016.06.021\n\nPubMedID: 27373157, https://www.ncbi.nlm.nih.gov/pubmed/27373157\nPubMedCentralID: PMC4956576, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4956576"},"doiId":"10.1016/j.celrep.2016.06.021","doiUrl":"https://dx.doi.org/10.1016/j.celrep.2016.06.021","featured":true,"firstPublished":{"html":"Jul 19, 2016 00:00:00 AM","text":"Jul 19, 2016 00:00:00 AM","value":"2016-07-19T00:00:00.000-07:00"},"mlaCitation":"Jahchan, Nadine S. et al. ?Identification and Targeting of Long-Term Tumor-Propagating Cells in Small Cell Lung Cancer.? <i>Cell reports</i> 16.3 (2016): 644?656. Print.","pubMedId":"27373157","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/27373157","publicationId":416504,"title":"Identification and Targeting of Long-Term Tumor-Propagating Cells in Small Cell Lung Cancer.","type":"article"},{"abstract":"Metastases are the main cause of cancer deaths, but the mechanisms underlying metastatic progression remain poorly understood. We isolated pure populations of cancer cells from primary tumors and metastases from a genetically engineered mouse model of human small cell lung cancer (SCLC) to investigate the mechanisms that drive the metastatic spread ofÂ this lethal cancer. Genome-wide characterization of chromatin accessibility revealed the opening of large numbers of distal regulatory elements across the genome during metastatic progression. These changes correlate with copy number amplification of the Nfib locus, and differentially accessible sites were highly enriched for Nfib transcription factor binding sites. Nfib is necessary and sufficient to increase chromatin accessibility at a large subset of the intergenic regions. Nfib promotes pro-metastatic neuronal gene expression programs and drives the metastatic ability of SCLC cells. The identification of widespread chromatin changes during SCLC progression reveals an unexpected global reprogramming during metastatic progression.","apaCitation":"Denny, S. K., Yang, D., Chuang, C.-H., Brady, J. J., Lim, J. S., Gruner, B. M., ? Winslow, M. M. (2016). Nfib Promotes Metastasis through a Widespread Increase in Chromatin Accessibility. <i>CELL</i>, <i>166</i>(2), 328?342.","capCitation":"<span class=\"title\">\n  <span>Nfib Promotes Metastasis through a Widespread Increase in Chromatin Accessibility</span>\n  <i>CELL</i>\n</span>\n<span class=\"authors\">Denny, S. K., Yang, D., Chuang, C., Brady, J. J., Lim, J. S., Gruner, B. M., Chiou, S., Schep, A. N., Baral, J., Hamard, C., Antoine, M., Wislez, M., Kong, C. S., Connolly, A. J., Park, K., Sage, J., Greenleaf, W. J., Winslow, M. M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">166 (2)</span><span class=\"pages\">: 328-342</span>\n</span>","chicagoCitation":"Denny, Sarah K., Dian Yang, Chen-Hua Chuang, Jennifer J. Brady, Jing Shan Lim, et al. 2016. ?Nfib Promotes Metastasis through a Widespread Increase in Chromatin Accessibility.? <i>CELL</i> 166 (2). CELL PRESS: 328?42.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Metastases are the main cause of cancer deaths, but the mechanisms underlying metastatic progression remain poorly understood. We isolated pure populations of cancer cells from primary tumors and metastases from a genetically engineered mouse model of human small cell lung cancer (SCLC) to investigate the mechanisms that drive the metastatic spread ofÂ this lethal cancer. Genome-wide characterization of chromatin accessibility revealed the opening of large numbers of distal regulatory elements across the genome during metastatic progression. These changes correlate with copy number amplification of the Nfib locus, and differentially accessible sites were highly enriched for Nfib transcription factor binding sites. Nfib is necessary and sufficient to increase chromatin accessibility at a large subset of the intergenic regions. Nfib promotes pro-metastatic neuronal gene expression programs and drives the metastatic ability of SCLC cells. The identification of widespread chromatin changes during SCLC progression reveals an unexpected global reprogramming during metastatic progression.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.cell.2016.05.052\">DOI 10.1016/j.cell.2016.05.052</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000380255400012\">Web of Science ID 000380255400012</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/27374332\">PubMedID 27374332</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5004630\">PubMedCentralID PMC5004630</a></span>\n  </p>","text":"Abstract:\nMetastases are the main cause of cancer deaths, but the mechanisms underlying metastatic progression remain poorly understood. We isolated pure populations of cancer cells from primary tumors and metastases from a genetically engineered mouse model of human small cell lung cancer (SCLC) to investigate the mechanisms that drive the metastatic spread ofÂ this lethal cancer. Genome-wide characterization of chromatin accessibility revealed the opening of large numbers of distal regulatory elements across the genome during metastatic progression. These changes correlate with copy number amplification of the Nfib locus, and differentially accessible sites were highly enriched for Nfib transcription factor binding sites. Nfib is necessary and sufficient to increase chromatin accessibility at a large subset of the intergenic regions. Nfib promotes pro-metastatic neuronal gene expression programs and drives the metastatic ability of SCLC cells. The identification of widespread chromatin changes during SCLC progression reveals an unexpected global reprogramming during metastatic progression.\nDOI: 10.1016/j.cell.2016.05.052, https://dx.doi.org/10.1016/j.cell.2016.05.052\nWeb of Science ID: 000380255400012, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000380255400012\nPubMedID: 27374332, https://www.ncbi.nlm.nih.gov/pubmed/27374332\nPubMedCentralID: PMC5004630, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5004630"},"doiId":"10.1016/j.cell.2016.05.052","doiUrl":"https://dx.doi.org/10.1016/j.cell.2016.05.052","featured":true,"firstPublished":{"html":"Jul 14, 2016 00:00:00 AM","text":"Jul 14, 2016 00:00:00 AM","value":"2016-07-14T00:00:00.000-07:00"},"mlaCitation":"Denny, Sarah K. et al. ?Nfib Promotes Metastasis through a Widespread Increase in Chromatin Accessibility.? <i>CELL</i> 166.2 (2016): 328?342. Print.","pubMedId":"27374332","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/27374332","publicationId":413068,"title":"Nfib Promotes Metastasis through a Widespread Increase in Chromatin Accessibility","type":"article","webOfScienceId":"000380255400012","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000380255400012"},{"abstract":"Small-cell lung cancer (SCLC) is a highly aggressive subtype of lung cancer with limited treatment options. CD47 is a cell-surface molecule that promotes immune evasion by engaging signal-regulatory protein alpha (SIRP?), which serves as an inhibitory receptor on macrophages. Here, we found that CD47 is highly expressed on the surface of human SCLC cells; therefore, we investigated CD47-blocking immunotherapies as a potential approach for SCLC treatment. Disruption of the interaction of CD47 with SIRP? using anti-CD47 antibodies induced macrophage-mediated phagocytosis of human SCLC patient cells in culture. In a murine model, administration of CD47-blocking antibodies or targeted inactivation of the Cd47 gene markedly inhibited SCLC tumor growth. Furthermore, using comprehensive antibody arrays, we identified several possible therapeutic targets on the surface of SCLC cells. Antibodies to these targets, including CD56/neural cell adhesion molecule (NCAM), promoted phagocytosis in human SCLC cell lines that was enhanced when combined with CD47-blocking therapies. In light of recent clinical trials for CD47-blocking therapies in cancer treatment, these findings identify disruption of the CD47/SIRP? axis as a potential immunotherapeutic strategy for SCLC. This approach could enable personalized immunotherapeutic regimens in patients with SCLC and other cancers.","apaCitation":"Weiskopf, K., Jahchan, N. S., Schnorr, P. J., Cristea, S., Ring, A. M., Maute, R. L., ? Sage, J. (2016). CD47-blocking immunotherapies stimulate macrophage-mediated destruction of small-cell lung cancer. <i>JOURNAL OF CLINICAL INVESTIGATION</i>, <i>126</i>(7), 2610?2620.","capCitation":"<span class=\"title\">\n  <span>CD47-blocking immunotherapies stimulate macrophage-mediated destruction of small-cell lung cancer</span>\n  <i>JOURNAL OF CLINICAL INVESTIGATION</i>\n</span>\n<span class=\"authors\">Weiskopf, K., Jahchan, N. S., Schnorr, P. J., Cristea, S., Ring, A. M., Maute, R. L., Volkmer, A. K., Volkmer, J., Liu, J., Lim, J. S., Yang, D., Seitz, G., Thuyen Nguyen, T., Wu, D., Jude, K., Guerston, H., Barkal, A., Trapani, F., George, J., Poirier, J. T., Gardner, E. E., Miles, L. A., de Stanchina, E., Lofgren, S. M., Vogel, H., Winslow, M. M., Dive, C., Thomas, R. K., Rudin, C. M., van de Rijn, M., Majeti, R., Garcia, K. C., Weissman, I. L., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">126 (7)</span><span class=\"pages\">: 2610-2620</span>\n</span>","chicagoCitation":"Weiskopf, Kipp, Nadine S. Jahchan, Peter J. Schnorr, Sandra Cristea, Aaron M. Ring, et al. 2016. ?CD47-Blocking Immunotherapies Stimulate Macrophage-Mediated Destruction of Small-Cell Lung Cancer.? <i>JOURNAL OF CLINICAL INVESTIGATION</i> 126 (7). AMER SOC CLINICAL INVESTIGATION INC: 2610?20.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small-cell lung cancer (SCLC) is a highly aggressive subtype of lung cancer with limited treatment options. CD47 is a cell-surface molecule that promotes immune evasion by engaging signal-regulatory protein alpha (SIRP?), which serves as an inhibitory receptor on macrophages. Here, we found that CD47 is highly expressed on the surface of human SCLC cells; therefore, we investigated CD47-blocking immunotherapies as a potential approach for SCLC treatment. Disruption of the interaction of CD47 with SIRP? using anti-CD47 antibodies induced macrophage-mediated phagocytosis of human SCLC patient cells in culture. In a murine model, administration of CD47-blocking antibodies or targeted inactivation of the Cd47 gene markedly inhibited SCLC tumor growth. Furthermore, using comprehensive antibody arrays, we identified several possible therapeutic targets on the surface of SCLC cells. Antibodies to these targets, including CD56/neural cell adhesion molecule (NCAM), promoted phagocytosis in human SCLC cell lines that was enhanced when combined with CD47-blocking therapies. In light of recent clinical trials for CD47-blocking therapies in cancer treatment, these findings identify disruption of the CD47/SIRP? axis as a potential immunotherapeutic strategy for SCLC. This approach could enable personalized immunotherapeutic regimens in patients with SCLC and other cancers.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1172/JCI81603\">DOI 10.1172/JCI81603</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000379094800024\">Web of Science ID 000379094800024</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/27294525\">PubMedID 27294525</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4922696\">PubMedCentralID PMC4922696</a></span>\n  </p>","text":"Abstract:\nSmall-cell lung cancer (SCLC) is a highly aggressive subtype of lung cancer with limited treatment options. CD47 is a cell-surface molecule that promotes immune evasion by engaging signal-regulatory protein alpha (SIRP?), which serves as an inhibitory receptor on macrophages. Here, we found that CD47 is highly expressed on the surface of human SCLC cells; therefore, we investigated CD47-blocking immunotherapies as a potential approach for SCLC treatment. Disruption of the interaction of CD47 with SIRP? using anti-CD47 antibodies induced macrophage-mediated phagocytosis of human SCLC patient cells in culture. In a murine model, administration of CD47-blocking antibodies or targeted inactivation of the Cd47 gene markedly inhibited SCLC tumor growth. Furthermore, using comprehensive antibody arrays, we identified several possible therapeutic targets on the surface of SCLC cells. Antibodies to these targets, including CD56/neural cell adhesion molecule (NCAM), promoted phagocytosis in human SCLC cell lines that was enhanced when combined with CD47-blocking therapies. In light of recent clinical trials for CD47-blocking therapies in cancer treatment, these findings identify disruption of the CD47/SIRP? axis as a potential immunotherapeutic strategy for SCLC. This approach could enable personalized immunotherapeutic regimens in patients with SCLC and other cancers.\nDOI: 10.1172/JCI81603, https://dx.doi.org/10.1172/JCI81603\nWeb of Science ID: 000379094800024, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000379094800024\nPubMedID: 27294525, https://www.ncbi.nlm.nih.gov/pubmed/27294525\nPubMedCentralID: PMC4922696, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4922696"},"doiId":"10.1172/JCI81603","doiUrl":"https://dx.doi.org/10.1172/JCI81603","featured":true,"firstPublished":{"html":"Jul 1, 2016 00:00:00 AM","text":"Jul 1, 2016 00:00:00 AM","value":"2016-07-01T00:00:00.000-07:00"},"mlaCitation":"Weiskopf, Kipp et al. ?CD47-Blocking Immunotherapies Stimulate Macrophage-Mediated Destruction of Small-Cell Lung Cancer.? <i>JOURNAL OF CLINICAL INVESTIGATION</i> 126.7 (2016): 2610?2620. Print.","pubMedId":"27294525","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/27294525","publicationId":416393,"title":"CD47-blocking immunotherapies stimulate macrophage-mediated destruction of small-cell lung cancer","type":"article","webOfScienceId":"000379094800024","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000379094800024"},{"apaCitation":"Reynoird, N., Mazur, P. K., Stellfeld, T., Flores, N. M., Lofgren, S. M., Carlson, S. M., ? Sage, J. (2016). Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer. <i>GENES &amp; DEVELOPMENT</i>, <i>30</i>(7), 772?785.","capCitation":"<span class=\"title\">\n  <span>Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer</span>\n  <i>GENES &amp; DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Reynoird, N., Mazur, P. K., Stellfeld, T., Flores, N. M., Lofgren, S. M., Carlson, S. M., Brambilla, E., Hainaut, P., Kaznowska, E. B., Arrowsmith, C. H., Khatri, P., Stresemann, C., Gozani, O., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">30 (7)</span><span class=\"pages\">: 772-785</span>\n</span>","chicagoCitation":"Reynoird, Nicolas, Pawel K. Mazur, Timo Stellfeld, Natasha M. Flores, Shane M. Lofgren, et al. 2016. ?Coordination of Stress Signals by the Lysine Methyltransferase SMYD2 Promotes Pancreatic Cancer.? <i>GENES &amp; DEVELOPMENT</i> 30 (7). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 772?85.","detail":{"html":"<p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.275529.115\">DOI 10.1101/gad.275529.115</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000373194300003\">Web of Science ID 000373194300003</a></span>\n  </p>","text":"DOI: 10.1101/gad.275529.115, https://dx.doi.org/10.1101/gad.275529.115\nWeb of Science ID: 000373194300003, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000373194300003"},"doiId":"10.1101/gad.275529.115","doiUrl":"https://dx.doi.org/10.1101/gad.275529.115","featured":true,"firstPublished":{"html":"Apr 1, 2016 00:00:00 AM","text":"Apr 1, 2016 00:00:00 AM","value":"2016-04-01T00:00:00.000-07:00"},"mlaCitation":"Reynoird, Nicolas et al. ?Coordination of Stress Signals by the Lysine Methyltransferase SMYD2 Promotes Pancreatic Cancer.? <i>GENES &amp; DEVELOPMENT</i> 30.7 (2016): 772?785. Print.","publicationId":405438,"title":"Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer","type":"article","webOfScienceId":"000373194300003","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000373194300003"},{"abstract":"Pancreatic ductal adenocarcinoma (PDAC) is a lethal form of cancer with few therapeutic options. We found that levels of the lysine methyltransferase SMYD2 (SET and MYND domain 2) are elevated in PDAC and that genetic and pharmacological inhibition of SMYD2 restricts PDAC growth. We further identified the stress response kinase MAPKAPK3 (MK3) as a new physiologic substrate of SMYD2 in PDAC cells. Inhibition of MAPKAPK3 impedes PDAC growth, identifying a potential new kinase target in PDAC. Finally, we show that inhibition of SMYD2 cooperates with standard chemotherapy to treat PDAC cells and tumors. These findings uncover a pivotal role for SMYD2 in promoting pancreatic cancer.","apaCitation":"Reynoird, N., Mazur, P. K., Stellfeld, T., Flores, N. M., Lofgren, S. M., Carlson, S. M., ? Sage, J. (2016). Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer. <i>Genes &amp; Development</i>, <i>30</i>(7), 772?785.","capCitation":"<span class=\"title\">\n  <span>Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer.</span>\n  <i>Genes &amp; development</i>\n</span>\n<span class=\"authors\">Reynoird, N., Mazur, P. K., Stellfeld, T., Flores, N. M., Lofgren, S. M., Carlson, S. M., Brambilla, E., Hainaut, P., Kaznowska, E. B., Arrowsmith, C. H., Khatri, P., Stresemann, C., Gozani, O., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2016</span>; <span class=\"volume\">30 (7)</span><span class=\"pages\">: 772-785</span>\n</span>","chicagoCitation":"Reynoird, Nicolas, Pawel K. Mazur, Timo Stellfeld, Natasha M. Flores, Shane M. Lofgren, et al. 2016. ?Coordination of Stress Signals by the Lysine Methyltransferase SMYD2 Promotes Pancreatic Cancer.? <i>Genes &amp; Development</i> 30 (7). COLD SPRING HARBOR LABORATORY PRESS: 772?85.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Pancreatic ductal adenocarcinoma (PDAC) is a lethal form of cancer with few therapeutic options. We found that levels of the lysine methyltransferase SMYD2 (SET and MYND domain 2) are elevated in PDAC and that genetic and pharmacological inhibition of SMYD2 restricts PDAC growth. We further identified the stress response kinase MAPKAPK3 (MK3) as a new physiologic substrate of SMYD2 in PDAC cells. Inhibition of MAPKAPK3 impedes PDAC growth, identifying a potential new kinase target in PDAC. Finally, we show that inhibition of SMYD2 cooperates with standard chemotherapy to treat PDAC cells and tumors. These findings uncover a pivotal role for SMYD2 in promoting pancreatic cancer.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.275529.115\">DOI 10.1101/gad.275529.115</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/26988419\">PubMedID 26988419</a></span>\n  </p>","text":"Abstract:\nPancreatic ductal adenocarcinoma (PDAC) is a lethal form of cancer with few therapeutic options. We found that levels of the lysine methyltransferase SMYD2 (SET and MYND domain 2) are elevated in PDAC and that genetic and pharmacological inhibition of SMYD2 restricts PDAC growth. We further identified the stress response kinase MAPKAPK3 (MK3) as a new physiologic substrate of SMYD2 in PDAC cells. Inhibition of MAPKAPK3 impedes PDAC growth, identifying a potential new kinase target in PDAC. Finally, we show that inhibition of SMYD2 cooperates with standard chemotherapy to treat PDAC cells and tumors. These findings uncover a pivotal role for SMYD2 in promoting pancreatic cancer.\nDOI: 10.1101/gad.275529.115, https://dx.doi.org/10.1101/gad.275529.115\n\nPubMedID: 26988419, https://www.ncbi.nlm.nih.gov/pubmed/26988419"},"doiId":"10.1101/gad.275529.115","doiUrl":"https://dx.doi.org/10.1101/gad.275529.115","featured":true,"firstPublished":{"html":"Apr 1, 2016 00:00:00 AM","text":"Apr 1, 2016 00:00:00 AM","value":"2016-04-01T00:00:00.000-07:00"},"mlaCitation":"Reynoird, Nicolas et al. ?Coordination of Stress Signals by the Lysine Methyltransferase SMYD2 Promotes Pancreatic Cancer.? <i>Genes &amp; development</i> 30.7 (2016): 772?785. Print.","pubMedId":"26988419","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/26988419","publicationId":405926,"title":"Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer.","type":"article"},{"abstract":"Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal human cancers and shows resistance to any therapeutic strategy used. Here we tested small-molecule inhibitors targeting chromatin regulators as possible therapeutic agents in PDAC. We show that JQ1, an inhibitor of the bromodomain and extraterminal (BET) family of proteins, suppresses PDAC development in mice by inhibiting both MYC activity and inflammatory signals. The histone deacetylase (HDAC) inhibitor SAHA synergizes with JQ1 to augment cell death and more potently suppress advanced PDAC. Finally, using a CRISPR-Cas9-based method for gene editing directly in the mouse adult pancreas, we show that de-repression of p57 (also known as KIP2 or CDKN1C) upon combined BET and HDAC inhibition is required for the induction of combination therapy-induced cell death in PDAC. SAHA is approved for human use, and molecules similar to JQ1 are being tested in clinical trials. Thus, these studies identify a promising epigenetic-based therapeutic strategy that may be rapidly implemented in fatal human tumors.","apaCitation":"Mazur, P. K., Herner, A., Mello, S. S., Wirth, M., Hausmann, S., SÃ¡nchez-Rivera, F. J., ? Siveke, J. T. (2015). Combined inhibition of BET family proteins and histone deacetylases as a potential epigenetics-based therapy for pancreatic ductal adenocarcinoma. <i>Nature Medicine</i>, <i>21</i>(10), 1163?1171.","capCitation":"<span class=\"title\">\n  <span>Combined inhibition of BET family proteins and histone deacetylases as a potential epigenetics-based therapy for pancreatic ductal adenocarcinoma.</span>\n  <i>Nature medicine</i>\n</span>\n<span class=\"authors\">Mazur, P. K., Herner, A., Mello, S. S., Wirth, M., Hausmann, S., SÃ¡nchez-Rivera, F. J., Lofgren, S. M., Kuschma, T., Hahn, S. A., Vangala, D., Trajkovic-Arsic, M., Gupta, A., Heid, I., NoÃ«l, P. B., Braren, R., Erkan, M., Kleeff, J., Sipos, B., Sayles, L. C., Heikenwalder, M., HeÃŸmann, E., Ellenrieder, V., Esposito, I., Jacks, T., Bradner, J. E., Khatri, P., Sweet-Cordero, E. A., Attardi, L. D., Schmid, R. M., Schneider, G., Sage, J., Siveke, J. T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2015</span>; <span class=\"volume\">21 (10)</span><span class=\"pages\">: 1163-1171</span>\n</span>","chicagoCitation":"Mazur, Pawel K., Alexander Herner, Stephano S. Mello, Matthias Wirth, Simone Hausmann, et al. 2015. ?Combined Inhibition of BET Family Proteins and Histone Deacetylases as a Potential Epigenetics-Based Therapy for Pancreatic Ductal Adenocarcinoma.? <i>Nature Medicine</i> 21 (10): 1163?71.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal human cancers and shows resistance to any therapeutic strategy used. Here we tested small-molecule inhibitors targeting chromatin regulators as possible therapeutic agents in PDAC. We show that JQ1, an inhibitor of the bromodomain and extraterminal (BET) family of proteins, suppresses PDAC development in mice by inhibiting both MYC activity and inflammatory signals. The histone deacetylase (HDAC) inhibitor SAHA synergizes with JQ1 to augment cell death and more potently suppress advanced PDAC. Finally, using a CRISPR-Cas9-based method for gene editing directly in the mouse adult pancreas, we show that de-repression of p57 (also known as KIP2 or CDKN1C) upon combined BET and HDAC inhibition is required for the induction of combination therapy-induced cell death in PDAC. SAHA is approved for human use, and molecules similar to JQ1 are being tested in clinical trials. Thus, these studies identify a promising epigenetic-based therapeutic strategy that may be rapidly implemented in fatal human tumors.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nm.3952\">DOI 10.1038/nm.3952</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/26390243\">PubMedID 26390243</a></span>\n  </p>","text":"Abstract:\nPancreatic ductal adenocarcinoma (PDAC) is one of the most lethal human cancers and shows resistance to any therapeutic strategy used. Here we tested small-molecule inhibitors targeting chromatin regulators as possible therapeutic agents in PDAC. We show that JQ1, an inhibitor of the bromodomain and extraterminal (BET) family of proteins, suppresses PDAC development in mice by inhibiting both MYC activity and inflammatory signals. The histone deacetylase (HDAC) inhibitor SAHA synergizes with JQ1 to augment cell death and more potently suppress advanced PDAC. Finally, using a CRISPR-Cas9-based method for gene editing directly in the mouse adult pancreas, we show that de-repression of p57 (also known as KIP2 or CDKN1C) upon combined BET and HDAC inhibition is required for the induction of combination therapy-induced cell death in PDAC. SAHA is approved for human use, and molecules similar to JQ1 are being tested in clinical trials. Thus, these studies identify a promising epigenetic-based therapeutic strategy that may be rapidly implemented in fatal human tumors.\nDOI: 10.1038/nm.3952, https://dx.doi.org/10.1038/nm.3952\n\nPubMedID: 26390243, https://www.ncbi.nlm.nih.gov/pubmed/26390243"},"doiId":"10.1038/nm.3952","doiUrl":"https://dx.doi.org/10.1038/nm.3952","featured":true,"firstPublished":{"html":"Oct 1, 2015 00:00:00 AM","text":"Oct 1, 2015 00:00:00 AM","value":"2015-10-01T00:00:00.000-07:00"},"mlaCitation":"Mazur, Pawel K. et al. ?Combined Inhibition of BET Family Proteins and Histone Deacetylases as a Potential Epigenetics-Based Therapy for Pancreatic Ductal Adenocarcinoma.? <i>Nature medicine</i> 21.10 (2015): 1163?1171. Print.","pubMedId":"26390243","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/26390243","publicationId":398340,"title":"Combined inhibition of BET family proteins and histone deacetylases as a potential epigenetics-based therapy for pancreatic ductal adenocarcinoma.","type":"article"},{"abstract":"We have sequenced the genomes of 110 small cell lung cancers (SCLC), one of the deadliest human cancers. In nearly all the tumours analysed we found bi-allelic inactivation of TP53 and RB1, sometimes by complex genomic rearrangements. Two tumours with wild-type RB1 had evidence of chromothripsis leading to overexpression of cyclin D1 (encoded by the CCND1 gene), revealing an alternative mechanism of Rb1 deregulation. Thus, loss of the tumour suppressors TP53 and RB1 is obligatory in SCLC. We discovered somatic genomic rearrangements of TP73 that create an oncogenic version of this gene, TP73?ex2/3. In rare cases, SCLC tumours exhibited kinase gene mutations, providing a possible therapeutic opportunity for individual patients. Finally, we observed inactivating mutations in NOTCH family genes in 25% of human SCLC. Accordingly, activation of Notch signalling in a pre-clinical SCLC mouse model strikingly reduced the number of tumours and extended the survival of the mutant mice. Furthermore, neuroendocrine gene expression was abrogated by Notch activity in SCLC cells. This first comprehensive study of somatic genome alterations in SCLC uncovers several key biological processes and identifies candidate therapeutic targets in this highly lethal form of cancer.","apaCitation":"George, J., Lim, J. S., Jang, S. J., Cun, Y., Ozretic, L., Kong, G., ? Thomas, R. K. (2015). Comprehensive genomic profiles of small cell lung cancer. <i>Nature</i>, <i>524</i>(7563), 47?53.","capCitation":"<span class=\"title\">\n  <span>Comprehensive genomic profiles of small cell lung cancer.</span>\n  <i>Nature</i>\n</span>\n<span class=\"authors\">George, J., Lim, J. S., Jang, S. J., Cun, Y., Ozretic, L., Kong, G., Leenders, F., Lu, X., FernÃ¡ndez-Cuesta, L., Bosco, G., MÃ¼ller, C., Dahmen, I., Jahchan, N. S., Park, K., Yang, D., Karnezis, A. N., Vaka, D., Torres, A., Wang, M. S., Korbel, J. O., Menon, R., Chun, S., Kim, D., Wilkerson, M., Hayes, N., Engelmann, D., PÃ¼tzer, B., Bos, M., Michels, S., Vlasic, I., Seidel, D., Pinther, B., Schaub, P., Becker, C., AltmÃ¼ller, J., Yokota, J., Kohno, T., Iwakawa, R., Tsuta, K., Noguchi, M., Muley, T., Hoffmann, H., Schnabel, P. A., Petersen, I., Chen, Y., Soltermann, A., Tischler, V., Choi, C., Kim, Y., Massion, P. P., Zou, Y., Jovanovic, D., Kontic, M., Wright, G. M., Russell, P. A., Solomon, B., Koch, I., Lindner, M., Muscarella, L. A., la Torre, A., Field, J. K., Jakopovic, M., Knezevic, J., CastaÃ±os-VÃ©lez, E., Roz, L., Pastorino, U., Brustugun, O., Lund-Iversen, M., Thunnissen, E., KÃ¶hler, J., Schuler, M., Botling, J., Sandelin, M., Sanchez-Cespedes, M., Salvesen, H. B., Achter, V., Lang, U., Bogus, M., Schneider, P. M., Zander, T., AnsÃ©n, S., Hallek, M., Wolf, J., Vingron, M., Yatabe, Y., Travis, W. D., NÃ¼rnberg, P., Reinhardt, C., Perner, S., Heukamp, L., BÃ¼ttner, R., Haas, S. A., Brambilla, E., Peifer, M., Sage, J., Thomas, R. K.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2015</span>; <span class=\"volume\">524 (7563)</span><span class=\"pages\">: 47-53</span>\n</span>","chicagoCitation":"George, Julie, Jing Shan Lim, Se Jin Jang, Yupeng Cun, Luka Ozretic, et al. 2015. ?Comprehensive Genomic Profiles of Small Cell Lung Cancer.? <i>Nature</i> 524 (7563): 47?53.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">We have sequenced the genomes of 110 small cell lung cancers (SCLC), one of the deadliest human cancers. In nearly all the tumours analysed we found bi-allelic inactivation of TP53 and RB1, sometimes by complex genomic rearrangements. Two tumours with wild-type RB1 had evidence of chromothripsis leading to overexpression of cyclin D1 (encoded by the CCND1 gene), revealing an alternative mechanism of Rb1 deregulation. Thus, loss of the tumour suppressors TP53 and RB1 is obligatory in SCLC. We discovered somatic genomic rearrangements of TP73 that create an oncogenic version of this gene, TP73?ex2/3. In rare cases, SCLC tumours exhibited kinase gene mutations, providing a possible therapeutic opportunity for individual patients. Finally, we observed inactivating mutations in NOTCH family genes in 25% of human SCLC. Accordingly, activation of Notch signalling in a pre-clinical SCLC mouse model strikingly reduced the number of tumours and extended the survival of the mutant mice. Furthermore, neuroendocrine gene expression was abrogated by Notch activity in SCLC cells. This first comprehensive study of somatic genome alterations in SCLC uncovers several key biological processes and identifies candidate therapeutic targets in this highly lethal form of cancer.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nature14664\">DOI 10.1038/nature14664</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/26168399\">PubMedID 26168399</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4861069\">PubMedCentralID PMC4861069</a></span>\n  </p>","text":"Abstract:\nWe have sequenced the genomes of 110 small cell lung cancers (SCLC), one of the deadliest human cancers. In nearly all the tumours analysed we found bi-allelic inactivation of TP53 and RB1, sometimes by complex genomic rearrangements. Two tumours with wild-type RB1 had evidence of chromothripsis leading to overexpression of cyclin D1 (encoded by the CCND1 gene), revealing an alternative mechanism of Rb1 deregulation. Thus, loss of the tumour suppressors TP53 and RB1 is obligatory in SCLC. We discovered somatic genomic rearrangements of TP73 that create an oncogenic version of this gene, TP73?ex2/3. In rare cases, SCLC tumours exhibited kinase gene mutations, providing a possible therapeutic opportunity for individual patients. Finally, we observed inactivating mutations in NOTCH family genes in 25% of human SCLC. Accordingly, activation of Notch signalling in a pre-clinical SCLC mouse model strikingly reduced the number of tumours and extended the survival of the mutant mice. Furthermore, neuroendocrine gene expression was abrogated by Notch activity in SCLC cells. This first comprehensive study of somatic genome alterations in SCLC uncovers several key biological processes and identifies candidate therapeutic targets in this highly lethal form of cancer.\nDOI: 10.1038/nature14664, https://dx.doi.org/10.1038/nature14664\n\nPubMedID: 26168399, https://www.ncbi.nlm.nih.gov/pubmed/26168399\nPubMedCentralID: PMC4861069, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4861069"},"doiId":"10.1038/nature14664","doiUrl":"https://dx.doi.org/10.1038/nature14664","featured":true,"firstPublished":{"html":"Aug 6, 2015 00:00:00 AM","text":"Aug 6, 2015 00:00:00 AM","value":"2015-08-06T00:00:00.000-07:00"},"mlaCitation":"George, Julie et al. ?Comprehensive Genomic Profiles of Small Cell Lung Cancer.? <i>Nature</i> 524.7563 (2015): 47?53. Print.","pubMedId":"26168399","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/26168399","publicationId":391110,"title":"Comprehensive genomic profiles of small cell lung cancer.","type":"article"},{"apaCitation":"George, J., Lim, J. S., Jang, S. J., Cun, Y., Ozretic, L., Kong, G., ? Thomas, R. K. (2015). Comprehensive genomic profiles of small cell lung cancer. <i>NATURE</i>, <i>524</i>(7563), 47?U73.","capCitation":"<span class=\"title\">\n  <span>Comprehensive genomic profiles of small cell lung cancer</span>\n  <i>NATURE</i>\n</span>\n<span class=\"authors\">George, J., Lim, J. S., Jang, S. J., Cun, Y., Ozretic, L., Kong, G., Leenders, F., Lu, X., Fernandez-Cuesta, L., Bosco, G., Mueller, C., Dahmen, I., Jahchan, N. S., Park, K., Yang, D., Karnezis, A. N., Vaka, D., Torres, A., Wang, M. S., Korbel, J. O., Menon, R., Chun, S., Kim, D., Wilkerson, M., Hayes, N., Engelmann, D., Puetzer, B., Bos, M., Michels, S., Vlasic, I., Seidel, D., Pinther, B., Schaub, P., Becker, C., Altmueller, J., Yokota, J., Kohno, T., Iwakawa, R., Tsuta, K., Noguchi, M., Muley, T., Hoffmann, H., Schnabel, P. A., Petersen, I., Chen, Y., Soltermann, A., Tischler, V., Choi, C., Kim, Y., Massion, P. P., Zou, Y., Jovanovic, D., Kontic, M., Wright, G. M., Russell, P. A., Solomon, B., Koch, I., Lindner, M., Muscarella, L. A., la Torre, A., Field, J. K., Jakopovic, M., Knezevic, J., Castanos-Velez, E., Roz, L., Pastorino, U., Brustugun, O., Lund-Iversen, M., Thunnissen, E., Koehler, J., Schuler, M., Botling, J., Sandelin, M., Sanchez-Cespedes, M., Salvesen, H. B., Achter, V., Lang, U., Bogus, M., Schneider, P. M., Zander, T., Ansen, S., Hallek, M., Wolf, J., Vingron, M., Yatabe, Y., Travis, W. D., Nuernberg, P., Reinhardt, C., Perner, S., Heukamp, L., Buettner, R., Haas, S. A., Brambilla, E., Peifer, M., Sage, J., Thomas, R. K.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2015</span>; <span class=\"volume\">524 (7563)</span><span class=\"pages\">: 47-U73</span>\n</span>","chicagoCitation":"George, Julie, Jing Shan Lim, Se Jin Jang, Yupeng Cun, Luka Ozretic, et al. 2015. ?Comprehensive Genomic Profiles of Small Cell Lung Cancer.? <i>NATURE</i> 524 (7563). NATURE PUBLISHING GROUP: 47?U73.","detail":{"html":"<p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nature14664\">DOI 10.1038/nature14664</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000359002300029\">Web of Science ID 000359002300029</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/26168399\">PubMedID 26168399</a></span>\n  </p>","text":"DOI: 10.1038/nature14664, https://dx.doi.org/10.1038/nature14664\nWeb of Science ID: 000359002300029, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000359002300029\nPubMedID: 26168399, https://www.ncbi.nlm.nih.gov/pubmed/26168399"},"doiId":"10.1038/nature14664","doiUrl":"https://dx.doi.org/10.1038/nature14664","featured":true,"firstPublished":{"html":"Aug 6, 2015 00:00:00 AM","text":"Aug 6, 2015 00:00:00 AM","value":"2015-08-06T00:00:00.000-07:00"},"mlaCitation":"George, Julie et al. ?Comprehensive Genomic Profiles of Small Cell Lung Cancer.? <i>NATURE</i> 524.7563 (2015): 47?U73. Print.","pubMedId":"26168399","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/26168399","publicationId":391969,"title":"Comprehensive genomic profiles of small cell lung cancer","type":"article","webOfScienceId":"000359002300029","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000359002300029"},{"abstract":"Mutations in the retinoblastoma tumor suppressor gene Rb are involved in many forms of human cancer. In this study, we investigated the early consequences of inactivating Rb in the context of cellular reprogramming. We found that Rb inactivation promotes the reprogramming of differentiated cells to a pluripotent state. Unexpectedly, this effect is cell cycle independent, and instead reflects direct binding of Rb to pluripotency genes, including Sox2 and Oct4, which leads to a repressed chromatin state. More broadly, this regulation of pluripotency networks and Sox2 in particular is critical for the initiation of tumors upon loss of Rb in mice. These studies therefore identify Rb as a global transcriptional repressor of pluripotency networks, providing a molecular basis for previous reports about its involvement in cell fate pliability, and implicate misregulation of pluripotency factors such as Sox2 in tumorigenesis related to loss of Rb function.","apaCitation":"Kareta, M. S., Gorges, L. L., Hafeez, S., Benayoun, B. A., Marro, S., Zmoos, A.-F., ? Wernig, M. (2015). Inhibition of pluripotency networks by the rb tumor suppressor restricts reprogramming and tumorigenesis. <i>Cell Stem Cell</i>, <i>16</i>(1), 39?50.","capCitation":"<span class=\"title\">\n  <span>Inhibition of pluripotency networks by the rb tumor suppressor restricts reprogramming and tumorigenesis.</span>\n  <i>Cell stem cell</i>\n</span>\n<span class=\"authors\">Kareta, M. S., Gorges, L. L., Hafeez, S., Benayoun, B. A., Marro, S., Zmoos, A., Cecchini, M. J., Spacek, D., Batista, L. F., O&#39;Brien, M., Ng, Y., Ang, C. E., Vaka, D., Artandi, S. E., Dick, F. A., Brunet, A., Sage, J., Wernig, M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2015</span>; <span class=\"volume\">16 (1)</span><span class=\"pages\">: 39-50</span>\n</span>","chicagoCitation":"Kareta, Michael S., Laura L. Gorges, Sana Hafeez, BÃ©rÃ©nice A. Benayoun, Samuele Marro, et al. 2015. ?Inhibition of Pluripotency Networks by the Rb Tumor Suppressor Restricts Reprogramming and Tumorigenesis.? <i>Cell Stem Cell</i> 16 (1). ELSEVIER INC: 39?50.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Mutations in the retinoblastoma tumor suppressor gene Rb are involved in many forms of human cancer. In this study, we investigated the early consequences of inactivating Rb in the context of cellular reprogramming. We found that Rb inactivation promotes the reprogramming of differentiated cells to a pluripotent state. Unexpectedly, this effect is cell cycle independent, and instead reflects direct binding of Rb to pluripotency genes, including Sox2 and Oct4, which leads to a repressed chromatin state. More broadly, this regulation of pluripotency networks and Sox2 in particular is critical for the initiation of tumors upon loss of Rb in mice. These studies therefore identify Rb as a global transcriptional repressor of pluripotency networks, providing a molecular basis for previous reports about its involvement in cell fate pliability, and implicate misregulation of pluripotency factors such as Sox2 in tumorigenesis related to loss of Rb function.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.stem.2014.10.019\">DOI 10.1016/j.stem.2014.10.019</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/25467916\">PubMedID 25467916</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4389904\">PubMedCentralID PMC4389904</a></span>\n  </p>","text":"Abstract:\nMutations in the retinoblastoma tumor suppressor gene Rb are involved in many forms of human cancer. In this study, we investigated the early consequences of inactivating Rb in the context of cellular reprogramming. We found that Rb inactivation promotes the reprogramming of differentiated cells to a pluripotent state. Unexpectedly, this effect is cell cycle independent, and instead reflects direct binding of Rb to pluripotency genes, including Sox2 and Oct4, which leads to a repressed chromatin state. More broadly, this regulation of pluripotency networks and Sox2 in particular is critical for the initiation of tumors upon loss of Rb in mice. These studies therefore identify Rb as a global transcriptional repressor of pluripotency networks, providing a molecular basis for previous reports about its involvement in cell fate pliability, and implicate misregulation of pluripotency factors such as Sox2 in tumorigenesis related to loss of Rb function.\nDOI: 10.1016/j.stem.2014.10.019, https://dx.doi.org/10.1016/j.stem.2014.10.019\n\nPubMedID: 25467916, https://www.ncbi.nlm.nih.gov/pubmed/25467916\nPubMedCentralID: PMC4389904, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4389904"},"doiId":"10.1016/j.stem.2014.10.019","doiUrl":"https://dx.doi.org/10.1016/j.stem.2014.10.019","featured":true,"firstPublished":{"html":"Jan 8, 2015 00:00:00 AM","text":"Jan 8, 2015 00:00:00 AM","value":"2015-01-08T00:00:00.000-08:00"},"mlaCitation":"Kareta, Michael S. et al. ?Inhibition of Pluripotency Networks by the Rb Tumor Suppressor Restricts Reprogramming and Tumorigenesis.? <i>Cell stem cell</i> 16.1 (2015): 39?50. Print.","pubMedId":"25467916","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/25467916","publicationId":346411,"title":"Inhibition of pluripotency networks by the rb tumor suppressor restricts reprogramming and tumorigenesis.","type":"article"},{"abstract":"In mammals, a cell's decision to divide is thought toÂ be under the control of the Rb/E2F pathway. We previously found that inactivation of the Rb family of cell cycle inhibitors (Rb, p107, and p130) in quiescent liver progenitors leads to uncontrolled division and cancer initiation. Here, we show that, in contrast, deletion of the entire Rb gene family in mature hepatocytes is not sufficient for their long-term proliferation. The cell cycle block in Rb family mutant hepatocytes is independent of the Arf/p53/p21 checkpoint but can be abrogated upon decreasing liver size. At the molecular level, we identify YAP, a transcriptional regulator involved in organ size control, as a factor required for the sustained expression of cell cycle genes in hepatocytes. These experiments identify a higher level of regulation of the cell cycle inÂ vivo in which signals regulating organ size are dominant regulators of the core cell cycle machinery.","apaCitation":"Ehmer, U., Zmoos, A.-F., Auerbach, R. K., Vaka, D., Butte, A. J., Kay, M. A., &amp; Sage, J. (2014). Organ Size Control Is Dominant over Rb Family Inactivation to Restrict Proliferation InÂ Vivo. <i>Cell Reports</i>, <i>8</i>(2), 371?381.","capCitation":"<span class=\"title\">\n  <span>Organ Size Control Is Dominant over Rb Family Inactivation to Restrict Proliferation InÂ Vivo.</span>\n  <i>Cell reports</i>\n</span>\n<span class=\"authors\">Ehmer, U., Zmoos, A., Auerbach, R. K., Vaka, D., Butte, A. J., Kay, M. A., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2014</span>; <span class=\"volume\">8 (2)</span><span class=\"pages\">: 371-381</span>\n</span>","chicagoCitation":"Ehmer, Ursula, Anne-Flore Zmoos, Raymond K. Auerbach, Dedeepya Vaka, Atul J. Butte, et al. 2014. ?Organ Size Control Is Dominant over Rb Family Inactivation to Restrict Proliferation InÂ Vivo.? <i>Cell Reports</i> 8 (2). ELSEVIER INC: 371?81.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">In mammals, a cell's decision to divide is thought toÂ be under the control of the Rb/E2F pathway. We previously found that inactivation of the Rb family of cell cycle inhibitors (Rb, p107, and p130) in quiescent liver progenitors leads to uncontrolled division and cancer initiation. Here, we show that, in contrast, deletion of the entire Rb gene family in mature hepatocytes is not sufficient for their long-term proliferation. The cell cycle block in Rb family mutant hepatocytes is independent of the Arf/p53/p21 checkpoint but can be abrogated upon decreasing liver size. At the molecular level, we identify YAP, a transcriptional regulator involved in organ size control, as a factor required for the sustained expression of cell cycle genes in hepatocytes. These experiments identify a higher level of regulation of the cell cycle inÂ vivo in which signals regulating organ size are dominant regulators of the core cell cycle machinery.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.celrep.2014.06.025\">DOI 10.1016/j.celrep.2014.06.025</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/25017070\">PubMedID 25017070</a></span>\n  </p>","text":"Abstract:\nIn mammals, a cell's decision to divide is thought toÂ be under the control of the Rb/E2F pathway. We previously found that inactivation of the Rb family of cell cycle inhibitors (Rb, p107, and p130) in quiescent liver progenitors leads to uncontrolled division and cancer initiation. Here, we show that, in contrast, deletion of the entire Rb gene family in mature hepatocytes is not sufficient for their long-term proliferation. The cell cycle block in Rb family mutant hepatocytes is independent of the Arf/p53/p21 checkpoint but can be abrogated upon decreasing liver size. At the molecular level, we identify YAP, a transcriptional regulator involved in organ size control, as a factor required for the sustained expression of cell cycle genes in hepatocytes. These experiments identify a higher level of regulation of the cell cycle inÂ vivo in which signals regulating organ size are dominant regulators of the core cell cycle machinery.\nDOI: 10.1016/j.celrep.2014.06.025, https://dx.doi.org/10.1016/j.celrep.2014.06.025\n\nPubMedID: 25017070, https://www.ncbi.nlm.nih.gov/pubmed/25017070"},"doiId":"10.1016/j.celrep.2014.06.025","doiUrl":"https://dx.doi.org/10.1016/j.celrep.2014.06.025","featured":true,"firstPublished":{"html":"Jul 24, 2014 00:00:00 AM","text":"Jul 24, 2014 00:00:00 AM","value":"2014-07-24T00:00:00.000-07:00"},"mlaCitation":"Ehmer, Ursula et al. ?Organ Size Control Is Dominant over Rb Family Inactivation to Restrict Proliferation InÂ Vivo.? <i>Cell reports</i> 8.2 (2014): 371?381. Print.","pubMedId":"25017070","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/25017070","publicationId":322996,"title":"Organ Size Control Is Dominant over Rb Family Inactivation to Restrict Proliferation InÂ Vivo.","type":"article"},{"abstract":"Deregulation of lysine methylation signalling has emerged as a common aetiological factor in cancer pathogenesis, with inhibitors of several histone lysine methyltransferases (KMTs) being developed as chemotherapeutics. The largely cytoplasmic KMT SMYD3 (SET and MYND domain containing protein 3) is overexpressed in numerous human tumours. However, the molecular mechanism by which SMYD3 regulates cancer pathways and its relationship to tumorigenesis in vivo are largely unknown. Here we show that methylation of MAP3K2 by SMYD3 increases MAP kinase signalling and promotes the formation of Ras-driven carcinomas. Using mouse models for pancreatic ductal adenocarcinoma and lung adenocarcinoma, we found that abrogating SMYD3 catalytic activity inhibits tumour development in response to oncogenic Ras. We used protein array technology to identify the MAP3K2 kinase as a target of SMYD3. In cancer cell lines, SMYD3-mediated methylation of MAP3K2 at lysine 260 potentiates activation of the Ras/Raf/MEK/ERK signalling module and SMYD3 depletion synergizes with a MEK inhibitor to block Ras-driven tumorigenesis. Finally, the PP2A phosphatase complex, a key negative regulator of the MAP kinase pathway, binds to MAP3K2 and this interaction is blocked by methylation. Together, our results elucidate a new role for lysine methylation in integrating cytoplasmic kinase-signalling cascades and establish a pivotal role for SMYD3 in the regulation of oncogenic Ras signalling.","apaCitation":"Mazur, P. K., Reynoird, N., Khatri, P., Jansen, P. W. T. C., Wilkinson, A. W., Liu, S., ? Gozani, O. (2014). SMYD3 links lysine methylation of MAP3K2 to Ras-driven cancer. <i>Nature</i>, <i>510</i>(7504), 283?287.","capCitation":"<span class=\"title\">\n  <span>SMYD3 links lysine methylation of MAP3K2 to Ras-driven cancer.</span>\n  <i>Nature</i>\n</span>\n<span class=\"authors\">Mazur, P. K., Reynoird, N., Khatri, P., Jansen, P. W., Wilkinson, A. W., Liu, S., Barbash, O., Van Aller, G. S., Huddleston, M., Dhanak, D., Tummino, P. J., Kruger, R. G., Garcia, B. A., Butte, A. J., Vermeulen, M., Sage, J., Gozani, O.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2014</span>; <span class=\"volume\">510 (7504)</span><span class=\"pages\">: 283-287</span>\n</span>","chicagoCitation":"Mazur, Pawel K., Nicolas Reynoird, Purvesh Khatri, Pascal WTC Jansen, Alex W. Wilkinson, et al. 2014. ?SMYD3 Links Lysine Methylation of MAP3K2 to Ras-Driven Cancer.? <i>Nature</i> 510 (7504): 283?87.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Deregulation of lysine methylation signalling has emerged as a common aetiological factor in cancer pathogenesis, with inhibitors of several histone lysine methyltransferases (KMTs) being developed as chemotherapeutics. The largely cytoplasmic KMT SMYD3 (SET and MYND domain containing protein 3) is overexpressed in numerous human tumours. However, the molecular mechanism by which SMYD3 regulates cancer pathways and its relationship to tumorigenesis in vivo are largely unknown. Here we show that methylation of MAP3K2 by SMYD3 increases MAP kinase signalling and promotes the formation of Ras-driven carcinomas. Using mouse models for pancreatic ductal adenocarcinoma and lung adenocarcinoma, we found that abrogating SMYD3 catalytic activity inhibits tumour development in response to oncogenic Ras. We used protein array technology to identify the MAP3K2 kinase as a target of SMYD3. In cancer cell lines, SMYD3-mediated methylation of MAP3K2 at lysine 260 potentiates activation of the Ras/Raf/MEK/ERK signalling module and SMYD3 depletion synergizes with a MEK inhibitor to block Ras-driven tumorigenesis. Finally, the PP2A phosphatase complex, a key negative regulator of the MAP kinase pathway, binds to MAP3K2 and this interaction is blocked by methylation. Together, our results elucidate a new role for lysine methylation in integrating cytoplasmic kinase-signalling cascades and establish a pivotal role for SMYD3 in the regulation of oncogenic Ras signalling.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nature13320\">DOI 10.1038/nature13320</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/24847881\">PubMedID 24847881</a></span>\n  </p>","text":"Abstract:\nDeregulation of lysine methylation signalling has emerged as a common aetiological factor in cancer pathogenesis, with inhibitors of several histone lysine methyltransferases (KMTs) being developed as chemotherapeutics. The largely cytoplasmic KMT SMYD3 (SET and MYND domain containing protein 3) is overexpressed in numerous human tumours. However, the molecular mechanism by which SMYD3 regulates cancer pathways and its relationship to tumorigenesis in vivo are largely unknown. Here we show that methylation of MAP3K2 by SMYD3 increases MAP kinase signalling and promotes the formation of Ras-driven carcinomas. Using mouse models for pancreatic ductal adenocarcinoma and lung adenocarcinoma, we found that abrogating SMYD3 catalytic activity inhibits tumour development in response to oncogenic Ras. We used protein array technology to identify the MAP3K2 kinase as a target of SMYD3. In cancer cell lines, SMYD3-mediated methylation of MAP3K2 at lysine 260 potentiates activation of the Ras/Raf/MEK/ERK signalling module and SMYD3 depletion synergizes with a MEK inhibitor to block Ras-driven tumorigenesis. Finally, the PP2A phosphatase complex, a key negative regulator of the MAP kinase pathway, binds to MAP3K2 and this interaction is blocked by methylation. Together, our results elucidate a new role for lysine methylation in integrating cytoplasmic kinase-signalling cascades and establish a pivotal role for SMYD3 in the regulation of oncogenic Ras signalling.\nDOI: 10.1038/nature13320, https://dx.doi.org/10.1038/nature13320\n\nPubMedID: 24847881, https://www.ncbi.nlm.nih.gov/pubmed/24847881"},"doiId":"10.1038/nature13320","doiUrl":"https://dx.doi.org/10.1038/nature13320","featured":true,"firstPublished":{"html":"Jun 12, 2014 00:00:00 AM","text":"Jun 12, 2014 00:00:00 AM","value":"2014-06-12T00:00:00.000-07:00"},"mlaCitation":"Mazur, Pawel K. et al. ?SMYD3 Links Lysine Methylation of MAP3K2 to Ras-Driven Cancer.? <i>Nature</i> 510.7504 (2014): 283?287. Print.","pubMedId":"24847881","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/24847881","publicationId":317914,"title":"SMYD3 links lysine methylation of MAP3K2 to Ras-driven cancer.","type":"article"},{"abstract":"Small cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer with high mortality. We used a systematic drug repositioning bioinformatics approach querying a large compendium of gene expression profiles to identify candidate U.S. Food and Drug Administration (FDA)-approved drugs to treat SCLC. We found that tricyclic antidepressants and related molecules potently induce apoptosis in both chemonaÃ¯ve and chemoresistant SCLC cells in culture, in mouse and human SCLC tumors transplanted into immunocompromised mice, and in endogenous tumors from a mouse model for human SCLC. The candidate drugs activate stress pathways and induce cell death in SCLC cells, at least in part by disrupting autocrine survival signals involving neurotransmitters and their G protein-coupled receptors. The candidate drugs inhibit the growth of other neuroendocrine tumors, including pancreatic neuroendocrine tumors and Merkel cell carcinoma. These experiments identify novel targeted strategies that can be rapidly evaluated in patients with neuroendocrine tumors through the repurposing of approved drugs.Our work shows the power of bioinformatics-based drug approaches to rapidly repurpose FDA-approved drugs and identifies a novel class of molecules to treat patients with SCLC, a cancer for which no effective novel systemic treatments have been identified in several decades. In addition, our experiments highlight the importance of novel autocrine mechanisms in promoting the growth of neuroendocrine tumor cells.","apaCitation":"Jahchan, N. S., Dudley, J. T., Mazur, P. K., Flores, N., Yang, D., Palmerton, A., ? Sage, J. (2013). A Drug Repositioning Approach Identifies Tricyclic Antidepressants as Inhibitors of Small Cell Lung Cancer and Other Neuroendocrine Tumors. <i>CANCER DISCOVERY</i>, <i>3</i>(12), 1364?1377.","capCitation":"<span class=\"title\">\n  <span>A Drug Repositioning Approach Identifies Tricyclic Antidepressants as Inhibitors of Small Cell Lung Cancer and Other Neuroendocrine Tumors</span>\n  <i>CANCER DISCOVERY</i>\n</span>\n<span class=\"authors\">Jahchan, N. S., Dudley, J. T., Mazur, P. K., Flores, N., Yang, D., Palmerton, A., Zmoos, A., Vaka, D., Tran, K. Q., Zhou, M., Krasinska, K., Riess, J. W., Neal, J. W., Khatri, P., Park, K. S., Butte, A. J., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2013</span>; <span class=\"volume\">3 (12)</span><span class=\"pages\">: 1364-1377</span>\n</span>","chicagoCitation":"Jahchan, Nadine S., Joel T. Dudley, Pawel K. Mazur, Natasha Flores, Dian Yang, et al. 2013. ?A Drug Repositioning Approach Identifies Tricyclic Antidepressants as Inhibitors of Small Cell Lung Cancer and Other Neuroendocrine Tumors.? <i>CANCER DISCOVERY</i> 3 (12). AMER ASSOC CANCER RESEARCH: 1364?77.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer with high mortality. We used a systematic drug repositioning bioinformatics approach querying a large compendium of gene expression profiles to identify candidate U.S. Food and Drug Administration (FDA)-approved drugs to treat SCLC. We found that tricyclic antidepressants and related molecules potently induce apoptosis in both chemonaÃ¯ve and chemoresistant SCLC cells in culture, in mouse and human SCLC tumors transplanted into immunocompromised mice, and in endogenous tumors from a mouse model for human SCLC. The candidate drugs activate stress pathways and induce cell death in SCLC cells, at least in part by disrupting autocrine survival signals involving neurotransmitters and their G protein-coupled receptors. The candidate drugs inhibit the growth of other neuroendocrine tumors, including pancreatic neuroendocrine tumors and Merkel cell carcinoma. These experiments identify novel targeted strategies that can be rapidly evaluated in patients with neuroendocrine tumors through the repurposing of approved drugs.Our work shows the power of bioinformatics-based drug approaches to rapidly repurpose FDA-approved drugs and identifies a novel class of molecules to treat patients with SCLC, a cancer for which no effective novel systemic treatments have been identified in several decades. In addition, our experiments highlight the importance of novel autocrine mechanisms in promoting the growth of neuroendocrine tumor cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1158/2159-8290.CD-13-0183\">DOI 10.1158/2159-8290.CD-13-0183</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000328257500023\">Web of Science ID 000328257500023</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/24078773\">PubMedID 24078773</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3864571\">PubMedCentralID PMC3864571</a></span>\n  </p>","text":"Abstract:\nSmall cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer with high mortality. We used a systematic drug repositioning bioinformatics approach querying a large compendium of gene expression profiles to identify candidate U.S. Food and Drug Administration (FDA)-approved drugs to treat SCLC. We found that tricyclic antidepressants and related molecules potently induce apoptosis in both chemonaÃ¯ve and chemoresistant SCLC cells in culture, in mouse and human SCLC tumors transplanted into immunocompromised mice, and in endogenous tumors from a mouse model for human SCLC. The candidate drugs activate stress pathways and induce cell death in SCLC cells, at least in part by disrupting autocrine survival signals involving neurotransmitters and their G protein-coupled receptors. The candidate drugs inhibit the growth of other neuroendocrine tumors, including pancreatic neuroendocrine tumors and Merkel cell carcinoma. These experiments identify novel targeted strategies that can be rapidly evaluated in patients with neuroendocrine tumors through the repurposing of approved drugs.Our work shows the power of bioinformatics-based drug approaches to rapidly repurpose FDA-approved drugs and identifies a novel class of molecules to treat patients with SCLC, a cancer for which no effective novel systemic treatments have been identified in several decades. In addition, our experiments highlight the importance of novel autocrine mechanisms in promoting the growth of neuroendocrine tumor cells.\nDOI: 10.1158/2159-8290.CD-13-0183, https://dx.doi.org/10.1158/2159-8290.CD-13-0183\nWeb of Science ID: 000328257500023, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000328257500023\nPubMedID: 24078773, https://www.ncbi.nlm.nih.gov/pubmed/24078773\nPubMedCentralID: PMC3864571, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3864571"},"doiId":"10.1158/2159-8290.CD-13-0183","doiUrl":"https://dx.doi.org/10.1158/2159-8290.CD-13-0183","featured":true,"firstPublished":{"html":"Dec 1, 2013 00:00:00 AM","text":"Dec 1, 2013 00:00:00 AM","value":"2013-12-01T00:00:00.000-08:00"},"mlaCitation":"Jahchan, Nadine S. et al. ?A Drug Repositioning Approach Identifies Tricyclic Antidepressants as Inhibitors of Small Cell Lung Cancer and Other Neuroendocrine Tumors.? <i>CANCER DISCOVERY</i> 3.12 (2013): 1364?1377. Print.","pubMedId":"24078773","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/24078773","publicationId":285360,"title":"A Drug Repositioning Approach Identifies Tricyclic Antidepressants as Inhibitors of Small Cell Lung Cancer and Other Neuroendocrine Tumors","type":"article","webOfScienceId":"000328257500023","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000328257500023"},{"abstract":"The mechanisms ensuring the long-term self-renewal of human embryonic stem cells are still only partly understood, limiting their use in cellular therapies. Here we found that increased activity of the RB cell cycle inhibitor in human embryonic stem cells induces cell cycle arrest, differentiation and cell death. Conversely, inactivation of the entire RB family (RB, p107 and p130) in human embryonic stem cells triggers G2/M arrest and cell death through functional activation of the p53 pathway and the cell cycle inhibitor p21. Differences in E2F target gene activation upon loss of RB family function between human embryonic stem cells, mouse embryonic stem cells and human fibroblasts underscore key differences in the cell cycle regulatory networks of human embryonic stem cells. Finally, loss of RB family function promotes genomic instability in both human and mouse embryonic stem cells, uncoupling cell cycle defects from chromosomal instability. These experiments indicate that a homeostatic level of RB activity is essential for the self-renewal and the survival of human embryonic stem cells.","apaCitation":"Conklin, J. F., Baker, J., &amp; Sage, J. (2012). The RB family is required for the self-renewal and survival of human embryonic stem cells. <i>NATURE COMMUNICATIONS</i>, <i>3</i>.","capCitation":"<span class=\"title\">\n  <span>The RB family is required for the self-renewal and survival of human embryonic stem cells</span>\n  <i>NATURE COMMUNICATIONS</i>\n</span>\n<span class=\"authors\">Conklin, J. F., Baker, J., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2012</span>; <span class=\"volume\">3</span>\n</span>","chicagoCitation":"Conklin, Jamie F., Julie Baker, and Julien Sage. 2012. ?The RB Family Is Required for the Self-Renewal and Survival of Human Embryonic Stem Cells.? <i>NATURE COMMUNICATIONS</i> 3. NATURE PUBLISHING GROUP.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The mechanisms ensuring the long-term self-renewal of human embryonic stem cells are still only partly understood, limiting their use in cellular therapies. Here we found that increased activity of the RB cell cycle inhibitor in human embryonic stem cells induces cell cycle arrest, differentiation and cell death. Conversely, inactivation of the entire RB family (RB, p107 and p130) in human embryonic stem cells triggers G2/M arrest and cell death through functional activation of the p53 pathway and the cell cycle inhibitor p21. Differences in E2F target gene activation upon loss of RB family function between human embryonic stem cells, mouse embryonic stem cells and human fibroblasts underscore key differences in the cell cycle regulatory networks of human embryonic stem cells. Finally, loss of RB family function promotes genomic instability in both human and mouse embryonic stem cells, uncoupling cell cycle defects from chromosomal instability. These experiments indicate that a homeostatic level of RB activity is essential for the self-renewal and the survival of human embryonic stem cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/ncomms2254\">DOI 10.1038/ncomms2254</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000316356700012\">Web of Science ID 000316356700012</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/23212373\">PubMedID 23212373</a></span>\n  </p>","text":"Abstract:\nThe mechanisms ensuring the long-term self-renewal of human embryonic stem cells are still only partly understood, limiting their use in cellular therapies. Here we found that increased activity of the RB cell cycle inhibitor in human embryonic stem cells induces cell cycle arrest, differentiation and cell death. Conversely, inactivation of the entire RB family (RB, p107 and p130) in human embryonic stem cells triggers G2/M arrest and cell death through functional activation of the p53 pathway and the cell cycle inhibitor p21. Differences in E2F target gene activation upon loss of RB family function between human embryonic stem cells, mouse embryonic stem cells and human fibroblasts underscore key differences in the cell cycle regulatory networks of human embryonic stem cells. Finally, loss of RB family function promotes genomic instability in both human and mouse embryonic stem cells, uncoupling cell cycle defects from chromosomal instability. These experiments indicate that a homeostatic level of RB activity is essential for the self-renewal and the survival of human embryonic stem cells.\nDOI: 10.1038/ncomms2254, https://dx.doi.org/10.1038/ncomms2254\nWeb of Science ID: 000316356700012, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000316356700012\nPubMedID: 23212373, https://www.ncbi.nlm.nih.gov/pubmed/23212373"},"doiId":"10.1038/ncomms2254","doiUrl":"https://dx.doi.org/10.1038/ncomms2254","featured":true,"firstPublished":{"html":"Dec 1, 2012 00:00:00 AM","text":"Dec 1, 2012 00:00:00 AM","value":"2012-12-01T00:00:00.000-08:00"},"mlaCitation":"Conklin, Jamie F., Julie Baker, and Julien Sage. ?The RB Family Is Required for the Self-Renewal and Survival of Human Embryonic Stem Cells.? <i>NATURE COMMUNICATIONS</i> 3 (2012): n. pag. Print.","pubMedId":"23212373","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/23212373","publicationId":127456,"title":"The RB family is required for the self-renewal and survival of human embryonic stem cells","type":"article","webOfScienceId":"000316356700012","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000316356700012"},{"abstract":"Small-cell lung cancer (SCLC) is an aggressive lung tumor subtype with poor prognosis. We sequenced 29 SCLC exomes, 2 genomes and 15 transcriptomes and found an extremely high mutation rate of 7.4Â±1 protein-changing mutations per million base pairs. Therefore, we conducted integrated analyses of the various data sets to identify pathogenetically relevant mutated genes. In all cases, we found evidence for inactivation of TP53 and RB1 and identified recurrent mutations in the CREBBP, EP300 and MLL genes that encode histone modifiers. Furthermore, we observed mutations in PTEN, SLIT2 and EPHA7, as well as focal amplifications of the FGFR1 tyrosine kinase gene. Finally, we detected many of the alterations found in humans in SCLC tumors from Tp53 and Rb1 double knockout mice. Our study implicates histone modification as a major feature of SCLC, reveals potentially therapeutically tractable genomic alterations and provides a generalizable framework for the identification of biologically relevant genes in the context of high mutational background.","apaCitation":"Peifer, M., Fernandez-Cuesta, L., Sos, M. L., George, J., Seidel, D., Kasper, L. H., ? Thomas, R. K. (2012). Integrative genome analyses identify key somatic driver mutations of small-cell lung cancer. <i>NATURE GENETICS</i>, <i>44</i>(10), 1104-?","capCitation":"<span class=\"title\">\n  <span>Integrative genome analyses identify key somatic driver mutations of small-cell lung cancer</span>\n  <i>NATURE GENETICS</i>\n</span>\n<span class=\"authors\">Peifer, M., Fernandez-Cuesta, L., Sos, M. L., George, J., Seidel, D., Kasper, L. H., Plenker, D., Leenders, F., Sun, R., Zander, T., Menon, R., Koker, M., Dahmen, I., Mueller, C., Di Cerbo, V., Schildhaus, H., Altmueller, J., Baessmann, I., Becker, C., De Wilde, B., Vandesompele, J., Boehm, D., Ansen, S., Gabler, F., Wilkening, I., Heynck, S., Heuckmann, J. M., Lu, X., Carter, S. L., Cibulskis, K., Banerji, S., Getz, G., Park, K., Rauh, D., Gruetter, C., Fischer, M., Pasqualucci, L., Wright, G., Wainer, Z., Russell, P., Petersen, I., Chen, Y., Stoelben, E., Ludwig, C., Schnabel, P., Hoffmann, H., Muley, T., Brockmann, M., Engel-Riedel, W., Muscarella, L. A., Fazio, V. M., Groen, H., Timens, W., Sietsma, H., Thunnissen, E., Smit, E., Heideman, D. A., Snijders, P. J., Cappuzzo, F., Ligorio, C., Damiani, S., Field, J., Solberg, S., Brustugun, O. T., Lund-Iversen, M., Saenger, J., Clement, J. H., Soltermann, A., Moch, H., Weder, W., Solomon, B., Soria, J., Validire, P., Besse, B., Brambilla, E., Brambilla, C., Lantuejoul, S., Lorimier, P., Schneider, P. M., Hallek, M., Pao, W., Meyerson, M., Sage, J., Shendure, J., Schneider, R., Buettner, R., Wolf, J., Nuernberg, P., Perner, S., Heukamp, L. C., Brindle, P. K., Haas, S., Thomas, R. K.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2012</span>; <span class=\"volume\">44 (10)</span><span class=\"pages\">: 1104-?</span>\n</span>","chicagoCitation":"Peifer, Martin, Lynnette Fernandez-Cuesta, Martin L. Sos, Julie George, Danila Seidel, et al. 2012. ?Integrative Genome Analyses Identify Key Somatic Driver Mutations of Small-Cell Lung Cancer.? <i>NATURE GENETICS</i> 44 (10). NATURE PUBLISHING GROUP: 1104-?","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small-cell lung cancer (SCLC) is an aggressive lung tumor subtype with poor prognosis. We sequenced 29 SCLC exomes, 2 genomes and 15 transcriptomes and found an extremely high mutation rate of 7.4Â±1 protein-changing mutations per million base pairs. Therefore, we conducted integrated analyses of the various data sets to identify pathogenetically relevant mutated genes. In all cases, we found evidence for inactivation of TP53 and RB1 and identified recurrent mutations in the CREBBP, EP300 and MLL genes that encode histone modifiers. Furthermore, we observed mutations in PTEN, SLIT2 and EPHA7, as well as focal amplifications of the FGFR1 tyrosine kinase gene. Finally, we detected many of the alterations found in humans in SCLC tumors from Tp53 and Rb1 double knockout mice. Our study implicates histone modification as a major feature of SCLC, reveals potentially therapeutically tractable genomic alterations and provides a generalizable framework for the identification of biologically relevant genes in the context of high mutational background.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/ng.2396\">DOI 10.1038/ng.2396</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000309550200009\">Web of Science ID 000309550200009</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/22941188\">PubMedID 22941188</a></span>\n  </p>","text":"Abstract:\nSmall-cell lung cancer (SCLC) is an aggressive lung tumor subtype with poor prognosis. We sequenced 29 SCLC exomes, 2 genomes and 15 transcriptomes and found an extremely high mutation rate of 7.4Â±1 protein-changing mutations per million base pairs. Therefore, we conducted integrated analyses of the various data sets to identify pathogenetically relevant mutated genes. In all cases, we found evidence for inactivation of TP53 and RB1 and identified recurrent mutations in the CREBBP, EP300 and MLL genes that encode histone modifiers. Furthermore, we observed mutations in PTEN, SLIT2 and EPHA7, as well as focal amplifications of the FGFR1 tyrosine kinase gene. Finally, we detected many of the alterations found in humans in SCLC tumors from Tp53 and Rb1 double knockout mice. Our study implicates histone modification as a major feature of SCLC, reveals potentially therapeutically tractable genomic alterations and provides a generalizable framework for the identification of biologically relevant genes in the context of high mutational background.\nDOI: 10.1038/ng.2396, https://dx.doi.org/10.1038/ng.2396\nWeb of Science ID: 000309550200009, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000309550200009\nPubMedID: 22941188, https://www.ncbi.nlm.nih.gov/pubmed/22941188"},"doiId":"10.1038/ng.2396","doiUrl":"https://dx.doi.org/10.1038/ng.2396","featured":true,"firstPublished":{"html":"Oct 1, 2012 00:00:00 AM","text":"Oct 1, 2012 00:00:00 AM","value":"2012-10-01T00:00:00.000-07:00"},"mlaCitation":"Peifer, Martin et al. ?Integrative Genome Analyses Identify Key Somatic Driver Mutations of Small-Cell Lung Cancer.? <i>NATURE GENETICS</i> 44.10 (2012): 1104-? Print.","pubMedId":"22941188","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/22941188","publicationId":187872,"title":"Integrative genome analyses identify key somatic driver mutations of small-cell lung cancer","type":"article","webOfScienceId":"000309550200009","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000309550200009"},{"abstract":"Stem cells play a critical role during embryonic development and in the maintenance of homeostasis in adult individuals. A better understanding of stem cell biology, including embryonic and adult stem cells, will allow the scientific community to better comprehend a number of pathologies and possibly design novel approaches to treat patients with a variety of diseases. The retinoblastoma tumor suppressor RB controls the proliferation, differentiation, and survival of cells, and accumulating evidence points to a central role for RB activity in the biology of stem and progenitor cells. In some contexts, loss of RB function in stem or progenitor cells is a key event in the initiation of cancer and determines the subtype of cancer arising from these pluripotent cells by altering their fate. In other cases, RB inactivation is often not sufficient to initiate cancer but may still lead to some stem cell expansion, raising the possibility that strategies aimed at transiently inactivating RB might provide a novel way to expand functional stem cell populations. Future experiments dedicated to better understanding how RB and the RB pathway control a stem cell's decisions to divide, self-renew, or give rise to differentiated progeny may eventually increase our capacity to control these decisions to enhance regeneration or help prevent cancer development.","apaCitation":"Sage, J. (2012). The retinoblastoma tumor suppressor and stem cell biology. <i>GENES &amp; DEVELOPMENT</i>, <i>26</i>(13), 1409?1420.","capCitation":"<span class=\"title\">\n  <span>The retinoblastoma tumor suppressor and stem cell biology</span>\n  <i>GENES &amp; DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2012</span>; <span class=\"volume\">26 (13)</span><span class=\"pages\">: 1409-1420</span>\n</span>","chicagoCitation":"Sage, Julien. 2012. ?The Retinoblastoma Tumor Suppressor and Stem Cell Biology.? <i>GENES &amp; DEVELOPMENT</i> 26 (13). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 1409?20.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Stem cells play a critical role during embryonic development and in the maintenance of homeostasis in adult individuals. A better understanding of stem cell biology, including embryonic and adult stem cells, will allow the scientific community to better comprehend a number of pathologies and possibly design novel approaches to treat patients with a variety of diseases. The retinoblastoma tumor suppressor RB controls the proliferation, differentiation, and survival of cells, and accumulating evidence points to a central role for RB activity in the biology of stem and progenitor cells. In some contexts, loss of RB function in stem or progenitor cells is a key event in the initiation of cancer and determines the subtype of cancer arising from these pluripotent cells by altering their fate. In other cases, RB inactivation is often not sufficient to initiate cancer but may still lead to some stem cell expansion, raising the possibility that strategies aimed at transiently inactivating RB might provide a novel way to expand functional stem cell populations. Future experiments dedicated to better understanding how RB and the RB pathway control a stem cell's decisions to divide, self-renew, or give rise to differentiated progeny may eventually increase our capacity to control these decisions to enhance regeneration or help prevent cancer development.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.193730.112\">DOI 10.1101/gad.193730.112</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000306175400002\">Web of Science ID 000306175400002</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/22751497\">PubMedID 22751497</a></span>\n  </p>","text":"Abstract:\nStem cells play a critical role during embryonic development and in the maintenance of homeostasis in adult individuals. A better understanding of stem cell biology, including embryonic and adult stem cells, will allow the scientific community to better comprehend a number of pathologies and possibly design novel approaches to treat patients with a variety of diseases. The retinoblastoma tumor suppressor RB controls the proliferation, differentiation, and survival of cells, and accumulating evidence points to a central role for RB activity in the biology of stem and progenitor cells. In some contexts, loss of RB function in stem or progenitor cells is a key event in the initiation of cancer and determines the subtype of cancer arising from these pluripotent cells by altering their fate. In other cases, RB inactivation is often not sufficient to initiate cancer but may still lead to some stem cell expansion, raising the possibility that strategies aimed at transiently inactivating RB might provide a novel way to expand functional stem cell populations. Future experiments dedicated to better understanding how RB and the RB pathway control a stem cell's decisions to divide, self-renew, or give rise to differentiated progeny may eventually increase our capacity to control these decisions to enhance regeneration or help prevent cancer development.\nDOI: 10.1101/gad.193730.112, https://dx.doi.org/10.1101/gad.193730.112\nWeb of Science ID: 000306175400002, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000306175400002\nPubMedID: 22751497, https://www.ncbi.nlm.nih.gov/pubmed/22751497"},"doiId":"10.1101/gad.193730.112","doiUrl":"https://dx.doi.org/10.1101/gad.193730.112","featured":true,"firstPublished":{"html":"Jul 1, 2012 00:00:00 AM","text":"Jul 1, 2012 00:00:00 AM","value":"2012-07-01T00:00:00.000-07:00"},"mlaCitation":"Sage, Julien. ?The Retinoblastoma Tumor Suppressor and Stem Cell Biology.? <i>GENES &amp; DEVELOPMENT</i> 26.13 (2012): 1409?1420. Print.","pubMedId":"22751497","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/22751497","publicationId":122469,"title":"The retinoblastoma tumor suppressor and stem cell biology","type":"article","webOfScienceId":"000306175400002","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000306175400002"},{"abstract":"Smyd3 is a lysine methyltransferase implicated in chromatin and cancer regulation. Here we show that Smyd3 catalyzes histone H4 methylation at lysine 5 (H4K5me). This novel histone methylation mark is detected in diverse cell types and its formation is attenuated by depletion of Smyd3 protein. Further, Smyd3-driven cancer cell phenotypes require its enzymatic activity. Thus, Smyd3, via H4K5 methylation, provides a potential new link between chromatin dynamics and neoplastic disease.","apaCitation":"Van Aller, G. S., Reynoird, N., Barbash, O., Huddleston, M., Liu, S., Zmoos, A.-F., ? Kruger, R. G. (2012). Smyd3 regulates cancer cell phenotypes and catalyzes histone H4 lysine 5 methylation. <i>EPIGENETICS</i>, <i>7</i>(4), 340?343.","capCitation":"<span class=\"title\">\n  <span>Smyd3 regulates cancer cell phenotypes and catalyzes histone H4 lysine 5 methylation</span>\n  <i>EPIGENETICS</i>\n</span>\n<span class=\"authors\">Van Aller, G. S., Reynoird, N., Barbash, O., Huddleston, M., Liu, S., Zmoos, A., McDevitt, P., Sinnamon, R., Le, B., Mas, G., Annan, R., Sage, J., Garcia, B. A., Tummino, P. J., Gozani, O., Kruger, R. G.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2012</span>; <span class=\"volume\">7 (4)</span><span class=\"pages\">: 340-343</span>\n</span>","chicagoCitation":"Van Aller, Glenn S., Nicolas Reynoird, Olena Barbash, Michael Huddleston, Shichong Liu, et al. 2012. ?Smyd3 Regulates Cancer Cell Phenotypes and Catalyzes Histone H4 Lysine 5 Methylation.? <i>EPIGENETICS</i> 7 (4). LANDES BIOSCIENCE: 340?43.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Smyd3 is a lysine methyltransferase implicated in chromatin and cancer regulation. Here we show that Smyd3 catalyzes histone H4 methylation at lysine 5 (H4K5me). This novel histone methylation mark is detected in diverse cell types and its formation is attenuated by depletion of Smyd3 protein. Further, Smyd3-driven cancer cell phenotypes require its enzymatic activity. Thus, Smyd3, via H4K5 methylation, provides a potential new link between chromatin dynamics and neoplastic disease.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000302493300004\">Web of Science ID 000302493300004</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/22419068\">PubMedID 22419068</a></span>\n  </p>","text":"Abstract:\nSmyd3 is a lysine methyltransferase implicated in chromatin and cancer regulation. Here we show that Smyd3 catalyzes histone H4 methylation at lysine 5 (H4K5me). This novel histone methylation mark is detected in diverse cell types and its formation is attenuated by depletion of Smyd3 protein. Further, Smyd3-driven cancer cell phenotypes require its enzymatic activity. Thus, Smyd3, via H4K5 methylation, provides a potential new link between chromatin dynamics and neoplastic disease.\n\nWeb of Science ID: 000302493300004, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000302493300004\nPubMedID: 22419068, https://www.ncbi.nlm.nih.gov/pubmed/22419068"},"featured":true,"firstPublished":{"html":"Apr 1, 2012 00:00:00 AM","text":"Apr 1, 2012 00:00:00 AM","value":"2012-04-01T00:00:00.000-07:00"},"mlaCitation":"Van Aller, Glenn S. et al. ?Smyd3 Regulates Cancer Cell Phenotypes and Catalyzes Histone H4 Lysine 5 Methylation.? <i>EPIGENETICS</i> 7.4 (2012): 340?343. Print.","pubMedId":"22419068","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/22419068","publicationId":118651,"title":"Smyd3 regulates cancer cell phenotypes and catalyzes histone H4 lysine 5 methylation","type":"article","webOfScienceId":"000302493300004","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000302493300004"},{"abstract":"Small-cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer for which there is no effective treatment. Using a mouse model in which deletion of Rb1 and Trp53 in the lung epithelium of adult mice induces SCLC, we found that the Hedgehog signaling pathway is activated in SCLC cells independently of the lung microenvironment. Constitutive activation of the Hedgehog signaling molecule Smoothened (Smo) promoted the clonogenicity of human SCLC in vitro and the initiation and progression of mouse SCLC in vivo. Reciprocally, deletion of Smo in Rb1 and Trp53-mutant lung epithelial cells strongly suppressed SCLC initiation and progression in mice. Furthermore, pharmacological blockade of Hedgehog signaling inhibited the growth of mouse and human SCLC, most notably following chemotherapy. These findings show a crucial cell-intrinsic role for Hedgehog signaling in the development and maintenance of SCLC and identify Hedgehog pathway inhibition as a therapeutic strategy to slow the progression of disease and delay cancer recurrence in individuals with SCLC.","apaCitation":"Park, K.-S., Martelotto, L. G., Peifer, M., Sos, M. L., Karnezis, A. N., Mahjoub, M. R., ? Sage, J. (2011). A crucial requirement for Hedgehog signaling in small cell lung cancer. <i>NATURE MEDICINE</i>, <i>17</i>(11), 1504?U1506.","capCitation":"<span class=\"title\">\n  <span>A crucial requirement for Hedgehog signaling in small cell lung cancer</span>\n  <i>NATURE MEDICINE</i>\n</span>\n<span class=\"authors\">Park, K., Martelotto, L. G., Peifer, M., Sos, M. L., Karnezis, A. N., Mahjoub, M. R., Bernard, K., Conklin, J. F., Szczepny, A., Yuan, J., Guo, R., Ospina, B., Falzon, J., Bennett, S., Brown, T. J., Markovic, A., Devereux, W. L., Ocasio, C. A., Chen, J. K., Stearns, T., Thomas, R. K., Dorsch, M., Buonamici, S., Watkins, D. N., Peacock, C. D., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">17 (11)</span><span class=\"pages\">: 1504-U1506</span>\n</span>","chicagoCitation":"Park, Kwon-Sik, Luciano G. Martelotto, Martin Peifer, Martin L. Sos, Anthony N. Karnezis, et al. 2011. ?A Crucial Requirement for Hedgehog Signaling in Small Cell Lung Cancer.? <i>NATURE MEDICINE</i> 17 (11). NATURE PUBLISHING GROUP: 1504?U1506.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small-cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer for which there is no effective treatment. Using a mouse model in which deletion of Rb1 and Trp53 in the lung epithelium of adult mice induces SCLC, we found that the Hedgehog signaling pathway is activated in SCLC cells independently of the lung microenvironment. Constitutive activation of the Hedgehog signaling molecule Smoothened (Smo) promoted the clonogenicity of human SCLC in vitro and the initiation and progression of mouse SCLC in vivo. Reciprocally, deletion of Smo in Rb1 and Trp53-mutant lung epithelial cells strongly suppressed SCLC initiation and progression in mice. Furthermore, pharmacological blockade of Hedgehog signaling inhibited the growth of mouse and human SCLC, most notably following chemotherapy. These findings show a crucial cell-intrinsic role for Hedgehog signaling in the development and maintenance of SCLC and identify Hedgehog pathway inhibition as a therapeutic strategy to slow the progression of disease and delay cancer recurrence in individuals with SCLC.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nm.2473\">DOI 10.1038/nm.2473</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000296779300043\">Web of Science ID 000296779300043</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21983857\">PubMedID 21983857</a></span>\n  </p>\n  <p class=\"pub-med-central\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3380617\">PubMedCentralID PMC3380617</a></span>\n  </p>","text":"Abstract:\nSmall-cell lung cancer (SCLC) is an aggressive neuroendocrine subtype of lung cancer for which there is no effective treatment. Using a mouse model in which deletion of Rb1 and Trp53 in the lung epithelium of adult mice induces SCLC, we found that the Hedgehog signaling pathway is activated in SCLC cells independently of the lung microenvironment. Constitutive activation of the Hedgehog signaling molecule Smoothened (Smo) promoted the clonogenicity of human SCLC in vitro and the initiation and progression of mouse SCLC in vivo. Reciprocally, deletion of Smo in Rb1 and Trp53-mutant lung epithelial cells strongly suppressed SCLC initiation and progression in mice. Furthermore, pharmacological blockade of Hedgehog signaling inhibited the growth of mouse and human SCLC, most notably following chemotherapy. These findings show a crucial cell-intrinsic role for Hedgehog signaling in the development and maintenance of SCLC and identify Hedgehog pathway inhibition as a therapeutic strategy to slow the progression of disease and delay cancer recurrence in individuals with SCLC.\nDOI: 10.1038/nm.2473, https://dx.doi.org/10.1038/nm.2473\nWeb of Science ID: 000296779300043, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000296779300043\nPubMedID: 21983857, https://www.ncbi.nlm.nih.gov/pubmed/21983857\nPubMedCentralID: PMC3380617, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3380617"},"doiId":"10.1038/nm.2473","doiUrl":"https://dx.doi.org/10.1038/nm.2473","featured":true,"firstPublished":{"html":"Nov 1, 2011 00:00:00 AM","text":"Nov 1, 2011 00:00:00 AM","value":"2011-11-01T00:00:00.000-07:00"},"mlaCitation":"Park, Kwon-Sik et al. ?A Crucial Requirement for Hedgehog Signaling in Small Cell Lung Cancer.? <i>NATURE MEDICINE</i> 17.11 (2011): 1504?U1506. Print.","pubMedId":"21983857","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21983857","publicationId":113515,"title":"A crucial requirement for Hedgehog signaling in small cell lung cancer","type":"article","webOfScienceId":"000296779300043","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000296779300043"},{"abstract":"Hepatocellular carcinoma (HCC) is the third cancer killer worldwide with >600,000 deaths every year. Although the major risk factors are known, therapeutic options in patients remain limited in part because of our incomplete understanding of the cellular and molecular mechanisms influencing HCC development. Evidence indicates that the retinoblastoma (RB) pathway is functionally inactivated in most cases of HCC by genetic, epigenetic, and/or viral mechanisms. To investigate the functional relevance of this observation, we inactivated the RB pathway in the liver of adult mice by deleting the three members of the Rb (Rb1) gene family: Rb, p107, and p130. Rb family triple knockout mice develop liver tumors with histopathological features and gene expression profiles similar to human HCC. In this mouse model, cancer initiation is associated with the specific expansion of populations of liver stem/progenitor cells, indicating that the RB pathway may prevent HCC development by maintaining the quiescence of adult liver progenitor cells. In addition, we show that during tumor progression, activation of the Notch pathway via E2F transcription factors serves as a negative feedback mechanism to slow HCC growth. The level of Notch activity is also able to predict survival of HCC patients, suggesting novel means to diagnose and treat HCC.","apaCitation":"Viatour, P., Ehmer, U., Saddic, L. A., Dorrell, C., Andersen, J. B., Lin, C. W., ? Sage, J. (2011). Notch signaling inhibits hepatocellular carcinoma following inactivation of the RB pathway. <i>JOURNAL OF EXPERIMENTAL MEDICINE</i>, <i>208</i>(10), 1963?1976.","capCitation":"<span class=\"title\">\n  <span>Notch signaling inhibits hepatocellular carcinoma following inactivation of the RB pathway</span>\n  <i>JOURNAL OF EXPERIMENTAL MEDICINE</i>\n</span>\n<span class=\"authors\">Viatour, P., Ehmer, U., Saddic, L. A., Dorrell, C., Andersen, J. B., Lin, C., Zmoos, A., Mazur, P. K., Schaffer, B. E., Ostermeier, A., Vogel, H., Sylvester, K. G., Thorgeirsson, S. S., Grompe, M., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">208 (10)</span><span class=\"pages\">: 1963-1976</span>\n</span>","chicagoCitation":"Viatour, Patrick, Ursula Ehmer, Louis A. Saddic, Craig Dorrell, Jesper B. Andersen, et al. 2011. ?Notch Signaling Inhibits Hepatocellular Carcinoma Following Inactivation of the RB Pathway.? <i>JOURNAL OF EXPERIMENTAL MEDICINE</i> 208 (10). ROCKEFELLER UNIV PRESS: 1963?76.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Hepatocellular carcinoma (HCC) is the third cancer killer worldwide with >600,000 deaths every year. Although the major risk factors are known, therapeutic options in patients remain limited in part because of our incomplete understanding of the cellular and molecular mechanisms influencing HCC development. Evidence indicates that the retinoblastoma (RB) pathway is functionally inactivated in most cases of HCC by genetic, epigenetic, and/or viral mechanisms. To investigate the functional relevance of this observation, we inactivated the RB pathway in the liver of adult mice by deleting the three members of the Rb (Rb1) gene family: Rb, p107, and p130. Rb family triple knockout mice develop liver tumors with histopathological features and gene expression profiles similar to human HCC. In this mouse model, cancer initiation is associated with the specific expansion of populations of liver stem/progenitor cells, indicating that the RB pathway may prevent HCC development by maintaining the quiescence of adult liver progenitor cells. In addition, we show that during tumor progression, activation of the Notch pathway via E2F transcription factors serves as a negative feedback mechanism to slow HCC growth. The level of Notch activity is also able to predict survival of HCC patients, suggesting novel means to diagnose and treat HCC.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1084/jem.20110198\">DOI 10.1084/jem.20110198</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000295318900005\">Web of Science ID 000295318900005</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21875955\">PubMedID 21875955</a></span>\n  </p>","text":"Abstract:\nHepatocellular carcinoma (HCC) is the third cancer killer worldwide with >600,000 deaths every year. Although the major risk factors are known, therapeutic options in patients remain limited in part because of our incomplete understanding of the cellular and molecular mechanisms influencing HCC development. Evidence indicates that the retinoblastoma (RB) pathway is functionally inactivated in most cases of HCC by genetic, epigenetic, and/or viral mechanisms. To investigate the functional relevance of this observation, we inactivated the RB pathway in the liver of adult mice by deleting the three members of the Rb (Rb1) gene family: Rb, p107, and p130. Rb family triple knockout mice develop liver tumors with histopathological features and gene expression profiles similar to human HCC. In this mouse model, cancer initiation is associated with the specific expansion of populations of liver stem/progenitor cells, indicating that the RB pathway may prevent HCC development by maintaining the quiescence of adult liver progenitor cells. In addition, we show that during tumor progression, activation of the Notch pathway via E2F transcription factors serves as a negative feedback mechanism to slow HCC growth. The level of Notch activity is also able to predict survival of HCC patients, suggesting novel means to diagnose and treat HCC.\nDOI: 10.1084/jem.20110198, https://dx.doi.org/10.1084/jem.20110198\nWeb of Science ID: 000295318900005, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000295318900005\nPubMedID: 21875955, https://www.ncbi.nlm.nih.gov/pubmed/21875955"},"doiId":"10.1084/jem.20110198","doiUrl":"https://dx.doi.org/10.1084/jem.20110198","featured":true,"firstPublished":{"html":"Sep 26, 2011 00:00:00 AM","text":"Sep 26, 2011 00:00:00 AM","value":"2011-09-26T00:00:00.000-07:00"},"mlaCitation":"Viatour, Patrick et al. ?Notch Signaling Inhibits Hepatocellular Carcinoma Following Inactivation of the RB Pathway.? <i>JOURNAL OF EXPERIMENTAL MEDICINE</i> 208.10 (2011): 1963?1976. Print.","pubMedId":"21875955","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21875955","publicationId":112237,"title":"Notch signaling inhibits hepatocellular carcinoma following inactivation of the RB pathway","type":"article","webOfScienceId":"000295318900005","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000295318900005"},{"abstract":"Retinoblastoma is a rare pediatric cancer that has served as a paradigm to investigate the mechanisms of tumorigenesis. In this issue of Genes & Development, Conkrite and colleagues (pp. 1734-1745) found high levels of the miR-17~92 and miR-106b-25 microRNAs in primary retinoblastomas and show that overexpression of miR-17~92 accelerates retinoblastoma development in mice by promoting proliferation, in part by reducing expression of the cell cycle inhibitor p21. These experiments identify the RB/miR-17~92/p21 axis as a critical regulator of retinoblastoma tumorigenesis and potentially many other cancers.","apaCitation":"Sage, J., &amp; Ventura, A. (2011). miR than meets the eye. <i>GENES &amp; DEVELOPMENT</i>, <i>25</i>(16), 1663?1667.","capCitation":"<span class=\"title\">\n  <span>miR than meets the eye</span>\n  <i>GENES &amp; DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Sage, J., Ventura, A.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2011</span>; <span class=\"volume\">25 (16)</span><span class=\"pages\">: 1663-1667</span>\n</span>","chicagoCitation":"Sage, Julien, and Andrea Ventura. 2011. ?MiR than Meets the Eye.? <i>GENES &amp; DEVELOPMENT</i> 25 (16). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 1663?67.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Retinoblastoma is a rare pediatric cancer that has served as a paradigm to investigate the mechanisms of tumorigenesis. In this issue of Genes & Development, Conkrite and colleagues (pp. 1734-1745) found high levels of the miR-17~92 and miR-106b-25 microRNAs in primary retinoblastomas and show that overexpression of miR-17~92 accelerates retinoblastoma development in mice by promoting proliferation, in part by reducing expression of the cell cycle inhibitor p21. These experiments identify the RB/miR-17~92/p21 axis as a critical regulator of retinoblastoma tumorigenesis and potentially many other cancers.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1101/gad.17454011\">DOI 10.1101/gad.17454011</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000294003800001\">Web of Science ID 000294003800001</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21852531\">PubMedID 21852531</a></span>\n  </p>","text":"Abstract:\nRetinoblastoma is a rare pediatric cancer that has served as a paradigm to investigate the mechanisms of tumorigenesis. In this issue of Genes & Development, Conkrite and colleagues (pp. 1734-1745) found high levels of the miR-17~92 and miR-106b-25 microRNAs in primary retinoblastomas and show that overexpression of miR-17~92 accelerates retinoblastoma development in mice by promoting proliferation, in part by reducing expression of the cell cycle inhibitor p21. These experiments identify the RB/miR-17~92/p21 axis as a critical regulator of retinoblastoma tumorigenesis and potentially many other cancers.\nDOI: 10.1101/gad.17454011, https://dx.doi.org/10.1101/gad.17454011\nWeb of Science ID: 000294003800001, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000294003800001\nPubMedID: 21852531, https://www.ncbi.nlm.nih.gov/pubmed/21852531"},"doiId":"10.1101/gad.17454011","doiUrl":"https://dx.doi.org/10.1101/gad.17454011","featured":true,"firstPublished":{"html":"Aug 15, 2011 00:00:00 AM","text":"Aug 15, 2011 00:00:00 AM","value":"2011-08-15T00:00:00.000-07:00"},"mlaCitation":"Sage, Julien, and Andrea Ventura. ?MiR than Meets the Eye.? <i>GENES &amp; DEVELOPMENT</i> 25.16 (2011): 1663?1667. Print.","pubMedId":"21852531","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21852531","publicationId":111910,"title":"miR than meets the eye","type":"article","webOfScienceId":"000294003800001","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000294003800001"},{"abstract":"The retinoblastoma tumor suppressor (RB) is a central cell cycle regulator and tumor suppressor. RB cellular functions are known to be regulated by a diversity of post-translational modifications such as phosphorylation and acetylation, raising the possibility that RB may also be methylated in cells. Here we demonstrate that RB can be methylated by SMYD2 at lysine 860, a highly conserved and novel site of modification. This methylation event occurs in vitro and in cells, and it is regulated during cell cycle progression, cellular differentiation, and in response to DNA damage. Furthermore, we show that RB monomethylation at lysine 860 provides a direct binding site for the methyl-binding domain of the transcriptional repressor L3MBTL1. These results support the idea that a code of post-translational modifications exists for RB and helps guide its functions in mammalian cells.","apaCitation":"Saddic, L. A., West, L. E., Aslanian, A., Yates, J. R., Rubin, S. M., Gozani, O., &amp; Sage, J. (2010). Methylation of the Retinoblastoma Tumor Suppressor by SMYD2. <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i>, <i>285</i>(48), 37733?37740.","capCitation":"<span class=\"title\">\n  <span>Methylation of the Retinoblastoma Tumor Suppressor by SMYD2</span>\n  <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i>\n</span>\n<span class=\"authors\">Saddic, L. A., West, L. E., Aslanian, A., Yates, J. R., Rubin, S. M., Gozani, O., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">285 (48)</span><span class=\"pages\">: 37733-37740</span>\n</span>","chicagoCitation":"Saddic, Louis A., Lisandra E. West, Aaron Aslanian, John R. Yates, Seth M. Rubin, et al. 2010. ?Methylation of the Retinoblastoma Tumor Suppressor by SMYD2.? <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i> 285 (48). AMER SOC BIOCHEMISTRY MOLECULAR BIOLOGY INC: 37733?40.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma tumor suppressor (RB) is a central cell cycle regulator and tumor suppressor. RB cellular functions are known to be regulated by a diversity of post-translational modifications such as phosphorylation and acetylation, raising the possibility that RB may also be methylated in cells. Here we demonstrate that RB can be methylated by SMYD2 at lysine 860, a highly conserved and novel site of modification. This methylation event occurs in vitro and in cells, and it is regulated during cell cycle progression, cellular differentiation, and in response to DNA damage. Furthermore, we show that RB monomethylation at lysine 860 provides a direct binding site for the methyl-binding domain of the transcriptional repressor L3MBTL1. These results support the idea that a code of post-translational modifications exists for RB and helps guide its functions in mammalian cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1074/jbc.M110.137612\">DOI 10.1074/jbc.M110.137612</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000284424000065\">Web of Science ID 000284424000065</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20870719\">PubMedID 20870719</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma tumor suppressor (RB) is a central cell cycle regulator and tumor suppressor. RB cellular functions are known to be regulated by a diversity of post-translational modifications such as phosphorylation and acetylation, raising the possibility that RB may also be methylated in cells. Here we demonstrate that RB can be methylated by SMYD2 at lysine 860, a highly conserved and novel site of modification. This methylation event occurs in vitro and in cells, and it is regulated during cell cycle progression, cellular differentiation, and in response to DNA damage. Furthermore, we show that RB monomethylation at lysine 860 provides a direct binding site for the methyl-binding domain of the transcriptional repressor L3MBTL1. These results support the idea that a code of post-translational modifications exists for RB and helps guide its functions in mammalian cells.\nDOI: 10.1074/jbc.M110.137612, https://dx.doi.org/10.1074/jbc.M110.137612\nWeb of Science ID: 000284424000065, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000284424000065\nPubMedID: 20870719, https://www.ncbi.nlm.nih.gov/pubmed/20870719"},"doiId":"10.1074/jbc.M110.137612","doiUrl":"https://dx.doi.org/10.1074/jbc.M110.137612","featured":true,"firstPublished":{"html":"Nov 26, 2010 00:00:00 AM","text":"Nov 26, 2010 00:00:00 AM","value":"2010-11-26T00:00:00.000-08:00"},"mlaCitation":"Saddic, Louis A. et al. ?Methylation of the Retinoblastoma Tumor Suppressor by SMYD2.? <i>JOURNAL OF BIOLOGICAL CHEMISTRY</i> 285.48 (2010): 37733?37740. Print.","pubMedId":"20870719","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20870719","publicationId":96868,"title":"Methylation of the Retinoblastoma Tumor Suppressor by SMYD2","type":"article","webOfScienceId":"000284424000065","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000284424000065"},{"abstract":"The ability of progenitor cells to exit the cell cycle is essential for proper embryonic development and homeostasis, but the mechanisms governing cell cycle exit are still not fully understood. Here, we tested the requirement for the retinoblastoma (Rb) protein and its family members p107 and p130 in G0/G1 arrest and differentiation in mammalian cells. We found that Rb family triple knockout (TKO) mouse embryos survive until days 9-11 of gestation. Strikingly, some TKO cells, including in epithelial and neural lineages, are able to exit the cell cycle in G0/G1 and differentiate in teratomas and in culture. This ability of TKO cells to arrest in G0/G1 is associated with the repression of key E2F target genes. Thus, G1 arrest is not always dependent on Rb family members, which illustrates the robustness of cell cycle regulatory networks during differentiation and allows for the identification of candidate pathways to inhibit the expansion of cancer cells with mutations in the Rb pathway.","apaCitation":"Wirt, S. E., Adler, A. S., Gebala, V., Weimann, J. M., Schaffer, B. E., Saddic, L. A., ? Sage, J. (2010). G1 arrest and differentiation can occur independently of Rb family function. <i>JOURNAL OF CELL BIOLOGY</i>, <i>191</i>(4), 809?825.","capCitation":"<span class=\"title\">\n  <span>G1 arrest and differentiation can occur independently of Rb family function</span>\n  <i>JOURNAL OF CELL BIOLOGY</i>\n</span>\n<span class=\"authors\">Wirt, S. E., Adler, A. S., Gebala, V., Weimann, J. M., Schaffer, B. E., Saddic, L. A., Viatour, P., Vogel, H., Chang, H. Y., Meissner, A., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">191 (4)</span><span class=\"pages\">: 809-825</span>\n</span>","chicagoCitation":"Wirt, Stacey E., Adam S. Adler, Veronique Gebala, James M. Weimann, Bethany E. Schaffer, et al. 2010. ?G1 Arrest and Differentiation Can Occur Independently of Rb Family Function.? <i>JOURNAL OF CELL BIOLOGY</i> 191 (4). ROCKEFELLER UNIV PRESS: 809?25.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The ability of progenitor cells to exit the cell cycle is essential for proper embryonic development and homeostasis, but the mechanisms governing cell cycle exit are still not fully understood. Here, we tested the requirement for the retinoblastoma (Rb) protein and its family members p107 and p130 in G0/G1 arrest and differentiation in mammalian cells. We found that Rb family triple knockout (TKO) mouse embryos survive until days 9-11 of gestation. Strikingly, some TKO cells, including in epithelial and neural lineages, are able to exit the cell cycle in G0/G1 and differentiate in teratomas and in culture. This ability of TKO cells to arrest in G0/G1 is associated with the repression of key E2F target genes. Thus, G1 arrest is not always dependent on Rb family members, which illustrates the robustness of cell cycle regulatory networks during differentiation and allows for the identification of candidate pathways to inhibit the expansion of cancer cells with mutations in the Rb pathway.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1083/jcb.201003048\">DOI 10.1083/jcb.201003048</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000284737200014\">Web of Science ID 000284737200014</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/21059851\">PubMedID 21059851</a></span>\n  </p>","text":"Abstract:\nThe ability of progenitor cells to exit the cell cycle is essential for proper embryonic development and homeostasis, but the mechanisms governing cell cycle exit are still not fully understood. Here, we tested the requirement for the retinoblastoma (Rb) protein and its family members p107 and p130 in G0/G1 arrest and differentiation in mammalian cells. We found that Rb family triple knockout (TKO) mouse embryos survive until days 9-11 of gestation. Strikingly, some TKO cells, including in epithelial and neural lineages, are able to exit the cell cycle in G0/G1 and differentiate in teratomas and in culture. This ability of TKO cells to arrest in G0/G1 is associated with the repression of key E2F target genes. Thus, G1 arrest is not always dependent on Rb family members, which illustrates the robustness of cell cycle regulatory networks during differentiation and allows for the identification of candidate pathways to inhibit the expansion of cancer cells with mutations in the Rb pathway.\nDOI: 10.1083/jcb.201003048, https://dx.doi.org/10.1083/jcb.201003048\nWeb of Science ID: 000284737200014, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000284737200014\nPubMedID: 21059851, https://www.ncbi.nlm.nih.gov/pubmed/21059851"},"doiId":"10.1083/jcb.201003048","doiUrl":"https://dx.doi.org/10.1083/jcb.201003048","featured":true,"firstPublished":{"html":"Nov 15, 2010 00:00:00 AM","text":"Nov 15, 2010 00:00:00 AM","value":"2010-11-15T00:00:00.000-08:00"},"mlaCitation":"Wirt, Stacey E. et al. ?G1 Arrest and Differentiation Can Occur Independently of Rb Family Function.? <i>JOURNAL OF CELL BIOLOGY</i> 191.4 (2010): 809?825. Print.","pubMedId":"21059851","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/21059851","publicationId":98964,"title":"G1 arrest and differentiation can occur independently of Rb family function","type":"article","webOfScienceId":"000284737200014","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000284737200014"},{"abstract":"The retinoblastoma tumor suppressor (Rb) is a potent and ubiquitously expressed cell cycle regulator, but patients with a germline Rb mutation develop a very specific tumor spectrum. This surprising observation raises the possibility that mechanisms that compensate for loss of Rb function are present or activated in many cell types. In particular, p107, a protein related to Rb, has been shown to functionally overlap for loss of Rb in several cellular contexts. To investigate the mechanisms underlying this functional redundancy between Rb and p107 in vivo, we used gene targeting in embryonic stem cells to engineer point mutations in two consensus E2F binding sites in the endogenous p107 promoter. Analysis of normal and mutant cells by gene expression and chromatin immunoprecipitation assays showed that members of the Rb and E2F families directly bound these two sites. Furthermore, we found that these two E2F sites controlled both the repression of p107 in quiescent cells and also its activation in cycling cells, as well as in Rb mutant cells. Cell cycle assays further indicated that activation of p107 transcription during S phase through the two E2F binding sites was critical for controlled cell cycle progression, uncovering a specific role for p107 to slow proliferation in mammalian cells. Direct transcriptional repression of p107 by Rb and E2F family members provides a molecular mechanism for a critical negative feedback loop during cell cycle progression and tumorigenesis. These experiments also suggest novel therapeutic strategies to increase the p107 levels in tumor cells.","apaCitation":"Burkhart, D. L., Wirt, S. E., Zmoos, A.-F., Kareta, M. S., &amp; Sage, J. (2010). Tandem E2F Binding Sites in the Promoter of the p107 Cell Cycle Regulator Control p107 Expression and Its Cellular Functions. <i>PLOS GENETICS</i>, <i>6</i>(6).","capCitation":"<span class=\"title\">\n  <span>Tandem E2F Binding Sites in the Promoter of the p107 Cell Cycle Regulator Control p107 Expression and Its Cellular Functions</span>\n  <i>PLOS GENETICS</i>\n</span>\n<span class=\"authors\">Burkhart, D. L., Wirt, S. E., Zmoos, A., Kareta, M. S., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">6 (6)</span>\n</span>","chicagoCitation":"Burkhart, Deborah L., Stacey E. Wirt, Anne-Flore Zmoos, Michael S. Kareta, and Julien Sage. 2010. ?Tandem E2F Binding Sites in the Promoter of the p107 Cell Cycle Regulator Control p107 Expression and Its Cellular Functions.? <i>PLOS GENETICS</i> 6 (6). PUBLIC LIBRARY SCIENCE.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma tumor suppressor (Rb) is a potent and ubiquitously expressed cell cycle regulator, but patients with a germline Rb mutation develop a very specific tumor spectrum. This surprising observation raises the possibility that mechanisms that compensate for loss of Rb function are present or activated in many cell types. In particular, p107, a protein related to Rb, has been shown to functionally overlap for loss of Rb in several cellular contexts. To investigate the mechanisms underlying this functional redundancy between Rb and p107 in vivo, we used gene targeting in embryonic stem cells to engineer point mutations in two consensus E2F binding sites in the endogenous p107 promoter. Analysis of normal and mutant cells by gene expression and chromatin immunoprecipitation assays showed that members of the Rb and E2F families directly bound these two sites. Furthermore, we found that these two E2F sites controlled both the repression of p107 in quiescent cells and also its activation in cycling cells, as well as in Rb mutant cells. Cell cycle assays further indicated that activation of p107 transcription during S phase through the two E2F binding sites was critical for controlled cell cycle progression, uncovering a specific role for p107 to slow proliferation in mammalian cells. Direct transcriptional repression of p107 by Rb and E2F family members provides a molecular mechanism for a critical negative feedback loop during cell cycle progression and tumorigenesis. These experiments also suggest novel therapeutic strategies to increase the p107 levels in tumor cells.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1371/journal.pgen.1001003\">DOI 10.1371/journal.pgen.1001003</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000279805200032\">Web of Science ID 000279805200032</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20585628\">PubMedID 20585628</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma tumor suppressor (Rb) is a potent and ubiquitously expressed cell cycle regulator, but patients with a germline Rb mutation develop a very specific tumor spectrum. This surprising observation raises the possibility that mechanisms that compensate for loss of Rb function are present or activated in many cell types. In particular, p107, a protein related to Rb, has been shown to functionally overlap for loss of Rb in several cellular contexts. To investigate the mechanisms underlying this functional redundancy between Rb and p107 in vivo, we used gene targeting in embryonic stem cells to engineer point mutations in two consensus E2F binding sites in the endogenous p107 promoter. Analysis of normal and mutant cells by gene expression and chromatin immunoprecipitation assays showed that members of the Rb and E2F families directly bound these two sites. Furthermore, we found that these two E2F sites controlled both the repression of p107 in quiescent cells and also its activation in cycling cells, as well as in Rb mutant cells. Cell cycle assays further indicated that activation of p107 transcription during S phase through the two E2F binding sites was critical for controlled cell cycle progression, uncovering a specific role for p107 to slow proliferation in mammalian cells. Direct transcriptional repression of p107 by Rb and E2F family members provides a molecular mechanism for a critical negative feedback loop during cell cycle progression and tumorigenesis. These experiments also suggest novel therapeutic strategies to increase the p107 levels in tumor cells.\nDOI: 10.1371/journal.pgen.1001003, https://dx.doi.org/10.1371/journal.pgen.1001003\nWeb of Science ID: 000279805200032, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000279805200032\nPubMedID: 20585628, https://www.ncbi.nlm.nih.gov/pubmed/20585628"},"doiId":"10.1371/journal.pgen.1001003","doiUrl":"https://dx.doi.org/10.1371/journal.pgen.1001003","featured":true,"firstPublished":{"html":"Jun 1, 2010 00:00:00 AM","text":"Jun 1, 2010 00:00:00 AM","value":"2010-06-01T00:00:00.000-07:00"},"mlaCitation":"Burkhart, Deborah L. et al. ?Tandem E2F Binding Sites in the Promoter of the p107 Cell Cycle Regulator Control p107 Expression and Its Cellular Functions.? <i>PLOS GENETICS</i> 6.6 (2010): n. pag. Print.","pubMedId":"20585628","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20585628","publicationId":93888,"title":"Tandem E2F Binding Sites in the Promoter of the p107 Cell Cycle Regulator Control p107 Expression and Its Cellular Functions","type":"article","webOfScienceId":"000279805200032","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000279805200032"},{"abstract":"Small-cell lung carcinoma (SCLC) is a neuroendocrine subtype of lung cancer. Although SCLC patients often initially respond to therapy, tumors nearly always recur, resulting in a 5-year survival rate of less than 10%. A mouse model has been developed based on the fact that the RB and p53 tumor suppressor genes are mutated in more than 90% of human SCLCs. Emerging evidence in patients and mouse models suggests that p130, a gene related to RB, may act as a tumor suppressor in SCLC cells. To test this idea, we used conditional mutant mice to delete p130 in combination with Rb and p53 in adult lung epithelial cells. We found that loss of p130 resulted in increased proliferation and significant acceleration of SCLC development in this triple-knockout mouse model. The histopathologic features of the triple-mutant mouse tumors closely resembled that of human SCLC. Genome-wide expression profiling experiments further showed that Rb/p53/p130-mutant mouse tumors were similar to human SCLC. These findings indicate that p130 plays a key tumor suppressor role in SCLC. Rb/p53/p130-mutant mice provide a novel preclinical mouse model to identify novel therapeutic targets against SCLC.","apaCitation":"Schaffer, B. E., Park, K.-S., Yiu, G., Conklin, J. F., Lin, C. W., Burkhart, D. L., ? Sage, J. (2010). Loss of p130 Accelerates Tumor Development in a Mouse Model for Human Small-Cell Lung Carcinoma. <i>CANCER RESEARCH</i>, <i>70</i>(10), 3877?3883.","capCitation":"<span class=\"title\">\n  <span>Loss of p130 Accelerates Tumor Development in a Mouse Model for Human Small-Cell Lung Carcinoma</span>\n  <i>CANCER RESEARCH</i>\n</span>\n<span class=\"authors\">Schaffer, B. E., Park, K., Yiu, G., Conklin, J. F., Lin, C., Burkhart, D. L., Karnezis, A. N., Sweet-Cordero, E. A., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2010</span>; <span class=\"volume\">70 (10)</span><span class=\"pages\">: 3877-3883</span>\n</span>","chicagoCitation":"Schaffer, Bethany E., Kwon-Sik Park, Gloria Yiu, Jamie F. Conklin, ChenWei Lin, et al. 2010. ?Loss of p130 Accelerates Tumor Development in a Mouse Model for Human Small-Cell Lung Carcinoma.? <i>CANCER RESEARCH</i> 70 (10). AMER ASSOC CANCER RESEARCH: 3877?83.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Small-cell lung carcinoma (SCLC) is a neuroendocrine subtype of lung cancer. Although SCLC patients often initially respond to therapy, tumors nearly always recur, resulting in a 5-year survival rate of less than 10%. A mouse model has been developed based on the fact that the RB and p53 tumor suppressor genes are mutated in more than 90% of human SCLCs. Emerging evidence in patients and mouse models suggests that p130, a gene related to RB, may act as a tumor suppressor in SCLC cells. To test this idea, we used conditional mutant mice to delete p130 in combination with Rb and p53 in adult lung epithelial cells. We found that loss of p130 resulted in increased proliferation and significant acceleration of SCLC development in this triple-knockout mouse model. The histopathologic features of the triple-mutant mouse tumors closely resembled that of human SCLC. Genome-wide expression profiling experiments further showed that Rb/p53/p130-mutant mouse tumors were similar to human SCLC. These findings indicate that p130 plays a key tumor suppressor role in SCLC. Rb/p53/p130-mutant mice provide a novel preclinical mouse model to identify novel therapeutic targets against SCLC.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1158/0008-5472.CAN-09-4228\">DOI 10.1158/0008-5472.CAN-09-4228</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000278486300004\">Web of Science ID 000278486300004</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/20406986\">PubMedID 20406986</a></span>\n  </p>","text":"Abstract:\nSmall-cell lung carcinoma (SCLC) is a neuroendocrine subtype of lung cancer. Although SCLC patients often initially respond to therapy, tumors nearly always recur, resulting in a 5-year survival rate of less than 10%. A mouse model has been developed based on the fact that the RB and p53 tumor suppressor genes are mutated in more than 90% of human SCLCs. Emerging evidence in patients and mouse models suggests that p130, a gene related to RB, may act as a tumor suppressor in SCLC cells. To test this idea, we used conditional mutant mice to delete p130 in combination with Rb and p53 in adult lung epithelial cells. We found that loss of p130 resulted in increased proliferation and significant acceleration of SCLC development in this triple-knockout mouse model. The histopathologic features of the triple-mutant mouse tumors closely resembled that of human SCLC. Genome-wide expression profiling experiments further showed that Rb/p53/p130-mutant mouse tumors were similar to human SCLC. These findings indicate that p130 plays a key tumor suppressor role in SCLC. Rb/p53/p130-mutant mice provide a novel preclinical mouse model to identify novel therapeutic targets against SCLC.\nDOI: 10.1158/0008-5472.CAN-09-4228, https://dx.doi.org/10.1158/0008-5472.CAN-09-4228\nWeb of Science ID: 000278486300004, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000278486300004\nPubMedID: 20406986, https://www.ncbi.nlm.nih.gov/pubmed/20406986"},"doiId":"10.1158/0008-5472.CAN-09-4228","doiUrl":"https://dx.doi.org/10.1158/0008-5472.CAN-09-4228","featured":true,"firstPublished":{"html":"May 15, 2010 00:00:00 AM","text":"May 15, 2010 00:00:00 AM","value":"2010-05-15T00:00:00.000-07:00"},"mlaCitation":"Schaffer, Bethany E. et al. ?Loss of p130 Accelerates Tumor Development in a Mouse Model for Human Small-Cell Lung Carcinoma.? <i>CANCER RESEARCH</i> 70.10 (2010): 3877?3883. Print.","pubMedId":"20406986","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/20406986","publicationId":91948,"title":"Loss of p130 Accelerates Tumor Development in a Mouse Model for Human Small-Cell Lung Carcinoma","type":"article","webOfScienceId":"000278486300004","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000278486300004"},{"abstract":"Mutations of the retinoblastoma tumor suppressor gene RB are frequently observed in human cancers, but rarely in non-small cell lung carcinomas (NSCLCs). Emerging evidence also suggests that the RB-related gene p130 is inactivated in a subset of human NSCLCs. To directly test the specific tumor suppressor roles of RB and p130 in NSCLC, we crossed Rb and p130 conditional mutant mice to mice carrying a conditional oncogenic K-Ras allele. In this model, controlled oncogenic K-Ras activation leads to the development of adenocarcinoma, a major subtype of NSCLC. We found that loss of p130 accelerated the death of mice, providing direct evidence in vivo that p130 is a tumor suppressor gene, albeit a weak one in this context. Loss of Rb increased the efficiency of lung cancer initiation and resulted in the development of high-grade adenocarcinomas and rapid death. Thus, despite the low frequency of RB mutations in human NSCLCs and reports that K-Ras activation and loss of RB function are rarely found in the same human tumors, loss of Rb clearly cooperates with activation of oncogenic K-Ras in lung adenocarcinoma development in mice.","apaCitation":"Ho, V. M., Schaffer, B. E., Karnezis, A. N., Park, K. S., &amp; Sage, J. (2009). The retinoblastoma gene Rb and its family member p130 suppress lung adenocarcinoma induced by oncogenic K-Ras. <i>ONCOGENE</i>, <i>28</i>(10), 1393?1399.","capCitation":"<span class=\"title\">\n  <span>The retinoblastoma gene Rb and its family member p130 suppress lung adenocarcinoma induced by oncogenic K-Ras</span>\n  <i>ONCOGENE</i>\n</span>\n<span class=\"authors\">Ho, V. M., Schaffer, B. E., Karnezis, A. N., Park, K. S., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2009</span>; <span class=\"volume\">28 (10)</span><span class=\"pages\">: 1393-1399</span>\n</span>","chicagoCitation":"Ho, V. M., B. E. Schaffer, A. N. Karnezis, K. S. Park, and J. Sage. 2009. ?The Retinoblastoma Gene Rb and Its Family Member p130 Suppress Lung Adenocarcinoma Induced by Oncogenic K-Ras.? <i>ONCOGENE</i> 28 (10). NATURE PUBLISHING GROUP: 1393?99.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Mutations of the retinoblastoma tumor suppressor gene RB are frequently observed in human cancers, but rarely in non-small cell lung carcinomas (NSCLCs). Emerging evidence also suggests that the RB-related gene p130 is inactivated in a subset of human NSCLCs. To directly test the specific tumor suppressor roles of RB and p130 in NSCLC, we crossed Rb and p130 conditional mutant mice to mice carrying a conditional oncogenic K-Ras allele. In this model, controlled oncogenic K-Ras activation leads to the development of adenocarcinoma, a major subtype of NSCLC. We found that loss of p130 accelerated the death of mice, providing direct evidence in vivo that p130 is a tumor suppressor gene, albeit a weak one in this context. Loss of Rb increased the efficiency of lung cancer initiation and resulted in the development of high-grade adenocarcinomas and rapid death. Thus, despite the low frequency of RB mutations in human NSCLCs and reports that K-Ras activation and loss of RB function are rarely found in the same human tumors, loss of Rb clearly cooperates with activation of oncogenic K-Ras in lung adenocarcinoma development in mice.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/onc.2008.491\">DOI 10.1038/onc.2008.491</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000264116000012\">Web of Science ID 000264116000012</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/19151761\">PubMedID 19151761</a></span>\n  </p>","text":"Abstract:\nMutations of the retinoblastoma tumor suppressor gene RB are frequently observed in human cancers, but rarely in non-small cell lung carcinomas (NSCLCs). Emerging evidence also suggests that the RB-related gene p130 is inactivated in a subset of human NSCLCs. To directly test the specific tumor suppressor roles of RB and p130 in NSCLC, we crossed Rb and p130 conditional mutant mice to mice carrying a conditional oncogenic K-Ras allele. In this model, controlled oncogenic K-Ras activation leads to the development of adenocarcinoma, a major subtype of NSCLC. We found that loss of p130 accelerated the death of mice, providing direct evidence in vivo that p130 is a tumor suppressor gene, albeit a weak one in this context. Loss of Rb increased the efficiency of lung cancer initiation and resulted in the development of high-grade adenocarcinomas and rapid death. Thus, despite the low frequency of RB mutations in human NSCLCs and reports that K-Ras activation and loss of RB function are rarely found in the same human tumors, loss of Rb clearly cooperates with activation of oncogenic K-Ras in lung adenocarcinoma development in mice.\nDOI: 10.1038/onc.2008.491, https://dx.doi.org/10.1038/onc.2008.491\nWeb of Science ID: 000264116000012, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000264116000012\nPubMedID: 19151761, https://www.ncbi.nlm.nih.gov/pubmed/19151761"},"doiId":"10.1038/onc.2008.491","doiUrl":"https://dx.doi.org/10.1038/onc.2008.491","featured":true,"firstPublished":{"html":"Mar 1, 2009 00:00:00 AM","text":"Mar 1, 2009 00:00:00 AM","value":"2009-03-01T00:00:00.000-08:00"},"mlaCitation":"Ho, V. M. et al. ?The Retinoblastoma Gene Rb and Its Family Member p130 Suppress Lung Adenocarcinoma Induced by Oncogenic K-Ras.? <i>ONCOGENE</i> 28.10 (2009): 1393?1399. Print.","pubMedId":"19151761","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/19151761","publicationId":77918,"title":"The retinoblastoma gene Rb and its family member p130 suppress lung adenocarcinoma induced by oncogenic K-Ras","type":"article","webOfScienceId":"000264116000012","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000264116000012"},{"abstract":"Individual members of the retinoblastoma (Rb) tumor suppressor gene family serve critical roles in the control of cellular proliferation and differentiation, but the extent of their contributions is masked by redundant and compensatory mechanisms. Here we employed a conditional knockout strategy to simultaneously inactivate all three members, Rb, p107, and p130, in adult hematopoietic stem cells (HSCs). Rb family triple knockout (TKO) mice develop a cell-intrinsic myeloproliferation that originates from hyperproliferative early hematopoietic progenitors and is accompanied by increased apoptosis in lymphoid progenitor populations. Loss of quiescence in the TKO HSC pool is associated with an expansion of these mutant stem cells but also with an enhanced mobilization and an impaired reconstitution potential upon transplantation. The presence of a single p107 allele is sufficient to largely rescue these defects. Thus, Rb family members collectively maintain HSC quiescence and the balance between lymphoid and myeloid cell fates in the hematopoietic system.","apaCitation":"Viatour, P., Somervaille, T. C., Venkatasubrahmanyam, S., Kogan, S., McLaughlin, M. E., Weissman, I. L., ? Sage, J. (2008). Hematopoietic Stem Cell Quiescence Is Maintained by Compound Contributions of the Retinoblastoma Gene Family. <i>CELL STEM CELL</i>, <i>3</i>(4), 416?428.","capCitation":"<span class=\"title\">\n  <span>Hematopoietic Stem Cell Quiescence Is Maintained by Compound Contributions of the Retinoblastoma Gene Family</span>\n  <i>CELL STEM CELL</i>\n</span>\n<span class=\"authors\">Viatour, P., Somervaille, T. C., Venkatasubrahmanyam, S., Kogan, S., McLaughlin, M. E., Weissman, I. L., Butte, A. J., Passegue, E., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2008</span>; <span class=\"volume\">3 (4)</span><span class=\"pages\">: 416-428</span>\n</span>","chicagoCitation":"Viatour, Patrick, Tim C. Somervaille, Shivkumar Venkatasubrahmanyam, Scott Kogan, Margaret E. McLaughlin, et al. 2008. ?Hematopoietic Stem Cell Quiescence Is Maintained by Compound Contributions of the Retinoblastoma Gene Family.? <i>CELL STEM CELL</i> 3 (4). CELL PRESS: 416?28.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Individual members of the retinoblastoma (Rb) tumor suppressor gene family serve critical roles in the control of cellular proliferation and differentiation, but the extent of their contributions is masked by redundant and compensatory mechanisms. Here we employed a conditional knockout strategy to simultaneously inactivate all three members, Rb, p107, and p130, in adult hematopoietic stem cells (HSCs). Rb family triple knockout (TKO) mice develop a cell-intrinsic myeloproliferation that originates from hyperproliferative early hematopoietic progenitors and is accompanied by increased apoptosis in lymphoid progenitor populations. Loss of quiescence in the TKO HSC pool is associated with an expansion of these mutant stem cells but also with an enhanced mobilization and an impaired reconstitution potential upon transplantation. The presence of a single p107 allele is sufficient to largely rescue these defects. Thus, Rb family members collectively maintain HSC quiescence and the balance between lymphoid and myeloid cell fates in the hematopoietic system.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1016/j.stem.2008.07.009\">DOI 10.1016/j.stem.2008.07.009</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000260149800012\">Web of Science ID 000260149800012</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/18940733\">PubMedID 18940733</a></span>\n  </p>","text":"Abstract:\nIndividual members of the retinoblastoma (Rb) tumor suppressor gene family serve critical roles in the control of cellular proliferation and differentiation, but the extent of their contributions is masked by redundant and compensatory mechanisms. Here we employed a conditional knockout strategy to simultaneously inactivate all three members, Rb, p107, and p130, in adult hematopoietic stem cells (HSCs). Rb family triple knockout (TKO) mice develop a cell-intrinsic myeloproliferation that originates from hyperproliferative early hematopoietic progenitors and is accompanied by increased apoptosis in lymphoid progenitor populations. Loss of quiescence in the TKO HSC pool is associated with an expansion of these mutant stem cells but also with an enhanced mobilization and an impaired reconstitution potential upon transplantation. The presence of a single p107 allele is sufficient to largely rescue these defects. Thus, Rb family members collectively maintain HSC quiescence and the balance between lymphoid and myeloid cell fates in the hematopoietic system.\nDOI: 10.1016/j.stem.2008.07.009, https://dx.doi.org/10.1016/j.stem.2008.07.009\nWeb of Science ID: 000260149800012, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000260149800012\nPubMedID: 18940733, https://www.ncbi.nlm.nih.gov/pubmed/18940733"},"doiId":"10.1016/j.stem.2008.07.009","doiUrl":"https://dx.doi.org/10.1016/j.stem.2008.07.009","featured":true,"firstPublished":{"html":"Oct 9, 2008 00:00:00 AM","text":"Oct 9, 2008 00:00:00 AM","value":"2008-10-09T00:00:00.000-07:00"},"mlaCitation":"Viatour, Patrick et al. ?Hematopoietic Stem Cell Quiescence Is Maintained by Compound Contributions of the Retinoblastoma Gene Family.? <i>CELL STEM CELL</i> 3.4 (2008): 416?428. Print.","pubMedId":"18940733","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/18940733","publicationId":75095,"title":"Hematopoietic Stem Cell Quiescence Is Maintained by Compound Contributions of the Retinoblastoma Gene Family","type":"article","webOfScienceId":"000260149800012","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000260149800012"},{"abstract":"The retinoblastoma (RB) tumour suppressor gene is functionally inactivated in a broad range of paediatric and adult cancers, and a plethora of cellular functions and partners have been identified for the RB protein. Data from human tumours and studies from mouse models indicate that loss of RB function contributes to both cancer initiation and progression. However, we still do not know the identity of the cell types in which RB normally prevents cancer initiation in vivo, and the specific functions of RB that suppress distinct aspects of the tumorigenic process are poorly understood.","apaCitation":"Burkhart, D. L., &amp; Sage, J. (2008). Cellular mechanisms of tumour suppression by the retinoblastoma gene. <i>NATURE REVIEWS CANCER</i>, <i>8</i>(9), 671?682.","capCitation":"<span class=\"title\">\n  <span>Cellular mechanisms of tumour suppression by the retinoblastoma gene</span>\n  <i>NATURE REVIEWS CANCER</i>\n</span>\n<span class=\"authors\">Burkhart, D. L., Sage, J.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2008</span>; <span class=\"volume\">8 (9)</span><span class=\"pages\">: 671-682</span>\n</span>","chicagoCitation":"Burkhart, Deborah L., and Julien Sage. 2008. ?Cellular Mechanisms of Tumour Suppression by the Retinoblastoma Gene.? <i>NATURE REVIEWS CANCER</i> 8 (9). NATURE PUBLISHING GROUP: 671?82.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma (RB) tumour suppressor gene is functionally inactivated in a broad range of paediatric and adult cancers, and a plethora of cellular functions and partners have been identified for the RB protein. Data from human tumours and studies from mouse models indicate that loss of RB function contributes to both cancer initiation and progression. However, we still do not know the identity of the cell types in which RB normally prevents cancer initiation in vivo, and the specific functions of RB that suppress distinct aspects of the tumorigenic process are poorly understood.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nrc2399\">DOI 10.1038/nrc2399</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000258662900012\">Web of Science ID 000258662900012</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/18650841\">PubMedID 18650841</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma (RB) tumour suppressor gene is functionally inactivated in a broad range of paediatric and adult cancers, and a plethora of cellular functions and partners have been identified for the RB protein. Data from human tumours and studies from mouse models indicate that loss of RB function contributes to both cancer initiation and progression. However, we still do not know the identity of the cell types in which RB normally prevents cancer initiation in vivo, and the specific functions of RB that suppress distinct aspects of the tumorigenic process are poorly understood.\nDOI: 10.1038/nrc2399, https://dx.doi.org/10.1038/nrc2399\nWeb of Science ID: 000258662900012, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000258662900012\nPubMedID: 18650841, https://www.ncbi.nlm.nih.gov/pubmed/18650841"},"doiId":"10.1038/nrc2399","doiUrl":"https://dx.doi.org/10.1038/nrc2399","featured":true,"firstPublished":{"html":"Sep 1, 2008 00:00:00 AM","text":"Sep 1, 2008 00:00:00 AM","value":"2008-09-01T00:00:00.000-07:00"},"mlaCitation":"Burkhart, Deborah L., and Julien Sage. ?Cellular Mechanisms of Tumour Suppression by the Retinoblastoma Gene.? <i>NATURE REVIEWS CANCER</i> 8.9 (2008): 671?682. Print.","pubMedId":"18650841","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/18650841","publicationId":72213,"title":"Cellular mechanisms of tumour suppression by the retinoblastoma gene","type":"article","webOfScienceId":"000258662900012","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000258662900012"},{"abstract":"Cell cycle checkpoints induced by DNA damage play an integral role in preservation of genomic stability by allowing cells to limit the propagation of deleterious mutations. The retinoblastoma tumor suppressor (RB) is crucial for the maintenance of the DNA damage checkpoint function because it elicits cell cycle arrest in response to a variety of genotoxic stresses. Although sporadic loss of RB is characteristic of most cancers and results in the bypass of the DNA damage checkpoint, the consequence of RB loss upon chemotherapeutic responsiveness has been largely uninvestigated. Here, we employed a conditional knockout approach to ablate RB in adult fibroblasts. This system enabled us to examine the DNA damage response of adult cells following acute RB deletion. Using this system, we demonstrated that loss of RB disrupted the DNA damage checkpoint elicited by either cisplatin or camptothecin exposure. Strikingly, this bypass was not associated with enhanced repair, but rather the accumulation of phosphorylated H2AX (gammaH2AX) foci, which indicate DNA double-strand breaks. The formation of gammaH2AX foci was due to ongoing replication following chemotherapeutic treatment in the RB-deficient cells. Additionally, peak gammaH2AX accumulation occurred in S-phase cells undergoing DNA replication in the presence of damage, and these gammaH2AX foci co-localized with replication foci. These results demonstrate that acute RB loss abrogates DNA damage-induced cell cycle arrest to induce gammaH2AX foci formation. Thus, secondary genetic lesions induced by RB loss have implications for the chemotherapeutic response and the development of genetic instability.","apaCitation":"Bosco, E. E., Mayhew, C. N., Hennigan, R. F., Sage, J., Jacks, T., &amp; Knudsen, E. S. (2004). RB signaling prevents replication-dependent DNA double-strand breaks following genotoxic insult. <i>NUCLEIC ACIDS RESEARCH</i>, <i>32</i>(1), 25?34.","capCitation":"<span class=\"title\">\n  <span>RB signaling prevents replication-dependent DNA double-strand breaks following genotoxic insult</span>\n  <i>NUCLEIC ACIDS RESEARCH</i>\n</span>\n<span class=\"authors\">Bosco, E. E., Mayhew, C. N., Hennigan, R. F., Sage, J., Jacks, T., Knudsen, E. S.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2004</span>; <span class=\"volume\">32 (1)</span><span class=\"pages\">: 25-34</span>\n</span>","chicagoCitation":"Bosco, E. E., C. N. Mayhew, R. F. Hennigan, J. Sage, T. Jacks, et al. 2004. ?RB Signaling Prevents Replication-Dependent DNA Double-Strand Breaks Following Genotoxic Insult.? <i>NUCLEIC ACIDS RESEARCH</i> 32 (1). OXFORD UNIV PRESS: 25?34.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Cell cycle checkpoints induced by DNA damage play an integral role in preservation of genomic stability by allowing cells to limit the propagation of deleterious mutations. The retinoblastoma tumor suppressor (RB) is crucial for the maintenance of the DNA damage checkpoint function because it elicits cell cycle arrest in response to a variety of genotoxic stresses. Although sporadic loss of RB is characteristic of most cancers and results in the bypass of the DNA damage checkpoint, the consequence of RB loss upon chemotherapeutic responsiveness has been largely uninvestigated. Here, we employed a conditional knockout approach to ablate RB in adult fibroblasts. This system enabled us to examine the DNA damage response of adult cells following acute RB deletion. Using this system, we demonstrated that loss of RB disrupted the DNA damage checkpoint elicited by either cisplatin or camptothecin exposure. Strikingly, this bypass was not associated with enhanced repair, but rather the accumulation of phosphorylated H2AX (gammaH2AX) foci, which indicate DNA double-strand breaks. The formation of gammaH2AX foci was due to ongoing replication following chemotherapeutic treatment in the RB-deficient cells. Additionally, peak gammaH2AX accumulation occurred in S-phase cells undergoing DNA replication in the presence of damage, and these gammaH2AX foci co-localized with replication foci. These results demonstrate that acute RB loss abrogates DNA damage-induced cell cycle arrest to induce gammaH2AX foci formation. Thus, secondary genetic lesions induced by RB loss have implications for the chemotherapeutic response and the development of genetic instability.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1093/nar/gkg919\">DOI 10.1093/nar/gkg919</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000188988700018\">Web of Science ID 000188988700018</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/14704340\">PubMedID 14704340</a></span>\n  </p>","text":"Abstract:\nCell cycle checkpoints induced by DNA damage play an integral role in preservation of genomic stability by allowing cells to limit the propagation of deleterious mutations. The retinoblastoma tumor suppressor (RB) is crucial for the maintenance of the DNA damage checkpoint function because it elicits cell cycle arrest in response to a variety of genotoxic stresses. Although sporadic loss of RB is characteristic of most cancers and results in the bypass of the DNA damage checkpoint, the consequence of RB loss upon chemotherapeutic responsiveness has been largely uninvestigated. Here, we employed a conditional knockout approach to ablate RB in adult fibroblasts. This system enabled us to examine the DNA damage response of adult cells following acute RB deletion. Using this system, we demonstrated that loss of RB disrupted the DNA damage checkpoint elicited by either cisplatin or camptothecin exposure. Strikingly, this bypass was not associated with enhanced repair, but rather the accumulation of phosphorylated H2AX (gammaH2AX) foci, which indicate DNA double-strand breaks. The formation of gammaH2AX foci was due to ongoing replication following chemotherapeutic treatment in the RB-deficient cells. Additionally, peak gammaH2AX accumulation occurred in S-phase cells undergoing DNA replication in the presence of damage, and these gammaH2AX foci co-localized with replication foci. These results demonstrate that acute RB loss abrogates DNA damage-induced cell cycle arrest to induce gammaH2AX foci formation. Thus, secondary genetic lesions induced by RB loss have implications for the chemotherapeutic response and the development of genetic instability.\nDOI: 10.1093/nar/gkg919, https://dx.doi.org/10.1093/nar/gkg919\nWeb of Science ID: 000188988700018, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000188988700018\nPubMedID: 14704340, https://www.ncbi.nlm.nih.gov/pubmed/14704340"},"doiId":"10.1093/nar/gkg919","doiUrl":"https://dx.doi.org/10.1093/nar/gkg919","featured":true,"firstPublished":{"html":"Jan 1, 2004 00:00:00 AM","text":"Jan 1, 2004 00:00:00 AM","value":"2004-01-01T00:00:00.000-08:00"},"mlaCitation":"Bosco, E. E. et al. ?RB Signaling Prevents Replication-Dependent DNA Double-Strand Breaks Following Genotoxic Insult.? <i>NUCLEIC ACIDS RESEARCH</i> 32.1 (2004): 25?34. Print.","pubMedId":"14704340","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/14704340","publicationId":242874,"title":"RB signaling prevents replication-dependent DNA double-strand breaks following genotoxic insult","type":"article","webOfScienceId":"000188988700018","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000188988700018"},{"abstract":"Cancer cells arise from normal cells through the acquisition of a series of mutations in oncogenes and tumour suppressor genes. Mouse models of human cancer often rely on germline alterations that activate or inactivate genes of interest. One limitation of this approach is that germline mutations might have effects other than somatic mutations, owing to developmental compensation. To model sporadic cancers associated with inactivation of the retinoblastoma (RB) tumour suppressor gene in humans, we have produced a conditional allele of the mouse Rb gene. We show here that acute loss of Rb in primary quiescent cells is sufficient for cell cycle entry and has phenotypic consequences different from germline loss of Rb function. This difference is explained in part by functional compensation by the Rb-related gene p107. We also show that acute loss of Rb in senescent cells leads to reversal of the cellular senescence programme. Thus, the use of conditional knockout strategies might refine our understanding of gene function and help to model human cancer more accurately.","apaCitation":"Sage, J., Miller, A. L., Perez-Mancera, P. A., Wysocki, J. M., &amp; Jacks, T. (2003). Acute mutation of retinoblastoma gene function is sufficient for cell cycle re-entry. <i>NATURE</i>, <i>424</i>(6945), 223?228.","capCitation":"<span class=\"title\">\n  <span>Acute mutation of retinoblastoma gene function is sufficient for cell cycle re-entry</span>\n  <i>NATURE</i>\n</span>\n<span class=\"authors\">Sage, J., Miller, A. L., Perez-Mancera, P. A., Wysocki, J. M., Jacks, T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2003</span>; <span class=\"volume\">424 (6945)</span><span class=\"pages\">: 223-228</span>\n</span>","chicagoCitation":"Sage, J., A. L. Miller, P. A. Perez-Mancera, J. M. Wysocki, and T. Jacks. 2003. ?Acute Mutation of Retinoblastoma Gene Function Is Sufficient for Cell Cycle Re-Entry.? <i>NATURE</i> 424 (6945). NATURE PUBLISHING GROUP: 223?28.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">Cancer cells arise from normal cells through the acquisition of a series of mutations in oncogenes and tumour suppressor genes. Mouse models of human cancer often rely on germline alterations that activate or inactivate genes of interest. One limitation of this approach is that germline mutations might have effects other than somatic mutations, owing to developmental compensation. To model sporadic cancers associated with inactivation of the retinoblastoma (RB) tumour suppressor gene in humans, we have produced a conditional allele of the mouse Rb gene. We show here that acute loss of Rb in primary quiescent cells is sufficient for cell cycle entry and has phenotypic consequences different from germline loss of Rb function. This difference is explained in part by functional compensation by the Rb-related gene p107. We also show that acute loss of Rb in senescent cells leads to reversal of the cellular senescence programme. Thus, the use of conditional knockout strategies might refine our understanding of gene function and help to model human cancer more accurately.</p>\n  <p class=\"doi\">\n    <span>View details for <a href=\"https://dx.doi.org/10.1038/nature01764\">DOI 10.1038/nature01764</a></span>\n  </p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000184032700049\">Web of Science ID 000184032700049</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/12853964\">PubMedID 12853964</a></span>\n  </p>","text":"Abstract:\nCancer cells arise from normal cells through the acquisition of a series of mutations in oncogenes and tumour suppressor genes. Mouse models of human cancer often rely on germline alterations that activate or inactivate genes of interest. One limitation of this approach is that germline mutations might have effects other than somatic mutations, owing to developmental compensation. To model sporadic cancers associated with inactivation of the retinoblastoma (RB) tumour suppressor gene in humans, we have produced a conditional allele of the mouse Rb gene. We show here that acute loss of Rb in primary quiescent cells is sufficient for cell cycle entry and has phenotypic consequences different from germline loss of Rb function. This difference is explained in part by functional compensation by the Rb-related gene p107. We also show that acute loss of Rb in senescent cells leads to reversal of the cellular senescence programme. Thus, the use of conditional knockout strategies might refine our understanding of gene function and help to model human cancer more accurately.\nDOI: 10.1038/nature01764, https://dx.doi.org/10.1038/nature01764\nWeb of Science ID: 000184032700049, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000184032700049\nPubMedID: 12853964, https://www.ncbi.nlm.nih.gov/pubmed/12853964"},"doiId":"10.1038/nature01764","doiUrl":"https://dx.doi.org/10.1038/nature01764","featured":true,"firstPublished":{"html":"Jul 10, 2003 00:00:00 AM","text":"Jul 10, 2003 00:00:00 AM","value":"2003-07-10T00:00:00.000-07:00"},"mlaCitation":"Sage, J. et al. ?Acute Mutation of Retinoblastoma Gene Function Is Sufficient for Cell Cycle Re-Entry.? <i>NATURE</i> 424.6945 (2003): 223?228. Print.","pubMedId":"12853964","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/12853964","publicationId":24327,"title":"Acute mutation of retinoblastoma gene function is sufficient for cell cycle re-entry","type":"article","webOfScienceId":"000184032700049","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000184032700049"},{"abstract":"The retinoblastoma protein, pRB, and the closely related proteins p107 and p130 are important regulators of the mammalian cell cycle. Biochemical and genetic studies have demonstrated overlapping as well as distinct functions for the three proteins in cell cycle control and mouse development. However, the role of the pRB family as a whole in the regulation of cell proliferation, cell death, or cell differentiation is not known. We generated embryonic stem (ES) cells and other cell types mutant for all three genes. Triple knock-out mouse embryonic fibroblasts (TKO MEFs) had a shorter cell cycle than wild-type, single, or double knock-out control cells. TKO cells were resistant to G(1) arrest following DNA damage, despite retaining functional p53 activity. They were also insensitive to G(1) arrest signals following contact inhibition or serum starvation. Finally, TKO MEFs did not undergo senescence in culture and do possess some characteristics of transformed cells. Our results confirm the essential role of the Rb family in the control of the G(1)/S transition, place the three Rb family members downstream of multiple cell cycle control pathways, and further the link between loss of cell cycle control and tumorigenesis.","apaCitation":"Sage, J., Mulligan, G. J., Attardi, L. D., Miller, A., Chen, S. Q., Williams, B., ? Jacks, T. (2000). Targeted disruption of the three Rb-related genes leads to loss of G(1) control and immortalization. <i>GENES &amp; DEVELOPMENT</i>, <i>14</i>(23), 3037?3050.","capCitation":"<span class=\"title\">\n  <span>Targeted disruption of the three Rb-related genes leads to loss of G(1) control and immortalization</span>\n  <i>GENES &amp; DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Sage, J., Mulligan, G. J., Attardi, L. D., Miller, A., Chen, S. Q., Williams, B., Theodorou, E., Jacks, T.</span>\n\n<span class=\"details\">\n   <span class=\"year\">2000</span>; <span class=\"volume\">14 (23)</span><span class=\"pages\">: 3037-3050</span>\n</span>","chicagoCitation":"Sage, J., G. J. Mulligan, L. D. Attardi, A. Miller, S. Q. Chen, et al. 2000. ?Targeted Disruption of the Three Rb-Related Genes Leads to Loss of G(1) Control and Immortalization.? <i>GENES &amp; DEVELOPMENT</i> 14 (23). COLD SPRING HARBOR LAB PRESS, PUBLICATIONS DEPT: 3037?50.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The retinoblastoma protein, pRB, and the closely related proteins p107 and p130 are important regulators of the mammalian cell cycle. Biochemical and genetic studies have demonstrated overlapping as well as distinct functions for the three proteins in cell cycle control and mouse development. However, the role of the pRB family as a whole in the regulation of cell proliferation, cell death, or cell differentiation is not known. We generated embryonic stem (ES) cells and other cell types mutant for all three genes. Triple knock-out mouse embryonic fibroblasts (TKO MEFs) had a shorter cell cycle than wild-type, single, or double knock-out control cells. TKO cells were resistant to G(1) arrest following DNA damage, despite retaining functional p53 activity. They were also insensitive to G(1) arrest signals following contact inhibition or serum starvation. Finally, TKO MEFs did not undergo senescence in culture and do possess some characteristics of transformed cells. Our results confirm the essential role of the Rb family in the control of the G(1)/S transition, place the three Rb family members downstream of multiple cell cycle control pathways, and further the link between loss of cell cycle control and tumorigenesis.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000165788600011\">Web of Science ID 000165788600011</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/11114892\">PubMedID 11114892</a></span>\n  </p>","text":"Abstract:\nThe retinoblastoma protein, pRB, and the closely related proteins p107 and p130 are important regulators of the mammalian cell cycle. Biochemical and genetic studies have demonstrated overlapping as well as distinct functions for the three proteins in cell cycle control and mouse development. However, the role of the pRB family as a whole in the regulation of cell proliferation, cell death, or cell differentiation is not known. We generated embryonic stem (ES) cells and other cell types mutant for all three genes. Triple knock-out mouse embryonic fibroblasts (TKO MEFs) had a shorter cell cycle than wild-type, single, or double knock-out control cells. TKO cells were resistant to G(1) arrest following DNA damage, despite retaining functional p53 activity. They were also insensitive to G(1) arrest signals following contact inhibition or serum starvation. Finally, TKO MEFs did not undergo senescence in culture and do possess some characteristics of transformed cells. Our results confirm the essential role of the Rb family in the control of the G(1)/S transition, place the three Rb family members downstream of multiple cell cycle control pathways, and further the link between loss of cell cycle control and tumorigenesis.\n\nWeb of Science ID: 000165788600011, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000165788600011\nPubMedID: 11114892, https://www.ncbi.nlm.nih.gov/pubmed/11114892"},"featured":true,"firstPublished":{"html":"Dec 1, 2000 00:00:00 AM","text":"Dec 1, 2000 00:00:00 AM","value":"2000-12-01T00:00:00.000-08:00"},"mlaCitation":"Sage, J. et al. ?Targeted Disruption of the Three Rb-Related Genes Leads to Loss of G(1) Control and Immortalization.? <i>GENES &amp; DEVELOPMENT</i> 14.23 (2000): 3037?3050. Print.","pubMedId":"11114892","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/11114892","publicationId":8961,"title":"Targeted disruption of the three Rb-related genes leads to loss of G(1) control and immortalization","type":"article","webOfScienceId":"000165788600011","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000165788600011"},{"abstract":"The Kit receptor and its ligand KL, which together constitute an essential effector at various stages of embryonic development, are both present during adult gametogenesis. In the testis, KL is expressed in Sertoli cells, and Kit in germ cells, starting at the premeiotic stages. A series of observations indicated previously a role in spermatogonia survival, without excluding a possible function at later stages. We identified a complex pattern of expression of the two components in the adult murine testis, suggestive of a role in the meiotic progression of spermatocytes. At stages VII-VIII of the cycle of the seminiferous epithelium, the time when spermatocytes enter meiosis, the membrane-associated form of KL extends on the Sertoli cell from the peripheral to the adluminal compartment of the tubule. We also found that the receptor is present on the surface of germ cells up to the pachytene stage. The availability of differentiated Sertoli cell lines, which express the KL protein and support part of the maturation of germ cells in coculture, allowed us to ask whether, in the in vitro reconstructed system, transit of spermatocytes through meiosis requires the Kit-KL interaction. Addition of a blocking monoclonal antibody against the Kit receptor (ACK2) inhibited extensively the appearance of haploid cells and the expression of a haploid-phase-specific gene (Prm1). Recognition of the supporting Sertoli cell by germ cells was not affected, indicating a requirement for the activity of the receptor for either entering or completing meiosis. Involvement of the membrane-associated form of the ligand was suggested by the observation that addition of the soluble form of KL was equally inhibitory.","apaCitation":"Vincent, S., Segretain, D., Nishikawa, S., Nishikawa, S., Sage, J., Cuzin, F., &amp; Rassoulzadegan, M. (1998). Stage-specific expression of the Kit receptor and its ligand (KL) during male gametogenesis in the mouse: a Kit-KL interaction critical for meiosis. <i>DEVELOPMENT</i>, <i>125</i>(22), 4585?4593.","capCitation":"<span class=\"title\">\n  <span>Stage-specific expression of the Kit receptor and its ligand (KL) during male gametogenesis in the mouse: a Kit-KL interaction critical for meiosis</span>\n  <i>DEVELOPMENT</i>\n</span>\n<span class=\"authors\">Vincent, S., Segretain, D., Nishikawa, S., Nishikawa, S., Sage, J., Cuzin, F., Rassoulzadegan, M.</span>\n\n<span class=\"details\">\n   <span class=\"year\">1998</span>; <span class=\"volume\">125 (22)</span><span class=\"pages\">: 4585-4593</span>\n</span>","chicagoCitation":"Vincent, S., D. Segretain, S. Nishikawa, S. Nishikawa, J. Sage, et al. 1998. ?Stage-Specific Expression of the Kit Receptor and Its Ligand (KL) during Male Gametogenesis in the Mouse: a Kit-KL Interaction Critical for Meiosis.? <i>DEVELOPMENT</i> 125 (22). COMPANY OF BIOLOGISTS LTD: 4585?93.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">The Kit receptor and its ligand KL, which together constitute an essential effector at various stages of embryonic development, are both present during adult gametogenesis. In the testis, KL is expressed in Sertoli cells, and Kit in germ cells, starting at the premeiotic stages. A series of observations indicated previously a role in spermatogonia survival, without excluding a possible function at later stages. We identified a complex pattern of expression of the two components in the adult murine testis, suggestive of a role in the meiotic progression of spermatocytes. At stages VII-VIII of the cycle of the seminiferous epithelium, the time when spermatocytes enter meiosis, the membrane-associated form of KL extends on the Sertoli cell from the peripheral to the adluminal compartment of the tubule. We also found that the receptor is present on the surface of germ cells up to the pachytene stage. The availability of differentiated Sertoli cell lines, which express the KL protein and support part of the maturation of germ cells in coculture, allowed us to ask whether, in the in vitro reconstructed system, transit of spermatocytes through meiosis requires the Kit-KL interaction. Addition of a blocking monoclonal antibody against the Kit receptor (ACK2) inhibited extensively the appearance of haploid cells and the expression of a haploid-phase-specific gene (Prm1). Recognition of the supporting Sertoli cell by germ cells was not affected, indicating a requirement for the activity of the receptor for either entering or completing meiosis. Involvement of the membrane-associated form of the ligand was suggested by the observation that addition of the soluble form of KL was equally inhibitory.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/000077649200022\">Web of Science ID 000077649200022</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/9778516\">PubMedID 9778516</a></span>\n  </p>","text":"Abstract:\nThe Kit receptor and its ligand KL, which together constitute an essential effector at various stages of embryonic development, are both present during adult gametogenesis. In the testis, KL is expressed in Sertoli cells, and Kit in germ cells, starting at the premeiotic stages. A series of observations indicated previously a role in spermatogonia survival, without excluding a possible function at later stages. We identified a complex pattern of expression of the two components in the adult murine testis, suggestive of a role in the meiotic progression of spermatocytes. At stages VII-VIII of the cycle of the seminiferous epithelium, the time when spermatocytes enter meiosis, the membrane-associated form of KL extends on the Sertoli cell from the peripheral to the adluminal compartment of the tubule. We also found that the receptor is present on the surface of germ cells up to the pachytene stage. The availability of differentiated Sertoli cell lines, which express the KL protein and support part of the maturation of germ cells in coculture, allowed us to ask whether, in the in vitro reconstructed system, transit of spermatocytes through meiosis requires the Kit-KL interaction. Addition of a blocking monoclonal antibody against the Kit receptor (ACK2) inhibited extensively the appearance of haploid cells and the expression of a haploid-phase-specific gene (Prm1). Recognition of the supporting Sertoli cell by germ cells was not affected, indicating a requirement for the activity of the receptor for either entering or completing meiosis. Involvement of the membrane-associated form of the ligand was suggested by the observation that addition of the soluble form of KL was equally inhibitory.\n\nWeb of Science ID: 000077649200022, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000077649200022\nPubMedID: 9778516, https://www.ncbi.nlm.nih.gov/pubmed/9778516"},"featured":true,"firstPublished":{"html":"Nov 1, 1998 00:00:00 AM","text":"Nov 1, 1998 00:00:00 AM","value":"1998-11-01T00:00:00.000-08:00"},"mlaCitation":"Vincent, S. et al. ?Stage-Specific Expression of the Kit Receptor and Its Ligand (KL) during Male Gametogenesis in the Mouse: a Kit-KL Interaction Critical for Meiosis.? <i>DEVELOPMENT</i> 125.22 (1998): 4585?4593. Print.","pubMedId":"9778516","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/9778516","publicationId":242885,"title":"Stage-specific expression of the Kit receptor and its ligand (KL) during male gametogenesis in the mouse: a Kit-KL interaction critical for meiosis","type":"article","webOfScienceId":"000077649200022","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/000077649200022"},{"abstract":"A cell culture system that supports the differentiation of male germ cells through meiosis is described. It takes advantage of the properties of a cell line, 15P-1, established from testicular cells of transgenic mice that express the large T protein of polyoma virus in the seminiferous epithelium. This line exhibits features characteristics of Sertoli cells, including transcription of the Wilms' tumor (WT1) and Steel genes. Cells of the 15P-1 type support the meiotic and postmeiotic differentiation in cocultures of diploid premeiotic germ cells into haploid spermatids expressing the protamine (Prm-1) gene. When cocultured with 15P-1 cells, testicular cells explanted from immature 9-day-old animals, before the onset of the first meiosis, generated tetrads of haploid cells with the morphology of round spermatids and initiated protamine transcription.","apaCitation":"Rassoulzadegan, M., PAQUISFLUCKLINGER, V., Bertino, B., Sage, J., Jasin, M., Miyagawa, K., ? Cuzin, F. (1993). TRANSMEIOTIC DIFFERENTIATION OF MALE GERM-CELLS IN CULTURE. <i>CELL</i>, <i>75</i>(5), 997?1006.","capCitation":"<span class=\"title\">\n  <span>TRANSMEIOTIC DIFFERENTIATION OF MALE GERM-CELLS IN CULTURE</span>\n  <i>CELL</i>\n</span>\n<span class=\"authors\">Rassoulzadegan, M., PAQUISFLUCKLINGER, V., Bertino, B., Sage, J., Jasin, M., Miyagawa, K., VANHEYNINGEN, V., Besmer, P., Cuzin, F.</span>\n\n<span class=\"details\">\n   <span class=\"year\">1993</span>; <span class=\"volume\">75 (5)</span><span class=\"pages\">: 997-1006</span>\n</span>","chicagoCitation":"Rassoulzadegan, M., V. PAQUISFLUCKLINGER, B. Bertino, J. Sage, M. Jasin, et al. 1993. ?TRANSMEIOTIC DIFFERENTIATION OF MALE GERM-CELLS IN CULTURE.? <i>CELL</i> 75 (5). CELL PRESS: 997?1006.","detail":{"html":"<h4>Abstract</h4>\n  <p class=\"abstract\">A cell culture system that supports the differentiation of male germ cells through meiosis is described. It takes advantage of the properties of a cell line, 15P-1, established from testicular cells of transgenic mice that express the large T protein of polyoma virus in the seminiferous epithelium. This line exhibits features characteristics of Sertoli cells, including transcription of the Wilms' tumor (WT1) and Steel genes. Cells of the 15P-1 type support the meiotic and postmeiotic differentiation in cocultures of diploid premeiotic germ cells into haploid spermatids expressing the protamine (Prm-1) gene. When cocultured with 15P-1 cells, testicular cells explanted from immature 9-day-old animals, before the onset of the first meiosis, generated tetrads of haploid cells with the morphology of round spermatids and initiated protamine transcription.</p>\n  <p class=\"wos\">\n    <span>View details for <a href=\"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&amp;rft_id=info:ut/A1993MK96600019\">Web of Science ID A1993MK96600019</a></span>\n  </p>\n  <p class=\"pub-med\">\n    <span>View details for <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/7504588\">PubMedID 7504588</a></span>\n  </p>","text":"Abstract:\nA cell culture system that supports the differentiation of male germ cells through meiosis is described. It takes advantage of the properties of a cell line, 15P-1, established from testicular cells of transgenic mice that express the large T protein of polyoma virus in the seminiferous epithelium. This line exhibits features characteristics of Sertoli cells, including transcription of the Wilms' tumor (WT1) and Steel genes. Cells of the 15P-1 type support the meiotic and postmeiotic differentiation in cocultures of diploid premeiotic germ cells into haploid spermatids expressing the protamine (Prm-1) gene. When cocultured with 15P-1 cells, testicular cells explanted from immature 9-day-old animals, before the onset of the first meiosis, generated tetrads of haploid cells with the morphology of round spermatids and initiated protamine transcription.\n\nWeb of Science ID: A1993MK96600019, https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/A1993MK96600019\nPubMedID: 7504588, https://www.ncbi.nlm.nih.gov/pubmed/7504588"},"featured":true,"firstPublished":{"html":"Dec 3, 1993 00:00:00 AM","text":"Dec 3, 1993 00:00:00 AM","value":"1993-12-03T00:00:00.000-08:00"},"mlaCitation":"Rassoulzadegan, M. et al. ?TRANSMEIOTIC DIFFERENTIATION OF MALE GERM-CELLS IN CULTURE.? <i>CELL</i> 75.5 (1993): 997?1006. Print.","pubMedId":"7504588","pubMedUrl":"https://www.ncbi.nlm.nih.gov/pubmed/7504588","publicationId":242891,"title":"TRANSMEIOTIC DIFFERENTIATION OF MALE GERM-CELLS IN CULTURE","type":"article","webOfScienceId":"A1993MK96600019","webOfScienceUrl":"https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/A1993MK96600019"}]}},"publicationsTabUrl":"/profiles/hesc/julien-sage?tab=publications","researchAndScholarshipTabUrl":"/profiles/hesc/julien-sage?tab=research-and-scholarship","showAcademicAppointments":true,"showAdministrativeAppointments":true,"showBio":false,"showBioScienceProgramAffiliations":false,"showClerkships":false,"showClinicalCare":false,"showClinicalFocus":false,"showClinicalTrialsOnResearchAndScholarship":true,"showCommunityAndInternationalWork":false,"showContactInfo":true,"showCourses":true,"showCurrentClinicalInterests":false,"showCurrentResearchAndScholarlyInterests":true,"showCurrentRoleAtStanford":false,"showGraduateAndFellowshipPrograms":true,"showHonorsAndAwards":true,"showIndependentStudies":true,"showIndustryRelationships":false,"showLabAffiliations":false,"showLinks":true,"showMembershipOrganizations":false,"showPatents":false,"showPersonalInterests":false,"showPostdoctoralAdvisees":true,"showPrimaryContactInfo":true,"showPriorYearCourses":false,"showProfessionalAffiliationsAndActivities":false,"showProfessionalEducation":true,"showProfessionalInterests":false,"showProfessionalOrganizationsOnBio":false,"showProjectsOnBio":false,"showProjectsOnResearchAndScholarship":false,"showPublications":true,"showResearchProjects":false,"showServiceWork":false,"showSimilarProfiles":true,"showStanfordAdvisees":true,"showStanfordAdvisors":false,"showSupervisors":false,"showTagCloud":true,"showWorkExperience":false,"similarProfileIds":"19907,5931,3851,72452,4726,4517,7991,3892,10445,4658,4713,6012,6066,6423,18937,37666,73156,8514,4159,9405,19524","stanfordAdvisees":[{"adviseesHtml":"<a href=\"https://profiles.stanford.edu/kevin-meng\">Kevin Meng</a>, <a href=\"https://profiles.stanford.edu/youcef-ouadah\">Youcef Ouadah</a>, <a href=\"https://profiles.stanford.edu/elizabeth-qin\">Elizabeth Qin</a>, <a href=\"https://profiles.stanford.edu/miranda-stratton\">Miranda Stratton</a>, <a href=\"https://profiles.stanford.edu/ian-winters\">Ian Winters</a>, <a href=\"https://profiles.stanford.edu/nora-yucel\">Nora Yucel</a>","relation":"Doctoral Dissertation Reader (AC)"},{"adviseesHtml":"<a href=\"https://profiles.stanford.edu/julia-arand\">Julia Arand</a>, <a href=\"https://profiles.stanford.edu/garry-coles\">Garry Coles</a>, <a href=\"https://profiles.stanford.edu/fangfei-qu\">Fangfei Qu</a>, <a href=\"https://profiles.stanford.edu/jamie-webster\">Jamie Webster</a>","relation":"Postdoctoral Faculty Sponsor"},{"adviseesHtml":"<a href=\"https://profiles.stanford.edu/dian-yang\">Dian Yang</a>","relation":"Doctoral Dissertation Advisor (AC)"}],"tabCount":4,"teachingTabUrl":"/profiles/hesc/julien-sage?tab=teaching"}},
	        
	        accessToken: '',
	        baseUrl: '/profiles',
	        namespace: 'hesc',
	        capApiUrl: '/profiles/proxy/api/cap',
	        capClassicSignInUrl: 'https://cap.stanford.edu/profiles/auth/',
	        isBannerSwapped: true
	      });
	      require(['main']);
	    </script>
    
    <script type="text/javascript">
    /* <![CDATA[ */
    function addLoadEvent(func) {
        var oldonload = window.onload;
        if (typeof window.onload != 'function') {
            window.onload = func;
        } else {
            window.onload = function() {
                if (oldonload) {
                    oldonload();
                }
                func();
            }
        }
    }

    function preloader() {
        var imageViewElts=document.getElementsByName('imageView');
        var imageType =  "big";
        for (var i=0; i<imageViewElts.length; i++) {
            var imageViewEltId = imageViewElts[i].id;
            var imageViewEltIdSplitResult = imageViewEltId.split("_");
            
            var appUrl = imageViewEltIdSplitResult[1];
            appUrl = appUrl.replace(/XxX/g,"\/");
            var fid = imageViewEltIdSplitResult[2];
            
            if(imageViewEltIdSplitResult.length > 3){
                var imageType = imageViewEltIdSplitResult[3];
            }
            
            if(imageViewEltIdSplitResult.length > 4){
                var profileLastModified = imageViewEltIdSplitResult[4];
            }
            
            imageViewElts[i].src = appUrl + "/viewImage?profileId=" + fid + "&type=" + imageType + "&showNoImage=true&lastModified="+ profileLastModified;
        }
    }

    addLoadEvent(preloader);
    /* ]]> */
    </script>    
  
</body></html>