<html><head>
	<meta charset="UTF-8">
	<title>Prof Shoumo Bhattacharya FMedSci FRCP - Division of Cardiovascular Medicine</title>
			<link rel="stylesheet" media="screen" href="/_static/website/stylesheets/general.css" type="text/css">
	<link rel="stylesheet" media="print" href="/_static/website/stylesheets/print.css" type="text/css">
	<link rel="stylesheet" media="screen" href="/_static/website/stylesheets/jquery.lightbox-0.5.css" type="text/css"><link rel="stylesheet" media="screen" href="/_static/website/stylesheets/jquery-ui-1.10.4.custom.min.css" type="text/css"><link rel="stylesheet" media="screen" href="/_static/website/stylesheets/leaflet.css" type="text/css"><link rel="stylesheet" media="screen" href="/_static/website/stylesheets/MarkerCluster.css" type="text/css"><link rel="stylesheet" media="screen" href="/_static/website/stylesheets/L.Control.ZoomMin.css" type="text/css">	<script async="" src="//www.google-analytics.com/analytics.js"></script><script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
	<script src="/_static/website/javascripts/content-tabbed-menus.js" type="text/javascript"></script><script src="/_static/website/javascripts/publications.js" type="text/javascript"></script><script id="altmetric-embed-js" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/altmetric_badges-d99e7772b00f31495fed09fece0b7237.js"></script><script src="/_static/website/javascripts/jquery.lightbox-0.5.min.js" type="text/javascript"></script><script src="/_static/website/javascripts/jquery-ui-1.10.4.custom.min.js" type="text/javascript"></script><script src="/_static/website/javascripts/leaflet.js" type="text/javascript"></script><script src="/_static/website/javascripts/leaflet.markercluster-src.js" type="text/javascript"></script><script src="/_static/website/javascripts/L.Control.ZoomMin.js" type="text/javascript"></script>		<link rel="shortcut icon" href="/_static/website/images/icons/rdm_favicon.ico" type="image/x-icon">
<link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed-deec67c8878bf9a094e44df5cac3dc81.css"></head>
<body class="rdm">
	<div id="header-wrapper">
		<div id="header">
			
			<a href="http://www.ox.ac.uk/" title="Link to Oxford University homepage" target="_blank"><img class="logo-box" src="/_static/website/images/ox-rectangle-border.gif" alt="Oxford University logo"></a>
			<a href="http://www.rdm.ox.ac.uk/" title="Link to Radcliffe Department of Medicine" target="_blank"><img id="rdm-logo-sub" src="/_static/website/images/RDMLogo8x3.png" alt="RDM logo"></a>
			<!-- <img style="float:left;margin-right:5px" src="/_asset/image/cardiov-logo.png" alt="CardioV logo" /> -->
			<h1 class="cardiov"><a href="/" title="Link to Division of Cardiovascular Medicine homepage"><img src="/_static/website/images/heading1-cardiov.gif" alt="Division of Cardiovascular Medicine"></a></h1>
			<ul id="primary-menu">
							<li><span><a href="/home" title="Home">Home</a></span></li>
							<li class="current"><span><a href="/research" title="Research">Research</a></span></li>
							<li><span><a href="/graduate-studies" title="Graduate Studies">Graduate Studies</a></span></li>
							<li><span><a href="/about-us" title="About Us">About Us</a></span></li>
							<li><span><a href="/clinical-care" title="Clinical Care">Clinical Care</a></span></li>
							<li><span><a href="/seminars-2" title="Seminars">Seminars</a></span></li>
						</ul>
			<!-- <form class="search-ndcls" action="/_search" method="get"><div class="search-box"><input id="search-field" name="q" type="text" size="10" value="Search..." onFocus="if(this.value == 'Search...') {this.value = '';}" onBlur="if (this.value == '') {this.value = 'Search...';}" /><input id="magnifier_image" name="b" type="image" src="/_static/website/images/icons/icon-search.gif" alt="magnifying glass" /></div></form> -->
			<form style="float:right" class="search" id="cse-search-box" action="/_search">
					
			 <input name="cx" value="013190107294343682755:qydnythmwca" type="hidden">
			 <input name="ie" value="UTF-8" type="hidden">
			 <input id="ndmsearchbox" style="width:120px;color:#002147" name="q" size="31" type="text">
			 
			 <input name="nojs" value="1" type="hidden">
		   </form>	
		</div>
	</div>
	<div id="content-wrapper">
									<ul id="secondary-menu" class="group">
							<li class="current"><a class="secondary-menu-link" href="/principal-investigators" title="Principal Investigators">Principal Investigators</a></li>
							<li><a class="secondary-menu-link" href="/research-areas" title="Research Areas">Research Areas</a></li>
							<li><a class="secondary-menu-link" href="/technology-exchange" title="Technology Exchange">Technology Exchange</a></li>
							<li><a target="_blank" class="secondary-menu-link" href="/find-funding" title="Find funding">Find funding</a></li>
							</ul>	
				
<script type="text/javascript">
$(function() {
	$('a.lightbox').lightBox({
		overlayOpacity: 0.6,
		imageLoading: '/_static/website/images/lightbox/lightbox-ico-loading.gif',
		imageBtnClose: '/_static/website/images/lightbox/lightbox-btn-close.gif',
		imageBtnPrev: '/_static/website/images/lightbox/lightbox-btn-prev.gif',
		imageBtnNext: '/_static/website/images/lightbox/lightbox-btn-next.gif',
		imageBlank: '/_static/website/images/lightbox/lightbox-blank.gif',
		fixedNavigation:true
	});
});
</script>
<div id="content">
		<h1>Prof Shoumo Bhattacharya FMedSci FRCP</h1>
	<ul class="content-menu">
		<li id="content-menu-research" class="current"><a href="#research" title="Research"><span>Research</span></a></li>
		<li id="content-menu-collaborations"><a href="#collaborations" title="Collaborations"><span>Collaborations</span></a></li>		<li id="content-menu-publications"><a href="#publications" title="Publications"><span>Publications</span></a></li>					</ul>
	<div id="content-research" class="content-panel">
		<table>
						<tbody><tr>
				<th>Research Area:</th>
				<td>Cell and Molecular Biology</td>
			</tr>
									<tr>
				<th>Technology Exchange:</th>
				<td>Drug discovery, Ex vivo models, Immunohistochemistry, In situ hybridisation, Magnetic resonance imaging &amp; spectroscopy, Microscopy (Confocal), Protein interaction, Transcript profiling and Transgenesis</td>
			</tr>
									<tr>
				<th>Scientific Themes:</th>
				<td>Cardiovascular Science and Molecular, Cell &amp; Systems Biology</td>
			</tr>
											</tbody></table>
				<div class="figure">
			<a class="lightbox" title="The Scar-In-A-Jar Assay allows us to identify novel drugs that block collagen secretion, and may be of potential use in fibrosis. Different doses of a candidate drug are shown, with corresponding effect on collagen, detected by immunofluorescence." href="http://tools.ndm.ox.ac.uk/pinfox/importers/pinfox/RDM/images/slide1-4_fit_900x600.jpg"><img src="http://tools.ndm.ox.ac.uk/pinfox/importers/pinfox/RDM/images/slide1-4_fit_204x204.jpg" alt="The Scar-In-A-Jar Assay allows us to identify novel drugs that block collagen secretion, and may be of potential use in fibrosis. Different doses of a candidate drug are shown, with corresponding effect on collagen, detected by immunofluorescence."></a>
			<p>The Scar-In-A-Jar Assay allows us to identify novel drugs that block collagen secretion, and may be ...</p>
		</div>
				<p></p><p>Our laboratory, based at the <a href="http://www.well.ox.ac.uk/">Wellcome Trust Centre for Human Genetics</a>, with collaborations at the <a href="http://www.tdi.ox.ac.uk/home">Target Discovery Institute</a> has three major themes. &nbsp;&nbsp;</p><p><strong>Theme 1 - Fibrosis and BMP/TGFb Signaling</strong></p><p><strong>Noel Faherty, Matt Benson, Angela Lee</strong></p><p>Fibrosis is a major obstacle to regeneration, and plays a key role in diseases of the heart, kidneys, lungs, and liver. A number of <a href="http://www.ncbi.nlm.nih.gov/pubmed/20538689">key cytokine pathways</a> that activate in cardiac fibrosis have been identified - including angiotensin II, TGFb, PDGF, CTGF, and endothelin-1, whereas <a href="http://www.ncbi.nlm.nih.gov/pubmed/19482601">BMP signalling is protective</a>. A key challenge is to identify therapeutic targets for fibrosis within these signalling pathways.&nbsp;</p><p><strong>a) BMP signalling:</strong> Using high-throughput siRNA approaches in mammalian cells we have identified a family of interacting genes that when knocked down enhance BMP signaling through a novel pathway.&nbsp; The goal of the project is to understand the pathway in depth, using cell based&nbsp; &amp; molecular approaches, and apply this to fibrotic disease.</p><p><strong>b) Collagen matrix formation:</strong> &nbsp;We have used network approaches to identify novel genes that are highly connected to the known fibrosis disease gene network. The high connectivity predicts that these genes are candidate drug targets for fibrosis. We have tested these candidates in a&nbsp;miniaturized “<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19785660">Scar-in-a-Jar assay</a>” using tool compounds,&nbsp;and have identified two novel classes of small molecules that affect collagen secretion in vitro.&nbsp; Goals of the project are to study the mechanisms of action of these molecules and validate them in models of cardiac fibrosis. We are also using the “Scar-in-a-Jar assay” to screen in an unbiased manner for further small molecules that affect collagen secretion at the <a href="http://www.tdi.ox.ac.uk/home">Target Discovery Institute</a> high-throughput screening facility.&nbsp;</p><p><strong>Theme 2 - New Biotechnology</strong></p><p><strong>Graham Davies, Kamayani Singh, Matt Benson, Adrian Gray</strong></p><p><strong>Yeast 3 hybrid system: </strong></p><p>&nbsp;We are establishing and improving a <a href="http://www.ncbi.nlm.nih.gov/pubmed/21499265">published system</a> that will allow us to efficiently identify the targets of small molecules identified in high throughput phenotypic screens.&nbsp; This project is in collaboration with <a href="http://www.tdi.ox.ac.uk/paul-brennan">Dr Paul Brennan</a>, Head of Medicinal Chemistry at the Target Discovery Institute. &nbsp;One goal of the project is to use candidate small molecules that have been identified in phenotypic screens (e.g. "Scar-in-a-Jar") and identify the cellular targets.</p><p><strong>Theme 3 - &nbsp;Heart Formation and Congenital Heart Disease</strong></p><p><strong>Dorota Szumska, Iain Dykes, Rathi Puliyadi, Linta Kucheria</strong></p><p>Congenital heart disease occurs in approximately 1% of all children, and is a major cause of illness and death. &nbsp;Using ENU mutagenesis (in collaboration with Steve Brown, MRC Harwell) we have identified a number of genes that affect heart development, and are studying the developmental mechanisms. In addition, &nbsp;we are part of two large programs: DMDD (Deciphering the Mechanisms of Developmental Disorders) which uses model organism genetics together with phenotyping of embryonic lethal mutants (with Dr Tim Mohun (NIMR) &amp; Jurgen Schneider (Oxford)); and UK10K (with Matt Hurles (WTSI) which uses exon sequencing to identify gene variants in patients with congenital heart disease. &nbsp;We believe that interfacing the DMDD and UK10K data will allow us to understand the mechanisms of congenital heart disease.</p><p><strong>Funding:&nbsp;</strong></p><p>British Heart Foundation and Wellcome Trust Program Grants,&nbsp;Wellcome Trust Strategic Award, BHF Centre of Research Excellence, BHF Centre for Regenerative Medicine.</p><p><strong>Affiliated DPhil Programs that provide funding:</strong></p><p><a href="http://www.cardioscience.ox.ac.uk/bhf-centre-of-research-excellence/graduate-studies">BHF Funded Programs</a></p><p><a href="http://www.rdm.ox.ac.uk/rdm-scholars-programme">RDM Scholars Programme</a></p><p></p>
	</div>
	<div id="content-collaborations" class="content-panel" style="display: none;">
				<div id="collab_map" class="hidden-xs leaflet-container leaflet-fade-anim" style="position: relative;" tabindex="0"><div class="leaflet-map-pane" style="transform: translate3d(0px, 0px, 0px);"><div class="leaflet-tile-pane"><div class="leaflet-layer"><div class="leaflet-tile-container"></div><div class="leaflet-tile-container leaflet-zoom-animated"><img class="leaflet-tile leaflet-tile-loaded" style="height: 256px; width: 256px; transform: translate3d(79px, -75px, 0px);" src="http://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/1/0/0"><img class="leaflet-tile leaflet-tile-loaded" style="height: 256px; width: 256px; transform: translate3d(335px, -75px, 0px);" src="http://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/1/0/1"><img class="leaflet-tile leaflet-tile-loaded" style="height: 256px; width: 256px; transform: translate3d(79px, 181px, 0px);" src="http://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/1/1/0"><img class="leaflet-tile leaflet-tile-loaded" style="height: 256px; width: 256px; transform: translate3d(335px, 181px, 0px);" src="http://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/1/1/1"><img class="leaflet-tile leaflet-tile-loaded" style="height: 256px; width: 256px; transform: translate3d(-177px, -75px, 0px);" src="http://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/1/0/1"><img class="leaflet-tile leaflet-tile-loaded" style="height: 256px; width: 256px; transform: translate3d(591px, -75px, 0px);" src="http://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/1/0/0"><img class="leaflet-tile leaflet-tile-loaded" style="height: 256px; width: 256px; transform: translate3d(-177px, 181px, 0px);" src="http://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/1/1/1"><img class="leaflet-tile leaflet-tile-loaded" style="height: 256px; width: 256px; transform: translate3d(591px, 181px, 0px);" src="http://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/1/1/0"></div></div></div><div class="leaflet-objects-pane"><div class="leaflet-shadow-pane"></div><div class="leaflet-overlay-pane"></div><div class="leaflet-marker-pane"><div class="leaflet-marker-icon mycluster leaflet-zoom-animated leaflet-clickable" style="margin-left: -20px; margin-top: -15px; width: 40px; height: 30px; transform: translate3d(333px, 95px, 0px); z-index: 95;" tabindex="0">4</div></div><div class="leaflet-popup-pane"></div></div></div><div class="leaflet-control-container"><div class="leaflet-top leaflet-left"><div class="leaflet-control-zoom leaflet-bar leaflet-control"><a class="leaflet-control-zoom-in" href="#" title="Zoom in">+</a><a class="leaflet-control-zoom-out leaflet-disabled" href="#" title="Zoom out">-</a><a class="leaflet-control-zoom-min leaflet-disabled" href="#" title="Zoom min">Zoom min</a></div></div><div class="leaflet-top leaflet-right"></div><div class="leaflet-bottom leaflet-left"></div><div class="leaflet-bottom leaflet-right"></div></div></div>
		<div id="map_disclaimer" class="hidden-xs">Beware: points on the map are generated from user entered data and may not always be accurately located</div>
		<table>
			<tbody><tr>
				<th>Name</th>
				<th>Department</th>
				<th>Institution</th>
				<th>Country</th>
			</tr>
															<tr>
								<td><a href="/principal-investigators/researcher/keith-channon">Prof Keith Channon FMedSci FRCP</a></td>
				<td>Cardiovascular Medicine</td>
				<td>Oxford University, John Radcliffe Hospital</td>
				<td>United Kingdom</td>
			</tr>
												<tr>
								<td><a href="/principal-investigators/researcher/hugh-watkins">Prof Hugh Watkins FRS FMedSci</a></td>
				<td>Cardiovascular Medicine</td>
				<td>Oxford University, West Wing, John Radcliffe Hospital</td>
				<td>United Kingdom</td>
			</tr>
												<tr>
								<td>Dr Timothy Mohun</td>
				<td></td>
				<td>NIMR</td>
				<td>United Kingdom</td>
			</tr>
												<tr>
								<td>Dr Angela Russell</td>
				<td>Chemistry</td>
				<td>University of Oxford</td>
				<td>United Kingdom</td>
			</tr>
												<tr>
								<td>Prof Paul Riley</td>
				<td>DPAG</td>
				<td>University of Oxford</td>
				<td>United Kingdom</td>
			</tr>
												<tr>
								<td>Dr Ben Davies (NDM)</td>
				<td>Wellcome Trust Centre for Human Genetics</td>
				<td>Oxford University, Henry Wellcome Building of Genomic Medicine</td>
				<td>United Kingdom</td>
			</tr>
												<tr>
								<td><a href="/principal-investigators/researcher/martin-farrall">Prof Martin Farrall</a></td>
				<td>Cardiovascular Medicine</td>
				<td>Oxford University, Henry Wellcome Building of Genomic Medicine</td>
				<td>United Kingdom</td>
			</tr>
												<tr>
								<td><a href="/principal-investigators/researcher/jurgen-schneider">Prof Jurgen E Schneider</a></td>
				<td>Cardiovascular Medicine</td>
				<td>Oxford University, Henry Wellcome Building of Genomic Medicine</td>
				<td>United Kingdom</td>
			</tr>
												<tr>
								<td>Professor Paul Brennan (NDM)</td>
				<td>Target Discovery Institute</td>
				<td>Oxford University, NDM Research Building</td>
				<td>United Kingdom</td>
			</tr>
												<tr>
								<td>Professor Stefan Knapp (NDM)</td>
				<td>Structural Genomics Consortium</td>
				<td>Oxford University, NDM Research Building</td>
				<td>United Kingdom</td>
			</tr>
					</tbody></table>
			</div>
	<div id="content-publications" class="content-panel" style="display: none;">
						<div id="pi_pub_display_options_spacer">
		    <span>
		    <div id="pi_pub_display_options_old">
			<a class="btn btn-primary btn-sm active" href="#" id="fav_pubs">Key Publications</a>
			<a class="btn btn-primary btn-sm" href="#" id="all_pubs">All Publications</a>
		    </div>
		    </span>
		</div>
								<div class="publication group all" symplectic_id="692368" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1186/s12861-017-0148-y" data-uuid="983f36c8-43a1-96fd-3189-33aee4eb0e86"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=19672503" style="display:inline-block;">
  <img alt="Article has an altmetric score of 2" src="https://badges.altmetric.com/?size=128&amp;score=2&amp;types=tttttttt" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">Szumska D</span><span class="author ">, Cioroch M</span><span class="author ">, Keeling A</span><span class="author ">, Prat A</span><span class="author ">, Seidah NG</span><span class="author ">, Bhattacharya S</span>.</span>		2017.		Pcsk5 is required in the early cranio-cardiac mesoderm for heart development.		<span class="journal-title">BMC Dev Biol</span>,		<span class="journal-volume">17</span>		(1),		pp. 6.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1186/s12861-017-0148-y" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-692368" class="pub_abstract" style="display: none;">
		BACKGROUND: Loss of proprotein convertase subtilisin/kexin type 5 (Pcsk5) results in multiple developmental anomalies including cardiac malformations, caudal regression, pre-sacral mass, renal agenesis, anteroposterior patterning defects, and tracheo-oesophageal and anorectal malformations, and is a model for VACTERL/caudal regression/Currarino syndromes (VACTERL association - Vertebral anomalies, Anal atresia, Cardiac defects, Tracheoesophageal fistula and/or Esophageal atresia, Renal &amp; Radial anomalies and Limb defects). RESULTS: Using magnetic resonance imaging (MRI), we examined heart development in mouse embryos with zygotic and cardiac specific deletion of Pcsk5. We show that conditional deletion of Pcsk5 in all epiblastic lineages recapitulates all developmental malformations except for tracheo-esophageal malformations. Using a conditional deletion strategy, we find that there is an essential and specific requirement for Pcsk5 in the cranio-cardiac mesoderm for cardiogenesis, but not for conotruncal septation or any other aspect of embryonic development. Surprisingly, deletion of Pcsk5 in cardiogenic or pharyngeal mesodermal progenitors that form later from the cranio-cardiac mesoderm does not affect heart development. Neither is Pcsk5 essential in the neural crest, which drives conotruncal septation. CONCLUSIONS: Our results suggest that Pcsk5 may have an essential and early role in the cranio-cardiac mesoderm for heart development. Alternatively, it is possible that Pcsk5 may still play a critical role in Nkx2.5-expressing cardiac progenitors, with persistence of mRNA or protein accounting for the lack of effect of deletion on heart development.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="637275" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1038/ng.3627" data-uuid="44b00096-b299-9d5e-7931-d2fb63f19773"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=10186891" style="display:inline-block;">
  <img alt="Article has an altmetric score of 219" src="https://badges.altmetric.com/?size=128&amp;score=219&amp;types=mtttttfg" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">13</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/27479907#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Sifrim A</span><span class="author ">, Hitz MP</span><span class="author ">, Wilsdon A</span><span class="author ">, Breckpot J</span><span class="author ">, Turki SH</span><span class="author ">, Thienpont B</span><span class="author ">, McRae J</span><span class="author ">, Fitzgerald TW</span><span class="author ">, Singh T</span><span class="author ">, Swaminathan GJ</span><span class="author hidden" style="display: none;">, Prigmore E</span><span class="author hidden" style="display: none;">, Rajan D</span><span class="author hidden" style="display: none;">, Abdul-Khaliq H</span><span class="author hidden" style="display: none;">, Banka S</span><span class="author hidden" style="display: none;">, Bauer UM</span><span class="author hidden" style="display: none;">, Bentham J</span><span class="author hidden" style="display: none;">, Berger F</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Bu'Lock F</span><span class="author hidden" style="display: none;">, Canham N</span><span class="author hidden" style="display: none;">, Colgiu IG</span><span class="author hidden" style="display: none;">, Cosgrove C</span><span class="author hidden" style="display: none;">, Cox H</span><span class="author hidden" style="display: none;">, Daehnert I</span><span class="author hidden" style="display: none;">, Daly A</span><span class="author hidden" style="display: none;">, Danesh J</span><span class="author hidden" style="display: none;">, Fryer A</span><span class="author hidden" style="display: none;">, Gewillig M</span><span class="author hidden" style="display: none;">, Hobson E</span><span class="author hidden" style="display: none;">, Hoff K</span><span class="author hidden" style="display: none;">, Homfray T</span><span class="author hidden" style="display: none;">, INTERVAL Study</span><span class="author hidden" style="display: none;">, Kahlert AK</span><span class="author hidden" style="display: none;">, Ketley A</span><span class="author hidden" style="display: none;">, Kramer HH</span><span class="author hidden" style="display: none;">, Lachlan K</span><span class="author hidden" style="display: none;">, Lampe AK</span><span class="author hidden" style="display: none;">, Louw JJ</span><span class="author hidden" style="display: none;">, Manickara AK</span><span class="author hidden" style="display: none;">, Manase D</span><span class="author hidden" style="display: none;">, McCarthy KP</span><span class="author hidden" style="display: none;">, Metcalfe K</span><span class="author hidden" style="display: none;">, Moore C</span><span class="author hidden" style="display: none;">, Newbury-Ecob R</span><span class="author hidden" style="display: none;">, Omer SO</span><span class="author hidden" style="display: none;">, Ouwehand WH</span><span class="author hidden" style="display: none;">, Park SM</span><span class="author hidden" style="display: none;">, Parker MJ</span><span class="author hidden" style="display: none;">, Pickardt T</span><span class="author hidden" style="display: none;">, Pollard MO</span><span class="author hidden" style="display: none;">, Robert L</span><span class="author hidden" style="display: none;">, Roberts DJ</span><span class="author hidden" style="display: none;">, Sambrook J</span><span class="author hidden" style="display: none;">, Setchfield K</span><span class="author hidden" style="display: none;">, Stiller B</span><span class="author hidden" style="display: none;">, Thornborough C</span><span class="author hidden" style="display: none;">, Toka O</span><span class="author hidden" style="display: none;">, Watkins H</span><span class="author hidden" style="display: none;">, Williams D</span><span class="author hidden" style="display: none;">, Wright M</span><span class="author hidden" style="display: none;">, Mital S</span><span class="author hidden" style="display: none;">, Daubeney PE</span><span class="author hidden" style="display: none;">, Keavney B</span><span class="author hidden" style="display: none;">, Goodship J</span><span class="author hidden" style="display: none;">, UK10K Consortium</span><span class="author hidden" style="display: none;">, Abu-Sulaiman RM</span><span class="author hidden" style="display: none;">, Klaassen S</span><span class="author hidden" style="display: none;">, Wright CF</span><span class="author hidden" style="display: none;">, Firth HV</span><span class="author hidden" style="display: none;">, Barrett JC</span><span class="author hidden" style="display: none;">, Devriendt K</span><span class="author hidden" style="display: none;">, FitzPatrick DR</span><span class="author hidden" style="display: none;">, Brook JD</span><span class="author hidden" style="display: none;">, Deciphering Developmental Disorders Study</span><span class="author hidden" style="display: none;">, Hurles ME</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2016.		Distinct genetic architectures for syndromic and nonsyndromic congenital heart defects identified by exome sequencing.		<span class="journal-title">Nat Genet</span>,		<span class="journal-volume">48</span>		(9),		pp. 1060-1065.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1038/ng.3627" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-637275" class="pub_abstract" style="display: none;">
		Congenital heart defects (CHDs) have a neonatal incidence of 0.8-1% (refs. 1,2). Despite abundant examples of monogenic CHD in humans and mice, CHD has a low absolute sibling recurrence risk (∼2.7%), suggesting a considerable role for de novo mutations (DNMs) and/or incomplete penetrance. De novo protein-truncating variants (PTVs) have been shown to be enriched among the 10% of 'syndromic' patients with extra-cardiac manifestations. We exome sequenced 1,891 probands, including both syndromic CHD (S-CHD, n = 610) and nonsyndromic CHD (NS-CHD, n = 1,281). In S-CHD, we confirmed a significant enrichment of de novo PTVs but not inherited PTVs in known CHD-associated genes, consistent with recent findings. Conversely, in NS-CHD we observed significant enrichment of PTVs inherited from unaffected parents in CHD-associated genes. We identified three genome-wide significant S-CHD disorders caused by DNMs in CHD4, CDK13 and PRKD1. Our study finds evidence for distinct genetic architectures underlying the low sibling recurrence risk in S-CHD and NS-CHD.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="637964" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1038/mt.2016.153" data-uuid="45d1c626-b4e4-94ec-9682-d0e6a3339454"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=10249757" style="display:inline-block;">
  <img alt="Article has an altmetric score of 5" src="https://badges.altmetric.com/?size=128&amp;score=5&amp;types=tttttttt" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">Al Haj Zen A</span><span class="author ">, Nawrot DA</span><span class="author ">, Howarth A</span><span class="author ">, Caporali A</span><span class="author ">, Ebner D</span><span class="author ">, Vernet A</span><span class="author ">, Schneider JE</span><span class="author ">, Bhattacharya S</span>.</span>		2016.		The Retinoid Agonist Tazarotene Promotes Angiogenesis and Wound Healing.		<span class="journal-title">Mol Ther</span>,		<span class="journal-volume">24</span>		(10),		pp. 1745-1759.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1038/mt.2016.153" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-637964" class="pub_abstract" style="display: none;">
		Therapeutic angiogenesis is a major goal of regenerative medicine, but no clinically approved small molecule exists that enhances new blood vessel formation. Here we show, using a phenotype-driven high-content imaging screen of an annotated chemical library of 1,280 bioactive small molecules, that the retinoid agonist Tazarotene, enhances in vitro angiogenesis, promoting branching morphogenesis, and tubule remodeling. The proangiogenic phenotype is mediated by retinoic acid receptor but not retinoic X receptor activation, and is characterized by secretion of the proangiogenic factors hepatocyte growth factor, vascular endothelial growth factor, plasminogen activator, urokinase and placental growth factor, and reduced secretion of the antiangiogenic factor pentraxin-3 from adjacent fibroblasts. In vivo, Tazarotene enhanced the growth of mature and functional microvessels in Matrigel implants and wound healing models, and increased blood flow. Notably, in ear punch wound healing model, Tazarotene promoted tissue repair characterized by rapid ear punch closure with normal-appearing skin containing new hair follicles, and maturing collagen fibers. Our study suggests that Tazarotene, an FDA-approved small molecule, could be potentially exploited for therapeutic applications in neovascularization and wound healing.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="629267" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1242/dev.123935" data-uuid="d015f776-702c-0e6a-bafe-3e57ff443f8f"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=8881433" style="display:inline-block;">
  <img alt="Article has an altmetric score of 16" src="https://badges.altmetric.com/?size=128&amp;score=16&amp;types=bbtttttf" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">Grijzenhout A</span><span class="author ">, Godwin J</span><span class="author ">, Koseki H</span><span class="author ">, Gdula MR</span><span class="author ">, Szumska D</span><span class="author ">, McGouran JF</span><span class="author ">, Bhattacharya S</span><span class="author ">, Kessler BM</span><span class="author ">, Brockdorff N</span><span class="author ">, Cooper S</span>.</span>		2016.		Functional analysis of AEBP2, a PRC2 Polycomb protein, reveals a Trithorax phenotype in embryonic development and in ESCs.		<span class="journal-title">Development</span>,		<span class="journal-volume">143</span>		(15),		pp. 2716-2723.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1242/dev.123935" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-629267" class="pub_abstract" style="display: none;">
		The Polycomb repressive complexes PRC1 and PRC2 are key mediators of heritable gene silencing in multicellular organisms. Here, we characterise AEBP2, a known PRC2 co-factor which, in vitro, has been shown to stimulate PRC2 activity. We show that AEBP2 localises specifically to PRC2 target loci, including the inactive X chromosome. Proteomic analysis confirms that AEBP2 associates exclusively with PRC2 complexes. However, analysis of embryos homozygous for a targeted mutation of Aebp2 unexpectedly revealed a Trithorax phenotype, normally linked to antagonism of Polycomb function. Consistent with this, we observe elevated levels of PRC2-mediated histone H3K27 methylation at target loci in Aebp2 mutant embryonic stem cells (ESCs). We further demonstrate that mutant ESCs assemble atypical hybrid PRC2 subcomplexes, potentially accounting for enhancement of Polycomb activity, and suggesting that AEBP2 normally plays a role in defining the mutually exclusive composition of PRC2 subcomplexes.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="611571" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1093/hmg/ddw071" data-uuid="07325ecb-bd72-2530-fafc-a216db67dc3a"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=6175878" style="display:inline-block;">
  <img alt="Article has an altmetric score of 2" src="https://badges.altmetric.com/?size=128&amp;score=2&amp;types=tttttttt" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">Hanchard NA</span><span class="author ">, Swaminathan S</span><span class="author ">, Bucasas K</span><span class="author ">, Furthner D</span><span class="author ">, Fernbach S</span><span class="author ">, Azamian MS</span><span class="author ">, Wang X</span><span class="author ">, Lewin M</span><span class="author ">, Towbin JA</span><span class="author ">, D'Alessandro LC</span><span class="author hidden" style="display: none;">, Morris SA</span><span class="author hidden" style="display: none;">, Dreyer W</span><span class="author hidden" style="display: none;">, Denfield S</span><span class="author hidden" style="display: none;">, Ayres NA</span><span class="author hidden" style="display: none;">, Franklin WJ</span><span class="author hidden" style="display: none;">, Justino H</span><span class="author hidden" style="display: none;">, Lantin-Hermoso MR</span><span class="author hidden" style="display: none;">, Ocampo EC</span><span class="author hidden" style="display: none;">, Santos AB</span><span class="author hidden" style="display: none;">, Parekh D</span><span class="author hidden" style="display: none;">, Moodie D</span><span class="author hidden" style="display: none;">, Jeewa A</span><span class="author hidden" style="display: none;">, Lawrence E</span><span class="author hidden" style="display: none;">, Allen HD</span><span class="author hidden" style="display: none;">, Penny DJ</span><span class="author hidden" style="display: none;">, Fraser CD</span><span class="author hidden" style="display: none;">, Lupski JR</span><span class="author hidden" style="display: none;">, Popoola M</span><span class="author hidden" style="display: none;">, Wadhwa L</span><span class="author hidden" style="display: none;">, Brook JD</span><span class="author hidden" style="display: none;">, Bu'Lock FA</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Lalani SR</span><span class="author hidden" style="display: none;">, Zender GA</span><span class="author hidden" style="display: none;">, Fitzgerald-Butt SM</span><span class="author hidden" style="display: none;">, Bowman J</span><span class="author hidden" style="display: none;">, Corsmeier D</span><span class="author hidden" style="display: none;">, White P</span><span class="author hidden" style="display: none;">, Lecerf K</span><span class="author hidden" style="display: none;">, Zapata G</span><span class="author hidden" style="display: none;">, Hernandez P</span><span class="author hidden" style="display: none;">, Goodship JA</span><span class="author hidden" style="display: none;">, Garg V</span><span class="author hidden" style="display: none;">, Keavney BD</span><span class="author hidden" style="display: none;">, Leal SM</span><span class="author hidden" style="display: none;">, Cordell HJ</span><span class="author hidden" style="display: none;">, Belmont JW</span><span class="author hidden" style="display: none;">, McBride KL</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2016.		A genome-wide association study of congenital cardiovascular left-sided lesions shows association with a locus on chromosome 20.		<span class="journal-title">Hum Mol Genet</span>,		<span class="journal-volume">25</span>		(11),		pp. 2331-2341.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1093/hmg/ddw071" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-611571" class="pub_abstract" style="display: none;">
		Congenital heart defects involving left-sided lesions (LSLs) are relatively common birth defects with substantial morbidity and mortality. Previous studies have suggested a high heritability with a complex genetic architecture, such that only a few LSL loci have been identified. We performed a genome-wide case-control association study to address the role of common variants using a discovery cohort of 778 cases and 2756 controls. We identified a genome-wide significant association mapping to a 200 kb region on chromosome 20q11 [P= 1.72 × 10(-8) for rs3746446; imputed Single Nucleotide Polymorphism (SNP) rs6088703 P= 3.01 × 10(-9), odds ratio (OR)= 1.6 for both]. This result was supported by transmission disequilibrium analyses using a subset of 541 case families (lowest P in region= 4.51 × 10(-5), OR= 1.5). Replication in a cohort of 367 LSL cases and 5159 controls showed nominal association (P= 0.03 for rs3746446) resulting in P= 9.49 × 10(-9) for rs3746446 upon meta-analysis of the combined cohorts. In addition, a group of seven SNPs on chromosome 1q21.3 met threshold for suggestive association (lowest P= 9.35 × 10(-7) for rs12045807). Both regions include genes involved in cardiac development-MYH7B/miR499A on chromosome 20 and CTSK, CTSS and ARNT on chromosome 1. Genome-wide heritability analysis using case-control genotyped SNPs suggested that the mean heritability of LSLs attributable to common variants is moderately high ([Formula: see text] range= 0.26-0.34) and consistent with previous assertions. These results provide evidence for the role of common variation in LSLs, proffer new genes as potential biological candidates, and give further insight to the complex genetic architecture of congenital heart disease.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="619932" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1093/nar/gkv1138" data-uuid="8cd92bb2-6d49-7e6e-9649-91f92b327313"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=5022343" style="display:inline-block;">
  <img alt="Article has an altmetric score of 2" src="https://badges.altmetric.com/?size=128&amp;score=2&amp;types=tttttttt" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">Wilson R</span><span class="author ">, McGuire C</span><span class="author ">, Mohun T</span><span class="author ">, DMDD Project</span>.</span>		2016.		Deciphering the mechanisms of developmental disorders: phenotype analysis of embryos from mutant mouse lines.		<span class="journal-title">Nucleic Acids Res</span>,		<span class="journal-volume">44</span>		(D1),		pp. D855-D861.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1093/nar/gkv1138" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-619932" class="pub_abstract" style="display: none;">
		The Deciphering the Mechanisms of Developmental Disorders (DMDD) consortium is a research programme set up to identify genes in the mouse, which if mutated (or knocked-out) result in embryonic lethality when homozygous, and initiate the study of why disruption of their function has such profound effects on embryo development and survival. The project uses a combination of comprehensive high resolution 3D imaging and tissue histology to identify abnormalities in embryo and placental structures of embryonic lethal lines. The image data we have collected and the phenotypes scored are freely available through the project website (http://dmdd.org.uk). In this article we describe the web interface to the images that allows the embryo data to be viewed at full resolution in different planes, discuss how to search the database for a phenotype, and our approach to organising the data for an embryo and a mutant line so it is easy to comprehend and intuitive to navigate.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="630054" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1038/srep28210" data-uuid="29bbe22a-0095-94f9-89be-91160fb63c89"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=8969412" style="display:inline-block;">
  <img alt="Article has an altmetric score of 3" src="https://badges.altmetric.com/?size=128&amp;score=3&amp;types=tttttttt" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">Faherty N</span><span class="author ">, Benson M</span><span class="author ">, Sharma E</span><span class="author ">, Lee A</span><span class="author ">, Howarth A</span><span class="author ">, Lockstone H</span><span class="author ">, Ebner D</span><span class="author ">, Bhattacharya S</span>.</span>		2016.		Negative autoregulation of BMP dependent transcription by SIN3B splicing reveals a role for RBM39.		<span class="journal-title">Sci Rep</span>,		<span class="journal-volume">6</span>		(1),		pp. 28210.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1038/srep28210" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-630054" class="pub_abstract" style="display: none;">
		BMP signalling is negatively autoregulated by several genes including SMAD6, Noggin and Gremlin, and autoregulators are possible targets for enhancing BMP signalling in disorders such as fibrosis and pulmonary hypertension. To identify novel negative regulators of BMP signalling, we used siRNA screening in mouse C2C12 cells with a BMP-responsive luciferase reporter. Knockdown of several splicing factors increased BMP4-dependent transcription and target gene expression. Knockdown of RBM39 produced the greatest enhancement in BMP activity. Transcriptome-wide RNA sequencing identified a change in Sin3b exon usage after RBM39 knockdown. SIN3B targets histone deacetylases to chromatin to repress transcription. In mouse, Sin3b produces long and short isoforms, with the short isoform lacking the ability to recruit HDACs. BMP4 induced a shift in SIN3B expression to the long isoform, and this change in isoform ratio was prevented by RBM39 knockdown. Knockdown of long isoform SIN3B enhanced BMP4-dependent transcription, whereas knockdown of the short isoform did not. We propose that BMP4-dependent transcription is negatively autoregulated in part by SIN3B alternative splicing, and that RBM39 plays a role in this process.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="579365" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1002/humu.22901" data-uuid="2e656598-d2c8-d4f5-06bb-02534170cae3"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=4494365" style="display:inline-block;">
  <img alt="Article has an altmetric score of 3" src="https://badges.altmetric.com/?size=128&amp;score=3&amp;types=ttttt111" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">24</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/26350204#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Grozeva D</span><span class="author ">, Carss K</span><span class="author ">, Spasic-Boskovic O</span><span class="author ">, Tejada MI</span><span class="author ">, Gecz J</span><span class="author ">, Shaw M</span><span class="author ">, Corbett M</span><span class="author ">, Haan E</span><span class="author ">, Thompson E</span><span class="author ">, Friend K</span><span class="author hidden" style="display: none;">, Hussain Z</span><span class="author hidden" style="display: none;">, Hackett A</span><span class="author hidden" style="display: none;">, Field M</span><span class="author hidden" style="display: none;">, Renieri A</span><span class="author hidden" style="display: none;">, Stevenson R</span><span class="author hidden" style="display: none;">, Schwartz C</span><span class="author hidden" style="display: none;">, Floyd JA</span><span class="author hidden" style="display: none;">, Bentham J</span><span class="author hidden" style="display: none;">, Cosgrove C</span><span class="author hidden" style="display: none;">, Keavney B</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Italian X-linked Mental Retardation Project</span><span class="author hidden" style="display: none;">, UK10K Consortium</span><span class="author hidden" style="display: none;">, GOLD Consortium</span><span class="author hidden" style="display: none;">, Hurles M</span><span class="author hidden" style="display: none;">, Raymond FL</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2015.		Targeted Next-Generation Sequencing Analysis of 1,000 Individuals with Intellectual Disability.		<span class="journal-title">Hum Mutat</span>,		<span class="journal-volume">36</span>		(12),		pp. 1197-1204.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1002/humu.22901" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-579365" class="pub_abstract" style="display: none;">
		To identify genetic causes of intellectual disability (ID), we screened a cohort of 986 individuals with moderate to severe ID for variants in 565 known or candidate ID-associated genes using targeted next-generation sequencing. Likely pathogenic rare variants were found in ∼11% of the cases (113 variants in 107/986 individuals: ∼8% of the individuals had a likely pathogenic loss-of-function [LoF] variant, whereas ∼3% had a known pathogenic missense variant). Variants in SETD5, ATRX, CUL4B, MECP2, and ARID1B were the most common causes of ID. This study assessed the value of sequencing a cohort of probands to provide a molecular diagnosis of ID, without the availability of DNA from both parents for de novo sequence analysis. This modeling is clinically relevant as 28% of all UK families with dependent children are single parent households. In conclusion, to diagnose patients with ID in the absence of parental DNA, we recommend investigation of all LoF variants in known genes that cause ID and assessment of a limited list of proven pathogenic missense variants in these genes. This will provide 11% additional diagnostic yield beyond the 10%-15% yield from array CGH alone.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="570702" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1038/nature14962" data-uuid="f2b7ec3d-64a5-892d-ffd4-09cb24db892f"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=4506746" style="display:inline-block;">
  <img alt="Article has an altmetric score of 217" src="https://badges.altmetric.com/?size=128&amp;score=217&amp;types=mbtttwfg" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">129</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/26367797#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">UK10K Consortium</span><span class="author ">, Walter K</span><span class="author ">, Min JL</span><span class="author ">, Huang J</span><span class="author ">, Crooks L</span><span class="author ">, Memari Y</span><span class="author ">, McCarthy S</span><span class="author ">, Perry JR</span><span class="author ">, Xu C</span><span class="author ">, Futema M</span><span class="author hidden" style="display: none;">, Lawson D</span><span class="author hidden" style="display: none;">, Iotchkova V</span><span class="author hidden" style="display: none;">, Schiffels S</span><span class="author hidden" style="display: none;">, Hendricks AE</span><span class="author hidden" style="display: none;">, Danecek P</span><span class="author hidden" style="display: none;">, Li R</span><span class="author hidden" style="display: none;">, Floyd J</span><span class="author hidden" style="display: none;">, Wain LV</span><span class="author hidden" style="display: none;">, Barroso I</span><span class="author hidden" style="display: none;">, Humphries SE</span><span class="author hidden" style="display: none;">, Hurles ME</span><span class="author hidden" style="display: none;">, Zeggini E</span><span class="author hidden" style="display: none;">, Barrett JC</span><span class="author hidden" style="display: none;">, Plagnol V</span><span class="author hidden" style="display: none;">, Richards JB</span><span class="author hidden" style="display: none;">, Greenwood CM</span><span class="author hidden" style="display: none;">, Timpson NJ</span><span class="author hidden" style="display: none;">, Durbin R</span><span class="author hidden" style="display: none;">, Soranzo N</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2015.		The UK10K project identifies rare variants in health and disease.		<span class="journal-title">Nature</span>,		<span class="journal-volume">526</span>		(7571),		pp. 82-90.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1038/nature14962" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-570702" class="pub_abstract" style="display: none;">
		The contribution of rare and low-frequency variants to human traits is largely unexplored. Here we describe insights from sequencing whole genomes (low read depth, 7×) or exomes (high read depth, 80×) of nearly 10,000 individuals from population-based and disease collections. In extensively phenotyped cohorts we characterize over 24 million novel sequence variants, generate a highly accurate imputation reference panel and identify novel alleles associated with levels of triglycerides (APOB), adiponectin (ADIPOQ) and low-density lipoprotein cholesterol (LDLR and RGAG1) from single-marker and rare variant aggregation tests. We describe population structure and functional annotation of rare and low-frequency variants, use the data to estimate the benefits of sequencing for association studies, and summarize lessons from disease-specific collections. Finally, we make available an extensive resource, including individual-level genetic and phenotypic data and web-based tools to facilitate the exploration of association results.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="567358" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1038/ncomms9111" data-uuid="67a8f0d0-a19e-15a7-6c5f-c0ee5eab7b0d"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=4506769" style="display:inline-block;">
  <img alt="Article has an altmetric score of 66" src="https://badges.altmetric.com/?size=128&amp;score=66&amp;types=mbtttttf" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">27</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/26368830#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Huang J</span><span class="author ">, Howie B</span><span class="author ">, McCarthy S</span><span class="author ">, Memari Y</span><span class="author ">, Walter K</span><span class="author ">, Min JL</span><span class="author ">, Danecek P</span><span class="author ">, Malerba G</span><span class="author ">, Trabetti E</span><span class="author ">, Zheng HF</span><span class="author hidden" style="display: none;">, UK10K Consortium</span><span class="author hidden" style="display: none;">, Gambaro G</span><span class="author hidden" style="display: none;">, Richards JB</span><span class="author hidden" style="display: none;">, Durbin R</span><span class="author hidden" style="display: none;">, Timpson NJ</span><span class="author hidden" style="display: none;">, Marchini J</span><span class="author hidden" style="display: none;">, Soranzo N</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2015.		Improved imputation of low-frequency and rare variants using the UK10K haplotype reference panel.		<span class="journal-title">Nat Commun</span>,		<span class="journal-volume">6</span>				pp. 8111.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1038/ncomms9111" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-567358" class="pub_abstract" style="display: none;">
		Imputing genotypes from reference panels created by whole-genome sequencing (WGS) provides a cost-effective strategy for augmenting the single-nucleotide polymorphism (SNP) content of genome-wide arrays. The UK10K Cohorts project has generated a data set of 3,781 whole genomes sequenced at low depth (average 7x), aiming to exhaustively characterize genetic variation down to 0.1% minor allele frequency in the British population. Here we demonstrate the value of this resource for improving imputation accuracy at rare and low-frequency variants in both a UK and an Italian population. We show that large increases in imputation accuracy can be achieved by re-phasing WGS reference panels after initial genotype calling. We also present a method for combining WGS panels to improve variant coverage and downstream imputation accuracy, which we illustrate by integrating 7,562 WGS haplotypes from the UK10K project with 2,184 haplotypes from the 1000 Genomes Project. Finally, we introduce a novel approximation that maintains speed without sacrificing imputation accuracy for rare variants.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="666964" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1186/2046-2530-4-S1-P37" data-uuid="36ed6b7a-244a-63a7-45d0-79a5f644cd72"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1186/2046-2530-4-s1-p37" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Leordean D</span><span class="author ">, Grimes D</span><span class="author ">, Keynton J</span><span class="author ">, Maier J</span><span class="author ">, Harfe B</span><span class="author ">, Benson M</span><span class="author ">, Gray A</span><span class="author ">, Bhattacharya S</span><span class="author ">, Norris D</span>.</span>		2015.		FOXA2 controls Pkd1l1 expression in the mouse node during left-right determination		<span class="journal-title">Cilia</span>,		<span class="journal-volume">4</span>		(Suppl 1),		pp. P37-P37.				 | <a href="http://dx.doi.org/10.1186/2046-2530-4-S1-P37" target="_blank">Read more</a> 		<p class="clearfix"></p>
				</div></div>
						<div class="publication group all" symplectic_id="531655" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.7554/eLife.06942" data-uuid="92469f5c-46b4-3ce2-97f9-c6f0bba09f07"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=4247734" style="display:inline-block;">
  <img alt="Article has an altmetric score of 7" src="https://badges.altmetric.com/?size=128&amp;score=7&amp;types=tttttttt" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">Bouveret R</span><span class="author ">, Waardenberg AJ</span><span class="author ">, Schonrock N</span><span class="author ">, Ramialison M</span><span class="author ">, Doan T</span><span class="author ">, de Jong D</span><span class="author ">, Bondue A</span><span class="author ">, Kaur G</span><span class="author ">, Mohamed S</span><span class="author ">, Fonoudi H</span><span class="author hidden" style="display: none;">, Chen CM</span><span class="author hidden" style="display: none;">, Wouters MA</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Plachta N</span><span class="author hidden" style="display: none;">, Dunwoodie SL</span><span class="author hidden" style="display: none;">, Chapman G</span><span class="author hidden" style="display: none;">, Blanpain C</span><span class="author hidden" style="display: none;">, Harvey RP</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2015.		NKX2-5 mutations causative for congenital heart disease retain functionality and are directed to hundreds of targets.		<span class="journal-title">Elife</span>,		<span class="journal-volume">4</span>		(JULY2015),				 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.7554/eLife.06942" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-531655" class="pub_abstract" style="display: none;">
		We take a functional genomics approach to congenital heart disease mechanism. We used DamID to establish a robust set of target genes for NKX2-5 wild type and disease associated NKX2-5 mutations to model loss-of-function in gene regulatory networks. NKX2-5 mutants, including those with a crippled homeodomain, bound hundreds of targets including NKX2-5 wild type targets and a unique set of "off-targets", and retained partial functionality. NKXΔHD, which lacks the homeodomain completely, could heterodimerize with NKX2-5 wild type and its cofactors, including E26 transformation-specific (ETS) family members, through a tyrosine-rich homophilic interaction domain (YRD). Off-targets of NKX2-5 mutants, but not those of an NKX2-5 YRD mutant, showed overrepresentation of ETS binding sites and were occupied by ETS proteins, as determined by DamID. Analysis of kernel transcription factor and ETS targets show that ETS proteins are highly embedded within the cardiac gene regulatory network. Our study reveals binding and activities of NKX2-5 mutations on WT target and off-targets, guided by interactions with their normal cardiac and general cofactors, and suggest a novel type of gain-of-function in congenital heart disease.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="527775" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1038/ncomms8074" data-uuid="fd1fc363-cc8c-ac88-af75-4e8f2a7dbc4c"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=4100719" style="display:inline-block;">
  <img alt="Article has an altmetric score of 11" src="https://badges.altmetric.com/?size=128&amp;score=11&amp;types=mttttttt" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">Schmidts M</span><span class="author ">, Hou Y</span><span class="author ">, Cortés CR</span><span class="author ">, Mans DA</span><span class="author ">, Huber C</span><span class="author ">, Boldt K</span><span class="author ">, Patel M</span><span class="author ">, van Reeuwijk J</span><span class="author ">, Plaza JM</span><span class="author ">, van Beersum SE</span><span class="author hidden" style="display: none;">, Yap ZM</span><span class="author hidden" style="display: none;">, Letteboer SJ</span><span class="author hidden" style="display: none;">, Taylor SP</span><span class="author hidden" style="display: none;">, Herridge W</span><span class="author hidden" style="display: none;">, Johnson CA</span><span class="author hidden" style="display: none;">, Scambler PJ</span><span class="author hidden" style="display: none;">, Ueffing M</span><span class="author hidden" style="display: none;">, Kayserili H</span><span class="author hidden" style="display: none;">, Krakow D</span><span class="author hidden" style="display: none;">, King SM</span><span class="author hidden" style="display: none;">, UK10K</span><span class="author hidden" style="display: none;">, Beales PL</span><span class="author hidden" style="display: none;">, Al-Gazali L</span><span class="author hidden" style="display: none;">, Wicking C</span><span class="author hidden" style="display: none;">, Cormier-Daire V</span><span class="author hidden" style="display: none;">, Roepman R</span><span class="author hidden" style="display: none;">, Mitchison HM</span><span class="author hidden" style="display: none;">, Witman GB</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2015.		TCTEX1D2 mutations underlie Jeune asphyxiating thoracic dystrophy with impaired retrograde intraflagellar transport.		<span class="journal-title">Nat Commun</span>,		<span class="journal-volume">6</span>				pp. 7074.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1038/ncomms8074" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-527775" class="pub_abstract" style="display: none;">
		The analysis of individuals with ciliary chondrodysplasias can shed light on sensitive mechanisms controlling ciliogenesis and cell signalling that are essential to embryonic development and survival. Here we identify TCTEX1D2 mutations causing Jeune asphyxiating thoracic dystrophy with partially penetrant inheritance. Loss of TCTEX1D2 impairs retrograde intraflagellar transport (IFT) in humans and the protist Chlamydomonas, accompanied by destabilization of the retrograde IFT dynein motor. We thus define TCTEX1D2 as an integral component of the evolutionarily conserved retrograde IFT machinery. In complex with several IFT dynein light chains, it is required for correct vertebrate skeletal formation but may be functionally redundant under certain conditions.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="525440" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1038/gim.2015.60" data-uuid="92ea467b-2516-ebbd-c008-bcd53c715b46"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=4051430" style="display:inline-block;">
  <img alt="Article has an altmetric score of 4" src="https://badges.altmetric.com/?size=128&amp;score=4&amp;types=ttttttff" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">D'Alessandro LC</span><span class="author ">, Al Turki S</span><span class="author ">, Manickaraj AK</span><span class="author ">, Manase D</span><span class="author ">, Mulder BJ</span><span class="author ">, Bergin L</span><span class="author ">, Rosenberg HC</span><span class="author ">, Mondal T</span><span class="author ">, Gordon E</span><span class="author ">, Lougheed J</span><span class="author hidden" style="display: none;">, Smythe J</span><span class="author hidden" style="display: none;">, Devriendt K</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Watkins H</span><span class="author hidden" style="display: none;">, Bentham J</span><span class="author hidden" style="display: none;">, Bowdin S</span><span class="author hidden" style="display: none;">, Hurles ME</span><span class="author hidden" style="display: none;">, Mital S</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2016.		Exome sequencing identifies rare variants in multiple genes in atrioventricular septal defect.		<span class="journal-title">Genet Med</span>,		<span class="journal-volume">18</span>		(2),		pp. 189-198.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1038/gim.2015.60" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-525440" class="pub_abstract" style="display: none;">
		PURPOSE: The genetic etiology of atrioventricular septal defect (AVSD) is unknown in 40% cases. Conventional sequencing and arrays have identified the etiology in only a minority of nonsyndromic individuals with AVSD. METHODS: Whole-exome sequencing was performed in 81 unrelated probands with AVSD to identify potentially causal variants in a comprehensive set of 112 genes with strong biological relevance to AVSD. RESULTS: A significant enrichment of rare and rare damaging variants was identified in the gene set, compared with controls (odds ratio (OR): 1.52; 95% confidence interval (CI): 1.35-1.71; P = 4.8 × 10(-11)). The enrichment was specific to AVSD probands, compared with a cohort without AVSD with tetralogy of Fallot (OR: 2.25; 95% CI: 1.84-2.76; P = 2.2 × 10(-16)). Six genes (NIPBL, CHD7, CEP152, BMPR1a, ZFPM2, and MDM4) were enriched for rare variants in AVSD compared with controls, including three syndrome-associated genes (NIPBL, CHD7, and CEP152). The findings were confirmed in a replication cohort of 81 AVSD probands. CONCLUSION: Mutations in genes with strong biological relevance to AVSD, including syndrome-associated genes, can contribute to AVSD, even in those with isolated heart disease. The identification of a gene set associated with AVSD will facilitate targeted genetic screening in this cohort.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="510471" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1038/ncomms6681" data-uuid="8dff8382-2031-4a22-6c8e-935cf3e8a976"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=3761380" style="display:inline-block;">
  <img alt="Article has an altmetric score of 28" src="https://badges.altmetric.com/?size=128&amp;score=28&amp;types=mssttttf" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">22</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/25743335#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Taylor PN</span><span class="author ">, Porcu E</span><span class="author ">, Chew S</span><span class="author ">, Campbell PJ</span><span class="author ">, Traglia M</span><span class="author ">, Brown SJ</span><span class="author ">, Mullin BH</span><span class="author ">, Shihab HA</span><span class="author ">, Min J</span><span class="author ">, Walter K</span><span class="author hidden" style="display: none;">, Memari Y</span><span class="author hidden" style="display: none;">, Huang J</span><span class="author hidden" style="display: none;">, Barnes MR</span><span class="author hidden" style="display: none;">, Beilby JP</span><span class="author hidden" style="display: none;">, Charoen P</span><span class="author hidden" style="display: none;">, Danecek P</span><span class="author hidden" style="display: none;">, Dudbridge F</span><span class="author hidden" style="display: none;">, Forgetta V</span><span class="author hidden" style="display: none;">, Greenwood C</span><span class="author hidden" style="display: none;">, Grundberg E</span><span class="author hidden" style="display: none;">, Johnson AD</span><span class="author hidden" style="display: none;">, Hui J</span><span class="author hidden" style="display: none;">, Lim EM</span><span class="author hidden" style="display: none;">, McCarthy S</span><span class="author hidden" style="display: none;">, Muddyman D</span><span class="author hidden" style="display: none;">, Panicker V</span><span class="author hidden" style="display: none;">, Perry JR</span><span class="author hidden" style="display: none;">, Bell JT</span><span class="author hidden" style="display: none;">, Yuan W</span><span class="author hidden" style="display: none;">, Relton C</span><span class="author hidden" style="display: none;">, Gaunt T</span><span class="author hidden" style="display: none;">, Schlessinger D</span><span class="author hidden" style="display: none;">, Abecasis G</span><span class="author hidden" style="display: none;">, Cucca F</span><span class="author hidden" style="display: none;">, Surdulescu GL</span><span class="author hidden" style="display: none;">, Woltersdorf W</span><span class="author hidden" style="display: none;">, Zeggini E</span><span class="author hidden" style="display: none;">, Zheng HF</span><span class="author hidden" style="display: none;">, Toniolo D</span><span class="author hidden" style="display: none;">, Dayan CM</span><span class="author hidden" style="display: none;">, Naitza S</span><span class="author hidden" style="display: none;">, Walsh JP</span><span class="author hidden" style="display: none;">, Spector T</span><span class="author hidden" style="display: none;">, Davey Smith G</span><span class="author hidden" style="display: none;">, Durbin R</span><span class="author hidden" style="display: none;">, Richards JB</span><span class="author hidden" style="display: none;">, Sanna S</span><span class="author hidden" style="display: none;">, Soranzo N</span><span class="author hidden" style="display: none;">, Timpson NJ</span><span class="author hidden" style="display: none;">, Wilson SG</span><span class="author hidden" style="display: none;">, UK0K Consortium</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2015.		Whole-genome sequence-based analysis of thyroid function.		<span class="journal-title">Nat Commun</span>,		<span class="journal-volume">6</span>				pp. 5681.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1038/ncomms6681" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-510471" class="pub_abstract" style="display: none;">
		Normal thyroid function is essential for health, but its genetic architecture remains poorly understood. Here, for the heritable thyroid traits thyrotropin (TSH) and free thyroxine (FT4), we analyse whole-genome sequence data from the UK10K project (N=2,287). Using additional whole-genome sequence and deeply imputed data sets, we report meta-analysis results for common variants (MAF≥1%) associated with TSH and FT4 (N=16,335). For TSH, we identify a novel variant in SYN2 (MAF=23.5%, P=6.15 × 10(-9)) and a new independent variant in PDE8B (MAF=10.4%, P=5.94 × 10(-14)). For FT4, we report a low-frequency variant near B4GALT6/SLC25A52 (MAF=3.2%, P=1.27 × 10(-9)) tagging a rare TTR variant (MAF=0.4%, P=2.14 × 10(-11)). All common variants explain ≥20% of the variance in TSH and FT4. Analysis of rare variants (MAF&lt;1%) using sequence kernel association testing reveals a novel association with FT4 in NRG1. Our results demonstrate that increased coverage in whole-genome sequence association studies identifies novel variants associated with thyroid function.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="463588" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1016/j.jss.2014.01.015" data-uuid="5fd068ca-3342-18f2-2bef-bf86385ac936"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1016/j.jss.2014.01.015" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Ji W</span><span class="author ">, Benson MA</span><span class="author ">, Bhattacharya S</span><span class="author ">, Chen Y</span><span class="author ">, Hu J</span><span class="author ">, Li F</span>.</span>		2014.		Characterization of transcription factor AP-2 beta mutations involved in familial isolated patent ductus arteriosus suggests haploinsufficiency		<span class="journal-title">Journal of Surgical Research</span>,		<span class="journal-volume">188</span>		(2),		pp. 466-472.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1016/j.jss.2014.01.015" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-463588" class="pub_abstract" style="display: none;">
		Background Patent ductus arteriosus (PDA) is one of the most common congenital heart defects. Transcription factor AP-2 beta (TFAP2B) mutations are associated with the Char syndrome, a disorder associated with PDA, and with facial and fingers abnormalities. Recently, we identified two TFAP2B mutations in two families without Char syndrome phenotype, c.601+5G&gt;A and c.435-438delCCGG, and these TFAP2B mutations were associated with familial isolated PDA. The aim of this study was to identify the effects of these mutations on TFAP2B function. Methods Plasmids containing the wild-type or mutated TFAP2B were constructed and transfected in cells. Plasmids containing the TFAP2B coactivator, Cpb/p300-interacting transactivator 2 (CITED2), was also transfected. TFAP2B expression was detected by luciferase expression and by Western blot analysis. Results These mutations resulted in loss of transactivation function, which could not be improved by Cpb/p300-interacting transactivator 2. The c.601+5G&gt;A mutated gene did not express any protein, whereas the c.435-438delCCGG mutation did not impact the transactivation function activated by the wild-type TFAP2B. Conclusions These results suggest that a haploinsufficiency effect of TFAP2B could be involved in familial isolated PDA. © 2014 Elsevier Inc. All rights reserved.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="463294" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1161/CIRCRESAHA.115.303300" data-uuid="fe19e698-d15a-134b-e34c-050bf1b47f7d"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=2464781" style="display:inline-block;">
  <img alt="Article has an altmetric score of 1" src="https://badges.altmetric.com/?size=128&amp;score=1&amp;types=tttttttt" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">Dykes IM</span><span class="author ">, van Bueren KL</span><span class="author ">, Ashmore RJ</span><span class="author ">, Floss T</span><span class="author ">, Wurst W</span><span class="author ">, Szumska D</span><span class="author ">, Bhattacharya S</span><span class="author ">, Scambler PJ</span>.</span>		2014.		HIC2 is a novel dosage-dependent regulator of cardiac development located within the distal 22q11 deletion syndrome region.		<span class="journal-title">Circ Res</span>,		<span class="journal-volume">115</span>		(1),		pp. 23-31.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1161/CIRCRESAHA.115.303300" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-463294" class="pub_abstract" style="display: none;">
		RATIONALE: 22q11 deletion syndrome arises from recombination between low-copy repeats on chromosome 22. Typical deletions result in hemizygosity for TBX1 associated with congenital cardiovascular disease. Deletions distal to the typically deleted region result in a similar cardiac phenotype but lack in extracardiac features of the syndrome, suggesting that a second haploinsufficient gene maps to this interval. OBJECTIVE: The transcription factor HIC2 is lost in most distal deletions, as well as in a minority of typical deletions. We used mouse models to test the hypothesis that HIC2 hemizygosity causes congenital heart disease. METHODS AND RESULTS: We created a genetrap mouse allele of Hic2. The genetrap reporter was expressed in the heart throughout the key stages of cardiac morphogenesis. Homozygosity for the genetrap allele was embryonic lethal before embryonic day E10.5, whereas the heterozygous condition exhibited a partially penetrant late lethality. One third of heterozygous embryos had a cardiac phenotype. MRI demonstrated a ventricular septal defect with over-riding aorta. Conditional targeting indicated a requirement for Hic2 within the Nkx2.5+ and Mesp1+ cardiovascular progenitor lineages. Microarray analysis revealed increased expression of Bmp10. CONCLUSIONS: Our results demonstrate a novel role for Hic2 in cardiac development. Hic2 is the first gene within the distal 22q11 interval to have a demonstrated haploinsufficient cardiac phenotype in mice. Together our data suggest that HIC2 haploinsufficiency likely contributes to the cardiac defects seen in distal 22q11 deletion syndrome.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="449186" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1016/j.jss.2014.01.015" data-uuid="d48bab0f-303f-1007-c722-85e9770e80fd"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1016/j.jss.2014.01.015" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Ji W</span><span class="author ">, Benson MA</span><span class="author ">, Bhattacharya S</span><span class="author ">, Chen Y</span><span class="author ">, Hu J</span><span class="author ">, Li F</span>.</span>		2014.		Characterization of transcription factor AP-2 β mutations involved in familial isolated patent ductus arteriosus suggests haploinsufficiency.		<span class="journal-title">J Surg Res</span>,		<span class="journal-volume">188</span>		(2),		pp. 466-472.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1016/j.jss.2014.01.015" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-449186" class="pub_abstract" style="display: none;">
		BACKGROUND: Patent ductus arteriosus (PDA) is one of the most common congenital heart defects. Transcription factor AP-2 beta (TFAP2B) mutations are associated with the Char syndrome, a disorder associated with PDA, and with facial and fingers abnormalities. Recently, we identified two TFAP2B mutations in two families without Char syndrome phenotype, c.601+5G&gt;A and c.435_438delCCGG, and these TFAP2B mutations were associated with familial isolated PDA. The aim of this study was to identify the effects of these mutations on TFAP2B function. METHODS: Plasmids containing the wild-type or mutated TFAP2B were constructed and transfected in cells. Plasmids containing the TFAP2B coactivator, Cpb/p300-interacting transactivator 2 (CITED2), was also transfected. TFAP2B expression was detected by luciferase expression and by Western blot analysis. RESULTS: These mutations resulted in loss of transactivation function, which could not be improved by Cpb/p300-interacting transactivator 2. The c.601+5G&gt;A mutated gene did not express any protein, whereas the c.435_438delCCGG mutation did not impact the transactivation function activated by the wild-type TFAP2B. CONCLUSIONS: These results suggest that a haploinsufficiency effect of TFAP2B could be involved in familial isolated PDA.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="484585" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1038/ncomms5871" data-uuid="c77b9e7f-1d28-95e1-81ef-005b4668dfcd"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=2687816" style="display:inline-block;">
  <img alt="Article has an altmetric score of 40" src="https://badges.altmetric.com/?size=128&amp;score=40&amp;types=mtttttfg" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">24</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/25225788#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Timpson NJ</span><span class="author ">, Walter K</span><span class="author ">, Min JL</span><span class="author ">, Tachmazidou I</span><span class="author ">, Malerba G</span><span class="author ">, Shin SY</span><span class="author ">, Chen L</span><span class="author ">, Futema M</span><span class="author ">, Southam L</span><span class="author ">, Iotchkova V</span><span class="author hidden" style="display: none;">, Cocca M</span><span class="author hidden" style="display: none;">, Huang J</span><span class="author hidden" style="display: none;">, Memari Y</span><span class="author hidden" style="display: none;">, McCarthy S</span><span class="author hidden" style="display: none;">, Danecek P</span><span class="author hidden" style="display: none;">, Muddyman D</span><span class="author hidden" style="display: none;">, Mangino M</span><span class="author hidden" style="display: none;">, Menni C</span><span class="author hidden" style="display: none;">, Perry JR</span><span class="author hidden" style="display: none;">, Ring SM</span><span class="author hidden" style="display: none;">, Gaye A</span><span class="author hidden" style="display: none;">, Dedoussis G</span><span class="author hidden" style="display: none;">, Farmaki AE</span><span class="author hidden" style="display: none;">, Burton P</span><span class="author hidden" style="display: none;">, Talmud PJ</span><span class="author hidden" style="display: none;">, Gambaro G</span><span class="author hidden" style="display: none;">, Spector TD</span><span class="author hidden" style="display: none;">, Smith GD</span><span class="author hidden" style="display: none;">, Durbin R</span><span class="author hidden" style="display: none;">, Richards JB</span><span class="author hidden" style="display: none;">, Humphries SE</span><span class="author hidden" style="display: none;">, Zeggini E</span><span class="author hidden" style="display: none;">, Soranzo N</span><span class="author hidden" style="display: none;">, UK1OK Consortium Members</span><span class="author hidden" style="display: none;">, UK1OK Consortium Members</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2014.		A rare variant in APOC3 is associated with plasma triglyceride and VLDL levels in Europeans.		<span class="journal-title">Nat Commun</span>,		<span class="journal-volume">5</span>				pp. 4871.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1038/ncomms5871" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-484585" class="pub_abstract" style="display: none;">
		The analysis of rich catalogues of genetic variation from population-based sequencing provides an opportunity to screen for functional effects. Here we report a rare variant in APOC3 (rs138326449-A, minor allele frequency ~0.25% (UK)) associated with plasma triglyceride (TG) levels (-1.43 s.d. (s.e.=0.27 per minor allele (P-value=8.0 × 10(-8))) discovered in 3,202 individuals with low read-depth, whole-genome sequence. We replicate this in 12,831 participants from five additional samples of Northern and Southern European origin (-1.0 s.d. (s.e.=0.173), P-value=7.32 × 10(-9)). This is consistent with an effect between 0.5 and 1.5 mmol l(-1) dependent on population. We show that a single predicted splice donor variant is responsible for association signals and is independent of known common variants. Analyses suggest an independent relationship between rs138326449 and high-density lipoprotein (HDL) levels. This represents one of the first examples of a rare, large effect variant identified from whole-genome sequencing at a population scale.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="450969" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1371/journal.pone.0089397" data-uuid="914376f4-40d2-46fd-3986-d7b46a9c29bc"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1371/journal.pone.0089397" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Bard-Chapeau EA</span><span class="author ">, Szumska D</span><span class="author ">, Jacob B</span><span class="author ">, Chua BQ</span><span class="author ">, Chatterjee GC</span><span class="author ">, Zhang Y</span><span class="author ">, Ward JM</span><span class="author ">, Urun F</span><span class="author ">, Kinameri E</span><span class="author ">, Vincent SD</span><span class="author hidden" style="display: none;">, Ahmed S</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Osato M</span><span class="author hidden" style="display: none;">, Perkins AS</span><span class="author hidden" style="display: none;">, Moore AW</span><span class="author hidden" style="display: none;">, Jenkins NA</span><span class="author hidden" style="display: none;">, Copeland NG</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2014.		Mice carrying a hypomorphic Evi1 allele are embryonic viable but exhibit severe congenital heart defects.		<span class="journal-title">PLoS One</span>,		<span class="journal-volume">9</span>		(2),		pp. e89397.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1371/journal.pone.0089397" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-450969" class="pub_abstract" style="display: none;">
		The ecotropic viral integration site 1 (Evi1) oncogenic transcription factor is one of a number of alternative transcripts encoded by the Mds1 and Evi1 complex locus (Mecom). Overexpression of Evi1 has been observed in a number of myeloid disorders and is associated with poor patient survival. It is also amplified and/or overexpressed in many epithelial cancers including nasopharyngeal carcinoma, ovarian carcinoma, ependymomas, and lung and colorectal cancers. Two murine knockout models have also demonstrated Evi1's critical role in the maintenance of hematopoietic stem cell renewal with its absence resulting in the death of mutant embryos due to hematopoietic failure. Here we characterize a novel mouse model (designated Evi1(fl3)) in which Evi1 exon 3, which carries the ATG start, is flanked by loxP sites. Unexpectedly, we found that germline deletion of exon3 produces a hypomorphic allele due to the use of an alternative ATG start site located in exon 4, resulting in a minor Evi1 N-terminal truncation and a block in expression of the Mds1-Evi1 fusion transcript. Evi1(δex3/δex3) mutant embryos showed only a mild non-lethal hematopoietic phenotype and bone marrow failure was only observed in adult Vav-iCre/+, Evi1(fl3/fl3) mice in which exon 3 was specifically deleted in the hematopoietic system. Evi1(δex3/δex3) knockout pups are born in normal numbers but die during the perinatal period from congenital heart defects. Database searches identified 143 genes with similar mutant heart phenotypes as those observed in Evi1(δex3/δex3) mutant pups. Interestingly, 42 of these congenital heart defect genes contain known Evi1-binding sites, and expression of 18 of these genes are also effected by Evi1 siRNA knockdown. These results show a potential functional involvement of Evi1 target genes in heart development and indicate that Evi1 is part of a transcriptional program that regulates cardiac development in addition to the development of blood.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="444671" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1186/1741-7007-11-107" data-uuid="30c3a270-da9e-5f9e-eb0e-3f0e51ccb33e"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=1836875" style="display:inline-block;">
  <img alt="Article has an altmetric score of 1" src="https://badges.altmetric.com/?size=128&amp;score=1&amp;types=tttttttt" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">Dai X</span><span class="author ">, Jiang W</span><span class="author ">, Zhang Q</span><span class="author ">, Xu L</span><span class="author ">, Geng P</span><span class="author ">, Zhuang S</span><span class="author ">, Petrich BG</span><span class="author ">, Jiang C</span><span class="author ">, Peng L</span><span class="author ">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Evans SM</span><span class="author hidden" style="display: none;">, Sun Y</span><span class="author hidden" style="display: none;">, Chen J</span><span class="author hidden" style="display: none;">, Liang X</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2013.		Requirement for integrin-linked kinase in neural crest migration and differentiation and outflow tract morphogenesis.		<span class="journal-title">BMC Biol</span>,		<span class="journal-volume">11</span>		(1),		pp. 107.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1186/1741-7007-11-107" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-444671" class="pub_abstract" style="display: none;">
		BACKGROUND: Neural crest defects lead to congenital heart disease involving outflow tract malformation. Integrin-linked-kinase (ILK) plays important roles in multiple cellular processes and embryogenesis. ILK is expressed in the neural crest, but its role in neural crest and outflow tract morphogenesis remains unknown. RESULTS: We ablated ILK specifically in the neural crest using the Wnt1-Cre transgene. ILK ablation resulted in abnormal migration and overpopulation of neural crest cells in the pharyngeal arches and outflow tract and a significant reduction in the expression of neural cell adhesion molecule (NCAM) and extracellular matrix components. ILK mutant embryos exhibited an enlarged common arterial trunk and ventricular septal defect. Reduced smooth muscle differentiation, but increased ossification and neurogenesis/innervation were observed in ILK mutant outflow tract that may partly be due to reduced transforming growth factor β2 (TGFβ2) but increased bone morphogenetic protein (BMP) signaling. Consistent with these observations, microarray analysis of fluorescence-activated cell sorting (FACS)-sorted neural crest cells revealed reduced expression of genes associated with muscle differentiation, but increased expression of genes of neurogenesis and osteogenesis. CONCLUSIONS: Our results demonstrate that ILK plays essential roles in neural crest and outflow tract development by mediating complex crosstalk between cell matrix and multiple signaling pathways. Changes in these pathways may collectively result in the unique neural crest and outflow tract phenotypes observed in ILK mutants.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="418604" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1074/jbc.M113.493023" data-uuid="9ae5e7bc-1cc6-108a-9515-d92593c341c1"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=1820305" style="display:inline-block;">
  <img alt="Article has an altmetric score of 1" src="https://badges.altmetric.com/?size=128&amp;score=1&amp;types=tttttttt" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">Benson MA</span><span class="author ">, Batchelor H</span><span class="author ">, Chuaiphichai S</span><span class="author ">, Bailey J</span><span class="author ">, Zhu H</span><span class="author ">, Stuehr DJ</span><span class="author ">, Bhattacharya S</span><span class="author ">, Channon KM</span><span class="author ">, Crabtree MJ</span>.</span>		2013.		A pivotal role for tryptophan 447 in enzymatic coupling of human endothelial nitric oxide synthase (eNOS): effects on tetrahydrobiopterin-dependent catalysis and eNOS dimerization.		<span class="journal-title">J Biol Chem</span>,		<span class="journal-volume">288</span>		(41),		pp. 29836-29845.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1074/jbc.M113.493023" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-418604" class="pub_abstract" style="display: none;">
		Tetrahydrobiopterin (BH4) is a required cofactor for the synthesis of NO by NOS. Bioavailability of BH4 is a critical factor in regulating the balance between NO and superoxide production by endothelial NOS (eNOS coupling). Crystal structures of the mouse inducible NOS oxygenase domain reveal a homologous BH4-binding site located in the dimer interface and a conserved tryptophan residue that engages in hydrogen bonding or aromatic stacking interactions with the BH4 ring. The role of this residue in eNOS coupling remains unexplored. We overexpressed human eNOS W447A and W447F mutants in novel cell lines with tetracycline-regulated expression of human GTP cyclohydrolase I, the rate-limiting enzyme in BH4 synthesis, to determine the importance of BH4 and Trp-447 in eNOS uncoupling. NO production was abolished in eNOS-W447A cells and diminished in cells expressing W447F, despite high BH4 levels. eNOS-derived superoxide production was significantly elevated in W447A and W447F versus wild-type eNOS, and this was sufficient to oxidize BH4 to 7,8-dihydrobiopterin. In uncoupled, BH4-deficient cells, the deleterious effects of W447A mutation were greatly exacerbated, resulting in further attenuation of NO and greatly increased superoxide production. eNOS dimerization was attenuated in W447A eNOS cells and further reduced in BH4-deficient cells, as demonstrated using a novel split Renilla luciferase biosensor. Reduction of cellular BH4 levels resulted in a switch from an eNOS dimer to an eNOS monomer. These data reveal a key role for Trp-447 in determining NO versus superoxide production by eNOS, by effects on BH4-dependent catalysis, and by modulating eNOS dimer formation.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="414725" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1161/CIRCGENETICS.113.000191" data-uuid="a9923664-5799-198f-827e-bd19e6108695"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=1705210" style="display:inline-block;">
  <img alt="Article has an altmetric score of 11" src="https://badges.altmetric.com/?size=128&amp;score=11&amp;types=mmmttttt" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">Mamasoula C</span><span class="author ">, Prentice RR</span><span class="author ">, Pierscionek T</span><span class="author ">, Pangilinan F</span><span class="author ">, Mills JL</span><span class="author ">, Druschel C</span><span class="author ">, Pass K</span><span class="author ">, Russell MW</span><span class="author ">, Hall D</span><span class="author ">, Töpf A</span><span class="author hidden" style="display: none;">, Brown DL</span><span class="author hidden" style="display: none;">, Zelenika D</span><span class="author hidden" style="display: none;">, Bentham J</span><span class="author hidden" style="display: none;">, Cosgrove C</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Riveron JG</span><span class="author hidden" style="display: none;">, Setchfield K</span><span class="author hidden" style="display: none;">, Brook JD</span><span class="author hidden" style="display: none;">, Bu'Lock FA</span><span class="author hidden" style="display: none;">, Thornborough C</span><span class="author hidden" style="display: none;">, Rahman TJ</span><span class="author hidden" style="display: none;">, Doza JP</span><span class="author hidden" style="display: none;">, Tan HL</span><span class="author hidden" style="display: none;">, O'Sullivan J</span><span class="author hidden" style="display: none;">, Stuart AG</span><span class="author hidden" style="display: none;">, Blue G</span><span class="author hidden" style="display: none;">, Winlaw D</span><span class="author hidden" style="display: none;">, Postma AV</span><span class="author hidden" style="display: none;">, Mulder BJ</span><span class="author hidden" style="display: none;">, Zwinderman AH</span><span class="author hidden" style="display: none;">, van Engelen K</span><span class="author hidden" style="display: none;">, Moorman AF</span><span class="author hidden" style="display: none;">, Rauch A</span><span class="author hidden" style="display: none;">, Gewillig M</span><span class="author hidden" style="display: none;">, Breckpot J</span><span class="author hidden" style="display: none;">, Devriendt K</span><span class="author hidden" style="display: none;">, Lathrop GM</span><span class="author hidden" style="display: none;">, Farrall M</span><span class="author hidden" style="display: none;">, Goodship JA</span><span class="author hidden" style="display: none;">, Cordell HJ</span><span class="author hidden" style="display: none;">, Brody LC</span><span class="author hidden" style="display: none;">, Keavney BD</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2013.		Association between C677T polymorphism of methylene tetrahydrofolate reductase and congenital heart disease: meta-analysis of 7697 cases and 13,125 controls.		<span class="journal-title">Circ Cardiovasc Genet</span>,		<span class="journal-volume">6</span>		(4),		pp. 347-353.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1161/CIRCGENETICS.113.000191" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-414725" class="pub_abstract" style="display: none;">
		BACKGROUND: Association between the C677T polymorphism of the methylene tetrahydrofolate reductase (MTHFR) gene and congenital heart disease (CHD) is contentious. METHODS AND RESULTS: We compared genotypes between CHD cases and controls and between mothers of CHD cases and controls. We placed our results in context by conducting meta-analyses of previously published studies. Among 5814 cases with primary genotype data and 10 056 controls, there was no evidence of association between MTHFR C677T genotype and CHD risk (odds ratio [OR], 0.96 [95% confidence interval, 0.87-1.07]). A random-effects meta-analysis of all studies (involving 7697 cases and 13 125 controls) suggested the presence of association (OR, 1.25 [95% confidence interval, 1.03-1.51]; P=0.022) but with substantial heterogeneity among contributing studies (I(2)=64.4%) and evidence of publication bias. Meta-analysis of large studies only (defined by a variance of the log OR &lt;0.05), which together contributed 83% of all cases, yielded no evidence of association (OR, 0.97 [95% confidence interval, 0.91-1.03]) without significant heterogeneity (I(2)=0). Moreover, meta-analysis of 1781 mothers of CHD cases (829 of whom were genotyped in this study) and 19 861 controls revealed no evidence of association between maternal C677T genotype and risk of CHD in offspring (OR, 1.13 [95% confidence interval, 0.87-1.47]). There was no significant association between MTHFR genotype and CHD risk in large studies from regions with different levels of dietary folate. CONCLUSIONS: The MTHFR C677T polymorphism, which directly influences plasma folate levels, is not associated with CHD risk. Publication biases appear to substantially contaminate the literature with regard to this genetic association.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="412009" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1038/ng.2637" data-uuid="6b8991ed-f7ec-9e52-4d27-778b8d4caff1"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=1511163" style="display:inline-block;">
  <img alt="Article has an altmetric score of 51" src="https://badges.altmetric.com/?size=128&amp;score=51&amp;types=mttttttf" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">44</p><p class="cited">Scopus</p>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Cordell HJ</span><span class="author ">, Bentham J</span><span class="author ">, Topf A</span><span class="author ">, Zelenika D</span><span class="author ">, Heath S</span><span class="author ">, Mamasoula C</span><span class="author ">, Cosgrove C</span><span class="author ">, Blue G</span><span class="author ">, Granados-Riveron J</span><span class="author ">, Setchfield K</span><span class="author hidden" style="display: none;">, Thornborough C</span><span class="author hidden" style="display: none;">, Breckpot J</span><span class="author hidden" style="display: none;">, Soemedi R</span><span class="author hidden" style="display: none;">, Martin R</span><span class="author hidden" style="display: none;">, Rahman TJ</span><span class="author hidden" style="display: none;">, Hall D</span><span class="author hidden" style="display: none;">, Van Engelen K</span><span class="author hidden" style="display: none;">, Moorman AFM</span><span class="author hidden" style="display: none;">, Zwinderman AH</span><span class="author hidden" style="display: none;">, Barnett P</span><span class="author hidden" style="display: none;">, Koopmann TT</span><span class="author hidden" style="display: none;">, Adriaens ME</span><span class="author hidden" style="display: none;">, Varro A</span><span class="author hidden" style="display: none;">, George AL</span><span class="author hidden" style="display: none;">, Dos Remedios C</span><span class="author hidden" style="display: none;">, Bishopric NH</span><span class="author hidden" style="display: none;">, Bezzina CR</span><span class="author hidden" style="display: none;">, O'Sullivan J</span><span class="author hidden" style="display: none;">, Gewillig M</span><span class="author hidden" style="display: none;">, Bu'Lock FA</span><span class="author hidden" style="display: none;">, Winlaw D</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Devriendt K</span><span class="author hidden" style="display: none;">, Brook JD</span><span class="author hidden" style="display: none;">, Mulder BJM</span><span class="author hidden" style="display: none;">, Mital S</span><span class="author hidden" style="display: none;">, Postma AV</span><span class="author hidden" style="display: none;">, Lathrop GM</span><span class="author hidden" style="display: none;">, Farrall M</span><span class="author hidden" style="display: none;">, Goodship JA</span><span class="author hidden" style="display: none;">, Keavney BD</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2013.		Genome-wide association study of multiple congenital heart disease phenotypes identifies a susceptibility locus for atrial septal defect at chromosome 4p16		<span class="journal-title">Nature Genetics</span>,		<span class="journal-volume">45</span>		(7),		pp. 822-824.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1038/ng.2637" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-412009" class="pub_abstract" style="display: none;">
		We carried out a genome-wide association study (GWAS) of congenital heart disease (CHD). Our discovery cohort comprised 1,995 CHD cases and 5,159 controls and included affected individuals from each of the 3 major clinical CHD categories (with septal, obstructive and cyanotic defects). When all CHD phenotypes were considered together, no region achieved genome-wide significant association. However, a region on chromosome 4p16, adjacent to the MSX1 and STX18 genes, was associated (P = 9.5 × 10 -7) with the risk of ostium secundum atrial septal defect (ASD) in the discovery cohort (N = 340 cases), and this association was replicated in a further 417 ASD cases and 2,520 controls (replication P = 5.0 × 10 -5; odds ratio (OR) in replication cohort = 1.40, 95% confidence interval (CI) = 1.19-1.65; combined P = 2.6 × 10 -10). Genotype accounted for ∼9% of the population-attributable risk of ASD. © 2013 Nature America, Inc. All rights reserved.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="407855" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1186/1471-2156-14-57" data-uuid="87493f3b-254a-eb08-47d3-f2ce10fbdea2"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=1670221" style="display:inline-block;">
  <img alt="Article has an altmetric score of 1" src="https://badges.altmetric.com/?size=128&amp;score=1&amp;types=tttttttt" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">Palomino Doza J</span><span class="author ">, Topf A</span><span class="author ">, Bentham J</span><span class="author ">, Bhattacharya S</span><span class="author ">, Cosgrove C</span><span class="author ">, Brook JD</span><span class="author ">, Granados-Riveron J</span><span class="author ">, Bu'Lock FA</span><span class="author ">, O'Sullivan J</span><span class="author ">, Stuart AG</span><span class="author hidden" style="display: none;">, Parsons J</span><span class="author hidden" style="display: none;">, Relton C</span><span class="author hidden" style="display: none;">, Goodship J</span><span class="author hidden" style="display: none;">, Henderson DJ</span><span class="author hidden" style="display: none;">, Keavney B</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2013.		Low-frequency intermediate penetrance variants in the ROCK1 gene predispose to Tetralogy of Fallot.		<span class="journal-title">BMC Genet</span>,		<span class="journal-volume">14</span>		(1),		pp. 57.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1186/1471-2156-14-57" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-407855" class="pub_abstract" style="display: none;">
		BACKGROUND: Epidemiological studies indicate a substantial excess familial recurrence of non-syndromic Tetralogy of Fallot (TOF), implicating genetic factors that remain largely unknown. The Rho induced kinase 1 gene (ROCK1) is a key component of the planar cell polarity signalling pathway, which plays an important role in normal cardiac development. The aim of this study was to investigate the role of genetic variation in ROCK1 on the risk of TOF. RESULTS: ROCK1 was sequenced in a discovery cohort of 93 non-syndromic TOF probands to identify rare variants. TagSNPs were selected to capture commoner variation in ROCK1. Novel variants and TagSNPs were genotyped in a discovery cohort of 458 TOF cases and 1331 healthy controls, and positive findings were replicated in a further 209 TOF cases and 1290 healthy controls. Association between genotypes and TOF was assessed using LAMP.A rare SNP (c.807C &gt; T; rs56085230) discovered by sequencing was associated with TOF risk (p = 0.006) in the discovery cohort. The variant was also significantly associated with the risk of TOF in the replication cohort (p = 0.018). In the combined cohorts the odds ratio for TOF was 2.61 (95% CI 1.58-4.30); p &lt; 0.0001. The minor allele frequency of rs56085230 in the cases was 0.02, and in the controls it was 0.007. The variant accounted for 1% of the population attributable risk (PAR) of TOF. We also found significant association with TOF for an uncommon TagSNP in ROCK1, rs288979 (OR 1.64 [95% CI 1.15-2.30]; p = 1.5x10⁻⁵). The minor allele frequency of rs288979 in the controls was 0.043, and the variant accounted for 11% of the PAR of TOF. These association signals were independent of each other, providing additional internal validation of our result. CONCLUSIONS: Low frequency intermediate penetrance (LFIP) variants in the ROCK1 gene predispose to the risk of TOF.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="401931" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1038/ng.2637" data-uuid="3b044601-dfb6-33dd-0739-1350fba11c5b"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=1511163" style="display:inline-block;">
  <img alt="Article has an altmetric score of 51" src="https://badges.altmetric.com/?size=128&amp;score=51&amp;types=mttttttf" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Web of Science Lite"><p class="cited">Cited:</p><p class="score">29</p><p class="cited">WOS</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/23708191#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Cordell HJ</span><span class="author ">, Bentham J</span><span class="author ">, Topf A</span><span class="author ">, Zelenika D</span><span class="author ">, Heath S</span><span class="author ">, Mamasoula C</span><span class="author ">, Cosgrove C</span><span class="author ">, Blue G</span><span class="author ">, Granados-Riveron J</span><span class="author ">, Setchfield K</span><span class="author hidden" style="display: none;">, Thornborough C</span><span class="author hidden" style="display: none;">, Breckpot J</span><span class="author hidden" style="display: none;">, Soemedi R</span><span class="author hidden" style="display: none;">, Martin R</span><span class="author hidden" style="display: none;">, Rahman TJ</span><span class="author hidden" style="display: none;">, Hall D</span><span class="author hidden" style="display: none;">, van Engelen K</span><span class="author hidden" style="display: none;">, Moorman AF</span><span class="author hidden" style="display: none;">, Zwinderman AH</span><span class="author hidden" style="display: none;">, Barnett P</span><span class="author hidden" style="display: none;">, Koopmann TT</span><span class="author hidden" style="display: none;">, Adriaens ME</span><span class="author hidden" style="display: none;">, Varro A</span><span class="author hidden" style="display: none;">, George AL</span><span class="author hidden" style="display: none;">, dos Remedios C</span><span class="author hidden" style="display: none;">, Bishopric NH</span><span class="author hidden" style="display: none;">, Bezzina CR</span><span class="author hidden" style="display: none;">, O'Sullivan J</span><span class="author hidden" style="display: none;">, Gewillig M</span><span class="author hidden" style="display: none;">, Bu'Lock FA</span><span class="author hidden" style="display: none;">, Winlaw D</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Devriendt K</span><span class="author hidden" style="display: none;">, Brook JD</span><span class="author hidden" style="display: none;">, Mulder BJ</span><span class="author hidden" style="display: none;">, Mital S</span><span class="author hidden" style="display: none;">, Postma AV</span><span class="author hidden" style="display: none;">, Lathrop GM</span><span class="author hidden" style="display: none;">, Farrall M</span><span class="author hidden" style="display: none;">, Goodship JA</span><span class="author hidden" style="display: none;">, Keavney BD</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2013.		Genome-wide association study of multiple congenital heart disease phenotypes identifies a susceptibility locus for atrial septal defect at chromosome 4p16.		<span class="journal-title">Nat Genet</span>,		<span class="journal-volume">45</span>		(7),		pp. 822-824.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1038/ng.2637" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-401931" class="pub_abstract" style="display: none;">
		We carried out a genome-wide association study (GWAS) of congenital heart disease (CHD). Our discovery cohort comprised 1,995 CHD cases and 5,159 controls and included affected individuals from each of the 3 major clinical CHD categories (with septal, obstructive and cyanotic defects). When all CHD phenotypes were considered together, no region achieved genome-wide significant association. However, a region on chromosome 4p16, adjacent to the MSX1 and STX18 genes, was associated (P = 9.5 × 10⁻⁷) with the risk of ostium secundum atrial septal defect (ASD) in the discovery cohort (N = 340 cases), and this association was replicated in a further 417 ASD cases and 2,520 controls (replication P = 5.0 × 10⁻⁵; odds ratio (OR) in replication cohort = 1.40, 95% confidence interval (CI) = 1.19-1.65; combined P = 2.6 × 10⁻¹⁰). Genotype accounted for ~9% of the population-attributable risk of ASD.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="406046" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1242/dmm.011833" data-uuid="2d6de5b7-6885-fe88-1478-268e7f48dcee"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=1307326" style="display:inline-block;">
  <img alt="Article has an altmetric score of 5" src="https://badges.altmetric.com/?size=128&amp;score=5&amp;types=tttttwff" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">28</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/23519032#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Adams D</span><span class="author ">, Baldock R</span><span class="author ">, Bhattacharya S</span><span class="author ">, Copp AJ</span><span class="author ">, Dickinson M</span><span class="author ">, Greene ND</span><span class="author ">, Henkelman M</span><span class="author ">, Justice M</span><span class="author ">, Mohun T</span><span class="author ">, Murray SA</span><span class="author hidden" style="display: none;">, Pauws E</span><span class="author hidden" style="display: none;">, Raess M</span><span class="author hidden" style="display: none;">, Rossant J</span><span class="author hidden" style="display: none;">, Weaver T</span><span class="author hidden" style="display: none;">, West D</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2013.		Bloomsbury report on mouse embryo phenotyping: recommendations from the IMPC workshop on embryonic lethal screening.		<span class="journal-title">Dis Model Mech</span>,		<span class="journal-volume">6</span>		(3),		pp. 571-579.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1242/dmm.011833" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-406046" class="pub_abstract" style="display: none;">
		Identifying genes that are important for embryo development is a crucial first step towards understanding their many functions in driving the ordered growth, differentiation and organogenesis of embryos. It can also shed light on the origins of developmental disease and congenital abnormalities. Current international efforts to examine gene function in the mouse provide a unique opportunity to pinpoint genes that are involved in embryogenesis, owing to the emergence of embryonic lethal knockout mutants. Through internationally coordinated efforts, the International Knockout Mouse Consortium (IKMC) has generated a public resource of mouse knockout strains and, in April 2012, the International Mouse Phenotyping Consortium (IMPC), supported by the EU InfraCoMP programme, convened a workshop to discuss developing a phenotyping pipeline for the investigation of embryonic lethal knockout lines. This workshop brought together over 100 scientists, from 13 countries, who are working in the academic and commercial research sectors, including experts and opinion leaders in the fields of embryology, animal imaging, data capture, quality control and annotation, high-throughput mouse production, phenotyping, and reporter gene analysis. This article summarises the outcome of the workshop, including (1) the vital scientific importance of phenotyping embryonic lethal mouse strains for basic and translational research; (2) a common framework to harmonise international efforts within this context; (3) the types of phenotyping that are likely to be most appropriate for systematic use, with a focus on 3D embryo imaging; (4) the importance of centralising data in a standardised form to facilitate data mining; and (5) the development of online tools to allow open access to and dissemination of the phenotyping data.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="405759" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1242/dmm.011957" data-uuid="8ee6c5a2-f5e6-aff3-c103-a2ce17028ab5"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=1307592" style="display:inline-block;">
  <img alt="Article has an altmetric score of 13" src="https://badges.altmetric.com/?size=128&amp;score=13&amp;types=bbttttff" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">Mohun T</span><span class="author ">, Adams DJ</span><span class="author ">, Baldock R</span><span class="author ">, Bhattacharya S</span><span class="author ">, Copp AJ</span><span class="author ">, Hemberger M</span><span class="author ">, Houart C</span><span class="author ">, Hurles ME</span><span class="author ">, Robertson E</span><span class="author ">, Smith JC</span><span class="author hidden" style="display: none;">, Weaver T</span><span class="author hidden" style="display: none;">, Weninger W</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2013.		Deciphering the Mechanisms of Developmental Disorders (DMDD): a new programme for phenotyping embryonic lethal mice.		<span class="journal-title">Dis Model Mech</span>,		<span class="journal-volume">6</span>		(3),		pp. 562-566.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1242/dmm.011957" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-405759" class="pub_abstract" style="display: none;">
		International efforts to test gene function in the mouse by the systematic knockout of each gene are creating many lines in which embryonic development is compromised. These homozygous lethal mutants represent a potential treasure trove for the biomedical community. Developmental biologists could exploit them in their studies of tissue differentiation and organogenesis; for clinical researchers they offer a powerful resource for investigating the origins of developmental diseases that affect newborns. Here, we outline a new programme of research in the UK aiming to kick-start research with embryonic lethal mouse lines. The 'Deciphering the Mechanisms of Developmental Disorders' (DMDD) programme has the ambitious goal of identifying all embryonic lethal knockout lines made in the UK over the next 5 years, and will use a combination of comprehensive imaging and transcriptomics to identify abnormalities in embryo structure and development. All data will be made freely available, enabling individual researchers to identify lines relevant to their research. The DMDD programme will coordinate its work with similar international efforts through the umbrella of the International Mouse Phenotyping Consortium [see accompanying Special Article (Adams et al., 2013)] and, together, these programmes will provide a novel database for embryonic development, linking gene identity with molecular profiles and morphology phenotypes.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="396002" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1093/hmg/ddt182" data-uuid="adeb6ee1-e87d-75b2-1ffe-777ea354c312"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1093/hmg/ddt182" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">33</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/23595884#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Rog-Zielinska EA</span><span class="author ">, Thomson A</span><span class="author ">, Kenyon CJ</span><span class="author ">, Brownstein DG</span><span class="author ">, Moran CM</span><span class="author ">, Szumska D</span><span class="author ">, Michailidou Z</span><span class="author ">, Richardson J</span><span class="author ">, Owen E</span><span class="author ">, Watt A</span><span class="author hidden" style="display: none;">, Morrison H</span><span class="author hidden" style="display: none;">, Forrester LM</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Holmes MC</span><span class="author hidden" style="display: none;">, Chapman KE</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2013.		Glucocorticoid receptor is required for foetal heart maturation.		<span class="journal-title">Hum Mol Genet</span>,		<span class="journal-volume">22</span>		(16),		pp. 3269-3282.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1093/hmg/ddt182" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-396002" class="pub_abstract" style="display: none;">
		Glucocorticoids are vital for the structural and functional maturation of foetal organs, yet excessive foetal exposure is detrimental to adult cardiovascular health. To elucidate the role of glucocorticoid signalling in late-gestation cardiovascular maturation, we have generated mice with conditional disruption of glucocorticoid receptor (GR) in cardiomyocytes and vascular smooth muscle cells using smooth muscle protein 22-driven Cre recombinase (SMGRKO mice) and compared them with mice with global deficiency in GR (GR(-/-)). Echocardiography shows impaired heart function in both SMGRKO and GR(-/-) mice at embryonic day (E)17.5, associated with generalized oedema. Cardiac ultrastructure is markedly disrupted in both SMGRKO and GR(-/-) mice at E17.5, with short, disorganized myofibrils and cardiomyocytes that fail to align in the compact myocardium. Failure to induce critical genes involved in contractile function, calcium handling and energy metabolism underpins this common phenotype. However, although hearts of GR(-/-) mice are smaller, with 22% reduced ventricular volume at E17.5, SMGRKO hearts are normally sized. Moreover, while levels of mRNA encoding atrial natriuretic peptide are reduced in E17.5 GR(-/-) hearts, they are normal in foetal SMGRKO hearts. These data demonstrate that structural, functional and biochemical maturation of the foetal heart is dependent on glucocorticoid signalling within cardiomyocytes and vascular smooth muscle, though some aspects of heart maturation (size, ANP expression) are independent of GR at these key sites.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="382915" style="display: none;"><div>
		<div class="altmetric-embed" data-uuid="e0e12822-93d7-047e-4a63-4141094bce6b"><img src="https://altmetric-badges.a.ssl.fastly.net/?size=64&amp;score=?&amp;types=????????" style="display: none;"></div>				<span class="journal-authors"><span class="author ">Davies B</span><span class="author ">, Davies G</span><span class="author ">, Preece C</span><span class="author ">, Bhattacharya S</span>.</span>		2013.		High efficiency sequence specific mutagenesis on CD1, C57Bl6/J and C3H genetic backgrounds by microinjection of TAL-Effector nucleases (TALENs) mRNA		<span class="journal-title">TRANSGENIC RESEARCH</span>,		<span class="journal-volume">22</span>		(1),		pp. 234-234.						<p class="clearfix"></p>
				</div></div>
						<div class="publication group all" symplectic_id="374254" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1093/hmg/dds552" data-uuid="9c30af90-548e-200e-9578-a012af7ed406"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=1258211" style="display:inline-block;">
  <img alt="Article has an altmetric score of 13" src="https://badges.altmetric.com/?size=128&amp;score=13&amp;types=bbbttttt" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">32</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/23297363#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Cordell HJ</span><span class="author ">, Töpf A</span><span class="author ">, Mamasoula C</span><span class="author ">, Postma AV</span><span class="author ">, Bentham J</span><span class="author ">, Zelenika D</span><span class="author ">, Heath S</span><span class="author ">, Blue G</span><span class="author ">, Cosgrove C</span><span class="author ">, Granados Riveron J</span><span class="author hidden" style="display: none;">, Darlay R</span><span class="author hidden" style="display: none;">, Soemedi R</span><span class="author hidden" style="display: none;">, Wilson IJ</span><span class="author hidden" style="display: none;">, Ayers KL</span><span class="author hidden" style="display: none;">, Rahman TJ</span><span class="author hidden" style="display: none;">, Hall D</span><span class="author hidden" style="display: none;">, Mulder BJ</span><span class="author hidden" style="display: none;">, Zwinderman AH</span><span class="author hidden" style="display: none;">, van Engelen K</span><span class="author hidden" style="display: none;">, Brook JD</span><span class="author hidden" style="display: none;">, Setchfield K</span><span class="author hidden" style="display: none;">, Bu'Lock FA</span><span class="author hidden" style="display: none;">, Thornborough C</span><span class="author hidden" style="display: none;">, O'Sullivan J</span><span class="author hidden" style="display: none;">, Stuart AG</span><span class="author hidden" style="display: none;">, Parsons J</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Winlaw D</span><span class="author hidden" style="display: none;">, Mital S</span><span class="author hidden" style="display: none;">, Gewillig M</span><span class="author hidden" style="display: none;">, Breckpot J</span><span class="author hidden" style="display: none;">, Devriendt K</span><span class="author hidden" style="display: none;">, Moorman AF</span><span class="author hidden" style="display: none;">, Rauch A</span><span class="author hidden" style="display: none;">, Lathrop GM</span><span class="author hidden" style="display: none;">, Keavney BD</span><span class="author hidden" style="display: none;">, Goodship JA</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2013.		Genome-wide association study identifies loci on 12q24 and 13q32 associated with tetralogy of Fallot.		<span class="journal-title">Hum Mol Genet</span>,		<span class="journal-volume">22</span>		(7),		pp. 1473-1481.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1093/hmg/dds552" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-374254" class="pub_abstract" style="display: none;">
		We conducted a genome-wide association study to search for risk alleles associated with Tetralogy of Fallot (TOF), using a northern European discovery set of 835 cases and 5159 controls. A region on chromosome 12q24 was associated (P = 1.4 × 10(-7)) and replicated convincingly (P = 3.9 × 10(-5)) in 798 cases and 2931 controls [per allele odds ratio (OR) = 1.27 in replication cohort, P = 7.7 × 10(-11) in combined populations]. Single nucleotide polymorphisms in the glypican 5 gene on chromosome 13q32 were also associated (P = 1.7 × 10(-7)) and replicated convincingly (P = 1.2 × 10(-5)) in 789 cases and 2927 controls (per allele OR = 1.31 in replication cohort, P = 3.03 × 10(-11) in combined populations). Four additional regions on chromosomes 10, 15 and 16 showed suggestive association accompanied by nominal replication. This study, the first genome-wide association study of a congenital heart malformation phenotype, provides evidence that common genetic variation influences the risk of TOF.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="470498" style="display: none;"><div>
		<div class="altmetric-embed" data-uuid="2c9bc8b0-f6d1-c154-818d-6ef320ff5963"><img src="https://altmetric-badges.a.ssl.fastly.net/?size=64&amp;score=?&amp;types=????????" style="display: none;"></div>				<span class="journal-authors"><span class="author ">Zen AAH</span><span class="author ">, Bhattacharya S</span>.</span>		2012.		High Content Imaging Assay to Study Sprouting Angiogenesis Demonstrates A Key Role for Macrophage Endothelial Cell Interactions		<span class="journal-title">CIRCULATION</span>,		<span class="journal-volume">126</span>		(21),								<p class="clearfix"></p>
				</div></div>
						<div class="publication group all" symplectic_id="350527" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1016/j.ajhg.2012.08.003" data-uuid="7f9a1270-243b-bfc6-7ad9-8154b341ff30"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=921337" style="display:inline-block;">
  <img alt="Article has an altmetric score of 2" src="https://badges.altmetric.com/?size=128&amp;score=2&amp;types=ttttttff" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Web of Science Lite"><p class="cited">Cited:</p><p class="score">86</p><p class="cited">WOS</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/22939634#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Soemedi R</span><span class="author ">, Wilson IJ</span><span class="author ">, Bentham J</span><span class="author ">, Darlay R</span><span class="author ">, Töpf A</span><span class="author ">, Zelenika D</span><span class="author ">, Cosgrove C</span><span class="author ">, Setchfield K</span><span class="author ">, Thornborough C</span><span class="author ">, Granados-Riveron J</span><span class="author hidden" style="display: none;">, Blue GM</span><span class="author hidden" style="display: none;">, Breckpot J</span><span class="author hidden" style="display: none;">, Hellens S</span><span class="author hidden" style="display: none;">, Zwolinkski S</span><span class="author hidden" style="display: none;">, Glen E</span><span class="author hidden" style="display: none;">, Mamasoula C</span><span class="author hidden" style="display: none;">, Rahman TJ</span><span class="author hidden" style="display: none;">, Hall D</span><span class="author hidden" style="display: none;">, Rauch A</span><span class="author hidden" style="display: none;">, Devriendt K</span><span class="author hidden" style="display: none;">, Gewillig M</span><span class="author hidden" style="display: none;">, O' Sullivan J</span><span class="author hidden" style="display: none;">, Winlaw DS</span><span class="author hidden" style="display: none;">, Bu'Lock F</span><span class="author hidden" style="display: none;">, Brook JD</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Lathrop M</span><span class="author hidden" style="display: none;">, Santibanez-Koref M</span><span class="author hidden" style="display: none;">, Cordell HJ</span><span class="author hidden" style="display: none;">, Goodship JA</span><span class="author hidden" style="display: none;">, Keavney BD</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2012.		Contribution of global rare copy-number variants to the risk of sporadic congenital heart disease.		<span class="journal-title">Am J Hum Genet</span>,		<span class="journal-volume">91</span>		(3),		pp. 489-501.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1016/j.ajhg.2012.08.003" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-350527" class="pub_abstract" style="display: none;">
		Previous studies have shown that copy-number variants (CNVs) contribute to the risk of complex developmental phenotypes. However, the contribution of global CNV burden to the risk of sporadic congenital heart disease (CHD) remains incompletely defined. We generated genome-wide CNV data by using Illumina 660W-Quad SNP arrays in 2,256 individuals with CHD, 283 trio CHD-affected families, and 1,538 controls. We found association of rare genic deletions with CHD risk (odds ratio [OR] = 1.8, p = 0.0008). Rare deletions in study participants with CHD had higher gene content (p = 0.001) with higher haploinsufficiency scores (p = 0.03) than they did in controls, and they were enriched with Wnt-signaling genes (p = 1 × 10(-5)). Recurrent 15q11.2 deletions were associated with CHD risk (OR = 8.2, p = 0.02). Rare de novo CNVs were observed in ~5% of CHD trios; 10 out of 11 occurred on the paternally transmitted chromosome (p = 0.01). Some of the rare de novo CNVs spanned genes known to be involved in heart development (e.g., HAND2 and GJA5). Rare genic deletions contribute ~4% of the population-attributable risk of sporadic CHD. Second to previously described CNVs at 1q21.1, deletions at 15q11.2 and those implicating Wnt signaling are the most significant contributors to the risk of sporadic CHD. Rare de novo CNVs identified in CHD trios exhibit paternal origin bias.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="350525" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.2459/JCM.0b013e328356a326" data-uuid="2db414f0-a8df-829b-092a-d956adbe19f0"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.2459/jcm.0b013e328356a326" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Beffagna G</span><span class="author ">, Cecchetto A</span><span class="author ">, Dal Bianco L</span><span class="author ">, Lorenzon A</span><span class="author ">, Angelini A</span><span class="author ">, Padalino M</span><span class="author ">, Vida V</span><span class="author ">, Bhattacharya S</span><span class="author ">, Stellin G</span><span class="author ">, Rampazzo A</span><span class="author ">, Daliento L</span>.</span>		2013.		R25C mutation in the NKX2.5 gene in Italian patients affected with non-syndromic and syndromic congenital heart disease.		<span class="journal-title">J Cardiovasc Med (Hagerstown)</span>,		<span class="journal-volume">14</span>		(8),		pp. 582-586.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.2459/JCM.0b013e328356a326" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-350525" class="pub_abstract" style="display: none;">
		AIMS: Heterozygous mutations in the transcription factor Nkx2.5 indicate a genetic cause for congenital heart diseases (CHDs) in human beings. The present study aimed to assess the prevalence of NKX2.5 mutations in Italian patients with sporadic non-syndromic and syndromic CHD, as well as to appraise any genotype-phenotype correlations. METHODS: One hundred Italian patients affected with CHD (90 had sporadic non-syndromic CHD and 10 had syndromic CHD) were screened for NKX2.5 mutations. The coding region and flanking regions involved in gene splicing of the CSX/NKX2.5 gene were amplified from genomic DNA by PCR, and mutational analysis was performed using denaturing high performance liquid chromatography and DNA sequencing. RESULTS: One previously reported NKX2.5 mutation (c.73C&gt;T, p.R25C) was identified in two of the 100 CHD patients (2%). We have detected the p.R25C alteration in a woman showing aneurysm of the membranous septum, aortic coarctation and bicuspid aortic valve, that was a different phenotype from those previously reported, and for the first time in a patient with syndromic CHD with Down's syndrome (posterior ventricular septal defect, atrial septal defect, left superior cava vein ' sinus, and patent ductus arteriosus). CONCLUSION: Our results confirm that NKX2.5 mutations are not a common cause of CHD; furthermore, the p.R25C variation may increase susceptibility to development of CHD in patients with and without chromosomal abnormalities.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="345893" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1002/dvdy.23812" data-uuid="f013df33-6e2f-54c0-8c7f-c805bd1bfd3e"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=787562" style="display:inline-block;">
  <img alt="Article has an altmetric score of 1" src="https://badges.altmetric.com/?size=128&amp;score=1&amp;types=tttttttt" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">Simrick S</span><span class="author ">, Szumska D</span><span class="author ">, Gardiner JR</span><span class="author ">, Jones K</span><span class="author ">, Sagar K</span><span class="author ">, Morrow B</span><span class="author ">, Bhattacharya S</span><span class="author ">, Basson MA</span>.</span>		2012.		Biallelic expression of Tbx1 protects the embryo from developmental defects caused by increased receptor tyrosine kinase signaling.		<span class="journal-title">Dev Dyn</span>,		<span class="journal-volume">241</span>		(8),		pp. 1310-1324.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1002/dvdy.23812" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-345893" class="pub_abstract" style="display: none;">
		BACKGROUND: 22q11.2 deletion syndrome (22q11DS) is the most common microdeletion syndrome in humans, characterized by cardiovascular defects such as interrupted aortic arch, outflow tract defects, thymus and parathyroid hypo- or aplasia, and cleft palate. Heterozygosity of Tbx1, the mouse homolog of the candidate TBX1 gene, results in mild defects dependent on genetic background, whereas complete inactivation results in severe malformations in multiple tissues. RESULTS: The loss of function of two Sprouty genes, which encode feedback antagonists of receptor tyrosine kinase (RTK) signaling, phenocopy many defects associated with 22q11DS in the mouse. The stepwise reduction of Sprouty gene dosage resulted in different phenotypes emerging at specific steps, suggesting that the threshold up to which a given developmental process can tolerate increased RTK signaling is different. Tbx1 heterozygosity significantly exacerbated the severity of all these defects, which correlated with a substantial increase in RTK signaling. CONCLUSIONS: Our findings suggest that TBX1 functions as an essential component of a mechanism that protects the embryo against perturbations in RTK signaling that may lead to developmental defects characteristic of 22q11DS. We propose that genetic factors that enhance RTK signaling ought to be considered as potential genetic modifiers of this syndrome.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="325587" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1007/s00335-012-9398-y" data-uuid="5f934907-0607-baa5-16ee-e95827189a65"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1007/s00335-012-9398-y" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Kowalczyk MS</span><span class="author ">, Hughes JR</span><span class="author ">, Babbs C</span><span class="author ">, Sanchez-Pulido L</span><span class="author ">, Szumska D</span><span class="author ">, Sharpe JA</span><span class="author ">, Sloane-Stanley JA</span><span class="author ">, Morriss-Kay GM</span><span class="author ">, Smoot LB</span><span class="author ">, Roberts AE</span><span class="author hidden" style="display: none;">, Watkins H</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Gibbons RJ</span><span class="author hidden" style="display: none;">, Ponting CP</span><span class="author hidden" style="display: none;">, Wood WG</span><span class="author hidden" style="display: none;">, Higgs DR</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2012.		Nprl3 is required for normal development of the cardiovascular system.		<span class="journal-title">Mamm Genome</span>,		<span class="journal-volume">23</span>		(7-8),		pp. 404-415.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1007/s00335-012-9398-y" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-325587" class="pub_abstract" style="display: none;">
		C16orf35 is a conserved and widely expressed gene lying adjacent to the human α-globin cluster in all vertebrate species. In-depth sequence analysis shows that C16orf35 (now called NPRL3) is an orthologue of the yeast gene Npr3 (nitrogen permease regulator 3) and, furthermore, is a paralogue of its protein partner Npr2. The yeast Npr2/3 dimeric protein complex senses amino acid starvation and appropriately adjusts cell metabolism via the TOR pathway. Here we have analysed a mouse model in which expression of Nprl3 has been abolished using homologous recombination. The predominant effect on RNA expression appears to involve genes that regulate protein synthesis and cell cycle, consistent with perturbation of the mTOR pathway. Embryos homozygous for this mutation die towards the end of gestation with a range of cardiovascular defects, including outflow tract abnormalities and ventriculoseptal defects consistent with previous observations, showing that perturbation of the mTOR pathway may affect development of the myocardium. NPRL3 is a candidate gene for harbouring mutations in individuals with developmental abnormalities of the cardiovascular system.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="325173" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1161/CIRCGENETICS.111.962035" data-uuid="c8ea9c0d-1ab8-74b0-6cbf-23f8a42d886e"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1161/circgenetics.111.962035" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Goodship JA</span><span class="author ">, Hall D</span><span class="author ">, Topf A</span><span class="author ">, Mamasoula C</span><span class="author ">, Griffin H</span><span class="author ">, Rahman TJ</span><span class="author ">, Glen E</span><span class="author ">, Tan H</span><span class="author ">, Palomino Doza J</span><span class="author ">, Relton CL</span><span class="author hidden" style="display: none;">, Bentham J</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Cosgrove C</span><span class="author hidden" style="display: none;">, Brook D</span><span class="author hidden" style="display: none;">, Granados-Riveron J</span><span class="author hidden" style="display: none;">, Bu'Lock FA</span><span class="author hidden" style="display: none;">, O'Sullivan J</span><span class="author hidden" style="display: none;">, Stuart AG</span><span class="author hidden" style="display: none;">, Parsons J</span><span class="author hidden" style="display: none;">, Cordell HJ</span><span class="author hidden" style="display: none;">, Keavney B</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2012.		A common variant in the PTPN11 gene contributes to the risk of tetralogy of Fallot.		<span class="journal-title">Circ Cardiovasc Genet</span>,		<span class="journal-volume">5</span>		(3),		pp. 287-292.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1161/CIRCGENETICS.111.962035" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-325173" class="pub_abstract" style="display: none;">
		BACKGROUND: Tetralogy of Fallot (TOF) is the commonest cyanotic form of congenital heart disease. In 80% of cases, TOF behaves as a complex genetic condition exhibiting significant heritability. As yet, no common genetic variants influencing TOF risk have been robustly identified. METHODS AND RESULTS: Two hundred and seven haplotype-tagging single nucleotide polymorphisms in 22 candidate genes were genotyped in a test cohort comprising 362 nonsyndromic British white patients with TOF together with 717 unaffected parents of patients and 183 unrelated healthy controls. Single nucleotide polymorphisms with suggestive evidence of association in the test cohort (P&lt;0.01) were taken forward for genotyping in an independent replication cohort comprising 392 cases of TOF, 218 unaffected parents of patients, and 1319 controls. Significant association was observed for 1 single nucleotide polymorphism, rs11066320 in the PTPN11 gene, in both the test and the replication cohort. Genotype at rs11066320 was associated with a per-allele odds ratio of 1.34 (95% confidence interval [CI], 1.19 to 1.52; P=2.9 × 10(-6)) in the total cohort of TOF cases and controls; this remained highly significant after Bonferroni correction for 207 analyses (corrected P=0.00061). Genotype at rs11066320 was responsible for a population-attributable risk of TOF of approximately 10%. CONCLUSIONS: Common variation in the linkage disequilibrium block including the PTPN11 gene contributes to the risk of nonsyndromic TOF. Rare mutations in PTPN11 are known to cause the autosomal dominant condition Noonan syndrome, which includes congenital heart disease, by upregulating Ras/mitogen-activated protein kinase (MAPK) signaling. Our results suggest a role for milder perturbations in PTPN11 function in sporadic, nonsyndromic congenital heart disease.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="314232" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1093/cvr/cvr314" data-uuid="1372348f-093e-34b2-14c5-adee95e122b7"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1093/cvr/cvr314" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Ammirabile G</span><span class="author ">, Tessari A</span><span class="author ">, Pignataro V</span><span class="author ">, Szumska D</span><span class="author ">, Sutera Sardo F</span><span class="author ">, Benes J</span><span class="author ">, Balistreri M</span><span class="author ">, Bhattacharya S</span><span class="author ">, Sedmera D</span><span class="author ">, Campione M</span>.</span>		2012.		Pitx2 confers left morphological, molecular, and functional identity to the sinus venosus myocardium.		<span class="journal-title">Cardiovasc Res</span>,		<span class="journal-volume">93</span>		(2),		pp. 291-301.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1093/cvr/cvr314" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-314232" class="pub_abstract" style="display: none;">
		AIMS: The sinus venous myocardium, comprising the sinoatrial node (SAN) and sinus horns (SH), is a region subject to congenital malformations and cardiac arrhythmias. It differentiates from symmetric bilateral mesenchymal precursors, but morphological, molecular, and functional left/right differences are progressively established through development. The role of the laterality gene Pitx2 in this process is unknown. We aimed to elucidate the molecular events driving left/right patterning in the sinus venosus (SV) myocardium by using a myocardial Pitx2 knockout mouse. METHODS AND RESULTS: We generated a myocardial specific Pitx2 knockout model (cTP mice). cTP embryos present several features of Pitx2 null, including right atrial isomerism with bilateral SANs and symmetric atrial entrance of the systemic veins. By in situ hybridization and optical mapping analysis, we compared throughout development the molecular and functional properties of the SV myocardium in wt and mutant embryos. We observed that Pitx2 prevents the expansion of the left-SAN primordium at the onset of its differentiation into myocardium; Pitx2 promotes expansion of the left SH through development; Pitx2 dose-dependently represses the autorhythmic properties of the left SV myocardium at mid-gestation (E14.5); Pitx2 modulates late foetal gene expression at the left SH-derived superior caval vein. CONCLUSION: Pitx2 drives left/right patterning of the SV myocardium through multiple developmental steps. Overall, Pitx2 plays a crucial functional role by negatively modulating a nodal-type programme in the left SV myocardium.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="311336" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1093/eurheartj/ehs056" data-uuid="eeec9e74-e5d4-2f1a-524a-56689e3378a1"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=700651" style="display:inline-block;">
  <img alt="Article has an altmetric score of 1" src="https://badges.altmetric.com/?size=128&amp;score=1&amp;types=tttttttt" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">MacDonald ST</span><span class="author ">, Bamforth SD</span><span class="author ">, Bragança J</span><span class="author ">, Chen CM</span><span class="author ">, Broadbent C</span><span class="author ">, Schneider JE</span><span class="author ">, Schwartz RJ</span><span class="author ">, Bhattacharya S</span>.</span>		2013.		A cell-autonomous role of Cited2 in controlling myocardial and coronary vascular development.		<span class="journal-title">Eur Heart J</span>,		<span class="journal-volume">34</span>		(32),		pp. 2557-2565.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1093/eurheartj/ehs056" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-311336" class="pub_abstract" style="display: none;">
		AIMS: Myocardial development is dependent on concomitant growth of cardiomyocytes and a supporting vascular network. The coupling of myocardial and coronary vascular development is partly mediated by vascular endothelial growth factor (VEGFA) signalling and additional unknown mechanisms. We examined the cardiomyocyte specific role of the transcriptional co-activator Cited2 on myocardial microstructure and vessel growth, in relation to Vegfa expression. METHODS AND RESULTS: A cardiomyocyte-specific knockout of mouse Cited2 (Cited2(Nkx)) was analysed using magnetic resonance imaging and histology. Ventricular septal defects and significant compact layer thinning (P &lt; 0.02 at right ventricular apex, P &lt; 0.009 at the left ventricular apex in Cited2(Nkx) vs. controls, n = 11 vs. n = 7, respectively) were found. This was associated with a significant decrease in the number of capillaries to larger vessels (ratio 1.56 ± 0.56 vs. 3.25 ± 1.63, P = 2.7 × 10(-6) Cited2(Nkx) vs. controls, n = 11 vs. n = 7, respectively) concomitant with a 1.5-fold reduction in Vegfa expression (P &lt; 0.02, Cited2(Nkx) vs. controls, n = 12 vs. n = 12, respectively). CITED2 was subsequently found at the Vegfa promoter in mouse embryonic hearts using chromatin immunoprecipitation, and moreover found to stimulate human VEGFA promoter activity in cooperation with TFAP2 transcription factors in transient transfection assays. There was no change in the myocardial expression of the left-right patterning gene Pitx2c, a previously known target of CITED2. CONCLUSIONS: This study delineates a novel cell-autonomous role of Cited2 in regulating VEGFA transcription and the development of myocardium and coronary vasculature in the mouse. We suggest that coupling of myocardial and coronary growth in the developing heart may occur in part through a Cited2→Vegfa pathway.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="352769" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1016/j.ajhg.2012.08.003" data-uuid="58608831-ef60-e860-ddd4-0b00623fe1c0"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=921337" style="display:inline-block;">
  <img alt="Article has an altmetric score of 2" src="https://badges.altmetric.com/?size=128&amp;score=2&amp;types=ttttttff" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">114</p><p class="cited">Scopus</p>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Soemedi R</span><span class="author ">, Wilson IJ</span><span class="author ">, Bentham J</span><span class="author ">, Darlay R</span><span class="author ">, Töpf A</span><span class="author ">, Zelenika D</span><span class="author ">, Cosgrove C</span><span class="author ">, Setchfield K</span><span class="author ">, Thornborough C</span><span class="author ">, Granados-Riveron J</span><span class="author hidden" style="display: none;">, Blue GM</span><span class="author hidden" style="display: none;">, Breckpot J</span><span class="author hidden" style="display: none;">, Hellens S</span><span class="author hidden" style="display: none;">, Zwolinkski S</span><span class="author hidden" style="display: none;">, Glen E</span><span class="author hidden" style="display: none;">, Mamasoula C</span><span class="author hidden" style="display: none;">, Rahman TJ</span><span class="author hidden" style="display: none;">, Hall D</span><span class="author hidden" style="display: none;">, Rauch A</span><span class="author hidden" style="display: none;">, Devriendt K</span><span class="author hidden" style="display: none;">, Gewillig M</span><span class="author hidden" style="display: none;">, O'sullivan J</span><span class="author hidden" style="display: none;">, Winlaw DS</span><span class="author hidden" style="display: none;">, Bu'lock F</span><span class="author hidden" style="display: none;">, Brook JD</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Lathrop M</span><span class="author hidden" style="display: none;">, Santibanez-Koref M</span><span class="author hidden" style="display: none;">, Cordell HJ</span><span class="author hidden" style="display: none;">, Goodship JA</span><span class="author hidden" style="display: none;">, Keavney BD</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2012.		Contribution of global rare copy-number variants to the risk of sporadic congenital heart disease		<span class="journal-title">American Journal of Human Genetics</span>,		<span class="journal-volume">91</span>		(3),		pp. 489-501.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1016/j.ajhg.2012.08.003" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-352769" class="pub_abstract" style="display: none;">
		Previous studies have shown that copy-number variants (CNVs) contribute to the risk of complex developmental phenotypes. However, the contribution of global CNV burden to the risk of sporadic congenital heart disease (CHD) remains incompletely defined. We generated genome-wide CNV data by using Illumina 660W-Quad SNP arrays in 2,256 individuals with CHD, 283 trio CHD-affected families, and 1,538 controls. We found association of rare genic deletions with CHD risk (odds ratio [OR] = 1.8, p = 0.0008). Rare deletions in study participants with CHD had higher gene content (p = 0.001) with higher haploinsufficiency scores (p = 0.03) than they did in controls, and they were enriched with Wnt-signaling genes (p = 1 × 10-5). Recurrent 15q11.2 deletions were associated with CHD risk (OR = 8.2, p = 0.02). Rare de novo CNVs were observed in ∼5% of CHD trios; 10 out of 11 occurred on the paternally transmitted chromosome (p = 0.01). Some of the rare de novo CNVs spanned genes known to be involved in heart development (e.g., HAND2 and GJA5). Rare genic deletions contribute ∼4% of the population-attributable risk of sporadic CHD. Second to previously described CNVs at 1q21.1, deletions at 15q11.2 and those implicating Wnt signaling are the most significant contributors to the risk of sporadic CHD. Rare de novo CNVs identified in CHD trios exhibit paternal origin bias. © 2012 The American Society of Human Genetics.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="356193" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1371/journal.pone.0046256" data-uuid="d004489c-aa72-e960-cddd-2108db89edf8"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1371/journal.pone.0046256" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Chen CM</span><span class="author ">, Bentham J</span><span class="author ">, Cosgrove C</span><span class="author ">, Braganca J</span><span class="author ">, Cuenda A</span><span class="author ">, Bamforth SD</span><span class="author ">, Schneider JE</span><span class="author ">, Watkins H</span><span class="author ">, Keavney B</span><span class="author ">, Davies B</span><span class="author ">, Bhattacharya S</span>.</span>		2012.		Functional significance of SRJ domain mutations in CITED2.		<span class="journal-title">PLoS One</span>,		<span class="journal-volume">7</span>		(10),		pp. e46256.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1371/journal.pone.0046256" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-356193" class="pub_abstract" style="display: none;">
		CITED2 is a transcriptional co-activator with 3 conserved domains shared with other CITED family members and a unique Serine-Glycine Rich Junction (SRJ) that is highly conserved in placental mammals. Loss of Cited2 in mice results in cardiac and aortic arch malformations, adrenal agenesis, neural tube and placental defects, and partially penetrant defects in left-right patterning. By screening 1126 sporadic congenital heart disease (CHD) cases and 1227 controls, we identified 19 variants, including 5 unique non-synonymous sequence variations (N62S, R92G, T166N, G180-A187del and A187T) in patients. Many of the CHD-specific variants identified in this and previous studies cluster in the SRJ domain. Transient transfection experiments show that T166N mutation impairs TFAP2 co-activation function and ES cell proliferation. We find that CITED2 is phosphorylated by MAPK1 in vitro at T166, and that MAPK1 activation enhances the coactivation function of CITED2 but not of CITED2-T166N. In order to investigate the functional significance in vivo, we generated a T166N mutation of mouse Cited2. We also used PhiC31 integrase-mediated cassette exchange to generate a Cited2 knock-in allele replacing the mouse Cited2 coding sequence with human CITED2 and with a mutant form deleting the entire SRJ domain. Mouse embryos expressing only CITED2-T166N or CITED2-SRJ-deleted alleles surprisingly show no morphological abnormalities, and mice are viable and fertile. These results indicate that the SRJ domain is dispensable for these functions of CITED2 in mice and that mutations clustering in the SRJ region are unlikely to be the sole cause of the malformations observed in patients with sporadic CHD. Our results also suggest that coding sequence mutations observed in case-control studies need validation using in vivo models and that predictions based on structural conservation and in vitro functional assays, or even in vivo global loss of function models, may be insufficient.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="353114" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1128/MCB.06331-11" data-uuid="eea50e15-2b34-a265-5875-b563d923a7ec"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1128/mcb.06331-11" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Kim W</span><span class="author ">, Essalmani R</span><span class="author ">, Szumska D</span><span class="author ">, Creemers JW</span><span class="author ">, Roebroek AJ</span><span class="author ">, D'Orleans-Juste P</span><span class="author ">, Bhattacharya S</span><span class="author ">, Seidah NG</span><span class="author ">, Prat A</span>.</span>		2012.		Loss of endothelial furin leads to cardiac malformation and early postnatal death.		<span class="journal-title">Mol Cell Biol</span>,		<span class="journal-volume">32</span>		(17),		pp. 3382-3391.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1128/MCB.06331-11" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-353114" class="pub_abstract" style="display: none;">
		In mammals, seven proprotein convertases (PCs) cleave secretory proteins after basic residues, and four of them are called furin-like PCs: furin, PC5, PACE4, and PC7. In vitro, they share many substrates. However, furin is essential during development since deficient embryos die at embryonic day 11 and exhibit multiple developmental defects, particularly defects related to the function of endothelial cells. To define the role of furin in endothelial cells, an endothelial cell-specific knockout (ecKO) of the Furin gene was generated. Newborns die shortly after birth, indicating that furin is essential in these cells. Magnetic resonance imaging revealed that ecKO embryos exhibit ventricular septal defects (VSD) and/or valve malformations. In addition, primary cultures of wild-type and ecKO lung endothelial cells revealed that ecKO cells are unable to grow. Growth was efficiently rescued by extracellular soluble furin. Analysis of the processing of precursors of endothelin-1 (ET-1), adrenomedullin (Adm), transforming growth factor β1 (TGF-β1), and bone morphogenetic protein 4 (BMP4) confirmed that ET-1, Adm, and TGF-β1 are in vivo substrates of endothelial furin. Mature ET-1 and BMP4 forms were reduced by ~90% in ecKO purified endothelial cells from lungs.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="254377" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1002/prot.23208" data-uuid="e454c13b-f557-0ae4-6e53-caffbbe8e2b7"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=403475" style="display:inline-block;">
  <img alt="Article has an altmetric score of 1" src="https://badges.altmetric.com/?size=128&amp;score=1&amp;types=tttttttt" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">El Omari K</span><span class="author ">, de Mesmaeker J</span><span class="author ">, Karia D</span><span class="author ">, Ginn H</span><span class="author ">, Bhattacharya S</span><span class="author ">, Mancini EJ</span>.</span>		2012.		Structure of the DNA-bound T-box domain of human TBX1, a transcription factor associated with the DiGeorge syndrome		<span class="journal-title">Proteins: Structure, Function and Bioinformatics</span>,		<span class="journal-volume">80</span>		(2),		pp. 655-660.				 | <a href="http://dx.doi.org/10.1002/prot.23208" target="_blank">Read more</a> 		<p class="clearfix"></p>
				</div></div>
						<div class="publication group all" symplectic_id="222338" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1093/hmg/ddr589" data-uuid="16927a79-18aa-d4cb-87a4-e7bcf8501e8d"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=509677" style="display:inline-block;">
  <img alt="Article has an altmetric score of 4" src="https://badges.altmetric.com/?size=128&amp;score=4&amp;types=twwwwwff" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">48</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/22199024#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Soemedi R</span><span class="author ">, Topf A</span><span class="author ">, Wilson IJ</span><span class="author ">, Darlay R</span><span class="author ">, Rahman T</span><span class="author ">, Glen E</span><span class="author ">, Hall D</span><span class="author ">, Huang N</span><span class="author ">, Bentham J</span><span class="author ">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Cosgrove C</span><span class="author hidden" style="display: none;">, Brook JD</span><span class="author hidden" style="display: none;">, Granados-Riveron J</span><span class="author hidden" style="display: none;">, Setchfield K</span><span class="author hidden" style="display: none;">, Bu'lock F</span><span class="author hidden" style="display: none;">, Thornborough C</span><span class="author hidden" style="display: none;">, Devriendt K</span><span class="author hidden" style="display: none;">, Breckpot J</span><span class="author hidden" style="display: none;">, Hofbeck M</span><span class="author hidden" style="display: none;">, Lathrop M</span><span class="author hidden" style="display: none;">, Rauch A</span><span class="author hidden" style="display: none;">, Blue GM</span><span class="author hidden" style="display: none;">, Winlaw DS</span><span class="author hidden" style="display: none;">, Hurles M</span><span class="author hidden" style="display: none;">, Santibanez-Koref M</span><span class="author hidden" style="display: none;">, Cordell HJ</span><span class="author hidden" style="display: none;">, Goodship JA</span><span class="author hidden" style="display: none;">, Keavney BD</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2012.		Phenotype-specific effect of chromosome 1q21.1 rearrangements and GJA5 duplications in 2436 congenital heart disease patients and 6760 controls.		<span class="journal-title">Hum Mol Genet</span>,		<span class="journal-volume">21</span>		(7),		pp. 1513-1520.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1093/hmg/ddr589" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-222338" class="pub_abstract" style="display: none;">
		Recurrent rearrangements of chromosome 1q21.1 that occur via non-allelic homologous recombination have been associated with variable phenotypes exhibiting incomplete penetrance, including congenital heart disease (CHD). However, the gene or genes within the ~1 Mb critical region responsible for each of the associated phenotypes remains unknown. We examined the 1q21.1 locus in 948 patients with tetralogy of Fallot (TOF), 1488 patients with other forms of CHD and 6760 ethnically matched controls using single nucleotide polymorphism genotyping arrays (Illumina 660W and Affymetrix 6.0) and multiplex ligation-dependent probe amplification. We found that duplication of 1q21.1 was more common in cases of TOF than in controls [odds ratio (OR) 30.9, 95% confidence interval (CI) 8.9-107.6); P = 2.2 × 10(-7)], but deletion was not. In contrast, deletion of 1q21.1 was more common in cases of non-TOF CHD than in controls [OR 5.5 (95% CI 1.4-22.0); P = 0.04] while duplication was not. We also detected rare (n = 3) 100-200 kb duplications within the critical region of 1q21.1 in cases of TOF. These small duplications encompassed a single gene in common, GJA5, and were enriched in cases of TOF in comparison to controls [OR = 10.7 (95% CI 1.8-64.3), P = 0.01]. These findings show that duplication and deletion at chromosome 1q21.1 exhibit a degree of phenotypic specificity in CHD, and implicate GJA5 as the gene responsible for the CHD phenotypes observed with copy number imbalances at this locus.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="206059" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1002/prot.23208" data-uuid="11899c70-86a9-bb58-7c09-38f6e14967c8"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=403475" style="display:inline-block;">
  <img alt="Article has an altmetric score of 1" src="https://badges.altmetric.com/?size=128&amp;score=1&amp;types=tttttttt" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">El Omari K</span><span class="author ">, De Mesmaeker J</span><span class="author ">, Karia D</span><span class="author ">, Ginn H</span><span class="author ">, Bhattacharya S</span><span class="author ">, Mancini EJ</span>.</span>		2012.		Structure of the DNA-bound T-box domain of human TBX1, a transcription factor associated with the DiGeorge syndrome.		<span class="journal-title">Proteins</span>,		<span class="journal-volume">80</span>		(2),		pp. 655-660.				 | <a href="http://dx.doi.org/10.1002/prot.23208" target="_blank">Read more</a> 		<p class="clearfix"></p>
				</div></div>
						<div class="publication group all" symplectic_id="164279" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1074/jbc.M111.233577" data-uuid="585e3259-4a7d-2a42-333e-bce05f24fe93"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1074/jbc.m111.233577" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">20</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/21550985#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Susan-Resiga D</span><span class="author ">, Essalmani R</span><span class="author ">, Hamelin J</span><span class="author ">, Asselin MC</span><span class="author ">, Benjannet S</span><span class="author ">, Chamberland A</span><span class="author ">, Day R</span><span class="author ">, Szumska D</span><span class="author ">, Constam D</span><span class="author ">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Prat A</span><span class="author hidden" style="display: none;">, Seidah NG</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2011.		Furin is the major processing enzyme of the cardiac-specific growth factor bone morphogenetic protein 10.		<span class="journal-title">J Biol Chem</span>,		<span class="journal-volume">286</span>		(26),		pp. 22785-22794.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1074/jbc.M111.233577" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-164279" class="pub_abstract" style="display: none;">
		Bone morphogenetic protein 10 (BMP10) is a member of the TGF-β superfamily and plays a critical role in heart development. In the postnatal heart, BMP10 is restricted to the right atrium. The inactive pro-BMP10 (∼60 kDa) is processed into active BMP10 (∼14 kDa) by an unknown protease. Proteolytic cleavage occurs at the RIRR(316)↓ site (human), suggesting the involvement of proprotein convertase(s) (PCs). In vitro digestion of a 12-mer peptide encompassing the predicted cleavage site with furin, PACE4, PC5/6, and PC7, showed that furin cleaves the best, whereas PC7 is inactive on this peptide. Ex vivo studies in COS-1 cells, a cell line lacking PC5/6, revealed efficient processing of pro-BMP10 by endogenous PCs other than PC5/6. The lack of processing of overexpressed pro-BMP10 in the furin- and PACE4-deficient cell line, CHO-FD11, and in furin-deficient LoVo cells, was restored by stable (CHO-FD11/Fur cells) or transient (LoVo cells) expression of furin. Use of cell-permeable and cell surface inhibitors suggested that endogenous PCs process pro-BMP10 mostly intracellularly, but also at the cell surface. Ex vivo experiments in mouse primary hepatocytes (wild type, PC5/6 knock-out, and furin knock-out) corroborated the above findings that pro-BMP10 is a substrate for endogenous furin. Western blot analyses of heart right atria extracts from wild type and PACE4 knock-out adult mice showed no significant difference in the processing of pro-BMP10, implying no in vivo role of PACE4. Overall, our in vitro, ex vivo, and in vivo data suggest that furin is the major convertase responsible for the generation of BMP10.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="167666" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1186/1471-2105-12-237" data-uuid="12d87161-7475-1bd2-b7dd-b0d90d333819"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1186/1471-2105-12-237" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Baghdadi L</span><span class="author ">, Zamyadi M</span><span class="author ">, Sled JG</span><span class="author ">, Schneider JE</span><span class="author ">, Bhattacharya S</span><span class="author ">, Henkelman RM</span><span class="author ">, Lerch JP</span>.</span>		2011.		Semi-automatic segmentation of multiple mouse embryos in MR images.		<span class="journal-title">BMC Bioinformatics</span>,		<span class="journal-volume">12</span>		(1),		pp. 237.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1186/1471-2105-12-237" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-167666" class="pub_abstract" style="display: none;">
		BACKGROUND: The motivation behind this paper is to aid the automatic phenotyping of mouse embryos, wherein multiple embryos embedded within a single tube were scanned using Magnetic Resonance Imaging (MRI). RESULTS: Our algorithm, a modified version of the simplex deformable model of Delingette, addresses various issues with deformable models including initialization and inability to adapt to boundary concavities. In addition, it proposes a novel technique for automatic collision detection of multiple objects which are being segmented simultaneously, hence avoiding major leaks into adjacent neighbouring structures. We address the initialization problem by introducing balloon forces which expand the initial spherical models close to the true boundaries of the embryos. This results in models which are less sensitive to initial minimum of two fold after each stage of deformation. To determine collision during segmentation, our unique collision detection algorithm finds the intersection between binary masks created from the deformed models after every few iterations of the deformation and modifies the segmentation parameters accordingly hence avoiding collision.We have segmented six tubes of three dimensional MR images of multiple mouse embryos using our modified deformable model algorithm. We have then validated the results of the our semi-automatic segmentation versus manual segmentation of the same embryos. Our Validation shows that except paws and tails we have been able to segment the mouse embryos with minor error. CONCLUSIONS: This paper describes our novel multiple object segmentation technique with collision detection using a modified deformable model algorithm. Further, it presents the results of segmenting magnetic resonance images of up to 32 mouse embryos stacked in one gel filled test tube and creating 32 individual masks.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="172055" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1371/journal.pone.0023376" data-uuid="f8fc95d0-234c-af35-dae9-1678612037fc"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1371/journal.pone.0023376" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Web of Science Lite"><p class="cited">Cited:</p><p class="score">34</p><p class="cited">WOS</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/21853122#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Chen CM</span><span class="author ">, Krohn J</span><span class="author ">, Bhattacharya S</span><span class="author ">, Davies B</span>.</span>		2011.		A comparison of exogenous promoter activity at the ROSA26 locus using a ΦiC31 integrase mediated cassette exchange approach in mouse ES cells.		<span class="journal-title">PLoS One</span>,		<span class="journal-volume">6</span>		(8),		pp. e23376.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1371/journal.pone.0023376" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-172055" class="pub_abstract" style="display: none;">
		The activities of nine ubiquitous promoters (ROSA26, CAG, CMV, CMVd1, UbC, EF1α, PGK, chicken β-actin and MC1) have been quantified and compared in mouse embryonic stem cells. To avoid the high variation in transgene expression which results from uncontrolled copy number and chromosomal position effects when using random insertion based transgenic approaches, we have adopted a PhiC31 integrase mediated cassette exchange method for the efficient insertion of transgenes at single copy within a defined and well characterized chromosomal position, ROSA26. This has enabled the direct comparison of constructs from within the same genomic context and allows a systematic and quantitative assessment of the strengths of the promoters in comparison with the endogenous ROSA26 promoter. The behavior of these exogenous promoters, when integrated at ROSA26 in both sense and antisense orientations, reveals a large variation in their levels of activity. In addition, a subset of promoters, EF1α, UbC and CAG, show an increased activity in the sense orientation as a consequence of integration. Transient transfection experiments confirmed these observations to reflect integration dependent effects and also revealed significant differences in the behaviour of these promoters when delivered transiently or stably. As well as providing an important reference which will facilitate the choice of an appropriate promoter to achieve the desired level of expression for a specific research question, this study also demonstrates the suitability of the cassette exchange methodology for the robust and reliable expression of multiple variant transgenes in ES cells.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="149749" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1038/ng.825" data-uuid="be8e48ad-9f24-1931-5cde-b0f9b854c118"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=877611" style="display:inline-block;">
  <img alt="Article has an altmetric score of 1" src="https://badges.altmetric.com/?size=128&amp;score=1&amp;types=tttttttt" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Web of Science Lite"><p class="cited">Cited:</p><p class="score">73</p><p class="cited">WOS</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/21532573#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Engelen E</span><span class="author ">, Akinci U</span><span class="author ">, Bryne JC</span><span class="author ">, Hou J</span><span class="author ">, Gontan C</span><span class="author ">, Moen M</span><span class="author ">, Szumska D</span><span class="author ">, Kockx C</span><span class="author ">, van Ijcken W</span><span class="author ">, Dekkers DH</span><span class="author hidden" style="display: none;">, Demmers J</span><span class="author hidden" style="display: none;">, Rijkers EJ</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Philipsen S</span><span class="author hidden" style="display: none;">, Pevny LH</span><span class="author hidden" style="display: none;">, Grosveld FG</span><span class="author hidden" style="display: none;">, Rottier RJ</span><span class="author hidden" style="display: none;">, Lenhard B</span><span class="author hidden" style="display: none;">, Poot RA</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2011.		Sox2 cooperates with Chd7 to regulate genes that are mutated in human syndromes.		<span class="journal-title">Nat Genet</span>,		<span class="journal-volume">43</span>		(6),		pp. 607-611.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1038/ng.825" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-149749" class="pub_abstract" style="display: none;">
		The HMG-box transcription factor Sox2 plays a role throughout neurogenesis and also acts at other stages of development, as illustrated by the multiple organs affected in the anophthalmia syndrome caused by SOX2 mutations. Here we combined proteomic and genomic approaches to characterize gene regulation by Sox2 in neural stem cells. Chd7, a chromatin remodeling ATPase associated with CHARGE syndrome, was identified as a Sox2 transcriptional cofactor. Sox2 and Chd7 physically interact, have overlapping genome-wide binding sites and regulate a set of common target genes including Jag1, Gli3 and Mycn, genes mutated in Alagille, Pallister-Hall and Feingold syndromes, which show malformations also associated with SOX2 anophthalmia syndrome or CHARGE syndrome. Regulation of disease-associated genes by a Sox2-Chd7 complex provides a plausible explanation for several malformations associated with SOX2 anophthalmia syndrome or CHARGE syndrome. Indeed, we found that Chd7-haploinsufficient embryos showed severely reduced expression of Jag1 in the developing inner ear.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="110422" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1093/hmg/ddq457" data-uuid="b9b7bde8-294f-c21a-5d16-3bc494add955"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1093/hmg/ddq457" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Cross SH</span><span class="author ">, McKie L</span><span class="author ">, West K</span><span class="author ">, Coghill EL</span><span class="author ">, Favor J</span><span class="author ">, Bhattacharya S</span><span class="author ">, Brown SD</span><span class="author ">, Jackson IJ</span>.</span>		2011.		The Opdc missense mutation of Pax2 has a milder than loss-of-function phenotype.		<span class="journal-title">Hum Mol Genet</span>,		<span class="journal-volume">20</span>		(2),		pp. 223-234.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1093/hmg/ddq457" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-110422" class="pub_abstract" style="display: none;">
		Renal-coloboma syndrome, also known as papillorenal syndrome, is an autosomal dominant human disorder in which optic disc coloboma is associated with kidney abnormalities. Mutations in the paired domain transcription factor PAX2 have been found to be the underlying cause of this disease. Disease severity varies between patients, and in some cases, renal hypoplasia has been found in the absence of any retinal defects. Here we report an N-ethyl-N-nitrosourea-induced mouse mutation, Opdc, which is an isoleucinetothreonine missense mutation, I40T, in the first α-helix of the Pax2 paired domain. The mutant protein binds target DNA sequences less strongly than the wild-type protein and acts poorly to transactivate target promoters in culture. The phenotypic consequence of this mutation on the development of the eye and ear is similar to that reported for null alleles of Pax2. However, in homozygotes, cerebellar development is normal on a genetic background in which loss of Pax2 results in failure of cerebellar formation. Moreover, there is a genetic background effect on the heterozygous phenotype such that on some strain backgrounds, kidney development is unaffected. Opdc is the first hypomorphic mutation reported for Pax2 that differs in phenotype from loss-of-function mutations. These results suggest that PAX2 is a strong candidate gene for cases in which human patients have optic disc coloboma not associated with renal dysplasia.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="105953" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1152/physiolgenomics.00091.2010" data-uuid="40f11c42-7d0e-5c18-e936-80a729949143"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1152/physiolgenomics.00091.2010" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">27</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/20682847#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Zamyadi M</span><span class="author ">, Baghdadi L</span><span class="author ">, Lerch JP</span><span class="author ">, Bhattacharya S</span><span class="author ">, Schneider JE</span><span class="author ">, Henkelman RM</span><span class="author ">, Sled JG</span>.</span>		2010.		Mouse embryonic phenotyping by morphometric analysis of MR images.		<span class="journal-title">Physiol Genomics</span>,		<span class="journal-volume">42A</span>		(2),		pp. 89-95.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1152/physiolgenomics.00091.2010" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-105953" class="pub_abstract" style="display: none;">
		A new method is described for automatic detection of subtle morphological phenotypes in mouse embryos. Based on high-resolution magnetic resonance imaging scanning and nonlinear image alignment, this method is demonstrated by comparing the morphology of two inbred strains, C57BL/6J and 129Sv/S1ImJ, at 15.5 days postconception. Mouse embryo morphology was found to be highly amenable to this kind of analysis with very low levels (on average 110 μm) of residual anatomical variation within strains after linear differences in pose and scale are removed. Mapping of local size differences showed that C57BL/6J embryos were larger than 129Sv/S1ImJ embryos, although these differences were not uniformly distributed across the anatomy. Expressed in terms of organ volumes, heart and lung were larger in C57BL/6J embryos, while brain and liver were comparable in volume between strains. The positive relationship between organ size and embryo size was consistent for the two strains but differed by organ, with the brain and liver being the least variable. Together these findings suggest the power of this technique for detecting subtle phenotypic differences arising from mutated genes.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="105186" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1093/hmg/ddq251" data-uuid="49fe09b4-ae84-a1f9-4234-fb48e108696a"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1093/hmg/ddq251" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Bentham J</span><span class="author ">, Michell AC</span><span class="author ">, Lockstone H</span><span class="author ">, Andrew D</span><span class="author ">, Schneider JE</span><span class="author ">, Brown NA</span><span class="author ">, Bhattacharya S</span>.</span>		2010.		Maternal high-fat diet interacts with embryonic Cited2 genotype to reduce Pitx2c expression and enhance penetrance of left-right patterning defects.		<span class="journal-title">Hum Mol Genet</span>,		<span class="journal-volume">19</span>		(17),		pp. 3394-3401.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1093/hmg/ddq251" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-105186" class="pub_abstract" style="display: none;">
		Deficiency of the transcription factor Cited2 in mice results in cardiac malformation, adrenal agenesis, neural tube, placental defects and partially penetrant cardiopulmonary laterality defects resulting from an abnormal Nodal-&gt;Pitx2c pathway. Here we show that a maternal high-fat diet more than doubles the penetrance of laterality defects and, surprisingly, induces palatal clefting in Cited2-deficient embryos. Both maternal diet and Cited2 deletion reduce embryo weight and kidney and thymus volume. Expression profiling identified 40 embryonic transcripts including Pitx2 that were significantly affected by embryonic genotype-maternal diet interaction. We show that a high-fat diet reduces Pitx2c levels &gt;2-fold in Cited2-deficient embryos. Taken together, these results define a novel interaction between maternal high-fat diet and embryonic Cited2 deficiency that affects Pitx2c expression and results in abnormal laterality. They suggest that appropriate modifications of maternal diet may prevent such defects in humans.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="105267" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1002/dvdy.22334" data-uuid="61983f8a-25c0-86a8-747b-f81e83618572"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1002/dvdy.22334" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Michell AC</span><span class="author ">, Bragança J</span><span class="author ">, Broadbent C</span><span class="author ">, Joyce B</span><span class="author ">, Franklyn A</span><span class="author ">, Schneider JE</span><span class="author ">, Bhattacharya S</span><span class="author ">, Bamforth SD</span>.</span>		2010.		A novel role for transcription factor Lmo4 in thymus development through genetic interaction with Cited2.		<span class="journal-title">Dev Dyn</span>,		<span class="journal-volume">239</span>		(7),		pp. 1988-1994.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1002/dvdy.22334" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-105267" class="pub_abstract" style="display: none;">
		Deletion of the transcriptional modulator Cited2 in the mouse results in embryonic lethality, cardiovascular malformations, adrenal agenesis, cranial ganglia fusion, exencephaly, and left-right patterning defects, all seen with a varying degree of penetrance. The phenotypic heterogeneity, observed on different genetic backgrounds, indicates the existence of both genetic and environmental modifiers. Mice lacking the LIM domain-containing protein Lmo4 share specific phenotypes with Cited2 null embryos, such as embryonic lethality, cranial ganglia fusion, and exencephaly. These shared phenotypes suggested that Lmo4 may be a potential genetic modifier of the Cited2 phenotype. Examination of Lmo4-deficient embryos revealed partially penetrant cardiovascular malformations and hypoplastic thymus. Examination of Lmo4;Cited2 compound mutants indicated that there is a genetic interaction between Cited2 and Lmo4 in control of thymus development. Our data suggest that this may occur, in part, through control of expression of a common target gene, Tbx1, which is necessary for normal thymus development.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="105926" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1002/mrm.22352" data-uuid="9dbd2911-6291-8656-6b49-f02f05642e15"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1002/mrm.22352" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Zhang X</span><span class="author ">, Schneider JE</span><span class="author ">, Portnoy S</span><span class="author ">, Bhattacharya S</span><span class="author ">, Henkelman RM</span>.</span>		2010.		Comparative SNR for high-throughput mouse embryo MR microscopy.		<span class="journal-title">Magn Reson Med</span>,		<span class="journal-volume">63</span>		(6),		pp. 1703-1707.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1002/mrm.22352" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-105926" class="pub_abstract" style="display: none;">
		MR microscopy is being explored as a useful imaging tool to phenotype mouse embryos due to its volume coverage with three-dimensional isotropic resolution. However, the main limitation for mouse embryo MR microscopy is the signal-to-noise ratio. Large numbers of embryos are needed for phenotypic screening, making high throughput essential. Two high-throughput imaging approaches, multi-embryo shared-coil (shared) and multi-embryo individual-coil (individual), have been developed for phenotyping mouse embryos. This study quantitatively compares the signal-to-noise ratio at equivalent times between these two established methods by compensating for differences that result from field strength. While the individual method provides 3.3 times as much signal-to-noise ratio as the shared method at equivalent conditions, it is more difficult and expensive to implement. Furthermore, the number of embryos that can be imaged concurrently is limited by the number of receiver channels. The objective of this study is to provide measured comparative data to guide choices for high-throughput mouse embryo MR microscopy and other similar applications.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="105921" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1186/1471-213X-10-54" data-uuid="420e36f1-1ce0-732d-3627-29c39dd8c99f"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1186/1471-213x-10-54" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Stevens J</span><span class="author ">, Ermakov A</span><span class="author ">, Braganca J</span><span class="author ">, Hilton H</span><span class="author ">, Underhill P</span><span class="author ">, Bhattacharya S</span><span class="author ">, Brown NA</span><span class="author ">, Norris DP</span>.</span>		2010.		Analysis of the asymmetrically expressed Ablim1 locus reveals existence of a lateral plate Nodal-independent left sided signal and an early, left-right independent role for nodal flow.		<span class="journal-title">BMC Dev Biol</span>,		<span class="journal-volume">10</span>		(1),		pp. 54.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1186/1471-213X-10-54" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-105921" class="pub_abstract" style="display: none;">
		BACKGROUND: Vertebrates show clear asymmetry in left-right (L-R) patterning of their organs and associated vasculature. During mammalian development a cilia driven leftwards flow of liquid leads to the left-sided expression of Nodal, which in turn activates asymmetric expression of the transcription factor Pitx2. While Pitx2 asymmetry drives many aspects of asymmetric morphogenesis, it is clear from published data that additional asymmetrically expressed loci must exist. RESULTS: A L-R expression screen identified the cytoskeletally-associated gene, actin binding lim protein 1 (Ablim1), as asymmetrically expressed in both the node and left lateral plate mesoderm (LPM). LPM expression closely mirrors that of Nodal. Significantly, Ablim1 LPM asymmetry was detected in the absence of detectable Nodal. In the node, Ablim1 was initially expressed symmetrically across the entire structure, resolving to give a peri-nodal ring at the headfold stage in a flow and Pkd2-dependent manner. The peri-nodal ring of Ablim1 expression became asymmetric by the mid-headfold stage, showing stronger right than left-sided expression. Node asymmetry became more apparent as development proceeded; expression retreated in an anticlockwise direction, disappearing first from the left anterior node. Indeed, at early somite stages Ablim1 shows a unique asymmetric expression pattern, in the left lateral plate and to the right side of the node. CONCLUSION: Left LPM Ablim1 is expressed in the absence of detectable LPM Nodal, clearly revealing existence of a Pitx2 and Nodal-independent left-sided signal in mammals. At the node, a previously unrecognised action of early nodal flow and Pkd2 activity, within the pit of the node, influences gene expression in a symmetric manner. Subsequent Ablim1 expression in the peri-nodal ring reveals a very early indication of L-R asymmetry. Ablim1 expression analysis at the node acts as an indicator of nodal flow. Together these results make Ablim1 a candidate for controlling aspects of L-R identity and patterning.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="105930" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1016/j.ydbio.2010.01.020" data-uuid="547d5cd7-0911-4676-8e30-75fa585afbb0"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1016/j.ydbio.2010.01.020" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">34</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/20122914#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">van Bueren KL</span><span class="author ">, Papangeli I</span><span class="author ">, Rochais F</span><span class="author ">, Pearce K</span><span class="author ">, Roberts C</span><span class="author ">, Calmont A</span><span class="author ">, Szumska D</span><span class="author ">, Kelly RG</span><span class="author ">, Bhattacharya S</span><span class="author ">, Scambler PJ</span>.</span>		2010.		Hes1 expression is reduced in Tbx1 null cells and is required for the development of structures affected in 22q11 deletion syndrome.		<span class="journal-title">Dev Biol</span>,		<span class="journal-volume">340</span>		(2),		pp. 369-380.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1016/j.ydbio.2010.01.020" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-105930" class="pub_abstract" style="display: none;">
		22q11 deletion syndrome (22q11DS) is characterised by aberrant development of the pharyngeal apparatus and the heart with haploinsufficiency of the transcription factor TBX1 being considered the major underlying cause of the disease. Tbx1 mutations in mouse phenocopy the disorder. In order to identify the transcriptional dysregulation in Tbx1-expressing lineages we optimised fluorescent-activated cell sorting of beta-galactosidase expressing cells (FACS-Gal) to compare the expression profile of Df1/Tbx1(lacZ) (effectively Tbx1 null) and Tbx1 heterozygous cells isolated from mouse embryos. Hes1, a major effector of Notch signalling, was identified as downregulated in Tbx1(-)(/)(-) mutants. Hes1 mutant mice exhibited a partially penetrant range of 22q11DS-like defects including pharyngeal arch artery (PAA), outflow tract, craniofacial and thymic abnormalities. Similar to Tbx1 mice, conditional mutagenesis revealed that Hes1 expression in embryonic pharyngeal ectoderm contributes to thymus and pharyngeal arch artery development. These results suggest that Hes1 acts downstream of Tbx1 in the morphogenesis of pharyngeal-derived structures.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="105256" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1007/s00246-009-9610-3" data-uuid="df306587-02bc-bec6-1198-8463befd9ff7"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=4860169" style="display:inline-block;">
  <img alt="Article has an altmetric score of 3" src="https://badges.altmetric.com/?size=128&amp;score=3&amp;types=wwwwwwww" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">Chen CM</span><span class="author ">, Norris D</span><span class="author ">, Bhattacharya S</span>.</span>		2010.		Transcriptional control of left-right patterning in cardiac development.		<span class="journal-title">Pediatr Cardiol</span>,		<span class="journal-volume">31</span>		(3),		pp. 371-377.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1007/s00246-009-9610-3" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-105256" class="pub_abstract" style="display: none;">
		The heart develops from a simple left-right (L-R) symmetrical tube. Through a complex process of looping and remodelling, it becomes a highly L-R asymmetrical organ with distinct asymmetries in both morphology and function. Abnormal cardiac L-R patterning can result in a spectrum of defects that include, dextrocardia (a malposition of the heart to the right), isomerism of the atria (both atria being morphologically right-sided or left-sided), abnormal ventricular topology (e.g. the morphological left ventricle being dextral to the morphological right ventricle) or mirror-image topology (associated with situs inversus). Intermediate forms include abnormalities such as situs ambiguus and heterotaxia. L-R patterning abnormalities are typically associated with cardiac malformations, and it has become clear that an isolated septal, outflow tract and aortic arch malformation may be the only presenting manifestation of an L-R patterning defect. In the last two decades, there have been seminal advances in our understanding of the mechanisms controlling L-R patterning, and how mutations in L-R patterning genes result in human cardiac malformation. In this review, we provide an overview of the transcriptional mechanisms that result in asymmetric gene activation in mammals, how they receive information from signalling pathways, and how this translates to abnormal cardiac development.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="426201" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1016/B978-0-12-381332-9.00036-0" data-uuid="6619a4be-635f-b880-b883-ab0f168af64f"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1016/b978-0-12-381332-9.00036-0" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Mohun T</span><span class="author ">, Weninger W</span><span class="author ">, Bhattacharya S</span>.</span>		2010.		Imaging Cardiac Developmental Malformations in the Mouse Embryo								pp. 779-791.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1016/B978-0-12-381332-9.00036-0" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-426201" class="pub_abstract" style="display: none;">
		This chapter discusses imaging cardiac developmental malfunctions in the mouse embryo. Studies of human genetic architecture cannot provide a mechanistic understanding of gene function in cardiac development, and analysis of gene-gene and gene-environment interactions in humans remains a challenge. These considerations necessitate the use of animal models, the most appropriate and widely used being the mouse. Like the human, the mouse has a four-chambered heart with a septated outflow tract, left-sided great arteries, and parallel pulmonary and systemic circulations. Common cardiac malformations such as septal, outflow tract, and aortic arch and pulmonary trunk defects can thus be identified in mouse embryos, and mouse mutations typically recapitulate the cardiac malformations observed in patients with mutations in the same genes (reviewed in Schneider and Bhattacharya, 2004). Imaging such a complex and dynamic structure as the developing heart plays a crucial role in such studies, underlying accurate assessment of phenotype. New anatomical imaging techniques such as optical projection tomography, high resolution episcopic microscopy and magnetic resonance imaging provide ways to supplement conventional histology with comprehensive image data, yielding three-dimensional models that allow systematic, quantitative comparisons of structure. These imaging methods have complementary strengths, and by combining them it is possible to envisage a powerful cardiac phenotyping pipeline. © 2010 Elsevier Inc. All rights reserved.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="104244" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1371/journal.pgen.1001000" data-uuid="40cfa1b5-c9b6-6d08-35ad-f1e11b151f88"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=1355759" style="display:inline-block;">
  <img alt="Article has an altmetric score of 6" src="https://badges.altmetric.com/?size=128&amp;score=6&amp;types=bbbbbbbb" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Web of Science Lite"><p class="cited">Cited:</p><p class="score">45</p><p class="cited">WOS</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/20585624#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Ashrafian H</span><span class="author ">, Docherty L</span><span class="author ">, Leo V</span><span class="author ">, Towlson C</span><span class="author ">, Neilan M</span><span class="author ">, Steeples V</span><span class="author ">, Lygate CA</span><span class="author ">, Hough T</span><span class="author ">, Townsend S</span><span class="author ">, Williams D</span><span class="author hidden" style="display: none;">, Wells S</span><span class="author hidden" style="display: none;">, Norris D</span><span class="author hidden" style="display: none;">, Glyn-Jones S</span><span class="author hidden" style="display: none;">, Land J</span><span class="author hidden" style="display: none;">, Barbaric I</span><span class="author hidden" style="display: none;">, Lalanne Z</span><span class="author hidden" style="display: none;">, Denny P</span><span class="author hidden" style="display: none;">, Szumska D</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Griffin JL</span><span class="author hidden" style="display: none;">, Hargreaves I</span><span class="author hidden" style="display: none;">, Fernandez-Fuentes N</span><span class="author hidden" style="display: none;">, Cheeseman M</span><span class="author hidden" style="display: none;">, Watkins H</span><span class="author hidden" style="display: none;">, Dear TN</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2010.		A mutation in the mitochondrial fission gene Dnm1l leads to cardiomyopathy.		<span class="journal-title">PLoS Genet</span>,		<span class="journal-volume">6</span>		(6),		pp. e1001000.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1371/journal.pgen.1001000" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-104244" class="pub_abstract" style="display: none;">
		Mutations in a number of genes have been linked to inherited dilated cardiomyopathy (DCM). However, such mutations account for only a small proportion of the clinical cases emphasising the need for alternative discovery approaches to uncovering novel pathogenic mutations in hitherto unidentified pathways. Accordingly, as part of a large-scale N-ethyl-N-nitrosourea mutagenesis screen, we identified a mouse mutant, Python, which develops DCM. We demonstrate that the Python phenotype is attributable to a dominant fully penetrant mutation in the dynamin-1-like (Dnm1l) gene, which has been shown to be critical for mitochondrial fission. The C452F mutation is in a highly conserved region of the M domain of Dnm1l that alters protein interactions in a yeast two-hybrid system, suggesting that the mutation might alter intramolecular interactions within the Dnm1l monomer. Heterozygous Python fibroblasts exhibit abnormal mitochondria and peroxisomes. Homozygosity for the mutation results in the death of embryos midway though gestation. Heterozygous Python hearts show reduced levels of mitochondria enzyme complexes and suffer from cardiac ATP depletion. The resulting energy deficiency may contribute to cardiomyopathy. This is the first demonstration that a defect in a gene involved in mitochondrial remodelling can result in cardiomyopathy, showing that the function of this gene is needed for the maintenance of normal cellular function in a relatively tissue-specific manner. This disease model attests to the importance of mitochondrial remodelling in the heart; similar defects might underlie human heart muscle disease.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="106107" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1128/MCB.00913-09" data-uuid="3e73f9f1-417a-5739-21bf-9016cda73045"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1128/mcb.00913-09" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">de Wit T</span><span class="author ">, Dekker S</span><span class="author ">, Maas A</span><span class="author ">, Breedveld G</span><span class="author ">, Knoch TA</span><span class="author ">, Langeveld A</span><span class="author ">, Szumska D</span><span class="author ">, Craig R</span><span class="author ">, Bhattacharya S</span><span class="author ">, Grosveld F</span><span class="author ">, Drabek D</span>.</span>		2010.		Tagged mutagenesis by efficient Minos-based germ line transposition.		<span class="journal-title">Mol Cell Biol</span>,		<span class="journal-volume">30</span>		(1),		pp. 68-77.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1128/MCB.00913-09" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-106107" class="pub_abstract" style="display: none;">
		Germ line gene transposition technology has been used to generate "libraries" of flies and worms carrying genomewide mutations. Phenotypic screening and DNA sequencing of such libraries provide functional information resulting from insertional events in target genes. There is also a great need to have a fast and efficient way to generate mouse mutants in vivo to model developmental defects and human diseases. Here we describe an optimized mammalian germ line transposition system active during early mouse spermatogenesis using the Minos transposon. Transposon-positive progeny carry on average more than 2 new transpositions, and 45 to 100% of the progeny carry an insertion in a gene. The optimized Minos-based system was tested in a small rapid dominant functional screen to identify mutated genes likely to cause measurable cardiovascular "disease" phenotypes in progeny/embryos. Importantly this system allows rapid screening for modifier genes.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="105941" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1016/j.stem.2009.11.001" data-uuid="fe6860b6-8e09-123f-274f-56a7650fb55d"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1016/j.stem.2009.11.001" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">52</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/19951693#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Kranc KR</span><span class="author ">, Schepers H</span><span class="author ">, Rodrigues NP</span><span class="author ">, Bamforth S</span><span class="author ">, Villadsen E</span><span class="author ">, Ferry H</span><span class="author ">, Bouriez-Jones T</span><span class="author ">, Sigvardsson M</span><span class="author ">, Bhattacharya S</span><span class="author ">, Jacobsen SE</span><span class="author ">, Enver T</span>.</span>		2009.		Cited2 is an essential regulator of adult hematopoietic stem cells.		<span class="journal-title">Cell Stem Cell</span>,		<span class="journal-volume">5</span>		(6),		pp. 659-665.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1016/j.stem.2009.11.001" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-105941" class="pub_abstract" style="display: none;">
		The regulatory pathways necessary for the maintenance of adult hematopoietic stem cells (HSCs) remain poorly defined. By using loss-of-function approaches, we report a selective and cell-autonomous requirement for the p300/CBP-binding transcriptional coactivator Cited2 in adult HSC maintenance. Conditional deletion of Cited2 in the adult mouse results in loss of HSCs causing multilineage bone marrow failure and increased lethality. In contrast, conditional ablation of Cited2 after lineage specification in lymphoid and myeloid lineages has no impact on the maintenance of these lineages. Additional deletion of Ink4a/Arf (encoding p16(Ink4a) and p19(Arf)) or Trp53 (encoding p53, a downstream target of p19(Arf)) in a Cited2-deficient background restores HSC functionality and rescues mice from bone marrow failure. Furthermore, we show that the critical role of Cited2 in primitive hematopoietic cells is conserved in humans. Taken together, our studies provide genetic evidence that Cited2 selectively maintains adult HSC functions, at least in part, via Ink4a/Arf and Trp53.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="110450" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1002/dvdy.21874" data-uuid="488bb541-2fb5-8a0f-5dc0-04b1bdfafe8f"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1002/dvdy.21874" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">23</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/19235720#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Ermakov A</span><span class="author ">, Stevens JL</span><span class="author ">, Whitehill E</span><span class="author ">, Robson JE</span><span class="author ">, Pieles G</span><span class="author ">, Brooker D</span><span class="author ">, Goggolidou P</span><span class="author ">, Powles-Glover N</span><span class="author ">, Hacker T</span><span class="author ">, Young SR</span><span class="author hidden" style="display: none;">, Dear N</span><span class="author hidden" style="display: none;">, Hirst E</span><span class="author hidden" style="display: none;">, Tymowska-Lalanne Z</span><span class="author hidden" style="display: none;">, Briscoe J</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Norris DP</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2009.		Mouse mutagenesis identifies novel roles for left-right patterning genes in pulmonary, craniofacial, ocular, and limb development.		<span class="journal-title">Dev Dyn</span>,		<span class="journal-volume">238</span>		(3),		pp. 581-594.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1002/dvdy.21874" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-110450" class="pub_abstract" style="display: none;">
		Vertebrate organs show consistent left-right (L-R) asymmetry in placement and patterning. To identify genes involved in this process we performed an ENU-based genetic screen. Of 135 lines analyzed 11 showed clear single gene defects affecting L-R patterning, including 3 new alleles of known L-R genes and mutants in novel L-R loci. We identified six lines (termed "gasping") that, in addition to abnormal L-R patterning and associated cardiovascular defects, had complex phenotypes including pulmonary agenesis, exencephaly, polydactyly, ocular and craniofacial malformations. These complex abnormalities are present in certain human disease syndromes (e.g., HYLS, SRPS, VACTERL). Gasping embryos also show defects in ciliogenesis, suggesting a role for cilia in these human congenital malformation syndromes. Our results indicate that genes controlling ciliogenesis and left-right asymmetry have, in addition to their known roles in cardiac patterning, major and unexpected roles in pulmonary, craniofacial, ocular and limb development with implications for human congenital malformation syndromes.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="105302" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1371/journal.pone.0004978" data-uuid="7bddc63b-4d2a-2b2a-c961-20eae0e639bb"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1371/journal.pone.0004978" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Griffin HR</span><span class="author ">, Hall DH</span><span class="author ">, Topf A</span><span class="author ">, Eden J</span><span class="author ">, Stuart AG</span><span class="author ">, Parsons J</span><span class="author ">, Peart I</span><span class="author ">, Deanfield JE</span><span class="author ">, O'Sullivan J</span><span class="author ">, Babu-Narayan SV</span><span class="author hidden" style="display: none;">, Gatzoulis MA</span><span class="author hidden" style="display: none;">, Bu'lock FA</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Bentham J</span><span class="author hidden" style="display: none;">, Farrall M</span><span class="author hidden" style="display: none;">, Granados Riveron J</span><span class="author hidden" style="display: none;">, Brook JD</span><span class="author hidden" style="display: none;">, Burn J</span><span class="author hidden" style="display: none;">, Cordell HJ</span><span class="author hidden" style="display: none;">, Goodship JA</span><span class="author hidden" style="display: none;">, Keavney B</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2009.		Genetic variation in VEGF does not contribute significantly to the risk of congenital cardiovascular malformation.		<span class="journal-title">PLoS One</span>,		<span class="journal-volume">4</span>		(3),		pp. e4978.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1371/journal.pone.0004978" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-105302" class="pub_abstract" style="display: none;">
		Several previous studies have investigated the role of common promoter variants in the vascular endothelial growth factor (VEGF) gene in causing congenital cardiovascular malformation (CVM). However, results have been discrepant between studies and no study to date has comprehensively characterised variation throughout the gene. We genotyped 771 CVM cases, of whom 595 had the outflow tract malformation Tetralogy of Fallot (TOF), and carried out TDT and case-control analyses using haplotype-tagging SNPs in VEGF. We carried out a meta-analysis of previous case-control or family-based studies that had typed VEGF promoter SNPs, which included an additional 570 CVM cases. To identify rare variants potentially causative of CVM, we carried out mutation screening in all VEGF exons and splice sites in 93 TOF cases. There was no significant effect of any VEGF haplotype-tagging SNP on the risk of CVM in our analyses of 771 probands. When the results of this and all previous studies were combined, there was no significant effect of the VEGF promoter SNPs rs699947 (OR 1.05 [95% CI 0.95-1.17]); rs1570360 (OR 1.17 [95% CI 0.99-1.26]); and rs2010963 (OR 1.04 [95% CI 0.93-1.16]) on the risk of CVM in 1341 cases. Mutation screening of 93 TOF cases revealed no VEGF coding sequence variants and no changes at splice consensus sequences. Genetic variation in VEGF appears to play a small role, if any, in outflow tract CVM susceptibility.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="110456" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.2337/db07-0337" data-uuid="286ddd58-dc6c-5d5c-5a6b-719cbee28f4b"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.2337/db07-0337" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">30</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/18477811#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Goldsworthy M</span><span class="author ">, Hugill A</span><span class="author ">, Freeman H</span><span class="author ">, Horner E</span><span class="author ">, Shimomura K</span><span class="author ">, Bogani D</span><span class="author ">, Pieles G</span><span class="author ">, Mijat V</span><span class="author ">, Arkell R</span><span class="author ">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Ashcroft FM</span><span class="author hidden" style="display: none;">, Cox RD</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2008.		Role of the transcription factor sox4 in insulin secretion and impaired glucose tolerance.		<span class="journal-title">Diabetes</span>,		<span class="journal-volume">57</span>		(8),		pp. 2234-2244.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.2337/db07-0337" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-110456" class="pub_abstract" style="display: none;">
		OBJECTIVES: To identify, map, clone, and functionally validate a novel mouse model for impaired glucose tolerance and insulin secretion. RESEARCH DESIGN AND METHODS: Haploinsufficiency of the insulin receptor and associated mild insulin resistance has been used to sensitize an N-ethyl-N-nitrosourea (ENU) screen to identify novel mutations resulting in impaired glucose tolerance and diabetes. The new impaired glucose tolerance 4 (IGT4) model was selected using an intraperitoneal glucose tolerance test and inheritance of the phenotype confirmed by generation of backcross progeny. Segregation of the phenotype was correlated with genotype information to map the location of the gene and candidates sequenced for mutations. The function of the SRY-related high mobility group (HMG)-box 4 (Sox4) gene in insulin secretion was tested using another ENU allele and by small interfering RNA silencing in insulinoma cells. RESULTS: We describe two allelic autosomal dominant mutations in the highly conserved HMG box of the transcription factor Sox4. Previously associated with pancreas development, Sox4 mutations in the adult mouse result in an insulin secretory defect, which exhibits impaired glucose tolerance in association with insulin receptor(+/-)-induced insulin resistance. Elimination of the Sox4 transcript in INS1 and Min6 cells resulted in the abolition of glucose-stimulated insulin release similar to that observed for silencing of the key metabolic enzyme glucokinase. Intracellular calcium measurements in treated cells indicate that this defect lies downstream of the ATP-sensitive K(+) channel (K(ATP) channel) and calcium influx. CONCLUSIONS: IGT4 represents a novel digenic model of insulin resistance coupled with an insulin secretory defect. The Sox4 gene has a role in insulin secretion in the adult beta-cell downstream of the K(ATP) channel.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="71830" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1101/gad.479408" data-uuid="ddfd3670-ae20-0776-e971-99669d87042f"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1101/gad.479408" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">72</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/18519639#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Szumska D</span><span class="author ">, Pieles G</span><span class="author ">, Essalmani R</span><span class="author ">, Bilski M</span><span class="author ">, Mesnard D</span><span class="author ">, Kaur K</span><span class="author ">, Franklyn A</span><span class="author ">, El Omari K</span><span class="author ">, Jefferis J</span><span class="author ">, Bentham J</span><span class="author hidden" style="display: none;">, Taylor JM</span><span class="author hidden" style="display: none;">, Schneider JE</span><span class="author hidden" style="display: none;">, Arnold SJ</span><span class="author hidden" style="display: none;">, Johnson P</span><span class="author hidden" style="display: none;">, Tymowska-Lalanne Z</span><span class="author hidden" style="display: none;">, Stammers D</span><span class="author hidden" style="display: none;">, Clarke K</span><span class="author hidden" style="display: none;">, Neubauer S</span><span class="author hidden" style="display: none;">, Morris A</span><span class="author hidden" style="display: none;">, Brown SD</span><span class="author hidden" style="display: none;">, Shaw-Smith C</span><span class="author hidden" style="display: none;">, Cama A</span><span class="author hidden" style="display: none;">, Capra V</span><span class="author hidden" style="display: none;">, Ragoussis J</span><span class="author hidden" style="display: none;">, Constam D</span><span class="author hidden" style="display: none;">, Seidah NG</span><span class="author hidden" style="display: none;">, Prat A</span><span class="author hidden" style="display: none;">, Bhattacharya S</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2008.		VACTERL/caudal regression/Currarino syndrome-like malformations in mice with mutation in the proprotein convertase Pcsk5.		<span class="journal-title">Genes Dev</span>,		<span class="journal-volume">22</span>		(11),		pp. 1465-1477.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1101/gad.479408" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-71830" class="pub_abstract" style="display: none;">
		We have identified an ethylnitrosourea (ENU)-induced recessive mouse mutation (Vcc) with a pleiotropic phenotype that includes cardiac, tracheoesophageal, anorectal, anteroposterior patterning defects, exomphalos, hindlimb hypoplasia, a presacral mass, renal and palatal agenesis, and pulmonary hypoplasia. It results from a C470R mutation in the proprotein convertase PCSK5 (PC5/6). Compound mutants (Pcsk5(Vcc/null)) completely recapitulate the Pcsk5(Vcc/Vcc) phenotype, as does an epiblast-specific conditional deletion of Pcsk5. The C470R mutation ablates a disulfide bond in the P domain, and blocks export from the endoplasmic reticulum and proprotein convertase activity. We show that GDF11 is cleaved and activated by PCSK5A, but not by PCSK5A-C470R, and that Gdf11-deficient embryos, in addition to having anteroposterior patterning defects and renal and palatal agenesis, also have a presacral mass, anorectal malformation, and exomphalos. Pcsk5 mutation results in abnormal expression of several paralogous Hox genes (Hoxa, Hoxc, and Hoxd), and of Mnx1 (Hlxb9). These include known Gdf11 targets, and are necessary for caudal embryo development. We identified nonsynonymous mutations in PCSK5 in patients with VACTERL (vertebral, anorectal, cardiac, tracheoesophageal, renal, limb malformation OMIM 192350) and caudal regression syndrome, the phenotypic features of which resemble the mouse mutation. We propose that Pcsk5, at least in part via GDF11, coordinately regulates caudal Hox paralogs, to control anteroposterior patterning, nephrogenesis, skeletal, and anorectal development.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="104872" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1196/annals.1420.003" data-uuid="d956508c-dcd6-f136-8b8f-73e7c21a5bb2"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1196/annals.1420.003" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">47</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/18375573#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Bentham J</span><span class="author ">, Bhattacharya S</span>.</span>		2008.		Genetic mechanisms controlling cardiovascular development.		<span class="journal-title">Ann N Y Acad Sci</span>,		<span class="journal-volume">1123</span>		(1),		pp. 10-19.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1196/annals.1420.003" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-104872" class="pub_abstract" style="display: none;">
		Congenital heart disease (CHD) is a major cause of childhood morbidity and death in the West; the incidence is approximately 1 in 145 live births. Mendelian and chromosomal syndromes account for approximately 20% of CHD. The genetic mechanisms underlying non-chromosomal or non-Mendelian "sporadic" CHD, which account for the remaining 80%, are poorly understood. The genetic architecture of sporadic CHD likely includes accumulation of rare nonsynonymous variants in cardiac developmental genes leading to mutational loading of cardiac developmental networks, copy number variation in cardiac developmental genes, and common variants that may not be obviously linked to cardiac development but may alter genetic buffering pathways (e.g., folate metabolism). The rare mutations typically associated with sporadic CHD likely arise from the severe decrease in reproductive fitness selecting against any CHD-causing gene variant. The resulting allelic heterogeneity reduces the power of genome-wide association studies for CHD. A complementary approach to the genetic analysis of CHD is to resequence candidate genes that have been shown to be necessary for mouse heart development. The number of such genes likely exceeds 1700. To identify these genes, we have developed an enabling technology (high-throughput magnetic resonance imaging of mouse embryos), which is used in combination with N-ethyl-N-nitrosourea/transposon mutagenesis and knockout techniques. Key future challenges now involve translating discoveries made in mouse models to human CHD genetics and understanding the mechanisms that create and disrupt genetic buffering. A long-term goal in CHD is to manipulate these pathways to enhance buffering and prevent disease in a manner analogous to the use of folate in preventing neural tube defects.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="110461" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1016/S0378-3782(07)70186-X" data-uuid="df7bff75-d30e-7afb-6ccd-42e168d4324e"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1016/s0378-3782(07)70186-x" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Asopa S</span><span class="author ">, Anthony FW</span><span class="author ">, Cagampang FRA</span><span class="author ">, Terroni PL</span><span class="author ">, Bhattacharya S</span><span class="author ">, Ohri SK</span><span class="author ">, Hanson MA</span>.</span>		2007.		Impact on the developmental profile of the murine heart by maternal protein restriction during pregnancy		<span class="journal-title">EARLY HUMAN DEVELOPMENT</span>,		<span class="journal-volume">83</span>				pp. S87-S87.				 | <a href="http://dx.doi.org/10.1016/S0378-3782(07)70186-X" target="_blank">Read more</a> 		<p class="clearfix"></p>
				</div></div>
						<div class="publication group all" symplectic_id="405742" style="display: none;"><div>
		<div class="altmetric-embed" data-uuid="cf5d5c28-b705-a68a-f759-8dd862147fb0"><img src="https://altmetric-badges.a.ssl.fastly.net/?size=64&amp;score=?&amp;types=????????" style="display: none;"></div>				<span class="journal-authors"><span class="author ">Macdonald ST</span><span class="author ">, Bamforth SD</span><span class="author ">, Farthing CR</span><span class="author ">, Franklyn AJ</span><span class="author ">, Broadbent C</span><span class="author ">, Schneider J</span><span class="author ">, Schwartz RJ</span><span class="author ">, Saga Y</span><span class="author ">, Bhattacharya S</span>.</span>		2007.		Epiblastic Cited2 function explains pleiotropy and penetrance of cardiac malformation resulting from its deficiency		<span class="journal-title">EUROPEAN HEART JOURNAL</span>,		<span class="journal-volume">28</span>				pp. 123-123.						<p class="clearfix"></p>
				</div></div>
						<div class="publication group all" symplectic_id="105154" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1111/j.1469-7580.2007.00746.x" data-uuid="67a2d71a-6914-06b5-1e26-a7b59ecf1cc4"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1111/j.1469-7580.2007.00746.x" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">32</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/17532797#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Pieles G</span><span class="author ">, Geyer SH</span><span class="author ">, Szumska D</span><span class="author ">, Schneider J</span><span class="author ">, Neubauer S</span><span class="author ">, Clarke K</span><span class="author ">, Dorfmeister K</span><span class="author ">, Franklyn A</span><span class="author ">, Brown SD</span><span class="author ">, Bhattacharya S</span><span class="author ">, Weninger WJ</span>.</span>		2007.		microMRI-HREM pipeline for high-throughput, high-resolution phenotyping of murine embryos.		<span class="journal-title">J Anat</span>,		<span class="journal-volume">211</span>		(1),		pp. 132-137.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1111/j.1469-7580.2007.00746.x" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-105154" class="pub_abstract" style="display: none;">
		Rapid and precise phenotyping analysis of large numbers of wild-type and mutant mouse embryos is essential for characterizing the genetic and epigenetic factors regulating embryogenesis. We present a novel methodology that permits precise high-throughput screening of the phenotype of embryos with both targeted and randomly generated mutations. To demonstrate the potential of this methodology we show embryo phenotyping results produced in a large-scale ENU-mutagenesis study. In essence this represents an analysis pipeline, which starts with simultaneous micro-magentic resonance imaging (microMRI) screening (voxel size: 25.4 x 25.4 x 24.4 microm) of 32 embryos in one run. Embryos with an indistinct phenotype are then cut into parts and suspect organs and structures are analysed with HREM (high-resolution episcopic microscopy). HREM is an imaging technique that employs 'positive' eosin staining and episcopic imaging for generating three-dimensional (3D) high-resolution (voxel size: 1.07 x 1.07 x 2 microm) digital data of near histological contrast and quality. The results show that our method guarantees the rapid availability of comprehensive phenotype information for high numbers of embryos in, if necessary, histological quality and detail. The combination of high-throughput microMRI with HREM provides an alternative screening pipeline with advantages over existing 3D phenotype screening methods as well as traditional histology. Thus, the microMRI-HREM phenotype analysis pipeline recommends itself as a routine tool for analysing the phenotype of transgenic and mutant embryos.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="110465" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1161/01.RES.0000259041.37059.8c" data-uuid="45cbcf4d-9fad-2b44-7cdf-e2f57399a6b1"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1161/01.res.0000259041.37059.8c" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">32</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/17272814#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Liang X</span><span class="author ">, Sun Y</span><span class="author ">, Schneider J</span><span class="author ">, Ding JH</span><span class="author ">, Cheng H</span><span class="author ">, Ye M</span><span class="author ">, Bhattacharya S</span><span class="author ">, Rearden A</span><span class="author ">, Evans S</span><span class="author ">, Chen J</span>.</span>		2007.		Pinch1 is required for normal development of cranial and cardiac neural crest-derived structures.		<span class="journal-title">Circ Res</span>,		<span class="journal-volume">100</span>		(4),		pp. 527-535.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1161/01.RES.0000259041.37059.8c" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-110465" class="pub_abstract" style="display: none;">
		Pinch1, an adaptor protein composed of 5 LIM domains, has been suggested to play an important role in multiple cellular processes. We found that Pinch1 is highly expressed in neural crest cells and their derivatives. To examine the requirement for Pinch1 in neural crest development, we generated neural crest conditional Pinch1 knockout mice using the Wnt1-Cre/loxP system. Neural crest conditional Pinch1 mutant embryos die perinatally from severe cardiovascular defects with an unusual aneurysmal common arterial trunk. Pinch1 mutants also exhibit multiple deficiencies in cranial neural crest-derived structures. Fate mapping demonstrated that initial migration of neural crest cells to the pharyngeal arch region occurs normally in the mutant embryos. However, in the cardiac outflow tract of mutants, neural crest cells exhibited hyperplasia and failed to differentiate into smooth muscle. Markedly increased apoptosis is observed in outflow tract cushions of mutants between embryonic days 11.5 and 13.5, likely contributing to the observed defects in cushion/valve remodeling and ventricular septation. Expression of transforming growth factor-beta(2), which plays a crucial role in outflow tract development, was decreased or absent in the outflow tract of the mutants. The decrease in transforming growth factor-beta(2) expression preceded neural crest cell death. Together, our results demonstrate that Pinch1 plays an essential role in neural crest development, perhaps in part through transforming growth factor-beta signaling.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="110466" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1093/hmg/ddl168" data-uuid="10d7bd58-430d-9fcb-0d99-6f45e406f6fa"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1093/hmg/ddl168" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">91</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/16825286#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Hart AW</span><span class="author ">, Morgan JE</span><span class="author ">, Schneider J</span><span class="author ">, West K</span><span class="author ">, McKie L</span><span class="author ">, Bhattacharya S</span><span class="author ">, Jackson IJ</span><span class="author ">, Cross SH</span>.</span>		2006.		Cardiac malformations and midline skeletal defects in mice lacking filamin A.		<span class="journal-title">Hum Mol Genet</span>,		<span class="journal-volume">15</span>		(16),		pp. 2457-2467.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1093/hmg/ddl168" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-110466" class="pub_abstract" style="display: none;">
		The X-linked gene filamin A (Flna) encodes a widely expressed actin-binding protein that crosslinks actin into orthogonal networks and interacts with a variety of other proteins including membrane proteins, integrins, transmembrane receptor complexes and second messengers, thus forming an important intracellular signalling scaffold. Heterozygous loss of function of human FLNA causes periventricular nodular heterotopia in females and is generally lethal (cause unknown) in hemizygous males. Missense FLNA mutations underlie a spectrum of disorders affecting both sexes that feature skeletal dysplasia accompanied by a variety of other abnormalities. Dilp2 is an X-linked male-lethal mouse mutation that was induced by N-ethyl-N-nitrosourea. We report here that Dilp2 is caused by a T-to-A transversion that converts a tyrosine codon to a stop codon in the Flna gene (Y2388X), leading to absence of the Flna protein and male lethality because of incomplete septation of the outflow tract of the heart, which produces common arterial trunk. A proportion of both male and female mutant mice have other cardiac defects including ventricular septal defect. In addition, mutant males have midline fusion defects manifesting as sternum and palate abnormalities. Carrier females exhibit milder sternum and palate defects and misshapen pupils. These results define crucial roles for Flna in development, demonstrate that X-linked male lethal mutations can be recovered from ENU mutagenesis screens and suggest possible explanations for lethality of human males hemizygous for null alleles of FLNA.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="110446" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1042/CS20060003" data-uuid="d92caa67-b936-80fa-aae5-c0ecf952d494"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1042/cs20060003" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">24</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/16764556#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Bhattacharya S</span><span class="author ">, Macdonald ST</span><span class="author ">, Farthing CR</span>.</span>		2006.		Molecular mechanisms controlling the coupled development of myocardium and coronary vasculature.		<span class="journal-title">Clin Sci (Lond)</span>,		<span class="journal-volume">111</span>		(1),		pp. 35-46.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1042/CS20060003" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-110446" class="pub_abstract" style="display: none;">
		Cardiac failure affects 1.5% of the adult population and is predominantly caused by myocardial dysfunction secondary to coronary vascular insufficiency. Current therapeutic strategies improve prognosis only modestly, as the primary cause -- loss of normally functioning cardiac myocytes -- is not being corrected. Adult cardiac myocytes are unable to divide and regenerate to any significant extent following injury. New cardiac myocytes are, however, created during embryogenesis from progenitor cells and then by cell division from existing cardiac myocytes. This process is intimately linked to the development of coronary vasculature from progenitors originating in the endothelium, the proepicardial organ and neural crest. In this review, we systematically evaluate approx. 90 mouse mutations that impair heart muscle growth during development. These studies provide genetic evidence for interactions between myocytes, endothelium and cells derived from the proepicardial organ and the neural crest that co-ordinate myocardial and coronary vascular development. Conditional knockout and transgenic rescue experiments indicate that Vegfa, Bmpr1a (ALK3), Fgfr1/2, Mapk14 (p38), Hand1, Hand2, Gata4, Zfpm2 (FOG2), Srf and Txnrd2 in cardiac myocytes, Rxra and Wt1 in the proepicardial organ, EfnB2, Tek, Mapk7, Pten, Nf1 and Casp8 in the endothelium, and Bmpr1a and Pax3 in neural crest cells are key molecules controlling myocardial development. Coupling of myocardial and coronary development is mediated by BMP (bone morphogenetic protein), FGF (fibroblast growth factor) and VEGFA (vascular endothelial growth factor A) signalling, and also probably involves hypoxia. Pharmacological targeting of these molecules and pathways could, in principle, be used to recreate the embryonic state and achieve coupled myocardial and coronary vascular regeneration in failing hearts.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="4709" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1007/s00335-005-0112-1" data-uuid="6dd9d264-1b90-18e5-3b3f-bb660955e07c"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1007/s00335-005-0112-1" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">65</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/16465593#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Solberg LC</span><span class="author ">, Valdar W</span><span class="author ">, Gauguier D</span><span class="author ">, Nunez G</span><span class="author ">, Taylor A</span><span class="author ">, Burnett S</span><span class="author ">, Arboledas-Hita C</span><span class="author ">, Hernandez-Pliego P</span><span class="author ">, Davidson S</span><span class="author ">, Burns P</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Hough T</span><span class="author hidden" style="display: none;">, Higgs D</span><span class="author hidden" style="display: none;">, Klenerman P</span><span class="author hidden" style="display: none;">, Cookson WO</span><span class="author hidden" style="display: none;">, Zhang Y</span><span class="author hidden" style="display: none;">, Deacon RM</span><span class="author hidden" style="display: none;">, Rawlins JN</span><span class="author hidden" style="display: none;">, Mott R</span><span class="author hidden" style="display: none;">, Flint J</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2006.		A protocol for high-throughput phenotyping, suitable for quantitative trait analysis in mice.		<span class="journal-title">Mamm Genome</span>,		<span class="journal-volume">17</span>		(2),		pp. 129-146.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1007/s00335-005-0112-1" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-4709" class="pub_abstract" style="display: none;">
		Whole-genome genetic association studies in outbred mouse populations represent a novel approach to identifying the molecular basis of naturally occurring genetic variants, the major source of quantitative variation between inbred strains of mice. Measuring multiple phenotypes in parallel on each mouse would make the approach cost effective, but protocols for phenotyping on a large enough scale have not been developed. In this article we describe the development and deployment of a protocol to collect measures on three models of human disease (anxiety, type II diabetes, and asthma) as well as measures of mouse blood biochemistry, immunology, and hematology. We report that the protocol delivers highly significant differences among the eight inbred strains (A/J, AKR/J, BALBc/J, CBA/J, C3H/HeJ, C57BL/6 J, DBA/2 J, and LP/J), the progenitors of a genetically heterogeneous stock (HS) of mice. We report the successful collection of multiple phenotypes from 2000 outbred HS animals. The phenotypes measured in the protocol form the basis of a large-scale investigation into the genetic basis of complex traits in mice designed to examine interactions between genes and between genes and environment, as well as the main effects of genetic variants on phenotypes.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="71812" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1073/pnas.0500584102" data-uuid="5c2f7fff-1c5d-ad17-e2d5-a022a2064023"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=145550" style="display:inline-block;">
  <img alt="Article has an altmetric score of 1" src="https://badges.altmetric.com/?size=128&amp;score=1&amp;types=11111111" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">24</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/16109771#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Bogani D</span><span class="author ">, Willoughby C</span><span class="author ">, Davies J</span><span class="author ">, Kaur K</span><span class="author ">, Mirza G</span><span class="author ">, Paudyal A</span><span class="author ">, Haines H</span><span class="author ">, McKeone R</span><span class="author ">, Cadman M</span><span class="author ">, Pieles G</span><span class="author hidden" style="display: none;">, Schneider JE</span><span class="author hidden" style="display: none;">, Bhattacharya S</span><span class="author hidden" style="display: none;">, Hardy A</span><span class="author hidden" style="display: none;">, Nolan PM</span><span class="author hidden" style="display: none;">, Tripodis N</span><span class="author hidden" style="display: none;">, Depew MJ</span><span class="author hidden" style="display: none;">, Chandrasekara R</span><span class="author hidden" style="display: none;">, Duncan G</span><span class="author hidden" style="display: none;">, Sharpe PT</span><span class="author hidden" style="display: none;">, Greenfield A</span><span class="author hidden" style="display: none;">, Denny P</span><span class="author hidden" style="display: none;">, Brown SD</span><span class="author hidden" style="display: none;">, Ragoussis J</span><span class="author hidden" style="display: none;">, Arkell RM</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2005.		Dissecting the genetic complexity of human 6p deletion syndromes by using a region-specific, phenotype-driven mouse screen.		<span class="journal-title">Proc Natl Acad Sci U S A</span>,		<span class="journal-volume">102</span>		(35),		pp. 12477-12482.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1073/pnas.0500584102" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-71812" class="pub_abstract" style="display: none;">
		Monosomy of the human chromosome 6p terminal region results in a variety of congenital malformations that include brain, craniofacial, and organogenesis abnormalities. To examine the genetic basis of these phenotypes, we have carried out an unbiased functional analysis of the syntenic region of the mouse genome (proximal Mmu13). A genetic screen for recessive mutations in this region recovered thirteen lines with phenotypes relevant to a variety of clinical conditions. These include two loci that cause holoprosencephaly, two that underlie anophthalmia, one of which also contributes to other craniofacial abnormalities such as microcephaly, agnathia, and palatogenesis defects, and one locus responsible for developmental heart and kidney defects. Analysis of heterozygous carriers of these mutations shows that a high proportion of these loci manifest with behavioral activity and sensorimotor deficits in the heterozygous state. This finding argues for the systematic, reciprocal phenotypic assessment of dominant and recessive mouse mutants. In addition to providing a resource of single gene mutants that model 6p-associated disorders, the work reveals unsuspected genetic complexity at this region. In particular, many of the phenotypes associated with 6p deletions can be elicited by mutation in one of a number of genes. This finding implies that phenotypes associated with contiguous gene deletion syndromes can result not only from dosage sensitivity of one gene in the region but also from the combined effect of monosomy for multiple genes that function within the same biological process.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="104860" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1038/ng1446" data-uuid="6655e8d0-627d-f9bc-4674-ee7b9f8f9018"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=3502041" style="display:inline-block;">
  <img alt="Article has an altmetric score of 3" src="https://badges.altmetric.com/?size=128&amp;score=3&amp;types=wwwwwwww" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">128</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/15475956#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Bamforth SD</span><span class="author ">, Bragança J</span><span class="author ">, Farthing CR</span><span class="author ">, Schneider JE</span><span class="author ">, Broadbent C</span><span class="author ">, Michell AC</span><span class="author ">, Clarke K</span><span class="author ">, Neubauer S</span><span class="author ">, Norris D</span><span class="author ">, Brown NA</span><span class="author hidden" style="display: none;">, Anderson RH</span><span class="author hidden" style="display: none;">, Bhattacharya S</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2004.		Cited2 controls left-right patterning and heart development through a Nodal-Pitx2c pathway.		<span class="journal-title">Nat Genet</span>,		<span class="journal-volume">36</span>		(11),		pp. 1189-1196.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1038/ng1446" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-104860" class="pub_abstract" style="display: none;">
		Malformations of the septum, outflow tract and aortic arch are the most common congenital cardiovascular defects and occur in mice lacking Cited2, a transcriptional coactivator of TFAP2. Here we show that Cited2(-/-) mice also develop laterality defects, including right isomerism, abnormal cardiac looping and hyposplenia, which are suppressed on a mixed genetic background. Cited2(-/-) mice lack expression of the Nodal target genes Pitx2c, Nodal and Ebaf in the left lateral plate mesoderm, where they are required for establishing laterality and cardiovascular development. CITED2 and TFAP2 were detected at the Pitx2c promoter in embryonic hearts, and they activate Pitx2c transcription in transient transfection assays. We propose that an abnormal Nodal-Pitx2c pathway represents a unifying mechanism for the cardiovascular malformations observed in Cited2(-/-) mice, and that such malformations may be the sole manifestation of a laterality defect.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="110444" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1158/0008-5472.CAN-04-0708" data-uuid="5b6eb68c-fd87-41ee-a4b2-876ca110fa9b"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1158/0008-5472.can-04-0708" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">38</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/15342390#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Fox SB</span><span class="author ">, Bragança J</span><span class="author ">, Turley H</span><span class="author ">, Campo L</span><span class="author ">, Han C</span><span class="author ">, Gatter KC</span><span class="author ">, Bhattacharya S</span><span class="author ">, Harris AL</span>.</span>		2004.		CITED4 inhibits hypoxia-activated transcription in cancer cells, and its cytoplasmic location in breast cancer is associated with elevated expression of tumor cell hypoxia-inducible factor 1alpha.		<span class="journal-title">Cancer Res</span>,		<span class="journal-volume">64</span>		(17),		pp. 6075-6081.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1158/0008-5472.CAN-04-0708" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-110444" class="pub_abstract" style="display: none;">
		The interaction of hypoxia-inducible factor 1alpha and the CH1 domain of the transcriptional coactivator p300/CBP is necessary for the expression of hypoxia responsive genes and tumor angiogenesis. The transcription factor CITED2 binds p300/CBP at the CH1 domain and functions as a negative regulator of hypoxia signaling by competing with hypoxia-inducible factor 1alpha. CITED4, a recently identified member of the CITED family, binds p300/CBP via the CH1 domain and functions as a coactivator for transcription factor AP-2. Here, we show that CITED4 blocks the binding of hypoxia-inducible factor 1alpha to p300 in vitro and inhibits hypoxia-inducible factor-1alpha transactivation and hypoxia-mediated reporter gene activation. These studies suggest that CITED4 might function as an inhibitor of hypoxia-inducible factor 1alpha. To explore the function of CITED4 in breast cancer, we determined its expression in normal, in situ and invasive breast cancers. We also correlated its expression in 286 invasive breast tumors with clinicopathological, hypoxia markers and survival. In contrast to the nuclear localization of CITED4 in normal breast tissue, breast tumors were characterized by cytoplasmic and nuclear localization. Nuclear CITED4 expression was significantly inversely associated with tumor hypoxia-inducible factor 1alpha (P &lt; 0.05), tumor size (P = 0.03), tumor grade (P = 0.0001), and Chalkley vessel count (P = 0.04). CITED4 showed no significant correlation with patient age (P = 0.45), estrogen receptor (P = 0.11), or epidermal growth factor receptor (P = 0.48). These results show that breast cancer development is characterized by either nuclear loss or cytoplasmic translocation of CITED4, with consequent loss of hypoxia-inducible factor-1alpha transcriptional antagonist activity. This may be an important mechanism by which tumors enhance hypoxia-inducible factor expression and result in an aggressive phenotype.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="105041" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1002/bdrc.20017" data-uuid="69b906fd-651f-3105-8f1f-e170f34e0d20"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1002/bdrc.20017" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">37</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/15495185#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Schneider JE</span><span class="author ">, Bhattacharya S</span>.</span>		2004.		Making the mouse embryo transparent: identifying developmental malformations using magnetic resonance imaging.		<span class="journal-title">Birth Defects Res C Embryo Today</span>,		<span class="journal-volume">72</span>		(3),		pp. 241-249.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1002/bdrc.20017" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-105041" class="pub_abstract" style="display: none;">
		Developmental malformations are a major cause of childhood mortality and are typically characterized by lesions that allow survival of the embryo through gestation. The genetics of developmental malformations are powerfully studied by using high-throughput, phenotype-driven screens (e.g., following zebrafish or mouse mutagenesis) or by genotype-driven studies using transgenic or knockout mice. With regard to either approach, the mouse is anatomically and phylogenetically closer to humans than any other genetically tractable model organism. This is particularly important in the cardiovascular and respiratory systems, which have unique mammalian features. The identification of murine models of developmental malformations is, however, hindered by the opacity of the late gestational mouse embryo. In this review, we describe recent advances in magnetic resonance imaging that make it possible to rapidly identify malformations in the developing mouse embryo with high efficiency.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="110472" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1016/j.cell.2004.07.025" data-uuid="2628db2d-9f3a-a9ad-ab77-d428b0789b99"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=151602" style="display:inline-block;">
  <img alt="Article has an altmetric score of 1" src="https://badges.altmetric.com/?size=128&amp;score=1&amp;types=11111111" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">437</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/15315760#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Kozar K</span><span class="author ">, Ciemerych MA</span><span class="author ">, Rebel VI</span><span class="author ">, Shigematsu H</span><span class="author ">, Zagozdzon A</span><span class="author ">, Sicinska E</span><span class="author ">, Geng Y</span><span class="author ">, Yu Q</span><span class="author ">, Bhattacharya S</span><span class="author ">, Bronson RT</span><span class="author hidden" style="display: none;">, Akashi K</span><span class="author hidden" style="display: none;">, Sicinski P</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2004.		Mouse development and cell proliferation in the absence of D-cyclins.		<span class="journal-title">Cell</span>,		<span class="journal-volume">118</span>		(4),		pp. 477-491.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1016/j.cell.2004.07.025" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-110472" class="pub_abstract" style="display: none;">
		D-type cyclins (cyclins D1, D2, and D3) are regarded as essential links between cell environment and the core cell cycle machinery. We tested the requirement for D-cyclins in mouse development and in proliferation by generating mice lacking all D-cyclins. We found that these cyclin D1(-/-)D2(-/-)D3(-/-) mice develop until mid/late gestation and die due to heart abnormalities combined with a severe anemia. Our analyses revealed that the D-cyclins are critically required for the expansion of hematopoietic stem cells. In contrast, cyclin D-deficient fibroblasts proliferate nearly normally but show increased requirement for mitogenic stimulation in cell cycle re-entry. We found that the proliferation of cyclin D1(-/-)D2(-/-)D3(-/-) cells is resistant to the inhibition by p16(INK4a), but it critically depends on CDK2. Lastly, we found that cells lacking D-cyclins display reduced susceptibility to the oncogenic transformation. Our results reveal the presence of alternative mechanisms that allow cell cycle progression in a cyclin D-independent fashion.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="105371" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1186/1471-213X-4-16" data-uuid="322580ff-13f5-082a-2044-be791422599d"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=4299943" style="display:inline-block;">
  <img alt="Article has an altmetric score of 6" src="https://badges.altmetric.com/?size=128&amp;score=6&amp;types=bbbbbbbb" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">101</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/15615595#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Schneider JE</span><span class="author ">, Böse J</span><span class="author ">, Bamforth SD</span><span class="author ">, Gruber AD</span><span class="author ">, Broadbent C</span><span class="author ">, Clarke K</span><span class="author ">, Neubauer S</span><span class="author ">, Lengeling A</span><span class="author ">, Bhattacharya S</span>.</span>		2004.		Identification of cardiac malformations in mice lacking Ptdsr using a novel high-throughput magnetic resonance imaging technique.		<span class="journal-title">BMC Dev Biol</span>,		<span class="journal-volume">4</span>				pp. 16.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1186/1471-213X-4-16" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-105371" class="pub_abstract" style="display: none;">
		BACKGROUND: Congenital heart defects are the leading non-infectious cause of death in children. Genetic studies in the mouse have been crucial to uncover new genes and signaling pathways associated with heart development and congenital heart disease. The identification of murine models of congenital cardiac malformations in high-throughput mutagenesis screens and in gene-targeted models is hindered by the opacity of the mouse embryo. RESULTS: We developed and optimized a novel method for high-throughput multi-embryo magnetic resonance imaging (MRI). Using this approach we identified cardiac malformations in phosphatidylserine receptor (Ptdsr) deficient embryos. These included ventricular septal defects, double-outlet right ventricle, and hypoplasia of the pulmonary artery and thymus. These results indicate that Ptdsr plays a key role in cardiac development. CONCLUSIONS: Our novel multi-embryo MRI technique enables high-throughput identification of murine models for human congenital cardiopulmonary malformations at high spatial resolution. The technique can be easily adapted for mouse mutagenesis screens and, thus provides an important new tool for identifying new mouse models for human congenital heart diseases.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="16766" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1128/MCB.23.21.7658-7666.2003" data-uuid="2859bacf-cb1a-2317-e80e-03dd826772b3"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1128/mcb.23.21.7658-7666.2003" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">64</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/14560011#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Kranc KR</span><span class="author ">, Bamforth SD</span><span class="author ">, Bragança J</span><span class="author ">, Norbury C</span><span class="author ">, van Lohuizen M</span><span class="author ">, Bhattacharya S</span>.</span>		2003.		Transcriptional coactivator Cited2 induces Bmi1 and Mel18 and controls fibroblast proliferation via Ink4a/ARF.		<span class="journal-title">Mol Cell Biol</span>,		<span class="journal-volume">23</span>		(21),		pp. 7658-7666.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1128/MCB.23.21.7658-7666.2003" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-16766" class="pub_abstract" style="display: none;">
		Cited2 (CBP/p300 interacting transactivator with ED-rich tail 2) is required for embryonic development, coactivation of transcription factor AP-2, and inhibition of hypoxia-inducible factor 1 transactivation. Cited2 is induced by multiple growth factors and cytokines and oncogenically transforms cells. Here, we show that the proliferation of Cited2(-/-) mouse embryonic fibroblasts ceases prematurely. This is associated with a reduction in growth fraction, senescent cellular morphology, and increased expression of the cell proliferation inhibitors p16(INK4a), p19(ARF), and p15(INK4b). Deletion of INK4a/ARF (encoding p16(INK4a) and p19(ARF)) completely rescued the defective proliferation of Cited2(-/-) fibroblasts. However, the deletion of INK4a/ARF did not rescue the embryonic malformations observed in Cited2(-/-) mice, indicating that INK4a/ARF-independent pathways are likely to be involved here. We found that Cited2(-/-) fibroblasts had reduced expression of the polycomb-group genes Bmi1 and Mel18, which function as INK4a/ARF and Hox repressors. Complementation with CITED2-expressing retrovirus enhanced proliferation, induced Bmi1/Mel18 expression, and decreased INK4a/ARF expression. Bmi1- and Mel18-expressing retroviruses enhanced the proliferation of Cited2(-/-) fibroblasts, indicating that they function downstream of Cited2. Our results provide genetic evidence that Cited2 controls the expression of INK4a/ARF and fibroblast proliferation, at least in part via the polycomb-group genes Bmi1 and Mel18.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="105501" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1016/S0092-8674(03)00645-7" data-uuid="7b93d902-f306-76cc-32d7-d76884c5cf07"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1016/s0092-8674(03)00645-7" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">491</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/12941272#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Geng Y</span><span class="author ">, Yu Q</span><span class="author ">, Sicinska E</span><span class="author ">, Das M</span><span class="author ">, Schneider JE</span><span class="author ">, Bhattacharya S</span><span class="author ">, Rideout WM</span><span class="author ">, Bronson RT</span><span class="author ">, Gardner H</span><span class="author ">, Sicinski P</span>.</span>		2003.		Cyclin E ablation in the mouse.		<span class="journal-title">Cell</span>,		<span class="journal-volume">114</span>		(4),		pp. 431-443.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1016/S0092-8674(03)00645-7" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-105501" class="pub_abstract" style="display: none;">
		E type cyclins (E1 and E2) are believed to drive cell entry into the S phase. It is widely assumed that the two E type cyclins are critically required for proliferation of all cell types. Here, we demonstrate that E type cyclins are largely dispensable for mouse development. However, endoreplication of trophoblast giant cells and megakaryocytes is severely impaired in the absence of cyclin E. Cyclin E-deficient cells proliferate actively under conditions of continuous cell cycling but are unable to reenter the cell cycle from the quiescent G(0) state. Molecular analyses revealed that cells lacking cyclin E fail to normally incorporate MCM proteins into DNA replication origins during G(0)--&gt;S progression. We also found that cyclin E-deficient cells are relatively resistant to oncogenic transformation. These findings define a molecular function for E type cyclins in cell cycle reentry and reveal a differential requirement for cyclin E in normal versus oncogenic proliferation.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="13341" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1038/nsb0703-501" data-uuid="8b135c72-d250-d8a7-3fc8-aa1a32db2c1c"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1038/nsb0703-501" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Bhattacharya S</span><span class="author ">, Ratcliffe PJ</span>.</span>		2003.		ExCITED about HIF.		<span class="journal-title">Nat Struct Biol</span>,		<span class="journal-volume">10</span>		(7),		pp. 501-503.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1038/nsb0703-501" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-13341" class="pub_abstract" style="display: none;">
		The structure of the CH1 domain of p300 in complex with the transactivation domain of CITED2 brings us one step closer toward understanding the molecular basis of the regulation of hypoxia response.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="110443" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1074/jbc.M208144200" data-uuid="c407f3b9-c3f5-36c9-43d1-9e5f7780a63e"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=3346489" style="display:inline-block;">
  <img alt="Article has an altmetric score of 3" src="https://badges.altmetric.com/?size=128&amp;score=3&amp;types=wwwwwwww" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">106</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/12586840#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Bragança J</span><span class="author ">, Eloranta JJ</span><span class="author ">, Bamforth SD</span><span class="author ">, Ibbitt JC</span><span class="author ">, Hurst HC</span><span class="author ">, Bhattacharya S</span>.</span>		2003.		Physical and functional interactions among AP-2 transcription factors, p300/CREB-binding protein, and CITED2.		<span class="journal-title">J Biol Chem</span>,		<span class="journal-volume">278</span>		(18),		pp. 16021-16029.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1074/jbc.M208144200" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-110443" class="pub_abstract" style="display: none;">
		The transcriptional co-activators and histone acetyltransferases p300/CREB-binding protein (CBP) interact with CITED2, a transcription factor AP-2 (TFAP2) co-activator. p300/CBP, CITED2, and TFAP2A are essential for normal neural tube and cardiac development. Here we show that p300 and CBP co-activate TFAP2A in the presence of CITED2. TFAP2A transcriptional activity was modestly impaired in p300(+/-) and CBP(+/-) mouse embryonic fibroblasts; this was rescued by ectopic expression of p300/CBP. p300, TFAP2A, and endogenous CITED2 could be co-immunoprecipitated from transfected U2-OS cells indicating that they can interact physically in vivo. CITED2 interacted with the dimerization domain of TFAP2C, which is highly conserved in TFAP2A/B. In mammalian two-hybrid experiments, full-length p300 and TFAP2A interacted only when CITED2 was co-transfected. N-terminal residues of TFAP2A, containing the transactivation domain, are both necessary and sufficient for interaction with p300, and this interaction was independent of CITED2. Consistent with this, N-terminal residues of TFAP2A were required for p300- and CITED2-dependent co-activation. A histone acetyltransferase-deficient p300 mutant (D1399Y) did not co-activate TFAP2A and did not affect the expression or cellular localization of TFAP2A or CITED2. In mammalian two-hybrid experiments p300D1399Y failed to interact with TFAP2A, explaining, at least in part, its failure to function as a co-activator. Our results suggest a model wherein interactions among TFAP2A, CITED2, and p300/CBP are necessary for TFAP2A-mediated transcriptional activation and for normal neural tube and cardiac development.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="105278" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1046/j.1469-7580.2003.00157.x" data-uuid="27d74452-a372-d69c-477e-2ded265315dd"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1046/j.1469-7580.2003.00157.x" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">38</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/12647873#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Schneider JE</span><span class="author ">, Bamforth SD</span><span class="author ">, Farthing CR</span><span class="author ">, Clarke K</span><span class="author ">, Neubauer S</span><span class="author ">, Bhattacharya S</span>.</span>		2003.		High-resolution imaging of normal anatomy, and neural and adrenal malformations in mouse embryos using magnetic resonance microscopy.		<span class="journal-title">J Anat</span>,		<span class="journal-volume">202</span>		(2),		pp. 239-247.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1046/j.1469-7580.2003.00157.x" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-105278" class="pub_abstract" style="display: none;">
		An efficient investigation of the effects of genetic or environmental manipulation on mouse development relies on the rapid and accurate screening of a substantial number of embryos for congenital malformations. Here we demonstrate that it is possible to examine normal organ development and identify malformations in mouse embryos by magnetic resonance microscopy in a substantially shorter time than by conventional histology. We imaged embryos in overnight runs of under 9 h, with an operator time of less than 1 h. In normal embryos we visualized the brain, spinal cord, ganglia, eyes, inner ear, pituitary, thyroid, thymus, trachea, bronchi, lungs, heart, kidneys, gonads, adrenals, oesophagus, stomach, intestines, spleen, liver and pancreas. Examination of the brain in embryos lacking the transcriptional coactivator Cited2 showed cerebellar and midbrain roof agenesis, in addition to exencephaly. In these embryos we were also able to detect agenesis of the adrenal gland. We confirmed all malformations by histological sectioning. Thus magnetic resonance microscopy can be used to rapidly identify developmental and organ malformations in mutant mouse embryos generated by transgenic techniques, in high-throughput mutagenesis screens, or in screens to identify teratogenic compounds and environmental factors contributing to developmental malformations.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="105072" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1007/s10334-003-0002-z" data-uuid="99464e42-4bb6-d5f3-d2cb-51154bce54cf"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1007/s10334-003-0002-z" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Web of Science Lite"><p class="cited">Cited:</p><p class="score">32</p><p class="cited">WOS</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/12695885#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Schneider JE</span><span class="author ">, Bamforth SD</span><span class="author ">, Grieve SM</span><span class="author ">, Clarke K</span><span class="author ">, Bhattacharya S</span><span class="author ">, Neubauer S</span>.</span>		2003.		High-resolution, high-throughput magnetic paragraph sign resonance imaging of mouse embryonic paragraph sign anatomy using a fast gradient-echo sequence.		<span class="journal-title">MAGMA</span>,		<span class="journal-volume">16</span>		(1),		pp. 43-51.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1007/s10334-003-0002-z" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-105072" class="pub_abstract" style="display: none;">
		Embryonic development in normal and genetically modified mice is commonly analysed by histological sectioning. This procedure is time-consuming, prone to artefact, and results in the loss of three-dimensional (3D) information. Magnetic resonance imaging (MRI) of embryos has the potential of noninvasively acquiring a complete 3D data set. Published methods have used spin-echo techniques with inherently high signal-to-noise ratio (SNR); however, they required either perfusion of the embryo with a contrast agent, or prolonged acquisition times to improve contrast and resolution. Here, we show that a standard preparation (i.e. paraformaldehyde fixation) of 15.5 days post-coitum embryos followed by MRI using a fast gradient-echo sequence with T(1)-weighting achieves high resolution and high throughput for investigating mouse embryonic anatomy. 3D data sets were acquired in overnight experiments (&lt;9 h) with an experimental resolution of approximately 25 microm(3). This spatial resolution is twofold higher than the values reported previously for comparable paraformaldehyde-fixed embryos, and it was obtained in less than a quarter of the time with sufficient SNR. Our approach combines speed, high resolution and contrast with a simple preparation technique and minimal operator time (&lt;1 h). It allows rapid routine 3D characterisation of normal and abnormal mouse embryonic anatomy.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="105040" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1016/S0022-2828(02)00291-2" data-uuid="9072f6a6-9b61-ac07-0f9e-2a73e6cfd318"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1016/s0022-2828(02)00291-2" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">58</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/12606262#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Schneider JE</span><span class="author ">, Bamforth SD</span><span class="author ">, Farthing CR</span><span class="author ">, Clarke K</span><span class="author ">, Neubauer S</span><span class="author ">, Bhattacharya S</span>.</span>		2003.		Rapid identification and 3D reconstruction of complex cardiac malformations in transgenic mouse embryos using fast gradient echo sequence magnetic resonance imaging.		<span class="journal-title">J Mol Cell Cardiol</span>,		<span class="journal-volume">35</span>		(2),		pp. 217-222.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1016/S0022-2828(02)00291-2" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-105040" class="pub_abstract" style="display: none;">
		Developmental malformations of the heart in mouse embryos are commonly studied by histological sectioning. This is slow, labour intensive, and results in the loss of three-dimensional (3D) information. Magnetic resonance studies of embryos typically use spin-echo sequences, using prolonged acquisition times (&gt;36 h) or perfusion with contrast agents to enhance resolution and contrast. This is technically difficult, and requires significant amounts of operator time. We imaged paraformaldehyde fixed embryos using a fast spoiled 3D gradient echo sequence with T(1)-weighting, in unattended overnight runs of less than 9 h. In wild-type embryos, we visualised normal cardiac structures, including cardiac chambers, the ventricular septum, primary and secondary atrial septa, valves, superior and inferior vena cava, aorta, pulmonary artery, and ductus arteriosus. In embryos lacking Cited2 (a transcriptional co-activator required for normal heart development), we identified cardiac malformations including atrial and ventricular septal defects, cono-truncal defects, and aortic arch malformations. We generated 3D reconstructions of normal and mutant hearts using contour identification and surface rendering computer software. The malformations were confirmed by histological sectioning. Our data indicate that fast gradient echo sequence magnetic resonance imaging can be used to rapidly and accurately identify complex cardiovascular malformations in transgenic and mutant mouse embryos.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="105999" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1007/s10334-003-0002-z" data-uuid="7d4ebefc-b6df-5637-ae1f-c059c4c159c1"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1007/s10334-003-0002-z" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Schneider JRE</span><span class="author ">, Bamforth SD</span><span class="author ">, Grieve SM</span><span class="author ">, Clarke K</span><span class="author ">, Bhattacharya S</span><span class="author ">, Neubauer S</span>.</span>		2003.		High-resolution, high-throughput magnetic resonance imaging of mouse embryonic anatomy using a fast gradient-echo sequence		<span class="journal-title">MAGMA Magnetic Resonance Materials in Physics, Biology and Medicine</span>,		<span class="journal-volume">16</span>		(1),		pp. 43-51.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1007/s10334-003-0002-z" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-105999" class="pub_abstract" style="display: none;">
		Embryonic development in normal and genetically modified mice is commonly analysed by histological sectioning. This procedure is time-consuming, prone to artefact, and results in the loss of three-dimensional (3D) information. Magnetic resonance imaging (MRI) of embryos has the potential of non-invasively acquiring a complete 3D data set. Published methods have used spin-echo techniques with inherently high signal-to-noise ratio (SNR); however, they required either perfusion of the embryo with a contrast agent, or prolonged acquisition times to improve contrast and resolution. Here, we show that a standard preparation (i.e. paraformaldehyde fixation) of 15.5 days post-coitum embryos followed by MRI using a fast gradient-echo sequence with T 1-weighting achieves high resolution and high throughput for investigating mouse embryonic anatomy. 3D data sets were acquired in overnight experiments (&lt;9 h) with an experimental resolution of approximately 25 μM3. This spatial resolution is twofold higher than the values reported previously for comparable paraformaldehyde-fixed embryos, and it was obtained in less than a quarter of the time with sufficient SNR. Our approach combines speed, high resolution and contrast with a simple preparation technique and minimal operator time (&lt;1 h). It allows rapid routine 3D characterisation of normal and abnormal mouse embryonic anatomy.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="110473" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1006/geno.2002.7005" data-uuid="ce5e26db-80c1-2de9-fea5-7d020a859eb1"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1006/geno.2002.7005" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">22</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/12504852#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Yahata T</span><span class="author ">, Takedatsu H</span><span class="author ">, Dunwoodie SL</span><span class="author ">, Bragança J</span><span class="author ">, Swingler T</span><span class="author ">, Withington SL</span><span class="author ">, Hur J</span><span class="author ">, Coser KR</span><span class="author ">, Isselbacher KJ</span><span class="author ">, Bhattacharya S</span><span class="author ">, Shioda T</span>.</span>		2002.		Cloning of mouse Cited4, a member of the CITED family p300/CBP-binding transcriptional coactivators: induced expression in mammary epithelial cells.		<span class="journal-title">Genomics</span>,		<span class="journal-volume">80</span>		(6),		pp. 601-613.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1006/geno.2002.7005" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-110473" class="pub_abstract" style="display: none;">
		The CITED family proteins bind to CBP/p300 transcriptional integrators through their conserved C-terminal acidic domain and function as coactivators. The 21-kDa mouse Cited4 protein, a novel member of the CITED family, interacted with CBP/p300 as well as isoforms of the TFAP2 transcription factor, coactivating TFAP2-dependent transcription. The cited4 gene consisted of only a single exon located on chromosome 4 at 56.5-56.8 cM flanked by marker genes kcnq4 and scml1. Expression of Cited4 protein was strong and selective in embryonic hematopoietic tissues and endothelial cells. In adult animals, Cited4 showed strong milk cycle-dependent induction in pregnant and lactating mammary epithelial cells. Strong induction of Cited4 expression was also observed in SCp2 mouse mammary epithelial cells during their prolactin-dependent in vitro differentiation. These results implied possible roles for Cited4 in regulation of gene expression during development and differentiation of blood cells, endothelial cells, and mammary epithelial cells.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="10301" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1074/jbc.C200273200" data-uuid="f14d2c84-5a52-7767-9dba-0ff53eace3bf"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=163364" style="display:inline-block;">
  <img alt="Article has an altmetric score of 4" src="https://badges.altmetric.com/?size=128&amp;score=4&amp;types=wwww1111" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">485</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/12042299#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Hewitson KS</span><span class="author ">, McNeill LA</span><span class="author ">, Riordan MV</span><span class="author ">, Tian YM</span><span class="author ">, Bullock AN</span><span class="author ">, Welford RW</span><span class="author ">, Elkins JM</span><span class="author ">, Oldham NJ</span><span class="author ">, Bhattacharya S</span><span class="author ">, Gleadle JM</span><span class="author hidden" style="display: none;">, Ratcliffe PJ</span><span class="author hidden" style="display: none;">, Pugh CW</span><span class="author hidden" style="display: none;">, Schofield CJ</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2002.		Hypoxia-inducible factor (HIF) asparagine hydroxylase is identical to factor inhibiting HIF (FIH) and is related to the cupin structural family.		<span class="journal-title">J Biol Chem</span>,		<span class="journal-volume">277</span>		(29),		pp. 26351-26355.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1074/jbc.C200273200" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-10301" class="pub_abstract" style="display: none;">
		Activity of the hypoxia-inducible factor (HIF) complex is controlled by oxygen-dependent hydroxylation of prolyl and asparaginyl residues. Hydroxylation of specific prolyl residues by 2-oxoglutarate (2-OG)-dependent oxygenases mediates ubiquitinylation and proteasomal destruction of HIF-alpha. Hydroxylation of an asparagine residue in the C-terminal transactivation domain (CAD) of HIF-alpha abrogates interaction with p300, preventing transcriptional activation. Yeast two-hybrid assays recently identified factor inhibiting HIF (FIH) as a protein that associates with the CAD region of HIF-alpha. Since FIH contains certain motifs present in iron- and 2-OG-dependent oxygenases we investigated whether FIH was the HIF asparaginyl hydroxylase. Assays using recombinant FIH and HIF-alpha fragments revealed that FIH is the enzyme that hydroxylates the CAD asparagine residue, that the activity is directly inhibited by cobalt(II) and limited by hypoxia, and that the oxygen in the alcohol of the hydroxyasparagine residue is directly derived from dioxygen. Sequence analyses involving FIH link the 2-OG oxygenases with members of the cupin superfamily, including Zn(II)-utilizing phosphomannose isomerase, revealing structural and evolutionary links between these metal-binding proteins that share common motifs.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="110442" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1074/jbc.M110850200" data-uuid="4d23fde7-4636-b431-7ed3-d4843db509ab"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=3346493" style="display:inline-block;">
  <img alt="Article has an altmetric score of 3" src="https://badges.altmetric.com/?size=128&amp;score=3&amp;types=wwwwwwww" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">64</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/11744733#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Bragança J</span><span class="author ">, Swingler T</span><span class="author ">, Marques FI</span><span class="author ">, Jones T</span><span class="author ">, Eloranta JJ</span><span class="author ">, Hurst HC</span><span class="author ">, Shioda T</span><span class="author ">, Bhattacharya S</span>.</span>		2002.		Human CREB-binding protein/p300-interacting transactivator with ED-rich tail (CITED) 4, a new member of the CITED family, functions as a co-activator for transcription factor AP-2.		<span class="journal-title">J Biol Chem</span>,		<span class="journal-volume">277</span>		(10),		pp. 8559-8565.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1074/jbc.M110850200" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-110442" class="pub_abstract" style="display: none;">
		Members of the CREB-binding protein/p300-interacting transactivator with ED-rich tail (CITED) family bind CREB-binding protein and p300 with high affinity and regulate gene transcription. Gene knockout studies indicate that CITED2 is required for neural crest and neural tube development and that it functions as a co-activator for transcription factor AP-2 (TFAP2). Here we describe human CITED4, a new member of this family, which is encoded by a single exon mapping to chromosome 1p34--1p35. CITED4 and p300/CREB-binding protein are present in endogenous naturally occurring complexes, indicating that they interact physiologically. The interaction occurs between the cysteine-histidine-rich domain 1 of p300 and the carboxyl terminus of CITED4. In keeping with this, CITED4 functions as a transactivator when artificially targeted to a promoter element. CITED4 physically interacts with all TFAP2 isoforms in vitro and strongly co-activates all TFAP2 isoforms in Hep3B cells. Co-activation of TFAP2 requires amino-terminal and carboxyl-terminal residues of CITED4. In HepG2 cells, CITED4 is significantly weaker than CITED2 for TFAP2C co-activation. These results suggest that CITED4 may function as a co-activator for TFAP2. They also suggest the existence of cell type- and TFAP2 isoform-specific co-activation by CITED2 and CITED4, which may result in differential modulation of TFAP2 function.		</p>
				</div></div>
						<div class="publication group all" symplectic_id="110441" style="display: none;"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1038/ng768" data-uuid="dede9cba-cc8b-088c-1d4b-f6d030e63ddb"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=166191" style="display:inline-block;">
  <img alt="Article has an altmetric score of 1" src="https://badges.altmetric.com/?size=128&amp;score=1&amp;types=11111111" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">211</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/11694877#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Bamforth SD</span><span class="author ">, Bragança J</span><span class="author ">, Eloranta JJ</span><span class="author ">, Murdoch JN</span><span class="author ">, Marques FI</span><span class="author ">, Kranc KR</span><span class="author ">, Farza H</span><span class="author ">, Henderson DJ</span><span class="author ">, Hurst HC</span><span class="author ">, Bhattacharya S</span>.</span>		2001.		Cardiac malformations, adrenal agenesis, neural crest defects and exencephaly in mice lacking Cited2, a new Tfap2 co-activator.		<span class="journal-title">Nat Genet</span>,		<span class="journal-volume">29</span>		(4),		pp. 469-474.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a href="http://dx.doi.org/10.1038/ng768" target="_blank">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-110441" class="pub_abstract" style="display: none;">
		The protein EP300 and its paralog CREBBP (CREB-binding protein) are ubiquitously expressed transcriptional co-activators and histone acetyl transferases. The gene EP300 is essential for normal cardiac and neural development, whereas CREBBP is essential for neurulation, hematopoietic differentiation, angiogenesis and skeletal and cardiac development. Mutations in CREBBP cause Rubinstein-Taybi syndrome, which is characterized by mental retardation, skeletal abnormalities and congenital cardiac defects. The CBP/p300-interacting transactivator with ED-rich tail 2 (CITED2) binds EP300 and CREBBP with high affinity and regulates gene transcription. Here we show that Cited2-/- embryos die with cardiac malformations, adrenal agenesis, abnormal cranial ganglia and exencephaly. The cardiac defects include atrial and ventricular septal defects, overriding aorta, double-outlet right ventricle, persistent truncus arteriosus and right-sided aortic arches. We find increased apoptosis in the midbrain region and a marked reduction in ErbB3-expressing neural crest cells in mid-embryogenesis. We show that CITED2 interacts with and co-activates all isoforms of transcription factor AP-2 (TFAP2). Transactivation by TFAP2 isoforms is defective in Cited2-/- embryonic fibroblasts and is rescued by ectopically expressed CITED2. As certain Tfap2 isoforms are essential in neural crest, neural tube and cardiac development, we propose that abnormal embryogenesis in mice lacking Cited2 results, at least in part, from its role as a Tfap2 co-activator.		</p>
				</div></div>
								<div class="publication group favourite" symplectic_id="311336"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1093/eurheartj/ehs056" data-uuid="9ac5348f-4a06-7c19-2735-85a9fd81b49c"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=700651" style="display:inline-block;">
  <img alt="Article has an altmetric score of 1" src="https://badges.altmetric.com/?size=128&amp;score=1&amp;types=tttttttt" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<span class="journal-authors"><span class="author ">MacDonald ST</span><span class="author ">, Bamforth SD</span><span class="author ">, Bragança J</span><span class="author ">, Chen CM</span><span class="author ">, Broadbent C</span><span class="author ">, Schneider JE</span><span class="author ">, Schwartz RJ</span><span class="author ">, Bhattacharya S</span>.</span>		2013.		A cell-autonomous role of Cited2 in controlling myocardial and coronary vascular development.		<span class="journal-title">Eur Heart J</span>,		<span class="journal-volume">34</span>		(32),		pp. 2557-2565.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a target="_blank" href="http://dx.doi.org/10.1093/eurheartj/ehs056">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-311336" class="pub_abstract" style="display: none;">
		AIMS: Myocardial development is dependent on concomitant growth of cardiomyocytes and a supporting vascular network. The coupling of myocardial and coronary vascular development is partly mediated by vascular endothelial growth factor (VEGFA) signalling and additional unknown mechanisms. We examined the cardiomyocyte specific role of the transcriptional co-activator Cited2 on myocardial microstructure and vessel growth, in relation to Vegfa expression. METHODS AND RESULTS: A cardiomyocyte-specific knockout of mouse Cited2 (Cited2(Nkx)) was analysed using magnetic resonance imaging and histology. Ventricular septal defects and significant compact layer thinning (P &lt; 0.02 at right ventricular apex, P &lt; 0.009 at the left ventricular apex in Cited2(Nkx) vs. controls, n = 11 vs. n = 7, respectively) were found. This was associated with a significant decrease in the number of capillaries to larger vessels (ratio 1.56 ± 0.56 vs. 3.25 ± 1.63, P = 2.7 × 10(-6) Cited2(Nkx) vs. controls, n = 11 vs. n = 7, respectively) concomitant with a 1.5-fold reduction in Vegfa expression (P &lt; 0.02, Cited2(Nkx) vs. controls, n = 12 vs. n = 12, respectively). CITED2 was subsequently found at the Vegfa promoter in mouse embryonic hearts using chromatin immunoprecipitation, and moreover found to stimulate human VEGFA promoter activity in cooperation with TFAP2 transcription factors in transient transfection assays. There was no change in the myocardial expression of the left-right patterning gene Pitx2c, a previously known target of CITED2. CONCLUSIONS: This study delineates a novel cell-autonomous role of Cited2 in regulating VEGFA transcription and the development of myocardium and coronary vasculature in the mouse. We suggest that coupling of myocardial and coronary growth in the developing heart may occur in part through a Cited2→Vegfa pathway.		</p>
				</div></div>
						<div class="publication group favourite" symplectic_id="105186"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1093/hmg/ddq251" data-uuid="708d7286-6989-a29a-13f3-c32028a9c7a1"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1093/hmg/ddq251" target="_blank"> </a></div>				<span class="journal-authors"><span class="author ">Bentham J</span><span class="author ">, Michell AC</span><span class="author ">, Lockstone H</span><span class="author ">, Andrew D</span><span class="author ">, Schneider JE</span><span class="author ">, Brown NA</span><span class="author ">, Bhattacharya S</span>.</span>		2010.		Maternal high-fat diet interacts with embryonic Cited2 genotype to reduce Pitx2c expression and enhance penetrance of left-right patterning defects.		<span class="journal-title">Hum Mol Genet</span>,		<span class="journal-volume">19</span>		(17),		pp. 3394-3401.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a target="_blank" href="http://dx.doi.org/10.1093/hmg/ddq251">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-105186" class="pub_abstract" style="display: none;">
		Deficiency of the transcription factor Cited2 in mice results in cardiac malformation, adrenal agenesis, neural tube, placental defects and partially penetrant cardiopulmonary laterality defects resulting from an abnormal Nodal-&gt;Pitx2c pathway. Here we show that a maternal high-fat diet more than doubles the penetrance of laterality defects and, surprisingly, induces palatal clefting in Cited2-deficient embryos. Both maternal diet and Cited2 deletion reduce embryo weight and kidney and thymus volume. Expression profiling identified 40 embryonic transcripts including Pitx2 that were significantly affected by embryonic genotype-maternal diet interaction. We show that a high-fat diet reduces Pitx2c levels &gt;2-fold in Cited2-deficient embryos. Taken together, these results define a novel interaction between maternal high-fat diet and embryonic Cited2 deficiency that affects Pitx2c expression and results in abnormal laterality. They suggest that appropriate modifications of maternal diet may prevent such defects in humans.		</p>
				</div></div>
						<div class="publication group favourite" symplectic_id="71830"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1101/gad.479408" data-uuid="0d30555f-4c58-796d-7651-f6d3a8e57ef8"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1101/gad.479408" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">72</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/18519639#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Szumska D</span><span class="author ">, Pieles G</span><span class="author ">, Essalmani R</span><span class="author ">, Bilski M</span><span class="author ">, Mesnard D</span><span class="author ">, Kaur K</span><span class="author ">, Franklyn A</span><span class="author ">, El Omari K</span><span class="author ">, Jefferis J</span><span class="author ">, Bentham J</span><span class="author hidden" style="display: none;">, Taylor JM</span><span class="author hidden" style="display: none;">, Schneider JE</span><span class="author hidden" style="display: none;">, Arnold SJ</span><span class="author hidden" style="display: none;">, Johnson P</span><span class="author hidden" style="display: none;">, Tymowska-Lalanne Z</span><span class="author hidden" style="display: none;">, Stammers D</span><span class="author hidden" style="display: none;">, Clarke K</span><span class="author hidden" style="display: none;">, Neubauer S</span><span class="author hidden" style="display: none;">, Morris A</span><span class="author hidden" style="display: none;">, Brown SD</span><span class="author hidden" style="display: none;">, Shaw-Smith C</span><span class="author hidden" style="display: none;">, Cama A</span><span class="author hidden" style="display: none;">, Capra V</span><span class="author hidden" style="display: none;">, Ragoussis J</span><span class="author hidden" style="display: none;">, Constam D</span><span class="author hidden" style="display: none;">, Seidah NG</span><span class="author hidden" style="display: none;">, Prat A</span><span class="author hidden" style="display: none;">, Bhattacharya S</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2008.		VACTERL/caudal regression/Currarino syndrome-like malformations in mice with mutation in the proprotein convertase Pcsk5.		<span class="journal-title">Genes Dev</span>,		<span class="journal-volume">22</span>		(11),		pp. 1465-1477.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a target="_blank" href="http://dx.doi.org/10.1101/gad.479408">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-71830" class="pub_abstract" style="display: none;">
		We have identified an ethylnitrosourea (ENU)-induced recessive mouse mutation (Vcc) with a pleiotropic phenotype that includes cardiac, tracheoesophageal, anorectal, anteroposterior patterning defects, exomphalos, hindlimb hypoplasia, a presacral mass, renal and palatal agenesis, and pulmonary hypoplasia. It results from a C470R mutation in the proprotein convertase PCSK5 (PC5/6). Compound mutants (Pcsk5(Vcc/null)) completely recapitulate the Pcsk5(Vcc/Vcc) phenotype, as does an epiblast-specific conditional deletion of Pcsk5. The C470R mutation ablates a disulfide bond in the P domain, and blocks export from the endoplasmic reticulum and proprotein convertase activity. We show that GDF11 is cleaved and activated by PCSK5A, but not by PCSK5A-C470R, and that Gdf11-deficient embryos, in addition to having anteroposterior patterning defects and renal and palatal agenesis, also have a presacral mass, anorectal malformation, and exomphalos. Pcsk5 mutation results in abnormal expression of several paralogous Hox genes (Hoxa, Hoxc, and Hoxd), and of Mnx1 (Hlxb9). These include known Gdf11 targets, and are necessary for caudal embryo development. We identified nonsynonymous mutations in PCSK5 in patients with VACTERL (vertebral, anorectal, cardiac, tracheoesophageal, renal, limb malformation OMIM 192350) and caudal regression syndrome, the phenotypic features of which resemble the mouse mutation. We propose that Pcsk5, at least in part via GDF11, coordinately regulates caudal Hox paralogs, to control anteroposterior patterning, nephrogenesis, skeletal, and anorectal development.		</p>
				</div></div>
						<div class="publication group favourite" symplectic_id="104860"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1038/ng1446" data-uuid="582f1776-9703-9101-6f46-e074c95d3bb9"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=3502041" style="display:inline-block;">
  <img alt="Article has an altmetric score of 3" src="https://badges.altmetric.com/?size=128&amp;score=3&amp;types=wwwwwwww" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">128</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/15475956#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Bamforth SD</span><span class="author ">, Bragança J</span><span class="author ">, Farthing CR</span><span class="author ">, Schneider JE</span><span class="author ">, Broadbent C</span><span class="author ">, Michell AC</span><span class="author ">, Clarke K</span><span class="author ">, Neubauer S</span><span class="author ">, Norris D</span><span class="author ">, Brown NA</span><span class="author hidden" style="display: none;">, Anderson RH</span><span class="author hidden" style="display: none;">, Bhattacharya S</span> <a class="et-al" href="#" style="display: inline;">et al</a>.</span>		2004.		Cited2 controls left-right patterning and heart development through a Nodal-Pitx2c pathway.		<span class="journal-title">Nat Genet</span>,		<span class="journal-volume">36</span>		(11),		pp. 1189-1196.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a target="_blank" href="http://dx.doi.org/10.1038/ng1446">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-104860" class="pub_abstract" style="display: none;">
		Malformations of the septum, outflow tract and aortic arch are the most common congenital cardiovascular defects and occur in mice lacking Cited2, a transcriptional coactivator of TFAP2. Here we show that Cited2(-/-) mice also develop laterality defects, including right isomerism, abnormal cardiac looping and hyposplenia, which are suppressed on a mixed genetic background. Cited2(-/-) mice lack expression of the Nodal target genes Pitx2c, Nodal and Ebaf in the left lateral plate mesoderm, where they are required for establishing laterality and cardiovascular development. CITED2 and TFAP2 were detected at the Pitx2c promoter in embryonic hearts, and they activate Pitx2c transcription in transient transfection assays. We propose that an abnormal Nodal-Pitx2c pathway represents a unifying mechanism for the cardiovascular malformations observed in Cited2(-/-) mice, and that such malformations may be the sole manifestation of a laterality defect.		</p>
				</div></div>
						<div class="publication group favourite" symplectic_id="16766"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1128/MCB.23.21.7658-7666.2003" data-uuid="9d22ca78-e791-9e90-b487-b6756b965b64"><a href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;doi=10.1128/mcb.23.21.7658-7666.2003" target="_blank"> </a></div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">64</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/14560011#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Kranc KR</span><span class="author ">, Bamforth SD</span><span class="author ">, Bragança J</span><span class="author ">, Norbury C</span><span class="author ">, van Lohuizen M</span><span class="author ">, Bhattacharya S</span>.</span>		2003.		Transcriptional coactivator Cited2 induces Bmi1 and Mel18 and controls fibroblast proliferation via Ink4a/ARF.		<span class="journal-title">Mol Cell Biol</span>,		<span class="journal-volume">23</span>		(21),		pp. 7658-7666.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a target="_blank" href="http://dx.doi.org/10.1128/MCB.23.21.7658-7666.2003">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-16766" class="pub_abstract" style="display: none;">
		Cited2 (CBP/p300 interacting transactivator with ED-rich tail 2) is required for embryonic development, coactivation of transcription factor AP-2, and inhibition of hypoxia-inducible factor 1 transactivation. Cited2 is induced by multiple growth factors and cytokines and oncogenically transforms cells. Here, we show that the proliferation of Cited2(-/-) mouse embryonic fibroblasts ceases prematurely. This is associated with a reduction in growth fraction, senescent cellular morphology, and increased expression of the cell proliferation inhibitors p16(INK4a), p19(ARF), and p15(INK4b). Deletion of INK4a/ARF (encoding p16(INK4a) and p19(ARF)) completely rescued the defective proliferation of Cited2(-/-) fibroblasts. However, the deletion of INK4a/ARF did not rescue the embryonic malformations observed in Cited2(-/-) mice, indicating that INK4a/ARF-independent pathways are likely to be involved here. We found that Cited2(-/-) fibroblasts had reduced expression of the polycomb-group genes Bmi1 and Mel18, which function as INK4a/ARF and Hox repressors. Complementation with CITED2-expressing retrovirus enhanced proliferation, induced Bmi1/Mel18 expression, and decreased INK4a/ARF expression. Bmi1- and Mel18-expressing retroviruses enhanced the proliferation of Cited2(-/-) fibroblasts, indicating that they function downstream of Cited2. Our results provide genetic evidence that Cited2 controls the expression of INK4a/ARF and fibroblast proliferation, at least in part via the polycomb-group genes Bmi1 and Mel18.		</p>
				</div></div>
						<div class="publication group favourite" symplectic_id="110443"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1074/jbc.M208144200" data-uuid="ee448e39-7568-4428-c438-eef639b96c2d"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=3346489" style="display:inline-block;">
  <img alt="Article has an altmetric score of 3" src="https://badges.altmetric.com/?size=128&amp;score=3&amp;types=wwwwwwww" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">106</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/12586840#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Bragança J</span><span class="author ">, Eloranta JJ</span><span class="author ">, Bamforth SD</span><span class="author ">, Ibbitt JC</span><span class="author ">, Hurst HC</span><span class="author ">, Bhattacharya S</span>.</span>		2003.		Physical and functional interactions among AP-2 transcription factors, p300/CREB-binding protein, and CITED2.		<span class="journal-title">J Biol Chem</span>,		<span class="journal-volume">278</span>		(18),		pp. 16021-16029.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a target="_blank" href="http://dx.doi.org/10.1074/jbc.M208144200">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-110443" class="pub_abstract" style="display: none;">
		The transcriptional co-activators and histone acetyltransferases p300/CREB-binding protein (CBP) interact with CITED2, a transcription factor AP-2 (TFAP2) co-activator. p300/CBP, CITED2, and TFAP2A are essential for normal neural tube and cardiac development. Here we show that p300 and CBP co-activate TFAP2A in the presence of CITED2. TFAP2A transcriptional activity was modestly impaired in p300(+/-) and CBP(+/-) mouse embryonic fibroblasts; this was rescued by ectopic expression of p300/CBP. p300, TFAP2A, and endogenous CITED2 could be co-immunoprecipitated from transfected U2-OS cells indicating that they can interact physically in vivo. CITED2 interacted with the dimerization domain of TFAP2C, which is highly conserved in TFAP2A/B. In mammalian two-hybrid experiments, full-length p300 and TFAP2A interacted only when CITED2 was co-transfected. N-terminal residues of TFAP2A, containing the transactivation domain, are both necessary and sufficient for interaction with p300, and this interaction was independent of CITED2. Consistent with this, N-terminal residues of TFAP2A were required for p300- and CITED2-dependent co-activation. A histone acetyltransferase-deficient p300 mutant (D1399Y) did not co-activate TFAP2A and did not affect the expression or cellular localization of TFAP2A or CITED2. In mammalian two-hybrid experiments p300D1399Y failed to interact with TFAP2A, explaining, at least in part, its failure to function as a co-activator. Our results suggest a model wherein interactions among TFAP2A, CITED2, and p300/CBP are necessary for TFAP2A-mediated transcriptional activation and for normal neural tube and cardiac development.		</p>
				</div></div>
						<div class="publication group favourite" symplectic_id="110442"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1074/jbc.M110850200" data-uuid="099c60d0-3619-7e2f-a24c-41b7f138b413"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=3346493" style="display:inline-block;">
  <img alt="Article has an altmetric score of 3" src="https://badges.altmetric.com/?size=128&amp;score=3&amp;types=wwwwwwww" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">64</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/11744733#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Bragança J</span><span class="author ">, Swingler T</span><span class="author ">, Marques FI</span><span class="author ">, Jones T</span><span class="author ">, Eloranta JJ</span><span class="author ">, Hurst HC</span><span class="author ">, Shioda T</span><span class="author ">, Bhattacharya S</span>.</span>		2002.		Human CREB-binding protein/p300-interacting transactivator with ED-rich tail (CITED) 4, a new member of the CITED family, functions as a co-activator for transcription factor AP-2.		<span class="journal-title">J Biol Chem</span>,		<span class="journal-volume">277</span>		(10),		pp. 8559-8565.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a target="_blank" href="http://dx.doi.org/10.1074/jbc.M110850200">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-110442" class="pub_abstract" style="display: none;">
		Members of the CREB-binding protein/p300-interacting transactivator with ED-rich tail (CITED) family bind CREB-binding protein and p300 with high affinity and regulate gene transcription. Gene knockout studies indicate that CITED2 is required for neural crest and neural tube development and that it functions as a co-activator for transcription factor AP-2 (TFAP2). Here we describe human CITED4, a new member of this family, which is encoded by a single exon mapping to chromosome 1p34--1p35. CITED4 and p300/CREB-binding protein are present in endogenous naturally occurring complexes, indicating that they interact physiologically. The interaction occurs between the cysteine-histidine-rich domain 1 of p300 and the carboxyl terminus of CITED4. In keeping with this, CITED4 functions as a transactivator when artificially targeted to a promoter element. CITED4 physically interacts with all TFAP2 isoforms in vitro and strongly co-activates all TFAP2 isoforms in Hep3B cells. Co-activation of TFAP2 requires amino-terminal and carboxyl-terminal residues of CITED4. In HepG2 cells, CITED4 is significantly weaker than CITED2 for TFAP2C co-activation. These results suggest that CITED4 may function as a co-activator for TFAP2. They also suggest the existence of cell type- and TFAP2 isoform-specific co-activation by CITED2 and CITED4, which may result in differential modulation of TFAP2 function.		</p>
				</div></div>
						<div class="publication group favourite" symplectic_id="110441"><div>
		<div class="altmetric-embed" data-hide-no-mentions="true" data-link-target="_blank" data-badge-type="donut" data-doi="http://dx.doi.org/10.1038/ng768" data-uuid="e8f12b11-c741-45b9-4d30-6d11fd8afdbd"><a target="_blank" href="http://www.altmetric.com/details.php?domain=www.cardiov.ox.ac.uk&amp;citation_id=166191" style="display:inline-block;">
  <img alt="Article has an altmetric score of 1" src="https://badges.altmetric.com/?size=128&amp;score=1&amp;types=11111111" style="border: 0px none; margin: 0px; max-width: none; display: none;" width="64px" height="64px">
</a>
</div>				<div class="citation_box">
		<div class="citation_count" title="Citations count from Scopus"><p class="cited">Cited:</p><p class="score">211</p><p class="cited">Scopus</p>
				<div class="citation_link">
		<p>
		<a target="_blank" href="http://europepmc.org/abstract/MED/11694877#fragment-related-citations">Citation information in Europe Pubmed Central</a>
		</p>
		</div>
				</div>
		</div>
				<span class="journal-authors"><span class="author ">Bamforth SD</span><span class="author ">, Bragança J</span><span class="author ">, Eloranta JJ</span><span class="author ">, Murdoch JN</span><span class="author ">, Marques FI</span><span class="author ">, Kranc KR</span><span class="author ">, Farza H</span><span class="author ">, Henderson DJ</span><span class="author ">, Hurst HC</span><span class="author ">, Bhattacharya S</span>.</span>		2001.		Cardiac malformations, adrenal agenesis, neural crest defects and exencephaly in mice lacking Cited2, a new Tfap2 co-activator.		<span class="journal-title">Nat Genet</span>,		<span class="journal-volume">29</span>		(4),		pp. 469-474.		 | <a class="show_abstract" href="#">Show Abstract</a> 		 | <a target="_blank" href="http://dx.doi.org/10.1038/ng768">Read more</a> 		<p class="clearfix"></p>
				<p id="abstract-110441" class="pub_abstract" style="display: none;">
		The protein EP300 and its paralog CREBBP (CREB-binding protein) are ubiquitously expressed transcriptional co-activators and histone acetyl transferases. The gene EP300 is essential for normal cardiac and neural development, whereas CREBBP is essential for neurulation, hematopoietic differentiation, angiogenesis and skeletal and cardiac development. Mutations in CREBBP cause Rubinstein-Taybi syndrome, which is characterized by mental retardation, skeletal abnormalities and congenital cardiac defects. The CBP/p300-interacting transactivator with ED-rich tail 2 (CITED2) binds EP300 and CREBBP with high affinity and regulates gene transcription. Here we show that Cited2-/- embryos die with cardiac malformations, adrenal agenesis, abnormal cranial ganglia and exencephaly. The cardiac defects include atrial and ventricular septal defects, overriding aorta, double-outlet right ventricle, persistent truncus arteriosus and right-sided aortic arches. We find increased apoptosis in the midbrain region and a marked reduction in ErbB3-expressing neural crest cells in mid-embryogenesis. We show that CITED2 interacts with and co-activates all isoforms of transcription factor AP-2 (TFAP2). Transactivation by TFAP2 isoforms is defective in Cited2-/- embryonic fibroblasts and is rescued by ectopically expressed CITED2. As certain Tfap2 isoforms are essential in neural crest, neural tube and cardiac development, we propose that abnormal embryogenesis in mice lacking Cited2 results, at least in part, from its role as a Tfap2 co-activator.		</p>
				</div></div>
					</div>
		</div>
<span id="js_markers" style="display:none">[["51.7642677"," -1.219434","&lt;div class='collaborator'&gt;&lt;h2&gt;Prof Keith Channon FMedSci FRCP&lt;\/h2&gt;&lt;p&gt;Oxford University, John Radcliffe Hospital&lt;\/p&gt;&lt;\/div&gt;"],["51.7651595","-1.2217654","&lt;div class='collaborator'&gt;&lt;h2&gt;Prof Hugh Watkins FRS FMedSci&lt;\/h2&gt;&lt;p&gt;Oxford University, West Wing, John Radcliffe Hospital&lt;\/p&gt;&lt;\/div&gt;"],["51.75179401","-1.21583361","&lt;div class='collaborator'&gt;&lt;h2&gt;Dr Ben Davies&lt;\/h2&gt;&lt;p&gt;Oxford University, Henry Wellcome Building of Genomic Medicine&lt;\/p&gt;&lt;\/div&gt;&lt;div class='collaborator'&gt;&lt;h2&gt;Prof Martin Farrall&lt;\/h2&gt;&lt;p&gt;Oxford University, Henry Wellcome Building of Genomic Medicine&lt;\/p&gt;&lt;\/div&gt;&lt;div class='collaborator'&gt;&lt;h2&gt;Prof Jurgen E Schneider&lt;\/h2&gt;&lt;p&gt;Oxford University, Henry Wellcome Building of Genomic Medicine&lt;\/p&gt;&lt;\/div&gt;"],["51.7529958","-1.2155378","&lt;div class='collaborator'&gt;&lt;h2&gt;Professor Paul Brennan&lt;\/h2&gt;&lt;p&gt;Oxford University, NDM Research Building&lt;\/p&gt;&lt;\/div&gt;&lt;div class='collaborator'&gt;&lt;h2&gt;Professor Stefan Knapp&lt;\/h2&gt;&lt;p&gt;Oxford University, NDM Research Building&lt;\/p&gt;&lt;\/div&gt;"]]</span>
<div id="side-bar">
	<div class="side-box">
		<h3>Profile</h3>
		<a class="lightbox" title="" href="http://tools.ndm.ox.ac.uk/pinfox/importers/pinfox/RDM/images/bhattacharya_fit_900x600.jpeg"><img id="pi-photo" src="http://tools.ndm.ox.ac.uk/pinfox/importers/pinfox/RDM/images/bhattacharya_fit_204x204.jpeg" alt="Prof Shoumo Bhattacharya FMedSci FRCP"></a>		<table>
			<tbody><tr>
				<th>Name:</th>
								<td><a class="email" href="mailto:sbhattac@well.ox.ac.uk">Prof Shoumo Bhattacharya FMedSci FRCP</a>&nbsp;&nbsp;<a class="email" href="mailto:sbhattac@well.ox.ac.uk"><img class="mail_icon" src="/_static/website/images/icons/icon-email.gif"></a></td>
							</tr>
									<tr>
				<th>Titles:</th>
				<td>BHF Chair of Cardiovascular Medicine, Honorary Consultant Cardiologist, Group Head / PI, Consultant Physician, Fellow and Member of congregation</td>
			</tr>
									<tr>
				<th>Location:</th>
				<td>Henry Wellcome Building of Genomic Medicine</td>
			</tr>
									<tr>
				<th>Division:</th>
				<td>Cardiovascular Medicine</td>
			</tr>
									<tr>
				<th>Group:</th>
				<td>Bhattacharya</td>
			</tr>
												<tr>
				<th>College:</th>
								<td><img id="pi-college-logo" src="http://tools.ndm.ox.ac.uk/pinfox/importers/pinfox/RDM/images/greentempletoncollege_fit_30x30.png"><a href="http://www.gtc.ox.ac.uk">Green Templeton College</a></td>
			</tr>
					</tbody></table>
	</div>
</div>
<div id="pi_id" dev="0">1461</div>

<script>
$(document).ready(function(){
		$('a[class="email"]').attr('href','mail'+'to'+':sbhattac'+'@'+'well.ox.ac.uk');
	if( typeof makePiTimeline !== 'undefined' )
	{
		makePiTimeline( 1461, 200 );
	}
	
});
</script>
<script type="text/javascript" src="/_static/website/javascripts/piTimelineNew.js"></script>
<script type="text/javascript" src="/_static/website/javascripts/swfobject.js"></script>
<script type="text/javascript">// <![CDATA[
$(document).ready(function(){
	$('div#register_interest_dialog').hide();
	$('div#register_success_dialog').hide();
	$('div#register_fail_dialog').hide();
	$('div#register_interest img').click(function(event){
		event.preventDefault();
		$('div#register_interest_dialog').dialog();
		$('div#register_interest_dialog').dialog("option", "width", 500);
	});
	$('a#send_reminder').click(function(event){
		event.preventDefault();
		sendReminderEmail();
		$('div#email_reminder_dialog').dialog('close');
	});
	var pieces = window.location.href.split("?");
	if (pieces[1]=='msg=success') {
		$('div#register_success_dialog').dialog({ buttons: [ { text: "Ok", click: function() { $( this ).dialog( "close" ); } } ] });
	}
	else if (pieces[1]=='msg=fail') {
		$('div#register_fail_dialog').dialog({ buttons: [ { text: "Ok", click: function() { $( this ).dialog( "close" ); } } ] });
	}

	$(document).delegate('*[data-toggle="lightbox"]:not([data-gallery="navigateTo"])', 'click', function(event) {
		event.preventDefault();
		return $(this).ekkoLightbox({
			onShown: function() {
			},
			onNavigate: function(direction, itemIndex) {
				if (window.console) {
					return console.log('Navigating '+direction+'. Current item: '+itemIndex);
				}
			},
			class: 'ekko-lightbox-nav-overlay'
		});
	});

	////Programatically call
	//$('#open-image').click(function (e) {
	//	e.preventDefault();
	//	$(this).ekkoLightbox();
	//});

    //var mymap = L.map('collab_map', {
    //    center: [51.505, -0.09],
    //    zoom: 3
    //}).setView([51.505, -0.09], 13);

    var southWest = L.latLng(-90, -195),
	northEast = L.latLng(90, 195),
	bounds = L.latLngBounds(southWest, northEast);
    
    var mymap = L.map('collab_map', {
	center: [21.505, -0.09],
	zoom: 1,
	zoomControl: false,
	maxBounds: bounds,
	attributionControl: false
    });
    
    //L.tileLayer('http://otile1.mqcdn.com/tiles/1.0.0/map/{z}/{x}/{y}.jpg', {
    L.tileLayer('http://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/{z}/{y}/{x}', {
	attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, <a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery � <a href="http://mapbox.com">Mapbox</a>',
	maxZoom: 18,
	minZoom: 1,
	id: 'demiwellard.0873jgj9',
	accessToken: 'pk.eyJ1IjoiZGVtaXdlbGxhcmQiLCJhIjoiY2lvcGtyaGxmMDAwNnc3bHgwNjNwMDBkcyJ9.fjE0ZDukBRTPUubnQw43Bg'
    }).addTo(mymap);
    
    js_markers = JSON.parse($('span#js_markers').html());
    
       //console.log(js_markers);
    var markers = [];
    js_markers.forEach(function(item, index) {
	var popup = L.popup().setContent((decodeHtml(item[2])));
	markers.push (L.marker([item[0], item[1]]).bindPopup(popup));
    });
    //console.log(markers);
    var layer_group = L.layerGroup(markers);
    
    mymap.on('zoomend', checkBounds);
    mymap.on('dragend', checkBounds);
    mymap.on('click', checkBounds);
    
    var markers2 = L.markerClusterGroup({
	    maxClusterRadius: 30,
	    iconCreateFunction: function (cluster) {
		    var markers = cluster.getAllChildMarkers();
		    var n = 0;
		    for (var i = 0; i < markers.length; i++) {
			    n += markers[i].number;
		    }
		    return L.divIcon({ html: n, className: 'mycluster', iconSize: L.point(40, 30) });
	    },
	    //Disable all of the defaults:
	    spiderfyOnMaxZoom: true, showCoverageOnHover: true, zoomToBoundsOnClick: true
    });

    function populate() {
	    js_markers.forEach(function(item, index) {
		var popup = L.popup({'maxHeight' : 150}).setContent((decodeHtml(item[2])));
		var m = L.marker([item[0], item[1]], { title: index }).bindPopup(popup);
		m.number = 1;
		markers2.addLayer(m);
	    });
	//    for (var i = 0; i < 100; i++) {
	//	    var m = L.marker(getRandomLatLng(mymap), { title: i });
	//	    m.number = i;
	//	    markers2.addLayer(m);
	//    }
	    return false;
    }

    populate();
    mymap.addLayer(markers2);

    mymap.addControl(new L.Control.ZoomMin())    

});

function checkBounds(args) {
    //console.log('checkBounds');
    if (args.target._zoom == 1) {
	//args.target.panTo([21.505, -0.09]);
	setTimeout(function(){setPanToCentre(args)}, 100);
        return;
    }
}


function setPanToCentre(args) {
    console.log('centre');
    args.target.panTo([21.505, -0.09]);
}

function decodeHtml(html) {
    var txt = document.createElement("textarea");
    txt.innerHTML = html;
    return txt.value;
}

</script>

		<div id="footer">
			<p>
				© 2017 Radcliffe Department of Medicine
			</p>
			
			<p id="power_and_drive">Driven by <a href="http://www.ndm.ox.ac.uk/operations"><img src="/_static/website/images/newt-favicon.gif" alt="newt"> Newt</a>, Powered by <a href="http://www.ndm.ox.ac.uk/operations"><img src="/_static/website/images/pinfox-favicon.gif" alt="pinfox"> Pinfox</a></p>
			<p>
				<a href="/">Home</a>
				| <a href="/_sitemap">Site Map</a>
				| <a href="/how-to-find-us">Contact</a>
								| <a href="http://www.admin.ox.ac.uk/foi/" target="_blank">Freedom of Information</a>
				| <a href="http://www.admin.ox.ac.uk/dataprotection/privacypolicy/" target="_blank">Privacy Policy</a> 
				| <a href="http://www.ox.ac.uk/copyright.html" target="_blank">Copyright</a>
				| <a href="http://www.ox.ac.uk/accessibility" target="_blank">Accessibility</a>
				| <a href="http://www.medsci.ox.ac.uk/" target="_blank">Medical Sciences Division</a>
				| <a href="http://www.ox.ac.uk/" target="_blank">Oxford University</a>
							</p>
		</div>
	</div>
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-52978141-1', 'auto');
  ga('require', 'displayfeatures');
  ga('send', 'pageview');
</script>

<script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1186/s12861-017-0148-y?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1038/ng.3627?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1038/mt.2016.153?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1242/dev.123935?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1093/hmg/ddw071?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1093/nar/gkv1138?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1038/srep28210?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1002/humu.22901?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1038/nature14962?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1038/ncomms9111?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1186/2046-2530-4-s1-p37?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.7554/elife.06942?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1038/ncomms8074?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1038/gim.2015.60?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1038/ncomms6681?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1016/j.jss.2014.01.015?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1161/circresaha.115.303300?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1016/j.jss.2014.01.015?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1038/ncomms5871?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1371/journal.pone.0089397?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1186/1741-7007-11-107?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1074/jbc.m113.493023?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1161/circgenetics.113.000191?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1038/ng.2637?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1186/1471-2156-14-57?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1038/ng.2637?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1242/dmm.011833?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1242/dmm.011957?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1093/hmg/ddt182?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1093/hmg/dds552?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1016/j.ajhg.2012.08.003?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.2459/jcm.0b013e328356a326?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1002/dvdy.23812?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1007/s00335-012-9398-y?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1161/circgenetics.111.962035?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1093/cvr/cvr314?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1093/eurheartj/ehs056?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1016/j.ajhg.2012.08.003?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1371/journal.pone.0046256?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1128/mcb.06331-11?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1002/prot.23208?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1093/hmg/ddr589?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1002/prot.23208?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1074/jbc.m111.233577?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1186/1471-2105-12-237?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1371/journal.pone.0023376?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1038/ng.825?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1093/hmg/ddq457?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1152/physiolgenomics.00091.2010?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1093/hmg/ddq251?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1002/dvdy.22334?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1002/mrm.22352?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1186/1471-213x-10-54?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1016/j.ydbio.2010.01.020?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1007/s00246-009-9610-3?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1016/b978-0-12-381332-9.00036-0?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1371/journal.pgen.1001000?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1128/mcb.00913-09?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1016/j.stem.2009.11.001?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1002/dvdy.21874?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1371/journal.pone.0004978?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.2337/db07-0337?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1101/gad.479408?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1196/annals.1420.003?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1016/s0378-3782(07)70186-x?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1111/j.1469-7580.2007.00746.x?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1161/01.res.0000259041.37059.8c?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1093/hmg/ddl168?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1042/cs20060003?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1007/s00335-005-0112-1?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1073/pnas.0500584102?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1038/ng1446?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1158/0008-5472.can-04-0708?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1002/bdrc.20017?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1016/j.cell.2004.07.025?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1186/1471-213x-4-16?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1128/mcb.23.21.7658-7666.2003?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1016/s0092-8674(03)00645-7?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1038/nsb0703-501?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1074/jbc.m208144200?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1046/j.1469-7580.2003.00157.x?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1007/s10334-003-0002-z?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1016/s0022-2828(02)00291-2?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1007/s10334-003-0002-z?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1006/geno.2002.7005?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1074/jbc.c200273200?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1074/jbc.m110850200?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1038/ng768?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1093/eurheartj/ehs056?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1093/hmg/ddq251?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1101/gad.479408?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1038/ng1446?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1128/mcb.23.21.7658-7666.2003?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1074/jbc.m208144200?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1074/jbc.m110850200?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1038/ng768?callback=_altmetric.embed_callback&amp;domain=www.cardiov.ox.ac.uk&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-21"></script></body></html>